
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:doi="http://dx.doi.org/"
      lang="en" xml:lang="en"
      itemscope itemtype="http://schema.org/Article"
      class="no-js">



<head prefix="og: http://ogp.me/ns#">
  <title>RNA-Seq Transcriptome Profiling Identifies CRISPLD2 as a Glucocorticoid Responsive Gene that Modulates Cytokine Function in Airway Smooth Muscle Cells</title>







<link rel="stylesheet" href="/plosone/resource/compiled/asset_DGQ6YYL26P3FNOQFYM3ZEASHBC46CKTH.css" />

  <!-- allows for  extra head tags -->


<!-- hello -->
<link rel="stylesheet" type="text/css"
      href="https://fonts.googleapis.com/css?family=Open+Sans:400,400i,600">

<link media="print" rel="stylesheet" type="text/css"  href="/plosone/resource/css/print.css"/>
    <script type="text/javascript">
        var siteUrlPrefix = "/plosone/";
    </script>
<script src="/plosone/resource/compiled/asset_SC5JIUGEUPR4P4P6VBUINUVOVUSU3NRY.js"></script>

  <link rel="shortcut icon" href="/plosone/resource/img/favicon.ico" type="image/x-icon"/>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>






  <link rel="canonical" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0099625" />
  <meta name="description" content="Asthma is a chronic inflammatory respiratory disease that affects over 300 million people worldwide. Glucocorticoids are a mainstay therapy for asthma because they exert anti-inflammatory effects in multiple lung tissues, including the airway smooth muscle (ASM). However, the mechanism by which glucocorticoids suppress inflammation in ASM remains poorly understood. Using RNA-Seq, a high-throughput sequencing method, we characterized transcriptomic changes in four primary human ASM cell lines that were treated with dexamethasone&mdash;a potent synthetic glucocorticoid (1 &micro;M for 18 hours). Based on a Benjamini-Hochberg corrected p-value &lt;0.05, we identified 316 differentially expressed genes, including both well known (DUSP1, KLF15, PER1, TSC22D3) and less investigated (C7, CCDC69, CRISPLD2) glucocorticoid-responsive genes. CRISPLD2, which encodes a secreted protein previously implicated in lung development and endotoxin regulation, was found to have SNPs that were moderately associated with inhaled corticosteroid resistance and bronchodilator response among asthma patients in two previously conducted genome-wide association studies. Quantitative RT-PCR and Western blotting showed that dexamethasone treatment significantly increased CRISPLD2 mRNA and protein expression in ASM cells. CRISPLD2 expression was also induced by the inflammatory cytokine IL1&beta;, and small interfering RNA-mediated knockdown of CRISPLD2 further increased IL1&beta;-induced expression of IL6 and IL8. Our findings offer a comprehensive view of the effect of a glucocorticoid on the ASM transcriptome and identify CRISPLD2 as an asthma pharmacogenetics candidate gene that regulates anti-inflammatory effects of glucocorticoids in the ASM." />

  <meta name="citation_abstract" content="Asthma is a chronic inflammatory respiratory disease that affects over 300 million people worldwide. Glucocorticoids are a mainstay therapy for asthma because they exert anti-inflammatory effects in multiple lung tissues, including the airway smooth muscle (ASM). However, the mechanism by which glucocorticoids suppress inflammation in ASM remains poorly understood. Using RNA-Seq, a high-throughput sequencing method, we characterized transcriptomic changes in four primary human ASM cell lines that were treated with dexamethasone&mdash;a potent synthetic glucocorticoid (1 &micro;M for 18 hours). Based on a Benjamini-Hochberg corrected p-value &lt;0.05, we identified 316 differentially expressed genes, including both well known (DUSP1, KLF15, PER1, TSC22D3) and less investigated (C7, CCDC69, CRISPLD2) glucocorticoid-responsive genes. CRISPLD2, which encodes a secreted protein previously implicated in lung development and endotoxin regulation, was found to have SNPs that were moderately associated with inhaled corticosteroid resistance and bronchodilator response among asthma patients in two previously conducted genome-wide association studies. Quantitative RT-PCR and Western blotting showed that dexamethasone treatment significantly increased CRISPLD2 mRNA and protein expression in ASM cells. CRISPLD2 expression was also induced by the inflammatory cytokine IL1&beta;, and small interfering RNA-mediated knockdown of CRISPLD2 further increased IL1&beta;-induced expression of IL6 and IL8. Our findings offer a comprehensive view of the effect of a glucocorticoid on the ASM transcriptome and identify CRISPLD2 as an asthma pharmacogenetics candidate gene that regulates anti-inflammatory effects of glucocorticoids in the ASM.">


  <meta name="keywords" content="Gene expression,Asthma,Small interfering RNAs,RNA sequencing,Bronchodilators,Cytokines,Inflammation,Transcriptome analysis" />


<meta name="citation_doi" content="10.1371/journal.pone.0099625"/>
<meta name="citation_author" content="Blanca E. Himes"/>
  <meta name="citation_author_institution" content="Channing Division of Network Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, Massachusetts, United States of America"/>
  <meta name="citation_author_institution" content="Partners HealthCare Personalized Medicine, Boston, Massachusetts, United States of America"/>
  <meta name="citation_author_institution" content="Children's Hospital Informatics Program, Boston, Massachusetts, United States of America"/>
<meta name="citation_author" content="Xiaofeng Jiang"/>
  <meta name="citation_author_institution" content="Program in Molecular and Integrative Physiological Sciences, Departments of Environmental Health, and Genetics and Complex Diseases, Harvard School of Public Health, Boston, Massachusetts, United States of America"/>
<meta name="citation_author" content="Peter Wagner"/>
  <meta name="citation_author_institution" content="Program in Molecular and Integrative Physiological Sciences, Departments of Environmental Health, and Genetics and Complex Diseases, Harvard School of Public Health, Boston, Massachusetts, United States of America"/>
<meta name="citation_author" content="Ruoxi Hu"/>
  <meta name="citation_author_institution" content="Program in Molecular and Integrative Physiological Sciences, Departments of Environmental Health, and Genetics and Complex Diseases, Harvard School of Public Health, Boston, Massachusetts, United States of America"/>
<meta name="citation_author" content="Qiyu Wang"/>
  <meta name="citation_author_institution" content="Program in Molecular and Integrative Physiological Sciences, Departments of Environmental Health, and Genetics and Complex Diseases, Harvard School of Public Health, Boston, Massachusetts, United States of America"/>
<meta name="citation_author" content="Barbara Klanderman"/>
  <meta name="citation_author_institution" content="Partners HealthCare Personalized Medicine, Boston, Massachusetts, United States of America"/>
<meta name="citation_author" content="Reid M. Whitaker"/>
  <meta name="citation_author_institution" content="Channing Division of Network Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, Massachusetts, United States of America"/>
<meta name="citation_author" content="Qingling Duan"/>
  <meta name="citation_author_institution" content="Channing Division of Network Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, Massachusetts, United States of America"/>
<meta name="citation_author" content="Jessica Lasky-Su"/>
  <meta name="citation_author_institution" content="Channing Division of Network Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, Massachusetts, United States of America"/>
<meta name="citation_author" content="Christina Nikolos"/>
  <meta name="citation_author_institution" content="Pulmonary, Allergy and Critical Care Division, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America"/>
<meta name="citation_author" content="William Jester"/>
  <meta name="citation_author_institution" content="Pulmonary, Allergy and Critical Care Division, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America"/>
<meta name="citation_author" content="Martin Johnson"/>
  <meta name="citation_author_institution" content="Pulmonary, Allergy and Critical Care Division, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America"/>
<meta name="citation_author" content="Reynold A. Panettieri Jr"/>
  <meta name="citation_author_institution" content="Pulmonary, Allergy and Critical Care Division, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America"/>
<meta name="citation_author" content="Kelan G. Tantisira"/>
  <meta name="citation_author_institution" content="Channing Division of Network Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, Massachusetts, United States of America"/>
<meta name="citation_author" content="Scott T. Weiss"/>
  <meta name="citation_author_institution" content="Channing Division of Network Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, Massachusetts, United States of America"/>
  <meta name="citation_author_institution" content="Partners HealthCare Personalized Medicine, Boston, Massachusetts, United States of America"/>
<meta name="citation_author" content="Quan Lu"/>
  <meta name="citation_author_institution" content="Program in Molecular and Integrative Physiological Sciences, Departments of Environmental Health, and Genetics and Complex Diseases, Harvard School of Public Health, Boston, Massachusetts, United States of America"/>

<meta name="citation_title" content="RNA-Seq Transcriptome Profiling Identifies CRISPLD2 as a Glucocorticoid Responsive Gene that Modulates Cytokine Function in Airway Smooth Muscle Cells"/>
<meta itemprop="name" content="RNA-Seq Transcriptome Profiling Identifies CRISPLD2 as a Glucocorticoid Responsive Gene that Modulates Cytokine Function in Airway Smooth Muscle Cells"/>
<meta name="citation_journal_title" content="PLOS ONE"/>
<meta name="citation_journal_abbrev" content="PLOS ONE"/>
<meta name="citation_date" content="Jun 13, 2014"/>
<meta name="citation_firstpage" content="e99625"/>
<meta name="citation_issue" content="6"/>
<meta name="citation_volume" content="9"/>
<meta name="citation_issn" content="1932-6203"/>
<meta name="citation_publisher" content="Public Library of Science"/>

  <meta name="citation_pdf_url" content="https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0099625&type=printable">

  <meta name="citation_article_type" content="Research Article">

<meta name="dc.identifier" content="10.1371/journal.pone.0099625" />


  <meta name="twitter:card" content="summary" />
  <meta name="twitter:site" content="@plosone"/>
    <meta name="twitter:title" content="RNA-Seq Transcriptome Profiling Identifies CRISPLD2 as a Glucocorticoid Responsive Gene that Modulates Cytokine Function in Airway Smooth Muscle Cells" />
    <meta property="twitter:description" content="Asthma is a chronic inflammatory respiratory disease that affects over 300 million people worldwide. Glucocorticoids are a mainstay therapy for asthma because they exert anti-inflammatory effects in multiple lung tissues, including the airway smooth muscle (ASM). However, the mechanism by which glucocorticoids suppress inflammation in ASM remains poorly understood. Using RNA-Seq, a high-throughput sequencing method, we characterized transcriptomic changes in four primary human ASM cell lines that were treated with dexamethasone&mdash;a potent synthetic glucocorticoid (1 &micro;M for 18 hours). Based on a Benjamini-Hochberg corrected p-value &lt;0.05, we identified 316 differentially expressed genes, including both well known (DUSP1, KLF15, PER1, TSC22D3) and less investigated (C7, CCDC69, CRISPLD2) glucocorticoid-responsive genes. CRISPLD2, which encodes a secreted protein previously implicated in lung development and endotoxin regulation, was found to have SNPs that were moderately associated with inhaled corticosteroid resistance and bronchodilator response among asthma patients in two previously conducted genome-wide association studies. Quantitative RT-PCR and Western blotting showed that dexamethasone treatment significantly increased CRISPLD2 mRNA and protein expression in ASM cells. CRISPLD2 expression was also induced by the inflammatory cytokine IL1&beta;, and small interfering RNA-mediated knockdown of CRISPLD2 further increased IL1&beta;-induced expression of IL6 and IL8. Our findings offer a comprehensive view of the effect of a glucocorticoid on the ASM transcriptome and identify CRISPLD2 as an asthma pharmacogenetics candidate gene that regulates anti-inflammatory effects of glucocorticoids in the ASM." />
    <meta property="twitter:image" content="https://journals.plos.org/plosone/article/figure/image?id=10.1371/journal.pone.0099625.g004&size=inline" />

<meta property="og:type" content="article" />
<meta property="og:url" content="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0099625"/>
<meta property="og:title" content="RNA-Seq Transcriptome Profiling Identifies CRISPLD2 as a Glucocorticoid Responsive Gene that Modulates Cytokine Function in Airway Smooth Muscle Cells"/>
<meta property="og:description" content="Asthma is a chronic inflammatory respiratory disease that affects over 300 million people worldwide. Glucocorticoids are a mainstay therapy for asthma because they exert anti-inflammatory effects in multiple lung tissues, including the airway smooth muscle (ASM). However, the mechanism by which glucocorticoids suppress inflammation in ASM remains poorly understood. Using RNA-Seq, a high-throughput sequencing method, we characterized transcriptomic changes in four primary human ASM cell lines that were treated with dexamethasone&mdash;a potent synthetic glucocorticoid (1 &micro;M for 18 hours). Based on a Benjamini-Hochberg corrected p-value &lt;0.05, we identified 316 differentially expressed genes, including both well known (DUSP1, KLF15, PER1, TSC22D3) and less investigated (C7, CCDC69, CRISPLD2) glucocorticoid-responsive genes. CRISPLD2, which encodes a secreted protein previously implicated in lung development and endotoxin regulation, was found to have SNPs that were moderately associated with inhaled corticosteroid resistance and bronchodilator response among asthma patients in two previously conducted genome-wide association studies. Quantitative RT-PCR and Western blotting showed that dexamethasone treatment significantly increased CRISPLD2 mRNA and protein expression in ASM cells. CRISPLD2 expression was also induced by the inflammatory cytokine IL1&beta;, and small interfering RNA-mediated knockdown of CRISPLD2 further increased IL1&beta;-induced expression of IL6 and IL8. Our findings offer a comprehensive view of the effect of a glucocorticoid on the ASM transcriptome and identify CRISPLD2 as an asthma pharmacogenetics candidate gene that regulates anti-inflammatory effects of glucocorticoids in the ASM."/>
<meta property="og:image" content="https://journals.plos.org/plosone/article/figure/image?id=10.1371/journal.pone.0099625.g004&size=inline"/>


<meta name="citation_reference" content="Akinbami L, Moorman J, Bailey C, Zahran H, King M,  et al .. (2012) Trends in asthma prevalence, health care use, and mortality in the United States, 2001–2010. Hyattsville, MD: National Center for Health Statistics."/>

<meta name="citation_reference" content="Global Initiative for Asthma (GINA). (2012) Global Strategy for Asthma Management and Prevention. Available from:  http://www.ginasthma.org/ ."/>
<meta name="citation_reference" content="citation_title=Antiinflammatory action of glucocorticoids—new mechanisms for old drugs;citation_author=T Rhen;citation_author=JA Cidlowski;citation_journal_title=N Engl J Med;citation_volume=353;citation_number=353;citation_first_page=1711;citation_last_page=1723;citation_publication_date=2005;"/>    
<meta name="citation_reference" content="citation_title=Asthma;citation_author=CH Fanta;citation_journal_title=N Engl J Med;citation_volume=360;citation_number=360;citation_first_page=1002;citation_last_page=1014;citation_publication_date=2009;"/>    
<meta name="citation_reference" content="citation_title=Chaperoning of glucocorticoid receptors;citation_author=WB Pratt;citation_author=Y Morishima;citation_author=M Murphy;citation_author=M Harrell;citation_journal_title=Handb Exp Pharmacol;citation_first_page=111;citation_last_page=138;citation_publication_date=2006;"/>    
<meta name="citation_reference" content="citation_title=Modelling the glucocorticoid receptor and producing therapeutic agents with anti-inflammatory effects but reduced side-effects;citation_author=A McMaster;citation_author=DW Ray;citation_journal_title=Exp Physiol;citation_volume=92;citation_number=92;citation_first_page=299;citation_last_page=309;citation_publication_date=2007;"/>    
<meta name="citation_reference" content="citation_title=Improvement in bronchodilation following deep inspiration after a course of high-dose oral prednisone in asthma;citation_author=AM Slats;citation_author=JK Sont;citation_author=RH van Klink;citation_author=EH Bel;citation_author=PJ Sterk;citation_journal_title=Chest;citation_volume=130;citation_number=130;citation_first_page=58;citation_last_page=65;citation_publication_date=2006;"/>    
<meta name="citation_reference" content="citation_title=Time course of inflammatory and remodeling events in a murine model of asthma: effect of steroid treatment;citation_author=A Trifilieff;citation_author=A El-Hashim;citation_author=C Bertrand;citation_journal_title=Am J Physiol Lung Cell Mol Physiol;citation_volume=279;citation_number=279;citation_first_page=L1120;citation_last_page=1128;citation_publication_date=2000;"/>    
<meta name="citation_reference" content="citation_title=Steroids augment relengthening of contracted airway smooth muscle: potential additional mechanism of benefit in asthma;citation_author=OJ Lakser;citation_author=ML Dowell;citation_author=FL Hoyte;citation_author=B Chen;citation_author=TL Lavoie;citation_journal_title=Eur Respir J;citation_volume=32;citation_number=32;citation_first_page=1224;citation_last_page=1230;citation_publication_date=2008;"/>    
<meta name="citation_reference" content="citation_title=Glucocorticoids inhibit eosinophil responses to granulocyte-macrophage colony-stimulating factor;citation_author=AM Lamas;citation_author=OG Leon;citation_author=RP Schleimer;citation_journal_title=J Immunol;citation_volume=147;citation_number=147;citation_first_page=254;citation_last_page=259;citation_publication_date=1991;"/>    
<meta name="citation_reference" content="citation_title=Glucocorticoids inhibit cytokine-mediated eosinophil survival;citation_author=N Wallen;citation_author=H Kita;citation_author=D Weiler;citation_author=GJ Gleich;citation_journal_title=J Immunol;citation_volume=147;citation_number=147;citation_first_page=3490;citation_last_page=3495;citation_publication_date=1991;"/>    
<meta name="citation_reference" content="citation_title=Genomic determination of the glucocorticoid response reveals unexpected mechanisms of gene regulation;citation_author=TE Reddy;citation_author=F Pauli;citation_author=RO Sprouse;citation_author=NF Neff;citation_author=KM Newberry;citation_journal_title=Genome Res;citation_volume=19;citation_number=19;citation_first_page=2163;citation_last_page=2171;citation_publication_date=2009;"/>    
<meta name="citation_reference" content="citation_title=Regulation of beta-adrenergic responses in airway smooth muscle;citation_author=SA Shore;citation_author=PE Moore;citation_journal_title=Respir Physiol Neurobiol;citation_volume=137;citation_number=137;citation_first_page=179;citation_last_page=195;citation_publication_date=2003;"/>    
<meta name="citation_reference" content="citation_title=Airway smooth muscle in bronchial tone, inflammation, and remodeling: basic knowledge to clinical relevance;citation_author=RA Panettieri Jr;citation_author=MI Kotlikoff;citation_author=WT Gerthoffer;citation_author=MB Hershenson;citation_author=PG Woodruff;citation_journal_title=Am J Respir Crit Care Med;citation_volume=177;citation_number=177;citation_first_page=248;citation_last_page=252;citation_publication_date=2008;"/>    
<meta name="citation_reference" content="citation_title=Beta-adrenergic bronchodilators;citation_author=HS Nelson;citation_journal_title=N Engl J Med;citation_volume=333;citation_number=333;citation_first_page=499;citation_last_page=506;citation_publication_date=1995;"/>    
<meta name="citation_reference" content="citation_title=Bronchial responsiveness to histamine or methacholine in asthma: measurement and clinical significance;citation_author=FE Hargreave;citation_author=G Ryan;citation_author=NC Thomson;citation_author=PM O'Byrne;citation_author=K Latimer;citation_journal_title=J Allergy Clin Immunol;citation_volume=68;citation_number=68;citation_first_page=347;citation_last_page=355;citation_publication_date=1981;"/>    
<meta name="citation_reference" content="citation_title=Expression profiling identifies Klf15 as a glucocorticoid target that regulates airway hyperresponsiveness;citation_author=K Masuno;citation_author=SM Haldar;citation_author=D Jeyaraj;citation_author=CM Mailloux;citation_author=X Huang;citation_journal_title=Am J Respir Cell Mol Biol;citation_volume=45;citation_number=45;citation_first_page=642;citation_last_page=649;citation_publication_date=2011;"/>    
<meta name="citation_reference" content="citation_title=Glucocorticoid- and protein kinase A-dependent transcriptome regulation in airway smooth muscle;citation_author=AM Misior;citation_author=DA Deshpande;citation_author=MJ Loza;citation_author=RM Pascual;citation_author=JD Hipp;citation_journal_title=Am J Respir Cell Mol Biol;citation_volume=41;citation_number=41;citation_first_page=24;citation_last_page=39;citation_publication_date=2009;"/>    
<meta name="citation_reference" content="citation_title=RNA-Seq: a revolutionary tool for transcriptomics;citation_author=Z Wang;citation_author=M Gerstein;citation_author=M Snyder;citation_journal_title=Nat Rev Genet;citation_volume=10;citation_number=10;citation_first_page=57;citation_last_page=63;citation_publication_date=2009;"/>    
<meta name="citation_reference" content="citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation;citation_author=C Trapnell;citation_author=BA Williams;citation_author=G Pertea;citation_author=A Mortazavi;citation_author=G Kwan;citation_journal_title=Nat Biotechnol;citation_volume=28;citation_number=28;citation_first_page=511;citation_last_page=515;citation_publication_date=2010;"/>    
<meta name="citation_reference" content="citation_title=Differential analysis of gene regulation at transcript resolution with RNA-seq;citation_author=C Trapnell;citation_author=DG Hendrickson;citation_author=M Sauvageau;citation_author=L Goff;citation_author=JL Rinn;citation_journal_title=Nat Biotechnol;citation_volume=31;citation_number=31;citation_first_page=46;citation_last_page=53;citation_publication_date=2012;"/>    
<meta name="citation_reference" content="citation_title=RNA-seq: an assessment of technical reproducibility and comparison with gene expression arrays;citation_author=JC Marioni;citation_author=CE Mason;citation_author=SM Mane;citation_author=M Stephens;citation_author=Y Gilad;citation_journal_title=Genome Res;citation_volume=18;citation_number=18;citation_first_page=1509;citation_last_page=1517;citation_publication_date=2008;"/>    
<meta name="citation_reference" content="citation_title=More powerful procedures for multiple significance testing;citation_author=Y Hochberg;citation_author=Y Benjamini;citation_journal_title=Stat Med;citation_volume=9;citation_number=9;citation_first_page=811;citation_last_page=818;citation_publication_date=1990;"/>    
<meta name="citation_reference" content="citation_title=Transcriptional regulation of human dual specificity protein phosphatase 1 (DUSP1) gene by glucocorticoids;citation_author=LE Shipp;citation_author=JV Lee;citation_author=CY Yu;citation_author=M Pufall;citation_author=P Zhang;citation_journal_title=PLoS One;citation_volume=5;citation_number=5;citation_first_page=e13754;citation_publication_date=2010;"/>    
<meta name="citation_reference" content="citation_title=Corticosteroid-induced gene expression in allergen-challenged asthmatic subjects taking inhaled budesonide;citation_author=MM Kelly;citation_author=EM King;citation_author=CF Rider;citation_author=C Gwozd;citation_author=NS Holden;citation_journal_title=Br J Pharmacol;citation_volume=165;citation_number=165;citation_first_page=1737;citation_last_page=1747;citation_publication_date=2012;"/>    
<meta name="citation_reference" content="citation_title=Circadian rhythms in the CNS and peripheral clock disorders: function of clock genes: influence of medication for bronchial asthma on circadian gene;citation_author=N Burioka;citation_author=Y Fukuoka;citation_author=M Takata;citation_author=M Endo;citation_author=M Miyata;citation_journal_title=J Pharmacol Sci;citation_volume=103;citation_number=103;citation_first_page=144;citation_last_page=149;citation_publication_date=2007;"/>    
<meta name="citation_reference" content="citation_title=The anti-inflammatory effect of glucocorticoids is mediated by glucocorticoid-induced leucine zipper in epithelial cells;citation_author=J Eddleston;citation_author=J Herschbach;citation_author=AL Wagelie-Steffen;citation_author=SC Christiansen;citation_author=BL Zuraw;citation_journal_title=J Allergy Clin Immunol;citation_volume=119;citation_number=119;citation_first_page=115;citation_last_page=122;citation_publication_date=2007;"/>    
<meta name="citation_reference" content="citation_title=Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources;citation_author=W Huang da;citation_author=BT Sherman;citation_author=RA Lempicki;citation_journal_title=Nat Protoc;citation_volume=4;citation_number=4;citation_first_page=44;citation_last_page=57;citation_publication_date=2009;"/>    
<meta name="citation_reference" content="citation_title=Long-acting beta2-adrenoceptor agonists synergistically enhance glucocorticoid-dependent transcription in human airway epithelial and smooth muscle cells;citation_author=M Kaur;citation_author=JE Chivers;citation_author=MA Giembycz;citation_author=R Newton;citation_journal_title=Molecular pharmacology;citation_volume=73;citation_number=73;citation_first_page=203;citation_last_page=214;citation_publication_date=2008;"/>    
<meta name="citation_reference" content="citation_title=Genome-Wide Association Analysis in Asthma Subjects Identifies SPATS2L as a Novel Bronchodilator Response Gene;citation_author=BE Himes;citation_author=X Jiang;citation_author=R Hu;citation_author=AC Wu;citation_author=JA Lasky-Su;citation_journal_title=PLoS Genet;citation_volume=8;citation_number=8;citation_first_page=e1002824;citation_publication_date=2012;"/>    
<meta name="citation_reference" content="citation_title=The novel lipopolysaccharide-binding protein CRISPLD2 is a critical serum protein to regulate endotoxin function;citation_author=ZQ Wang;citation_author=WM Xing;citation_author=HH Fan;citation_author=KS Wang;citation_author=HK Zhang;citation_journal_title=J Immunol;citation_volume=183;citation_number=183;citation_first_page=6646;citation_last_page=6656;citation_publication_date=2009;"/>    
<meta name="citation_reference" content="citation_title=Role of IL-6 in asthma and other inflammatory pulmonary diseases;citation_author=M Rincon;citation_author=CG Irvin;citation_journal_title=Int J Biol Sci;citation_volume=8;citation_number=8;citation_first_page=1281;citation_last_page=1290;citation_publication_date=2012;"/>    
<meta name="citation_reference" content="citation_title=Essential involvement of interleukin-8 (IL-8) in acute inflammation;citation_author=A Harada;citation_author=N Sekido;citation_author=T Akahoshi;citation_author=T Wada;citation_author=N Mukaida;citation_journal_title=J Leukoc Biol;citation_volume=56;citation_number=56;citation_first_page=559;citation_last_page=564;citation_publication_date=1994;"/>    
<meta name="citation_reference" content="citation_title=Mechanisms involved in the side effects of glucocorticoids;citation_author=H Schacke;citation_author=WD Docke;citation_author=K Asadullah;citation_journal_title=Pharmacol Ther;citation_volume=96;citation_number=96;citation_first_page=23;citation_last_page=43;citation_publication_date=2002;"/>    

<meta name="citation_reference" content="Thierry-Mieg D, Thierry-Mieg J (2006) AceView: a comprehensive cDNA-supported gene and transcripts annotation. Genome Biol  7  Suppl 1: S12 11–14."/>
<meta name="citation_reference" content="citation_title=Glucocorticoid receptor isoforms generate transcription specificity;citation_author=NZ Lu;citation_author=JA Cidlowski;citation_journal_title=Trends in cell biology;citation_volume=16;citation_number=16;citation_first_page=301;citation_last_page=307;citation_publication_date=2006;"/>    
<meta name="citation_reference" content="citation_title=Cell-specific regulation of PTX3 by glucocorticoid hormones in hematopoietic and nonhematopoietic cells;citation_author=A Doni;citation_author=G Mantovani;citation_author=C Porta;citation_author=J Tuckermann;citation_author=HM Reichardt;citation_journal_title=The Journal of biological chemistry;citation_volume=283;citation_number=283;citation_first_page=29983;citation_last_page=29992;citation_publication_date=2008;"/>    
<meta name="citation_reference" content="citation_title=Phosphodiesterase 4 inhibitors augment the ability of formoterol to enhance glucocorticoid-dependent gene transcription in human airway epithelial cells: a novel mechanism for the clinical efficacy of roflumilast in severe chronic obstructive pulmonary disease;citation_author=T Moodley;citation_author=SM Wilson;citation_author=T Joshi;citation_author=CF Rider;citation_author=P Sharma;citation_journal_title=Molecular pharmacology;citation_volume=83;citation_number=83;citation_first_page=894;citation_last_page=906;citation_publication_date=2013;"/>    
<meta name="citation_reference" content="citation_title=Sequence features and chromatin structure around the genomic regions bound by 119 human transcription factors;citation_author=J Wang;citation_author=J Zhuang;citation_author=S Iyer;citation_author=X Lin;citation_author=TW Whitfield;citation_journal_title=Genome research;citation_volume=22;citation_number=22;citation_first_page=1798;citation_last_page=1812;citation_publication_date=2012;"/>    
<meta name="citation_reference" content="citation_title=Biological role of the CCAAT/enhancer-binding protein family of transcription factors;citation_author=J Lekstrom-Himes;citation_author=KG Xanthopoulos;citation_journal_title=The Journal of biological chemistry;citation_volume=273;citation_number=273;citation_first_page=28545;citation_last_page=28548;citation_publication_date=1998;"/>    
<meta name="citation_reference" content="citation_title=Glucocorticoids regulate the CCSP and CYP2B1 promoters via C/EBPbeta and delta in lung cells;citation_author=T Berg;citation_author=TN Cassel;citation_author=PE Schwarze;citation_author=M Nord;citation_journal_title=Biochemical and biophysical research communications;citation_volume=293;citation_number=293;citation_first_page=907;citation_last_page=912;citation_publication_date=2002;"/>    
<meta name="citation_reference" content="citation_title=Expression and activity of C/EBPbeta and delta are upregulated by dexamethasone in skeletal muscle;citation_author=H Yang;citation_author=J Mammen;citation_author=W Wei;citation_author=M Menconi;citation_author=A Evenson;citation_journal_title=Journal of cellular physiology;citation_volume=204;citation_number=204;citation_first_page=219;citation_last_page=226;citation_publication_date=2005;"/>    
<meta name="citation_reference" content="citation_title=Glucocorticoids increase C/EBPbeta activity in the lung epithelium via phosphorylation;citation_author=T Berg;citation_author=L Didon;citation_author=J Barton;citation_author=O Andersson;citation_author=M Nord;citation_journal_title=Biochemical and biophysical research communications;citation_volume=334;citation_number=334;citation_first_page=638;citation_last_page=645;citation_publication_date=2005;"/>    
<meta name="citation_reference" content="citation_title=A nuclear factor for IL-6 expression (NF-IL6) is a member of a C/EBP family;citation_author=S Akira;citation_author=H Isshiki;citation_author=T Sugita;citation_author=O Tanabe;citation_author=S Kinoshita;citation_journal_title=The EMBO journal;citation_volume=9;citation_number=9;citation_first_page=1897;citation_last_page=1906;citation_publication_date=1990;"/>    
<meta name="citation_reference" content="citation_title=Antisense oligodeoxynucleotides decrease LGL1 mRNA and protein levels and inhibit branching morphogenesis in fetal rat lung;citation_author=L Oyewumi;citation_author=F Kaplan;citation_author=S Gagnon;citation_author=NB Sweezey;citation_journal_title=Am J Respir Cell Mol Biol;citation_volume=28;citation_number=28;citation_first_page=232;citation_last_page=240;citation_publication_date=2003;"/>    
<meta name="citation_reference" content="citation_title=CRISPLD2: a novel NSCLP candidate gene;citation_author=BT Chiquet;citation_author=AC Lidral;citation_author=S Stal;citation_author=JB Mulliken;citation_author=LM Moreno;citation_journal_title=Hum Mol Genet;citation_volume=16;citation_number=16;citation_first_page=2241;citation_last_page=2248;citation_publication_date=2007;"/>    
<meta name="citation_reference" content="citation_title=CRISPLD2 is expressed at low levels during septic shock and is associated with procalcitonin;citation_author=T Wang;citation_author=ZQ Wang;citation_author=L Wang;citation_author=L Yan;citation_author=J Wan;citation_journal_title=PloS one;citation_volume=8;citation_number=8;citation_first_page=e65743;citation_publication_date=2013;"/>    
<meta name="citation_reference" content="citation_title=Systemic steroids in severe sepsis and septic shock;citation_author=GP Patel;citation_author=RA Balk;citation_journal_title=American journal of respiratory and critical care medicine;citation_volume=185;citation_number=185;citation_first_page=133;citation_last_page=139;citation_publication_date=2012;"/>    
<meta name="citation_reference" content="citation_title=Modulation of asthma by endotoxin;citation_author=V Doreswamy;citation_author=DB Peden;citation_journal_title=Clin Exp Allergy;citation_volume=41;citation_number=41;citation_first_page=9;citation_last_page=19;citation_publication_date=2011;"/>    
<meta name="citation_reference" content="citation_title=Lung epithelial-C/EBPbeta contributes to LPS-induced inflammation and its suppression by formoterol;citation_author=AB Roos;citation_author=JL Barton;citation_author=A Miller-Larsson;citation_author=B Dahlberg;citation_author=T Berg;citation_journal_title=Biochemical and biophysical research communications;citation_volume=423;citation_number=423;citation_first_page=134;citation_last_page=139;citation_publication_date=2012;"/>    
<meta name="citation_reference" content="citation_title=Glucocorticoid-induced Changes in Gene Expression of Airway Smooth Muscle in Patients with Asthma;citation_author=CY Yick;citation_author=AH Zwinderman;citation_author=PW Kunst;citation_author=K Grunberg;citation_author=T Mauad;citation_journal_title=Am J Respir Crit Care Med;citation_volume=187;citation_number=187;citation_first_page=1076;citation_last_page=1084;citation_publication_date=2013;"/>    
<meta name="citation_reference" content="citation_title=A human airway smooth muscle cell line that retains physiological responsiveness;citation_author=RA Panettieri;citation_author=RK Murray;citation_author=LR DePalo;citation_author=PA Yadvish;citation_author=MI Kotlikoff;citation_journal_title=Am J Physiol;citation_volume=256;citation_number=256;citation_first_page=C329;citation_last_page=335;citation_publication_date=1989;"/>    
<meta name="citation_reference" content="citation_title=20-HETE mediates ozone-induced, neutrophil-independent airway hyper-responsiveness in mice;citation_author=PR Cooper;citation_author=AC Mesaros;citation_author=J Zhang;citation_author=P Christmas;citation_author=CM Stark;citation_journal_title=PLoS One;citation_volume=5;citation_number=5;citation_first_page=e10235;citation_publication_date=2010;"/>    

<meta name="citation_reference" content="Andrews S FastQC A Quality Control tool for High Throughput Sequence Data."/>
<meta name="citation_reference" content="citation_title=Comparison of DNA sequences with protein sequences;citation_author=WR Pearson;citation_author=T Wood;citation_author=Z Zhang;citation_author=W Miller;citation_journal_title=Genomics;citation_volume=46;citation_number=46;citation_first_page=24;citation_last_page=36;citation_publication_date=1997;"/>    
<meta name="citation_reference" content="citation_title=TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions;citation_author=D Kim;citation_author=G Pertea;citation_author=C Trapnell;citation_author=H Pimentel;citation_author=R Kelley;citation_journal_title=Genome Biol;citation_volume=14;citation_number=14;citation_first_page=R36;citation_publication_date=2013;"/>    
<meta name="citation_reference" content="citation_title=BamTools: a C++ API and toolkit for analyzing and managing BAM files;citation_author=DW Barnett;citation_author=EK Garrison;citation_author=AR Quinlan;citation_author=MP Stromberg;citation_author=GT Marth;citation_journal_title=Bioinformatics;citation_volume=27;citation_number=27;citation_first_page=1691;citation_last_page=1692;citation_publication_date=2011;"/>    
<meta name="citation_reference" content="citation_title=Mapping and quantifying mammalian transcriptomes by RNA-Seq;citation_author=A Mortazavi;citation_author=BA Williams;citation_author=K McCue;citation_author=L Schaeffer;citation_author=B Wold;citation_journal_title=Nat Methods;citation_volume=5;citation_number=5;citation_first_page=621;citation_last_page=628;citation_publication_date=2008;"/>    
<meta name="citation_reference" content="citation_title=Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks;citation_author=C Trapnell;citation_author=A Roberts;citation_author=L Goff;citation_author=G Pertea;citation_author=D Kim;citation_journal_title=Nat Protoc;citation_volume=7;citation_number=7;citation_first_page=562;citation_last_page=578;citation_publication_date=2012;"/>    
<meta name="citation_reference" content="citation_title=The Childhood Asthma Management Program (CAMP): design, rationale, and methods;citation_journal_title=Control Clin Trials;citation_volume=20;citation_number=20;citation_first_page=91;citation_last_page=120;citation_publication_date=1999;"/>    
<meta name="citation_reference" content="citation_title=Randomized comparison of strategies for reducing treatment in mild persistent asthma;citation_author=SP Peters;citation_author=N Anthonisen;citation_author=M Castro;citation_author=JT Holbrook;citation_author=CG Irvin;citation_journal_title=N Engl J Med;citation_volume=356;citation_number=356;citation_first_page=2027;citation_last_page=2039;citation_publication_date=2007;"/>    
<meta name="citation_reference" content="citation_title=The Prevention of Early Asthma in Kids study: design, rationale and methods for the Childhood Asthma Research and Education network;citation_author=TW Guilbert;citation_author=WJ Morgan;citation_author=M Krawiec;citation_author=RF Lemanske Jr;citation_author=C Sorkness;citation_journal_title=Control Clin Trials;citation_volume=25;citation_number=25;citation_first_page=286;citation_last_page=310;citation_publication_date=2004;"/>    

<meta name="citation_reference" content="Denlinger LC, Sorkness CA, Chinchilli VM, Lemanske RF, Jr. (2007) Guideline-defining asthma clinical trials of the National Heart, Lung, and Blood Institute&amp;apos;s Asthma Clinical Research Network and Childhood Asthma Research and Education Network. J Allergy Clin Immunol  119 : : 3–11; quiz 12–13."/>
<meta name="citation_reference" content="citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses;citation_author=S Purcell;citation_author=B Neale;citation_author=K Todd-Brown;citation_author=L Thomas;citation_author=MA Ferreira;citation_journal_title=Am J Hum Genet;citation_volume=81;citation_number=81;citation_first_page=559;citation_last_page=575;citation_publication_date=2007;"/>    

<meta name="citation_reference" content="Smyth GK (2005) Limma: linear models for microarray data. In: Gentleman R, Carey V, Dudoit S, Irizarry R, Huber W, editors. Bioinformatics and Computational Biology Solutions using R and Bioconductor. New York: Springer. pp. 397–420."/>
<meta name="citation_reference" content="citation_title=affy—analysis of Affymetrix GeneChip data at the probe level;citation_author=L Gautier;citation_author=L Cope;citation_author=BM Bolstad;citation_author=RA Irizarry;citation_journal_title=Bioinformatics;citation_volume=20;citation_number=20;citation_first_page=307;citation_last_page=315;citation_publication_date=2004;"/>    
<meta name="citation_reference" content="citation_title=NIH Image to ImageJ: 25 years of image analysis;citation_author=CA Schneider;citation_author=WS Rasband;citation_author=KW Eliceiri;citation_journal_title=Nature methods;citation_volume=9;citation_number=9;citation_first_page=671;citation_last_page=675;citation_publication_date=2012;"/>    
<meta name="citation_reference" content="citation_title=LocusZoom: regional visualization of genome-wide association scan results;citation_author=RJ Pruim;citation_author=RP Welch;citation_author=S Sanna;citation_author=TM Teslovich;citation_author=PS Chines;citation_journal_title=Bioinformatics;citation_volume=26;citation_number=26;citation_first_page=2336;citation_last_page=2337;citation_publication_date=2010;"/>    


<!-- DoubleClick overall ad setup script -->
<script type='text/javascript'>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
  (function() {
    var gads = document.createElement('script');
    gads.async = true;
    gads.type = 'text/javascript';
    var useSSL = 'https:' == document.location.protocol;
    gads.src = (useSSL ? 'https:' : 'http:') +
        '//www.googletagservices.com/tag/js/gpt.js';
    var node = document.getElementsByTagName('script')[0];
    node.parentNode.insertBefore(gads, node);
  })();
</script>

<!-- DoubleClick ad slot setup script -->

  <script id="doubleClickSetupScript" type='text/javascript'>
    googletag.cmd.push(function() {
  googletag.defineSlot('/75507958/PONE_728x90_ATF', [728, 90], 'div-gpt-ad-1458247671871-0').addService(googletag.pubads());
  googletag.defineSlot('/75507958/PONE_160x600_BTF', [160, 600], 'div-gpt-ad-1458247671871-1').addService(googletag.pubads());
      googletag.pubads().enableSingleRequest();
      googletag.enableServices();
    });
  </script>



<script type="text/javascript">
    var WombatConfig = WombatConfig || {};
    WombatConfig.resourcePath = "/plosone/resource/";
    WombatConfig.imgPath = "/plosone/resource/img/";
    WombatConfig.journalKey = "PLoSONE";
    WombatConfig.figurePath = "/plosone/article/figure/image";
    WombatConfig.figShareInstitutionString = "plos";
    WombatConfig.doiResolverPrefix = "http://dx.plos.org/";
</script>

<script type="text/javascript">
  var WombatConfig = WombatConfig || {};
  WombatConfig.metrics = WombatConfig.metrics || {};
  WombatConfig.metrics.referenceUrl      = "http://lagotto.io/plos";
  WombatConfig.metrics.googleScholarUrl  = "https://scholar.google.com/scholar";
  WombatConfig.metrics.googleScholarCitationUrl  = WombatConfig.metrics.googleScholarUrl + "?hl=en&lr=&q=";
  WombatConfig.metrics.crossrefUrl  = "https://www.crossref.org";
</script>
<script src="https://code.jquery.com/jquery-2.1.4.min.js" ></script>
<script>window.jQuery || document.write('<script src="/plosone/resource/js/vendor/jquery-2.1.4.min.js""><\/script>')</script>

          <script type="text/javascript" src="https://widgets.figshare.com/static/figshare.js"></script>










<!--For Google Tag manager to be able to track site information  -->
<script>

  dataLayer = [{
    'mobileSite': 'false',
    'desktopSite': 'true'
  }];

</script>

</head>



<body class="article plosone">


<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-TP26BH"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
    '//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-TP26BH');</script>

<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-MQQMGF"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
    '//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-MQQMGF');</script>
<!-- End Google Tag Manager -->



<header>

  <div id="topslot" class="head-top">

<div class="center">
<div class="title">Advertisement</div>
<!-- DoubleClick Ad Zone -->
  <div class='advertisement' id='div-gpt-ad-1458247671871-0' style='width:728px; height:90px;'>
    <script type='text/javascript'>
      googletag.cmd.push(function() { googletag.display('div-gpt-ad-1458247671871-0'); });
    </script>
  </div>
</div>
  </div>

  <div id="user" class="nav">
    <ul class="nav-user">




      <li ><a href="https://www.plos.org">plos.org</a></li>


        <li ><a href="https://community.plos.org/registration/new">create account</a></li>


          <li class="highlighted"><a href="/plosone/user/secure/login?page=%2Fplosone%2Farticle%3Fid%3D10.1371%2Fjournal.pone.0099625">sign in</a></li>

    </ul>
  </div>
  <div id="pagehdr">

    <nav class="nav-main">




<h1 class="logo">
  <a href="/plosone/.">PLOS ONE</a>
</h1>

<section class="top-bar-section"> 

<ul class="nav-elements">


    <li class="multi-col-parent menu-section-header has-dropdown" id="publish">
    Publish
      <div class="dropdown mega ">
        <ul class="multi-col" id="publish-dropdown-list">

  <li class="menu-section-header " id="submissions">
    <span class="menu-section-header-title">  Submissions </span>

    <ul class="menu-section "
        id="submissions-dropdown-list">
      <li>
    <a href="/plosone/s/getting-started" >Getting Started</a>
  </li>

      <li>
    <a href="/plosone/s/submission-guidelines" >Submission Guidelines</a>
  </li>

      <li>
    <a href="/plosone/s/figures" >Figures</a>
  </li>

      <li>
    <a href="/plosone/s/tables" >Tables</a>
  </li>

      <li>
    <a href="/plosone/s/supporting-information" >Supporting Information</a>
  </li>

      <li>
    <a href="/plosone/s/latex" >LaTeX</a>
  </li>

      <li>
    <a href="/plosone/s/preprints" >Preprints</a>
  </li>

      <li>
    <a href="/plosone/s/revising-your-manuscript" >Revising Your Manuscript</a>
  </li>

      <li>
    <a href="/plosone/s/submit-now" >Submit Now</a>
  </li>

    </ul>

  </li>


  <li class="menu-section-header " id="policies">
    <span class="menu-section-header-title">  Policies </span>

    <ul class="menu-section "
        id="policies-dropdown-list">
      <li>
    <a href="/plosone/s/best-practices-in-research-reporting" >Best Practices in Research Reporting</a>
  </li>

      <li>
    <a href="/plosone/s/human-subjects-research" >Human Subjects Research</a>
  </li>

      <li>
    <a href="/plosone/s/animal-research" >Animal Research</a>
  </li>

      <li>
    <a href="/plosone/s/competing-interests" >Competing Interests</a>
  </li>

      <li>
    <a href="/plosone/s/disclosure-of-funding-sources" >Disclosure of Funding Sources</a>
  </li>

      <li>
    <a href="/plosone/s/licenses-and-copyright" >Licenses and Copyright</a>
  </li>

      <li>
    <a href="/plosone/s/data-availability" >Data Availability</a>
  </li>

      <li>
    <a href="/plosone/s/materials-and-software-sharing" >Materials and Software Sharing</a>
  </li>

      <li>
    <a href="/plosone/s/ethical-publishing-practice" >Ethical Publishing Practice</a>
  </li>

      <li>
    <a href="/plosone/s/authorship" >Authorship</a>
  </li>

      <li>
    <a href="/plosone/s/downloads-and-translations" >Downloads and Translations</a>
  </li>

    </ul>

  </li>


  <li class="menu-section-header " id="manuscript-review-and-publication">
    <span class="menu-section-header-title">  Manuscript Review and Publication </span>

    <ul class="menu-section "
        id="manuscript-review-and-publication-dropdown-list">
      <li>
    <a href="/plosone/s/criteria-for-publication" >Criteria for Publication</a>
  </li>

      <li>
    <a href="/plosone/s/editorial-and-peer-review-process" >Editorial and Peer Review Process</a>
  </li>

      <li>
    <a href="/plosone/s/editor-center" >Editor Center</a>
  </li>

      <li>
    <a href="/plosone/s/reviewer-guidelines" >Guidelines for Reviewers</a>
  </li>

      <li>
    <a href="/plosone/s/accepted-manuscripts" >Accepted Manuscripts</a>
  </li>

      <li>
    <a href="/plosone/s/corrections-and-retractions" >Corrections and Retractions</a>
  </li>

      <li>
    <a href="/plosone/s/comments" >Comments</a>
  </li>

      <li>
    <a href="/plosone/s/article-level-metrics" >Article-Level Metrics</a>
  </li>

    </ul>

  </li>
        </ul>
          <div class="calloutcontainer">



  <h3 class="callout-headline">Submit Your Manuscript</h3>

  <div class="action-contain">
  <p class="callout-content">
  Discover a faster, simpler path to publishing in a high-quality journal. <em>PLOS ONE</em> promises fair, rigorous peer review,
  broad scope, and wide readership – a perfect fit for your research every time.
  </p>

  <p class="button-contain special">
    <a class="button button-default" href="/plosone/static/publish">
     Learn More
    </a>
    <a class="button-link" href="https://www.editorialmanager.com/pone/default.asp">
      Submit Now
    </a>
  </p>
  </div>  <!-- opens in siteMenuCalloutDescription -->


          </div>
      </div>
    </li>



  <li class="menu-section-header has-dropdown " id="about">
    <span class="menu-section-header-title">  About </span>

    <ul class="menu-section dropdown "
        id="about-dropdown-list">
      <li>
    <a href="/plosone/static/publish" >Why Publish with PLOS ONE</a>
  </li>

      <li>
    <a href="/plosone/s/journal-information" >Journal Information</a>
  </li>

      <li>
    <a href="/plosone/s/staff-editors" >Staff Editors</a>
  </li>

      <li>
    <a href="/plosone/static/editorial-board" >Editorial Board</a>
  </li>

      <li>
    <a href="/plosone/s/section-editors" >Section Editors</a>
  </li>

      <li>
    <a href="/plosone/s/advisory-groups" >Advisory Groups</a>
  </li>

      <li>
    <a href="/plosone/s/find-and-read-articles" >Find and Read Articles</a>
  </li>

      <li>
    <a href="/plosone/s/publishing-information" >Publishing Information</a>
  </li>

      <li>
    <a href="/plosone/s/publication-fees" >Publication Fees</a>
  </li>

      <li>
    <a href="/plosone/s/press-and-media" >Press and Media</a>
  </li>

      <li>
    <a href="/plosone/s/contact" >Contact</a>
  </li>

    </ul>

  </li>

  <li data-js-tooltip-hover="trigger" class="subject-area menu-section-header">
  Browse
  </li>

      <li id="navsearch" class="head-search">


    <form name="searchForm" action="/plosone/search" method="get">
      <fieldset>
        <legend>Search</legend>
        <label for="search">Search</label>
        <div class="search-contain">
          <input id="search" type="text" name="q" placeholder="SEARCH" required/>
          <button id="headerSearchButton" type="submit"><span class="search-icon"></span></button>
        </div>
      </fieldset>
      <input type="hidden" name="filterJournals" value="PLoSONE"/>
    </form>

    <a id="advSearch"
       href="/plosone/search">
      advanced search
    </a>




      </li>

      </ul>     
      </section>  
    </nav>
  </div>

</header><section id="taxonomyContainer">

<div id="taxonomy-browser" class="areas" data-search-url="/plosone/browse">
  <div class="wrapper">
    <div class="taxonomy-header">
      Browse Subject Areas
      <div id="subjInfo">?</div>
      <div id="subjInfoText">
        <p>Click through the PLOS taxonomy to find articles in your field.</p>
        <p>For more information about PLOS Subject Areas, click
          <a href="https://github.com/PLOS/plos-thesaurus/blob/develop/README.md" target="_blank" title="Link opens in new window">here</a>.
        </p>
      </div>
    </div>
    <div class="levels">
      <div class="levels-container cf">
        <div class="levels-position"></div>
      </div>
      <a href="#" class="prev"></a>
      <a href="#" class="next active"></a>
    </div>
  </div>
  <div class="taxonomy-browser-border-bottom"></div>
</div></section>
<main> <div class="set-grid">

<header class="title-block">



<script type="text/javascript">
  var COUNTER_HOST = "https://counter.plos.org/api/v1.0/stats/totals/doi";
</script>

<script type="text/javascript">
  var ALM_CONFIG = ALM_CONFIG || {};
  ALM_CONFIG.hostname = "https://alm.plos.org";
  ALM_CONFIG.apiKey = "3pezRBRXdyzYW6ztfwft";
  ALM_CONFIG.host = "https://alm.plos.org/api/v5/articles";
</script>

<ul id="almSignposts" class="signposts">
  <li id="loadingMetrics">
    <p>Loading metrics</p>
  </li>
</ul>

<script type="text/template" id="signpostsGeneralErrorTemplate">
  <li id="metricsError">Article metrics are unavailable at this time. Please try again later.</li>
</script>

<script type="text/template" id="signpostsNewArticleErrorTemplate">
  <li></li><li></li><li id="tooSoon">Article metrics are unavailable for recently published articles.</li>
</script>

<script type="text/template" id="signpostsTemplate">
    <li id="almSaves">
      <%= s.numberFormat(saveCount, 0) %>
      <div class="tools" data-js-tooltip-hover="trigger">
        <a class="metric-term" href="/plosone/article/metrics?id=10.1371/journal.pone.0099625#savedHeader">Save</a>
        <p class="saves-tip" data-js-tooltip-hover="target"><a href="/plosone/article/metrics?id=10.1371/journal.pone.0099625#savedHeader">Total Mendeley and CiteULike bookmarks.</a></p>
      </div>
    </li>

    <li id="almCitations">
      <%= s.numberFormat(citationCount, 0) %>
      <div class="tools" data-js-tooltip-hover="trigger">
        <a class="metric-term" href="/plosone/article/metrics?id=10.1371/journal.pone.0099625#citedHeader">Citation</a>
        <p class="citations-tip" data-js-tooltip-hover="target"><a href="/plosone/article/metrics?id=10.1371/journal.pone.0099625#citedHeader">Paper's citation count computed by Scopus.</a></p>
      </div>
    </li>

    <li id="almViews">
      <%= s.numberFormat(viewCount, 0) %>
      <div class="tools" data-js-tooltip-hover="trigger">
        <a class="metric-term" href="/plosone/article/metrics?id=10.1371/journal.pone.0099625#viewedHeader">View</a>
        <p class="views-tip" data-js-tooltip-hover="target"><a href="/plosone/article/metrics?id=10.1371/journal.pone.0099625#viewedHeader">Sum of PLOS and PubMed Central page views and downloads.</a></p>
      </div>
    </li>

    <li id="almShares">
      <%= s.numberFormat(shareCount, 0) %>
      <div class="tools" data-js-tooltip-hover="trigger">
        <a class="metric-term" href="/plosone/article/metrics?id=10.1371/journal.pone.0099625#discussedHeader">Share</a>
        <p class="shares-tip" data-js-tooltip-hover="target"><a href="/plosone/article/metrics?id=10.1371/journal.pone.0099625#discussedHeader">Sum of Facebook and Twitter activity.</a></p>
      </div>
    </li>
</script>

    <div class="article-meta">

<div class="classifications">
  <p class="license-short" id="licenseShort">Open Access</p>
  <p class="peer-reviewed" id="peerReviewed">Peer-reviewed</p>

<div class="article-type" >
  <p class="type-article" id="artType">Research Article</p>
</div>


</div>


    </div>
    <div class="article-title-etc">



<div class="title-authors">
  <h1 id="artTitle"><?xml version="1.0" encoding="UTF-8"?>RNA-Seq Transcriptome Profiling Identifies <i>CRISPLD2</i> as a Glucocorticoid Responsive Gene that Modulates Cytokine Function in Airway Smooth Muscle Cells</h1>

<ul class="author-list clearfix"  data-js-tooltip="tooltip_container" id="author-list">



<li
  data-js-tooltip="tooltip_trigger"
  
>
   <a  data-author-id="0" class="author-name" >
Blanca E. Himes <span class="contribute"> </span> <span class="email">  </span>,</a>    <div id="author-meta-0" class="author-info" data-js-tooltip="tooltip_target">

  <p>
    <span class="contribute"> </span> Contributed equally to this work with:
    Blanca E. Himes,
    Xiaofeng Jiang
  </p>
  
  <p id="authCorresponding-0"> <span class="email">* E-mail:</span> <a href="mailto:blanca.himes@channing.harvard.edu">blanca.himes@channing.harvard.edu</a> (BH); <a href="mailto:qlu@hsph.harvard.edu">qlu@hsph.harvard.edu</a> (QL)</p>
  <p id="authAffiliations-0"><span class="type">Affiliations</span>
    Channing Division of Network Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, Massachusetts, United States of America, 
    Partners HealthCare Personalized Medicine, Boston, Massachusetts, United States of America, 
    Children's Hospital Informatics Program, Boston, Massachusetts, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose0"> &#x02A2F; </a>
    </div>
</li>

<li
  data-js-tooltip="tooltip_trigger"
  
>
   <a  data-author-id="1" class="author-name" >
Xiaofeng Jiang <span class="contribute"> </span>,</a>    <div id="author-meta-1" class="author-info" data-js-tooltip="tooltip_target">

  <p>
    <span class="contribute"> </span> Contributed equally to this work with:
    Blanca E. Himes,
    Xiaofeng Jiang
  </p>
  
  
  <p id="authAffiliations-1"><span class="type">Affiliation</span>
    Program in Molecular and Integrative Physiological Sciences, Departments of Environmental Health, and Genetics and Complex Diseases, Harvard School of Public Health, Boston, Massachusetts, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose1"> &#x02A2F; </a>
    </div>
</li>

<li
  data-js-tooltip="tooltip_trigger"
  
>
   <a  data-author-id="2" class="author-name" >
Peter Wagner,</a>    <div id="author-meta-2" class="author-info" data-js-tooltip="tooltip_target">

  
  
  <p id="authAffiliations-2"><span class="type">Affiliation</span>
    Program in Molecular and Integrative Physiological Sciences, Departments of Environmental Health, and Genetics and Complex Diseases, Harvard School of Public Health, Boston, Massachusetts, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose2"> &#x02A2F; </a>
    </div>
</li>

<li
  data-js-tooltip="tooltip_trigger"
  
>
   <a  data-author-id="3" class="author-name" >
Ruoxi Hu,</a>    <div id="author-meta-3" class="author-info" data-js-tooltip="tooltip_target">

  
  
  <p id="authAffiliations-3"><span class="type">Affiliation</span>
    Program in Molecular and Integrative Physiological Sciences, Departments of Environmental Health, and Genetics and Complex Diseases, Harvard School of Public Health, Boston, Massachusetts, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose3"> &#x02A2F; </a>
    </div>
</li>

<li
  data-js-tooltip="tooltip_trigger"
  
>
   <a  data-author-id="4" class="author-name" >
Qiyu Wang,</a>    <div id="author-meta-4" class="author-info" data-js-tooltip="tooltip_target">

  
  
  <p id="authAffiliations-4"><span class="type">Affiliation</span>
    Program in Molecular and Integrative Physiological Sciences, Departments of Environmental Health, and Genetics and Complex Diseases, Harvard School of Public Health, Boston, Massachusetts, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose4"> &#x02A2F; </a>
    </div>
</li>

<li
  data-js-tooltip="tooltip_trigger"
  
>
   <a  data-author-id="5" class="author-name" >
Barbara Klanderman,</a>    <div id="author-meta-5" class="author-info" data-js-tooltip="tooltip_target">

  
  
  <p id="authAffiliations-5"><span class="type">Affiliation</span>
    Partners HealthCare Personalized Medicine, Boston, Massachusetts, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose5"> &#x02A2F; </a>
    </div>
</li>

<li
  data-js-tooltip="tooltip_trigger"
  
>
   <a  data-author-id="6" class="author-name" >
Reid M. Whitaker,</a>    <div id="author-meta-6" class="author-info" data-js-tooltip="tooltip_target">

  
  
  <p id="authAffiliations-6"><span class="type">Affiliation</span>
    Channing Division of Network Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, Massachusetts, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose6"> &#x02A2F; </a>
    </div>
</li>

<li
  data-js-tooltip="tooltip_trigger"
  
>
   <a  data-author-id="7" class="author-name" >
Qingling Duan,</a>    <div id="author-meta-7" class="author-info" data-js-tooltip="tooltip_target">

  
  
  <p id="authAffiliations-7"><span class="type">Affiliation</span>
    Channing Division of Network Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, Massachusetts, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose7"> &#x02A2F; </a>
    </div>
</li>

<li
  data-js-tooltip="tooltip_trigger"
  
>
   <a  data-author-id="8" class="author-name" >
Jessica Lasky-Su,</a>    <div id="author-meta-8" class="author-info" data-js-tooltip="tooltip_target">

  
  
  <p id="authAffiliations-8"><span class="type">Affiliation</span>
    Channing Division of Network Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, Massachusetts, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose8"> &#x02A2F; </a>
    </div>
</li>

<li
  data-js-tooltip="tooltip_trigger"
  
>
   <a  data-author-id="9" class="author-name" >
Christina Nikolos,</a>    <div id="author-meta-9" class="author-info" data-js-tooltip="tooltip_target">

  
  
  <p id="authAffiliations-9"><span class="type">Affiliation</span>
    Pulmonary, Allergy and Critical Care Division, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose9"> &#x02A2F; </a>
    </div>
</li>

<li
  data-js-tooltip="tooltip_trigger"
  
>
   <a  data-author-id="10" class="author-name" >
William Jester,</a>    <div id="author-meta-10" class="author-info" data-js-tooltip="tooltip_target">

  
  
  <p id="authAffiliations-10"><span class="type">Affiliation</span>
    Pulmonary, Allergy and Critical Care Division, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose10"> &#x02A2F; </a>
    </div>
</li>

<li
  data-js-tooltip="tooltip_trigger"
  
>
   <a  data-author-id="11" class="author-name" >
Martin Johnson,</a>    <div id="author-meta-11" class="author-info" data-js-tooltip="tooltip_target">

  
  
  <p id="authAffiliations-11"><span class="type">Affiliation</span>
    Pulmonary, Allergy and Critical Care Division, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose11"> &#x02A2F; </a>
    </div>
</li>

<li
  data-js-tooltip="tooltip_trigger"
  
>
   <a  data-author-id="12" class="author-name" >
Reynold A. Panettieri Jr,</a>    <div id="author-meta-12" class="author-info" data-js-tooltip="tooltip_target">

  
  
  <p id="authAffiliations-12"><span class="type">Affiliation</span>
    Pulmonary, Allergy and Critical Care Division, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose12"> &#x02A2F; </a>
    </div>
</li>

<li
  data-js-tooltip="tooltip_trigger"
   data-js-toggle="toggle_target" data-visibility= "none"
>
   <a  data-author-id="13" class="author-name" >
Kelan G. Tantisira,</a>    <div id="author-meta-13" class="author-info" data-js-tooltip="tooltip_target">

  
  
  <p id="authAffiliations-13"><span class="type">Affiliation</span>
    Channing Division of Network Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, Massachusetts, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose13"> &#x02A2F; </a>
    </div>
</li>

<li
  data-js-tooltip="tooltip_trigger"
   data-js-toggle="toggle_target" data-visibility= "none"
>
   <a  data-author-id="14" class="author-name" >
Scott T. Weiss,</a>    <div id="author-meta-14" class="author-info" data-js-tooltip="tooltip_target">

  
  
  <p id="authAffiliations-14"><span class="type">Affiliations</span>
    Channing Division of Network Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, Massachusetts, United States of America, 
    Partners HealthCare Personalized Medicine, Boston, Massachusetts, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose14"> &#x02A2F; </a>
    </div>
</li>


<li data-js-toggle="toggle_add">&nbsp;[ ... ],</li>


<li
  data-js-tooltip="tooltip_trigger"
  
>
   <a  data-author-id="15" class="author-name" >
Quan Lu <span class="email">  </span></a>    <div id="author-meta-15" class="author-info" data-js-tooltip="tooltip_target">

  
  <p id="authCorresponding-15"> <span class="email">* E-mail:</span> <a href="mailto:blanca.himes@channing.harvard.edu">blanca.himes@channing.harvard.edu</a> (BH); <a href="mailto:qlu@hsph.harvard.edu">qlu@hsph.harvard.edu</a> (QL)</p>
  <p id="authAffiliations-15"><span class="type">Affiliation</span>
    Program in Molecular and Integrative Physiological Sciences, Departments of Environmental Health, and Genetics and Complex Diseases, Harvard School of Public Health, Boston, Massachusetts, United States of America
  </p>

      <a data-js-tooltip="tooltip_close" class="close" id="tooltipClose15"> &#x02A2F; </a>
    </div>
</li>
  <li data-js-toggle="toggle_trigger" >
    <a class="more-authors active" id="authors-show">[ view all ]</a>
    </li>
  <li data-js-toggle="toggle_trigger" data-visibility="none">
    <a class="author-less" id="author-hide">[ view less ]</a>
  </li>

</ul>

</div>


<div id="floatTitleTop" data-js-floater="title_author" class="float-title">
  <div class="set-grid">
    <div class="float-title-inner">
      <h1><?xml version="1.0" encoding="UTF-8"?>RNA-Seq Transcriptome Profiling Identifies <i>CRISPLD2</i> as a Glucocorticoid Responsive Gene that Modulates Cytokine Function in Airway Smooth Muscle Cells</h1>

<ul id="floatAuthorList" data-js-floater="floated_authors">

  <li data-float-index="1">Blanca E. Himes,&nbsp;

  </li>
  <li data-float-index="2">Xiaofeng Jiang,&nbsp;

  </li>
  <li data-float-index="3">Peter Wagner,&nbsp;

  </li>
  <li data-float-index="4">Ruoxi Hu,&nbsp;

  </li>
  <li data-float-index="5">Qiyu Wang,&nbsp;

  </li>
  <li data-float-index="6">Barbara Klanderman,&nbsp;

  </li>
  <li data-float-index="7">Reid M. Whitaker,&nbsp;

  </li>
  <li data-float-index="8">Qingling Duan,&nbsp;

  </li>
  <li data-float-index="9">Jessica Lasky-Su,&nbsp;

  </li>
  <li data-float-index="10">Christina Nikolos

</ul>



    </div>
    <div class="logo-close" id="titleTopCloser">
      <img src="/plosone/resource/img/logo.plos.95.png" alt="PLOS" />
      <div class="close-floater" title="close">x</div>
    </div>
  </div>
</div>

      <ul class="date-doi">
        <li id="artPubDate">Published: June 13, 2014</li>
        <li id="artDoi">
<a   href="https://doi.org/10.1371/journal.pone.0099625">https://doi.org/10.1371/journal.pone.0099625</a>
        </li>
      </ul>

    </div>
  <div>

  </div>
</header>

  <section class="article-body">



<ul class="article-tabs">

          <li class="tab-title active" id="tabArticle">
            <a href="/plosone/article?id=10.1371/journal.pone.0099625" class="article-tab-1">Article</a>
        </li>


              <li class="tab-title " id="tabAuthors">
            <a href="/plosone/article/authors?id=10.1371/journal.pone.0099625" class="article-tab-2">Authors</a>
        </li>


          <li class="tab-title " id="tabMetrics">
            <a href="/plosone/article/metrics?id=10.1371/journal.pone.0099625" class="article-tab-3">Metrics</a>
        </li>


          <li class="tab-title " id="tabComments">
            <a href="/plosone/article/comments?id=10.1371/journal.pone.0099625" class="article-tab-4">Comments</a>
        </li>

          <li class="tab-title " id="tabRelated">
            <a href="/plosone/article/related?id=10.1371/journal.pone.0099625" class="article-tab-5">Media Coverage</a>
        </li>

</ul>

    <div class="article-container">


<div id="nav-article">
  <ul class="nav-secondary">

    <li class="nav-comments" id="nav-comments">
      <a href="article/comments?id=10.1371/journal.pone.0099625">Reader Comments (0)</a>
    </li>

    <li class="nav-media" id="nav-media" data-doi="10.1371/journal.pone.0099625">
      <a href="/plosone/article/related?id=10.1371/journal.pone.0099625">
        Media Coverage <span id="media-coverage-count"></span>
      </a>
    </li>

    <li id="nav-figures"><a href="#" data-doi="10.1371/journal.pone.0099625">Figures</a></li>
  </ul>
</div>

<div id="figure-lightbox-container"></div>

<script id="figure-lightbox-template" type="text/template">
  <div id="figure-lightbox" class="reveal-modal full" data-reveal aria-hidden="true"
       role="dialog">
    <div class="lb-header">
      <h1 id="lb-title"><%= articleTitle %></h1>

      <div id="lb-authors">
              <span>Blanca E. Himes</span>
              <span>Xiaofeng Jiang</span>
          <a class="more-authors" href="/plosone/article/authors?id=10.1371/journal.pone.0099625">...</a>
          <span>Quan Lu</span>
      </div>

      <div class="lb-close" title="close">&nbsp;</div>
    </div>
    <div class="img-container">
      <div class="loader"> <i class="fa-spinner"></i> </div>
      <img class="main-lightbox-image" src=""/>
      <aside id="figures-list">
        <% figureList.each(function (ix, figure) { %>
        <div class="change-img" data-doi="<%= figure.getAttribute('data-doi') %>">
          <img class="aside-figure" src="/plosone/article/figure/image?size=inline&id=<%= figure.getAttribute('data-doi') %>" />
        </div>
        <% }) %>
        <div class="dummy-figure">
        </div>
      </aside>
    </div>
    <div id="lightbox-footer">

      <div id="btns-container" class="lightbox-row <% if(figureList.length <= 1) { print('one-figure-only') } %>">
        <div class="fig-btns-container reset-zoom-wrapper left">
          <span class="fig-btn reset-zoom-btn">Reset zoom</span>
        </div>
        <div class="zoom-slider-container">
          <div class="range-slider-container">
            <span id="lb-zoom-min"></span>
            <div class="range-slider round" data-slider data-options="start: 20; end: 200; initial: 20;">
              <span class="range-slider-handle" role="slider" tabindex="0"></span>
              <span class="range-slider-active-segment"></span>
              <input type="hidden">
            </div>
            <span id="lb-zoom-max"></span>
          </div>
        </div>
        <% if(figureList.length > 1) { %>
        <div class="fig-btns-container">
          <span class="fig-btn all-fig-btn"><i class="icon icon-all"></i> All Figures</span>
          <span class="fig-btn next-fig-btn"><i class="icon icon-next"></i> Next</span>
          <span class="fig-btn prev-fig-btn"><i class="icon icon-prev"></i> Previous</span>
        </div>
        <% } %>
      </div>
      <div id="image-context">
      </div>
    </div>
  </div>
</script>

<script id="image-context-template" type="text/template">
  <div class="footer-text">
    <div id="figure-description-wrapper">
      <div id="view-more-wrapper" style="<% descriptionExpanded? print('display:none;') : '' %>">
        <span id="figure-title"><%= title %></span>
        <p id="figure-description">
          <%= description %>&nbsp;&nbsp;
        </p>
        <span id="view-more">show more<i class="icon-arrow-right"></i></span>

      </div>
      <div id="view-less-wrapper" style="<% descriptionExpanded? print('display:inline-block;') : '' %>" >
        <span id="figure-title"><%= title %></span>
        <p id="full-figure-description">
          <%= description %>&nbsp;&nbsp;
          <span id="view-less">show less<i class="icon-arrow-left"></i></span>
        </p>
      </div>
    </div>
  </div>
  <div id="show-context-container">
    <a class="btn show-context" href="<%= showInContext(strippedDoi) %>">Show in Context</a>
  </div>
  <div id="download-buttons">
    <h3>Download:</h3>
    <div class="item">
      <a href="/plosone/article/figure/image?size=original&download=&id=<%= doi %>" title="original image">
        <span class="download-btn">TIFF</span>
      </a>
      <span class="file-size"><%= fileSizes.original %></span>
    </div>
    <div class="item">
      <a href="/plosone/article/figure/image?size=large&download=&id=<%= doi %>" title="large image">
        <span class="download-btn">PNG</span>
      </a>
      <span class="file-size"><%= fileSizes.large %></span>
    </div>
    <div class="item">
      <a href="/plosone/article/figure/powerpoint?id=<%= doi %>" title="PowerPoint slide">
        <span class="download-btn">PPT</span>
      </a>
    </div>

  </div>
</script>
      <div class="article-content">







<div id="figure-carousel-section">
  <h2>Figures</h2>

  <div id="figure-carousel">

    <div class="carousel-wrapper">
      <div class="slider">

          <div class="carousel-item lightbox-figure" data-doi="10.1371/journal.pone.0099625.g001">

                <img src="/plosone/article/figure/image?size=inline&amp;id=10.1371/journal.pone.0099625.g001"
                     alt="Figure 1"
                />

            </div>

          <div class="carousel-item lightbox-figure" data-doi="10.1371/journal.pone.0099625.t001">

                <img src="/plosone/article/figure/image?size=inline&amp;id=10.1371/journal.pone.0099625.t001"
                     alt="Table 1"
                />

            </div>

          <div class="carousel-item lightbox-figure" data-doi="10.1371/journal.pone.0099625.g002">

                <img src="/plosone/article/figure/image?size=inline&amp;id=10.1371/journal.pone.0099625.g002"
                     alt="Figure 2"
                />

            </div>

          <div class="carousel-item lightbox-figure" data-doi="10.1371/journal.pone.0099625.t002">

                <img src="/plosone/article/figure/image?size=inline&amp;id=10.1371/journal.pone.0099625.t002"
                     alt="Table 2"
                />

            </div>

          <div class="carousel-item lightbox-figure" data-doi="10.1371/journal.pone.0099625.t003">

                <img src="/plosone/article/figure/image?size=inline&amp;id=10.1371/journal.pone.0099625.t003"
                     alt="Table 3"
                />

            </div>

          <div class="carousel-item lightbox-figure" data-doi="10.1371/journal.pone.0099625.g003">

                <img src="/plosone/article/figure/image?size=inline&amp;id=10.1371/journal.pone.0099625.g003"
                     alt="Figure 3"
                />

            </div>

          <div class="carousel-item lightbox-figure" data-doi="10.1371/journal.pone.0099625.g004">

                <img src="/plosone/article/figure/image?size=inline&amp;id=10.1371/journal.pone.0099625.g004"
                     alt="Figure 4"
                />

            </div>
      </div>
    </div>

    <div class="carousel-control">
      <span class="button previous"></span>
      <span class="button next"></span>
    </div>
    <div class="carousel-page-buttons">

    </div>
  </div>
</div>


        <div class="article-text" id="artText">
          



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>Asthma is a chronic inflammatory respiratory disease that affects over 300 million people worldwide. Glucocorticoids are a mainstay therapy for asthma because they exert anti-inflammatory effects in multiple lung tissues, including the airway smooth muscle (ASM). However, the mechanism by which glucocorticoids suppress inflammation in ASM remains poorly understood. Using RNA-Seq, a high-throughput sequencing method, we characterized transcriptomic changes in four primary human ASM cell lines that were treated with dexamethasone—a potent synthetic glucocorticoid (1 µM for 18 hours). Based on a Benjamini-Hochberg corrected p-value &lt;0.05, we identified 316 differentially expressed genes, including both well known (<em>DUSP1, KLF15, PER1, TSC22D3</em>) and less investigated (<em>C7, CCDC69, CRISPLD2</em>) glucocorticoid-responsive genes. <em>CRISPLD2</em>, which encodes a secreted protein previously implicated in lung development and endotoxin regulation, was found to have SNPs that were moderately associated with inhaled corticosteroid resistance and bronchodilator response among asthma patients in two previously conducted genome-wide association studies. Quantitative RT-PCR and Western blotting showed that dexamethasone treatment significantly increased <em>CRISPLD2</em> mRNA and protein expression in ASM cells. <em>CRISPLD2</em> expression was also induced by the inflammatory cytokine IL1β, and small interfering RNA-mediated knockdown of <em>CRISPLD2</em> further increased IL1β-induced expression of <em>IL6</em> and <em>IL8</em>. Our findings offer a comprehensive view of the effect of a glucocorticoid on the ASM transcriptome and identify <em>CRISPLD2</em> as an asthma pharmacogenetics candidate gene that regulates anti-inflammatory effects of glucocorticoids in the ASM.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Himes BE, Jiang X, Wagner P, Hu R, Wang Q, Klanderman B, et al.  (2014) RNA-Seq Transcriptome Profiling Identifies <em>CRISPLD2</em> as a Glucocorticoid Responsive Gene that Modulates Cytokine Function in Airway Smooth Muscle Cells. PLoS ONE 9(6):
           e99625.
        
        https://doi.org/10.1371/journal.pone.0099625</p><p><strong>Editor: </strong>Jan Peter Tuckermann, University of Ulm, Germany</p><p><strong>Received: </strong>December 6, 2013; <strong>Accepted: </strong>May 17, 2014; <strong>Published: </strong> June 13, 2014</p><p><strong>Copyright: </strong> © 2014 Himes et al. This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Funding was provided by National Institutes of Health (NIH) U01 HL65899, an NIH Pharmacogenomics Research Network (PGRN) – RIKEN Center for Genomic Medicine (CGM) Global Alliance, R01 HL097796, R01 HL114769, and P30 ES013508. BEH was funded by NIH K99 HL105663. XJ was supported by NIH training grant T32 HL007118. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors declare that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="s1" name="s1" data-toc="s1" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Asthma, a chronic inflammatory respiratory disease that affects over 25 million Americans and 300 million people world-wide, is characterized by variable airflow limitation and airway hyperresponsiveness <a href="#pone.0099625-Akinbami1" class="ref-tip">[1]</a>, <a href="#pone.0099625-Global1" class="ref-tip">[2]</a>. Glucocorticoids (GCs) are common medications used to treat various inflammatory diseases, including asthma <a href="#pone.0099625-Rhen1" class="ref-tip">[3]</a>. Inhaled corticosteroids, GC medications that act directly in the lung, are among the most common asthma controller medications and treatment of asthma patients with them leads to improved clinical outcomes, including decreased asthma symptoms and exacerbations <a href="#pone.0099625-Fanta1" class="ref-tip">[4]</a>. At a cellular level, GCs act by binding to GC receptors (GRs), causing them to translocate to cell nuclei where they modulate transcription of various genes in a tissue-dependent fashion <a href="#pone.0099625-Pratt1" class="ref-tip">[5]</a>. The anti-inflammatory action of GCs is partly a result of 1) GC-GR complexes stimulating anti-inflammatory genes by directly binding to DNA at glucocorticoid receptor enhancer elements, and of 2) GC-GR complexes inhibiting proinflammatory transcription factors such as nuclear factor kappa-light-chain-enhancer of activated B cells (NFκB) <a href="#pone.0099625-McMaster1" class="ref-tip">[6]</a>. In addition to directly reducing inflammation, GCs have been shown to affect other asthma-related phenotypes, including bronchodilation <a href="#pone.0099625-Slats1" class="ref-tip">[7]</a>, airway hyperresponsiveness <a href="#pone.0099625-Trifilieff1" class="ref-tip">[8]</a>, and airway smooth muscle (ASM) contractility <a href="#pone.0099625-Lakser1" class="ref-tip">[9]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>Many cells and tissues are involved in asthma and are targeted by GCs, including inflammatory <a href="#pone.0099625-Lamas1" class="ref-tip">[10]</a>, <a href="#pone.0099625-Wallen1" class="ref-tip">[11]</a>, airway epithelium <a href="#pone.0099625-Reddy1" class="ref-tip">[12]</a>, and ASM <a href="#pone.0099625-Shore1" class="ref-tip">[13]</a>. Of these, the ASM is involved in altered airway contractility <a href="#pone.0099625-Panettieri1" class="ref-tip">[14]</a>, a major asthma-specific trait that is assessed clinically and for research studies by measures such as bronchodilator response <a href="#pone.0099625-Nelson1" class="ref-tip">[15]</a> and airway hyperresponsiveness <a href="#pone.0099625-Hargreave1" class="ref-tip">[16]</a>. However, compared to the other airway cells, much less is known about how GCs work specifically in the ASM to alleviate asthma. Because GCs function by activating GR to directly modulate transcriptional gene expression, a better understanding of how the ASM transcriptome responds to GCs is needed to provide mechanistic insights for improving asthma therapy. Several studies have been conducted to identify GCs-induced transcript changes in the ASM. For example, two microarray-based gene expression studies have measured the effect of GCs on ASM cells using <em>in vitro</em> models where human ASM cells were stimulated with dexamethasone or fluticasone <a href="#pone.0099625-Masuno1" class="ref-tip">[17]</a>, <a href="#pone.0099625-Misior1" class="ref-tip">[18]</a>. Although both were limited by the inherent biases of microarrays, these studies identified some genes involved in the ASM GC response, with one focusing on validating the function of the <em>KLF15</em> gene in airway hyperresponsiveness <a href="#pone.0099625-Masuno1" class="ref-tip">[17]</a> and the other on the overlap between GC and beta-agonist response of the ASM <a href="#pone.0099625-Misior1" class="ref-tip">[18]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>Recent advances in sequencing technologies have made possible the comprehensive and in-depth characterization of transcriptomes via a technique known as RNA-Seq <a href="#pone.0099625-Wang1" class="ref-tip">[19]</a>–<a href="#pone.0099625-Trapnell2" class="ref-tip">[21]</a>. Compared to the use of microarrays, RNA-Seq is able to (1) quantify more RNA species, including non-coding and novel splice variants, (2) quantify RNA at baseline, rather than only measure fold changes across conditions, and (3) cover a wider dynamic range of signal <a href="#pone.0099625-Marioni1" class="ref-tip">[22]</a>. In this study, we used RNA-Seq to comprehensively characterize changes of the ASM transcriptome in response to GCs using an <em>in vitro</em> model. We identified 316 significantly differentially expressed genes representing various functional categories such as glycoprotein/extracellular matrix, vasculature and lung development, regulation of cell migration, and extracellular matrix organization. One of these genes, cysteine-rich secretory protein LCCL domain-containing, 2 (<em>CRISPLD2</em>; OMIM 612434), had single nucleotide polymorphisms (SNPs) that were nominally associated with two asthma drug response phenotypes (i.e., inhaled corticosteroid response and short-acting bronchodilator response). Functional experiments showed that in ASM cells, <em>CRISPLD2</em> mRNA and protein levels changed in response to treatment with a glucocorticoid or proinflammatory cytokine, and that knockdown of <em>CRISPLD2</em> resulted in increased levels of IL1β-induced <em>IL6</em> and <em>IL8</em> mRNA expression.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2" name="s2" data-toc="s2" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="s2a" name="s2a" class="link-target" title="RNA-Seq Transcriptome Profiling of GC-treated Primary Human ASM Cells"></a>
<h3>RNA-Seq Transcriptome Profiling of GC-treated Primary Human ASM Cells</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>To identify GC-responsive genes in ASM, we performed RNA-Seq expression profiling of primary ASM cells from four white male donors treated with 1 µM dexamethasone (DEX) or control vehicle for 18 h, a treatment protocol that captures a large set of genes regulated by the GR <a href="#pone.0099625-Masuno1" class="ref-tip">[17]</a>. We obtained an average of 58.9 million raw sequencing reads per sample (range 44.2–71.3 million reads per sample). Of these reads, an average of 83.36% were aligned to hg19 genome reference files downloaded from Illumina's iGenomes project (range 81.94%–84.34%) [<a href="#pone.0099625.s012">Table S1</a>]. An average of 26.43% of the mapped reads spanned junctions. Most bases in mapped reads corresponded to mRNA (&gt;98%) [<a href="#pone.0099625.s013">Table S2</a>]. Plots of normalized read coverage of transcripts vs. normalized position, reveals that there was even coverage of transcripts by reads [<a href="#pone.0099625.s001">Figure S1</a>]. Based on these and various quality control (QC) summary metrics, including ERCC spike-in dose response plots, the sequencing and alignment results for each sample were deemed of sufficiently high quality to include in differential expression analyses. Quantification of transcript and gene expression levels was performed using Cufflinks according to hg19 RefSeq annotation files from Illumina's iGenomes Project.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2" class="link-target"></a><p>Overall, 316 genes were significantly differentially expressed after correcting for false discovery rate by the Benjamini-Hochberg <a href="#pone.0099625-Hochberg1" class="ref-tip">[23]</a> approach [<a href="#pone-0099625-g001">Figure 1A</a>, <a href="#pone.0099625.s014">Table S3</a>]. <a href="#pone-0099625-t001">Table 1</a> contains the genes with Q-value &lt;1E-10 that were considered for further study. Some of these top genes have been previously related to steroid responsiveness and inflammation (i.e., <em>DUSP1</em> <a href="#pone.0099625-Shipp1" class="ref-tip">[<em>24</em>]</a>, <em>FKBP5</em> <a href="#pone.0099625-Kelly1" class="ref-tip">[25]</a>, <em>KLF15</em> <a href="#pone.0099625-Masuno1" class="ref-tip">[<em>17</em>]</a>, <em>PER1</em> <a href="#pone.0099625-Reddy1" class="ref-tip">[<em>12</em>]</a>, <a href="#pone.0099625-Burioka1" class="ref-tip">[<em>26</em>]</a>, and <em>TSC22D3</em> <a href="#pone.0099625-Kelly1" class="ref-tip">[<em>25</em>]</a>, <a href="#pone.0099625-Eddleston1" class="ref-tip">[27]</a>), and their upregulation by 1 µM for 18 DEX was confirmed by quantitative real time PCR (qRT-PCR) in ASM cells from three donors [<a href="#pone-0099625-g001">Figure 1B</a>]. qRT-PCR results for the fourth donor used in the RNA-Seq experiment were also consistent [<a href="#pone.0099625.s002">Figure S2</a>]. Other genes identified via the RNA-Seq experiment were considered potentially novel GC-responsive genes as they have little published evidence regarding a relationship with steroid responsiveness and/or inflammation. Gene set enrichment analysis using the NIH DAVID tool <a href="#pone.0099625-Huangda1" class="ref-tip">[28]</a> identified various Gene Ontology and other annotation categories that were overrepresented by the 316 genes. The top six gene functional annotation clusters (enrichment scores &gt;3) had terms related to: glycoprotein/extracellular matrix, vasculature development, circulatory system process, response to nutrients, thrombospondin type-1, and response to hormone stimulus terms [<a href="#pone.0099625.s015">Table S4</a>]. Other clusters among the 19 with enrichment scores &gt;1.5 that may be relevant to lung disease included lung development, regulation of cell migration, and extracellular matrix organization.</p>
<a class="link-target" id="pone-0099625-g001" name="pone-0099625-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0099625.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0099625.g001" data-doi="info:doi/10.1371/journal.pone.0099625" data-uri="info:doi/10.1371/journal.pone.0099625.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0099625.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0099625.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0099625.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0099625.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0099625.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0099625.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0099625.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 1. </span> RNA-Seq profiling of DEX-treated ASM cells.</div><p class="caption_target"><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1" class="link-target"></a><p><strong>A</strong>) Volcano plot of overall gene-based differential expression results for four cell lines treated with DEX vs. left untreated (each dot corresponds to a gene). The y-axis corresponds to the negative log (base 10) of P-values while the x-axis corresponds to the negative log (base 2) of the fold change for difference in expression when cells were stimulated with DEX. There were 316 differentially expressed genes according to an adjusted p-value &lt;0.05 (blue dots). <strong>B</strong>) Validation of known GC-responsive genes through qRT-PCR. After ASM cells were treated with 1 µM DEX for 18 h, the mRNA levels of indicated genes were measured by qRT-PCR and the folds of change in mRNA induced by DEX were calculated. Each column bar represents an individual cell line. Experiments for each cell line were performed in triplicate, and the error bars are SE values corresponding to a cell line's replicates. The dotted line indicates a fold change of 1. ** <em>P</em>&lt;0.005, * <em>P</em>&lt;0.05 (t test).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0099625.g001">
              https://doi.org/10.1371/journal.pone.0099625.g001</a></p></div><a class="link-target" id="pone-0099625-t001" name="pone-0099625-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0099625.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0099625.t001" data-doi="info:doi/10.1371/journal.pone.0099625" data-uri="info:doi/10.1371/journal.pone.0099625.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0099625.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0099625.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0099625.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0099625.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0099625.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0099625.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0099625.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> Top (Q-value &lt;1E-10) differentially expressed genes.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0099625.t001">
              https://doi.org/10.1371/journal.pone.0099625.t001</a></p></div></div>

<div id="section2" class="section toc-section"><a id="s2b" name="s2b" class="link-target" title="Verification of GC-responsive Genes by q-PCR"></a>
<h3>Verification of GC-responsive Genes by q-PCR</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>A subset of the top differentially expressed genes (i.e., <em>CRISPLD2, C13orf15, KCTD12, SERPINA3</em>) was selected for follow-up based on each gene's potential to be a novel steroid responsiveness gene. Differential expression for these four genes and one additional gene selected from the top 316 differentially expressed ones (i.e., <em>PTX3</em>) was verified via qRT-PCR by treating with 1 µM DEX for 18 h three of the ASM cell lines used for RNA-Seq [<a href="#pone-0099625-g002">Figure 2</a>] to compare biological sample variability and effect sizes obtained via RNA-Seq vs. qRT-PCR. qRT-PCR results for the fourth donor were consistent with those for the other three cell lines [<a href="#pone.0099625.s002">Figure S2</a>]. Gene expression levels varied among the primary cell lines, suggesting an inherent heterogeneity in individual GC responsiveness. Nevertheless, the qRT-PCR data for each of the genes was consistent in direction of fold-change with the RNA-Seq results.</p>
<a class="link-target" id="pone-0099625-g002" name="pone-0099625-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0099625.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0099625.g002" data-doi="info:doi/10.1371/journal.pone.0099625" data-uri="info:doi/10.1371/journal.pone.0099625.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0099625.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0099625.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0099625.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0099625.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0099625.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0099625.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0099625.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 2. </span> Validation of GC responsive genes.</div><p class="caption_target"><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1" class="link-target"></a><p>After cells from three individual ASM lines were treated with 1 µM DEX for 18 h, the mRNA levels of indicated genes were measured by qRT-PCR and the folds of change induced by DEX were calculated. Each column bar represents an individual cell line; experiments for each cell line were performed in triplicate. Error bars are SE values corresponding to a cell line's replicates.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0099625.g002">
              https://doi.org/10.1371/journal.pone.0099625.g002</a></p></div></div>

<div id="section3" class="section toc-section"><a id="s2c" name="s2c" class="link-target" title="CRISPLD2 Variants Associated with Asthma Pharmacogenetic Phenotypes"></a>
<h3><em>CRISPLD2</em> Variants Associated with Asthma Pharmacogenetic Phenotypes</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>Inhaled corticosteroid (ICS) responsiveness is a measure of improvement in pulmonary function after treatment with a glucocorticoid. To determine whether any of the differentially expressed genes were associated with this pharmacogenetic phenotype, defined as unchanged improvement in lung function among asthma patients after receiving ICS therapy for 4–8 weeks, we obtained previously conducted ICS GWAS results (unpublished) for SNPs within, or spanning 50 kb on either side, each of the genes in <a href="#pone-0099625-t001">Table 1</a>. Based on a threshold of 1E-03, the <em>CRISPLD2</em> gene had SNPs that were nominally associated with ICS resistance [<a href="#pone-0099625-t002">Table 2</a>; <a href="#pone.0099625.s003">Figure S3</a>]. Because the beta-agonist and glucocorticoid pathways are known to overlap <a href="#pone.0099625-Kaur1" class="ref-tip">[29]</a>, we also examined the association of the differentially expressed genes with bronchodilator response, which measures the effect of beta-agonists on lung function. Based on bronchodilator response GWAS results from a previous study where the phenotype was defined as change in FEV<sub>1</sub> in response to administration of the beta-agonist albuterol <a href="#pone.0099625-Himes1" class="ref-tip">[30]</a>, SNPs in <em>CRISPLD2</em> and an additional gene <em>CCDC69</em> were nominally associated with the bronchodilator response [<a href="#pone-0099625-t002">Table 2</a>; <a href="#pone.0099625.s003">Figure S3</a>]. Additionally, replication results for one SNP (rs8047416) from this bronchodilator response GWAS that had a primary P-value of 4.5E-04 had been obtained for 552 white subjects from the Severe Asthma Research Program (SARP) cohort and found to have a P-value of 0.038 (overall P-value 9.0E-05). Together these results suggest a role for <em>CRISPLD2</em> in modulating two asthma pharmacogenetic phenotypes.</p>
<a class="link-target" id="pone-0099625-t002" name="pone-0099625-t002"></a><div class="figure" data-doi="10.1371/journal.pone.0099625.t002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0099625.t002" data-doi="info:doi/10.1371/journal.pone.0099625" data-uri="info:doi/10.1371/journal.pone.0099625.t002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0099625.t002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0099625.t002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0099625.t002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0099625.t002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0099625.t002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0099625.t002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0099625.t002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 2. </span> SNPs within 50<a href="#pone-0099625-t001">Table 1</a> that are associated (overall P-value &lt;1E-03) with bronchodilator response (BDR) or inhaled corticosteroid (ICS) resistance in human clinical trial cohorts.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0099625.t002">
              https://doi.org/10.1371/journal.pone.0099625.t002</a></p></div></div>

<div id="section4" class="section toc-section"><a id="s2d" name="s2d" class="link-target" title="CRISPLD2 Expression Changes in Previous Microarray Studies of the ASM GC Response"></a>
<h3><em>CRISPLD2</em> Expression Changes in Previous Microarray Studies of the ASM GC Response</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>We analyzed publicly available data from two published gene expression microarray studies (GSE34313 <a href="#pone.0099625-Masuno1" class="ref-tip">[17]</a> and GSE13168 <a href="#pone.0099625-Misior1" class="ref-tip">[18]</a>) that measured the effect of GCs on human ASM cells to determine whether these previous studies supported our <em>CRISPLD2</em> differential expression results. Although <em>CRISPLD2</em> did not rank as one of the most highly differentially expressed genes in these studies, all comparisons available between ASM cells treated with a GC vs. baseline conditions demonstrate that <em>CRISPLD2</em> had significant adjusted P-values [<a href="#pone-0099625-t003">Table 3</a>]. Specifically, the GSE34313 study found that <em>CRISPLD2</em> was differentially expressed both 4 and 24 hours after ASM cells were treated with DEX, and the GSE13168 study found that the differential <em>CRISPLD2</em> expression was strongest when ASM cells were treated with a GC (i.e. fluticasone) vs. left untreated, than when cells were also stimulated with pro-inflammatory cytokines (i.e. EGF and IL1β).</p>
<a class="link-target" id="pone-0099625-t003" name="pone-0099625-t003"></a><div class="figure" data-doi="10.1371/journal.pone.0099625.t003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0099625.t003" data-doi="info:doi/10.1371/journal.pone.0099625" data-uri="info:doi/10.1371/journal.pone.0099625.t003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0099625.t003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0099625.t003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0099625.t003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0099625.t003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0099625.t003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0099625.t003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0099625.t003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 3. </span> Differential expression results for <em>CRISPLD2</em> obtained from publicly available data of two previous microarray studies that investigated the effects of GCs on human ASM cells.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0099625.t003">
              https://doi.org/10.1371/journal.pone.0099625.t003</a></p></div></div>

<div id="section5" class="section toc-section"><a id="s2e" name="s2e" class="link-target" title="GC Induced CRISPLD2 mRNA and Protein Expression in Primary Human ASM Cells"></a>
<h3>GC Induced <em>CRISPLD2</em> mRNA and Protein Expression in Primary Human ASM Cells</h3>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1" class="link-target"></a><p>Because of its potential to modulate two important asthma drug response phenotypes vis-à-vis these associations and published evidence of its involvement in lung development and endotoxin regulation <a href="#pone.0099625-Wang2" class="ref-tip">[31]</a>, we focused our functional studies on the <em>CRISPLD2</em> gene to investigate its potential role in steroid and immune response in ASM cells. We grew the most GC sensitive ASM cell line among those tested in <a href="#pone-0099625-g002">Figure 2</a>, treated those cells with DEX, and extracted RNA for qRT-PCR and protein for immune-blot analysis. Upon DEX treatment, <em>CRISPLD2</em> mRNA increased 8.1-fold [<a href="#pone-0099625-g003">Figure 3A</a>]. Consistent with mRNA changes, protein levels of <em>CRISPLD2</em> in ASM cells also increased upon DEX treatment by 1.7-fold at 24 hours [<a href="#pone-0099625-g003">Figure 3B</a>]. Using cells from a single donor, the effect of DEX on <em>CRISPLD2</em> expression was found to be time [<a href="#pone.0099625.s004">Figure S4A</a>] and dose dependent [<a href="#pone.0099625.s004">Figure S4B</a>]. The induction of <em>CRISPLD2</em> by DEX that was observed in ASM did not occur in A549 pulmonary epithelial cells derived from a lung carcinoma tissue, as analogous treatment of A549 cells with DEX caused a decrease of <em>CRISPLD2</em> mRNA [<a href="#pone.0099625.s005">Figure S5</a>].</p>
<a class="link-target" id="pone-0099625-g003" name="pone-0099625-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0099625.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0099625.g003" data-doi="info:doi/10.1371/journal.pone.0099625" data-uri="info:doi/10.1371/journal.pone.0099625.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0099625.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0099625.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0099625.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0099625.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0099625.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0099625.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0099625.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 3. </span> <em>CRISPLD2</em> is a GC- and IL1β-responsive gene.</div><p class="caption_target"><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1" class="link-target"></a><p>ASM cells were treated with 100<strong>A</strong>) increased <em>CRISPLD2</em> mRNA expression as measured by qRT-PCR, <strong>B</strong>) increased <em>CRISPLD2</em> protein expression as measured by immuno-blotting. ASM cells were treated with 5 ng/mL IL1β for 24 h, resulting in <strong>C</strong>) increased <em>CRISPLD2</em> mRNA expression as measured by qRT-PCR, and <strong>D</strong>) increased <em>CRISPLD2</em> protein expression as measured by immuno-blotting. <em>CRISPLD2</em> mRNA levels were measured in triplicate. CRISPLD2 protein levels are shown as normalized blot densitometry values, and the error bars are SE values across three independent experiments. * <em>P</em>&lt;0.05 (<em>t</em> test).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0099625.g003">
              https://doi.org/10.1371/journal.pone.0099625.g003</a></p></div></div>

<div id="section6" class="section toc-section"><a id="s2f" name="s2f" class="link-target" title="CRISPLD2 is Induced by IL1β and Modulates the Expression of Two Immuno-Responsive Genes"></a>
<h3><em>CRISPLD2</em> is Induced by IL1β and Modulates the Expression of Two Immuno-Responsive Genes</h3>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1" class="link-target"></a><p>Because GCs exert anti-inflammatory effects, we tested the role of GC-induced <em>CRISPLD2</em> expression in regulating inflammatory responses in the ASM. Treatment of a single ASM cell line with the proinflammatory cytokine IL1β (5 ng/mL for 24 h) increased <em>CRISPLD2</em> mRNA by 10.4-fold and protein levels by 1.9-fold [<a href="#pone-0099625-g003">Figure 3C and 3D</a>], suggesting that <em>CRISPLD2</em> is not only GC-inducible but also immuno-responsive. We next performed knockdown experiments to assess whether <em>CRISPLD2</em> modulates IL1β-induced cytokine responses using a single ASM cell line. Because IL1β acts as an important mediator of inflammatory responses by activating other cytokines, we investigated the role of <em>CRISPLD2</em> in IL1β-induced expression of other known immune-response genes (i.e. <em>IL6</em> <a href="#pone.0099625-Rincon1" class="ref-tip">[32]</a> and <em>IL8</em> <a href="#pone.0099625-Harada1" class="ref-tip">[33]</a>). In ASM cells transfected with <em>CRISPLD2</em>-specific siRNA, <em>CRISPLD2</em> mRNA expression was decreased by 74% and protein levels decreased by 60% [<a href="#pone-0099625-g004">Figure 4A</a>].</p>
<a class="link-target" id="pone-0099625-g004" name="pone-0099625-g004"></a><div class="figure" data-doi="10.1371/journal.pone.0099625.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0099625.g004" data-doi="info:doi/10.1371/journal.pone.0099625" data-uri="info:doi/10.1371/journal.pone.0099625.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0099625.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0099625.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0099625.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0099625.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0099625.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0099625.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0099625.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 4. </span> <em>CRISPLD2</em> regulates the response to inflammatory cytokines.</div><p class="caption_target"><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1" class="link-target"></a><p><strong>A</strong>) Effect of <em>CRISPLD2</em>-specific siRNA on <em>CRISPLD2</em> mRNA and protein levels. ASM cells were transfected with <em>CRISPLD2</em>-specific siRNA or non-targeting (NT) siRNA, and 72 h later, <em>CRISPLD2</em> mRNA and protein levels were determined by qRT-PCR (levels normalized to those in control cells transfected with NT siRNA) and immuno-blotting, respectively. The effect of <em>CRISPLD2</em> knockdown on IL1β-induced cytokine expression was assessed by transfecting ASM cells with <em>CRISPLD2</em>-specific or NT siRNA, and 72 h later, treating cells for 24 h with <strong>B</strong>) 5 ng/mL IL1β, <strong>C</strong>) 100 nM DEX, or <strong>D</strong>) 5 ng/mL IL1β and 100 nM DEX. <em>IL6</em> expression was determined by qRT-PCR. Normalized mRNA levels are shown. Experiments were performed in triplicate, and the error bars are SE values for three samples. * <em>P</em>&lt;0.05 (<em>t</em> test).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0099625.g004">
              https://doi.org/10.1371/journal.pone.0099625.g004</a></p></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2" class="link-target"></a><p>While expression levels of <em>IL6</em> did not change in response to <em>CRISPLD2</em> knockdown, treatment of ASM cells with IL1β induced significantly higher expression of <em>IL6</em> in <em>CRISPLD2</em>-knockdown cells as compared to NT siRNA control cells [<a href="#pone-0099625-g004">Figure 4B</a>], suggesting that <em>CRISPLD2</em> is an inhibitory modulator of immuno-response in ASM cells. Consistent with this notion, another cytokine's (i.e. <em>IL8</em>'s) induction by IL1β was also enhanced by <em>CRISPLD2</em> knockdown [<a href="#pone.0099625.s006">Figure S6</a>]. To further characterize the effect of <em>CRISPLD2</em> on immune response, we treated cells with 100 nM DEX alone or in combination with 5 ng/mL IL1β. <em>IL6</em> expression was decreased with DEX treatment, but <em>CRISPLD2</em> knockdown did not significantly change the <em>IL6</em> response to DEX [<a href="#pone-0099625-g004">Figure 4C</a>]. However, <em>IL6</em> mRNA levels in <em>CRISPLD2</em> knockdown cells were higher than that in NT siRNA control cells when IL1β and DEX were administered simultaneously [<a href="#pone-0099625-g004">Figure 4D</a>], again supporting a role for <em>CRISPLD2</em> in modulating IL1β response.</p>
</div>

<div id="section7" class="section toc-section"><a id="s2g" name="s2g" class="link-target" title="CRISPLD2 is Not Required for the Expression of GC Target Genes"></a>
<h3><em>CRISPLD2</em> is Not Required for the Expression of GC Target Genes</h3>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1" class="link-target"></a><p>We performed siRNA-mediated <em>CRISPLD2</em> knockdown experiments to assess whether <em>CRISPLD2</em> affects the transcriptional expression of well-known GC-response genes (i.e. <em>DUSP1 </em><a href="#pone.0099625-Shipp1" class="ref-tip">[<em>24</em>]</a><em>, FKBP5</em> <a href="#pone.0099625-Kelly1" class="ref-tip">[25]</a> or <em>TSC22D3 </em><a href="#pone.0099625-Kelly1" class="ref-tip">[<em>25</em>]</a>, <a href="#pone.0099625-Eddleston1" class="ref-tip">[<em>27</em>]</a>). The expression levels of three GR target genes were not significantly altered in <em>CRISPLD2</em>-knockdown ASM cells relative to control non-targeting siRNA-transfected cells [<a href="#pone.0099625.s007">Figure S7</a>], suggesting that <em>CRIPSLD2</em> does not modulate the direct transcriptional actions of GCs in the ASM.</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="s3" name="s3" data-toc="s3" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1" class="link-target"></a><p>Due to their widespread use in the treatment of various inflammatory diseases and the notoriety of their undesirable side effects, GCs have been extensively characterized for their effects on specific tissues <a href="#pone.0099625-Rhen1" class="ref-tip">[3]</a>, <a href="#pone.0099625-Schacke1" class="ref-tip">[34]</a>. In the case of asthma, an important target of GCs is the ASM <a href="#pone.0099625-Shore1" class="ref-tip">[13]</a>. Our RNA-Seq, qRT-PCR and immuno-blot results demonstrated that <em>CRISPLD2</em> is a GC responsive gene, with DEX increasing its mRNA and protein levels. We found that <em>CRISPLD2</em> mRNA and protein levels increase in response to treatment with a known pro-inflammatory cytokine (IL1β), and that <em>CRISPLD2</em> knockdown increased the IL1β responsiveness of two inflammatory genes (i.e. <em>IL6</em> and <em>IL8</em>), suggesting that <em>CRISPLD2</em> may regulate immune response. Specifically, <em>CRISPLD2</em> may interfere with IL1β-induced cytokine production and act to reduce immune response via a negative feedback loop that can be activated by IL1β. This negative feedback loop may also play a role in cytokine level modulation in response to DEX treatment, as evidenced by the increased levels of <em>IL6</em> observed when both IL1β and DEX were administered to <em>CRISPLD2</em> knockdown ASM cells vs. when DEX was administered alone.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2" class="link-target"></a><p>The <em>CRISPLD2</em> gene maps to chromosome 16 at 16q24.1, covering 118.32 kb. According to AceView <a href="#pone.0099625-ThierryMieg1" class="ref-tip">[35]</a>, <em>CRISPLD2</em> is highly expressed in many human tissues, including lung and trachea. While AceView describes 14 different mRNAs as transcript products for this gene, only one of these (NM_031476) was part of the RefSeq annotation file used in the RNA-Seq analysis. Based on mapping of raw reads, each exon of this reported mRNA isoform was expressed in our ASM samples [<a href="#pone.0099625.s008">Figure S8</a>]. Results obtained after repeating the alignment and transcript reconstruction with parameters that allow for the discovery of novel isoforms while using the reference hg19 genome as a guide, also suggested that a single <em>CRISPLD2</em> mRNA isoform (NM_031476) was present in the ASM samples. Further studies are required to find out whether <em>CRISPLD2</em> splicing variation occurs in different individuals, at rare frequencies, and/or under other biological conditions.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3" class="link-target"></a><p>The cell-specific effects of GCs are well known <a href="#pone.0099625-Pratt1" class="ref-tip">[5]</a> and understanding the mechanisms of such specificity is an active area of research <a href="#pone.0099625-Reddy1" class="ref-tip">[12]</a>, <a href="#pone.0099625-Lu1" class="ref-tip">[36]</a>. For example, a study of the cell-specific responsiveness of <em>PTX3</em> to GCs found that in fibroblasts and endothelial cells, the GR functioned as a ligand-dependent transcription factor to induce <em>PTX3</em> gene expression, while in macrophages and myeloid dendritic cells, the GR repressed <em>PTX3</em> transcription by interfering with the action of other signaling pathways <a href="#pone.0099625-Doni1" class="ref-tip">[37]</a>. Comparison of baseline <em>CRISPLD2</em> expression in ASM vs. A549 pulmonary epithelial cells revealed that <em>CRISPLD2</em> is more highly expressed in ASM [<a href="#pone.0099625.s005">Figure S5A</a>]. This finding is consistent with the results from Reddy <em>et al</em>, who found low levels of <em>CRISPLD2</em> mRNA in A549 cells <a href="#pone.0099625-Reddy1" class="ref-tip">[12]</a>. While ASM <em>CRISPLD2</em> levels increased in response to DEX, levels decreased in A549 cells according to our experiments [<a href="#pone.0099625.s005">Figure S5B</a>] and were not significantly changed after 100 nM DEX treatment for 1 hour in the Reddy <em>et al</em> <a href="#pone.0099625-Reddy1" class="ref-tip">[12]</a> study. In a study using BEAS-2B cells, <em>CRISPLD2</em> expression was induced by both a glucocorticoid (fluticasone) and a long-acting beta-agonist (formoterol) <a href="#pone.0099625-Moodley1" class="ref-tip">[38]</a>. Further studies are required to understand the cell-specific expression of <em>CRISPLD2</em> and its interactions with other asthma medications.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4" class="link-target"></a><p>A hypothesis of the mechanism by which GCs activate <em>CRISPLD2</em> in ASM is provided by the chromatin immunoprecipitation followed by DNA sequencing (ChIP-Seq) results from Reddy <em>et al</em>, who sought to identify regions of the genome where GRs bind at various concentrations of DEX (100 nM 50 nM, 5 nM, 500 pM) <a href="#pone.0099625-Reddy1" class="ref-tip">[12]</a>. Their results, which are part of the Transcription Factor ChIP-Seq V4 results from ENCODE <a href="#pone.0099625-Wang3" class="ref-tip">[39]</a>, in/near <em>CRISPLD2</em> indicate that a region between its transcription start site (TSS) and first exon binds GRs [<a href="#pone.0099625.s008">Figure S8</a>]. While Reddy <em>et al</em> did not further characterize the GR binding of <em>CRISPLD2</em> because this gene was not DEX-responsive in their study, our RNA-Seq results along with their ChIP-Seq results suggest that GCs may activate <em>CRISPLD2</em> in ASM by binding to GR enhancer regions. However, future ChIP experiments are necessary to validate the exact location and sequence of this potential GR-binding site that may increase <em>CRISPLD2</em> expression in ASM cells.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5" class="link-target"></a><p>CCAAT/Enhancer Binding Proteins (CEBPs) are a family of transcription factors that help regulate a wide variety of processes, including inflammatory response <a href="#pone.0099625-LekstromHimes1" class="ref-tip">[40]</a>. Two of these factors, <em>CEBPB</em> and <em>CEBPD</em>, have been shown to be induced by GCs in lung epithelium <a href="#pone.0099625-Berg1" class="ref-tip">[41]</a> and skeletal muscle cells <a href="#pone.0099625-Yang1" class="ref-tip">[42]</a>. Additionally, the transcription of <em>CEBPB</em> has been shown to be induced by IL1, IL6, and lipopolysaccharide (LPS) <a href="#pone.0099625-LekstromHimes1" class="ref-tip">[40]</a>. ENCODE Transcription Factor ChIP-Seq V4 results suggest that CEBPs may also induce transcription of <em>CRISPLD2</em> [<a href="#pone.0099625.s008">Figure S8</a>]. Specifically, two CEBPB binding regions detected in untreated A549, HepG2 (hepatocellular carcinoma), IMR90 (fetal lung fibroblasts), and K562 (immortalized chronic myelogenous leukemia) cells between the TSS and first exon of <em>CRISPLD2</em> are salient as they have cluster scores of 1000 out of 1000 (all others near/in <em>CRISPLD2</em> have scores of &lt;435). According to our RNA-Seq data, both <em>CEBPB</em> and <em>CEBPD</em> were expressed in ASM [<a href="#pone.0099625.s009">Figure S9</a>], but only <em>CEBPD</em> had significantly increased mRNA levels in response to treatment with DEX [Q-value 4.8E-04, Ln of fold-change 1.47]. Thus, if DEX does indeed mediate changes in expression of <em>CRISPLD2</em> in the ASM via CEBPB binding, it likely does so via a post-translational mechanism such as by changing phosphorylation <a href="#pone.0099625-Berg2" class="ref-tip">[43]</a>. Future ChIP experiments are necessary to confirm that CEBPB binding contributes to <em>CRISPLD2</em> expression in ASM.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6" class="link-target"></a><p>The potential presence of both GR and CEBPB binding sites in <em>CRISPLD2</em> is consistent with our observation that both DEX and IL1β increased <em>CRISPLD2</em> mRNA expression, findings that may seem at odds because one mechanism by which GCs act is to decrease expression of cytokines such as IL1β. Because IL1 induces <em>CEBPB</em> expression <a href="#pone.0099625-Akira1" class="ref-tip">[44]</a>, IL1β may induce transcription of <em>CRISPLD2</em> via increased binding of CEBPB to a CEBPB transcription factor in <em>CRISPLD2</em> in the absence of a GC [<a href="#pone-0099625-g003">Figure 3C</a>]. The control ASM cell lines used to obtain RNA-Seq results expressed low levels of <em>IL1β</em> both at baseline (FPKM  = 0.04) and when treated with DEX (FPKM  = 0.01), and thus, we did not characterize the relationships among DEX, <em>IL1β</em> and <em>CRISPLD2</em> that would be expected of a disease involving high levels of IL1β. Further studies examining changes of expression of <em>CRISPLD2</em> under varying concentrations of DEX and IL1βwould help to clarify the relationships among them.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7" class="link-target"></a><p><em>CRISPLD2</em> (a.k.a. <em>Lgl1</em> in rat) has been identified as a developmental gene that modulates branching morphogenesis in fetal rat lung <a href="#pone.0099625-Oyewumi1" class="ref-tip">[45]</a> and its variants have been related to non-syndromic cleft lip with or without cleft palate in human association studies <a href="#pone.0099625-Chiquet1" class="ref-tip">[46]</a>. In a recent study of CRISPLD2's role in endotoxin regulation, CRISPLD2 was found to bind to LPS, thereby preventing LPS from binding target peripheral blood mononuclear cells (PBMCs) and inhibiting the release of proinflammatory markers (i.e. TNFα and IL6) by these target cells <a href="#pone.0099625-Wang2" class="ref-tip">[31]</a>. Endogenous CRISPLD2 in healthy human serum was found to downregulate LPS-induced TNFα production <em>in vitro</em>, and CRISPLD2 was found to protect mice from endotoxic shock. A subsequent study of CRISPLD2 found that its protein levels were decreased in blood serum of patients with septic shock compared to controls, patients with sepsis and patients with severe sepsis <a href="#pone.0099625-Wang4" class="ref-tip">[47]</a>. However, CRISPLD2 levels were not related to clinical outcomes (e.g., survival). Although their use is highly controversial, GCs have been used to treat septic shock <a href="#pone.0099625-Patel1" class="ref-tip">[48]</a>, and thus, one area of future study could be to characterize the relationships among GCs, CRISPLD2 and LPS in septic shock. Our findings that <em>CRISPLD2</em> knockdown increased <em>IL6</em> and <em>IL8</em> levels, while DEX increased <em>CRISPLD2</em> levels, taken together with the study by Wang, <em>et al</em> demonstrating that CRISPLD2 plays a role in endotoxin regulation, suggest that CRISPLD2, in part, may modulate asthma phenotypes by decreasing the ASM inflammatory response to exogenous LPS-containing bacteria. However, the role of LPS and endotoxin in the development of asthma or its exacerbations is not fully understood <a href="#pone.0099625-Doreswamy1" class="ref-tip">[49]</a>, and further studies are needed to test potential roles of <em>CRIPSLD2</em> in linking GCs and inflammation to endotoxin regulation.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8" class="link-target"></a><p>In addition to published evidence that <em>CRISPLD2</em> may indirectly play a role in asthma, SNPs of this gene were associated with two asthma pharmacogenetic traits measured in asthma clinical trials. Each of these traits was relevant to GC response and ASM contractility: ICS resistance was a direct measure of treatment with a glucocorticoid, while bronchodilator response was a measure of beta-agonist effects, and the beta-agonist and glucocorticoid pathways are known to overlap <a href="#pone.0099625-Kaur1" class="ref-tip">[29]</a>. Interestingly, the region of association with BDR overlaps with the GR and CEBPB DNA-binding regions mentioned above [<a href="#pone.0099625.s010">Figure S10</a>]. In a previous study, the long-acting beta-agonist formoterol was found to activate a CEBP-luciferase reporter construct in BEAS-2B cells, and mice with a lung epithelial-specific knockout of CEBPB were found to have an impaired suppression of LPS-induced neutrophilia by formoterol compared to control littermates <a href="#pone.0099625-Roos1" class="ref-tip">[50]</a>. Thus, it is possible that this region of association with bronchodilator response reflects a functional change that alters GR and/or CEBPB binding, but further experiments are required to test this hypothesis. While the nominal associations with ICS resistance and bronchodilator response do not reach genome-wide significance, and hence, would not suggest that <em>CRISPLD2</em> variants be prioritized for further study based on the GWAS data alone, in the context of the current GC responsiveness results, they suggest that specific regions in/near <em>CRISPLD2</em> may modulate asthma phenotypes in humans.</p>
<a id="article1.body1.sec3.p9" name="article1.body1.sec3.p9" class="link-target"></a><p>Results from two publicly available gene expression microarray studies that have measured the effect of GCs on human ASM cells using <em>in vitro</em> models supported our <em>CRISPLD2</em> findings [<a href="#pone-0099625-t003">Table 3</a>]. The first study by Masuno, <em>et al</em> (GSE34313) investigated the effects of DEX at 4 and 24 hours and focused on the functional validation of the <em>KLF15</em> gene, which was found to modulate airway hyperresponsiveness, but not inflammatory response, in an ovalbumin challenge mouse asthma model <a href="#pone.0099625-Masuno1" class="ref-tip">[17]</a>. While not among their top-ranked findings, <em>CRISPLD2</em> expression was increased by DEX at both 4 and 24 hours in the experiments by Masuno, <em>et al.</em> Consistent with our findings, DEX increased the expression of <em>CRISPLD2</em> at both 4 and 24 hours in the experiments by Masuno, <em>et al.</em> Consistent with the results of Masuno, <em>et al</em>, <em>KLF15</em> was among the top differentially expressed genes that we identified. Another microarray study of the ASM transcriptome by Misior <em>et al</em> (GSE13168) focused on the overlap of GC and beta-agonist gene responses <a href="#pone.0099625-Misior1" class="ref-tip">[18]</a>. Of most relevance to our work, <em>CRISPLD2</em> had significantly increased expression levels when ASM cells were treated with fluticasone. Further, the effect of fluticasone on <em>CRISPLD2</em> expression was diminished when ASM were also treated with IL1β and/or EGF pro-inflammatory cytokines [<a href="#pone-0099625-t003">Table 3</a>]. This suggests that <em>CRISPLD2</em> levels were increased by both EGF and IL1β and is consistent with our result that IL1β increased <em>CRISPLD2</em> expression [<a href="#pone-0099625-g003">Figures 3C and 3D</a>].</p>
<a id="article1.body1.sec3.p10" name="article1.body1.sec3.p10" class="link-target"></a><p>While <em>in vitro</em> studies of the ASM response to GCs that use RNA-Seq have not been published, a recent study used RNA-Seq to investigate the effects of a 2-week course of oral prednisolone on ASM gene expression in patients with mild asthma, using ASM extracted via laser caption microdissection from bronchoscopy samples <a href="#pone.0099625-Yick1" class="ref-tip">[51]</a>. Comparing samples from 6 patients assigned to GC treatment vs. 6 patients assigned to placebo, this study found that 15 genes were significantly differentially expressed between groups, and two of the 15 genes were also associated with airway hyperresponsiveness. Of these 15 genes, only one was significant in our study (i.e. <em>SYNPO2</em>, adjusted P-value 0.015) [<a href="#pone.0099625.s014">Table S3</a>]. Future studies of the <em>in vivo</em> GC response of ASM may help clarify the differences between <em>in vitro</em> and <em>in vivo</em> study results.</p>
<a id="article1.body1.sec3.p11" name="article1.body1.sec3.p11" class="link-target"></a><p>Our study has identified well-known and more novel GC responsive genes, but it was also subject to limitations. We used cell lines from four white male subjects, and based on both RNA-Seq and qRT-PCR results, there was significant variability in gene expression levels among subjects. For example, the qRT-PCR data in <a href="#pone-0099625-g002">Figure 2</a> suggests that there is an inverse correlation between levels of <em>CRISPLD2</em> and <em>C13orf15</em>. That is, DEX seemed to induce greater levels <em>CRISPLD2</em> in cells that had lower levels of <em>C13orf15</em>. Because the cell lines used were derived from “an outbred human population,” the heterogeneity of responses is consistent with the complexity of the pathophysiology of asthma. Future studies with a larger number of individuals and individuals of diverse gender and racial/ethnic backgrounds may shed light on individualized profiles of GC response, including identifying individuals who are GC insensitive, and increase our understanding of how the expression of various genes relates to each other. Our RNA-Seq analyses were limited to the hg19 RefSeq annotation files downloaded from Illumina's iGenomes project. Thus, we did not characterize the expression long-non-coding RNA or mRNA transcript isoforms that were not part of the reference file used. We opted for use of a well-annotated reference file for our investigation of the ASM transcriptome to reduce the number of false-positive results that we may have selected for follow-up. Future studies with more comprehensive annotation files and a greater number of individuals and/or greater sequencing depth will yield additional insight into the ASM transcriptome.</p>
<a id="article1.body1.sec3.p12" name="article1.body1.sec3.p12" class="link-target"></a><p>While ASM is a target tissue in the GC treatment of asthma, our ASM samples were not from asthma patients. Although the study by Masuno <em>et al</em> found that there was general concordance between response to DEX in 16 genes among four control ASM cell lines and those of two asthma patients <a href="#pone.0099625-Masuno1" class="ref-tip">[17]</a>, there are likely some differences in the GC response between asthma patients and individuals without asthma. Further studies that include ASM from asthma patients may help clarify such differences. Finally, it is known that the response to GCs changes in time. For example, the Masuno <em>et al</em> study compared the ASM GC response at both 4 and 24 hours, and it found that while some genes had consistent changes at both time points, others had varied ones <a href="#pone.0099625-Masuno1" class="ref-tip">[17]</a>. We selected an 18-hour DEX treatment period partly because the study by Masuno <em>et al</em> suggested that the set of genes regulated by the GR expands greatly between 4 and 24 hours. Future studies that evaluate the GC response over time would further define how GCs alter the ASM transcriptome.</p>
<a id="article1.body1.sec3.p13" name="article1.body1.sec3.p13" class="link-target"></a><p>In summary, we identified 316 GC responsive genes in primary ASM cell lines. The <em>CRISPLD2</em> gene was selected for functional studies based on having SNPs that were nominally associated with ICS resistance and bronchodilator response, as well as having published studies relating it to lung development and endotoxin response. Based on results of <em>in vitro</em> experiments, <em>CRISPLD2</em> was found to be a modulator of IL1β response in ASM cells. Our findings identified <em>CRISPLD2</em> as a novel asthma pharmacogenetics candidate gene and provide transcriptome data to further explore the anti-inflammatory effects of GCs in the ASM.</p>
</div>

<div id="section4" class="section toc-section"><a id="s4" name="s4" data-toc="s4" class="link-target" title="Materials and Methods"></a><h2>Materials and Methods</h2>
<div id="section1" class="section toc-section"><a id="s4a" name="s4a" class="link-target" title="Ethics Statement"></a>
<h3>Ethics Statement</h3>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1" class="link-target"></a><p>Lung tissue was obtained from the National Disease Resource Interchange (NDRI) and its use approved by the University of Pennsylvania Institutional Review Board; use of the cells does not constitute human research. Approval of the GWAS studies was issued by the Partners Healthcare, Inc. Partners Human Research Committee, which ensured that all procedures followed were in accordance with the ethical standards of the responsible committee on human experimentation, including obtaining written informed consent for all study participants.</p>
</div>

<div id="section2" class="section toc-section"><a id="s4b" name="s4b" class="link-target" title="ASM Cell Culture and GC Treatment for RNA-Seq Experiment"></a>
<h3>ASM Cell Culture and GC Treatment for RNA-Seq Experiment</h3>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1" class="link-target"></a><p>Primary ASM cells were isolated from four white aborted lung transplant donors with no chronic illness. ASM cell cultivation and characterization were described previously <a href="#pone.0099625-Panettieri2" class="ref-tip">[52]</a>, <a href="#pone.0099625-Cooper1" class="ref-tip">[53]</a>. Passages 4 to 7 ASM cells maintained in Ham's F12 medium supplemented with 10% FBS were used in all experiments. For the RNA-Seq and qRT-PCR validation experiments, cells from each donor were treated with 1 µM DEX (Sigma-Aldrich, St. Louis, MO) or control vehicle for 18 h. For other experiments, cells were treated with 100 nM DEX for 24 or 48 h.</p>
</div>

<div id="section3" class="section toc-section"><a id="s4c" name="s4c" class="link-target" title="RNA-Seq Library Construction and Sequencing"></a>
<h3>RNA-Seq Library Construction and Sequencing</h3>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1" class="link-target"></a><p>Total RNA was extracted from control and DEX-treated ASM cells using the miRNAeasy mini kit (Qiagen Sciences, Inc., Germantown, MD). Approximately 1 µg of RNA from each sample was used to generate RNA-Seq cDNA libraries for sequencing using the TruSeq RNA Sample Prep Kit v2 (Illumina, Inc., San Diego, CA). Sample preparation followed the manufacturer's protocol with a workflow that included isolation of poly-adenylated RNA molecules using poly-T oligo-attached magnetic beads, enzymatic RNA fragmentation, cDNA synthesis, ligation of bar-coded adapters, and PCR amplification. Ambion External RNA Controls Consortium (ERCC) RNA Spike-In Control Mix 1 (Life Technologies Corporation, Carlsbad, CA) was added to the samples. The amplified cDNA fragments were analyzed using the 2100 Bioanalyzer (Agilent Technologies, Inc., Santa Clara, CA) to determine fragment quality and size. Library concentrations were determined by Qubit Fluorometric Quantitiation (Life Technologies Corporation, Carlsbad, CA). Sequencing of 75 bp paired-end reads was performed with an Illumina HiSeq 2000 instrument at the Partners HealthCare Center for Personalized Genetic Medicine (Boston, MA).</p>
</div>

<div id="section4" class="section toc-section"><a id="s4d" name="s4d" class="link-target" title="RNA-Seq Data Analysis"></a>
<h3>RNA-Seq Data Analysis</h3>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1" class="link-target"></a><p>Preliminary processing of raw reads was performed using Casava 1.8 (Illumina, Inc., San Diego, CA). Subsequently, Taffeta scripts (<a href="https://github.com/blancahimes/taffeta">https://github.com/blancahimes/taffeta</a>) were used to analyze RNA-Seq data, which included use of FastQC <a href="#pone.0099625-Andrews1" class="ref-tip">[54]</a> (v.0.10.0) to obtain overall QC metrics. Based on having sequence bias in the initial 12 bases on the 5′ end of reads, the first 12 bases of all reads were trimmed with the FASTX Toolkit (v.0.0.13) <a href="#pone.0099625-Pearson1" class="ref-tip">[55]</a>. FastQC reports for each sample revealed that each was successfully sequenced. Trimmed reads for each sample were aligned to the reference hg19 genome and known ERCC transcripts using TopHat <a href="#pone.0099625-Kim1" class="ref-tip">[56]</a> (v.2.0.4), while constraining mapped reads to be within reference hg19 or ERCC transcripts. Additional QC parameters were obtained to assess whether reads were appropriately mapped. Bamtools <a href="#pone.0099625-Barnett1" class="ref-tip">[57]</a> was used to the number of mapped reads, including junction spanning reads. The Picard Tools (<a href="http://picard.sourceforge.net">http://picard.sourceforge.net</a>) RnaSeqMetrics function was used to compute the number of bases assigned to various classes of RNA, according to the hg19 refFlat file available as a UCSC Genome Table. For each sample, Cufflinks <a href="#pone.0099625-Trapnell2" class="ref-tip">[21]</a> (v.2.0.2) was used to quantify ERCC Spike-In and hg19 transcripts based on reads that mapped to the provided hg19 and ERCC reference files. For three samples that contained ERCC Spike-Ins, we created dose response curves (i.e. plots of ERCC transcript FPKM vs. ERCC transcript molecules) following the manufacturer's protocol <a href="#pone.0099625-Mortazavi1" class="ref-tip">[58]</a>. Ideally, the slope and R<sup>2</sup> would equal 1.0. For our samples (Dex.2, Control.4, Dex.4), the slope (R<sup>2</sup>) values were 0.90 (0.90), 0.92 (0.84), 0.82 (0.86), respectively. Raw read plots were created by displaying bigwig files for each sample in the UCSC Genome Browser.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2" class="link-target"></a><p>Differential expression of genes and transcripts in samples treated with DEX vs. untreated samples was obtained using Cuffdiff <a href="#pone.0099625-Trapnell2" class="ref-tip">[21]</a> (v.2.0.2) with the quantified transcripts computed by Cufflinks (v.2.0.2), while applying bias correction. The CummeRbund <a href="#pone.0099625-Trapnell3" class="ref-tip">[59]</a> R package (v.0.1.3) was used to measure significance of differentially expressed genes and create plots of the results. As a positive control of gene expression, the FPKM values for four housekeeping genes (i.e., <em>B2M</em>, <em>GABARAP</em>, <em>GAPDH</em>, <em>RPL19</em>) were obtained. Each had high FPKM values that did not differ significantly by treatment status [<a href="#pone.0099625.s011">Figure S11</a>]. The NIH Database for Annotation, Visualization and Integrated Discovery (DAVID) was used to perform gene functional annotation clustering using Homo Sapiens as background, and default options and annotation categories (Disease: OMIM_DISEASE; Functional Categories: COG_ONTOLOGY, SP_PIR_KEYWORDS, UP_SEQ_FEATURE; Gene_Ontology: GOTERM_BP_FAT, GOTERM_CC_FAT, GOTERM_MF_FAT; Pathway: BBID, BIOCARTA, KEGG_PATHWAY; Protein_Domains: INTERPRO, PIR_SUPERFAMILY, SMART) <a href="#pone.0099625-Huangda1" class="ref-tip">[28]</a>. The RNA-Seq data is available at the Gene Expression Omnibus Web site (<a href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>) under accession GSE52778.</p>
</div>

<div id="section5" class="section toc-section"><a id="s4e" name="s4e" class="link-target" title="Genome-wide Association Studies"></a>
<h3>Genome-wide Association Studies</h3>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1" class="link-target"></a><p>Two GWAS of asthma-related traits that may be related to ASM GC response were selected to measure association of SNPs in/near top differentially expressed genes identified by RNA-Seq:</p>


<ol class="order">

<li>A GWAS of inhaled corticosteroid (ICS) response was conducted in 723 non-Hispanic white asthmatics from the following drug clinical trials: Childhood Asthma Management Program (CAMP) <a href="#pone.0099625-Childhood1" class="ref-tip">[60]</a>, Leukotriene Modifier or Corticosteroid Salmeterol study (LOCCS) <a href="#pone.0099625-Peters1" class="ref-tip">[61]</a>, and subsets of trials within the Childhood Asthma Research and Education (CARE) network <a href="#pone.0099625-Guilbert1" class="ref-tip">[62]</a>, and the Asthma Clinical Research Network (ACRN) <a href="#pone.0099625-Denlinger1" class="ref-tip">[63]</a> participating in the NHLBI SNP Health Association Resource (SHARe) Asthma Resource project (SHARP). ICS response was quantified as the percent change in pre-BD FEV<sub>1</sub> following 4–8 weeks of ICS therapy [i.e. (onICSpreFEV<sub>1</sub> – offICSpreFEV<sub>1</sub>)/offICSpreFEV<sub>1</sub>)]. Association of imputed SNPs (based on the June 2010 release of the 1000 Genome Project reference) with ICS response was measured using a linear regression model using PLINK <a href="#pone.0099625-Purcell1" class="ref-tip">[64]</a>.</li>

<li>Previously, a GWAS of bronchodilator response was conducted in 1,644 non-Hispanic white asthmatics from drug clinical trials <a href="#pone.0099625-Himes1" class="ref-tip">[30]</a>. Briefly, bronchodilator response was quantified as the percent change in FEV<sub>1</sub> in response to administration of a β<sub>2</sub>-agonist bronchodilator (BD) [i.e. (post-BD FEV<sub>1</sub> – pre-BD FEV<sub>1</sub>)/pre-BD FEV<sub>1</sub>], and association of single nucleotide polymorphisms (SNPs) with bronchodilator response was measured using a linear regression model, while adjusting for age, sex, and height. Results for 4,571,615 imputed SNPs (based on the June 2010 release of the 1000 Genome Project reference) were available.</li>

</ol><a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2" class="link-target"></a><p>Originally computed hg18 coordinates were converted to those of the hg19 human genome assembly using the liftOver tool of the UCSC genome browser. Variant names were converted to those of the dbSNP build 138.</p>
</div>

<div id="section6" class="section toc-section"><a id="s4f" name="s4f" class="link-target" title="Publicly Available GC-treated ASM Microarray Expression Data Analysis"></a>
<h3>Publicly Available GC-treated ASM Microarray Expression Data Analysis</h3>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1" class="link-target"></a><p>Datasets from two publicly available gene expression microarray studies that measured the effect of GCs on human ASM cells using <em>in vitro</em> models were obtained from the Gene Expression Omnibus (GEO): GSE34313 and GSE13168. Raw signals of the GSE34313 experiment, which used four replicates of a single ASM cell line to investigate the effects of DEX at 4 and 24 hours using the Agilent-014850 Whole Human Genome Microarray 4x44K platform <a href="#pone.0099625-Masuno1" class="ref-tip">[17]</a>, were preprocessed with background correction and quantile normalization using the limma R package <a href="#pone.0099625-Smyth1" class="ref-tip">[65]</a>. The GSE13168 dataset consisted of ASM cell cultures extracted from four donor tracheas that were stimulated with a GC (fluticasone) or a protein kinase A inhibitor as well as pro-inflammatory agents (i.e. EGF and/or IL1β) or control and used the Affymetrix U133A platform to measure gene expression changes <a href="#pone.0099625-Misior1" class="ref-tip">[18]</a>. Its raw signal intensities were preprocessed with RMA as implemented in the affy R package <a href="#pone.0099625-Gautier1" class="ref-tip">[66]</a>. For both datasets, gene-based differential expression analysis was conducted using the limma R package by averaging probe intensities for individual genes <a href="#pone.0099625-Smyth1" class="ref-tip">[65]</a>.</p>
</div>

<div id="section7" class="section toc-section"><a id="s4g" name="s4g" class="link-target" title="Cell culture, Chemical Treatment and siRNA Transfection"></a>
<h3>Cell culture, Chemical Treatment and siRNA Transfection</h3>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1" class="link-target"></a><p>ASM cells were maintained in Ham's F12 medium supplemented with 24 mM HEPES, 1.7 mM CaCl2, 12 Mm NaOH and 10% FBS. For chemical treatment, cells grown in the above medium were washed with PBS and switched to the serum deprivation medium. A549 human lung epithelial cells were maintained in high-glucose DMEM medium containing 10% FBS. For chemical treatment, A549 cells were washed and switched to DMEM/F12(1∶1) medium supplemented with 3% dialyzed FBS. 100 nM DEX (Sigma-Aldrich Corporation, St. Louis, MO) or 5 ng/mL IL1β (Invitrogen, Life Technologies, Grand Island, NY) was then added to the medium. Transfection of <em>CRISPLD2</em> and non-targeting siRNA (siRNA universal non-targeting control 1, Sigma-Aldrich Corporation, St. Louis, MO) was performed using DharmaFECT 1 reagent according to the recommended protocol from the manufacturer (Thermo Scientific, Lafayette, CO). The final concentration of siRNA was 25 nM; siRNA sequences for <em>CRSIPLD2</em> knockdown were 5′-GAACCAACAUCUAUGCAGA(dT)(dT)-3′ and 5′-UCUGCAUAGAUCUUGGUUC(dT)(dT)-3′.</p>
</div>

<div id="section8" class="section toc-section"><a id="s4h" name="s4h" class="link-target" title="Quantitative Real-Time PCR (qRT-PCR) Analysis"></a>
<h3>Quantitative Real-Time PCR (qRT-PCR) Analysis</h3>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1" class="link-target"></a><p>Total RNA was isolated from cells by using QIAshredder and RNeasy kits (Qiagen Sciences, Inc., Germantown, MD). Oligo(dT)-primed cDNA was prepared from 50 ng of total RNA by using SuperScript III First-strand Synthesis System (Invitrogen, Life Technologies, Grand Island, NY). qRT-PCR was set up in the presence of 0.5 µM primers by using QuantiTect SYBR Green PCR kit (Qiagen Sciences, Inc., Germantown, MD). qRT-PCR was performed on an StepOne Plus real time PCR machine (Applied Biosystems, Life Technologies, Grand Island, NY). β-actin was used as an internal control for data normalization. Each sample was measured in triplicate.</p>
</div>

<div id="section9" class="section toc-section"><a id="s4i" name="s4i" class="link-target" title="Immunoblot Analysis"></a>
<h3>Immunoblot Analysis</h3>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1" class="link-target"></a><p>Cells were washed with PBS and lysed in NP-40 lysing buffer (50 mM Tris-HCl pH 7.5, 150 mM NaCl<sub>2</sub>, 0.5% Nonidet P-40) containing protease inhibitor cocktail (Roche, Genentech, Inc., South San Francisco, CA). For the secreted <em>CRISPLD2</em> protein analysis, the medium was concentrated using Ultra-15 10 K Centrifugal Filter Devices (Merck Millipore, Tullagreen, IRL). Protein samples were separated on NuPAGE 4–12% Bis-Tris gels (Invitrogen, Life Technologies, Grand Island, NY) and transferred to PVDF membranes (Bio-Rad, Life Science Research, Hercules, CA). Immunoblot signals were developed using SuperSignal West Pico Chemiluminescent Substrates (Pierce Biotechnology, Inc., Rockford, IL). Primary antibodies used in the study included rabbit polyclonal CRISPLD2 antibody (Abcam, Cambridge, MA) and mouse monoclonal β-actin antibody (Santa Cruz Biotechnology, Inc., Dallas, TX). Relative sample intensities were computed with scanned and quantified immunoblot data obtained using Image J software <a href="#pone.0099625-Schneider1" class="ref-tip">[67]</a>. Each Western blot experiment was performed 3 times; representative image results are shown. Semi-quantitative results are reported as the mean +/− SE.</p>
</div>
</div>

<div id="section5" class="section toc-section"><a id="s5" name="s5" data-toc="s5" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0099625"></div><div class="supplementary-material"><a name="pone.0099625.s001" id="pone.0099625.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0099625.s001">Figure S1. </a></h3><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Estimated read coverage across transcripts for each sample.</strong> Position and coverage are normalized by adjusting for transcript lengths and total number of reads mapped per sample.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0099625.s001">https://doi.org/10.1371/journal.pone.0099625.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(PNG)</p>
</div><div class="supplementary-material"><a name="pone.0099625.s002" id="pone.0099625.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0099625.s002">Figure S2. </a></h3><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Confirmation of qRT-PCR results supporting RNA-Seq findings in the ASM cell line that was not used for validation in </strong><a href="#pone-0099625-g001"><strong>Figure 1B</strong></a><strong> or </strong><a href="#pone-0099625-g002"><strong>Figure 2</strong></a><strong>.</strong> The mRNA levels of the indicated genes were measured by qRT-PCR and the folds of change induced by DEX were calculated for a single replicate.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0099625.s002">https://doi.org/10.1371/journal.pone.0099625.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pone.0099625.s003" id="pone.0099625.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0099625.s003">Figure S3. </a></h3><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Association of SNPs near </strong><strong><em>CRISPLD2</em></strong><strong> with A) bronchodilator response and B) ICS resistance.</strong> The x-axes denote position along Chromosome 16 according to the hg18 genome build. The y-axes denote –Log<sub>10</sub>(P) corresponding to 1000GP imputed data P-values. LD between the SNP with the lowest P-values (chr16:83425008 and chr16:83522091) to each SNP in the plot is denoted in colors and was computed according to 1000GP June 2010 CEU data. Plot was created using LocusZoom <a href="#pone.0099625-Pruim1" class="ref-tip">[68]</a>.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0099625.s003">https://doi.org/10.1371/journal.pone.0099625.s003</a></p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pone.0099625.s004" id="pone.0099625.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0099625.s004">Figure S4. </a></h3><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Time and dose dependent effects of DEX on CRISPLD2 expression.</strong> <strong>A</strong>) ASM cells were treated with 100 nM DEX for 24 and 48 h. <strong>B</strong>) ASM cells were treated with DEX at indicated concentrations for 24 h. CRISPLD2 protein was measured by immune-blotting.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0099625.s004">https://doi.org/10.1371/journal.pone.0099625.s004</a></p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pone.0099625.s005" id="pone.0099625.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0099625.s005">Figure S5. </a></h3><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>DEX induced expression change of </strong><strong><em>CRISPLD2</em></strong><strong> in A549 pulmonary epithelial cells.</strong> <strong>A</strong>) Basal mRNA level of <em>CRISPLD2</em> in A549 and ASM cells. <strong>B</strong>) A549 cells were treated with 100 nM DEX for 24 h and <em>CRISPLD2</em> mRNA levels were measured by qRT-PCR. Relative values of gene expression shown. Experiment was performed in triplicate using cells from a single donor. ** <em>P</em>&lt;0.005, * <em>P</em>&lt;0.05 (<em>t</em> test).</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0099625.s005">https://doi.org/10.1371/journal.pone.0099625.s005</a></p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pone.0099625.s006" id="pone.0099625.s006" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0099625.s006">Figure S6. </a></h3><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong><em>CRISPLD2</em></strong><strong> regulates </strong><strong><em>IL8</em></strong><strong> expression.</strong> ASM cells were transfected with <em>CRISPLD2</em>-specific siRNA or non-targeting (NT) siRNA, and 72 h later cell were treated with 5 ng/mL IL1β for 24 h. <em>IL8</em> mRNA expression was determined by qRT-PCR. Normalized mRNA levels shown. All measurements were performed in triplicate samples. * <em>P</em>&lt;0.05 (<em>t</em> test).</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0099625.s006">https://doi.org/10.1371/journal.pone.0099625.s006</a></p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pone.0099625.s007" id="pone.0099625.s007" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0099625.s007">Figure S7. </a></h3><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Effect of </strong><strong><em>CRISPLD2</em></strong><strong> knockdown on GR target gene expression.</strong> ASM cells were first transfected with <em>CRISPLD2</em>-specific or NT siRNA and then stimulated with 100 nM DEX for 24 h. Induced expression (DEX treatment vs. control) of three GR target genes was determined by qRT-PCR. None of the target genes were found to be differentially expressed (i.e. all had t-test <em>P</em>&gt;0.05).</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0099625.s007">https://doi.org/10.1371/journal.pone.0099625.s007</a></p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pone.0099625.s008" id="pone.0099625.s008" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0099625.s008">Figure S8. </a></h3><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Raw RNA-Seq read plots for four ASM cell lines treated with DEX (red) or left untreated (blue) along the human (hg19) chromosome 16 region containing </strong><strong><em>CRISPLD2</em></strong><strong>.</strong> Reads mapped to each exon of the RefSeq version of CRISPLD2. An increased number of mapped reads is observed in each sample after DEX treatment vs. when left untreated. Shown below the RefSeq gene track are ENCODE ChIP-Seq tracks: four are for sites found to bind the GR (official gene name <em>NR3C1</em>) in A549 pulmonary epithelial cells at various dosages of DEX (100 nM 50 nM, 5 nM, 500 pM) <a href="#pone.0099625-Reddy1" class="ref-tip">[12]</a>, and below these are Transcription Factor ChIP-Seq V4 results for GR, CEBPB, and CEBPD. Darker vertical lines represent binding sites with higher scores (i.e. detected more strongly).</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0099625.s008">https://doi.org/10.1371/journal.pone.0099625.s008</a></p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pone.0099625.s009" id="pone.0099625.s009" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0099625.s009">Figure S9. </a></h3><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>RNA-Seq results from four ASM cell lines treated with DEX expressed as FPKM for </strong><strong><em>CEBPB</em></strong><strong> and </strong><strong><em>CEBPD</em></strong><strong> by condition status (i.e. DEX vs. untreated) show the presence of both genes.</strong> DEX treatment did not significantly change the expression levels of <em>CEBPB</em> but did change the expression levels of <em>CEBPD</em> (Q-value 4.8E-04, Ln of fold-change 1.47).</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0099625.s009">https://doi.org/10.1371/journal.pone.0099625.s009</a></p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pone.0099625.s010" id="pone.0099625.s010" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0099625.s010">Figure S10. </a></h3><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Region of </strong><strong><em>CRISPLD2</em></strong><strong> where SNPs were most strongly associated with bronchodilator response (BDR) along with GR- and CEBPB-binding sites identified by ENCODE Transcription Factor ChIP-Seq V4 results.</strong> The x-axis denotes position along Chromosome 16 in hg19 genome build coordinates. The vertical axis of the BDR GWAS Results denotes –Log<sub>10</sub>(P-values), and the horizontal line at 1.3 represents a nominal significance threshold of P-value = 0.05. Shown below the RefSeq gene track are ENCODE ChIP-Seq tracks for sites found to bind the GR (official gene name <em>NR3C1</em>) in A549 pulmonary epithelial cells at various dosages of DEX (100 nM 50 nM, 5 nM, 500 pM) <a href="#pone.0099625-Reddy1" class="ref-tip">[12]</a>, and below these are Transcription Factor ChIP-Seq V4 results for GR, CEBPB, and CEBPD. Darker ChIP-Seq regions represent binding sites with higher scores (i.e. detected more strongly).</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0099625.s010">https://doi.org/10.1371/journal.pone.0099625.s010</a></p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pone.0099625.s011" id="pone.0099625.s011" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0099625.s011">Figure S11. </a></h3><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>RNA-Seq results from four ASM cell lines treated with DEX expressed as FPKM for four housekeeping genes (i.e. </strong><strong><em>B2M</em></strong><strong>, </strong><strong><em>GABARAP</em></strong><strong>, </strong><strong><em>GAPDH</em></strong><strong>, </strong><strong><em>RPL19</em></strong><strong>) by condition status (i.e. DEX vs. untreated) show high levels of expression for each gene that did not significantly differ with DEX treatment.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0099625.s011">https://doi.org/10.1371/journal.pone.0099625.s011</a></p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pone.0099625.s012" id="pone.0099625.s012" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0099625.s012">Table S1. </a></h3><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1" class="link-target"></a><p class="preSiDOI">Number of various read types per sample in millions. Values in parenthesis for unmapped and mapped reads correspond to percentages of total reads per sample, while values in parenthesis for all other entries correspond to percentages of total mapped reads per sample.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0099625.s012">https://doi.org/10.1371/journal.pone.0099625.s012</a></p><a id="article1.body1.sec5.supplementary-material12.caption1.p2" name="article1.body1.sec5.supplementary-material12.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0099625.s013" id="pone.0099625.s013" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0099625.s013">Table S2. </a></h3><a id="article1.body1.sec5.supplementary-material13.caption1.p1" name="article1.body1.sec5.supplementary-material13.caption1.p1" class="link-target"></a><p class="preSiDOI">Percentage of mapped bases according to hg19 reference refFlat file base type.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0099625.s013">https://doi.org/10.1371/journal.pone.0099625.s013</a></p><a id="article1.body1.sec5.supplementary-material13.caption1.p2" name="article1.body1.sec5.supplementary-material13.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0099625.s014" id="pone.0099625.s014" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0099625.s014">Table S3. </a></h3><a id="article1.body1.sec5.supplementary-material14.caption1.p1" name="article1.body1.sec5.supplementary-material14.caption1.p1" class="link-target"></a><p class="preSiDOI">Significantly differentially expressed genes.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0099625.s014">https://doi.org/10.1371/journal.pone.0099625.s014</a></p><a id="article1.body1.sec5.supplementary-material14.caption1.p2" name="article1.body1.sec5.supplementary-material14.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0099625.s015" id="pone.0099625.s015" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0099625.s015">Table S4. </a></h3><a id="article1.body1.sec5.supplementary-material15.caption1.p1" name="article1.body1.sec5.supplementary-material15.caption1.p1" class="link-target"></a><p class="preSiDOI">Functional annotation clusters obtained with the NIH DAVID tool using differentially expressed genes in <a href="#pone.0099625.s014">Table S3</a>. Clusters with enrichment scores &gt;1.5 are shown. Individual P-values listed correspond to EASE Scores, or modified Fisher Exact P-Values computed by DAVID.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0099625.s015">https://doi.org/10.1371/journal.pone.0099625.s015</a></p><a id="article1.body1.sec5.supplementary-material15.caption1.p2" name="article1.body1.sec5.supplementary-material15.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div></div>



<div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>Conceived and designed the experiments: BEH XJ RAP QL. Wrote the paper: BEH XJ QL. Analyzed RNA-Seq data: BEH. Analyzed microarray data: BEH RMW. Performed ASM DEX experiments for RNA-Seq: CN WJ MJ. Performed RNA-Seq experiments: PW RH BK. Performed all other experiments: XJ QW. Performed ICS Resistance GWAS: QD JL. Contributed to the overall study concept and design: KGT STW. Edited the manuscript and approved its final draft: BEH XJ PW RH QW BK RMW QD JL CN WJ MJ RAP KGT STW QL.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0099625-Akinbami1" id="pone.0099625-Akinbami1" class="link-target"></a>Akinbami L, Moorman J, Bailey C, Zahran H, King M, et al.. (2012) Trends in asthma prevalence, health care use, and mortality in the United States, 2001–2010. Hyattsville, MD: National Center for Health Statistics. <ul class="find-nolinks"></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0099625-Global1" id="pone.0099625-Global1" class="link-target"></a>Global Initiative for Asthma (GINA). (2012) Global Strategy for Asthma Management and Prevention. Available from: <a href="http://www.ginasthma.org/">http://www.ginasthma.org/</a>. <ul class="find-nolinks"></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0099625-Rhen1" id="pone.0099625-Rhen1" class="link-target"></a>Rhen T, Cidlowski JA (2005) Antiinflammatory action of glucocorticoids—new mechanisms for old drugs. N Engl J Med 353: 1711–1723. <ul class="reflinks"><li><a href="#" data-author="Rhen" data-cit="RhenT%2C%20CidlowskiJA%20%282005%29%20Antiinflammatory%20action%20of%20glucocorticoids%E2%80%94new%20mechanisms%20for%20old%20drugs.%20N%20Engl%20J%20Med%20353%3A%201711%E2%80%931723." data-title="Antiinflammatory%20action%20of%20glucocorticoids%E2%80%94new%20mechanisms%20for%20old%20drugs" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Antiinflammatory+action+of+glucocorticoids%E2%80%94new+mechanisms+for+old+drugs+Rhen+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0099625-Fanta1" id="pone.0099625-Fanta1" class="link-target"></a>Fanta CH (2009) Asthma. N Engl J Med 360: 1002–1014. <ul class="reflinks"><li><a href="#" data-author="Fanta" data-cit="FantaCH%20%282009%29%20Asthma.%20N%20Engl%20J%20Med%20360%3A%201002%E2%80%931014." data-title="Asthma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Asthma+Fanta+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0099625-Pratt1" id="pone.0099625-Pratt1" class="link-target"></a>Pratt WB, Morishima Y, Murphy M, Harrell M (2006) Chaperoning of glucocorticoid receptors. Handb Exp Pharmacol: 111–138. <ul class="reflinks"><li><a href="#" data-author="Pratt" data-cit="PrattWB%2C%20MorishimaY%2C%20MurphyM%2C%20HarrellM%20%282006%29%20Chaperoning%20of%20glucocorticoid%20receptors.%20Handb%20Exp%20Pharmacol%3A%20111%E2%80%93138." data-title="Chaperoning%20of%20glucocorticoid%20receptors" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Chaperoning+of+glucocorticoid+receptors+Pratt+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0099625-McMaster1" id="pone.0099625-McMaster1" class="link-target"></a>McMaster A, Ray DW (2007) Modelling the glucocorticoid receptor and producing therapeutic agents with anti-inflammatory effects but reduced side-effects. Exp Physiol 92: 299–309. <ul class="reflinks"><li><a href="#" data-author="McMaster" data-cit="McMasterA%2C%20RayDW%20%282007%29%20Modelling%20the%20glucocorticoid%20receptor%20and%20producing%20therapeutic%20agents%20with%20anti-inflammatory%20effects%20but%20reduced%20side-effects.%20Exp%20Physiol%2092%3A%20299%E2%80%93309." data-title="Modelling%20the%20glucocorticoid%20receptor%20and%20producing%20therapeutic%20agents%20with%20anti-inflammatory%20effects%20but%20reduced%20side-effects" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Modelling+the+glucocorticoid+receptor+and+producing+therapeutic+agents+with+anti-inflammatory+effects+but+reduced+side-effects+McMaster+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0099625-Slats1" id="pone.0099625-Slats1" class="link-target"></a>Slats AM, Sont JK, van Klink RH, Bel EH, Sterk PJ (2006) Improvement in bronchodilation following deep inspiration after a course of high-dose oral prednisone in asthma. Chest 130: 58–65. <ul class="reflinks"><li><a href="#" data-author="Slats" data-cit="SlatsAM%2C%20SontJK%2C%20van%20KlinkRH%2C%20BelEH%2C%20SterkPJ%20%282006%29%20Improvement%20in%20bronchodilation%20following%20deep%20inspiration%20after%20a%20course%20of%20high-dose%20oral%20prednisone%20in%20asthma.%20Chest%20130%3A%2058%E2%80%9365." data-title="Improvement%20in%20bronchodilation%20following%20deep%20inspiration%20after%20a%20course%20of%20high-dose%20oral%20prednisone%20in%20asthma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Improvement+in+bronchodilation+following+deep+inspiration+after+a+course+of+high-dose+oral+prednisone+in+asthma+Slats+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0099625-Trifilieff1" id="pone.0099625-Trifilieff1" class="link-target"></a>Trifilieff A, El-Hashim A, Bertrand C (2000) Time course of inflammatory and remodeling events in a murine model of asthma: effect of steroid treatment. Am J Physiol Lung Cell Mol Physiol 279: L1120–1128. <ul class="reflinks"><li><a href="#" data-author="Trifilieff" data-cit="TrifilieffA%2C%20El-HashimA%2C%20BertrandC%20%282000%29%20Time%20course%20of%20inflammatory%20and%20remodeling%20events%20in%20a%20murine%20model%20of%20asthma%3A%20effect%20of%20steroid%20treatment.%20Am%20J%20Physiol%20Lung%20Cell%20Mol%20Physiol%20279%3A%20L1120%E2%80%931128." data-title="Time%20course%20of%20inflammatory%20and%20remodeling%20events%20in%20a%20murine%20model%20of%20asthma%3A%20effect%20of%20steroid%20treatment" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Time+course+of+inflammatory+and+remodeling+events+in+a+murine+model+of+asthma%3A+effect+of+steroid+treatment+Trifilieff+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0099625-Lakser1" id="pone.0099625-Lakser1" class="link-target"></a>Lakser OJ, Dowell ML, Hoyte FL, Chen B, Lavoie TL, et al. (2008) Steroids augment relengthening of contracted airway smooth muscle: potential additional mechanism of benefit in asthma. Eur Respir J 32: 1224–1230. <ul class="reflinks"><li><a href="#" data-author="Lakser" data-cit="LakserOJ%2C%20DowellML%2C%20HoyteFL%2C%20ChenB%2C%20LavoieTL%2C%20et%20al.%20%282008%29%20Steroids%20augment%20relengthening%20of%20contracted%20airway%20smooth%20muscle%3A%20potential%20additional%20mechanism%20of%20benefit%20in%20asthma.%20Eur%20Respir%20J%2032%3A%201224%E2%80%931230." data-title="Steroids%20augment%20relengthening%20of%20contracted%20airway%20smooth%20muscle%3A%20potential%20additional%20mechanism%20of%20benefit%20in%20asthma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Steroids+augment+relengthening+of+contracted+airway+smooth+muscle%3A+potential+additional+mechanism+of+benefit+in+asthma+Lakser+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0099625-Lamas1" id="pone.0099625-Lamas1" class="link-target"></a>Lamas AM, Leon OG, Schleimer RP (1991) Glucocorticoids inhibit eosinophil responses to granulocyte-macrophage colony-stimulating factor. J Immunol 147: 254–259. <ul class="reflinks"><li><a href="#" data-author="Lamas" data-cit="LamasAM%2C%20LeonOG%2C%20SchleimerRP%20%281991%29%20Glucocorticoids%20inhibit%20eosinophil%20responses%20to%20granulocyte-macrophage%20colony-stimulating%20factor.%20J%20Immunol%20147%3A%20254%E2%80%93259." data-title="Glucocorticoids%20inhibit%20eosinophil%20responses%20to%20granulocyte-macrophage%20colony-stimulating%20factor" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Glucocorticoids+inhibit+eosinophil+responses+to+granulocyte-macrophage+colony-stimulating+factor+Lamas+1991" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0099625-Wallen1" id="pone.0099625-Wallen1" class="link-target"></a>Wallen N, Kita H, Weiler D, Gleich GJ (1991) Glucocorticoids inhibit cytokine-mediated eosinophil survival. J Immunol 147: 3490–3495. <ul class="reflinks"><li><a href="#" data-author="Wallen" data-cit="WallenN%2C%20KitaH%2C%20WeilerD%2C%20GleichGJ%20%281991%29%20Glucocorticoids%20inhibit%20cytokine-mediated%20eosinophil%20survival.%20J%20Immunol%20147%3A%203490%E2%80%933495." data-title="Glucocorticoids%20inhibit%20cytokine-mediated%20eosinophil%20survival" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Glucocorticoids+inhibit+cytokine-mediated+eosinophil+survival+Wallen+1991" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0099625-Reddy1" id="pone.0099625-Reddy1" class="link-target"></a>Reddy TE, Pauli F, Sprouse RO, Neff NF, Newberry KM, et al. (2009) Genomic determination of the glucocorticoid response reveals unexpected mechanisms of gene regulation. Genome Res 19: 2163–2171. <ul class="reflinks"><li><a href="#" data-author="Reddy" data-cit="ReddyTE%2C%20PauliF%2C%20SprouseRO%2C%20NeffNF%2C%20NewberryKM%2C%20et%20al.%20%282009%29%20Genomic%20determination%20of%20the%20glucocorticoid%20response%20reveals%20unexpected%20mechanisms%20of%20gene%20regulation.%20Genome%20Res%2019%3A%202163%E2%80%932171." data-title="Genomic%20determination%20of%20the%20glucocorticoid%20response%20reveals%20unexpected%20mechanisms%20of%20gene%20regulation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genomic+determination+of+the+glucocorticoid+response+reveals+unexpected+mechanisms+of+gene+regulation+Reddy+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0099625-Shore1" id="pone.0099625-Shore1" class="link-target"></a>Shore SA, Moore PE (2003) Regulation of beta-adrenergic responses in airway smooth muscle. Respir Physiol Neurobiol 137: 179–195. <ul class="reflinks"><li><a href="#" data-author="Shore" data-cit="ShoreSA%2C%20MoorePE%20%282003%29%20Regulation%20of%20beta-adrenergic%20responses%20in%20airway%20smooth%20muscle.%20Respir%20Physiol%20Neurobiol%20137%3A%20179%E2%80%93195." data-title="Regulation%20of%20beta-adrenergic%20responses%20in%20airway%20smooth%20muscle" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Regulation+of+beta-adrenergic+responses+in+airway+smooth+muscle+Shore+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0099625-Panettieri1" id="pone.0099625-Panettieri1" class="link-target"></a>Panettieri RA Jr, Kotlikoff MI, Gerthoffer WT, Hershenson MB, Woodruff PG, et al. (2008) Airway smooth muscle in bronchial tone, inflammation, and remodeling: basic knowledge to clinical relevance. Am J Respir Crit Care Med 177: 248–252. <ul class="reflinks"><li><a href="#" data-author="Panettieri" data-cit="PanettieriRAJr%2C%20KotlikoffMI%2C%20GerthofferWT%2C%20HershensonMB%2C%20WoodruffPG%2C%20et%20al.%20%282008%29%20Airway%20smooth%20muscle%20in%20bronchial%20tone%2C%20inflammation%2C%20and%20remodeling%3A%20basic%20knowledge%20to%20clinical%20relevance.%20Am%20J%20Respir%20Crit%20Care%20Med%20177%3A%20248%E2%80%93252." data-title="Airway%20smooth%20muscle%20in%20bronchial%20tone%2C%20inflammation%2C%20and%20remodeling%3A%20basic%20knowledge%20to%20clinical%20relevance" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Airway+smooth+muscle+in+bronchial+tone%2C+inflammation%2C+and+remodeling%3A+basic+knowledge+to+clinical+relevance+Panettieri+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0099625-Nelson1" id="pone.0099625-Nelson1" class="link-target"></a>Nelson HS (1995) Beta-adrenergic bronchodilators. N Engl J Med 333: 499–506. <ul class="reflinks"><li><a href="#" data-author="Nelson" data-cit="NelsonHS%20%281995%29%20Beta-adrenergic%20bronchodilators.%20N%20Engl%20J%20Med%20333%3A%20499%E2%80%93506." data-title="Beta-adrenergic%20bronchodilators" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Beta-adrenergic+bronchodilators+Nelson+1995" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0099625-Hargreave1" id="pone.0099625-Hargreave1" class="link-target"></a>Hargreave FE, Ryan G, Thomson NC, O'Byrne PM, Latimer K, et al. (1981) Bronchial responsiveness to histamine or methacholine in asthma: measurement and clinical significance. J Allergy Clin Immunol 68: 347–355. <ul class="reflinks"><li><a href="#" data-author="Hargreave" data-cit="HargreaveFE%2C%20RyanG%2C%20ThomsonNC%2C%20O%27ByrnePM%2C%20LatimerK%2C%20et%20al.%20%281981%29%20Bronchial%20responsiveness%20to%20histamine%20or%20methacholine%20in%20asthma%3A%20measurement%20and%20clinical%20significance.%20J%20Allergy%20Clin%20Immunol%2068%3A%20347%E2%80%93355." data-title="Bronchial%20responsiveness%20to%20histamine%20or%20methacholine%20in%20asthma%3A%20measurement%20and%20clinical%20significance" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Bronchial+responsiveness+to+histamine+or+methacholine+in+asthma%3A+measurement+and+clinical+significance+Hargreave+1981" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0099625-Masuno1" id="pone.0099625-Masuno1" class="link-target"></a>Masuno K, Haldar SM, Jeyaraj D, Mailloux CM, Huang X, et al. (2011) Expression profiling identifies Klf15 as a glucocorticoid target that regulates airway hyperresponsiveness. Am J Respir Cell Mol Biol 45: 642–649. <ul class="reflinks"><li><a href="#" data-author="Masuno" data-cit="MasunoK%2C%20HaldarSM%2C%20JeyarajD%2C%20MaillouxCM%2C%20HuangX%2C%20et%20al.%20%282011%29%20Expression%20profiling%20identifies%20Klf15%20as%20a%20glucocorticoid%20target%20that%20regulates%20airway%20hyperresponsiveness.%20Am%20J%20Respir%20Cell%20Mol%20Biol%2045%3A%20642%E2%80%93649." data-title="Expression%20profiling%20identifies%20Klf15%20as%20a%20glucocorticoid%20target%20that%20regulates%20airway%20hyperresponsiveness" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Expression+profiling+identifies+Klf15+as+a+glucocorticoid+target+that+regulates+airway+hyperresponsiveness+Masuno+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0099625-Misior1" id="pone.0099625-Misior1" class="link-target"></a>Misior AM, Deshpande DA, Loza MJ, Pascual RM, Hipp JD, et al. (2009) Glucocorticoid- and protein kinase A-dependent transcriptome regulation in airway smooth muscle. Am J Respir Cell Mol Biol 41: 24–39. <ul class="reflinks"><li><a href="#" data-author="Misior" data-cit="MisiorAM%2C%20DeshpandeDA%2C%20LozaMJ%2C%20PascualRM%2C%20HippJD%2C%20et%20al.%20%282009%29%20Glucocorticoid-%20and%20protein%20kinase%20A-dependent%20transcriptome%20regulation%20in%20airway%20smooth%20muscle.%20Am%20J%20Respir%20Cell%20Mol%20Biol%2041%3A%2024%E2%80%9339." data-title="Glucocorticoid-%20and%20protein%20kinase%20A-dependent%20transcriptome%20regulation%20in%20airway%20smooth%20muscle" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Glucocorticoid-+and+protein+kinase+A-dependent+transcriptome+regulation+in+airway+smooth+muscle+Misior+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0099625-Wang1" id="pone.0099625-Wang1" class="link-target"></a>Wang Z, Gerstein M, Snyder M (2009) RNA-Seq: a revolutionary tool for transcriptomics. Nat Rev Genet 10: 57–63. <ul class="reflinks"><li><a href="#" data-author="Wang" data-cit="WangZ%2C%20GersteinM%2C%20SnyderM%20%282009%29%20RNA-Seq%3A%20a%20revolutionary%20tool%20for%20transcriptomics.%20Nat%20Rev%20Genet%2010%3A%2057%E2%80%9363." data-title="RNA-Seq%3A%20a%20revolutionary%20tool%20for%20transcriptomics" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=RNA-Seq%3A+a+revolutionary+tool+for+transcriptomics+Wang+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0099625-Trapnell1" id="pone.0099625-Trapnell1" class="link-target"></a>Trapnell C, Williams BA, Pertea G, Mortazavi A, Kwan G, et al. (2010) Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation. Nat Biotechnol 28: 511–515. <ul class="reflinks"><li><a href="#" data-author="Trapnell" data-cit="TrapnellC%2C%20WilliamsBA%2C%20PerteaG%2C%20MortazaviA%2C%20KwanG%2C%20et%20al.%20%282010%29%20Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation.%20Nat%20Biotechnol%2028%3A%20511%E2%80%93515." data-title="Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Transcript+assembly+and+quantification+by+RNA-Seq+reveals+unannotated+transcripts+and+isoform+switching+during+cell+differentiation+Trapnell+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0099625-Trapnell2" id="pone.0099625-Trapnell2" class="link-target"></a>Trapnell C, Hendrickson DG, Sauvageau M, Goff L, Rinn JL, et al. (2012) Differential analysis of gene regulation at transcript resolution with RNA-seq. Nat Biotechnol 31: 46–53. <ul class="reflinks"><li><a href="#" data-author="Trapnell" data-cit="TrapnellC%2C%20HendricksonDG%2C%20SauvageauM%2C%20GoffL%2C%20RinnJL%2C%20et%20al.%20%282012%29%20Differential%20analysis%20of%20gene%20regulation%20at%20transcript%20resolution%20with%20RNA-seq.%20Nat%20Biotechnol%2031%3A%2046%E2%80%9353." data-title="Differential%20analysis%20of%20gene%20regulation%20at%20transcript%20resolution%20with%20RNA-seq" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Differential+analysis+of+gene+regulation+at+transcript+resolution+with+RNA-seq+Trapnell+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0099625-Marioni1" id="pone.0099625-Marioni1" class="link-target"></a>Marioni JC, Mason CE, Mane SM, Stephens M, Gilad Y (2008) RNA-seq: an assessment of technical reproducibility and comparison with gene expression arrays. Genome Res 18: 1509–1517. <ul class="reflinks"><li><a href="#" data-author="Marioni" data-cit="MarioniJC%2C%20MasonCE%2C%20ManeSM%2C%20StephensM%2C%20GiladY%20%282008%29%20RNA-seq%3A%20an%20assessment%20of%20technical%20reproducibility%20and%20comparison%20with%20gene%20expression%20arrays.%20Genome%20Res%2018%3A%201509%E2%80%931517." data-title="RNA-seq%3A%20an%20assessment%20of%20technical%20reproducibility%20and%20comparison%20with%20gene%20expression%20arrays" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=RNA-seq%3A+an+assessment+of+technical+reproducibility+and+comparison+with+gene+expression+arrays+Marioni+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0099625-Hochberg1" id="pone.0099625-Hochberg1" class="link-target"></a>Hochberg Y, Benjamini Y (1990) More powerful procedures for multiple significance testing. Stat Med 9: 811–818. <ul class="reflinks"><li><a href="#" data-author="Hochberg" data-cit="HochbergY%2C%20BenjaminiY%20%281990%29%20More%20powerful%20procedures%20for%20multiple%20significance%20testing.%20Stat%20Med%209%3A%20811%E2%80%93818." data-title="More%20powerful%20procedures%20for%20multiple%20significance%20testing" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=More+powerful+procedures+for+multiple+significance+testing+Hochberg+1990" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0099625-Shipp1" id="pone.0099625-Shipp1" class="link-target"></a>Shipp LE, Lee JV, Yu CY, Pufall M, Zhang P, et al. (2010) Transcriptional regulation of human dual specificity protein phosphatase 1 (DUSP1) gene by glucocorticoids. PLoS One 5: e13754. <ul class="reflinks"><li><a href="#" data-author="Shipp" data-cit="ShippLE%2C%20LeeJV%2C%20YuCY%2C%20PufallM%2C%20ZhangP%2C%20et%20al.%20%282010%29%20Transcriptional%20regulation%20of%20human%20dual%20specificity%20protein%20phosphatase%201%20%28DUSP1%29%20gene%20by%20glucocorticoids.%20PLoS%20One%205%3A%20e13754." data-title="Transcriptional%20regulation%20of%20human%20dual%20specificity%20protein%20phosphatase%201%20%28DUSP1%29%20gene%20by%20glucocorticoids" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Transcriptional+regulation+of+human+dual+specificity+protein+phosphatase+1+%28DUSP1%29+gene+by+glucocorticoids+Shipp+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0099625-Kelly1" id="pone.0099625-Kelly1" class="link-target"></a>Kelly MM, King EM, Rider CF, Gwozd C, Holden NS, et al. (2012) Corticosteroid-induced gene expression in allergen-challenged asthmatic subjects taking inhaled budesonide. Br J Pharmacol 165: 1737–1747. <ul class="reflinks"><li><a href="#" data-author="Kelly" data-cit="KellyMM%2C%20KingEM%2C%20RiderCF%2C%20GwozdC%2C%20HoldenNS%2C%20et%20al.%20%282012%29%20Corticosteroid-induced%20gene%20expression%20in%20allergen-challenged%20asthmatic%20subjects%20taking%20inhaled%20budesonide.%20Br%20J%20Pharmacol%20165%3A%201737%E2%80%931747." data-title="Corticosteroid-induced%20gene%20expression%20in%20allergen-challenged%20asthmatic%20subjects%20taking%20inhaled%20budesonide" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Corticosteroid-induced+gene+expression+in+allergen-challenged+asthmatic+subjects+taking+inhaled+budesonide+Kelly+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0099625-Burioka1" id="pone.0099625-Burioka1" class="link-target"></a>Burioka N, Fukuoka Y, Takata M, Endo M, Miyata M, et al. (2007) Circadian rhythms in the CNS and peripheral clock disorders: function of clock genes: influence of medication for bronchial asthma on circadian gene. J Pharmacol Sci 103: 144–149. <ul class="reflinks"><li><a href="#" data-author="Burioka" data-cit="BuriokaN%2C%20FukuokaY%2C%20TakataM%2C%20EndoM%2C%20MiyataM%2C%20et%20al.%20%282007%29%20Circadian%20rhythms%20in%20the%20CNS%20and%20peripheral%20clock%20disorders%3A%20function%20of%20clock%20genes%3A%20influence%20of%20medication%20for%20bronchial%20asthma%20on%20circadian%20gene.%20J%20Pharmacol%20Sci%20103%3A%20144%E2%80%93149." data-title="Circadian%20rhythms%20in%20the%20CNS%20and%20peripheral%20clock%20disorders%3A%20function%20of%20clock%20genes%3A%20influence%20of%20medication%20for%20bronchial%20asthma%20on%20circadian%20gene" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Circadian+rhythms+in+the+CNS+and+peripheral+clock+disorders%3A+function+of+clock+genes%3A+influence+of+medication+for+bronchial+asthma+on+circadian+gene+Burioka+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0099625-Eddleston1" id="pone.0099625-Eddleston1" class="link-target"></a>Eddleston J, Herschbach J, Wagelie-Steffen AL, Christiansen SC, Zuraw BL (2007) The anti-inflammatory effect of glucocorticoids is mediated by glucocorticoid-induced leucine zipper in epithelial cells. J Allergy Clin Immunol 119: 115–122. <ul class="reflinks"><li><a href="#" data-author="Eddleston" data-cit="EddlestonJ%2C%20HerschbachJ%2C%20Wagelie-SteffenAL%2C%20ChristiansenSC%2C%20ZurawBL%20%282007%29%20The%20anti-inflammatory%20effect%20of%20glucocorticoids%20is%20mediated%20by%20glucocorticoid-induced%20leucine%20zipper%20in%20epithelial%20cells.%20J%20Allergy%20Clin%20Immunol%20119%3A%20115%E2%80%93122." data-title="The%20anti-inflammatory%20effect%20of%20glucocorticoids%20is%20mediated%20by%20glucocorticoid-induced%20leucine%20zipper%20in%20epithelial%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+anti-inflammatory+effect+of+glucocorticoids+is+mediated+by+glucocorticoid-induced+leucine+zipper+in+epithelial+cells+Eddleston+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0099625-Huangda1" id="pone.0099625-Huangda1" class="link-target"></a>Huang da W, Sherman BT, Lempicki RA (2009) Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources. Nat Protoc 4: 44–57. <ul class="reflinks"><li><a href="#" data-author="Huang%20da" data-cit="Huang%20daW%2C%20ShermanBT%2C%20LempickiRA%20%282009%29%20Systematic%20and%20integrative%20analysis%20of%20large%20gene%20lists%20using%20DAVID%20bioinformatics%20resources.%20Nat%20Protoc%204%3A%2044%E2%80%9357." data-title="Systematic%20and%20integrative%20analysis%20of%20large%20gene%20lists%20using%20DAVID%20bioinformatics%20resources" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Systematic+and+integrative+analysis+of+large+gene+lists+using+DAVID+bioinformatics+resources+Huang+da+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0099625-Kaur1" id="pone.0099625-Kaur1" class="link-target"></a>Kaur M, Chivers JE, Giembycz MA, Newton R (2008) Long-acting beta2-adrenoceptor agonists synergistically enhance glucocorticoid-dependent transcription in human airway epithelial and smooth muscle cells. Molecular pharmacology 73: 203–214. <ul class="reflinks"><li><a href="#" data-author="Kaur" data-cit="KaurM%2C%20ChiversJE%2C%20GiembyczMA%2C%20NewtonR%20%282008%29%20Long-acting%20beta2-adrenoceptor%20agonists%20synergistically%20enhance%20glucocorticoid-dependent%20transcription%20in%20human%20airway%20epithelial%20and%20smooth%20muscle%20cells.%20Molecular%20pharmacology%2073%3A%20203%E2%80%93214." data-title="Long-acting%20beta2-adrenoceptor%20agonists%20synergistically%20enhance%20glucocorticoid-dependent%20transcription%20in%20human%20airway%20epithelial%20and%20smooth%20muscle%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Long-acting+beta2-adrenoceptor+agonists+synergistically+enhance+glucocorticoid-dependent+transcription+in+human+airway+epithelial+and+smooth+muscle+cells+Kaur+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0099625-Himes1" id="pone.0099625-Himes1" class="link-target"></a>Himes BE, Jiang X, Hu R, Wu AC, Lasky-Su JA, et al. (2012) Genome-Wide Association Analysis in Asthma Subjects Identifies SPATS2L as a Novel Bronchodilator Response Gene. PLoS Genet 8: e1002824. <ul class="reflinks"><li><a href="#" data-author="Himes" data-cit="HimesBE%2C%20JiangX%2C%20HuR%2C%20WuAC%2C%20Lasky-SuJA%2C%20et%20al.%20%282012%29%20Genome-Wide%20Association%20Analysis%20in%20Asthma%20Subjects%20Identifies%20SPATS2L%20as%20a%20Novel%20Bronchodilator%20Response%20Gene.%20PLoS%20Genet%208%3A%20e1002824." data-title="Genome-Wide%20Association%20Analysis%20in%20Asthma%20Subjects%20Identifies%20SPATS2L%20as%20a%20Novel%20Bronchodilator%20Response%20Gene" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genome-Wide+Association+Analysis+in+Asthma+Subjects+Identifies+SPATS2L+as+a+Novel+Bronchodilator+Response+Gene+Himes+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0099625-Wang2" id="pone.0099625-Wang2" class="link-target"></a>Wang ZQ, Xing WM, Fan HH, Wang KS, Zhang HK, et al. (2009) The novel lipopolysaccharide-binding protein CRISPLD2 is a critical serum protein to regulate endotoxin function. J Immunol 183: 6646–6656. <ul class="reflinks"><li><a href="#" data-author="Wang" data-cit="WangZQ%2C%20XingWM%2C%20FanHH%2C%20WangKS%2C%20ZhangHK%2C%20et%20al.%20%282009%29%20The%20novel%20lipopolysaccharide-binding%20protein%20CRISPLD2%20is%20a%20critical%20serum%20protein%20to%20regulate%20endotoxin%20function.%20J%20Immunol%20183%3A%206646%E2%80%936656." data-title="The%20novel%20lipopolysaccharide-binding%20protein%20CRISPLD2%20is%20a%20critical%20serum%20protein%20to%20regulate%20endotoxin%20function" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+novel+lipopolysaccharide-binding+protein+CRISPLD2+is+a+critical+serum+protein+to+regulate+endotoxin+function+Wang+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0099625-Rincon1" id="pone.0099625-Rincon1" class="link-target"></a>Rincon M, Irvin CG (2012) Role of IL-6 in asthma and other inflammatory pulmonary diseases. Int J Biol Sci 8: 1281–1290. <ul class="reflinks"><li><a href="#" data-author="Rincon" data-cit="RinconM%2C%20IrvinCG%20%282012%29%20Role%20of%20IL-6%20in%20asthma%20and%20other%20inflammatory%20pulmonary%20diseases.%20Int%20J%20Biol%20Sci%208%3A%201281%E2%80%931290." data-title="Role%20of%20IL-6%20in%20asthma%20and%20other%20inflammatory%20pulmonary%20diseases" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Role+of+IL-6+in+asthma+and+other+inflammatory+pulmonary+diseases+Rincon+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0099625-Harada1" id="pone.0099625-Harada1" class="link-target"></a>Harada A, Sekido N, Akahoshi T, Wada T, Mukaida N, et al. (1994) Essential involvement of interleukin-8 (IL-8) in acute inflammation. J Leukoc Biol 56: 559–564. <ul class="reflinks"><li><a href="#" data-author="Harada" data-cit="HaradaA%2C%20SekidoN%2C%20AkahoshiT%2C%20WadaT%2C%20MukaidaN%2C%20et%20al.%20%281994%29%20Essential%20involvement%20of%20interleukin-8%20%28IL-8%29%20in%20acute%20inflammation.%20J%20Leukoc%20Biol%2056%3A%20559%E2%80%93564." data-title="Essential%20involvement%20of%20interleukin-8%20%28IL-8%29%20in%20acute%20inflammation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Essential+involvement+of+interleukin-8+%28IL-8%29+in+acute+inflammation+Harada+1994" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0099625-Schacke1" id="pone.0099625-Schacke1" class="link-target"></a>Schacke H, Docke WD, Asadullah K (2002) Mechanisms involved in the side effects of glucocorticoids. Pharmacol Ther 96: 23–43. <ul class="reflinks"><li><a href="#" data-author="Schacke" data-cit="SchackeH%2C%20DockeWD%2C%20AsadullahK%20%282002%29%20Mechanisms%20involved%20in%20the%20side%20effects%20of%20glucocorticoids.%20Pharmacol%20Ther%2096%3A%2023%E2%80%9343." data-title="Mechanisms%20involved%20in%20the%20side%20effects%20of%20glucocorticoids" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Mechanisms+involved+in+the+side+effects+of+glucocorticoids+Schacke+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0099625-ThierryMieg1" id="pone.0099625-ThierryMieg1" class="link-target"></a>Thierry-Mieg D, Thierry-Mieg J (2006) AceView: a comprehensive cDNA-supported gene and transcripts annotation. Genome Biol 7  Suppl 1: S12 11–14. <ul class="find-nolinks"></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0099625-Lu1" id="pone.0099625-Lu1" class="link-target"></a>Lu NZ, Cidlowski JA (2006) Glucocorticoid receptor isoforms generate transcription specificity. Trends in cell biology 16: 301–307. <ul class="reflinks"><li><a href="#" data-author="Lu" data-cit="LuNZ%2C%20CidlowskiJA%20%282006%29%20Glucocorticoid%20receptor%20isoforms%20generate%20transcription%20specificity.%20Trends%20in%20cell%20biology%2016%3A%20301%E2%80%93307." data-title="Glucocorticoid%20receptor%20isoforms%20generate%20transcription%20specificity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Glucocorticoid+receptor+isoforms+generate+transcription+specificity+Lu+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0099625-Doni1" id="pone.0099625-Doni1" class="link-target"></a>Doni A, Mantovani G, Porta C, Tuckermann J, Reichardt HM, et al. (2008) Cell-specific regulation of PTX3 by glucocorticoid hormones in hematopoietic and nonhematopoietic cells. The Journal of biological chemistry 283: 29983–29992. <ul class="reflinks"><li><a href="#" data-author="Doni" data-cit="DoniA%2C%20MantovaniG%2C%20PortaC%2C%20TuckermannJ%2C%20ReichardtHM%2C%20et%20al.%20%282008%29%20Cell-specific%20regulation%20of%20PTX3%20by%20glucocorticoid%20hormones%20in%20hematopoietic%20and%20nonhematopoietic%20cells.%20The%20Journal%20of%20biological%20chemistry%20283%3A%2029983%E2%80%9329992." data-title="Cell-specific%20regulation%20of%20PTX3%20by%20glucocorticoid%20hormones%20in%20hematopoietic%20and%20nonhematopoietic%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cell-specific+regulation+of+PTX3+by+glucocorticoid+hormones+in+hematopoietic+and+nonhematopoietic+cells+Doni+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pone.0099625-Moodley1" id="pone.0099625-Moodley1" class="link-target"></a>Moodley T, Wilson SM, Joshi T, Rider CF, Sharma P, et al. (2013) Phosphodiesterase 4 inhibitors augment the ability of formoterol to enhance glucocorticoid-dependent gene transcription in human airway epithelial cells: a novel mechanism for the clinical efficacy of roflumilast in severe chronic obstructive pulmonary disease. Molecular pharmacology 83: 894–906. <ul class="reflinks"><li><a href="#" data-author="Moodley" data-cit="MoodleyT%2C%20WilsonSM%2C%20JoshiT%2C%20RiderCF%2C%20SharmaP%2C%20et%20al.%20%282013%29%20Phosphodiesterase%204%20inhibitors%20augment%20the%20ability%20of%20formoterol%20to%20enhance%20glucocorticoid-dependent%20gene%20transcription%20in%20human%20airway%20epithelial%20cells%3A%20a%20novel%20mechanism%20for%20the%20clinical%20efficacy%20of%20roflumilast%20in%20severe%20chronic%20obstructive%20pulmonary%20disease.%20Molecular%20pharmacology%2083%3A%20894%E2%80%93906." data-title="Phosphodiesterase%204%20inhibitors%20augment%20the%20ability%20of%20formoterol%20to%20enhance%20glucocorticoid-dependent%20gene%20transcription%20in%20human%20airway%20epithelial%20cells%3A%20a%20novel%20mechanism%20for%20the%20clinical%20efficacy%20of%20roflumilast%20in%20severe%20chronic%20obstructive%20pulmonary%20disease" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Phosphodiesterase+4+inhibitors+augment+the+ability+of+formoterol+to+enhance+glucocorticoid-dependent+gene+transcription+in+human+airway+epithelial+cells%3A+a+novel+mechanism+for+the+clinical+efficacy+of+roflumilast+in+severe+chronic+obstructive+pulmonary+disease+Moodley+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pone.0099625-Wang3" id="pone.0099625-Wang3" class="link-target"></a>Wang J, Zhuang J, Iyer S, Lin X, Whitfield TW, et al. (2012) Sequence features and chromatin structure around the genomic regions bound by 119 human transcription factors. Genome research 22: 1798–1812. <ul class="reflinks"><li><a href="#" data-author="Wang" data-cit="WangJ%2C%20ZhuangJ%2C%20IyerS%2C%20LinX%2C%20WhitfieldTW%2C%20et%20al.%20%282012%29%20Sequence%20features%20and%20chromatin%20structure%20around%20the%20genomic%20regions%20bound%20by%20119%20human%20transcription%20factors.%20Genome%20research%2022%3A%201798%E2%80%931812." data-title="Sequence%20features%20and%20chromatin%20structure%20around%20the%20genomic%20regions%20bound%20by%20119%20human%20transcription%20factors" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Sequence+features+and+chromatin+structure+around+the+genomic+regions+bound+by+119+human+transcription+factors+Wang+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pone.0099625-LekstromHimes1" id="pone.0099625-LekstromHimes1" class="link-target"></a>Lekstrom-Himes J, Xanthopoulos KG (1998) Biological role of the CCAAT/enhancer-binding protein family of transcription factors. The Journal of biological chemistry 273: 28545–28548. <ul class="reflinks"><li><a href="#" data-author="Lekstrom-Himes" data-cit="Lekstrom-HimesJ%2C%20XanthopoulosKG%20%281998%29%20Biological%20role%20of%20the%20CCAAT%2Fenhancer-binding%20protein%20family%20of%20transcription%20factors.%20The%20Journal%20of%20biological%20chemistry%20273%3A%2028545%E2%80%9328548." data-title="Biological%20role%20of%20the%20CCAAT%2Fenhancer-binding%20protein%20family%20of%20transcription%20factors" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Biological+role+of+the+CCAAT%2Fenhancer-binding+protein+family+of+transcription+factors+Lekstrom-Himes+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="pone.0099625-Berg1" id="pone.0099625-Berg1" class="link-target"></a>Berg T, Cassel TN, Schwarze PE, Nord M (2002) Glucocorticoids regulate the CCSP and CYP2B1 promoters via C/EBPbeta and delta in lung cells. Biochemical and biophysical research communications 293: 907–912. <ul class="reflinks"><li><a href="#" data-author="Berg" data-cit="BergT%2C%20CasselTN%2C%20SchwarzePE%2C%20NordM%20%282002%29%20Glucocorticoids%20regulate%20the%20CCSP%20and%20CYP2B1%20promoters%20via%20C%2FEBPbeta%20and%20delta%20in%20lung%20cells.%20Biochemical%20and%20biophysical%20research%20communications%20293%3A%20907%E2%80%93912." data-title="Glucocorticoids%20regulate%20the%20CCSP%20and%20CYP2B1%20promoters%20via%20C%2FEBPbeta%20and%20delta%20in%20lung%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Glucocorticoids+regulate+the+CCSP+and+CYP2B1+promoters+via+C%2FEBPbeta+and+delta+in+lung+cells+Berg+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="pone.0099625-Yang1" id="pone.0099625-Yang1" class="link-target"></a>Yang H, Mammen J, Wei W, Menconi M, Evenson A, et al. (2005) Expression and activity of C/EBPbeta and delta are upregulated by dexamethasone in skeletal muscle. Journal of cellular physiology 204: 219–226. <ul class="reflinks"><li><a href="#" data-author="Yang" data-cit="YangH%2C%20MammenJ%2C%20WeiW%2C%20MenconiM%2C%20EvensonA%2C%20et%20al.%20%282005%29%20Expression%20and%20activity%20of%20C%2FEBPbeta%20and%20delta%20are%20upregulated%20by%20dexamethasone%20in%20skeletal%20muscle.%20Journal%20of%20cellular%20physiology%20204%3A%20219%E2%80%93226." data-title="Expression%20and%20activity%20of%20C%2FEBPbeta%20and%20delta%20are%20upregulated%20by%20dexamethasone%20in%20skeletal%20muscle" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Expression+and+activity+of+C%2FEBPbeta+and+delta+are+upregulated+by+dexamethasone+in+skeletal+muscle+Yang+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="pone.0099625-Berg2" id="pone.0099625-Berg2" class="link-target"></a>Berg T, Didon L, Barton J, Andersson O, Nord M (2005) Glucocorticoids increase C/EBPbeta activity in the lung epithelium via phosphorylation. Biochemical and biophysical research communications 334: 638–645. <ul class="reflinks"><li><a href="#" data-author="Berg" data-cit="BergT%2C%20DidonL%2C%20BartonJ%2C%20AnderssonO%2C%20NordM%20%282005%29%20Glucocorticoids%20increase%20C%2FEBPbeta%20activity%20in%20the%20lung%20epithelium%20via%20phosphorylation.%20Biochemical%20and%20biophysical%20research%20communications%20334%3A%20638%E2%80%93645." data-title="Glucocorticoids%20increase%20C%2FEBPbeta%20activity%20in%20the%20lung%20epithelium%20via%20phosphorylation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Glucocorticoids+increase+C%2FEBPbeta+activity+in+the+lung+epithelium+via+phosphorylation+Berg+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref44"><span class="order">44.
            </span><a name="pone.0099625-Akira1" id="pone.0099625-Akira1" class="link-target"></a>Akira S, Isshiki H, Sugita T, Tanabe O, Kinoshita S, et al. (1990) A nuclear factor for IL-6 expression (NF-IL6) is a member of a C/EBP family. The EMBO journal 9: 1897–1906. <ul class="reflinks"><li><a href="#" data-author="Akira" data-cit="AkiraS%2C%20IsshikiH%2C%20SugitaT%2C%20TanabeO%2C%20KinoshitaS%2C%20et%20al.%20%281990%29%20A%20nuclear%20factor%20for%20IL-6%20expression%20%28NF-IL6%29%20is%20a%20member%20of%20a%20C%2FEBP%20family.%20The%20EMBO%20journal%209%3A%201897%E2%80%931906." data-title="A%20nuclear%20factor%20for%20IL-6%20expression%20%28NF-IL6%29%20is%20a%20member%20of%20a%20C%2FEBP%20family" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+nuclear+factor+for+IL-6+expression+%28NF-IL6%29+is+a+member+of+a+C%2FEBP+family+Akira+1990" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span class="order">45.
            </span><a name="pone.0099625-Oyewumi1" id="pone.0099625-Oyewumi1" class="link-target"></a>Oyewumi L, Kaplan F, Gagnon S, Sweezey NB (2003) Antisense oligodeoxynucleotides decrease LGL1 mRNA and protein levels and inhibit branching morphogenesis in fetal rat lung. Am J Respir Cell Mol Biol 28: 232–240. <ul class="reflinks"><li><a href="#" data-author="Oyewumi" data-cit="OyewumiL%2C%20KaplanF%2C%20GagnonS%2C%20SweezeyNB%20%282003%29%20Antisense%20oligodeoxynucleotides%20decrease%20LGL1%20mRNA%20and%20protein%20levels%20and%20inhibit%20branching%20morphogenesis%20in%20fetal%20rat%20lung.%20Am%20J%20Respir%20Cell%20Mol%20Biol%2028%3A%20232%E2%80%93240." data-title="Antisense%20oligodeoxynucleotides%20decrease%20LGL1%20mRNA%20and%20protein%20levels%20and%20inhibit%20branching%20morphogenesis%20in%20fetal%20rat%20lung" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Antisense+oligodeoxynucleotides+decrease+LGL1+mRNA+and+protein+levels+and+inhibit+branching+morphogenesis+in+fetal+rat+lung+Oyewumi+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref46"><span class="order">46.
            </span><a name="pone.0099625-Chiquet1" id="pone.0099625-Chiquet1" class="link-target"></a>Chiquet BT, Lidral AC, Stal S, Mulliken JB, Moreno LM, et al. (2007) CRISPLD2: a novel NSCLP candidate gene. Hum Mol Genet 16: 2241–2248. <ul class="reflinks"><li><a href="#" data-author="Chiquet" data-cit="ChiquetBT%2C%20LidralAC%2C%20StalS%2C%20MullikenJB%2C%20MorenoLM%2C%20et%20al.%20%282007%29%20CRISPLD2%3A%20a%20novel%20NSCLP%20candidate%20gene.%20Hum%20Mol%20Genet%2016%3A%202241%E2%80%932248." data-title="CRISPLD2%3A%20a%20novel%20NSCLP%20candidate%20gene" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=CRISPLD2%3A+a+novel+NSCLP+candidate+gene+Chiquet+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref47"><span class="order">47.
            </span><a name="pone.0099625-Wang4" id="pone.0099625-Wang4" class="link-target"></a>Wang T, Wang ZQ, Wang L, Yan L, Wan J, et al. (2013) CRISPLD2 is expressed at low levels during septic shock and is associated with procalcitonin. PloS one 8: e65743. <ul class="reflinks"><li><a href="#" data-author="Wang" data-cit="WangT%2C%20WangZQ%2C%20WangL%2C%20YanL%2C%20WanJ%2C%20et%20al.%20%282013%29%20CRISPLD2%20is%20expressed%20at%20low%20levels%20during%20septic%20shock%20and%20is%20associated%20with%20procalcitonin.%20PloS%20one%208%3A%20e65743." data-title="CRISPLD2%20is%20expressed%20at%20low%20levels%20during%20septic%20shock%20and%20is%20associated%20with%20procalcitonin" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=CRISPLD2+is+expressed+at+low+levels+during+septic+shock+and+is+associated+with+procalcitonin+Wang+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref48"><span class="order">48.
            </span><a name="pone.0099625-Patel1" id="pone.0099625-Patel1" class="link-target"></a>Patel GP, Balk RA (2012) Systemic steroids in severe sepsis and septic shock. American journal of respiratory and critical care medicine 185: 133–139. <ul class="reflinks"><li><a href="#" data-author="Patel" data-cit="PatelGP%2C%20BalkRA%20%282012%29%20Systemic%20steroids%20in%20severe%20sepsis%20and%20septic%20shock.%20American%20journal%20of%20respiratory%20and%20critical%20care%20medicine%20185%3A%20133%E2%80%93139." data-title="Systemic%20steroids%20in%20severe%20sepsis%20and%20septic%20shock" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Systemic+steroids+in+severe+sepsis+and+septic+shock+Patel+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref49"><span class="order">49.
            </span><a name="pone.0099625-Doreswamy1" id="pone.0099625-Doreswamy1" class="link-target"></a>Doreswamy V, Peden DB (2011) Modulation of asthma by endotoxin. Clin Exp Allergy 41: 9–19. <ul class="reflinks"><li><a href="#" data-author="Doreswamy" data-cit="DoreswamyV%2C%20PedenDB%20%282011%29%20Modulation%20of%20asthma%20by%20endotoxin.%20Clin%20Exp%20Allergy%2041%3A%209%E2%80%9319." data-title="Modulation%20of%20asthma%20by%20endotoxin" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Modulation+of+asthma+by+endotoxin+Doreswamy+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref50"><span class="order">50.
            </span><a name="pone.0099625-Roos1" id="pone.0099625-Roos1" class="link-target"></a>Roos AB, Barton JL, Miller-Larsson A, Dahlberg B, Berg T, et al. (2012) Lung epithelial-C/EBPbeta contributes to LPS-induced inflammation and its suppression by formoterol. Biochemical and biophysical research communications 423: 134–139. <ul class="reflinks"><li><a href="#" data-author="Roos" data-cit="RoosAB%2C%20BartonJL%2C%20Miller-LarssonA%2C%20DahlbergB%2C%20BergT%2C%20et%20al.%20%282012%29%20Lung%20epithelial-C%2FEBPbeta%20contributes%20to%20LPS-induced%20inflammation%20and%20its%20suppression%20by%20formoterol.%20Biochemical%20and%20biophysical%20research%20communications%20423%3A%20134%E2%80%93139." data-title="Lung%20epithelial-C%2FEBPbeta%20contributes%20to%20LPS-induced%20inflammation%20and%20its%20suppression%20by%20formoterol" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Lung+epithelial-C%2FEBPbeta+contributes+to+LPS-induced+inflammation+and+its+suppression+by+formoterol+Roos+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref51"><span class="order">51.
            </span><a name="pone.0099625-Yick1" id="pone.0099625-Yick1" class="link-target"></a>Yick CY, Zwinderman AH, Kunst PW, Grunberg K, Mauad T, et al. (2013) Glucocorticoid-induced Changes in Gene Expression of Airway Smooth Muscle in Patients with Asthma. Am J Respir Crit Care Med 187: 1076–1084. <ul class="reflinks"><li><a href="#" data-author="Yick" data-cit="YickCY%2C%20ZwindermanAH%2C%20KunstPW%2C%20GrunbergK%2C%20MauadT%2C%20et%20al.%20%282013%29%20Glucocorticoid-induced%20Changes%20in%20Gene%20Expression%20of%20Airway%20Smooth%20Muscle%20in%20Patients%20with%20Asthma.%20Am%20J%20Respir%20Crit%20Care%20Med%20187%3A%201076%E2%80%931084." data-title="Glucocorticoid-induced%20Changes%20in%20Gene%20Expression%20of%20Airway%20Smooth%20Muscle%20in%20Patients%20with%20Asthma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Glucocorticoid-induced+Changes+in+Gene+Expression+of+Airway+Smooth+Muscle+in+Patients+with+Asthma+Yick+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref52"><span class="order">52.
            </span><a name="pone.0099625-Panettieri2" id="pone.0099625-Panettieri2" class="link-target"></a>Panettieri RA, Murray RK, DePalo LR, Yadvish PA, Kotlikoff MI (1989) A human airway smooth muscle cell line that retains physiological responsiveness. Am J Physiol 256: C329–335. <ul class="reflinks"><li><a href="#" data-author="Panettieri" data-cit="PanettieriRA%2C%20MurrayRK%2C%20DePaloLR%2C%20YadvishPA%2C%20KotlikoffMI%20%281989%29%20A%20human%20airway%20smooth%20muscle%20cell%20line%20that%20retains%20physiological%20responsiveness.%20Am%20J%20Physiol%20256%3A%20C329%E2%80%93335." data-title="A%20human%20airway%20smooth%20muscle%20cell%20line%20that%20retains%20physiological%20responsiveness" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+human+airway+smooth+muscle+cell+line+that+retains+physiological+responsiveness+Panettieri+1989" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref53"><span class="order">53.
            </span><a name="pone.0099625-Cooper1" id="pone.0099625-Cooper1" class="link-target"></a>Cooper PR, Mesaros AC, Zhang J, Christmas P, Stark CM, et al. (2010) 20-HETE mediates ozone-induced, neutrophil-independent airway hyper-responsiveness in mice. PLoS One 5: e10235. <ul class="reflinks"><li><a href="#" data-author="Cooper" data-cit="CooperPR%2C%20MesarosAC%2C%20ZhangJ%2C%20ChristmasP%2C%20StarkCM%2C%20et%20al.%20%282010%29%2020-HETE%20mediates%20ozone-induced%2C%20neutrophil-independent%20airway%20hyper-responsiveness%20in%20mice.%20PLoS%20One%205%3A%20e10235." data-title="20-HETE%20mediates%20ozone-induced%2C%20neutrophil-independent%20airway%20hyper-responsiveness%20in%20mice" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=20-HETE+mediates+ozone-induced%2C+neutrophil-independent+airway+hyper-responsiveness+in+mice+Cooper+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref54"><span class="order">54.
            </span><a name="pone.0099625-Andrews1" id="pone.0099625-Andrews1" class="link-target"></a>Andrews S FastQC A Quality Control tool for High Throughput Sequence Data. <ul class="find-nolinks"></ul></li><li id="ref55"><span class="order">55.
            </span><a name="pone.0099625-Pearson1" id="pone.0099625-Pearson1" class="link-target"></a>Pearson WR, Wood T, Zhang Z, Miller W (1997) Comparison of DNA sequences with protein sequences. Genomics 46: 24–36. <ul class="reflinks"><li><a href="#" data-author="Pearson" data-cit="PearsonWR%2C%20WoodT%2C%20ZhangZ%2C%20MillerW%20%281997%29%20Comparison%20of%20DNA%20sequences%20with%20protein%20sequences.%20Genomics%2046%3A%2024%E2%80%9336." data-title="Comparison%20of%20DNA%20sequences%20with%20protein%20sequences" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Comparison+of+DNA+sequences+with+protein+sequences+Pearson+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref56"><span class="order">56.
            </span><a name="pone.0099625-Kim1" id="pone.0099625-Kim1" class="link-target"></a>Kim D, Pertea G, Trapnell C, Pimentel H, Kelley R, et al. (2013) TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions. Genome Biol 14: R36. <ul class="reflinks"><li><a href="#" data-author="Kim" data-cit="KimD%2C%20PerteaG%2C%20TrapnellC%2C%20PimentelH%2C%20KelleyR%2C%20et%20al.%20%282013%29%20TopHat2%3A%20accurate%20alignment%20of%20transcriptomes%20in%20the%20presence%20of%20insertions%2C%20deletions%20and%20gene%20fusions.%20Genome%20Biol%2014%3A%20R36." data-title="TopHat2%3A%20accurate%20alignment%20of%20transcriptomes%20in%20the%20presence%20of%20insertions%2C%20deletions%20and%20gene%20fusions" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=TopHat2%3A+accurate+alignment+of+transcriptomes+in+the+presence+of+insertions%2C+deletions+and+gene+fusions+Kim+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref57"><span class="order">57.
            </span><a name="pone.0099625-Barnett1" id="pone.0099625-Barnett1" class="link-target"></a>Barnett DW, Garrison EK, Quinlan AR, Stromberg MP, Marth GT (2011) BamTools: a C++ API and toolkit for analyzing and managing BAM files. Bioinformatics 27: 1691–1692. <ul class="reflinks"><li><a href="#" data-author="Barnett" data-cit="BarnettDW%2C%20GarrisonEK%2C%20QuinlanAR%2C%20StrombergMP%2C%20MarthGT%20%282011%29%20BamTools%3A%20a%20C%2B%2B%20API%20and%20toolkit%20for%20analyzing%20and%20managing%20BAM%20files.%20Bioinformatics%2027%3A%201691%E2%80%931692." data-title="BamTools%3A%20a%20C%2B%2B%20API%20and%20toolkit%20for%20analyzing%20and%20managing%20BAM%20files" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=BamTools%3A+a+C%2B%2B+API+and+toolkit+for+analyzing+and+managing+BAM+files+Barnett+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref58"><span class="order">58.
            </span><a name="pone.0099625-Mortazavi1" id="pone.0099625-Mortazavi1" class="link-target"></a>Mortazavi A, Williams BA, McCue K, Schaeffer L, Wold B (2008) Mapping and quantifying mammalian transcriptomes by RNA-Seq. Nat Methods 5: 621–628. <ul class="reflinks"><li><a href="#" data-author="Mortazavi" data-cit="MortazaviA%2C%20WilliamsBA%2C%20McCueK%2C%20SchaefferL%2C%20WoldB%20%282008%29%20Mapping%20and%20quantifying%20mammalian%20transcriptomes%20by%20RNA-Seq.%20Nat%20Methods%205%3A%20621%E2%80%93628." data-title="Mapping%20and%20quantifying%20mammalian%20transcriptomes%20by%20RNA-Seq" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Mapping+and+quantifying+mammalian+transcriptomes+by+RNA-Seq+Mortazavi+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref59"><span class="order">59.
            </span><a name="pone.0099625-Trapnell3" id="pone.0099625-Trapnell3" class="link-target"></a>Trapnell C, Roberts A, Goff L, Pertea G, Kim D, et al. (2012) Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks. Nat Protoc 7: 562–578. <ul class="reflinks"><li><a href="#" data-author="Trapnell" data-cit="TrapnellC%2C%20RobertsA%2C%20GoffL%2C%20PerteaG%2C%20KimD%2C%20et%20al.%20%282012%29%20Differential%20gene%20and%20transcript%20expression%20analysis%20of%20RNA-seq%20experiments%20with%20TopHat%20and%20Cufflinks.%20Nat%20Protoc%207%3A%20562%E2%80%93578." data-title="Differential%20gene%20and%20transcript%20expression%20analysis%20of%20RNA-seq%20experiments%20with%20TopHat%20and%20Cufflinks" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Differential+gene+and+transcript+expression+analysis+of+RNA-seq+experiments+with+TopHat+and+Cufflinks+Trapnell+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref60"><span class="order">60.
            </span><a name="pone.0099625-Childhood1" id="pone.0099625-Childhood1" class="link-target"></a>Childhood Asthma Management Program Research Group (1999) The Childhood Asthma Management Program (CAMP): design, rationale, and methods. Control Clin Trials 20: 91–120. <ul class="reflinks"><li><a href="#" data-author="" data-cit="Childhood%20Asthma%20Management%20Program%20Research%20Group%20%281999%29%20The%20Childhood%20Asthma%20Management%20Program%20%28CAMP%29%3A%20design%2C%20rationale%2C%20and%20methods.%20Control%20Clin%20Trials%2020%3A%2091%E2%80%93120." data-title="The%20Childhood%20Asthma%20Management%20Program%20%28CAMP%29%3A%20design%2C%20rationale%2C%20and%20methods" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+Childhood+Asthma+Management+Program+%28CAMP%29%3A+design%2C+rationale%2C+and+methods++1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref61"><span class="order">61.
            </span><a name="pone.0099625-Peters1" id="pone.0099625-Peters1" class="link-target"></a>Peters SP, Anthonisen N, Castro M, Holbrook JT, Irvin CG, et al. (2007) Randomized comparison of strategies for reducing treatment in mild persistent asthma. N Engl J Med 356: 2027–2039. <ul class="reflinks"><li><a href="#" data-author="Peters" data-cit="PetersSP%2C%20AnthonisenN%2C%20CastroM%2C%20HolbrookJT%2C%20IrvinCG%2C%20et%20al.%20%282007%29%20Randomized%20comparison%20of%20strategies%20for%20reducing%20treatment%20in%20mild%20persistent%20asthma.%20N%20Engl%20J%20Med%20356%3A%202027%E2%80%932039." data-title="Randomized%20comparison%20of%20strategies%20for%20reducing%20treatment%20in%20mild%20persistent%20asthma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Randomized+comparison+of+strategies+for+reducing+treatment+in+mild+persistent+asthma+Peters+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref62"><span class="order">62.
            </span><a name="pone.0099625-Guilbert1" id="pone.0099625-Guilbert1" class="link-target"></a>Guilbert TW, Morgan WJ, Krawiec M, Lemanske RF Jr, Sorkness C, et al. (2004) The Prevention of Early Asthma in Kids study: design, rationale and methods for the Childhood Asthma Research and Education network. Control Clin Trials 25: 286–310. <ul class="reflinks"><li><a href="#" data-author="Guilbert" data-cit="GuilbertTW%2C%20MorganWJ%2C%20KrawiecM%2C%20LemanskeRFJr%2C%20SorknessC%2C%20et%20al.%20%282004%29%20The%20Prevention%20of%20Early%20Asthma%20in%20Kids%20study%3A%20design%2C%20rationale%20and%20methods%20for%20the%20Childhood%20Asthma%20Research%20and%20Education%20network.%20Control%20Clin%20Trials%2025%3A%20286%E2%80%93310." data-title="The%20Prevention%20of%20Early%20Asthma%20in%20Kids%20study%3A%20design%2C%20rationale%20and%20methods%20for%20the%20Childhood%20Asthma%20Research%20and%20Education%20network" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+Prevention+of+Early+Asthma+in+Kids+study%3A+design%2C+rationale+and+methods+for+the+Childhood+Asthma+Research+and+Education+network+Guilbert+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref63"><span class="order">63.
            </span><a name="pone.0099625-Denlinger1" id="pone.0099625-Denlinger1" class="link-target"></a>Denlinger LC, Sorkness CA, Chinchilli VM, Lemanske RF, Jr. (2007) Guideline-defining asthma clinical trials of the National Heart, Lung, and Blood Institute's Asthma Clinical Research Network and Childhood Asthma Research and Education Network. J Allergy Clin Immunol 119: : 3–11; quiz 12–13. <ul class="find-nolinks"></ul></li><li id="ref64"><span class="order">64.
            </span><a name="pone.0099625-Purcell1" id="pone.0099625-Purcell1" class="link-target"></a>Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, et al. (2007) PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet 81: 559–575. <ul class="reflinks"><li><a href="#" data-author="Purcell" data-cit="PurcellS%2C%20NealeB%2C%20Todd-BrownK%2C%20ThomasL%2C%20FerreiraMA%2C%20et%20al.%20%282007%29%20PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses.%20Am%20J%20Hum%20Genet%2081%3A%20559%E2%80%93575." data-title="PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=PLINK%3A+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses+Purcell+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref65"><span class="order">65.
            </span><a name="pone.0099625-Smyth1" id="pone.0099625-Smyth1" class="link-target"></a>Smyth GK (2005) Limma: linear models for microarray data. In: Gentleman R, Carey V, Dudoit S, Irizarry R, Huber W, editors. Bioinformatics and Computational Biology Solutions using R and Bioconductor. New York: Springer. pp. 397–420. <ul class="find-nolinks"></ul></li><li id="ref66"><span class="order">66.
            </span><a name="pone.0099625-Gautier1" id="pone.0099625-Gautier1" class="link-target"></a>Gautier L, Cope L, Bolstad BM, Irizarry RA (2004) affy—analysis of Affymetrix GeneChip data at the probe level. Bioinformatics 20: 307–315. <ul class="reflinks"><li><a href="#" data-author="Gautier" data-cit="GautierL%2C%20CopeL%2C%20BolstadBM%2C%20IrizarryRA%20%282004%29%20affy%E2%80%94analysis%20of%20Affymetrix%20GeneChip%20data%20at%20the%20probe%20level.%20Bioinformatics%2020%3A%20307%E2%80%93315." data-title="affy%E2%80%94analysis%20of%20Affymetrix%20GeneChip%20data%20at%20the%20probe%20level" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=affy%E2%80%94analysis+of+Affymetrix+GeneChip+data+at+the+probe+level+Gautier+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref67"><span class="order">67.
            </span><a name="pone.0099625-Schneider1" id="pone.0099625-Schneider1" class="link-target"></a>Schneider CA, Rasband WS, Eliceiri KW (2012) NIH Image to ImageJ: 25 years of image analysis. Nature methods 9: 671–675. <ul class="reflinks"><li><a href="#" data-author="Schneider" data-cit="SchneiderCA%2C%20RasbandWS%2C%20EliceiriKW%20%282012%29%20NIH%20Image%20to%20ImageJ%3A%2025%20years%20of%20image%20analysis.%20Nature%20methods%209%3A%20671%E2%80%93675." data-title="NIH%20Image%20to%20ImageJ%3A%2025%20years%20of%20image%20analysis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=NIH+Image+to+ImageJ%3A+25+years+of+image+analysis+Schneider+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref68"><span class="order">68.
            </span><a name="pone.0099625-Pruim1" id="pone.0099625-Pruim1" class="link-target"></a>Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, et al. (2010) LocusZoom: regional visualization of genome-wide association scan results. Bioinformatics 26: 2336–2337. <ul class="reflinks"><li><a href="#" data-author="Pruim" data-cit="PruimRJ%2C%20WelchRP%2C%20SannaS%2C%20TeslovichTM%2C%20ChinesPS%2C%20et%20al.%20%282010%29%20LocusZoom%3A%20regional%20visualization%20of%20genome-wide%20association%20scan%20results.%20Bioinformatics%2026%3A%202336%E2%80%932337." data-title="LocusZoom%3A%20regional%20visualization%20of%20genome-wide%20association%20scan%20results" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=LocusZoom%3A+regional+visualization+of+genome-wide+association+scan+results+Pruim+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>



          <div class="ref-tooltip">
            <div class="ref_tooltip-content">

            </div>
          </div>

        </div>
      </div>
    </div>

  </section>
  <aside class="article-aside">


<!--[if IE 9]>
<style>
.dload-xml {margin-top: 38px}
</style>
<![endif]-->
<div class="dload-menu">
  <div class="dload-pdf">
    <a href="/plosone/article/file?id=10.1371/journal.pone.0099625&type=printable"
       id="downloadPdf" target="_blank">Download PDF</a>
  </div>
  <div data-js-tooltip-hover="trigger" class="dload-hover">&nbsp;
    <ul class="dload-xml" data-js-tooltip-hover="target">
      <li><a href="/plosone/article/citation?id=10.1371/journal.pone.0099625"
             id="downloadCitation">Citation</a></li>
      <li><a href="/plosone/article/file?id=10.1371/journal.pone.0099625&type=manuscript"
             id="downloadXml">XML</a>
      </li>
    </ul>

  </div>
</div>

<div class="aside-container">

<div class="print-article" id="printArticle" data-js-tooltip-hover="trigger">

  Print
  <ul class="print-options" data-js-tooltip-hover="target">
    <li>
        <a href="#" onclick="window.print(); return false;" class="preventDefault" id="printBrowser" title="Print
        Article">Print article</a>
    </li>




<li>
<a title="Odyssey Press" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=7&amp;doi=10.1371%2Fjournal.pone.0099625&amp;volume=&amp;issue=&amp;title=RNA-Seq%20Transcriptome%20Profiling%20Identifies%20CRISPLD2%20as%20a%20Glucocorticoid%20Responsive%20Gene%20that%20Modulates%20Cytokine%20Function%20in%20Airway%20Smooth%20Muscle%20Cells&amp;author_name=Blanca%20E.%20Himes%2C%20Xiaofeng%20Jiang%2C%20Peter%20Wagner%2C%20Ruoxi%20Hu%2C%20Qiyu%20Wang%2C%20Barbara%20Klanderman%2C%20Reid%20M.%20Whitaker%2C%20Qingling%20Duan%2C%20Jessica%20Lasky-Su%2C%20Christina%20Nikolos%2C%20William%20Jester%2C%20Martin%20Johnson%2C%20Reynold%20A.%20Panettieri%20Jr%2C%20Kelan%20G.%20Tantisira%2C%20Scott%20T.%20Weiss%2C%20Quan%20Lu&amp;start_page=1&amp;end_page=13">EzReprint  </a>
</li>

  </ul>
</div>
<div class="share-article" id="shareArticle" data-js-tooltip-hover="trigger">
  Share
  <ul data-js-tooltip-hover="target" class="share-options" id="share-options">

<li><a href="https://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0099625" id="shareReddit" target="_blank" title="Submit to Reddit"><img src="/plosone/resource/img/icon.reddit.16.png" width="16" height="16" alt="Reddit">Reddit</a></li>

<li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0099625" id="shareGoogle" target="_blank" title="Share on Google+"><img src="/plosone/resource/img/icon.gplus.16.png" width="16" height="16" alt="Google+">Google+</a></li>

<li><a href="https://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0099625&t=RNA-Seq Transcriptome Profiling Identifies CRISPLD2 as a Glucocorticoid Responsive Gene that Modulates Cytokine Function in Airway Smooth Muscle Cells" id="shareFacebook" target="_blank" title="Share on Facebook"><img src="/plosone/resource/img/icon.fb.16.png" width="16" height="16" alt="Facebook">Facebook</a></li>

<li><a href="https://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0099625&title=RNA-Seq Transcriptome Profiling Identifies CRISPLD2 as a Glucocorticoid Responsive Gene that Modulates Cytokine Function in Airway Smooth Muscle Cells&summary=Checkout this article I found at PLOS"  id="shareLinkedIn" target="_blank" title="Add to LinkedIn"><img src="/plosone/resource/img/icon.linkedin.16.png" width="16" height="16" alt="LinkedIn">LinkedIn</a></li>

<li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0099625&title=RNA-Seq Transcriptome Profiling Identifies CRISPLD2 as a Glucocorticoid Responsive Gene that Modulates Cytokine Function in Airway Smooth Muscle Cells"  id="shareCiteULike" target="_blank" title="Add to CiteULike"><img src="/plosone/resource/img/icon.cul.16.png" width="16" height="16" alt="CiteULike">CiteULike</a></li>

<li><a href="https://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0099625"  id="shareMendeley" target="_blank" title="Add to Mendeley"><img src="/plosone/resource/img/icon.mendeley.16.png" width="16" height="16" alt="Mendeley">Mendeley</a></li>

<li><a href="https://www.pubchase.com/library?add_aid=10.1371/journal.pone.0099625&source=plos"  id="sharePubChase" target="_blank" title="Add to PubChase"><img src="/plosone/resource/img/icon.pc.16.png" width="16" height="16" alt="PubChase">PubChase</a></li>

  <li><a href="https://twitter.com/intent/tweet?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0099625&text=%23PLOSONE%3A%20RNA-Seq Transcriptome Profiling Identifies CRISPLD2 as a Glucocorticoid Responsive Gene that Modulates Cytokine Function in Airway Smooth Muscle Cells" target="_blank" title="share on Twitter" id="twitter-share-link"><img src="/plosone/resource/img/icon.twtr.16.png" width="16" height="16" alt="Twitter">Twitter</a></li>

<li><a href="/plosone/article/email?id=10.1371/journal.pone.0099625" id="shareEmail" title="Email this article"><img src="/plosone/resource/img/icon.email.16.png" width="16" height="16" alt="Email">Email</a></li>

  </ul>
</div></div>


 <!-- Crossmark 2.0 widget -->
<script src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"></script>
<a data-target="crossmark"><img width="150" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_BW_horizontal.svg"></a>
<!-- End Crossmark 2.0 widget -->







<div class="skyscraper-container">
  <div class="title">Advertisement</div>
<!-- DoubleClick Ad Zone -->
  <div class='advertisement' id='div-gpt-ad-1458247671871-1' style='width:160px; height:600px;'>
    <script type='text/javascript'>
      googletag.cmd.push(function() { googletag.display('div-gpt-ad-1458247671871-1'); });
    </script>
  </div>
</div>




<div class="subject-areas-container">
  <h3>Subject Areas <div id="subjInfo">?</div>
    <div id="subjInfoText">
      <p>For more information about PLOS Subject Areas, click
        <a href="https://github.com/PLOS/plos-thesaurus/blob/develop/README.md" target="_blank" title="Link opens in new window">here</a>.</p>
      <span class="inline-intro">We want your feedback.</span> Do these Subject Areas make sense for this article? Click the target next to the incorrect Subject Area and let us know. Thanks for your help!


    </div>
  </h3>
  <ul id="subjectList">
      <li>
              <a class="taxo-term" title="Search for articles about Gene expression"
                 href="/plosone/search?filterSubjects=Gene+expression&filterJournals=PLoSONE&q=">Gene expression</a>
        <span class="taxo-flag">&nbsp;</span>
        <div class="taxo-tooltip" data-categoryname="Gene expression"><p class="taxo-explain">Is the Subject Area <strong>"Gene expression"</strong> applicable to this article?
          <button id="noFlag" data-action="remove">Yes</button>
          <button id="flagIt" value="flagno" data-action="add">No</button></p>
          <p class="taxo-confirm">Thanks for your feedback.</p>
        </div>
      </li>
      <li>
              <a class="taxo-term" title="Search for articles about Asthma"
                 href="/plosone/search?filterSubjects=Asthma&filterJournals=PLoSONE&q=">Asthma</a>
        <span class="taxo-flag">&nbsp;</span>
        <div class="taxo-tooltip" data-categoryname="Asthma"><p class="taxo-explain">Is the Subject Area <strong>"Asthma"</strong> applicable to this article?
          <button id="noFlag" data-action="remove">Yes</button>
          <button id="flagIt" value="flagno" data-action="add">No</button></p>
          <p class="taxo-confirm">Thanks for your feedback.</p>
        </div>
      </li>
      <li>
              <a class="taxo-term" title="Search for articles about Small interfering RNAs"
                 href="/plosone/search?filterSubjects=Small+interfering+RNAs&filterJournals=PLoSONE&q=">Small interfering RNAs</a>
        <span class="taxo-flag">&nbsp;</span>
        <div class="taxo-tooltip" data-categoryname="Small interfering RNAs"><p class="taxo-explain">Is the Subject Area <strong>"Small interfering RNAs"</strong> applicable to this article?
          <button id="noFlag" data-action="remove">Yes</button>
          <button id="flagIt" value="flagno" data-action="add">No</button></p>
          <p class="taxo-confirm">Thanks for your feedback.</p>
        </div>
      </li>
      <li>
              <a class="taxo-term" title="Search for articles about RNA sequencing"
                 href="/plosone/search?filterSubjects=RNA+sequencing&filterJournals=PLoSONE&q=">RNA sequencing</a>
        <span class="taxo-flag">&nbsp;</span>
        <div class="taxo-tooltip" data-categoryname="RNA sequencing"><p class="taxo-explain">Is the Subject Area <strong>"RNA sequencing"</strong> applicable to this article?
          <button id="noFlag" data-action="remove">Yes</button>
          <button id="flagIt" value="flagno" data-action="add">No</button></p>
          <p class="taxo-confirm">Thanks for your feedback.</p>
        </div>
      </li>
      <li>
              <a class="taxo-term" title="Search for articles about Bronchodilators"
                 href="/plosone/search?filterSubjects=Bronchodilators&filterJournals=PLoSONE&q=">Bronchodilators</a>
        <span class="taxo-flag">&nbsp;</span>
        <div class="taxo-tooltip" data-categoryname="Bronchodilators"><p class="taxo-explain">Is the Subject Area <strong>"Bronchodilators"</strong> applicable to this article?
          <button id="noFlag" data-action="remove">Yes</button>
          <button id="flagIt" value="flagno" data-action="add">No</button></p>
          <p class="taxo-confirm">Thanks for your feedback.</p>
        </div>
      </li>
      <li>
              <a class="taxo-term" title="Search for articles about Cytokines"
                 href="/plosone/search?filterSubjects=Cytokines&filterJournals=PLoSONE&q=">Cytokines</a>
        <span class="taxo-flag">&nbsp;</span>
        <div class="taxo-tooltip" data-categoryname="Cytokines"><p class="taxo-explain">Is the Subject Area <strong>"Cytokines"</strong> applicable to this article?
          <button id="noFlag" data-action="remove">Yes</button>
          <button id="flagIt" value="flagno" data-action="add">No</button></p>
          <p class="taxo-confirm">Thanks for your feedback.</p>
        </div>
      </li>
      <li>
              <a class="taxo-term" title="Search for articles about Inflammation"
                 href="/plosone/search?filterSubjects=Inflammation&filterJournals=PLoSONE&q=">Inflammation</a>
        <span class="taxo-flag">&nbsp;</span>
        <div class="taxo-tooltip" data-categoryname="Inflammation"><p class="taxo-explain">Is the Subject Area <strong>"Inflammation"</strong> applicable to this article?
          <button id="noFlag" data-action="remove">Yes</button>
          <button id="flagIt" value="flagno" data-action="add">No</button></p>
          <p class="taxo-confirm">Thanks for your feedback.</p>
        </div>
      </li>
      <li>
              <a class="taxo-term" title="Search for articles about Transcriptome analysis"
                 href="/plosone/search?filterSubjects=Transcriptome+analysis&filterJournals=PLoSONE&q=">Transcriptome analysis</a>
        <span class="taxo-flag">&nbsp;</span>
        <div class="taxo-tooltip" data-categoryname="Transcriptome analysis"><p class="taxo-explain">Is the Subject Area <strong>"Transcriptome analysis"</strong> applicable to this article?
          <button id="noFlag" data-action="remove">Yes</button>
          <button id="flagIt" value="flagno" data-action="add">No</button></p>
          <p class="taxo-confirm">Thanks for your feedback.</p>
        </div>
      </li>
  </ul>
</div>
<div id="subjectErrors"></div>


<div class="twitter-container">
  <h3>Archived Tweets</h3>
  <ul id="tweetList">

  </ul>
  <div class="load-more">Load more <span></span></div>
  <div class="view-all"><a href="https://alm.plos.org/works/doi.org/10.1371/journal.pone.0099625?source_id=twitter">View all tweets</a>
  </div>
</div>

<script type="text/template" id="twitterModuleItemTemplate">
  <% _.each(items, function(item) { %>
    <li>
      <div class="tweet-info">
        <a href="https://twitter.com/<%= item.user %>">
          <span class="imgholder">
            <img class="imgLoad" src="<%= item.user_profile_image %>">
          </span>
          <div class="tweetDate"><%= item.created_at %></div>
          <div class="tweetUser">
            <strong><%= item.user_name %></strong>
            <span>@<%= item.user %></span>
          </div>
        </a>
      </div>
      <div class="tweetText">
        <%= item.text %>
      </div>
      <div id="tweetActions">
        <a class="tweet-reply" href="https://twitter.com/intent/tweet?in_reply_to<%= item.id %>&amp;text=@<%= item.user %>">
          <div>&nbsp;</div> Reply
        </a>
        <a class="tweet-retweet" href="https://twitter.com/intent/retweet?tweet_id=<%= item.id %>">
          <div>&nbsp;</div> Retweet
        </a>
        <a class="tweet-favorite" href="https://twitter.com/intent/favorite?tweet_id=<%= item.id %>">
          <div>&nbsp;</div> Favorite
        </a>
      </div>
    </li>
  <% }); %>
</script>


  </aside>
</div>




</main>

<footer id="pageftr">
  <div class="row">
  

     <div class="block x-small">   
      

<ul class="nav nav-secondary">
    <li class="ftr-header"><a href="https://www.plos.org/publications/journals/">Publications</a></li>
    <li><a href="/plosbiology/" id="ftr-bio">PLOS Biology</a></li>
    <li><a href="/plosmedicine/" id="ftr-med">PLOS Medicine</a></li>
    <li><a href="/ploscompbiol/" id="ftr-compbio">PLOS Computational Biology</a></li>
    <li><a href="/plosgenetics/" id="ftr-gen">PLOS Genetics</a></li>
    <li><a href="/plospathogens/" id="ftr-path">PLOS Pathogens</a></li>
    <li><a href="/plosone/" id="ftr-one">PLOS ONE</a></li>
    <li><a href="/plosntds/" id="ftr-ntds">PLOS Neglected Tropical Diseases</a></li>
  </ul>
  </div>
   
    <div class="block xx-small">	


<ul class="nav nav-tertiary">
  <li>
    <a href="https://www.plos.org" id="ftr-home">Home</a>
  </li>
  <li>
    <a href="https://blogs.plos.org" id="ftr-blog">Blogs</a>
  </li>
  <li>
    <a href="https://collections.plos.org" id="ftr-collections">Collections</a>
  </li>
  <li>
    <a href="/plosone/feedback" id="ftr-feedback">Give feedback</a>
  </li>
  <li>
    <a href="/plosone/lockss-manifest" id="ftr-lockss">LOCKSS</a>
  </li>
</ul>
    </div>
    <div class="block xx-small">

<ul class="nav nav-primary">
  <li><a href="https://www.plos.org/privacy-policy" id="ftr-privacy">Privacy Policy</a></li>
  <li><a href="https://www.plos.org/terms-of-use" id="ftr-terms">Terms of Use</a></li>
  <li><a href="https://www.plos.org/advertise/" id="ftr-advertise">Advertise</a></li>
  <li><a href="https://www.plos.org/media-inquiries" id="ftr-media">Media Inquiries</a></li>
  <li><a href="https://www.plos.org/contact" id="ftr-contact">Contact</a></li>
</ul>
     </div>
     <div class="block x-small">


<p class="footer-non-profit-statement">PLOS is a nonprofit 501(c)(3) corporation, #C2354500, based in San Francisco, California, US</p>	 		 <img src="/plosone/resource/img/logo-plos-footer.png" alt="PLOS" class="logo-footer"/>
    </div>
	 


&nbsp;
<!--
  Webapp build:  3.7.8 at 20190503150719 by teamcity, commit: 
  Service build: 2.5.2 at 20190506160336 by teamcity, commit: 
  Enabled dev features: []
  -->
     
  </div>
 


</footer>




<script type="text/javascript">
  var ArticleData = {
    doi: '10.1371/journal.pone.0099625',
    title: '<article-title xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">RNA-Seq Transcriptome Profiling Identifies <italic>CRISPLD2<\/italic> as a Glucocorticoid Responsive Gene that Modulates Cytokine Function in Airway Smooth Muscle Cells<\/article-title>',
    date: 'Jun 13, 2014'
  };
</script>

<script type="text/javascript">
  var ALM_CONFIG = ALM_CONFIG || {};
  ALM_CONFIG.hostname = "https://alm.plos.org";
  ALM_CONFIG.apiKey = "3pezRBRXdyzYW6ztfwft";
  ALM_CONFIG.host = "https://alm.plos.org/api/v5/articles";
</script>
<script type="text/javascript">
  var ALM_CONFIG = ALM_CONFIG || {};
  ALM_CONFIG.hostname = "https://alm.plos.org";
  ALM_CONFIG.apiKey = "3pezRBRXdyzYW6ztfwft";
  ALM_CONFIG.host = "https://alm.plos.org/api/v5/articles";
</script>






<script type="text/javascript" async src="https://platform.twitter.com/widgets.js"></script>





<!-- This file should be loaded before the renderJs, to avoid conflicts with the FigShare, that implements the MathJax also. -->

<!--  mathjax configuration options  -->
<!-- more can be found at http://docs.mathjax.org/en/latest/ -->
<script type="text/x-mathjax-config">
MathJax.Hub.Config({
  "HTML-CSS": {
    scale: 100,
    availableFonts: ["STIX","TeX"],
    preferredFont: "STIX",
    webFont: "STIX-Web",
    linebreaks: { automatic: false }
  },
  jax: ["input/MathML", "output/HTML-CSS"]
});
</script>

<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=MML_HTMLorMML"></script>

<script src="/plosone/resource/compiled/asset_4GTWTU3NT6K5SKRGQAMOYA77BVM7OPDS.js"></script>
<div class="reveal-modal-bg"></div>
</body>
</html>
