---
title: "Lecture 14 - Genome Informatics Pt2"
author: "ACGeffre"
date: "May 17th 2019"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

Aside: We are going to install bioConductor and DESeq packages to R; please see the RMD file for this document to see the specific code (it is different from the typical install, as BioConductor is very large. 
```{r, include=FALSE}
#Bioconductor packages are installed differently than “regular” R packages from CRAN. 
# To install the core Bioconductor packages, copy and paste these lines of code into your R console one at a time.

#    install.packages("BiocManager")
#    BiocManager::install()

# For this class, you'll also need DESeq2:

#    BiocManager::install("DESeq2")

# We installed nothing from source files and didn't update any packages during installation
library("BiocManager")
library("DESeq2")
```

# Working in BioConductor
For working with DESeq2, we will have unique data structure:
Please see: https://bioboot.github.io/bggn213_S19/class-material/lab-14-bggn213.html

1) our data, which should be comprised of reads (Y), along treatments (X), and 2) a dataframe that expands the metadata encoded by treatment designation in the previous data frame's X value (e.g. treatment, gender, etc.).

## Glucorticosteroid Expression Set
Let's import our data for today:
```{r}
counts <- read.csv("airway_scaledcounts.csv", stringsAsFactors = FALSE)
metadata <-  read.csv("airway_metadata.csv", stringsAsFactors = FALSE)
```
Please see the following background provided by BJGrant about these data:
"The data for this hands-on session comes from a published RNA-seq experiment where airway smooth muscle cells were treated with dexamethasone, a synthetic glucocorticoid steroid with anti-inflammatory effects (Himes et al. 2014).

Glucocorticoids are used, for example, by people with asthma to reduce inflammation of the airways. The anti-inflammatory effects on airway smooth muscle (ASM) cells has been known for some time but the underlying molecular mechanisms are unclear.

Himes et al. used RNA-seq to profile gene expression changes in four different ASM cell lines treated with dexamethasone glucocorticoid. They found a number of differentially expressed genes comparing dexamethasone-treated to control cells, but focus much of the discussion on a gene called CRISPLD2. This gene encodes a secreted protein known to be involved in lung development, and SNPs in this gene in previous GWAS studies are associated with inhaled corticosteroid resistance and bronchodilator response in asthma patients. They confirmed the upregulated CRISPLD2 mRNA expression with qPCR and increased protein expression using Western blotting.

In the experiment, four primary human ASM cell lines were treated with 1 micromolar dexamethasone for 18 hours. For each of the four cell lines, we have a treated and an untreated sample. They did their analysis using Tophat and Cufflinks similar to our last day's hands-on session. For a more detailed description of their analysis see the PubMed entry 24926665 and for raw data see the GEO entry GSE52778.

In this session we will read and explore the gene expression data from this experiment using base R functions and then perform a detailed analysis with the DESeq2 package from Bioconductor."


### Question 1
How many genes are in this dataset?
```{r}
print(paste("There are", nrow(counts), "genes in this dataset.")) # the gene names are each rows in this dataframe
```

We can check if the cols of 'counts' is the same as the id column in "metadata":
```{r}
colnames(counts[,-1]) # or colnames(counts)[-1]
metadata$id

# Do they match?
colnames(counts[,-1]) == metadata$id
# If true, then yes they match

# Compare everything at once
all(colnames(counts[,-1]) == metadata$id)
# If there are any falses, the value will return "FALSE"; if everything matches, it returns "TRUE"
```


### Question 2
How many 'control' cell lines do we have?
```{r}
View(metadata)
print(paste("There are", count(metadata$dex == "control"), "control cell lines in this experiment"))
```


## Toy Differential gene expression methods
From Dr. Grant: "Let's perform some exploratory differential gene expression analysis. *Note: this analysis is for demonstration only. NEVER do differential expression analysis this way!*"

Let's make a control treatment average set...
```{r}
control <- metadata[metadata[,"dex"]=="control",]
control.mean <- rowSums( counts[ ,control$id] )/count(metadata$dex == "control")
names(control.mean) <- counts$ensgene
```

...and a treated treatment average set.
```{r}
treated <- metadata[metadata[,"dex"]=="treated",]
treated.mean <- rowSums( counts[ ,treated$id] )/count(metadata$dex == "treated")
names(treated.mean) <- counts$ensgene
```

And finally, we will make a concise dataframe containing the meat from both files:
```{r}
meancounts <- data.frame(control.mean, treated.mean)
head(meancounts)
# For each gene, prints the mean expression (counts) for each treatement.
```

Lastly, we plot them:
```{r}
plot(meancounts[,1],meancounts[,2], xlab="Control", ylab="Treated") #plots normally
plot(meancounts[,1],meancounts[,2], xlab="Control", ylab="Treated", log="xy") # Plots expression on logarithimic scale
```

### Calculate the fold change between treatments (log2 fold change)
Let's create a variable housing the fold change in expression, compared to the control, that happens during treatment. Bioinformaticists often use log(base2), or log2 fold change.

```{r}
meancounts$log2fc <- log2(meancounts[,"treated.mean"]/meancounts[,"control.mean"])
head(meancounts)
```
 
Nite that this generates "NaN" and "-Inf", both of which are nonsense values that we can't interpret.  How to deal with these?

First, we can check which raw mean values are 0:
```{r}
zero.vals <- which(meancounts[,1:2]==0, arr.ind=TRUE) 
# which() creates a vector of all the variables in meancounts[,1:2] that are 0. 
# arr.ind=TRUE means arrange the vector above in a matrix including the row and column ID for each value=0

to.rm <- unique(zero.vals[,1]) 
# pulls all zero values from the matrix we just made where there is a zero in only ONE of the columns (e.g. Unique)
mycounts <- meancounts[-to.rm,]
# Pops out all the unique 0 rows (e.g. genes with no expression)
head(mycounts)
# Now there are no more "NaNs" and such.
```

For thsi type of gene expression comparison, note that log2 values >2 are expressed MORE, and <-2 are expressed LESS.  Let's prepare a data set for these:
```{r}
up.ind <- mycounts$log2fc > 2
down.ind <- mycounts$log2fc < (-2)
```

We can use the count() fucntion to determine how many genes are up-regulated and down-regulated.
```{r}
print(paste("There are", count(up.ind == TRUE), "upregulated genes and", count(down.ind == TRUE), "downregulated genes in our data set"))
```

Or we can use the sum() function to sum up the Boolean values (True = 1, False = 0).
```{r}
print(paste("There are", sum(up.ind == TRUE), "upregulated genes and", sum(down.ind == TRUE), "downregulated genes in our data set"))
```

## Renaming Genes with Proper Annotation
We have found out genes that are up/down regulated, so now we should try to figure out what those genes are called.  We have Ensemble gene IDs (ENSXXXXX... and such), but those are not well-known names.  Let's import names from our annotation data. We have this information from Ensemble. 

### The Annotation File of Annotation-ness
We have an annotation file for all the human genes and their annotation information (downloadable from Ensemble)!
```{r}
anno <- read.csv("annotables_grch38.csv")
# Check the deets, yo.
head(anno)
```

### Merge them shits together
We will use the **merge()** function to squish the two data sets together

```{r}
merge(x=mycounts, y=anno, by.x="row.names", by.y="ensgene")
# Merges mycounts with anno, pastes rows together according to matching ens IDs in the "ensgene" column
```

## Do it all, but with BioConductor
We can also use BioConductor to do all of this (perhaps more easily too)
```{r, include=FALSE}
library("AnnotationDbi")
# BiocManager::install("org.Hs.eg.db")
library("org.Hs.eg.db")
```

Let's look at the types of keys available to search our genes against.
```{r}
columns(org.Hs.eg.db)
```
We should be able to print corresponding labels for any of these types of IDs, given our intial ens IDs.

Let's take our original data from mycounts, and annotate it thusly:
```{r}
mycounts$symbol <- mapIds(org.Hs.eg.db,
                     keys=row.names(mycounts), # Our genenames
                     keytype="ENSEMBL",        # The format of our genenames
                     column="SYMBOL",          # The new format we want to add
                     multiVals="first")
# Makes a new column in mycounts ("symbol") with the legible gene ID by processing the samples using an "ENSEMBL" key type.
head(mycounts)
```

Now let's add the Entrez ID and UniProt accession numbers:
```{r}
mycounts$entrez <- mapIds(org.Hs.eg.db,
                     keys=row.names(mycounts), # Our genenames
                     keytype="ENSEMBL",        # The format of our genenames
                     column="ENTREZID",          # The new format we want to add
                     multiVals="first")

mycounts$uniprot <- mapIds(org.Hs.eg.db,
                     keys=row.names(mycounts), # Our genenames
                     keytype="ENSEMBL",        # The format of our genenames
                     column="UNIPROT",          # The new format we want to add
                     multiVals="first")
head(mycounts)
```

## Assemble the annotation with comparative counts with DESeq2
```{r, include=FALSE}
library(DESeq2)
citation("DESeq2")
```
To use DESeq, we need to compile a specific file to work with; please see BJGrant about this:
"Bioconductor software packages often define and use custom class objects for storing data. This helps to ensure that all the needed data for analysis (and the results) are available. DESeq works on a particular type of object called a DESeqDataSet. The DESeqDataSet is a single object that contains input values, intermediate calculations like how things are normalized, and all results of a differential expression analysis.

You can construct a DESeqDataSet from (1) a count matrix, (2) a metadata file, and (3) a formula indicating the design of the experiment.

We have talked about (1) and (2) previously. The third needed item that has to be specified at the beginning of the analysis is a design formula. This tells DESeq2 which columns in the sample information table (colData) specify the experimental design (i.e. which groups the samples belong to) and how these factors should be used in the analysis. Essentially, this formula expresses how the counts for each gene depend on the variables in colData.

Take a look at metadata again. The thing we're interested in is the dex column, which tells us which samples are treated with dexamethasone versus which samples are untreated controls. We'll specify the design with a tilde, like this: design=~dex. (The tilde is the shifted key to the left of the number 1 key on my keyboard. It looks like a little squiggly line).

We will use the DESeqDataSetFromMatrix() function to build the required DESeqDataSet object and call it dds, short for our DESeqDataSet. If you get a warning about "some variables in design formula are characters, converting to factors" don't worry about it. Take a look at the dds object once you create it."

### Import raw data files
We will give DESeq the raw counts file, along with the metadata associated with those columns. The function **DESeqDataSetFromMatrix()** creates the specisl DESeqDataSet file we can use to do expression analysis.
```{r}
dds <- DESeqDataSetFromMatrix(countData=counts, 
                              colData=metadata, 
                              design=~dex, 
                              tidy=TRUE)
dds
```

### Run the seq comparisons
```{r}
dds <- DESeq(dds)

# Now we have a bunch of information 
sizeFactors(dds)
dispersions(dds)
res <- results(dds)
```

Let's survey our results from DESeq2.
```{r}
as.data.frame(res)
```

And summarizd them:
```{r}
summary(res)
```

Let's manipulate the alpha value to be 0.05 instead of 0.1
```{r}
res05 <- results(dds, alpha = 0.05)
summary(res05)
```

### Volcano Plot these results
The volcano plot usually plots the fold change as X, and p-value as Y. 

About the log(pvalue): "Typically these plots shows the log fold change on the X-axis, and the −log10 of the p-value on the Y-axis (the more significant the p-value, the larger the −log10 of that value will be). A very dull (i.e. non colored and labeled) version can be created with a quick call to plot() like so:"
```{r}
plot( res$log2FoldChange,  -log(res$padj), 
      xlab="Log2(FoldChange)",
      ylab="-Log(P-value)")
abline(v=c(-2,2), col = c("blue", "blue"), lty = 2)
abline( h = (-log(0.05)), col="purple", lty = 2)
```

```{r}
# Setup our custom point color vector 
mycols <- rep("gray", nrow(res)) 
# create a color vector that assigns everything to gray
mycols[ abs(res$log2FoldChange) > 2 ]  <- "purple" 
# however if there are abs(values) > 2, overwrite them with "purple"; 
# this means all values beyond the threshold will appear purple, now.  

inds <- (res$padj < 0.01) & (abs(res$log2FoldChange) > 2 )
# find all abs(value) > 2 with pvalue < 0.01 andf put them in a data set.
mycols[ inds ] <- "blue"
# THis means for all values in inds that overlap with mycolsm replace their value with "blue"

# Volcano plot with custom colors 
plot( res$log2FoldChange,  -log(res$padj), 
 col=mycols, ylab="-Log(P-value)", xlab="Log2(FoldChange)" )

# Cut-off lines
abline(v=c(-2,2), col="blue",lwd = 3,  lty=2)
abline(h=-log(0.1), col="purple", lwd = 3, lty=2)
```

### Enhanced volcano!
First, let's make a data set to use with enhanced volcano.
```{r}
x <- as.data.frame(res)
x$symbol <- mapIds(org.Hs.eg.db, 
                   keys=row.names(x),
                   keytype="ENSEMBL",
                   column="SYMBOL",
                   multiVals="first")
```

```{r}
# BiocInstaller::install("EnhancedVolcano")
# install.packages("EnhancedVolcano")
# library(EnhancedVolcano)
```

Ooops! Enhanced volcano isn't available for my version for R (3.6.1). 
