

    <!DOCTYPE html>
<html class="">

        
<head>
    <title>Differential analyses for RNA-seq: transcript-level... | F1000Research</title>
    <meta charset="utf-8">
    <!--<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>-->
    <!--<meta lang="$locale">-->
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="google-site-verification" content="9-QVycOO2_ob3Z9QzRmXv2CF08A9oyYXqWyTiVdKPlU" />
    <!-- This is commented out to fix display problems on mobile devices.
    We may use it again once we implement a responsive design that supports native device resolutions.
    <meta name="viewport" content="width=device-width"> -->

    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

                
    
    <link rel="alternate" title="Recent articles published in F1000Research" href="/rss" type="application/rss+xml">
            <link rel="alternate" type="application/pdf" title="PDF" href="https://f1000research.com/articles/4-1521/pdf"/>
        <link rel="canonical" href="https://f1000research.com/articles/4-1521" />
    
            <meta name="description" content="Read the full article on F1000Research: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences">
    
            <meta name="og:title" content="F1000Research Article: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences.">
            <meta name="og:description" content="Read the latest article version by Charlotte Soneson, Michael I. Love, Mark D. Robinson, at F1000Research.">
            <meta name="version-id" content="8770">
            <meta name="article-id" content="7563">
            <meta name="DC.identifier" content="doi:10.12688/f1000research.7563.2">
            <meta name="citation_title" content="Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences">
            <meta name="citation_abstract" content="High-throughput sequencing of cDNA (RNA-seq) is used extensively to characterize the transcriptome of cells. Many transcriptomic studies aim at comparing either abundance levels or the transcriptome composition between given conditions, and as a first step, the sequencing reads must be used as the basis for abundance quantification of transcriptomic features of interest, such as genes or transcripts. Various quantification approaches have been proposed, ranging from simple counting of reads that overlap given genomic regions to more complex estimation of underlying transcript abundances. In this paper, we show that gene-level abundance estimates and statistical inference offer advantages over transcript-level analyses, in terms of performance and interpretability. We also illustrate that the presence of differential isoform usage can lead to inflated false discovery rates in differential gene expression analyses on simple count matrices but that&amp;nbsp;this can be addressed by incorporating offsets derived from transcript-level abundance estimates. We also show that the problem is relatively minor in several real data sets. Finally, we provide an R package ( tximport ) to help users integrate transcript-level abundance estimates from common quantification pipelines into count-based statistical inference engines.">
            <meta name="citation_description" content="High-throughput sequencing of cDNA (RNA-seq) is used extensively to characterize the transcriptome of cells. Many transcriptomic studies aim at comparing either abundance levels or the transcriptome composition between given conditions, and as a first step, the sequencing reads must be used as the basis for abundance quantification of transcriptomic features of interest, such as genes or transcripts. Various quantification approaches have been proposed, ranging from simple counting of reads that overlap given genomic regions to more complex estimation of underlying transcript abundances. In this paper, we show that gene-level abundance estimates and statistical inference offer advantages over transcript-level analyses, in terms of performance and interpretability. We also illustrate that the presence of differential isoform usage can lead to inflated false discovery rates in differential gene expression analyses on simple count matrices but that&amp;nbsp;this can be addressed by incorporating offsets derived from transcript-level abundance estimates. We also show that the problem is relatively minor in several real data sets. Finally, we provide an R package ( tximport ) to help users integrate transcript-level abundance estimates from common quantification pipelines into count-based statistical inference engines.">
            <meta name="citation_keywords" content="RNA-seq, quantification, gene expression, transcriptomics">
            <meta name="citation_journal_title" content="F1000Research">
            <meta name="citation_author" content="Charlotte Soneson">
            <meta name="citation_author_institution" content="SIB Swiss Institute of Bioinformatics, University of Zurich, Zurich, 8057, Switzerland">
            <meta name="citation_author_institution" content="Institute for Molecular Life Sciences, University of Zurich, Zurich, 8057, Switzerland">
            <meta name="citation_author" content="Michael I. Love">
            <meta name="citation_author_institution" content="Department of Biostatistics, Harvard TH Chan School of Public Health, Boston, MA, 02115, USA">
            <meta name="citation_author_institution" content="Department of Biostatistics and Computational Biology, Dana-Farber Cancer Institute, Boston, MA, 02210, USA">
            <meta name="citation_author" content="Mark D. Robinson">
            <meta name="citation_author_institution" content="SIB Swiss Institute of Bioinformatics, University of Zurich, Zurich, 8057, Switzerland">
            <meta name="citation_author_institution" content="Institute for Molecular Life Sciences, University of Zurich, Zurich, 8057, Switzerland">
            <meta name="citation_date" content="2016/02/29">
            <meta name="citation_volume" content="4">
            <meta name="citation_publication_number" content="1521">
            <meta name="citation_version_number" content="2">
            <meta name="citation_doi" content="10.12688/f1000research.7563.2">
            <meta name="citation_pdf_url" content="https://f1000research.com/articles/4-1521/v2/pdf">
            <meta name="keywords" content="RNA-seq, quantification, gene expression, transcriptomics, F1000, F1000Research, Faculty of 1000, Open Access, publishing platform, post-publication peer review, rapid publication, fast publication, science publishing, Science, Medicine, Biology, life sciences, peer review, publishing, research articles, negative results, null results, open data, open science, open peer review, replication studies, sound science, cell biology, ecology, neuroscience, physiology, genetics, immunology, health, oncology, anesthesiology, emergency medicine">
    

    
    <link href="/img/favicon-research.ico" rel="shortcut icon" type="image/ico">
    <link href="/img/favicon-research.ico" rel="icon" type="image/ico">

        <link rel="stylesheet" href="/1557818118695/css/mdl/material-design-lite.min.css" type="text/css" media="all" />
    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i" rel="stylesheet">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/MaterialDesign-Webfont/2.2.43/css/materialdesignicons.min.css" />

        
            
    <link rel="stylesheet" href="/1557818118695/css/F1000Research.css" type="text/css" media="all" />
    <link rel="stylesheet" href="/css/F1000ResearchFontIcons/F1000ResearchFontIcons.css" type="text/css" media="all" />
    <link rel="stylesheet" href="/css/F1000ResearchFontIcons/animation.css" type="text/css" media="all" />

        <!--[if IE 7]><link rel="stylesheet" href="/css/F1000ResearchFontIcons/F1000ResearchFontIcons-ie7.css" media="all" /><![endif]-->

    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.8.1/jquery.min.js"></script>
    <script>window.jQuery || document.write('<script src="/js/vendor/jquery-1.8.1.min.js"><\/script>')</script>
    <script type="text/javascript" src="/1557818118695/js/vendor/modernizr-2.6.1-respond-1.1.0.min.js"></script>
    <script type="text/javascript" src="/1557818118695/js/shared_scripts/sticky.js"></script>
    <script type="text/javascript" src="/1557818118695/js/shared_scripts/helpers.js"></script>
    <script type="text/javascript" src="/1557818118695/js/shared_scripts/menu.js"></script>
    <script type="text/javascript" src="/1557818118695/js/shared_scripts/navbar.js"></script>
    <script type="text/javascript" src="/1557818118695/js/shared_scripts/platforms.js"></script>
    <script type="text/javascript" src="/1557818118695/js/shared_scripts/object-polyfills.js"></script>
            <script type="text/javascript" src="/1557818118695/js/vendor/lodash.min.js"></script>
        <script>CKEDITOR_BASEPATH='https://f1000research.com/js/ckeditor/'</script>
    <script src="https://d1hiluowqo0t4b.cloudfront.net/js/F1000R/plugins.js"></script>
    <script type="text/javascript" src="/1557818118695/js/shared_scripts/helpers.js"></script>
    <script type="text/javascript" src="/1557818118695/js/app/research.js"></script>
    <script type="text/javascript" src="/1557818118695/js/app/research.ui.js"></script>
    <script type="text/javascript" src="/1557818118695/js/app/login.js"></script>
    <script type="text/javascript" src="/1557818118695/js/app/main.js"></script>
    <script type="text/javascript" src="/1557818118695/js/app/js-date-format.min.js"></script>
    <script type="text/javascript" src="/1557818118695/js/app/search.js"></script>
    <script type="text/javascript" src="/1557818118695/js/app/cookies_warning.js"></script>
    <script type="text/javascript" src="/1557818118695/js/mdl/mdl.min.js"></script>

    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
                <script type="text/javascript" src="https://d1hiluowqo0t4b.cloudfront.net/js/F1000R/ckeditor.js"></script>
            <script type="text/javascript" src="/js/ckeditor/adapters/jquery.js"></script>
                <script type="text/javascript" src="/js/article/article_scrolling_module.js"></script>
            <script type="text/javascript" src="/js/article/article_stats.js"></script>
            <script type="text/javascript" src="/js/article/article.js"></script>
            <script type="text/javascript" src="/js/shared_scripts/referee_timeline_pagination.js"></script>
            <script type="text/javascript" src="/js/app/text_editor_controller.js"></script>
            <script type="text/javascript" src="//s7.addthis.com/js/250/addthis_widget.js#pubid=ra-503e5e99593dc42c"></script>
            <script type="text/javascript" src="/js/article/article_metrics.js"></script>
                <script type="text/javascript">
        if (window.location.hash == '#_=_'){
            window.location = window.location.href.split('#')[0]
        }
    </script>

            
        
    <!-- pixelId: 1641728616063202 :: assetPixelId: 6034867600215 :: funderPixelId:  -->

            <!-- Facebook pixel code (merged with EP GTM code) -->
        <script>
            !function(f,b,e,v,n,t,s){if(f.fbq)return;n=f.fbq=function()

            {n.callMethod? n.callMethod.apply(n,arguments):n.queue.push(arguments)}
            ;if(!f._fbq)f._fbq=n;
            n.push=n;n.loaded=!0;n.version='2.0';n.queue=[];t=b.createElement(e);t.async=!0;
            t.src=v;s=b.getElementsByTagName(e)[0];s.parentNode.insertBefore(t,s)}(window,
            document,'script','https://connect.facebook.net/en_US/fbevents.js');

            fbq('init', '1641728616063202');

            
            fbq('track', "PixelInitialized", {});
        </script>

        <noscript><img height="1" width="1" style="display:none"
            src="https://www.facebook.com/tr?id=1641728616063202&noscript=1&amp;ev=PixelInitialized"
        /></noscript>
        <!-- End Facebook Pixel Code -->
    
        <script>
        (function(h,o,t,j,a,r){
            h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
            h._hjSettings={hjid:917825,hjsv:6};
            a=o.getElementsByTagName('head')[0];
            r=o.createElement('script');r.async=1;
            r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
            a.appendChild(r);
        })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
    </script>

</head>
<body  class="o-page-container no-js p-article o-layout-reset   ">

    
        
        
    <div class="o-page">

        <div id="notify-container"></div>
        <div id="pageWarning"></div>
        <div id="pageMessage"></div>
        <div id="pageFooterMessage"></div>
                <div id="f1r-ga-data" data-name="f1r-ga-data" class="f1r-ga-data"
            data-user-registered="false"
            data-user-module=""
            data-current-path=""
            data-location=""
            data-website="F1000Research"
            data-websiteDisplayName="F1000Research">
        </div>

                
        
        <div class="header-wrapper   js-navbar-space ">
                            


    











        
                                
    
            



<nav class="c-navbar js-navbar js-mini-nav js-sticky c-navbar--js-sticky c-navbar--userSite c-navbar__platform-bgcolor  c-navbar--bg-f1000research ">

    <div class="c-navbar__content">

                                <div class="c-navbar__extras">
            <div class="o-wrapper">
                <div class="o-actions o-actions--middle c-navbar__extras-row">
                    <div class="o-actions__primary">
                        
                                            </div>
                    <div class="o-actions__secondary">
                                                    <div class="c-navbar__extras-menu">
                                                                                                            <a href="http://f1000.com/prime"  target="_blank"  class="c-navbar__extras-item  ">F1000Prime</a>
                                                                                                                                                <a href="https://f1000workspace.com"  target="_blank"  class="c-navbar__extras-item  ">F1000Workspace</a>
                                                                                                    <p>item : </p>
                            </div>
                                            </div>
                </div>
            </div>
        </div>

        <div class="o-wrapper t-inverted js-sticky-start">

            <div class="c-navbar__branding-row">
                <div class="c-navbar__row">


                                        
                    <div class="c-navbar__primary u-mr--2">

                                                                                                                                                                                                    <a href="/" class="c-navbar__branding u-ib u-middle"   data-test-id="nav_branding"  >
                                <img class="u-ib u-middle" src="/img/F1000R_logo_header.png" alt="F1000Research">
                            </a>
                                            </div>

                    <div class="c-navbar__secondary c-navbar__row">


                                                
                                                    <form action="/search" class="-navbar__secondary u-mr--2 c-search-form js-search-form u-hide u-show@navbar">
                                <label for="searchInput" class="c-search-form__label _mdl-layout">
                                    <input name="q" type="search" class="c-search-form__input" id="searchInput" placeholder="Search">
                                    <button type="submit" class="c-search-form__submit mdl-button mdl-js-button mdl-button--icon"><i class="material-icons">search</i></button>
                                </label>
                            </form>
                        
                                                
                                                    <div class="c-navbar__primary u-hide u-show@navbar">
                                <div class="_mdl-layout c-navbar__cta">
                                    <a class="mdl-button mdl-js-button mdl-button--raised mdl-button--no-shadow mdl-button--multi-line mdl-js-ripple-effect mdl-button--inverted c-navbar__submit" href="/for-authors/publish-your-research"   data-test-id="nav_submit_research"  ><i class="material-icons">file_upload</i>Submit your research</a>
                                </div>
                            </div>
                        

                                                
                        <span class="u-hide@navbar _mdl-layout u-nowrap">

                                                            <button type="button" class="mdl-button mdl-js-button mdl-js-ripple-effect mdl-button--icon c-navbar__toggle c-navbar__toggle--menu js-navbar-toggle" data-focus="#navbar_mob_search_input" data-toggle="navbarMenu" data-target="navbarMenu" aria-controls="navbarMenu" aria-expanded="false" aria-label="Toggle navigation"   data-test-id="nav_menu_search_mob"  ><i class="material-icons">search</i></button>
                            
                                                            <button class="mdl-button mdl-js-button mdl-js-ripple-effect mdl-button--icon mdl-button--multi-line c-navbar__toggle c-navbar__toggle--menu js-navbar-toggle" type="button" data-toggle="navbarMenu" data-target="navbarMenu" aria-controls="navbarMenu" aria-expanded="false" aria-label="Toggle navigation"   data-test-id="nav_menu_toggle_mob"  >
                                    <i class="material-icons c-navbar__toggle-open">menu</i>
                                    <i class="material-icons c-navbar__toggle-close">close</i>
                                </button>
                                                    </span>
                    </div>

                </div>
            </div>

                        
                            <div class="c-navbar__menu-row js-navbar-block is-collapsed" id="navbarMenu">

                                                                                                                                                
                                                                    
                                                                    
                                                                                                                                                                                                                                                                                                                                                                                                                                                
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
                                            

                                                                
                                                                                                                                                                                                                                
                                            
                    <div class="c-navbar__menu-row-content">

                                                                            <div class="u-hide@navbar c-navbar__menu-bar-spacing">
                                <form action="/search" class="c-search-form js-search-form">
                                    <label for="navbar_mob_search_input" class="c-search-form__label _mdl-layout">
                                        <input id="navbar_mob_search_input" name="q" type="search" class="c-search-form__input" placeholder="Search">
                                        <button type="submit" class="c-search-form__submit mdl-button mdl-js-button mdl-button--icon"><i class="material-icons">search</i></button>
                                    </label>
                                </form>
                            </div>
                        
                        <div class="o-actions o-actions--middle">

                            <div class="o-actions__primary">

                                                                
    <ul class="c-menubar c-navbar__menu-bar js-main-menu"   id="main-menu"   role="menubar" aria-label="Main Navigation"  data-menu-group="navbar" >

        
            
                                                                
                                
                                
                                    <li role="none"
                        data-index="0"
                        class="c-menubar__item js-menu-item  c-navbar__menu-bar-item
                                                              ">

                        <a href="/browse/articles" class="c-navbar__menu-bar-link js-menu-link " role="menuitem"
                                                                                    tabindex="0"
                              data-test-id="nav_browse"                              >Browse</a>

                                            </li>
                
            
        
            
                                                                
                
                                
                                    <li role="none"
                        data-index="1"
                        class="c-menubar__item js-menu-item  c-navbar__menu-bar-item
                                                              ">

                        <a href="/gateways" class="c-navbar__menu-bar-link js-menu-link " role="menuitem"
                                                                                    tabindex="-1"
                              data-test-id="nav_gatewaysViewAndBrowse"                              >Gateways & Collections</a>

                                            </li>
                
            
        
            
                                                                
                
                                
                                    <li role="none"
                        data-index="2"
                        class="c-menubar__item js-menu-item  c-navbar__menu-bar-item
                                                               c-menubar__item--selected c-navbar__menu-bar-item--parent ">

                        <a href="#" class="c-navbar__menu-bar-link js-menu-link " role="menuitem"
                                                         aria-haspopup="true" aria-expanded="false"                             tabindex="-1"
                              data-test-id="nav_for-authors"                              >How to Publish</a>

                                                    
    <ul class="c-menu js-menu
                is-collapsed                c-menubar__menu c-navbar__menu"
                  role="menu"
        aria-label="How to Publish">

        
            
                                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_submit-manuscript"                      href="/for-authors/publish-your-research"
                    role="menuitem"
                    tabindex="0">Submit your Research</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_my-submissions"                      href="/for-authors/my-submissions"
                    role="menuitem"
                    tabindex="-1">Submissions</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_article-guidelines"                      href="/for-authors/article-guidelines"
                    role="menuitem"
                    tabindex="-1">Article Guidelines</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_data-guidelines"                      href="/for-authors/data-guidelines"
                    role="menuitem"
                    tabindex="-1">Data Guidelines</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_asset-guidelines"                      href="/for-authors/posters-and-slides-guidelines"
                    role="menuitem"
                    tabindex="-1">Posters and Slides Guidelines</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_article-processing-charges"                      href="/for-authors/article-processing-charges"
                    role="menuitem"
                    tabindex="-1">Article Processing Charges</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_finding-referees"                      href="/for-authors/tips-for-finding-referees"
                    role="menuitem"
                    tabindex="-1">Finding Article Reviewers</a>

                </li>

                        </ul>
                                            </li>
                
            
        
            
                                                                
                
                                
                                    <li role="none"
                        data-index="3"
                        class="c-menubar__item js-menu-item  c-navbar__menu-bar-item
                                                               c-menubar__item--selected c-navbar__menu-bar-item--parent ">

                        <a href="#" class="c-navbar__menu-bar-link js-menu-link " role="menuitem"
                                                         aria-haspopup="true" aria-expanded="false"                             tabindex="-1"
                              data-test-id="nav_about-contact"                              >About</a>

                                                    
    <ul class="c-menu js-menu
                is-collapsed                c-menubar__menu c-navbar__menu"
                  role="menu"
        aria-label="About">

        
            
                                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_about-page"                      href="/about"
                    role="menuitem"
                    tabindex="0">How it Works</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_referee-guidelines"                      href="/for-referees/guidelines"
                    role="menuitem"
                    tabindex="-1">For Reviewers</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_advisoryPanel"                      href="/advisors"
                    role="menuitem"
                    tabindex="-1">Our Advisors</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_policy-page"                      href="/about/policies"
                    role="menuitem"
                    tabindex="-1">Policies</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_glossary-page"                      href="/glossary"
                    role="menuitem"
                    tabindex="-1">Glossary</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_faqs-page"                      href="/faqs"
                    role="menuitem"
                    tabindex="-1">FAQs</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              u-hide u-show@navbar"
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      u-hide u-show@navbar"
                                          data-test-id="nav_for-developers"                      href="/developers"
                    role="menuitem"
                    tabindex="-1">For Developers</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_contact-page"                      href="/contact"
                    role="menuitem"
                    tabindex="-1">Contact</a>

                </li>

                        </ul>
                                            </li>
                
            
        
            
                                                                
                
                                
                                    <li role="none"
                        data-index="4"
                        class="c-menubar__item js-menu-item  c-navbar__menu-bar-item
                                                              ">

                        <a href="https://blog.f1000.com/blogs/f1000research/" class="c-navbar__menu-bar-link js-menu-link " role="menuitem"
                                                                                    tabindex="-1"
                              data-test-id="nav_blog"                              >Blog</a>

                                            </li>
                
            
        
    </ul>



                            </div>

                            <div class="o-actions__secondary">

                                                                
    <ul class="c-menubar c-navbar__menu-bar js-main-menu"   id="secondary-items"   role="menubar" aria-label="My Account"  data-menu-group="navbar" >

        
            
                                                                
                                
                                
                                    <li role="none"
                        data-index="0"
                        class="c-menubar__item js-menu-item  c-navbar__menu-bar-item
                                                               c-menubar__item--selected c-navbar__menu-bar-item--parent ">

                        <a href="#" class="c-navbar__menu-bar-link js-menu-link " role="menuitem"
                                                         aria-haspopup="true" aria-expanded="false"                             tabindex="0"
                              data-test-id="nav_my-research"                              >My Research</a>

                                                    
    <ul class="c-menu js-menu
                is-collapsed                c-menubar__menu c-navbar__menu"
                  role="menu"
        aria-label="My Research">

        
            
                                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_my-submissions"                      href="/login?originalPath=/my/submissions"
                    role="menuitem"
                    tabindex="0">Submissions</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_my-email-alerts"                      href="/login?originalPath=/my/email-alerts"
                    role="menuitem"
                    tabindex="-1">Tracking and Alerts</a>

                </li>

                    
            
                
                                
                <li class="c-menu__item js-menu-item
                                              "
                        role="none">

                <a  class="c-menu__link js-menu-link
                                                      "
                                          data-test-id="nav_my-user-details"                      href="/login?originalPath=/my/user-details"
                    role="menuitem"
                    tabindex="-1">My Details</a>

                </li>

                        </ul>
                                            </li>
                
            
        
            
                                                                
                
                                
                                    <li role="none"
                        data-index="1"
                        class="c-menubar__item js-menu-item  c-navbar__menu-bar-item
                                                              ">

                        <a href="/login?originalPath=/articles/4-1521/v2" class="c-navbar__menu-bar-link js-menu-link " role="menuitem"
                                                                                    tabindex="-1"
                              data-test-id="nav_sign-in"                              >Sign In</a>

                                            </li>
                
            
        
    </ul>



                            </div>

                                                                                        <div class="_mdl-layout c-navbar__cta u-hide@navbar c-navbar__menu-bar-spacing">
                                    <a class="mdl-button mdl-js-button mdl-button--raised mdl-button--multi-line mdl-button--no-shadow mdl-js-ripple-effect mdl-button--inverted c-navbar__submit" href="/for-authors/publish-your-research"   data-test-id="nav_submit_research_mob"  ><i class="material-icons">file_upload</i>Submit your research</a>
                                </div>
                                                    </div>

                    </div>

                </div>
            
        </div>

    </div>

</nav>
                    </div>

        <div class="content-wrapper o-page__main row ">
            <div id="highlight-area" class="content ">
                





<div id=article-metadata class=hidden><input type=hidden name=versionId value=8770 /><input type=hidden id=articleId name=articleId value=7563 /><input type=hidden id=xmlUrl value="/articles/4-1521/v2/xml"/><input type=hidden id=article_uuid value=b8f453a6-0659-4c35-ae8c-7ba13e292a10 /><input type=hidden id=referer value=""/><input type=hidden id=meta-article-title value="Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences"/><input type=hidden id=workspace-export-url value="https://f1000.com/work/api/import/external?doi=10.12688/f1000research.7563.2"/><input type=hidden id=versionDoi value="10.12688/f1000research.7563.2"/><input type=hidden id=usePmcStats value=true /></div><main class="o-wrapper p-article__wrapper js-wrapper"><div class="o-layout o-layout--right-gutter"><div id=article_secondary-column class="p-article__main o-layout__item u-font-size--legal u-2/3@article not-expanded "><div class="float-left f1r-article-desk"><div class=breadcrumbs>&nbsp;</div></div><div class="article-badges-container no-breadcrumb"><div class=crossmark-new><script src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"></script><a data-target=crossmark><img height=30 width=150 src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg"/></a></div><div id=crossmark-dialog style="display: none;" title=""><iframe id=crossmark-dialog-frame frameborder=0></iframe></div></div><div class=article-interaction-container><div id=main-article-count-box class=article-count-box><div class="article-metrics-wrapper metrics-icon-wrapper" data-version-id=8770 data-id=7563 data-downloads="" data-views="" data-scholar="10.12688/f1000research.7563.2" data-recommended=726181976 data-doi="10.12688/f1000research.7563.2" data-f1r-ga-helper="Article Page Metrics (Desktop)"><span class="metrics-on-browse article-metrics-icon f1r-icon icon-89_metrics"></span><div class="count-title article-metrics-text">Metrics</div><div class=js-article-metrics-container></div></div><div true title="Total views and downloads from F1000Research and PubMed Central"><div class=count-delimiter></div><div class="count-container view-count js-views-count">-</div><div class=count-title><span class="count-title-icon count-title-views-icon"></span>Views</div><div class="count-delimiter download-counts" hidden></div><div class="count-container download-counts js-downloads-count" hidden></div><div class="count-title download-counts" hidden><span class="count-title-icon f1r-icon icon-76_download_file"></span>Downloads</div></div></div><div id=main-article-interaction-box class="article-interaction-box has-control-tab"><div class="article-interaction-info article-page"><div class=article-interaction-button><span class="f1r-icon icon-102_download_pdf"></span><a href="https://f1000research.com/articles/4-1521/v2/pdf?article_uuid=b8f453a6-0659-4c35-ae8c-7ba13e292a10" title="Download PDF" class="button-link download pdf-download-helper" target=_blank>Get PDF</a></div></div><div class="article-interaction-info article-page"><div class=article-interaction-button><span class="f1r-icon icon-103_download_xml"></span><a id=download-xml href="#" class="button-link download" title="Download XML">Get XML</a></div></div><div class="article-interaction-info article-page"><div class="cite-article-popup-wrapper article-page-interaction-box"><div class="article-interaction-button cite-article-button" title="Cite this article" data-windowref=cite-article-popup-7563-2><span class="f1r-icon icon-82_quote"></span><a href="#" class="button-link cite-article-popup-link" title="Cite Article">Cite</a></div><div id=cite-article-popup-7563-2 class="popup-window-wrapper is-hidden"><div class=cite-popup-background></div><div class="popup-window top-popup cite-this-article-box research-layout"><div class="popup-window-title small cite-title">How to cite this article</div><span id=cite-article-text-7563-2><span class="article-title-and-info in-popup">Soneson C, Love MI and Robinson MD. Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences [version 2; peer review: 2 approved]</span>. <i>F1000Research</i> 2016, <b>4</b>:1521<br/>(<a class=new-orange href="https://doi.org/10.12688/f1000research.7563.2" target=_blank>https://doi.org/10.12688/f1000research.7563.2</a>) </span><div class="popup-window-title small margin-top-20 margin-bottom-20 note"><strong>NOTE:</strong> it is important to ensure the information in square brackets after the title is included in all citations of this article. </div><div class=float-right><button class="secondary no-fill orange-text-and-border margin-right-20 close-cite-popup uppercase">Close</button><button id=copy-citation-details class="secondary orange copy-cite-article-version uppercase js-clipboard" title="Copy the current citation details to the clipboard." data-clipboard-target="#cite-article-text-7563-2">Copy Citation Details</button></div></div></div></div></div><div class="article-interaction-info article-page"><div class=article-interaction-button><span class="f1r-icon icon-76_download_file"></span><a id=export-citation href="#" class="button-link download" title="Export Citation">Export</a></div><div class="modal-window-wrapper is-hidden"><div id=export-citation-popup class="modal-window padding-20"><div class=modal-window-close-button></div><div class=modal-window-title>Export Citation</div><div class=modal-window-row><div><input type=radio name=export-citation-option value=WORKSPACE /><span class=radio-label>F1000Workspace</span></div><div><input type=radio name=export-citation-option value=ENDNOTE /><span class=radio-label>EndNote</span></div><div><input type=radio name=export-citation-option value=REF_MANAGER /><span class=radio-label>Ref. Manager</span></div><div><input type=radio name=export-citation-option value=BIBTEX /><span class=radio-label>Bibtex</span></div><div><input type=radio name=export-citation-option value=PROCITE /><span class=radio-label>ProCite</span></div><div><input type=radio name=export-citation-option value=SENTE /><span class=radio-label>Sente</span></div></div><div class=modal-window-footer><button class=general-white-orange-button id=export-citation-submit>EXPORT</button></div><div class=default-error style="display: none;">Select a format first</div></div></div></div><div class="article-interaction-info article-page"><div class=article-interaction-button><span class="f1r-icon icon-90_track"></span><a class="button-link track-article" data-article-id=7563 id=track-article-signin-7563 title="Be alerted to any updates to this article." href="/login?originalPath=/trackArticle/7563?target=/articles/4-1521/v2">Track</a></div></div><div class="article-interaction-info article-page"><div class="article-interaction-button email-article"><span class="f1r-icon icon-6_email"></span><a href="#" class=button-link title="Email this article">Email</a></div><div class="email-article-version-container small-tooltip _chrome-fix"><div class=close-icon><span class="f1r-icon icon-3_close_big"></span></div><script src='https://www.recaptcha.net/recaptcha/api.js'></script><form class="recommend-version-form research-layout"><p>All fields are required.</p><input name=versionId type=hidden value=8770 /><input name=articleId type=hidden value=7563 /><input name=senderName class="form-input-field reg-form" value="" type=text placeholder="Your name"/><input name=senderEmail class="form-input-field reg-form margin-top" value="" type=text placeholder="Your email address"/><textarea name=recipientEmails class="form-textarea-field ninetynine-percent-wide margin-top no-resize" placeholder="Recipient email address(es) (comma delimited)"></textarea><input class="form-input-field reg-form margin-top" name=subject type=text value="Interesting article on F1000Research" placeholder=Subject /><textarea name=message class="form-textarea-field reg-form margin-top no-resize">I thought this article from F1000Research (https://f1000research.com) would be of interest to you.</textarea><div class="g-recaptcha margin-top" data-sitekey=6LcHqxoUAAAAANP3_0TzpGG6qFvl4DhbUcuRzw7W></div><input value="" name=captcha type=hidden /><p>A full article citation will be automatically included.</p><p><img class="ticker-email-article-details hidden" src="/img/ticker.gif" alt=loading /></p><button class="secondary orange margin-bottom" data-test-id=version_share_email_send>SEND EMAIL</button><div class="orange-message margin-bottom is-hidden" data-test-id=version_share_email_message></div></form></div></div><div class="article-interaction-info article-page"><div class=article-interaction-button><span class="f1r-icon icon-34_share"></span><a href="#" class="button-link last addthis_button share-article" title="Share this article">Share</a></div></div></div><div id=article-interaction-control-tab class=article-interaction-control-tab><div id=hide-article-interaction class=article-interaction-control title="Hide Toolbox">&#9644;</div><div id=show-article-interaction class="article-interaction-control open" title="Show Toolbox">&#10010;</div></div></div><div class="article-header-information article-page"><div class="research-layout prime-recommended-wrapper"><a href="http://f1000.com/prime/726181976" target=_blank class="prime-recommended-iconlink big" title="F1000 Recommended"><span class="f1r-icon icon-79_faculty_recommended_badge biggest red vmiddle"></span><span class="prime-red article-text">F1000 recommended</span></a></div><div class="f1r-article-mobile article-heading-bar"></div><div class="article-type article-display">Method Article </div><div class="article-title-and-info article-view highlighted-article" id=anchor-title><span title="#springMessage("NewVersionReasonType.REVISED.tooltip")" class=" updated-box updated-box-bg orange-bg revised helper">Revised</span> Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences<span class=other-info> [version 2; peer review: 2 approved]</span></div><div class=article-subtitle></div><div class=f1r-article-desk><div class="authors _mdl-layout"><span class=""><a href="mailto:charlotte.soneson@uzh.ch" title="Send email" class="cauthor research-layout"><span class='f1r-icon icon-6_email orange'></span><span>Charlotte Soneson</span></a><a href="https://orcid.org/0000-0003-3833-2169" target=_blank id=author-orcid-0><span class=orcid-logo-for-author-list></span></a><div class="mdl-tooltip mdl-tooltip--wider" for=author-orcid-0><span class=orcid-logo-for-author-list></span> https://orcid.org/0000-0003-3833-2169</div><sup>1,2</sup>,&nbsp;</span><span class=""><a href="mailto:michaelisaiahlove@gmail.com" title="Send email" class="cauthor research-layout"><span class='f1r-icon icon-6_email orange'></span><span>Michael I. Love</span></a><a href="https://orcid.org/0000-0001-8401-0545" target=_blank id=author-orcid-1><span class=orcid-logo-for-author-list></span></a><div class="mdl-tooltip mdl-tooltip--wider" for=author-orcid-1><span class=orcid-logo-for-author-list></span> https://orcid.org/0000-0001-8401-0545</div><sup>3,4</sup>,&nbsp;</span><span class=""><a href="mailto:mark.robinson@imls.uzh.ch" title="Send email" class="cauthor research-layout"><span class='f1r-icon icon-6_email orange'></span><span>Mark D. Robinson</span></a><a href="https://orcid.org/0000-0002-3048-5518" target=_blank id=author-orcid-2><span class=orcid-logo-for-author-list></span></a><div class="mdl-tooltip mdl-tooltip--wider" for=author-orcid-2><span class=orcid-logo-for-author-list></span> https://orcid.org/0000-0002-3048-5518</div><sup>1,2</sup></span></div></div><div class=f1r-article-mobile><div class="authors _mdl-layout"><span class=""><a href="mailto:charlotte.soneson@uzh.ch" title="Send email" class="cauthor research-layout"><span class='f1r-icon icon-6_email orange'></span><span>Charlotte Soneson</span></a><a href="http://orcid.org/0000-0003-3833-2169" target=_blank id=mauthor-orcid-0><span class=orcid-logo-for-author-list></span></a><div class="mdl-tooltip mdl-tooltip--wider" for=mauthor-orcid-0><span class=orcid-logo-for-author-list></span> https://orcid.org/0000-0003-3833-2169</div><sup>1,2</sup>,&nbsp;</span><span class=""><a href="mailto:michaelisaiahlove@gmail.com" title="Send email" class="cauthor research-layout"><span class='f1r-icon icon-6_email orange'></span><span>Michael I. Love</span></a><a href="http://orcid.org/0000-0001-8401-0545" target=_blank id=mauthor-orcid-1><span class=orcid-logo-for-author-list></span></a><div class="mdl-tooltip mdl-tooltip--wider" for=mauthor-orcid-1><span class=orcid-logo-for-author-list></span> https://orcid.org/0000-0001-8401-0545</div><sup>3,4</sup>,&nbsp;</span><span class=""><a href="mailto:mark.robinson@imls.uzh.ch" title="Send email" class="cauthor research-layout"><span class='f1r-icon icon-6_email orange'></span><span>Mark D. Robinson</span></a><a href="http://orcid.org/0000-0002-3048-5518" target=_blank id=mauthor-orcid-2><span class=orcid-logo-for-author-list></span></a><div class="mdl-tooltip mdl-tooltip--wider" for=mauthor-orcid-2><span class=orcid-logo-for-author-list></span> https://orcid.org/0000-0002-3048-5518</div><sup>1,2</sup></span></div></div><div class=f1r-article-mobile><div class=article-pubinfo-mobile> PUBLISHED 29 Feb 2016 </div></div><span class=Z3988 title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft_id=info:doi/10.12688%2Ff1000research.7563.2"></span><div class=f1r-article-desk><div class="contracted-details first"><a href="#" class="contracted-details-label author-affiliations"><span class=contracted></span>Author details</a><a href="#" class=section-title>Author details</a><span class="f1r-icon icon-14_more_small section-control"></span><span class="f1r-icon icon-10_less_small section-control"></span><div class="expanded-details affiliations is-hidden"><sup>1</sup> Institute for Molecular Life Sciences, University of Zurich, Zurich, 8057, Switzerland<br/><sup>2</sup> SIB Swiss Institute of Bioinformatics, University of Zurich, Zurich, 8057, Switzerland<br/><sup>3</sup> Department of Biostatistics and Computational Biology, Dana-Farber Cancer Institute, Boston, MA, 02210, USA<br/><sup>4</sup> Department of Biostatistics, Harvard TH Chan School of Public Health, Boston, MA, 02115, USA<br/><p></p></div></div></div><div class=f1r-article-mobile><div class="article-page-section-box margin-bottom-40 research-layout"><span class=box-title><span class="f1r-icon icon-85_peer_review"></span> OPEN PEER REVIEW </span><button class="tertiary grey float-right" data-scrollto=article-reports>DETAILS</button><div class="status-row referee-reports-container"> REVIEWER STATUS <span class=status-icons><span class="f1r-icon icon-86_approved status-green smaller" title=Approved data-refInfo=13464-11761></span><span class="f1r-icon icon-86_approved status-green smaller" title=Approved data-refInfo=11706-11745></span></span></div></div></div><div class="f1r-article-desk padding-top"><div class=article-collection-wrapper><div class="o-media o-media--sm o-media--center"><div class=o-media__img><a href="https://f1000research.com/gateways/bioconductor" title="Open Gateway"><img src="https://f1000researchdata.s3.amazonaws.com/thumbnails/18bd34a0-c579-44be-873b-4a18b149eaee_collection.thumbnail"></a></div><div class=o-media__body><p class="u-mt--0 u-mb--0">This article is included in the <a href="https://f1000research.com/gateways/bioconductor">Bioconductor</a> gateway.</p></div></div></div><div class=article-collection-wrapper><div class="o-media o-media--sm o-media--center"><div class=o-media__img><a href="https://f1000research.com/gateways/rpackage" title="Open Gateway"><img src="https://f1000researchdata.s3.amazonaws.com/thumbnails/2c9ffab2-7c03-43a4-ba50-66f9a5a94633_collection.thumbnail"></a></div><div class=o-media__body><p class="u-mt--0 u-mb--0">This article is included in the <a href="https://f1000research.com/gateways/rpackage">RPackage</a> gateway.</p></div></div></div></div></div><h2 class="article-headings article-page-abstract" id=anchor-abstract><span class="f1r-article-mobile-inline abstract-heading-border"></span> Abstract </h2><div class="article-abstract article-page-general-text-mobile research-layout"><div class="abstract-text is-expanded"> High-throughput sequencing of cDNA (RNA-seq) is used extensively to characterize the transcriptome of cells. Many transcriptomic studies aim at comparing either abundance levels or the transcriptome composition between given conditions, and as a first step, the sequencing reads must be used as the basis for abundance quantification of transcriptomic features of interest, such as genes or transcripts. Various quantification approaches have been proposed, ranging from simple counting of reads that overlap given genomic regions to more complex estimation of underlying transcript abundances. In this paper, we show that gene-level abundance estimates and statistical inference offer advantages over transcript-level analyses, in terms of performance and interpretability. We also illustrate that the presence of differential isoform usage can lead to inflated false discovery rates in differential gene expression analyses on simple count matrices but that&nbsp;this can be addressed by incorporating offsets derived from transcript-level abundance estimates. We also show that the problem is relatively minor in several real data sets. Finally, we provide an R package (<i>tximport</i>) to help users integrate transcript-level abundance estimates from common quantification pipelines into count-based statistical inference engines. </div><div class=abstract-for-mobile><div class="margin-top-30 padding-bottom-30 research-layout is-centered"><button class="primary orange-text white-bg bigger-text abstract-expand-button-mobile with-border show" style="display: none;"> READ ALL <span class="f1r-icon icon-14_more_small orange vmiddle big"></span></button><button class="primary orange-text white-bg bigger-text abstract-expand-button-mobile with-border hide"> READ LESS <span class="f1r-icon icon-10_less_small orange vmiddle big"></span></button></div></div></div><div class=clearfix></div><div class="article-context no-divider"><div class="article-abstract article-page-general-text-mobile research-layout generated-article-body"><h2 class=main-title>Keywords</h2><p class="u-mb--0 u-pb--2"> RNA-seq, quantification, gene expression, transcriptomics </p></div></div><div class=article-information><span class="info-separation padding-bottom"><div id=corresponding-author-icon class="email-icon float-left"><span class="f1r-icon icon-6_email orange"></span><div id=corresponding-author-window class="margin-top-20 popup-window-wrapper is-hidden"><div class="popup-window corresponding-authors-popup"><div class=corresponding-author-container><div class="popup-window-title small">Corresponding Author(s)</div><div class=authors> Charlotte Soneson (<a href="mailto:charlotte.soneson@uzh.ch">charlotte.soneson@uzh.ch</a>) <br> Michael I. Love (<a href="mailto:michaelisaiahlove@gmail.com">michaelisaiahlove@gmail.com</a>) <br> Mark D. Robinson (<a href="mailto:mark.robinson@imls.uzh.ch">mark.robinson@imls.uzh.ch</a>) </div></div><div class="margin-top margin-bottom float-left"><button id=close-popup-window class=general-white-orange-button>Close</button></div></div></div></div><span class="icon-text float-left" data-test-id=box-corresponding-author><b>Corresponding authors:</b> Charlotte Soneson, Michael I. Love, Mark D. Robinson </span><div class=clearfix></div></span><span class="info-separation padding-bottom competing-interests-display"><span class=competing-interests-title>Competing interests:</span> No competing interests were disclosed. </span><div class="info-separation padding-bottom grant-information-display"><span class=grant-information-title>Grant information:</span> MDR and CS acknowledge support from the "RNA & Disease" National Center of Competence in Research, an SNSF project grant (143883) and from the European Commission through the 7th Framework Collaborative Project RADIANT (Grant Agreement Number: 305626). MIL was supported by NIH grant 5T32CA009337- 35. <br/><i>The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</i></div><span class="f1r-article-desk info-separation padding-bottom"><span class="copywrite-icon float-left"><span class="f1r-icon icon-100_open_access"></span></span><span class="icon-text float-left" data-test-id=box-copyright-text><b>Copyright:</b>&nbsp;  2016 Soneson C <em>et al</em>. This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/" target=_blank data-test-id=box-licence-link>Creative Commons Attribution Licence</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. Data associated with the article are available under the terms of the <a href="http://creativecommons.org/publicdomain/zero/1.0/" target=_blank data-test-id=box-data-licence-link>Creative Commons Zero "No rights reserved" data waiver</a> (CC0 1.0 Public domain dedication). </span><div class=clearfix></div></span><span class="info-separation padding-bottom" data-test-id=box-how-to-cite><b>How to cite:</b><span class="article-title-and-info in-article-box"> Soneson C, Love MI and Robinson MD. Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences [version 2; peer review: 2 approved]</span>. <i>F1000Research</i> 2016, <b>4</b>:1521 (<a href="https://doi.org/10.12688/f1000research.7563.2" target=_blank>https://doi.org/10.12688/f1000research.7563.2</a>) </span><span class=info-separation data-test-id=box-first-published><b>First published:</b> 30 Dec 2015, <b>4</b>:1521 (<a href="https://doi.org/10.12688/f1000research.7563.1" target=_blank>https://doi.org/10.12688/f1000research.7563.1</a>)</span><span class=info-separation data-test-id=box-latest-published><b>Latest published:</b> 29 Feb 2016, <b>4</b>:1521 (<a href="https://doi.org/10.12688/f1000research.7563.2" target=_blank>https://doi.org/10.12688/f1000research.7563.2</a>)</span></div><div class="update-box-wrapper update-text-view-control article-updated-box "><h2 class="h2-title no-top-margin"><span title="#springMessage("NewVersionReasonType.REVISED.tooltip")" class=" updated-box updated-box-bg orange-bg revised ">Revised</span> Amendments from Version 1 <span class="research-layout f1r-article-mobile-inline update-box-control"><span class="f1r-icon icon-14_more_small big midgrey"></span><span class="f1r-icon icon-10_less_small big midgrey" style="display: none;"></span></span></h2><div class=f1r-article-desk><p>In version 2 of the manuscript, we have reworded and improved the clarity of the text in several places, to better convey the differences between the contrasted approaches and clarify the questions that are addressed by each analysis. We have also expanded the supplementary material with additional analyses of accuracy of abundance estimates among paralogous genes, and added more detailed descriptions of aspects such as the calculation of each abundance measure and the generation of the incomplete annotation files, information that was previously only available in the respective Datasets. The Discussion section has been extended, mainly to include a discussion of differential expression analysis methods incorporating variance estimates.</p></div><div class="f1r-article-mobile research-layout article-page-general-text-mobile padding-top-20" style="display: none;"> In version 2 of the manuscript, we have reworded and improved the clarity of the text in several places, to better convey the differences between the contrasted approaches and clarify the questions that are addressed by each analysis. We have also expanded the supplementary material with additional analyses of accuracy of abundance estimates among paralogous genes, and added more detailed descriptions of aspects such as the calculation of each abundance measure and the generation of the incomplete annotation files, information that was previously only available in the respective Datasets. The Discussion section has been extended, mainly to include a discussion of differential expression analysis methods incorporating variance estimates. </div><p class=f1r-article-desk><a href="#article-reports" class="black-link bold">See reviewer responses</a></p><span class=f1r-article-mobile><div class="margin-top-30 margin-bottom research-layout is-centered" style="display: none;"><a href="#article-reports"><button class="primary orange extra-padding">READ REVIEWER RESPONSES</button></a></div></span></div><div class=clearfix></div><div id=article-context class=article-context><div id=article1-body class=generated-article-body><h2 class=main-title id=d15331e274>Introduction</h2><p class="" id=d15331e277>Quantification and comparison of isoform- or gene-level expression based on high throughput sequencing reads from cDNA (RNA-seq) are arguably among the most common tasks in modern computational molecular biology. Currently, one of the most widely used approaches amounts to defining the genomic locations of a set of non-overlapping targets (typically, genes) and using the number of aligned reads overlapping a target as a measure of its abundance, or expression level. Several software packages have been developed for performing such simple counting (e.g., <i>featureCounts</i><sup><a href="#ref-1">1</a></sup> and <i>HTSeq-count</i><sup><a href="#ref-2">2</a></sup>). More recently, the field has seen a surge in methods aimed at quantifying the abundances of individual <i>transcripts</i> (e.g., <i>Cufflinks</i><sup><a href="#ref-3">3</a></sup>, <i>RSEM</i><sup><a href="#ref-4">4</a></sup>, <i>BitSeq</i><sup><a href="#ref-5">5</a></sup>, <i>kallisto</i><sup><a href="#ref-6">6</a></sup> and <i>Salmon</i><sup><a href="#ref-7">7</a></sup>). These methods provide higher resolution than simple counting, and by circumventing the computationally costly read alignment step, some (notably, <i>kallisto</i> and <i>Salmon</i>) are also considerably faster. However, isoform quantification is more complex than the simple counting, due to the high degree of overlap among transcripts. Currently, there is no consensus regarding the optimal resolution or method for quantification and downstream analysis of transcriptomic output.</p><p class="" id=d15331e333>Another point of debate is the unit in which abundances are given. The traditional R/FPKM<sup><a href="#ref-8">8</a>,<a href="#ref-9">9</a></sup> (reads/fragments per kilobase per million reads) have been largely superseded by the TPM<sup><a href="#ref-10">10</a></sup> (transcripts per million), since the latter is more consistent across libraries. Regardless, all these units attempt to correct for sequencing depth and feature length and thus do not reflect the influence of these on quantification uncertainty. In order to account for these aspects, most statistical tools for analysis of RNA-seq data operate instead on the <i>count</i> scale. Most of these tools were designed to be applied to simple read counts, and the degree to which their performance is affected by using fractional estimated counts resulting from portioning reads aligning to multiple transcripts is still an open question. The fact that the most common sequencing protocols provide reads that are much shorter than the average transcript implies that the observed read counts depend on a transcripts length as well as its abundance; thus, simple counts are arguably less accurate measures than TPMs of the true abundance of RNA molecules from given genes. The use of gene counts as input to statistical tools typically assumes that the length of the expressed part of a gene does not change across samples and thus its impact can be ignored for differential analysis.</p><p class="" id=d15331e350>In the analysis of transcriptomic data, as for any other application, it is of utmost importance that the question of interest is precisely defined before a computational approach is selected. Often, the interest lies in comparing the transcriptional output between different conditions, and most RNA-seq studies can be classified as either: 1) differential gene expression (DGE) studies, where the overall transcriptional output of each gene is compared between conditions; 2) differential transcript/exon usage (DTU/DEU) studies, where the composition of a genes isoform abundance spectrum is compared between conditions, or 3) differential transcript expression (DTE) studies, where the interest lies in whether individual transcripts show differential expression between conditions. DTE analysis results can be represented on the individual transcript level, or aggregated to the gene level, e.g., by evaluating whether <i>at least one</i> of the isoforms shows evidence of differential abundance.</p><p class="" id=d15331e356>In this report, we make and give evidence for three claims: 1) gene-level estimation is considerably more accurate than transcript-level; 2) regardless of the level at which abundance estimation is done, <i>inferences</i> at the gene level are appealing in terms of robustness, statistical performance and interpretation; 3) taking advantage of transcript-level abundance estimates when defining or analyzing gene-level abundances leads to improved DGE results compared to simple counting for genes exhibiting DTU. The magnitude of the effect in a given data set thus depends on the extent of DTU, and the global impact is relatively small in several real data sets analyzed in this study.</p><p class="" id=d15331e363>To facilitate a broad range of analysis choices, depending on the biological question of interest, we provide an R/Bioconductor package, <i>tximport</i>, to import transcript lengths and abundance estimates from several popular quantification packages and export (estimated) count matrices and, optionally, average transcript length correction terms (i.e., offsets) that can be used as inputs to common statistical engines, such as <i>DESeq2</i><sup><a href="#ref-11">11</a></sup>, <i>edgeR</i><sup><a href="#ref-12">12</a></sup> and <i>limma</i><sup><a href="#ref-13">13</a></sup>.</p></div><div id=article1-body class=generated-article-body><h2 class=main-title id=d15331e390>Data and methods</h2><p class="" id=d15331e393>Throughout this manuscript, we utilize two simulated data sets and four experimental data sets (Bottomly<sup><a href="#ref-14">14</a></sup> [<a href="#DS2">Data set 3</a>], GSE64570<sup><a href="#ref-15">15</a></sup> [<a href="#DS3">Data set 4</a>], GSE69244<sup><a href="#ref-16">16</a></sup> [<a href="#DS4">Data set 5</a>], GSE72165<sup><a href="#ref-17">17</a></sup> [<a href="#DS5">Data set 6</a>], see <a href="#SM1">Supplementary File 1</a> for further details) for illustration. Details on the data generation and full records of the analyses are provided in the data sets and <a href="#SM1">Supplementary File 1</a>. The first simulated data set (sim1; <a href="#DS0">Data set 1</a>) is the synthetic human data set from Soneson <i>et al</i>.<sup><a href="#ref-18">18</a></sup>, comprising 20,410 genes and 145,342 transcripts and is available from ArrayExpress (accession E-MTAB-3766). This data set consists of three biological replicates from each of two simulated conditions, and differential isoform usage was introduced for 1,000 genes by swapping the relative expression levels of the two most dominant isoforms between conditions. For each gene in this data set, the total transcriptional output is the same in the two conditions (i.e., no overall DGE); it is worth noting that this is an extreme situation, but provides a useful test set for contrasting DGE, DTU and DTE. The second simulated data set (sim2; <a href="#DS1">Data set 2</a>) is a synthetic data set comprising the 3,858 genes and 15,677 transcripts from the human chromosome 1. It is available from ArrayExpress with accession E-MTAB-4119. Also here, we simulated two conditions with three biological replicates each. For this data set, we simulated both overall DGE, where all transcripts of the affected gene showed the same fold change between the conditions (420 genes), differential transcript usage (DTU), where the total transcriptional output was kept constant but the relative contribution from the transcripts changed (420 genes) and differential transcript expression (DTE), where the expression of 10% of the transcripts of each affected gene was modified (422 genes, 528 transcripts). The three sets of modified genes were disjoint. Again, this synthetic data set represents an extreme situation compared to most real data sets, but provides a useful test case to identify underlying causes of differences between results from various analysis pipelines.</p><p class="" id=d15331e445>In addition to <a href="#DS0">Data set 1</a><a href="#DS5">Data set 6</a>, which contain all code for reproducing our analyses, further method descriptions are given in <a href="#SM1">Supplementary File 1</a>.</p><div class="table-wrap panel dataset clearfix"><a name=DS0 class=n-a></a><div class="dataset-thumbnail float-left is-hidden" thumbnail-id=DS0></div><div class=h5-label><span class=normal-text>Dataset 1.</span>Data set 1. </div><span class=first id=d15331e464><a target=xrefwindow href="http://dx.doi.org/10.5256/f1000research.7563.d114722" id=d15331e466>http://dx.doi.org/10.5256/f1000research.7563.d114722</a></span><span id=d15331e470>Dataset 1 (html) contains all the R code that was used to perform the analyses and generate the figures for the <b>sim1</b> data set<sup><a href="#ref-30">30</a></sup>.</span></div><div class="flashwrapper is-hidden"><div class=flashContent id=DS0></div></div><div class="table-wrap panel dataset clearfix"><a name=DS1 class=n-a></a><div class="dataset-thumbnail float-left is-hidden" thumbnail-id=DS1></div><div class=h5-label><span class=normal-text>Dataset 2.</span>Data set 2. </div><span class=first id=d15331e489><a target=xrefwindow href="http://dx.doi.org/10.5256/f1000research.7563.d114723" id=d15331e491>http://dx.doi.org/10.5256/f1000research.7563.d114723</a></span><span id=d15331e495>Data set 2 (html) contains all the R code that was used to perform the analyses and generate the figures for the <b>sim2</b> data set<sup><a href="#ref-31">31</a></sup>.</span></div><div class="flashwrapper is-hidden"><div class=flashContent id=DS1></div></div><div class="table-wrap panel dataset clearfix"><a name=DS2 class=n-a></a><div class="dataset-thumbnail float-left is-hidden" thumbnail-id=DS2></div><div class=h5-label><span class=normal-text>Dataset 3.</span>Data set 3. </div><span class=first id=d15331e515><a target=xrefwindow href="http://dx.doi.org/10.5256/f1000research.7563.d114724" id=d15331e517>http://dx.doi.org/10.5256/f1000research.7563.d114724</a></span><span id=d15331e521>Data set 3 (html) contains all the R code that was used to perform the analyses and generate the figures for the <b>Bottomly</b> data set<sup><a href="#ref-32">32</a></sup>.</span></div><div class="flashwrapper is-hidden"><div class=flashContent id=DS2></div></div><div class="table-wrap panel dataset clearfix"><a name=DS3 class=n-a></a><div class="dataset-thumbnail float-left is-hidden" thumbnail-id=DS3></div><div class=h5-label><span class=normal-text>Dataset 4.</span>Data set 4. </div><span class=first id=d15331e540><a target=xrefwindow href="http://dx.doi.org/10.5256/f1000research.7563.d114725" id=d15331e542>http://dx.doi.org/10.5256/f1000research.7563.d114725</a></span><span id=d15331e546>Data set 4 (html) contains all the R code that was used to perform the analyses and generate the figures for the <b>GSE64570</b> data set<sup><a href="#ref-33">33</a></sup>.</span></div><div class="flashwrapper is-hidden"><div class=flashContent id=DS3></div></div><div class="table-wrap panel dataset clearfix"><a name=DS4 class=n-a></a><div class="dataset-thumbnail float-left is-hidden" thumbnail-id=DS4></div><div class=h5-label><span class=normal-text>Dataset 5.</span>Data set 5. </div><span class=first id=d15331e565><a target=xrefwindow href="http://dx.doi.org/10.5256/f1000research.7563.d114726" id=d15331e567>http://dx.doi.org/10.5256/f1000research.7563.d114726</a></span><span id=d15331e571>Data set 5 (html) contains all the R code that was used to perform the analyses and generate the figures for the <b>GSE69244</b> data set<sup><a href="#ref-34">34</a></sup>.</span></div><div class="flashwrapper is-hidden"><div class=flashContent id=DS4></div></div><div class="table-wrap panel dataset clearfix"><a name=DS5 class=n-a></a><div class="dataset-thumbnail float-left is-hidden" thumbnail-id=DS5></div><div class=h5-label><span class=normal-text>Dataset 6.</span>Data set 6. </div><span class=first id=d15331e590><a target=xrefwindow href="http://dx.doi.org/10.5256/f1000research.7563.d114730" id=d15331e592>http://dx.doi.org/10.5256/f1000research.7563.d114730</a></span><span id=d15331e596>Data set 6 (html) contain all the R code that was used to perform the analyses and generate the figures for the <b>GSE72165</b> data set<sup><a href="#ref-35">35</a></sup>.</span></div><div class="flashwrapper is-hidden"><div class=flashContent id=DS5></div></div></div><div id=article1-body class=generated-article-body><h2 class=main-title id=d15331e611>Gene abundance estimates are more accurate than transcript abundance estimates</h2><p class="" id=d15331e614>To evaluate the accuracy of abundance estimation with transcript and gene resolution, we used the quasi-mapping mode of <i>Salmon</i><sup><a href="#ref-7">7</a></sup> (v0.5.1) to estimate the TPM for each transcript in each of the data sets. Gene-level TPM estimates, representing the overall transcriptional output of each gene, were obtained by summing the corresponding transcript-level TPM estimates. For the two simulated data sets, the true underlying TPM of each feature was known and we could thus evaluate the accuracy of the estimates. Unsurprisingly, gene-level estimates were more accurate than transcript-level estimates (<a href="#f1">Figure 1A</a>, <a href="#SM1">Supplementary Figures 4,5</a>). We also derived TPM estimates from simple gene-level counts obtained from traditional alignment of the reads to the genome using STAR followed by counting with <i>featureCounts</i>, by dividing the read count for each gene with a reasonable measure of the length of the gene (the length of the union of its exons) and the total number of mapped reads, and scaling the estimates to sum to 1 million. The simple count estimates showed a lower correlation with the true TPMs than the <i>Salmon</i> estimates, in line with previous observations<sup><a href="#ref-19">19</a></sup>. It is worth noting that we are comparing entire (typical) workflows, and that differences may also occur if the set of reads that STAR is able to align to the genome is not identical to the set of reads that are contributing to the abundance estimation of <i>Salmon</i>. However, due to the large fraction of aligned reads and the high mapping rate with <i>Salmon</i> (both exceeding 99.8%, more than 95% of the reads were subsequently unambiguously assigned to genes by <i>featureCounts</i>), we do not expect this to have a major impact on the results shown in <a href="#f1">Figure 1A</a>.</p><div class="fig panel clearfix"><a name=f1 class=n-a></a><a target=_blank href="https://f1000researchdata.s3.amazonaws.com/manuscripts/8770/aca07401-7c11-4875-92b3-89c387175c53_figure1.gif"><img alt="aca07401-7c11-4875-92b3-89c387175c53_figure1.gif" src="https://f1000researchdata.s3.amazonaws.com/manuscripts/8770/aca07401-7c11-4875-92b3-89c387175c53_figure1.gif"></a><div class=caption><h3>Figure 1 (sim2). </h3><p class=first id=d15331e659><b>A</b>: Accuracy of gene- and transcript-level TPM estimates from <i>Salmon</i> and scaled FPKM estimates derived from simple counts from <i>featureCounts</i>, in one of the simulated samples (sampleA1). Spearman correlations are indicated in the respective panels. Top row: using the complete annotation. Bottom row: using an incomplete annotation, with 20% of the transcripts randomly removed. Gene-level estimates are more accurate than transcript-level estimates. Gene-level estimates from <i>Salmon</i> are more accurate than those from <i>featureCounts</i>. <b>B</b>: Distribution of the coefficients of variation of gene- and transcript-level abundance estimates from <i>Salmon</i>, calculated across 30 bootstrap samples of one of the simulated samples (sampleA1). Gene-level estimates are less variable than transcript-level estimates. <b>C</b>: An example of unidentifiable transcript-level estimates, as uneven coverage does not cover the critical regions that would determine the amount that each transcript is expressed, while gene-level estimation is still possible.</p></div></div><p class="" id=d15331e691>Gene-level estimates derived from both simple counts and <i>Salmon</i> tended to show a high degree of robustness against incompleteness of the annotation catalog, as evidenced from estimation errors after first removing (at random) 20% of the transcripts (<a href="#f1">Figure 1A</a>, see also <a href="#SM1">Supplementary File 1</a>); in contrast, <i>Salmon</i>s transcript estimate accuracies deteriorated. To further compare the merits of genome alignment-based vs alignment-free quantification, especially in their handling of multi-mapping reads, we investigated the accuracy of the abundance estimates within sets of paralogous genes (<a href="#SM1">Supplementary Figures 13</a>). Also here, <i>Salmon</i> provided more consistently accurate estimates than STAR+<i>featureCounts</i>. From the bootstrap estimates generated by <i>Salmon</i>, we also estimated the coefficient of variation of the abundance estimates. The gene-level estimates showed considerably lower variability than the transcript-level estimates in both simulated and experimental data (<a href="#f1">Figure 1B</a>, <a href="#SM1">Supplementary Figures 6,7</a>). Taken together, these observations suggest that the gene-level estimates are more accurate than transcript-level estimates and therefore potentially allow a more accurate and stable statistical analysis. A further argument in favor of gene-level analysis is the unidentifiability of transcript expression that can result from uneven coverage caused by underlying technical biases. While some extent of coverage variability might be alleviated by corrections for sequence- or position-specific biases<sup><a href="#ref-20">20</a></sup>, there remain cases where transcript expression cannot be inferred from data (<a href="#f1">Figure 1C</a>). Intermediate approaches, grouping together indistinguishable features are also conceiveable<sup><a href="#ref-21">21</a></sup>, but not yet standard practice.</p></div><div id=article1-body class=generated-article-body><h2 class=main-title id=d15331e740>DTE is more powerful and easier to interpret on gene level than for individual transcripts</h2><p class="" id=d15331e743>DTE is concerned with inference of changes in abundance at transcript resolution, and thus invokes a statistical test for each transcript. We argue that this can lead to several complications: the first is conceptual, since the rows (transcripts) in the result table will in many cases not be interpreted independently, since the researcher is often interested in comparing the results for transcripts from the same gene locus, and the second one is more technical, since the number of transcripts is considerably larger than the number of genes, which could lead to lower power due to the portioning of the total set of reads across a larger number of features and a potentially higher multiple testing penalty. We tested for DTE on the simulated data by applying <i>edgeR</i><sup><a href="#ref-12">12</a></sup> to the transcript counts obtained from <i>Salmon</i> (the application of count models to <i>estimated</i> counts is discussed in the next Section), and represented the results as transcript-level p-values or aggregated these to the gene level by using the <i>perGeneQValue</i> function from the <i>DEXSeq</i><sup><a href="#ref-22">22</a></sup> R package. Note that the transcript-level DTE test assesses the null hypothesis that individual transcripts do not change their expression, whereas the gene-level DTE test assesses the null hypothesis that <i>all</i> transcripts from a given gene exhibit no change in expression. Framing the DTE question at the gene level results in higher power, without sacrificing false discovery rate control (<a href="#f2">Figure 2A</a>). This is not surprising given the different null hypotheses, and, in fact, for many of the genes detected as true positives with the gene-level test, only a subset of the truly changing transcripts were detected (<a href="#SM1">Supplementary Figure 8</a>). We note that this type of gene-level aggregation may favor genes in which one transcript shows strong changes, and that other approaches to increase power against specific alternatives are conceivable, e.g., capitalizing on the rich collection of methods for gene set analysis.</p><div class="fig panel clearfix"><a name=f2 class=n-a></a><a target=_blank href="https://f1000researchdata.s3.amazonaws.com/manuscripts/8770/aca07401-7c11-4875-92b3-89c387175c53_figure2.gif"><img alt="aca07401-7c11-4875-92b3-89c387175c53_figure2.gif" src="https://f1000researchdata.s3.amazonaws.com/manuscripts/8770/aca07401-7c11-4875-92b3-89c387175c53_figure2.gif"></a><div class=caption><h3>Figure 2 (sim2). </h3><p class=first id=d15331e784><b>A</b>: DTE detection performance on transcript- and gene-level, using <i>edgeR</i> applied to transcript-level estimated counts from <i>Salmon</i>. The statistical analysis was performed on transcript level and aggregated for each gene using the <i>perGeneQValue</i> function from the <i>DEXSeq</i> R package; aggregated results show higher detection power. The curves trace out the observed FDR and TPR for each significance cutoff value. The three circles mark the performance at adjusted p-value cutoffs of 0.01, 0.05 and 0.1. <b>B</b>: Schematic illustration of different ways in which differential transcript expression (DTE) can arise, in terms of absence or presence of differential gene expression (DGE) and differential transcript usage (DTU).</p></div></div><p class="" id=d15331e810>While DTE analysis is more suitable than DGE analysis for detecting genes with changes in absolute or relative isoform expression but no or only minor change in overall output (<a href="#SM1">Supplementary Figure 9</a>), we argue that even gene-level DTE results may suffer from lack of interpretability. DTE can manifest in several different ways, as an overall differential expression of the gene or differential relative usage of its transcripts, or a combination of the two (<a href="#f2">Figure 2B</a>). We argue that the biological question of interest is in many cases more readily interpretable as a combination of DGE and DTU, rather than DTE. It has been our experience that results reported at the transcript level are still often cast to the gene level (i.e., given a differentially expressed transcript, researchers want to know whether also other isoforms of the gene are changing), suggesting that asking two specific gene-level questions (Is the overall abundance changing? Are the isoform abundances changing proportionally?) trumps the interpretability of one broad question addressing the transcript abundances (Are there changes in any of the isoform expression levels?), despite the increased need for multiple testing correction associated with performing two tests for each gene rather than one. There are of course also situations when a transcript-centric approach provides superior interpretability, for example in targeted experiments where specific isoforms are expected to change due to an administered treatment.</p></div><div id=article1-body class=generated-article-body><h2 class=main-title id=d15331e822>Incorporating transcript-level estimates leads to more accurate DGE results</h2><p class="" id=d15331e825>DGE (i.e., testing for changes in the overall transcriptional output of a gene) is typically performed by applying a count-based inference method from statistical packages such as <i>edgeR</i><sup><a href="#ref-12">12</a></sup> or <i>DESeq2</i><sup><a href="#ref-11">11</a></sup> to gene counts obtained by read counting software such as <i>featureCounts</i><sup><a href="#ref-1">1</a></sup>, <i>HTSeq-count</i><sup><a href="#ref-2">2</a></sup> or functions from the <i>GenomicAlignments</i><sup><a href="#ref-23">23</a></sup> R package. A lot has been written about how simple counting approaches are prone to give erroneous results for genes with changes in relative isoform usage, due to the direct dependence of the observed read count on the transcript length<sup><a href="#ref-24">24</a></sup>, and alternatives, such as <i>Cuffdiff</i><sup><a href="#ref-24">24</a></sup>, which utilizes estimated transcript abundances, have been proposed. However, the extent of the problem in real data has not been thoroughly investigated. Here, we show that taking advantage of transcript-resolution estimates (e.g., obtained by <i>Salmon</i>) in count-based inference methods can lead to improved DGE results. We propose two alternative ways of integrating transcript abundance estimates into the DGE pipeline: to define an artificial count matrix, or to calculate offsets that can be used in the statistical modeling of the observed gene counts from, e.g., <i>featureCounts</i>. Both approaches are implemented in the accompanying <i>tximport</i> R package (available from <a target=xrefwindow href="http://bioconductor.org/packages/tximport" id=d15331e878>http://bioconductor.org/packages/tximport</a>).</p><p class="" id=d15331e882>For the DGE analyses, we defined three different gene-level count matrices for each data set (see also <a href="#SM1">Supplementary File 1</a>): 1) using <i>featureCounts</i> from the <i>Rsubread</i><sup><a href="#ref-1">1</a></sup> R package (denoted <b>featureCounts</b> below), 2) summing the estimated transcript counts from <i>Salmon</i> within genes (<b>simplesum</b>), 3) summing the estimated transcript TPMs from <i>Salmon</i> within genes, and multiplying with the total library size in millions (<b>scaledTPM</b>). We note that the scaledTPM values are artificial values, transforming underlying abundance measures to the count scale to incorporate the information provided by the sequencing depth. We further used the effective transcript lengths and estimated TPMs from <i>Salmon</i> to define average transcript lengths for each gene and each sample (normalization factors) as described in the <a href="#SM1">Supplementary material</a>, to be used as offsets for <i>edgeR</i> and <i>DESeq2</i> when analyzing the featureCounts and simplesum count matrices (<b>featureCounts_avetxl</b> and <b>simplesum_avetxl</b>).</p><p class="" id=d15331e932>Overall, the counts obtained by all methods were highly correlated (<a href="#SM1">Supplementary Figures 1012</a>), which is not surprising since any differences are likely to affect a relatively small subset of the genes. In general, the simplesum and featureCounts matrices led to similar conclusions in all considered data sets, even though there are differences between the two approaches in terms of how multi-mapping reads and reads partly overlapping intronic regions are handled<sup><a href="#ref-25">25</a></sup>. Previous studies have also shown that some loss of sensitivity for certain genes may be encountered from discarding multi-mapping fragments, which may be recovered through the use of transcript abundance estimators such as <i>Salmon</i><sup><a href="#ref-21">21</a></sup>. The concordance between simplesum and featureCounts results also suggests that statistical methods based on the Negative Binomial assumption are applicable also to summarized, gene-level <i>estimated</i> counts, which is further supported by the similarity between the p-value histograms as well as the mean-variance relationships observed with the three types of count matrices (<a href="#SM1">Supplementary Figures 1318</a>).</p><p class="" id=d15331e954>Accounting for the potentially varying average transcript length across samples when performing DGE, either in the definition of the count matrix (scaledTPM) or by defining offsets (featureCounts_avetxl, simplesum_avetxl), led to considerably improved false discovery rate (FDR) control compared to using the observed <i>featureCounts</i> or aggregated <i>Salmon</i> counts directly (<a href="#f3">Figure 3A</a>, <a href="#T1">Table 1</a>). It is important to note that this improvement is entirely attributable to an improved handling of genes with changes in isoform composition between the conditions (<a href="#f3">Figure 3B</a>, <a href="#SM1">Supplementary Figure 19</a>), that we purposely introduced strong signals in the simulated data set in order to pinpoint these underlying causes, and that the overall effect in a real data set will depend on the extent to which considerable DTU is present. Experiments on various real data sets (<a href="#SM1">Supplementary Figure 20</a>) show only small differences in the collections of significant genes found with the simplesum and simplesum_avetxl approaches, suggesting that the extent of the problem in many real data sets is limited, and that most findings obtained with simple counting are not induced by counting artifacts. Further support for this conclusion is shown in <a href="#f4">Figure 4</a> (see also <a href="#SM1">Supplementary Figures 2123</a> and <a href="#SM1">Supplementary Table 1</a>), where log-fold change estimates from <i>edgeR</i>, based on the simplesum and scaledTPM matrices, are contrasted. For the genes with induced DTU in the sim2 data set, log-fold changes based on the simplesum matrix are overestimated, as expected. However, this effect is almost absent in all the real data sets, again highlighting the extreme nature of our simulated data and suggesting that the effect of using different count matrices is considerably smaller for many real data sets. <a href="#T1">Table 1</a> further suggests that the lack of error control for simplesum and featureCounts matrices is more pronounced when there is a large difference in length between the differentially used isoforms. In the group with smallest length difference, where the longer differentially used isoform is less than 34% longer than the shorter one, all approaches controlled the type I error satisfactorily. It is worth noting that among all human transcript pairs in which both transcripts belong to the same gene, the median length ratio is 1.85, and for one third of such pairs the longer isoform is less than 38% longer than the shorter one (see <a href="#DS0">Data set 1</a>).</p><div class="fig panel clearfix"><a name=f3 class=n-a></a><a target=_blank href="https://f1000researchdata.s3.amazonaws.com/manuscripts/8770/aca07401-7c11-4875-92b3-89c387175c53_figure3.gif"><img alt="aca07401-7c11-4875-92b3-89c387175c53_figure3.gif" src="https://f1000researchdata.s3.amazonaws.com/manuscripts/8770/aca07401-7c11-4875-92b3-89c387175c53_figure3.gif"></a><div class=caption><h3>Figure 3 (sim2). </h3><p class=first id=d15331e1006><b>A</b>: DGE detection performance of <i>edgeR</i> applied to three different count matrices (simplesum, scaledTPM, featureCounts), with or without including an offset representing the average transcript length (for simplesum and featureCounts, avetxl indicates that such offsets were used). Including the offset or using the scaledTPM count matrix leads to improved FDR control compared to using simplesum or featureCounts matrices without offset. The curves trace out the observed FDR and TPR for each significance cutoff value. The three circles mark the performance at adjusted p-value cutoffs of 0.01, 0.05 and 0.1. <b>B</b>: stratification of the results in <b>A</b> by the presence of differential isoform usage. The improvement in FDR control seen in <b>A</b> results from an improved treatment of genes with differential isoform usage, while all methods perform similarly for genes without differential isoform usage.</p></div></div><div class="table-wrap panel clearfix"><a name=T1 class=n-a></a><div class=caption><h3>Table 1 (sim1). Observed false positive rates from a differential gene expression analysis using <i>edgeR</i> applied to various count matrices (with a nominal p-value cutoff at 0.05), limited to genes with true underlying differential isoform usage (recall that no genes are truly differentially expressed in this data set).</h3><p id=d15331e1041>The results are stratified by effect size (the difference in relative abundance between the two differentially used isoforms) and the length ratio between the longer and the shorter of the differentially used isoforms. FPRs below the nominal p-value threshold (0.05) are marked in bold. For more details, see <a href="#DS1">Data set 1</a>.</p></div><table xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class=article-table frame=below><a name=d15331e1048 class=n-a></a><thead><a name=d15331e1050 class=n-a></a><tr><a name=d15331e1052 class=n-a></a><td colspan=1 rowspan=1><a name=d15331e1054 class=n-a></a></td><th align=center colspan=1 rowspan=1><a name=d15331e1056 class=n-a></a>simplesum</th><th align=center colspan=1 rowspan=1><a name=d15331e1059 class=n-a></a>featureCounts</th><th align=center colspan=1 rowspan=1><a name=d15331e1062 class=n-a></a>simplesum_avetxl</th><th align=center colspan=1 rowspan=1><a name=d15331e1065 class=n-a></a>featureCounts_avetxl</th><th align=center colspan=1 rowspan=1><a name=d15331e1069 class=n-a></a>scaledTPM</th></tr></thead><tbody><a name=d15331e1074 class=n-a></a><tr><a name=d15331e1076 class=n-a></a><td colspan=1 rowspan=1><a name=d15331e1078 class=n-a></a>[0,0.33), [1,1.34)</td><td align=center colspan=1 rowspan=1><a name=d15331e1081 class=n-a></a><b>0.019</b></td><td align=center colspan=1 rowspan=1><a name=d15331e1087 class=n-a></a><b>0.019</b></td><td align=center colspan=1 rowspan=1><a name=d15331e1093 class=n-a></a><b>0.023</b></td><td align=center colspan=1 rowspan=1><a name=d15331e1099 class=n-a></a><b>0.023</b></td><td align=center colspan=1 rowspan=1><a name=d15331e1106 class=n-a></a><b>0.023</b></td></tr><tr><a name=d15331e1113 class=n-a></a><td colspan=1 rowspan=1><a name=d15331e1115 class=n-a></a>[0.33,0.67), [1,1.34)</td><td align=center colspan=1 rowspan=1><a name=d15331e1118 class=n-a></a>0.059</td><td align=center colspan=1 rowspan=1><a name=d15331e1121 class=n-a></a>0.059</td><td align=center colspan=1 rowspan=1><a name=d15331e1124 class=n-a></a>0.059</td><td align=center colspan=1 rowspan=1><a name=d15331e1127 class=n-a></a>0.059</td><td align=center colspan=1 rowspan=1><a name=d15331e1131 class=n-a></a>0.059</td></tr><tr><a name=d15331e1135 class=n-a></a><td colspan=1 rowspan=1><a name=d15331e1137 class=n-a></a>[0.67,1), [1,1.34)</td><td align=center colspan=1 rowspan=1><a name=d15331e1140 class=n-a></a><b>0.000</b></td><td align=center colspan=1 rowspan=1><a name=d15331e1146 class=n-a></a>0.053</td><td align=center colspan=1 rowspan=1><a name=d15331e1149 class=n-a></a>0.053</td><td align=center colspan=1 rowspan=1><a name=d15331e1152 class=n-a></a>0.053</td><td align=center colspan=1 rowspan=1><a name=d15331e1156 class=n-a></a>0.053</td></tr><tr><a name=d15331e1160 class=n-a></a><td colspan=1 rowspan=1><a name=d15331e1162 class=n-a></a>[0,0.33), [1.34,2.57)</td><td align=center colspan=1 rowspan=1><a name=d15331e1165 class=n-a></a>0.075</td><td align=center colspan=1 rowspan=1><a name=d15331e1168 class=n-a></a>0.070</td><td align=center colspan=1 rowspan=1><a name=d15331e1171 class=n-a></a>0.070</td><td align=center colspan=1 rowspan=1><a name=d15331e1174 class=n-a></a>0.065</td><td align=center colspan=1 rowspan=1><a name=d15331e1178 class=n-a></a>0.065</td></tr><tr><a name=d15331e1182 class=n-a></a><td colspan=1 rowspan=1><a name=d15331e1184 class=n-a></a>[0.33,0.67), [1.34,2.57)</td><td align=center colspan=1 rowspan=1><a name=d15331e1187 class=n-a></a>0.240</td><td align=center colspan=1 rowspan=1><a name=d15331e1190 class=n-a></a>0.220</td><td align=center colspan=1 rowspan=1><a name=d15331e1193 class=n-a></a><b>0.050</b></td><td align=center colspan=1 rowspan=1><a name=d15331e1199 class=n-a></a><b>0.033</b></td><td align=center colspan=1 rowspan=1><a name=d15331e1206 class=n-a></a>0.066</td></tr><tr><a name=d15331e1211 class=n-a></a><td colspan=1 rowspan=1><a name=d15331e1213 class=n-a></a>[0.67,1), [1.34,2.57)</td><td align=center colspan=1 rowspan=1><a name=d15331e1216 class=n-a></a>0.420</td><td align=center colspan=1 rowspan=1><a name=d15331e1219 class=n-a></a>0.540</td><td align=center colspan=1 rowspan=1><a name=d15331e1222 class=n-a></a><b>0.038</b></td><td align=center colspan=1 rowspan=1><a name=d15331e1228 class=n-a></a>0.077</td><td align=center colspan=1 rowspan=1><a name=d15331e1232 class=n-a></a><b>0.038</b></td></tr><tr><a name=d15331e1239 class=n-a></a><td colspan=1 rowspan=1><a name=d15331e1241 class=n-a></a>[0,0.33), [2.57,35.4]</td><td align=center colspan=1 rowspan=1><a name=d15331e1244 class=n-a></a>0.150</td><td align=center colspan=1 rowspan=1><a name=d15331e1247 class=n-a></a>0.140</td><td align=center colspan=1 rowspan=1><a name=d15331e1250 class=n-a></a><b>0.037</b></td><td align=center colspan=1 rowspan=1><a name=d15331e1256 class=n-a></a><b>0.043</b></td><td align=center colspan=1 rowspan=1><a name=d15331e1263 class=n-a></a><b>0.037</b></td></tr><tr><a name=d15331e1270 class=n-a></a><td colspan=1 rowspan=1><a name=d15331e1272 class=n-a></a>[0.33,0.67), [2.57,35.4]</td><td align=center colspan=1 rowspan=1><a name=d15331e1275 class=n-a></a>0.650</td><td align=center colspan=1 rowspan=1><a name=d15331e1278 class=n-a></a>0.650</td><td align=center colspan=1 rowspan=1><a name=d15331e1281 class=n-a></a>0.060</td><td align=center colspan=1 rowspan=1><a name=d15331e1284 class=n-a></a>0.060</td><td align=center colspan=1 rowspan=1><a name=d15331e1288 class=n-a></a><b>0.034</b></td></tr><tr><a name=d15331e1295 class=n-a></a><td colspan=1 rowspan=1><a name=d15331e1297 class=n-a></a>[0.67,1), [2.57,35.4]</td><td align=center colspan=1 rowspan=1><a name=d15331e1300 class=n-a></a>0.970</td><td align=center colspan=1 rowspan=1><a name=d15331e1303 class=n-a></a>0.970</td><td align=center colspan=1 rowspan=1><a name=d15331e1306 class=n-a></a><b>0.034</b></td><td align=center colspan=1 rowspan=1><a name=d15331e1312 class=n-a></a><b>0.034</b></td><td align=center colspan=1 rowspan=1><a name=d15331e1319 class=n-a></a><b>0.034</b></td></tr></tbody></table></div><div class="fig panel clearfix"><a name=f4 class=n-a></a><a target=_blank href="https://f1000researchdata.s3.amazonaws.com/manuscripts/8770/aca07401-7c11-4875-92b3-89c387175c53_figure4.gif"><img alt="aca07401-7c11-4875-92b3-89c387175c53_figure4.gif" src="https://f1000researchdata.s3.amazonaws.com/manuscripts/8770/aca07401-7c11-4875-92b3-89c387175c53_figure4.gif"></a><div class=caption><h3>Figure 4. Comparison of log-fold change estimates from <i>edgeR</i>, based on <b>simplesum</b> and <b>scaledTPM</b> count matrices, in four different data sets.</h3><p id=d15331e1348>For the simulated data set (<b>sim2</b>), where signals have been exaggerated to pinpoint underlying causes of various observations, genes with induced DTU (whose true overall log-fold change is 0) show a clear overestimation of log-fold changes when using <b>simplesum</b> counts. However, none of the real data sets contain a similar population of genes, suggesting that for many real data sets, simple gene counting leads to overall similar conclusions as accounting for underlying changes in transcript usage.</p></div></div></div><div id=article1-body class=generated-article-body><h2 class=main-title id=d15331e1365>Discussion</h2><p class="" id=d15331e1368>In this article, we have contrasted transcript- and gene-resolution analyses in terms of both abundance estimation and statistical inference, and illustrated that gene-level results are often more accurate, powerful and interpretable than transcript-level results. Not surprisingly, however, accurate transcript-level estimation and inference play an important role in deriving appropriate gene-level results, and it is therefore imperative to continue improving abundance estimation and inference methods applicable to individual transcripts, since misestimation can propagate to the gene level. We have shown that when testing for changes in overall gene expression (DGE), traditional gene counting approaches may lead to an inflated false discovery rate compared to methods aggregating transcript-level TPM values or incorporating correction factors derived from these, for genes where the relative isoform usage differs between the compared conditions. These correction factors can be calculated from the output of transcript abundance programs, using e.g., the provided R package (<i>tximport</i>). It is important to note that the average transcript length offsets must account for the differences in transcript usage between the samples and thus using (sample-independent) exon-union gene lengths will not improve performance.</p><p class="" id=d15331e1374>On the six data sets studied here, simple counting with <i>featureCounts</i> led to very similar conclusions as estimated gene counts from <i>Salmon</i>, when combined with count-based statistical inference tools such as <i>edgeR</i> and <i>DESeq2</i>. Moreover, p-value distributions and mean-variance relationships were similar for actual and estimated counts. Taken together, this suggests that the negative binomial assumption made by the count-based tools is flexible enough to accommodate also estimated counts. All evaluated counting approaches, with and without the inclusion of average transcript length offsets, gave comparable DGE results for genes where DTU was not present. Thus, the extent of the FDR inflation in experimental data depends on the extent of DTU between the compared conditions; notably, our simulation introduced rather extreme levels of DTU, hence the inflated FDR, and the difference between the approaches was considerably smaller in real data sets. Recent studies have also shown that many genes express mainly one, dominant isoform<sup><a href="#ref-26">26</a></sup> and for such genes, we expect that simple gene counting will work well.</p><p class="" id=d15331e1393>All evaluations in this study were performed using well-established count-based differential analysis tools. These methods take as input a matrix of counts, which is assumed to correctly represent the origin of each read in a particular set of libraries. However, due to sequence similarities among transcripts or genes, there is often a hidden uncertainty in the feature abundance estimates, even when the set of input reads is fixed. With the development of fast, alignment-free abundance estimation methods, this uncertainty can now be estimated rapidly using bootstrap approaches (see e.g. <a href="#f1">Figure 1b</a>). Method development is currently underway in the field to account for this uncertainty in the differential expression analysis (e.g., MetaDiff<sup><a href="#ref-27">27</a></sup>, sleuth<sup><a href="#ref-6">6</a></sup>), which has the potential to improve performance of both DTE and DGE analyses. If such methods are based on (potentially transformed) aggregated transcript counts as gene-level abundance measures, DGE analysis will still be affected by the presence of DTU, and thus could benefit from the inclusion of average transcript length offsets, or by instead using the sum of transcript TPMs as gene abundance measures.</p><p class="" id=d15331e1407>Our results highlight the importance of carefully specifying the question of interest before selecting a statistical approach. Summarization of abundance estimates at the gene level before performing the statistical testing should be the method of choice if the interest is in finding changes in the overall transcriptional output of a gene. However, it is suboptimal if the goal is to identify genes for which <i>at least one</i> of the transcripts show differences in transcriptional output, since it may miss genes where two transcripts change in opposite directions, or where a lowly expressed transcript changes. For gene-level detection of DTE (that is, whether any transcript showed a change in expression between the conditions), statistical testing applied to aggregated gene counts led to reduced power and slightly inflated FDR compared to performing the statistical test on the transcript level and aggregating results within genes (<a href="#SM1">Supplementary Figure 9</a>). Statistical inference on aggregated transcript TPMs (scaledTPM) showed low power for detecting changes that did not affect the overall transcriptional output of the gene, as expected. An alternative to DTE analysis, for potential improved interpretability, is to perform a combination of DGE and DTU analyses, both resulting in gene-level inferences. <a href="#T2">Table 2</a> summarizes our results and give suggested workflows for the different types of analyses we have considered.</p><div class="table-wrap panel clearfix"><a name=T2 class=n-a></a><div class=caption><h3>Table 2. Summary of suitable analysis approaches for the three types of comparative analyses discussed in the manuscript (DGE, DTE and DTU).</h3></div><table xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class=article-table frame=below><a name=d15331e1431 class=n-a></a><thead><a name=d15331e1433 class=n-a></a><tr><a name=d15331e1435 class=n-a></a><th align=left colspan=1 rowspan=1><a name=d15331e1437 class=n-a></a>Task</th><th align=left colspan=1 rowspan=1><a name=d15331e1440 class=n-a></a>Input data</th><th align=left colspan=1 rowspan=1><a name=d15331e1443 class=n-a></a>Software (examples)</th><th align=left colspan=1 rowspan=1><a name=d15331e1446 class=n-a></a>Post-processing</th></tr></thead><tbody><a name=d15331e1451 class=n-a></a><tr><a name=d15331e1453 class=n-a></a><td colspan=1 rowspan=3 valign=top><a name=d15331e1455 class=n-a></a>DGE</td><td colspan=1 rowspan=3 valign=top><a name=d15331e1458 class=n-a></a>Aggregated transcript counts +<br class=br>average transcript length offsets, or<br class=br>simple counts + average transcript<br class=br>length offsets</td><td colspan=1 rowspan=1><a name=d15331e1467 class=n-a></a>Salmon, kallisto, BitSeq, RSEM</td><td colspan=1 rowspan=3 valign=top><a name=d15331e1470 class=n-a></a></td></tr><tr><a name=d15331e1473 class=n-a></a><td colspan=1 rowspan=1><a name=d15331e1475 class=n-a></a>tximport</td></tr><tr><a name=d15331e1479 class=n-a></a><td colspan=1 rowspan=1><a name=d15331e1481 class=n-a></a>DESeq2, edgeR, voom/limma</td></tr><tr><a name=d15331e1485 class=n-a></a><td colspan=1 rowspan=3 valign=top><a name=d15331e1487 class=n-a></a>DTE</td><td colspan=1 rowspan=3 valign=top><a name=d15331e1490 class=n-a></a>Transcript counts</td><td colspan=1 rowspan=1><a name=d15331e1493 class=n-a></a>Salmon, kallisto, BitSeq, RSEM</td><td colspan=1 rowspan=3 valign=top><a name=d15331e1496 class=n-a></a>Optional gene-level<br class=br>aggregation</td></tr><tr><a name=d15331e1502 class=n-a></a><td colspan=1 rowspan=1><a name=d15331e1504 class=n-a></a>tximport</td></tr><tr><a name=d15331e1509 class=n-a></a><td colspan=1 rowspan=1><a name=d15331e1511 class=n-a></a>DESeq2, edgeR, sleuth, voom/limma</td></tr><tr><a name=d15331e1515 class=n-a></a><td colspan=1 rowspan=2 valign=top><a name=d15331e1517 class=n-a></a>DTU/DEU</td><td colspan=1 rowspan=2 valign=top><a name=d15331e1520 class=n-a></a>Transcript counts or bin counts,<br class=br>depending on interpretation potential<sup><a href="#ref-18">18</a></sup></td><td colspan=1 rowspan=1><a name=d15331e1529 class=n-a></a>Salmon, kallisto, BitSeq, RSEM</td><td colspan=1 rowspan=2 valign=top><a name=d15331e1532 class=n-a></a>Optional gene-level<br class=br>aggregation</td></tr><tr><a name=d15331e1538 class=n-a></a><td colspan=1 rowspan=1><a name=d15331e1540 class=n-a></a>DEXSeq</td></tr></tbody></table></div><p class="" id=d15331e1547>Finally, we note that abundance estimation at the gene level can reduce the impact of technical biases on expression levels, which have been shown to lead to estimation errors, such as expression being attributed to the wrong isoform<sup><a href="#ref-28">28</a></sup>. Non-uniform coverage from amplification bias or from position bias (3 coverage bias from poly-(A) selection) can result in unidentifiable transcript-level estimation. While correction of technical artifacts in coverage can be attempted computationally, through estimation of sequence- and position-specific biases<sup><a href="#ref-20">20</a></sup>, we note that such errors and estimation problems are also minimized when summarizing expression to the gene level. This being said, there may of course be situations where a direct transcript-level analysis is appropriate. For example, in a cancer setting where a specific deleterious splice variant is of interest (e.g., AR-V7 in prostate cancer<sup><a href="#ref-29">29</a></sup>), inferences directly at the transcript level may be preferred. However, while this may be preferred for individual known transcripts, transcriptome-wide differential expression analyses may not be warranted, given the associated multiple testing cost.</p></div><div id=article1-body class=generated-article-body><h2 class=main-title id=d15331e1565>Data availability</h2><p class="" id=d15331e1568><i>F1000Research</i>: Dataset 1. Data set 1, <a target=xrefwindow href="http://dx.doi.org/10.5256/f1000research.7563.d114722" id=d15331e1572>10.5256/f1000research.7563.d114722</a></p><p class="" id=d15331e1576><i>F1000Research</i>: Dataset 2. Data set 2, <a target=xrefwindow href="http://dx.doi.org/10.5256/f1000research.7563.d114723" id=d15331e1580>10.5256/f1000research.7563.d114723</a></p><p class="" id=d15331e1584><i>F1000Research</i>: Dataset 3. Data set 3, <a target=xrefwindow href="http://dx.doi.org/10.5256/f1000research.7563.d114724" id=d15331e1588>10.5256/f1000research.7563.d114724</a></p><p class="" id=d15331e1592><i>F1000Research</i>: Dataset 4. Data set 4, <a target=xrefwindow href="http://dx.doi.org/10.5256/f1000research.7563.d114725" id=d15331e1596>10.5256/f1000research.7563.d114725</a></p><p class="" id=d15331e1601><i>F1000Research</i>: Dataset 5. Data set 5, <a target=xrefwindow href="http://dx.doi.org/10.5256/f1000research.7563.d114726" id=d15331e1605>10.5256/f1000research.7563.d114726</a></p><p class="" id=d15331e1609><i>F1000Research</i>: Dataset 6. Data set 6, <a target=xrefwindow href="http://dx.doi.org/10.5256/f1000research.7563.d114730" id=d15331e1613>10.5256/f1000research.7563.d114730</a></p></div><div id=article1-body class=generated-article-body><h2 class=main-title id=d15331e1620>Software availability</h2><div class=section><a name=d15331e1623 class=n-a></a><h3 class=section-title>Software access</h3><p class="" id=d15331e1628><a target=xrefwindow href="http://bioconductor.org/packages/tximport" id=d15331e1630>http://bioconductor.org/packages/tximport</a></p></div><div class=section><a name=d15331e1635 class=n-a></a><h3 class=section-title>Source code as at the time of publication</h3><p class="" id=d15331e1640><a target=xrefwindow href="https://github.com/F1000Research/tximport" id=d15331e1642>https://github.com/F1000Research/tximport</a></p></div><div class=section><a name=d15331e1647 class=n-a></a><h3 class=section-title>Archived source code as at the time of publication</h3><p class="" id=d15331e1652><a target=xrefwindow href="http://dx.doi.org/10.5281/Zenodo.35123" id=d15331e1654>http://dx.doi.org/10.5281/Zenodo.35123</a></p></div><div class=section><a name=d15331e1659 class=n-a></a><h3 class=section-title>Software license</h3><p class="" id=d15331e1664><i>tximport</i> is released under a GNU Public License (GPL).</p></div></div><div id=article1-back class=generated-article-footer><div class=back-section><a name=d15331e1 class=n-a></a><h2 class=main-title id=d15511>Author contributions</h2><p class=metadata-entry><a name=d15331e157 class=n-a></a><p id=d15331e159> CS, MIL and MDR conceived the study and developed methodology. CS and MDR designed and carried out the computational experiments and drafted the manuscript. MIL implemented the <i>tximport</i> R package and wrote parts of the manuscript. All authors read and approved the final manuscript and have agreed to the content.</p></p></div><div class=back-section><a name=d15331e1 class=n-a></a><h2 class=main-title id=d15515>Competing interests</h2><p class=metadata-entry><a name=d15331e166 class=n-a></a><p id=d15331e168> No competing interests were disclosed.</p></p></div><div class=back-section><a name=d15331e1 class=n-a></a><h2 class=main-title id=d15517>Grant information</h2><p>MDR and CS acknowledge support from the RNA &amp; Disease National Center of Competence in Research, an SNSF project grant (143883) and from the European Commission through the 7th Framework Collaborative Project RADIANT (Grant Agreement Number: 305626). MIL was supported by NIH grant 5T32CA009337-35.</p><p><i>The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</i></p></div><div class=back-section><a name=d15331e1675 class=n-a></a><h2 class=main-title id=d15331e1677>Acknowledgments</h2><p class="" id=d15331e1680>The authors would like to thank Magnus Rattray, Alexander Kanitz, Hubert Rehrauer, Xiaobei Zhou and Rafael Irizarry for helpful comments on earlier versions of this manuscript. The authors would also like to thank the reviewers, Rob Patro and Stephen Floor, for their excellent suggestions that have helped to improve this manuscript.</p></div><div class=back-section><a name=SM1 class=n-a></a><h2 class=main-title id=d15331e1686>Supplementary material</h2><p class="" id=d15331e1689><i>Supplementary file 1.</i></p><p class="" id=d15331e1695>Supplementary File 1 (pdf) contains more detailed information about the data sets, supplementary methods and supplementary figures referred to in the text.</p><p class="" id=d15331e1698><a target=xrefwindow href="https://f1000researchdata.s3.amazonaws.com/supplementary/7563/9487b780-1cec-4a38-bd8a-e4c4ed5e7c5a.pdf" id=d15331e1700>Click here to access the data</a>.</p></div><div class=back-section><a name=d15331e1705 class=n-a></a><span class="research-layout prime-recommended-wrapper reference-heading is-hidden"><span class="f1r-icon icon-79_faculty_recommended_badge red vmiddle default-cursor"></span><span class="prime-red big">F1000 recommended</span></span><h2 class=main-title id=d15643>References</h2><div class="section ref-list"><a name=d15331e1705 class=n-a></a><ul><li><a name=ref-1 class=n-a></a><span class=label>1. </span>&nbsp;<span class=citation><a name=d15331e1712 class=n-a></a>Liao Y, Smyth GK, Shi W: featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. <i>Bioinformatics.</i> 2014; <b>30</b>(7): 92330. <a target=xrefwindow id=d15331e1720 href="http://www.ncbi.nlm.nih.gov/pubmed/24227677">PubMed Abstract </a> | <a target=xrefwindow id=d15331e1723 href="http://dx.doi.org/10.1093/bioinformatics/btt656">Publisher Full Text </a></span></li><li><a name=ref-2 class=n-a></a><span class=label>2. </span>&nbsp;<span class=citation><a name=d15331e1732 class=n-a></a>Anders S, Pyl PT, Huber W: HTSeq--a Python framework to work with high-throughput sequencing data. <i>Bioinformatics.</i> 2015; <b>31</b>(2): 166169. <a target=xrefwindow id=d15331e1740 href="http://www.ncbi.nlm.nih.gov/pubmed/25260700">PubMed Abstract </a> | <a target=xrefwindow id=d15331e1743 href="http://dx.doi.org/10.1093/bioinformatics/btu638">Publisher Full Text </a> | <a target=xrefwindow id=d15331e1746 href="http://www.ncbi.nlm.nih.gov/pmc/articles/4287950">Free Full Text </a></span></li><li><a name=ref-3 class=n-a></a><span class=label>3. </span>&nbsp;<span class=citation><a name=d15331e1755 class=n-a></a>Trapnell C, Roberts A, Goff L, <i> et al.</i>: Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks. <i>Nat Protoc.</i> 2012; <b>7</b>(3): 56278. <a target=xrefwindow id=d15331e1766 href="http://www.ncbi.nlm.nih.gov/pubmed/22383036">PubMed Abstract </a> | <a target=xrefwindow id=d15331e1769 href="http://dx.doi.org/10.1038/nprot.2012.016">Publisher Full Text </a> | <a target=xrefwindow id=d15331e1773 href="http://www.ncbi.nlm.nih.gov/pmc/articles/3334321">Free Full Text </a></span></li><li><a name=ref-4 class=n-a></a><span class=label>4. </span>&nbsp;<span class=citation><a name=d15331e1782 class=n-a></a>Li B, Dewey CN: RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome. <i>BMC Bioinformatics.</i> 2011; <b>12</b>: 323. <a target=xrefwindow id=d15331e1790 href="http://www.ncbi.nlm.nih.gov/pubmed/21816040">PubMed Abstract </a> | <a target=xrefwindow id=d15331e1793 href="http://dx.doi.org/10.1186/1471-2105-12-323">Publisher Full Text </a> | <a target=xrefwindow id=d15331e1796 href="http://www.ncbi.nlm.nih.gov/pmc/articles/3163565">Free Full Text </a></span></li><li><a name=ref-5 class=n-a></a><span class=label>5. </span>&nbsp;<span class=citation><a name=d15331e1805 class=n-a></a>Glaus P, Honkela A, Rattray M: Identifying differentially expressed transcripts from RNA-seq data with biological variation. <i>Bioinformatics.</i> 2012; <b>28</b>(13): 17211728. <a target=xrefwindow id=d15331e1813 href="http://www.ncbi.nlm.nih.gov/pubmed/22563066">PubMed Abstract </a> | <a target=xrefwindow id=d15331e1816 href="http://dx.doi.org/10.1093/bioinformatics/bts260">Publisher Full Text </a> | <a target=xrefwindow id=d15331e1819 href="http://www.ncbi.nlm.nih.gov/pmc/articles/3381971">Free Full Text </a></span></li><li><a name=ref-6 class=n-a></a><span class=label>6. </span>&nbsp;<span class=citation><a name=d15331e1829 class=n-a></a>Bray N, Pimentel H, Melsted P, <i> et al.</i>: Near-optimal RNA-Seq quantification. <i>arXiv:1505.02710</i>. 2015. <a target=xrefwindow id=d15331e1837 href="http://arxiv.org/ftp/arxiv/papers/1505/1505.02710.pdf">Reference Source</a></span></li><li><a name=ref-7 class=n-a></a><span class=label>7. </span>&nbsp;<span class=citation><a name=d15331e1846 class=n-a></a>Patro R, Duggal G, Kingsford C: Accurate, fast, and model-aware transcript expression quantification with Salmon. <i>bioRxiv.</i> 2015. <a target=xrefwindow id=d15331e1851 href="http://dx.doi.org/10.1101/021592">Publisher Full Text </a></span></li><li><a name=ref-8 class=n-a></a><span class=label>8. </span>&nbsp;<span class=citation><a name=d15331e1860 class=n-a></a>Mortazavi A, Williams BA, McCue K, <i> et al.</i>: Mapping and quantifying mammalian transcriptomes by RNA-Seq. <i>Nat Methods.</i> 2008; <b>5</b>(7): 621628. <a target=xrefwindow id=d15331e1871 href="http://www.ncbi.nlm.nih.gov/pubmed/18516045">PubMed Abstract </a> | <a target=xrefwindow id=d15331e1874 href="http://dx.doi.org/10.1038/nmeth.1226">Publisher Full Text </a></span></li><li><a name=ref-9 class=n-a></a><span class=label>9. </span>&nbsp;<span class=citation><a name=d15331e1883 class=n-a></a>Trapnell C, Williams BA, Pertea G, <i> et al.</i>: Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation. <i>Nat Biotechnol.</i> 2010; <b>28</b>(5): 511515. <a target=xrefwindow id=d15331e1894 href="http://www.ncbi.nlm.nih.gov/pubmed/20436464">PubMed Abstract </a> | <a target=xrefwindow id=d15331e1897 href="http://dx.doi.org/10.1038/nbt.1621">Publisher Full Text </a> | <a target=xrefwindow id=d15331e1901 href="http://www.ncbi.nlm.nih.gov/pmc/articles/3146043">Free Full Text </a></span></li><li><a name=ref-10 class=n-a></a><span class=label>10. </span>&nbsp;<span class=citation><a name=d15331e1910 class=n-a></a>Wagner GP, Kin K, Lynch VJ: Measurement of mRNA abundance using RNA-seq data: RPKM measure is inconsistent among samples. <i>Theory Biosci.</i> 2012; <b>131</b>(4): 281285. <a target=xrefwindow id=d15331e1918 href="http://www.ncbi.nlm.nih.gov/pubmed/22872506">PubMed Abstract </a> | <a target=xrefwindow id=d15331e1921 href="http://dx.doi.org/10.1007/s12064-012-0162-3">Publisher Full Text </a></span></li><li><a name=ref-11 class=n-a></a><span class=label>11. </span>&nbsp;<span class=citation><a name=d15331e1930 class=n-a></a>Love MI, Huber W, Anders S: Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. <i>Genome Biol.</i> 2014; <b>15</b>(12): 550. <a target=xrefwindow id=d15331e1938 href="http://www.ncbi.nlm.nih.gov/pubmed/25516281">PubMed Abstract </a> | <a target=xrefwindow id=d15331e1941 href="http://dx.doi.org/10.1186/s13059-014-0550-8">Publisher Full Text </a> | <a target=xrefwindow id=d15331e1944 href="http://www.ncbi.nlm.nih.gov/pmc/articles/4302049">Free Full Text </a></span></li><li><a name=ref-12 class=n-a></a><span class=label>12. </span>&nbsp;<span class=citation><a name=d15331e1954 class=n-a></a>Robinson MD, McCarthy DJ, Smyth GK: edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. <i>Bioinformatics.</i> 2010; <b>26</b>(1): 13940. <a target=xrefwindow id=d15331e1962 href="http://www.ncbi.nlm.nih.gov/pubmed/19910308">PubMed Abstract </a> | <a target=xrefwindow id=d15331e1965 href="http://dx.doi.org/10.1093/bioinformatics/btp616">Publisher Full Text </a> | <a target=xrefwindow id=d15331e1968 href="http://www.ncbi.nlm.nih.gov/pmc/articles/2796818">Free Full Text </a></span></li><li><a name=ref-13 class=n-a></a><span class=label>13. </span>&nbsp;<span class=citation><a name=d15331e1977 class=n-a></a>Ritchie ME, Phipson B, Wu D, <i> et al.</i>: <i>limma</i> powers differential expression analyses for RNA-sequencing and microarray studies. <i>Nucleic Acids Res.</i> 2015; <b>43</b>(7): e47. <a target=xrefwindow id=d15331e1991 href="http://www.ncbi.nlm.nih.gov/pubmed/25605792">PubMed Abstract </a> | <a target=xrefwindow id=d15331e1995 href="http://dx.doi.org/10.1093/nar/gkv007">Publisher Full Text </a> | <a target=xrefwindow id=d15331e1998 href="http://www.ncbi.nlm.nih.gov/pmc/articles/4402510">Free Full Text </a></span></li><li><a name=ref-14 class=n-a></a><span class=label>14. </span>&nbsp;<span class=citation><a name=d15331e2007 class=n-a></a>Bottomly D, Walter NA, Hunter JE, <i> et al.</i>: Evaluating gene expression in C57BL/6J and DBA/2J mouse striatum using RNA-Seq and microarrays. <i>PLoS One.</i> 2011; <b>6</b>(3): e17820. <a target=xrefwindow id=d15331e2018 href="http://www.ncbi.nlm.nih.gov/pubmed/21455293">PubMed Abstract </a> | <a target=xrefwindow id=d15331e2021 href="http://dx.doi.org/10.1371/journal.pone.0017820">Publisher Full Text </a> | <a target=xrefwindow id=d15331e2025 href="http://www.ncbi.nlm.nih.gov/pmc/articles/3063777">Free Full Text </a></span></li><li><a name=ref-15 class=n-a></a><span class=label>15. </span>&nbsp;<span class=citation><a name=d15331e2034 class=n-a></a>Yang S, Marn-Juez R, Meijer AH, <i> et al.</i>: Common and specific downstream signaling targets controlled by Tlr2 and Tlr5 innate immune signaling in zebrafish. <i>BMC Genomics.</i> 2015; <b>16</b>(1): 547. <a target=xrefwindow id=d15331e2045 href="http://www.ncbi.nlm.nih.gov/pubmed/26208853">PubMed Abstract </a> | <a target=xrefwindow id=d15331e2048 href="http://dx.doi.org/10.1186/s12864-015-1740-9">Publisher Full Text </a> | <a target=xrefwindow id=d15331e2052 href="http://www.ncbi.nlm.nih.gov/pmc/articles/4514945">Free Full Text </a></span></li><li><a name=ref-16 class=n-a></a><span class=label>16. </span>&nbsp;<span class=citation><a name=d15331e2061 class=n-a></a>Currais A, Goldberg J, Farrokhi C, <i> et al.</i>: A comprehensive multiomics approach toward understanding the relationship between aging and dementia. <i>Aging (Albany NY).</i> 2015; <b>7</b>(11): 937955. <a target=xrefwindow id=d15331e2072 href="http://www.ncbi.nlm.nih.gov/pubmed/26564964">PubMed Abstract </a> | <a target=xrefwindow id=d15331e2075 href="http://www.ncbi.nlm.nih.gov/pmc/articles/4694064">Free Full Text </a></span></li><li><a name=ref-17 class=n-a></a><span class=label>17. </span>&nbsp;<span class=citation><a name=d15331e2084 class=n-a></a>Chang AJ, Ortega FE, Riegler J, <i> et al.</i>: Oxygen regulation of breathing through an olfactory receptor activated by lactate. <i>Nature.</i> 2015; <b>527</b>(7577): 240244. <a target=xrefwindow id=d15331e2095 href="http://www.ncbi.nlm.nih.gov/pubmed/26560302">PubMed Abstract </a> | <a target=xrefwindow id=d15331e2098 href="http://dx.doi.org/10.1038/nature15721">Publisher Full Text </a></span></li><li><a name=ref-18 class=n-a></a><span class=label>18. </span>&nbsp;<span class=citation><a name=d15331e2108 class=n-a></a>Soneson C, Matthes KL, Nowicka M, <i> et al.</i>: Differential transcript usage from RNA-seq data: isoform pre-filtering improves performance of count-based methods. <i>bioRxiv.</i> 2015. <a target=xrefwindow id=d15331e2116 href="http://dx.doi.org/10.1101/025387">Publisher Full Text </a></span></li><li><a name=ref-19 class=n-a></a><span class=label>19. </span>&nbsp;<span class=citation><a name=d15331e2125 class=n-a></a>Kanitz A, Gypas F, Gruber AJ, <i> et al.</i>: Comparative assessment of methods for the computational inference of transcript isoform abundance from RNA-seq data. <i>Genome Biol.</i> 2015; <b>16</b>(1): 150. <a target=xrefwindow id=d15331e2136 href="http://www.ncbi.nlm.nih.gov/pubmed/26201343">PubMed Abstract </a> | <a target=xrefwindow id=d15331e2139 href="http://dx.doi.org/10.1186/s13059-015-0702-5">Publisher Full Text </a> | <a target=xrefwindow id=d15331e2143 href="http://www.ncbi.nlm.nih.gov/pmc/articles/4511015">Free Full Text </a></span></li><li><a name=ref-20 class=n-a></a><span class=label>20. </span>&nbsp;<span class=citation><a name=d15331e2152 class=n-a></a>Roberts A, Trapnell C, Donaghey J, <i> et al.</i>: Improving RNA-Seq expression estimates by correcting for fragment bias. <i>Genome Biol.</i> 2011; <b>12</b>(3): R22. <a target=xrefwindow id=d15331e2163 href="http://www.ncbi.nlm.nih.gov/pubmed/21410973">PubMed Abstract </a> | <a target=xrefwindow id=d15331e2166 href="http://dx.doi.org/10.1186/gb-2011-12-3-r22">Publisher Full Text </a> | <a target=xrefwindow id=d15331e2170 href="http://www.ncbi.nlm.nih.gov/pmc/articles/3129672">Free Full Text </a></span></li><li><a name=ref-21 class=n-a></a><span class=label>21. </span>&nbsp;<span class=citation><a name=d15331e2179 class=n-a></a>Robert C, Watson M: Errors in RNA-Seq quantification affect genes of relevance to human disease. <i>Genome Biol.</i> 2015; <b>16</b>(1): 177. <a target=xrefwindow id=d15331e2187 href="http://www.ncbi.nlm.nih.gov/pubmed/26335491">PubMed Abstract </a> | <a target=xrefwindow id=d15331e2190 href="http://dx.doi.org/10.1186/s13059-015-0734-x">Publisher Full Text </a> | <a target=xrefwindow id=d15331e2193 href="http://www.ncbi.nlm.nih.gov/pmc/articles/4558956">Free Full Text </a></span></li><li><a name=ref-22 class=n-a></a><span class=label>22. </span>&nbsp;<span class=citation><a name=d15331e2202 class=n-a></a>Anders S, Reyes A, Huber W: Detecting differential usage of exons from RNA-seq data. <i>Genome Res.</i> 2012; <b>22</b>(10): 200817. <a target=xrefwindow id=d15331e2210 href="http://www.ncbi.nlm.nih.gov/pubmed/22722343">PubMed Abstract </a> | <a target=xrefwindow id=d15331e2213 href="http://dx.doi.org/10.1101/gr.133744.111">Publisher Full Text </a> | <a target=xrefwindow id=d15331e2216 href="http://www.ncbi.nlm.nih.gov/pmc/articles/3460195">Free Full Text </a></span></li><li><a name=ref-23 class=n-a></a><span class=label>23. </span>&nbsp;<span class=citation><a name=d15331e2225 class=n-a></a>Lawrence M, Huber W, Pags H, <i> et al.</i>: Software for computing and annotating genomic ranges. <i>PLoS Comput Biol.</i> 2013; <b>9</b>(8): e1003118. <a target=xrefwindow id=d15331e2236 href="http://www.ncbi.nlm.nih.gov/pubmed/23950696">PubMed Abstract </a> | <a target=xrefwindow id=d15331e2239 href="http://dx.doi.org/10.1371/journal.pcbi.1003118">Publisher Full Text </a> | <a target=xrefwindow id=d15331e2243 href="http://www.ncbi.nlm.nih.gov/pmc/articles/3738458">Free Full Text </a></span></li><li><a name=ref-24 class=n-a></a><span class=label>24. </span>&nbsp;<span class=citation><a name=d15331e2253 class=n-a></a>Trapnell C, Hendrickson DG, Sauvageau M, <i> et al.</i>: Differential analysis of gene regulation at transcript resolution with RNA-seq. <i>Nat Biotechnol.</i> 2013; <b>31</b>(1): 4653. <a target=xrefwindow id=d15331e2264 href="http://www.ncbi.nlm.nih.gov/pubmed/23222703">PubMed Abstract </a> | <a target=xrefwindow id=d15331e2267 href="http://dx.doi.org/10.1038/nbt.2450">Publisher Full Text </a> | <a target=xrefwindow id=d15331e2271 href="http://www.ncbi.nlm.nih.gov/pmc/articles/3869392">Free Full Text </a></span></li><li><a name=ref-25 class=n-a></a><span class=label>25. </span>&nbsp;<span class=citation><a name=d15331e2280 class=n-a></a>Zhao S, Xi L, Zhang B: Union Exon Based Approach for RNA-Seq Gene Quantification: To Be or Not to Be? <i>PLoS One.</i> 2015; <b>10</b>(11): e0141910. <a target=xrefwindow id=d15331e2288 href="http://www.ncbi.nlm.nih.gov/pubmed/26559532">PubMed Abstract </a> | <a target=xrefwindow id=d15331e2291 href="http://dx.doi.org/10.1371/journal.pone.0141910">Publisher Full Text </a> | <a target=xrefwindow id=d15331e2294 href="http://www.ncbi.nlm.nih.gov/pmc/articles/4641603">Free Full Text </a></span></li><li><a name=ref-26 class=n-a></a><span class=label>26. </span>&nbsp;<span class=citation><a name=d15331e2303 class=n-a></a>Gonzlez-Porta M, Frankish A, Rung J, <i> et al.</i>: Transcriptome analysis of human tissues and cell lines reveals one dominant transcript per gene. <i>Genome Biol.</i> 2013; <b>14</b>(7): R70. <a target=xrefwindow id=d15331e2314 href="http://www.ncbi.nlm.nih.gov/pubmed/23815980">PubMed Abstract </a> | <a target=xrefwindow id=d15331e2317 href="http://dx.doi.org/10.1186/gb-2013-14-7-r70">Publisher Full Text </a> | <a target=xrefwindow id=d15331e2321 href="http://www.ncbi.nlm.nih.gov/pmc/articles/4053754">Free Full Text </a></span></li><li><a name=ref-27 class=n-a></a><span class=label>27. </span>&nbsp;<span class=citation><a name=d15331e2330 class=n-a></a>Jia C, Guan W, Yang A, <i> et al.</i>: MetaDiff: differential isoform expression analysis using random-effects meta-regression. <i>BMC Bioinformatics.</i> 2015; <b>16</b>(1): 208. <a target=xrefwindow id=d15331e2341 href="http://www.ncbi.nlm.nih.gov/pubmed/26134005">PubMed Abstract </a> | <a target=xrefwindow id=d15331e2344 href="http://dx.doi.org/10.1186/s12859-015-0623-z">Publisher Full Text </a> | <a target=xrefwindow id=d15331e2348 href="http://www.ncbi.nlm.nih.gov/pmc/articles/4489045">Free Full Text </a></span></li><li><a name=ref-28 class=n-a></a><span class=label>28. </span>&nbsp;<span class=citation><a name=d15331e2357 class=n-a></a>Love MI, Hogenesch JB, Irizarry RA: Modeling of RNA-seq fragment sequence bias reduces systematic errors in transcript abundance estimation. <i>bioRxiv.</i> 2015. <a target=xrefwindow id=d15331e2362 href="http://dx.doi.org/10.1101/025767">Publisher Full Text </a></span></li><li><a name=ref-29 class=n-a></a><span class=label>29. </span>&nbsp;<span class=citation><a name=d15331e2371 class=n-a></a>Antonarakis ES, Lu C, Wang H, <i> et al.</i>: AR-V7 and resistance to enzalutamide and abiraterone in prostate cancer. <i>N Engl J Med.</i> 2014; <b>371</b>(11): 102838. <a target=xrefwindow id=d15331e2382 href="http://www.ncbi.nlm.nih.gov/pubmed/25184630">PubMed Abstract </a> | <a target=xrefwindow id=d15331e2385 href="http://dx.doi.org/10.1056/NEJMoa1315815">Publisher Full Text </a> | <a target=xrefwindow id=d15331e2389 href="http://www.ncbi.nlm.nih.gov/pmc/articles/4201502">Free Full Text </a></span></li><li><a name=ref-30 class=n-a></a><span class=label>30. </span>&nbsp;<span class=citation><a name=d15331e2399 class=n-a></a>Soneson C, Love MI, Robinson MD: Data set 1 in: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences. <i>F1000Research.</i> 2016. <a target=xrefwindow id=d15331e2404 href="http://dx.doi.org/10.5256/f1000research.7563.d114722">Data Source</a></span></li><li><a name=ref-31 class=n-a></a><span class=label>31. </span>&nbsp;<span class=citation><a name=d15331e2413 class=n-a></a>Soneson C, Love MI, Robinson MD: Data set 2 in: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences. <i>F1000Research.</i> 2016. <a target=xrefwindow id=d15331e2418 href="http://dx.doi.org/10.5256/f1000research.7563.d114723">Data Source</a></span></li><li><a name=ref-32 class=n-a></a><span class=label>32. </span>&nbsp;<span class=citation><a name=d15331e2427 class=n-a></a>Soneson C, Love MI, Robinson MD: Data set 3 in: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences. <i>F1000Research.</i> 2016. <a target=xrefwindow id=d15331e2432 href="http://dx.doi.org/10.5256/f1000research.7563.d114724">Data Source</a></span></li><li><a name=ref-33 class=n-a></a><span class=label>33. </span>&nbsp;<span class=citation><a name=d15331e2441 class=n-a></a>Soneson C, Love MI, Robinson MD: Data set 4 in: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences. <i>F1000Research.</i> 2016. <a target=xrefwindow id=d15331e2446 href="http://dx.doi.org/10.5256/f1000research.7563.d114725">Data Source</a></span></li><li><a name=ref-34 class=n-a></a><span class=label>34. </span>&nbsp;<span class=citation><a name=d15331e2455 class=n-a></a>Soneson C, Love MI, Robinson MD: Data set 5 in: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences. <i>F1000Research.</i> 2016. <a target=xrefwindow id=d15331e2460 href="http://dx.doi.org/10.5256/f1000research.7563.d114726">Data Source</a></span></li><li><a name=ref-35 class=n-a></a><span class=label>35. </span>&nbsp;<span class=citation><a name=d15331e2469 class=n-a></a>Soneson C, Love MI, Robinson MD: Data set 6 in: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences. <i>F1000Research.</i> 2016. <a target=xrefwindow id=d15331e2474 href="http://dx.doi.org/10.5256/f1000research.7563.d114730">Data Source</a></span></li></ul></div></div></div></div><div class=f1r-article-desk><section class="o-box o-box--medium u-bg--2 u-mt--2"><h4 class="u-mt--0 u-mb--2 t-h3 u-weight--md">Looking for the Open Peer Review Reports?</h4><p class="u-mt--0 u-mb--0 t-h4">They can now be found at the top of the panel on the right, linked from the box entitled Open Peer Review. Choose the reviewer report you wish to read and click the 'read' link. You can also read all the peer review reports by <a target=_blank href="https://f1000research.com/articles/4-1521/v2/pdf">downloading the PDF</a>.</p></section></div><div id=article-comments class="article-comments padding-bottom-20"><div class=current-article-comment-section><h2 class=main-title name=add-new-comment id=add-new-comment><span class="research-layout f1r-article-mobile-inline valign-middle"><span class="f1r-icon icon-104_comments size30"></span></span><span class=f1r-article-desk-inline>Comments on this article</span><span class=f1r-article-mobile-inline>Comments (4)</span><span class="f1r-article-mobile-inline float-right"><span class="f1r-icon icon-14_more_small"></span><span class="f1r-icon icon-10_less_small"></span></span></h2></div><div class=comments-list-container><div class="f1r-article-desk-inline referee-report-info-box referee-report-version-box"> Version 2 </div><div class="f1r-article-mobile research-layout mobile-version-info padding-top-30"><span class="mversion ">VERSION 2</span><span class=details>PUBLISHED 29 Feb 2016</span><span class="article-pubinfo-mobile versions-section"><div title="#springMessage("NewVersionReasonType.REVISED.tooltip")" class="version-info-box revised ">Revised</div></span></div><ul class=all-comment-wrapper data-reportID=NONE data-versionID=2><li class="comment-wrapper not-sortable " data-commentID=3453><div class="one-comment version-comments "><a name=article-comment-3453></a><div class="f1r-article-desk commenter-role"> Author Response 23 Feb 2018 </div><div><span class=name>Mark Robinson</span><span class=name>, </span><span class=commenter-affiliation>University of Zurich, Zurich, Switzerland</span></div><div class="f1r-article-mobile the-comment-date"> 23 Feb 2018 </div><div class=f1r-article-mobile><div class="article-pubinfo-mobile comments-section"><span class="version-info-box comments"> Author Response </span></div></div><div class="comment-content article-comment-list"><span class=truncated-comment> Written by Charlotte Soneson, Michael I Love and Mark D Robinson.<br/><br/>One of the main points of our paper was to point out and illustrate that defining the null hypothesis of ... <span class="continue-reading-container response-and-comment">Continue reading</span></span><span class="full-comment hidden"><span id=full-comment-original-3453 class=hidden>Written by Charlotte Soneson, Michael I Love and Mark D Robinson.<br/><br/>One of the main points of our paper was to point out and illustrate that defining the null hypothesis of interest is of great importance when it comes to choosing the right data processing and analysis approach (see e.g., the Introduction, or Table 2 for a summary). In particular, we considered three main null hypotheses on the gene level, testing for what we may denote: <ol><li>DGE (differential gene expression)</li><li>DTE+G (differential transcript expression aggregated to the gene level)</li><li>DTU (differential transcript usage).</li></ol> The first null hypothesis (tested in DGE analyses) is that the total &quot;transcriptional output&quot; of a gene* is the same between the compared groups. This is the null hypothesis considered in Figure 3, and indeed for most differential gene expression analyses to date. Now, consider for simplicity a gene with two isoforms of very different length, and assume that only the short isoform is expressed in experimental condition 1, and only the long isoform is expressed in condition 2, but that the total expression of the gene is the same in the two groups. Using the summarized gene counts (in the paper denoted as simplesum) to test the DGE null hypothesis will (falsely) detect a difference between the groups, since the observed gene count depends on the length of the expressed transcripts. Incorporating the average transcript length offsets, or using scaled TPMs instead of simplesum counts, addresses this problem and improves the inference (again, for the DGE null hypothesis) in situations with considerable differential transcript usage. We show this conceptually with (as stated) &quot;extreme&quot; simulated data, but we also illustrate that (as Dr. Pachter comments) for many real data sets, the overall effect is much smaller, and thus, simplesum aggregation indeed provides similar global performance as scaled TPMs. This does not exclude that there may be important effects for individual genes or for other data sets, and therefore we recommend users to follow the steps suggested in the tximport vignette (again, assuming that DGE is the relevant hypothesis to test).<br/><br/>It is important to recall that the concern that DTU among isoforms of different length could induce false positives in DGE analysis was the main issue that <a href="https://www.nature.com/articles/nbt.2450">Trapnell et al (2013)</a> had with competing count-based statistical methods. From the Results section of that paper:<br/><br/><i>When all of a gene's isoforms are up- or downregulated between two conditions, raw count methods recover true change in gene expression. However, when some isoforms are upregulated and others downregulated, raw count methods are inaccurate.</i><br/><br/>In the comment, Dr. Pachter notes that our average transcript length or scaled TPM corrections for performing DGE, which protect against potential false positives induced by DTU, may provide similar global performance as simplesum for a given dataset. This would imply that the major concern of Trapnell et al (2013) with count-based statistical methods may not be a large problem for every dataset. However, this does not invalidate the concern of Trapnell et al (2013), rather it means that the problem primarily affects individual genes with DTU among isoforms of different length, and this may not occur in every dataset, or may not affect the global operating characteristics.<br/><br/>The null hypothesis tested in Supplementary Figure 2 of the Ntranos/Yi et al paper mentioned by Dr. Pachter is very different from the DGE null hypothesis discussed above. In their figure, a gene is considered &quot;truly differential&quot; if any of its transcripts are truly differentially expressed between the compared groups. We consider this null hypothesis for bulk, full-length RNA-seq in our Supplementary Figure 9, and we show that, indeed, performing DGE on the simplesum counts detects more of the truly differential genes (defined as those with any transcript DE rather than where the total concentration of transcripts changes) than performing DGE by including the average transcript length offsets or using scaled TPMs. This is natural and expected, since the offsets are explicitly designed to eliminate the effect of DTU on the overall (aggregated) gene count, to avoid this affecting DGE analysis as described above. Thus, a gene as the one mentioned in the first paragraph will have different gene-level counts due (only) to the usage of different isoforms in the two conditions, not to a change in overall expression, and will be detected in a DGE analysis applied to summarized gene counts, but not if offsets are included. However, the important message of Supplementary Figure 9 is that neither of these gene-level summarization approaches should be applied if the goal is to find genes where any of the transcripts are differentially expressed (the hypothesis tested in Ntranos/Yi et al). Instead, in this situation, transcript-level information should be used. In our Supplementary Figure 9, we exemplify this by performing differential expression analysis on the individual transcript level, followed by aggregation of the test results to the gene level. As expected, this correctly detects many more of the genes with any transcript DE.<br/><br/>Thus, as we discuss in depth in our paper, the answer to the question &quot;How should I sum transcript counts on the gene level with tximport?&quot; depends on the null hypothesis that one is interested in testing. If the null hypothesis of interest is that the total transcriptional output of a gene is the same between conditions, incorporating the average transcript length offsets or using scaled (or length-scaled) TPMs will protect against effects of expressed transcript length on observed gene abundances (these are the steps one will find for DGE analysis in the tximport vignette). If the null hypothesis of interest is that none of the transcripts of a gene are differentially expressed, then the transcript-level abundances should not be aggregated to the gene level. Instead, we recommend using transcript-level information to answer this question (see, for example the methods papers for <a href="https://bioconductor.org/packages/DRIMSeq">DRIMSeq</a> or <a href="https://github.com/statOmics/stageR">stageR</a>).<br/><br/>In the context of transcript-level analysis, it is worth noting that statements about our F1000 paper written in <a href="https://liorpachter.wordpress.com/2018/02/15/gde%C2%B2-dge%C2%B2-dtu%C2%B2-dte%E2%82%81%C2%B2-dte%E2%82%82%C2%B2/">Dr. Pachters blog</a> (and at <a href="http://www.rna-seqblog.com/dont-bother-with-transcript-level-analysis/">RNASeqBlog</a>) are not true. In the 15 Feb 2018 version of Dr. Pachters blog, it is stated Soneson et al. 2016 suggest that while DTE and DTU may be appropriate in certain niche applications, generally its better to choose DGE, and they therefore advise not to bother with transcript-level analysis. However, we highly recommend transcript-level analysis, whether it be a combination of DGE and DTU, or via DTE+G; in particular, text from our Discussion is copied below with emphasis added:<br/><br/><i>Our results highlight the importance of carefully specifying the question of interest before selecting a statistical approach. Summarization of abundance estimates at the gene level before performing the statistical testing should be the method of choice if the interest is in finding changes in the overall transcriptional output of a gene. <b>However, it is suboptimal if the goal is to identify genes for which at least one of the transcripts show differences in transcriptional output, since it may miss genes where two transcripts change in opposite directions, or where a lowly expressed transcript changes</b>. For gene-level detection of DTE (that is, whether any transcript showed a change in expression between the conditions), statistical testing applied to aggregated gene counts led to reduced power and slightly inflated FDR compared to <b>performing the statistical test on the transcript level and aggregating results within genes</b> (Supplementary Figure 9). Statistical inference on aggregated transcript TPMs (scaledTPM) showed low power for detecting changes that did not affect the overall transcriptional output of the gene, as expected. <b>An alternative to DTE analysis, for potential improved interpretability, is to perform a combination of DGE and DTU analyses, both resulting in gene-level inferences. Table 2 summarizes our results and give suggested workflows for the different types of analyses we have considered.</b></i><br/><br/>Our main message overall is that analysts should think about and carefully specify the differential expression question of interest among the many possibilities.<br/><br/>* =concentration of transcript molecules, summed over all isoforms</span><div class=padding-bottom id=full-comment-3453>Written by Charlotte Soneson, Michael I Love and Mark D Robinson.<br/><br/>One of the main points of our paper was to point out and illustrate that defining the null hypothesis of interest is of great importance when it comes to choosing the right data processing and analysis approach (see e.g., the Introduction, or Table 2 for a summary). In particular, we considered three main null hypotheses on the gene level, testing for what we may denote: <ol><li>DGE (differential gene expression)</li><li>DTE+G (differential transcript expression aggregated to the gene level)</li><li>DTU (differential transcript usage).</li></ol> The first null hypothesis (tested in DGE analyses) is that the total &quot;transcriptional output&quot; of a gene* is the same between the compared groups. This is the null hypothesis considered in Figure 3, and indeed for most differential gene expression analyses to date. Now, consider for simplicity a gene with two isoforms of very different length, and assume that only the short isoform is expressed in experimental condition 1, and only the long isoform is expressed in condition 2, but that the total expression of the gene is the same in the two groups. Using the summarized gene counts (in the paper denoted as simplesum) to test the DGE null hypothesis will (falsely) detect a difference between the groups, since the observed gene count depends on the length of the expressed transcripts. Incorporating the average transcript length offsets, or using scaled TPMs instead of simplesum counts, addresses this problem and improves the inference (again, for the DGE null hypothesis) in situations with considerable differential transcript usage. We show this conceptually with (as stated) &quot;extreme&quot; simulated data, but we also illustrate that (as Dr. Pachter comments) for many real data sets, the overall effect is much smaller, and thus, simplesum aggregation indeed provides similar global performance as scaled TPMs. This does not exclude that there may be important effects for individual genes or for other data sets, and therefore we recommend users to follow the steps suggested in the tximport vignette (again, assuming that DGE is the relevant hypothesis to test).<br/><br/>It is important to recall that the concern that DTU among isoforms of different length could induce false positives in DGE analysis was the main issue that <a href="https://www.nature.com/articles/nbt.2450">Trapnell et al (2013)</a> had with competing count-based statistical methods. From the Results section of that paper:<br/><br/><i>When all of a gene's isoforms are up- or downregulated between two conditions, raw count methods recover true change in gene expression. However, when some isoforms are upregulated and others downregulated, raw count methods are inaccurate.</i><br/><br/>In the comment, Dr. Pachter notes that our average transcript length or scaled TPM corrections for performing DGE, which protect against potential false positives induced by DTU, may provide similar global performance as simplesum for a given dataset. This would imply that the major concern of Trapnell et al (2013) with count-based statistical methods may not be a large problem for every dataset. However, this does not invalidate the concern of Trapnell et al (2013), rather it means that the problem primarily affects individual genes with DTU among isoforms of different length, and this may not occur in every dataset, or may not affect the global operating characteristics.<br/><br/>The null hypothesis tested in Supplementary Figure 2 of the Ntranos/Yi et al paper mentioned by Dr. Pachter is very different from the DGE null hypothesis discussed above. In their figure, a gene is considered &quot;truly differential&quot; if any of its transcripts are truly differentially expressed between the compared groups. We consider this null hypothesis for bulk, full-length RNA-seq in our Supplementary Figure 9, and we show that, indeed, performing DGE on the simplesum counts detects more of the truly differential genes (defined as those with any transcript DE rather than where the total concentration of transcripts changes) than performing DGE by including the average transcript length offsets or using scaled TPMs. This is natural and expected, since the offsets are explicitly designed to eliminate the effect of DTU on the overall (aggregated) gene count, to avoid this affecting DGE analysis as described above. Thus, a gene as the one mentioned in the first paragraph will have different gene-level counts due (only) to the usage of different isoforms in the two conditions, not to a change in overall expression, and will be detected in a DGE analysis applied to summarized gene counts, but not if offsets are included. However, the important message of Supplementary Figure 9 is that neither of these gene-level summarization approaches should be applied if the goal is to find genes where any of the transcripts are differentially expressed (the hypothesis tested in Ntranos/Yi et al). Instead, in this situation, transcript-level information should be used. In our Supplementary Figure 9, we exemplify this by performing differential expression analysis on the individual transcript level, followed by aggregation of the test results to the gene level. As expected, this correctly detects many more of the genes with any transcript DE.<br/><br/>Thus, as we discuss in depth in our paper, the answer to the question &quot;How should I sum transcript counts on the gene level with tximport?&quot; depends on the null hypothesis that one is interested in testing. If the null hypothesis of interest is that the total transcriptional output of a gene is the same between conditions, incorporating the average transcript length offsets or using scaled (or length-scaled) TPMs will protect against effects of expressed transcript length on observed gene abundances (these are the steps one will find for DGE analysis in the tximport vignette). If the null hypothesis of interest is that none of the transcripts of a gene are differentially expressed, then the transcript-level abundances should not be aggregated to the gene level. Instead, we recommend using transcript-level information to answer this question (see, for example the methods papers for <a href="https://bioconductor.org/packages/DRIMSeq">DRIMSeq</a> or <a href="https://github.com/statOmics/stageR">stageR</a>).<br/><br/>In the context of transcript-level analysis, it is worth noting that statements about our F1000 paper written in <a href="https://liorpachter.wordpress.com/2018/02/15/gde%C2%B2-dge%C2%B2-dtu%C2%B2-dte%E2%82%81%C2%B2-dte%E2%82%82%C2%B2/">Dr. Pachters blog</a> (and at <a href="http://www.rna-seqblog.com/dont-bother-with-transcript-level-analysis/">RNASeqBlog</a>) are not true. In the 15 Feb 2018 version of Dr. Pachters blog, it is stated Soneson et al. 2016 suggest that while DTE and DTU may be appropriate in certain niche applications, generally its better to choose DGE, and they therefore advise not to bother with transcript-level analysis. However, we highly recommend transcript-level analysis, whether it be a combination of DGE and DTU, or via DTE+G; in particular, text from our Discussion is copied below with emphasis added:<br/><br/><i>Our results highlight the importance of carefully specifying the question of interest before selecting a statistical approach. Summarization of abundance estimates at the gene level before performing the statistical testing should be the method of choice if the interest is in finding changes in the overall transcriptional output of a gene. <b>However, it is suboptimal if the goal is to identify genes for which at least one of the transcripts show differences in transcriptional output, since it may miss genes where two transcripts change in opposite directions, or where a lowly expressed transcript changes</b>. For gene-level detection of DTE (that is, whether any transcript showed a change in expression between the conditions), statistical testing applied to aggregated gene counts led to reduced power and slightly inflated FDR compared to <b>performing the statistical test on the transcript level and aggregating results within genes</b> (Supplementary Figure 9). Statistical inference on aggregated transcript TPMs (scaledTPM) showed low power for detecting changes that did not affect the overall transcriptional output of the gene, as expected. <b>An alternative to DTE analysis, for potential improved interpretability, is to perform a combination of DGE and DTU analyses, both resulting in gene-level inferences. Table 2 summarizes our results and give suggested workflows for the different types of analyses we have considered.</b></i><br/><br/>Our main message overall is that analysts should think about and carefully specify the differential expression question of interest among the many possibilities.<br/><br/>* =concentration of transcript molecules, summed over all isoforms</div><span class="disclosures competing-interests-display padding-top"><strong>Competing Interests:</strong><span class=c-i-d>No competing interests were disclosed.</span></span><span class="hide-comment-container response-and-comment">Close</span></span><div class="report-concern-wrapper last-comment"><a href="mailto:research@f1000.com?subject=Report a concern (7563) " class=report-a-concern-link>Report a concern</a></div></div></div></li><li class="comment-wrapper not-sortable " data-commentID=3436><div class="one-comment version-comments "><a name=article-comment-3436></a><div class="f1r-article-desk commenter-role"> Reviewer Response 20 Feb 2018 </div><div><span class=name>Lior Pachter</span><span class=name>, </span><span class=commenter-affiliation>University of California Berkeley, Berkeley, USA</span></div><div class="f1r-article-mobile the-comment-date"> 20 Feb 2018 </div><div class=f1r-article-mobile><div class="article-pubinfo-mobile comments-section"><span class="version-info-box comments"> Reviewer Response </span></div></div><div class="comment-content article-comment-list"><span class=truncated-comment> Many users have inquired about which of the tximport options they should use when summing transcript-level abundances.&nbsp;The paper provides some mixed messaging in this regard, first stating that &quot;Accounting for ... <span class="continue-reading-container response-and-comment">Continue reading</span></span><span class="full-comment hidden"><span id=full-comment-original-3436 class=hidden>Many users have inquired about which of the tximport options they should use when summing transcript-level abundances.&nbsp;The paper provides some mixed messaging in this regard, first stating that &quot;Accounting for the potentially varying average transcript length across samples when performing DGE, either in the definition of the count matrix (scaledTPM) or by defining offsets (featureCounts_avetxl, simplesum_avetxl), led to considerably improved false discovery rate (FDR) control&quot; and then&nbsp;&quot;Experiments on various real data sets (<a>Supplementary Figure 20</a>) show only small differences in the collections of significant genes found with the simplesum and simplesum_avetxl approaches, suggesting that the extent of the problem in many real data sets is limited.&quot;<br/><br/> To shed further light on this question, we recently examined the performance of tximport in the single-cell RNA-Seq setting. In Supplementary Figure 2 of the paper <a href="https://www.biorxiv.org/content/early/2018/02/14/258566">Ntranos, Yi&nbsp;<i>et al.</i>&nbsp;2018</a>&nbsp;we see almost identical performance when using tximport (+DESeq2) with the simplesum, scaledTPM or lengthscaledTPM options (current options available via countsFromAbundance in tximport). Our result is from a simulation study where the simulation parameters are based on transcript changes seen in a biological dataset.<br/><br/> Our result indicates that simplesum may be adequate in practice (in our experiment even slightly outperforming the other normalizations), which is useful to know because the summation of transcript estimated counts to the gene-level is so simple to perform that it does not require tximport.</span><div class=padding-bottom id=full-comment-3436>Many users have inquired about which of the tximport options they should use when summing transcript-level abundances.&nbsp;The paper provides some mixed messaging in this regard, first stating that &quot;Accounting for the potentially varying average transcript length across samples when performing DGE, either in the definition of the count matrix (scaledTPM) or by defining offsets (featureCounts_avetxl, simplesum_avetxl), led to considerably improved false discovery rate (FDR) control&quot; and then&nbsp;&quot;Experiments on various real data sets (<a>Supplementary Figure 20</a>) show only small differences in the collections of significant genes found with the simplesum and simplesum_avetxl approaches, suggesting that the extent of the problem in many real data sets is limited.&quot;<br/><br/> To shed further light on this question, we recently examined the performance of tximport in the single-cell RNA-Seq setting. In Supplementary Figure 2 of the paper <a href="https://www.biorxiv.org/content/early/2018/02/14/258566">Ntranos, Yi&nbsp;<i>et al.</i>&nbsp;2018</a>&nbsp;we see almost identical performance when using tximport (+DESeq2) with the simplesum, scaledTPM or lengthscaledTPM options (current options available via countsFromAbundance in tximport). Our result is from a simulation study where the simulation parameters are based on transcript changes seen in a biological dataset.<br/><br/> Our result indicates that simplesum may be adequate in practice (in our experiment even slightly outperforming the other normalizations), which is useful to know because the summation of transcript estimated counts to the gene-level is so simple to perform that it does not require tximport.</div><span class="disclosures competing-interests-display padding-top"><strong>Competing Interests:</strong><span class=c-i-d>I have publicly accused the authors of another paper, Patro et al., Nature Methods 2017, of plagiarism. Mike Love (co-author of this paper) is a coauthor of the Patro et al. paper.</span></span><span class="hide-comment-container response-and-comment">Close</span></span><div class="report-concern-wrapper last-comment"><a href="mailto:research@f1000.com?subject=Report a concern (7563) " class=report-a-concern-link>Report a concern</a></div></div></div></li><li class="comment-wrapper not-sortable button-wrapper"><div class="one-comment version-comments add-the-comment"><a href="/login?originalPath=/articles/4-1521/v2&scrollTo=add-new-comments" class=register-report-comment-button data-test-id=add-comment><span class=f1r-article-desk><span class=contracted></span>Comment</span><span class="f1r-article-mobile research-layout "><button class="primary orange extra-padding comment-on-this-article">ADD YOUR COMMENT</button></span></a></div></li></ul><div class="f1r-article-mobile research-layout"><div class="mobile-sections-divider versions-divider"></div></div><div class="f1r-article-desk-inline referee-report-info-box referee-report-version-box"> Version 1 </div><div class="f1r-article-mobile research-layout mobile-version-info padding-top-30"><span class="mversion previous-version">VERSION 1</span><span class=details>PUBLISHED 30 Dec 2015</span><span class="article-pubinfo-mobile versions-section"></span></div><div class="comments-closed f1r-article-mobile"> Discussion is closed on this version, please comment on the latest version above. </div><ul class=all-comment-wrapper data-reportID=NONE data-versionID=1><li class="comment-wrapper not-sortable " data-commentID=1792><div class="one-comment version-comments "><a name=article-comment-1792></a><div class="f1r-article-desk commenter-role"> Author Response 02 Feb 2016 </div><div><span class=name>Charlotte Soneson</span><span class=name>, </span><span class=commenter-affiliation>University of Zurich, Switzerland</span></div><div class="f1r-article-mobile the-comment-date"> 02 Feb 2016 </div><div class=f1r-article-mobile><div class="article-pubinfo-mobile comments-section"><span class="version-info-box comments"> Author Response </span></div></div><div class="comment-content article-comment-list"><span class=truncated-comment> Hi Nick,<br/><br/>thanks for your comments.&nbsp;<br/><br/>Regarding the increased accuracy and robustness of gene-level estimates compared to transcript-level ones (even with the full annotation, bootstrap variances of gene estimates are lower than ... <span class="continue-reading-container response-and-comment">Continue reading</span></span><span class="full-comment hidden"><span id=full-comment-original-1792 class=hidden>Hi Nick,<br/><br/>thanks for your comments.&nbsp;<br/><br/>Regarding the increased accuracy and robustness of gene-level estimates compared to transcript-level ones (even with the full annotation, bootstrap variances of gene estimates are lower than those of transcript estimates), we agree that it is not surprising. However, we still found it worth pointing out since it implies that in some situations, gene-level statistical analyses may be preferred due to the increased precision of the input data.<br/><br/>For your second point, note that Figure 2 does not in fact compare DGE to DTE, but rather DTE summarized on gene- and transcript-level. You are of course right that just reducing the number of tests does not automatically result in higher power. Instead, the difference between the gene- and transcript-level results in Figure 2 is largely due to the different null hypotheses (as outlined in the text) and, consequently, the type of signal we require to call a feature (gene or transcript) significant. In fact, the gene-level summarized analysis is answering a somewhat &quot;easier&quot; question, from the sensitivity point of view. Consider for example a gene with 5 differentially expressed transcripts. On the gene level, we can reach a power of 100% for this gene if <b>any </b>of the transcripts is considered significantly DE. On the transcript level, we would need <b>all</b> of the transcripts to be significant to reach the same power. Looking at this particular data set, for many of the genes that are found as true positives with the gene level test, not all truly DE transcripts are actually detected. This explains the lower power of the transcript-level analysis. We will clarify this in the revised version. Regarding the choice of DTE vs DGE+DTU, it is clear that one solution will not always be the optimal choice, and it will likely depend on the particular problem as well as on the person interpreting the results. However, we have found that in many of our own collaborations, the biological question can be more clearly stated in terms of DGE+DTU (not necessarily performed sequentially, rather in parallel). Part of the reason for this discussion was to encourage researchers to think about what question they really want to answer before starting their analyses.&nbsp;<br/><br/>Finally, note that the only place where we are actually comparing &quot;summarizing DTE at the gene level&quot; and DGE is in Supplementary Figure 5 (on simulated data). In Figures 3-4, the question of interest is always comparing the total transcriptional output of a gene between conditions (i.e., DGE), and all methods are based on aggregating gene abundance estimates before the statistical test is applied. As you note, we don't see a big global effect of including offsets accounting for average transcript lengths for real data. However, there may of course still be important effects for individual genes, where isoforms of different lengths are expressed in different conditions.&nbsp;<br/><br/>Thanks again for your comments, and we are glad you found the paper useful!</span><div class=padding-bottom id=full-comment-1792>Hi Nick,<br/><br/>thanks for your comments.&nbsp;<br/><br/>Regarding the increased accuracy and robustness of gene-level estimates compared to transcript-level ones (even with the full annotation, bootstrap variances of gene estimates are lower than those of transcript estimates), we agree that it is not surprising. However, we still found it worth pointing out since it implies that in some situations, gene-level statistical analyses may be preferred due to the increased precision of the input data.<br/><br/>For your second point, note that Figure 2 does not in fact compare DGE to DTE, but rather DTE summarized on gene- and transcript-level. You are of course right that just reducing the number of tests does not automatically result in higher power. Instead, the difference between the gene- and transcript-level results in Figure 2 is largely due to the different null hypotheses (as outlined in the text) and, consequently, the type of signal we require to call a feature (gene or transcript) significant. In fact, the gene-level summarized analysis is answering a somewhat &quot;easier&quot; question, from the sensitivity point of view. Consider for example a gene with 5 differentially expressed transcripts. On the gene level, we can reach a power of 100% for this gene if <b>any </b>of the transcripts is considered significantly DE. On the transcript level, we would need <b>all</b> of the transcripts to be significant to reach the same power. Looking at this particular data set, for many of the genes that are found as true positives with the gene level test, not all truly DE transcripts are actually detected. This explains the lower power of the transcript-level analysis. We will clarify this in the revised version. Regarding the choice of DTE vs DGE+DTU, it is clear that one solution will not always be the optimal choice, and it will likely depend on the particular problem as well as on the person interpreting the results. However, we have found that in many of our own collaborations, the biological question can be more clearly stated in terms of DGE+DTU (not necessarily performed sequentially, rather in parallel). Part of the reason for this discussion was to encourage researchers to think about what question they really want to answer before starting their analyses.&nbsp;<br/><br/>Finally, note that the only place where we are actually comparing &quot;summarizing DTE at the gene level&quot; and DGE is in Supplementary Figure 5 (on simulated data). In Figures 3-4, the question of interest is always comparing the total transcriptional output of a gene between conditions (i.e., DGE), and all methods are based on aggregating gene abundance estimates before the statistical test is applied. As you note, we don't see a big global effect of including offsets accounting for average transcript lengths for real data. However, there may of course still be important effects for individual genes, where isoforms of different lengths are expressed in different conditions.&nbsp;<br/><br/>Thanks again for your comments, and we are glad you found the paper useful!</div><span class="disclosures competing-interests-display padding-top"><strong>Competing Interests:</strong><span class=c-i-d>No competing interests were disclosed.</span></span><span class="hide-comment-container response-and-comment">Close</span></span><div class="report-concern-wrapper last-comment"><a href="mailto:research@f1000.com?subject=Report a concern (7563) " class=report-a-concern-link>Report a concern</a></div></div></div></li><li class="comment-wrapper not-sortable " data-commentID=1787><div class="one-comment version-comments "><a name=article-comment-1787></a><div class="f1r-article-desk commenter-role"> Reader Comment (<span class=italic>Member of the F1000 Faculty</span>) 29 Jan 2016 </div><div><span class=name>Nick Schurch</span><span class=name>, </span><span class=commenter-affiliation>The Barton Group, Division of Computational Biology College of Life Sciences, University of Dundee, UK</span></div><div class="f1r-article-mobile the-comment-date"> 29 Jan 2016 </div><div class=f1r-article-mobile><div class="article-pubinfo-mobile comments-section"><span class="version-info-box comments"> Reader Comment </span><span class="version-info-box comments fm">Member of the F1000 Faculty</span></div></div><div class="comment-content article-comment-list"><span class=truncated-comment> I read this work with great interest for a literature review and found it engaging and informative. I thought I'd post my resulting thoughts here.<br/><br/>This paper focusses on the advantages ... <span class="continue-reading-container response-and-comment">Continue reading</span></span><span class="full-comment hidden"><span id=full-comment-original-1787 class=hidden>I read this work with great interest for a literature review and found it engaging and informative. I thought I'd post my resulting thoughts here.<br/><br/>This paper focusses on the advantages and disadvantages of performing differential expression analysis at either the gene level or the transcript level, relying heavily on simulated data. Initially the authors use simulated data to show that gene level estimations of expression are more stable then transcript level estimations. They do this by showing that with the removal of ~20% of the transcripts from the annotation, the quantification the transcript level estimations becomes less accurate while the gene level estimation remains relatively unchanged. I found this conclusion to be unsurprising; the transcripts are the primary annotation used for transcript level quantification but are not the primary annotation for the gene level estimation. Except in rare cases where the removal of the transcripts result in the removal of the parent gene or removal of the longest transcripts results in a considerable change in the length of the parent gene, the gene annotation will remain largely unaltered by the removal of transcripts.<br/><br/>The authors then consider the performance of differential expression at the gene level (DGE) vs differential expression at the transcript level (DTE), concluding that inference at the gene level is more robust, easier to interpret and has better statistical performance. While the argument for improved robustness is compelling, I found the arguments for improved inference and statistical performance a little misleading. The improved statistical performance of DGE is due essentially to performing fewer tests. By this argument, anything that reduced the number of tests always improves the inference (e.g., using a smaller annotation) which is clearly not always the case. The authors also suggest that rather than identifying DTE, a better approach with information summarized at the gene level may be to first identify which genes are changing and then identify differential isoform usage. There is no clear demonstration of the advantages of this approach or what the appropriate multiple testing correction is for the combined analysis. In particular, l find the interpretability of DTE far more straightforward than the combined approach.<br/><br/>The paper also examined the differences between the differential gene expression conclusions resulting from summarizing DTE at the gene level and DGE called on simple gene counts on both simulated and real data and find that, again somewhat unsurprisingly in my opinion, that the difference in the results is smaller and less obvious for real datasets than for the simulated data where they have added a strong signal. I found the evidence for using an adjusted transcript length is compelling despite the relatively small apparent impact it has for real datasets.<br/><br/>All in all I liked the paper and it was interesting to talk about it and discuss the issues it raises with my group.</span><div class=padding-bottom id=full-comment-1787>I read this work with great interest for a literature review and found it engaging and informative. I thought I'd post my resulting thoughts here.<br/><br/>This paper focusses on the advantages and disadvantages of performing differential expression analysis at either the gene level or the transcript level, relying heavily on simulated data. Initially the authors use simulated data to show that gene level estimations of expression are more stable then transcript level estimations. They do this by showing that with the removal of ~20% of the transcripts from the annotation, the quantification the transcript level estimations becomes less accurate while the gene level estimation remains relatively unchanged. I found this conclusion to be unsurprising; the transcripts are the primary annotation used for transcript level quantification but are not the primary annotation for the gene level estimation. Except in rare cases where the removal of the transcripts result in the removal of the parent gene or removal of the longest transcripts results in a considerable change in the length of the parent gene, the gene annotation will remain largely unaltered by the removal of transcripts.<br/><br/>The authors then consider the performance of differential expression at the gene level (DGE) vs differential expression at the transcript level (DTE), concluding that inference at the gene level is more robust, easier to interpret and has better statistical performance. While the argument for improved robustness is compelling, I found the arguments for improved inference and statistical performance a little misleading. The improved statistical performance of DGE is due essentially to performing fewer tests. By this argument, anything that reduced the number of tests always improves the inference (e.g., using a smaller annotation) which is clearly not always the case. The authors also suggest that rather than identifying DTE, a better approach with information summarized at the gene level may be to first identify which genes are changing and then identify differential isoform usage. There is no clear demonstration of the advantages of this approach or what the appropriate multiple testing correction is for the combined analysis. In particular, l find the interpretability of DTE far more straightforward than the combined approach.<br/><br/>The paper also examined the differences between the differential gene expression conclusions resulting from summarizing DTE at the gene level and DGE called on simple gene counts on both simulated and real data and find that, again somewhat unsurprisingly in my opinion, that the difference in the results is smaller and less obvious for real datasets than for the simulated data where they have added a strong signal. I found the evidence for using an adjusted transcript length is compelling despite the relatively small apparent impact it has for real datasets.<br/><br/>All in all I liked the paper and it was interesting to talk about it and discuss the issues it raises with my group.</div><span class="disclosures competing-interests-display padding-top"><strong>Competing Interests:</strong><span class=c-i-d>No competing interests were disclosed.</span></span><span class="hide-comment-container response-and-comment">Close</span></span><div class="report-concern-wrapper last-comment"><a href="mailto:research@f1000.com?subject=Report a concern (7563) " class=report-a-concern-link>Report a concern</a></div></div></div></li><li class="comment-wrapper not-sortable f1r-article-desk"><div class="bold padding-top padding-left-20"> Discussion is closed on this version, please comment on the latest version above. </div></li></ul></div></div><div class="f1r-article-mobile margin-bottom-30"><div class=contracted-details><a href="#" class="contracted-details-label author-affiliations"><span class=contracted></span>Author details</a><a href="#" class=section-title>Author details</a><span class="f1r-icon icon-14_more_small section-control"></span><span class="f1r-icon icon-10_less_small section-control"></span><div class="expanded-details affiliations is-hidden"><sup>1</sup> Institute for Molecular Life Sciences, University of Zurich, Zurich, 8057, Switzerland<br/><sup>2</sup> SIB Swiss Institute of Bioinformatics, University of Zurich, Zurich, 8057, Switzerland<br/><sup>3</sup> Department of Biostatistics and Computational Biology, Dana-Farber Cancer Institute, Boston, MA, 02210, USA<br/><sup>4</sup> Department of Biostatistics, Harvard TH Chan School of Public Health, Boston, MA, 02115, USA<br/><p></p></div></div><div class=contracted-details><a href="#" class=section-title>Article Versions (2)</a><span class="f1r-icon icon-14_more_small section-control"></span><span class="f1r-icon icon-10_less_small section-control"></span><div class="expanded-details grant-information article-page is-hidden"><div><a href="https://f1000research.com/articles/4-1521/v2" title="Open version 2 of this article." class="" gahelper=2>version 2</a><span class="article-pubinfo-mobile versions-section"><div title="#springMessage("NewVersionReasonType.REVISED.tooltip")" class="version-info-box revised ">Revised</div></span></div><div> Published: 29 Feb 2016, 4:1521 </div><div class=margin-bottom><a href="https://doi.org/10.12688/f1000research.7563.2">https://doi.org/10.12688/f1000research.7563.2</a></div><div><a href="https://f1000research.com/articles/4-1521/v1" title="Open version 1 of this article." class="" gahelper=1>version 1</a><span class="article-pubinfo-mobile versions-section"></span></div><div> Published: 30 Dec 2015, 4:1521 </div><div class=""><a href="https://doi.org/10.12688/f1000research.7563.1">https://doi.org/10.12688/f1000research.7563.1</a></div></div></div><div class=contracted-details><a href="#" class=section-title><span class="f1r-icon icon-100_open_access"></span> Copyright </a><span class="f1r-icon icon-14_more_small section-control"></span><span class="f1r-icon icon-10_less_small section-control"></span><div class="expanded-details grant-information article-page is-hidden">  2016 Soneson C <em>et al</em>. This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/" target=_blank data-test-id=box-licence-link>Creative Commons Attribution Licence</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. Data associated with the article are available under the terms of the <a href="http://creativecommons.org/publicdomain/zero/1.0/" target=_blank data-test-id=box-data-licence-link>Creative Commons Zero "No rights reserved" data waiver</a> (CC0 1.0 Public domain dedication). </div></div><div class="padding-top-30 research-layout"><div class=article-toolbox-wrapper-mobile><div class=article-tools-icon-mobile data-section=download><span class="f1r-icon icon-76_download_file white"></span></div><div class="article-tools-icon-mobile mobile-metrics article-metrics-wrapper metrics-icon-wrapper" data-section=metrics data-version-id=8770 data-id=7563 data-downloads="" data-views="" data-scholar="10.12688/f1000research.7563.2" data-recommended=726181976 data-f1r-ga-helper="Article Page Metrics (Mobile)"><span class="f1r-icon icon-89_metrics white"></span></div><div class=article-tools-divider-mobile></div><div class=article-tools-icon-mobile data-section=cite><span class="f1r-icon icon-82_quote white"></span></div><div class="article-tools-icon-mobile " data-section=track><span class="f1r-icon icon-90_track white"></span></div><div class=article-tools-divider-mobile></div><div class=article-tools-icon-mobile data-section=share><span class="f1r-icon icon-34_share white"></span></div><span class=article-toolbox-stretch></span></div><div class=article-toolbox-content-mobile><div class="toolbox-section download"><div class=toolbox-section-heading>Download</div><div class=toolbox-section-content><a href="https://f1000research.com/articles/4-1521/v2/pdf" title="Download PDF" class=no-decoration><span class="f1r-icon icon-102_download_pdf toolbox-section-icon"></span></a><div class=toolbox-section-option-divider>&nbsp;</div><a id=mobile-download-xml class=no-decoration href="#" title="Download XML"><span class="f1r-icon icon-103_download_xml toolbox-section-icon"></span></a></div><div class=toolbox-section-divider></div><div class=toolbox-section-heading>Export To</div><div class=toolbox-section-content><button class="primary no-fill orange-text-and-border mobile-export" data-etype=WORKSPACE>F1000Workspace</button><button class="primary no-fill orange-text-and-border mobile-export" data-etype=BIBTEX>Bibtex</button><button class="primary no-fill orange-text-and-border mobile-export" data-etype=ENDNOTE>EndNote</button><button class="primary no-fill orange-text-and-border mobile-export" data-etype=PROCITE>ProCite</button><button class="primary no-fill orange-text-and-border mobile-export" data-etype=REF_MANAGER>Ref. Manager (RIS)</button><button class="primary no-fill orange-text-and-border mobile-export" data-etype=SENTE>Sente</button></div></div><div class="toolbox-section metrics"><div class="toolbox-section-heading no-top-border">metrics</div><div class=toolbox-section-divider></div><div class=toolbox-section-colsplit><div class=toolbox-section-heading>VIEWS</div><div class="toolbox-section-count view-count js-views-count">$counts.viewCount</div></div><div class="toolbox-section-option-divider metrics">&nbsp;</div><div class=toolbox-section-colsplit><div class=toolbox-section-heading>downloads</div><div class="toolbox-section-count download-count js-downloads-count"></div></div><span class=metrics-citations-container><div class=toolbox-section-divider></div><div class="toolbox-section-heading u-mb--1">Citations</div><div><div class=toolbox-section-colsplit><div class=citations-scopus-logo><a href="" target=_blank class="is-hidden metrics-citation-icon" title="View full citation details at www.scopus.com"><i class="material-icons scopus-icon">open_in_new</i></a></div><div class="toolbox-section-count scopus"><a href='' class='scopus-citation-link is-hidden' target=_blank title='View full citation details at www.scopus.com'>0</a></div></div><div class="toolbox-section-option-divider metrics">&nbsp;</div><div class=toolbox-section-colsplit><div class="citations-pubmed-logo f1000research"><a href="" target=_blank class="is-hidden metrics-citation-icon f1000research" title="View full citation details"><i class="material-icons scopus-icon">open_in_new</i></a></div><div class="toolbox-section-count pubmed"><a href='' class='pubmed-citation-link is-hidden' target=_blank title='View full citation details'>0</a></div></div><div class=toolbox-section-divider></div><div class=toolbox-section-content><div class="citations-scholar-logo f1000research"><a href="" target=_blank class="is-hidden metrics-citation-icon google-scholar f1000research" title="View full citation details" data-scholar="10.12688/f1000research.7563.2"><i class="material-icons scopus-icon">open_in_new</i></a></div></div></div></span><span class=metrics-details-container><div class=toolbox-section-divider></div><div class="toolbox-section-content altmetric-section"><div class=altmetrics-image></div><div class=altmetrics-more-link><a href="" target=_blank class=f1r-standard-link>SEE MORE DETAILS</a></div><div class=altmetric-mobile-column-counts></div><div class=altmetric-mobile-column-readers></div><div class=toolbox-section-divider></div></div></span></div><div class="toolbox-section cite"><div class="toolbox-section-heading no-top-border">CITE</div><div class=toolbox-section-divider></div><div class=toolbox-section-heading>how to cite this article</div><div id=citation-copy-mobile class="toolbox-section-content text-content heading9 small"> Soneson C, Love MI and Robinson MD. Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences [version 2; peer review: 2 approved] <i>F1000Research</i> 2016, <b>4</b>:1521 (<a href="https://doi.org/10.12688/f1000research.7563.2" target=_blank>https://doi.org/10.12688/f1000research.7563.2</a>) </div><div class=toolbox-section-divider></div><div class="toolbox-section-content text-content heading9 small"> NOTE: <em>it is important to ensure the information in <b>square brackets after the title</b> is included in all citations of this article.</em></div><div class=toolbox-section-content><button class="primary orange extra-padding copy-cite-article-mobile js-clipboard" data-clipboard-target="#citation-copy-mobile" title="Copy the current citation details.">COPY CITATION DETAILS</button></div></div><div class="toolbox-section track"><div class="toolbox-section-heading no-top-border">track</div><div class=toolbox-section-divider></div><div class=toolbox-section-heading>receive updates on this article</div><div class="toolbox-section-content padding-left-20 padding-right-20 heading9 small"> Track an article to receive email alerts on any updates to this article. </div><div class=toolbox-section-content><a data-article-id=7563 id=mobile-track-article-signin-7563 title="Be alerted to any updates to this article." href="/login?originalPath=/trackArticle/7563?target=/articles/4-1521/v2"><button class="primary orange extra-padding"> TRACK THIS ARTICLE </button></a></div></div><div class="toolbox-section share"><div class="toolbox-section-heading no-top-border">Share</div><div class=toolbox-section-divider></div><div class=toolbox-section-content><a target=_blank class="f1r-shares-icon-square f1r-shares-email" title="Email this article"></a><a target=_blank class="f1r-shares-icon-square f1r-shares-twitter" title="Share on Twitter"></a><a target=_blank class="f1r-shares-icon-square f1r-shares-facebook" title="Share on Facebook"></a><a target=_blank class="f1r-shares-icon-square f1r-shares-linkedin" title="Share on LinkedIn"></a><a target=_blank class="f1r-shares-icon-square f1r-shares-reddit" title="Share on Reddit"></a><a target=_blank class="f1r-shares-icon-square f1r-shares-citeulike" title="Share on citeulike"></a><a target=_blank class="f1r-shares-icon-square f1r-shares-mendelay" title="Share on Mendeley"></a><div class="email-article-wrapper email-article-version-container"><div class=toolbox-section-divider></div><script src='https://www.recaptcha.net/recaptcha/api.js'></script><form class="recommend-version-form-mobile research-layout"><p>All fields are required.</p><input name=versionId type=hidden value=8770 /><input name=articleId type=hidden value=7563 /><input name=senderName class="form-input-field reg-form" value="" type=text placeholder="Your name"/><input name=senderEmail class="form-input-field reg-form margin-top" value="" type=text placeholder="Your email address"/><textarea name=recipientEmails class="form-textarea-field ninetynine-percent-wide margin-top no-resize" placeholder="Recipient email address(es) (comma delimited)"></textarea><input class="form-input-field reg-form margin-top" name=subject type=text value="Interesting article on F1000Research" placeholder=Subject /><textarea name=message class="form-textarea-field reg-form margin-top no-resize">I thought this article from F1000Research (https://f1000research.com) would be of interest to you.</textarea><div class="g-recaptcha margin-top" data-sitekey=6LcHqxoUAAAAANP3_0TzpGG6qFvl4DhbUcuRzw7W></div><input value="" name=captcha type=hidden /><p>A full article citation will be automatically included.</p><p><img class="ticker-email-article-details hidden" src="/img/ticker.gif" alt=loading /></p><button class="secondary orange margin-bottom" data-test-id=version_share_email_send>SEND EMAIL</button><div class="orange-message margin-bottom is-hidden" data-test-id=version_share_email_message></div></form></div></div></div></div></div><a name=article-reports></a><div id=article-reports class="u-mt--3 reports-comments no-divider"><div class="current-referee-status "><h2 class=main-title id=current-referee-status><span class="research-layout f1r-article-mobile-inline valign-middle"><span class="f1r-icon icon-85_peer_review size30"></span></span> Open Peer Review <span class="f1r-article-mobile-inline float-right"><span class="f1r-icon icon-14_more_small"></span><span class="f1r-icon icon-10_less_small"></span></span></h2><a name=current-referee-status></a><div class=current-referee-status__content name=add-new-report-comment id=add-new-report-comment> Current Reviewer Status: <span class="research-layout f1r-article-desk-inline"><span class="f1r-icon icon-86_approved status-green smaller" title=Approved></span></span><span class="research-layout f1r-article-mobile float-right"><span class="f1r-icon icon-86_approved status-green smaller" title=Approved></span></span><span class="research-layout f1r-article-desk-inline"><span class="f1r-icon icon-86_approved status-green smaller" title=Approved></span></span><span class="research-layout f1r-article-mobile float-right"><span class="f1r-icon icon-86_approved status-green smaller" title=Approved></span></span><span class="circle-icon-small to-right" data-window=about-referee-status title=Help>?</span><span class="research-layout f1r-article-mobile"><div class=mobile-ref-status-help> Key to Reviewer Statuses <span class=referee-status-pointer></span><span class="view-control float-right">VIEW</span><span class="view-control float-right is-hidden">HIDE</span><div class=mobile-ref-status-help-content><div class="cf margin-top"><span class="f1r-icon icon-86_approved status-green smaller float-left margin-bottom-40 margin-right" title=Approved></span><span class=title>Approved</span>The paper is scientifically sound in its current form and only minor, if any, improvements are suggested </div><div class="cf margin-top"><span class="f1r-icon icon-87_approved_reservations status-green smaller float-left margin-bottom-40 margin-right" title="Approved with Reservations"></span><span class=title>Approved with reservations</span>Key revisions are required to address specific details and make the paper fully scientifically sound </div><div class="cf margin-top"><span class="f1r-icon icon-88_not_approved status-red small float-left margin-bottom-30 margin-right" title="Not Approved"></span><span class=title>Not approved</span>Fundamental flaws in the paper seriously undermine the findings and conclusions </div></div></div></span></div></div><div class="f1r-article-desk-inline referee-report-info-box referee-report-version-box">Version 1</div><div class="f1r-article-mobile research-layout mobile-version-info padding-top-30 padding-bottom-30"><div class="mversion previous-version">VERSION 1</div><span class=details>PUBLISHED 30 Dec 2015</span><span class="article-pubinfo-mobile versions-section"></span></div><div class="referee-response-container no-border" data-reportID=11745><div class="f1r-article-mobile research-layout"><div class="version-border previous"></div></div><a name=referee-response-11745></a><div id=report-metadata-11745 class=is-hidden><input type=hidden name=reportId value=11745 /></div><div class="f1r-article-mobile-inline research-layout float-right margin-right-20"><span class="f1r-icon-circle-wrapper with-border white-bg big referee-reports"><span class="f1r-icon icon-86_approved status-green smaller" title=Approved></span></span></div><div class="article-count-box response-and-comment has-shadow"><div class=comments-count-title>Views</div><div class=comments-count-container>0</div><div class="article-interaction-info article-page"><div class="cite-article-popup-wrapper article-page-interaction-box"><div class="article-interaction-button cite-article-button" title="Cite this report" data-windowref=cite-report-popup-11745-1><span class="f1r-icon icon-82_quote"></span><a href="#" class="button-link cite-article-popup-link response-and-comment" title="Cite Report" data-test-id=cite-report-1>Cite</a></div><div id=cite-report-popup-11745-1 class="popup-window-wrapper is-hidden"><div class=cite-popup-background></div><div class="popup-window cite-this-article-box research-layout"><div class="popup-window-title small cite-title">How to cite this report:</div><span id=cite-report-text-11745-1><span class="article-title-and-info in-popup">Patro R. Reviewer Report For: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences [version 2; peer review: 2 approved]</span>. <i>F1000Research</i> 2016, <b>4</b>:1521<br/>(<a class=new-orange href="https://doi.org/10.5256/f1000research.8143.r11745" target=_blank>https://doi.org/10.5256/f1000research.8143.r11745</a>) </span><div class=padding-top data-test-id=report-url> The direct URL for this report is:<br><a href="https://f1000research.com/articles/4-1521/v1#referee-response-11745" class=new-orange target=_blank>https://f1000research.com/articles/4-1521/v1#referee-response-11745</a></div><div class="popup-window-title small note margin-top-20 margin-bottom-20"><strong>NOTE:</strong> it is important to ensure the information in square brackets after the title is included in this citation. </div><div class=float-right><button class="secondary no-fill orange-text-and-border margin-right-20 close-cite-popup uppercase">Close</button><button class="secondary orange copy-cite-article-version uppercase js-clipboard" title="Copy the current citation details to the clipboard." data-clipboard-target="#cite-report-text-11745-1">Copy Citation Details</button></div></div></div></div></div></div><div class="referee-report-info-box referee-report-details-box"><span class="bold mobile-report-text">Reviewer Report</span>&nbsp;<span class=mobile-report-info>12 Jan 2016</span></div><span class="info-separation padding-bottom padding-left"><div><span class=bold>Rob Patro<a href="http://orcid.org/0000-0001-8463-1675" target=_blank title="Open ORCID"><span class=orcid-logo-for-author-list></span></a></span><span class=f1r-article-desk-inline>,</span> Department of Computer Science, Stony Brook University, Stony Brook, NY, USA&nbsp; </div></span><div class="research-layout referee-report-status-line"><span class="f1r-icon icon-86_approved status-green smaller" title=Approved></span><span class=status-text>Approved</span></div><div class=f1r-article-mobile><span class=article-pubinfo-mobile><div class="version-info-box views">VIEWS 0</div></span></div><div class=referee-report-content><div class=hidden-text><a href="https://doi.org/10.5256/f1000research.8143.r11745" target=_blank>https://doi.org/10.5256/f1000research.8143.r11745</a></div><span class=truncated-comment> In this manuscript, the authors address a few questions of considerable (and perennial) interest in the analysis of RNA-seq data. Specifically, they provide evidence that, using available methods (e.g. DESeq2 / edgeR), assessing differential expression at the gene-level (DGE) is ... <span class="f1r-article-desk-inline continue-reading-container response-and-comment increment-view-count" data-reportID=11745>Continue reading</span><div class="f1r-article-mobile research-layout margin-top-30 is-centered"><button class="primary orange-text white-bg bigger-text expand-button-mobile continue-reading-container show with-border"> READ ALL <span class="f1r-icon icon-14_more_small orange vmiddle big"></span></button></div></span><span class="hidden hidden-report-text"><div class=hidden-report-text__comment>In this manuscript, the authors address a few questions of considerable (and perennial) interest in the analysis of RNA-seq data. Specifically, they provide evidence that, using available methods (e.g. DESeq2 / edgeR), assessing differential expression at the gene-level (DGE) is more robust than at the transcript level (DTE). Further, they convincingly argue that estimating abundance at the level of transcripts, and then aggregating these abundances to the gene level leads to improved estimation of differential gene expression. They demonstrate that one of the major factors in this improved estimation is the availability of a sample-specific feature length for each gene (derived from the abundance-weighted length of the expressed transcripts of this gene), which is not possible to obtain with any fixed gene model used by count-based methods. Finally, the authors argue that much of the analysis of interest at the transcript level does not actually require differential transcript expression testing, but rather can be inferred from a combination of DGE and differential transcript usage (DTU); this is an interesting proposition that merits further discussion and analysis. Overall, this is a well-written paper, with extensive and compelling supplementary and supporting data, that addresses a ubiquitous analysis task involving RNA-seq. &nbsp;It should be of broad interest to the community and makes a valuable contribution. The accompanying software, <i>tximport</i>, is user-friendly and makes it easy to apply the type of analysis recommended herein; it too should be widely useful.<br/><br/><b><u>Major comments:</u></b><br/><br/>It would be very useful to provide the equations used for calculating each of the abundance measured considered directly. Section 4 of the supplementary information is useful to this end, but the reader still has to search a bit to see exactly how each metric is computed (though the fantastic R-Markdown included with the figures means that these computations can be found explicitly).<br/><br/>Similarly, it would be useful to the reader to provide a description, in prose, of how specific experiments were performed (again, the reproducible nature of most of these experiments makes tracking down this information possible, but sometimes time-consuming). For example, how, precisely, was removal of transcripts handled at the level of the genome annotation? If a transcript consists only of constitutive exons, were all of those exons retained in the genome annotation used for STAR + featureCounts, while the transcript was removed in the Salmon index? &nbsp;<br/><br/>The result that transcript-level abundance estimation is more sensitive to the removal of transcripts than gene-level abundance estimation  this seems intuitive. However, I agree with Dr. Floors suggestion that:<br/><br/><i>&quot;The assertion that simple counts tended to show a high degree of robustness against incompleteness of the annotation catalog, as evidenced from estimation errors after first removing (at random) 20% of the transcripts seems misleading since Salmon-derived gene-level abundances actually show a higher Spearman correlation than count-derived gene-level abundances when subjected to removing a random 20% of transcripts.</i><br/><br/>I would suggest rewording this sentence, as the main result seems to be that gene-level analysis is more robust to an incomplete annotation than transcript-level analysis. Transcript-level abundance estimation followed by gene-level analysis seems to perform just as well (actually, better) than gene-level counting in this scenario.<br/><br/>The experiments in the section Incorporating transcript-level estimates leads to more accurate DGE results suggests the (reasonable) interpretation that the main benefit of incorporating transcript-level abundance estimates when assessing DGE is a more accurate measure of the feature length of the gene. The authors state  It is important to note that this improvement is entirely attributable to an improved handling of genes with changes in isoform composition between the conditions. This is supported by the fact that using the abundance-weighted average transcript length (i.e. offsets) with counting based approaches improves the results substantially. &nbsp;However, one other place where transcript-level abundance estimates are useful in the context of DGE is when assessing the expression of paralogous genes. While most multi-mapping reads that derive from different isoforms of the same gene will be uniquely mappable at the level of the genome, and hence will be included in the counts for that gene, reads that map ambiguously among paralogs may not be. In such cases, count-based methods do not have a principled way of apportioning a read between the paralogs involved, and discarding multi-mapping reads may negatively affect estimation of the abundance of the paralogs, even at the gene level. While this case is likely much less common than mis-estimation of DGE as a result of DTU, it is certainly of biological interest. I would suggest adding an analysis, restricted to sets of paralogous genes, comparing how the different approaches perform in this case. This may help to highlight the importance of not only deriving appropriately weighted and sample-dependent lengths for genes, but also on resolving multi-mapping ambiguity that occurs between genomically distinct loci.<br/><br/>The argument that most transcript-level analyses of interest may be addressed by looking at DGE in conjunction with DTU is an interesting one. It is certainly that case that not all tasks for which DTE is used actually require assessing differential expression at the transcript level. One issue with the DGE + DTU-based analysis which warrants further discussion in the manuscript is that I believe that this approach, too, would require correcting for multiple hypothesis testing. Specifically, one is testing both the DGE and the DTU hypotheses for each gene (or for a relevant subset of interest). The correction here is likely to be less harsh than in the case of assessing DTE, but is still worth discussing.<br/><br/>Minor comments:<br/><br/>As per Dr. Floors statement, Salmon (and Sailfish) also incorporate sequence-specific bias correction. Further, RSEM and Salmon (and a few other transcript-level abundance estimation tools) also incorporate the modeling of non-uniform fragment start position distributions. Of course, modeling a non-uniform start position distribution cannot overcome a complete lack of sampling in critical regions that might make determining transcript-level fragment assignment impossible, but it may help in properly apportioning an ambiguously-mapped fragment between transcripts depending on its relative position in each.<br/><br/>One potential added source of variability here is that all Salmon estimates presented in the manuscript make use of Salmons quasi-mapping of reads, while the STAR + featureCount pipeline makes use of traditional alignments. This is the primary intended usage mode of Salmon, and absolutely does represent a typical pipeline for methods that avoid alignment (Salmon, Sailfish, kallisto). However, it would probably be best to mention this as a potential (though likely negligible) additional source of variability.<br/><br/>In the discussion, the authors argue that ... it is therefore imperative to continue improving abundance estimation and inference methods applicable to individual transcripts, since misestimation can propagate to the gene level. This is, of course, an important and valid suggestion. Another direction, on which it would be useful to get the authors thoughts and suggestions, is the development of differential expression tools (at either the transcript or gene level) that can make use of the variance estimates that some tools (like Salmon) can provide. To what extent might incorporating this information help control false positive rates and improve DTE or even DGE estimates?</div><p style="margin-top: 10px;"><b class=mobile-block-display>Competing Interests:</b> No competing interests were disclosed.</p><span class="info-separation padding-top padding-bottom comment-statement-mobile"><div class=f1r-article-mobile><div class="content-divider report-info"></div></div><strong> I have read this submission. I believe that I have an appropriate level of expertise to confirm that it is of an acceptable scientific standard. </strong></span><div class=f1r-article-mobile><div class="content-divider report-info"></div></div><span class="f1r-article-desk hide-comment-container response-and-comment">Close</span><div class="f1r-article-mobile research-layout margin-top-30 is-centered"><button class="primary orange-text white-bg bigger-text expand-button-mobile hide-comment-container hide with-border"> READ LESS <span class="f1r-icon icon-10_less_small orange vmiddle big"></span></button></div></span><div class="f1r-article-mobile cite-report-wrapper"><a href="#" class=mobile-cite-report>CITE</a></div><div class="research-layout mobile-cite-report-popup-wrapper"><div class=bgmask></div><div class=mobile-cite-report-popup><span class="f1r-icon icon-88_not_approved close-cite-report-popup"></span><div class=cite-report-popup-title>CITE</div><span class=cite-report-popup-top-content> HOW TO CITE THIS REPORT </span><div class=cite-report-popup-content><span class=copy-cite-report id=cite-report-text-mobile-11745-1><span class=article-title-and-info>Patro R. Reviewer Report For: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences [version 2; peer review: 2 approved]</span>. <i>F1000Research</i> 2016, <b>4</b>:1521 (<a class=new-orange href="https://doi.org/10.5256/f1000research.8143.r11745" target=_blank>https://doi.org/10.5256/f1000research.8143.r11745</a>) </span></div><div class=cite-report-popup-divider></div><div class=cite-report-popup-content data-test-id=report-url> The direct URL for this report is:<br><a href="https://f1000research.com/articles/4-1521/v1#referee-response-11745" class=new-orangetarget=_blank>https://f1000research.com/articles/4-1521/v1#referee-response-11745</a></div><div class=cite-report-popup-divider></div><div class=cite-report-popup-content> NOTE: <em>it is important to ensure the information in <b>square brackets after the title</b> is included in all citations of this article.</em></div><div class=cite-report-popup-buttons><button type=button class="primary orange extra-padding copy-cite-report-mobile js-clipboard" title="Copy the current citation details to the clipboard." data-clipboard-target="#cite-report-text-mobile-11745-1">COPY CITATION DETAILS</button></div></div></div><div class=report-concern-wrapper><a class=report-a-concern-link href="mailto:research@f1000.com?subject=Report a concern (7563) ">Report a concern</a></div></div><div class=comments-list-container id=referee-comments-11745-1><div class=f1r-article-desk><ul class=all-comment-wrapper data-reportID=11745><li class="comment-wrapper not-sortable " data-commentID=1820><div class="one-comment with-divider "><a name=article-comment-1820></a><div class="f1r-article-desk commenter-role"> Author Response 29 Feb 2016 </div><div><span class=name>Mark Robinson</span><span class=name>, </span><span class=commenter-affiliation>University of Zurich, Zurich, Switzerland</span></div><div class="f1r-article-mobile the-comment-date"> 29 Feb 2016 </div><div class=f1r-article-mobile><div class="article-pubinfo-mobile comments-section"><span class="version-info-box comments"> Author Response </span></div></div><div class="comment-content "><span class=truncated-comment> Thank you for taking the time to read and review our paper. As per your suggestions, we have expanded the supplementary pdf document with equations defining each of the abundance ... <span class="continue-reading-container response-and-comment">Continue reading</span></span><span class="full-comment hidden"><span id=full-comment-original-1820 class=hidden>Thank you for taking the time to read and review our paper. As per your suggestions, we have expanded the supplementary pdf document with equations defining each of the abundance measures that we used as well as a detailed description of the generation of the incomplete annotation files. We have also reworded the paragraph discussing the influence of an incomplete annotation catalog, and added further discussion points regarding multiple testing correction for DGE+DTU as well as regarding new methods incorporating variance estimates into the differential analysis. However, a deeper discussion of the latter point falls outside the scope of this article.<br/>&nbsp;<br/>We found your suggestion to restrict the abundance accuracy comparison to paralogous genes interesting, and the supplement has been expanded with several examples comparing the accuracy of gene-level abundance estimates from Salmon and from STAR+featureCounts, restricted to sets of paralogous genes. While we see a clear advantage of Salmon for many of the paralogous gene groups, the overall difference is only slightly larger for sets of paralogous genes compared to random sets of genes (as can be seen in the revised version of Dataset 1). In addition, we have added the following text to the relevant section in the results:<br/>&nbsp;<br/>...suggesting that the extent of the problem in many real data sets is limited... though some loss of sensitivity for certain genes may be encountered from discarding multi-mapping fragments, which may be recovered through the use of transcript abundance estimators such as Salmon.<br/>&nbsp;<br/>While we acknowledge that many of the popular transcript abundance estimation methods incorporate some type of bias estimation/correction (albeit not at the fragment level), and that it is definitely an important (and difficult) research area, a thorough discussion on the relative merits of different bias correction approaches is outside the scope of this paper. We have added citations to relevant literature on coverage bias, have simplified and clarified the last paragraph of the discussion, and have reworded the sentence about unidentifiable estimation due to coverage biases:<br/>&nbsp;<br/>While some extent of coverage variability might be alleviated by corrections for sequence- or position-specific biases, there remain cases where transcript expression cannot be inferred from data ( Figure 1C)</span><div class=padding-bottom id=full-comment-1820>Thank you for taking the time to read and review our paper. As per your suggestions, we have expanded the supplementary pdf document with equations defining each of the abundance measures that we used as well as a detailed description of the generation of the incomplete annotation files. We have also reworded the paragraph discussing the influence of an incomplete annotation catalog, and added further discussion points regarding multiple testing correction for DGE+DTU as well as regarding new methods incorporating variance estimates into the differential analysis. However, a deeper discussion of the latter point falls outside the scope of this article.<br/>&nbsp;<br/>We found your suggestion to restrict the abundance accuracy comparison to paralogous genes interesting, and the supplement has been expanded with several examples comparing the accuracy of gene-level abundance estimates from Salmon and from STAR+featureCounts, restricted to sets of paralogous genes. While we see a clear advantage of Salmon for many of the paralogous gene groups, the overall difference is only slightly larger for sets of paralogous genes compared to random sets of genes (as can be seen in the revised version of Dataset 1). In addition, we have added the following text to the relevant section in the results:<br/>&nbsp;<br/>...suggesting that the extent of the problem in many real data sets is limited... though some loss of sensitivity for certain genes may be encountered from discarding multi-mapping fragments, which may be recovered through the use of transcript abundance estimators such as Salmon.<br/>&nbsp;<br/>While we acknowledge that many of the popular transcript abundance estimation methods incorporate some type of bias estimation/correction (albeit not at the fragment level), and that it is definitely an important (and difficult) research area, a thorough discussion on the relative merits of different bias correction approaches is outside the scope of this paper. We have added citations to relevant literature on coverage bias, have simplified and clarified the last paragraph of the discussion, and have reworded the sentence about unidentifiable estimation due to coverage biases:<br/>&nbsp;<br/>While some extent of coverage variability might be alleviated by corrections for sequence- or position-specific biases, there remain cases where transcript expression cannot be inferred from data ( Figure 1C)</div><span class="disclosures competing-interests-display padding-top"><strong>Competing Interests:</strong><span class=c-i-d>No competing interests were disclosed.</span></span><span class="hide-comment-container response-and-comment">Close</span></span><div class="report-concern-wrapper last-comment"><a href="mailto:research@f1000.com?subject=Report a concern (7563) " class=report-a-concern-link>Report a concern</a></div></div></div></li><li class="comment-wrapper not-sortable"><div class="one-comment respond-or-comment"><a href="/login?originalPath=/articles/4-1521/v2&scrollTo=referee-response-11745" class=register-report-comment-button data-test-id=add-response-comment><span class=contracted></span>Respond or Comment</a></div></li></ul></div><div class="f1r-article-mobile research-layout"><div class=reader-comment-header>COMMENTS ON THIS REPORT</div><ul class=all-comment-wrapper data-reportID=11745><li class="comment-wrapper not-sortable response-on-mobile" data-commentID=1820><div class="one-comment with-divider "><a name=article-comment-1820></a><div class="f1r-article-desk commenter-role"> Author Response 29 Feb 2016 </div><div><span class=name>Mark Robinson</span><span class=name>, </span><span class=commenter-affiliation>University of Zurich, Zurich, Switzerland</span></div><div class="f1r-article-mobile the-comment-date"> 29 Feb 2016 </div><div class=f1r-article-mobile><div class="article-pubinfo-mobile comments-section"><span class="version-info-box comments"> Author Response </span></div></div><div class="comment-content "><span class=truncated-comment> Thank you for taking the time to read and review our paper. As per your suggestions, we have expanded the supplementary pdf document with equations defining each of the abundance ... <span class="continue-reading-container response-and-comment">Continue reading</span></span><span class="full-comment hidden"><span id=full-comment-original-1820 class=hidden>Thank you for taking the time to read and review our paper. As per your suggestions, we have expanded the supplementary pdf document with equations defining each of the abundance measures that we used as well as a detailed description of the generation of the incomplete annotation files. We have also reworded the paragraph discussing the influence of an incomplete annotation catalog, and added further discussion points regarding multiple testing correction for DGE+DTU as well as regarding new methods incorporating variance estimates into the differential analysis. However, a deeper discussion of the latter point falls outside the scope of this article.<br/>&nbsp;<br/>We found your suggestion to restrict the abundance accuracy comparison to paralogous genes interesting, and the supplement has been expanded with several examples comparing the accuracy of gene-level abundance estimates from Salmon and from STAR+featureCounts, restricted to sets of paralogous genes. While we see a clear advantage of Salmon for many of the paralogous gene groups, the overall difference is only slightly larger for sets of paralogous genes compared to random sets of genes (as can be seen in the revised version of Dataset 1). In addition, we have added the following text to the relevant section in the results:<br/>&nbsp;<br/>...suggesting that the extent of the problem in many real data sets is limited... though some loss of sensitivity for certain genes may be encountered from discarding multi-mapping fragments, which may be recovered through the use of transcript abundance estimators such as Salmon.<br/>&nbsp;<br/>While we acknowledge that many of the popular transcript abundance estimation methods incorporate some type of bias estimation/correction (albeit not at the fragment level), and that it is definitely an important (and difficult) research area, a thorough discussion on the relative merits of different bias correction approaches is outside the scope of this paper. We have added citations to relevant literature on coverage bias, have simplified and clarified the last paragraph of the discussion, and have reworded the sentence about unidentifiable estimation due to coverage biases:<br/>&nbsp;<br/>While some extent of coverage variability might be alleviated by corrections for sequence- or position-specific biases, there remain cases where transcript expression cannot be inferred from data ( Figure 1C)</span><div class=padding-bottom id=full-comment-1820>Thank you for taking the time to read and review our paper. As per your suggestions, we have expanded the supplementary pdf document with equations defining each of the abundance measures that we used as well as a detailed description of the generation of the incomplete annotation files. We have also reworded the paragraph discussing the influence of an incomplete annotation catalog, and added further discussion points regarding multiple testing correction for DGE+DTU as well as regarding new methods incorporating variance estimates into the differential analysis. However, a deeper discussion of the latter point falls outside the scope of this article.<br/>&nbsp;<br/>We found your suggestion to restrict the abundance accuracy comparison to paralogous genes interesting, and the supplement has been expanded with several examples comparing the accuracy of gene-level abundance estimates from Salmon and from STAR+featureCounts, restricted to sets of paralogous genes. While we see a clear advantage of Salmon for many of the paralogous gene groups, the overall difference is only slightly larger for sets of paralogous genes compared to random sets of genes (as can be seen in the revised version of Dataset 1). In addition, we have added the following text to the relevant section in the results:<br/>&nbsp;<br/>...suggesting that the extent of the problem in many real data sets is limited... though some loss of sensitivity for certain genes may be encountered from discarding multi-mapping fragments, which may be recovered through the use of transcript abundance estimators such as Salmon.<br/>&nbsp;<br/>While we acknowledge that many of the popular transcript abundance estimation methods incorporate some type of bias estimation/correction (albeit not at the fragment level), and that it is definitely an important (and difficult) research area, a thorough discussion on the relative merits of different bias correction approaches is outside the scope of this paper. We have added citations to relevant literature on coverage bias, have simplified and clarified the last paragraph of the discussion, and have reworded the sentence about unidentifiable estimation due to coverage biases:<br/>&nbsp;<br/>While some extent of coverage variability might be alleviated by corrections for sequence- or position-specific biases, there remain cases where transcript expression cannot be inferred from data ( Figure 1C)</div><span class="disclosures competing-interests-display padding-top"><strong>Competing Interests:</strong><span class=c-i-d>No competing interests were disclosed.</span></span><span class="hide-comment-container response-and-comment">Close</span></span><div class="report-concern-wrapper last-comment"><a href="mailto:research@f1000.com?subject=Report a concern (7563) " class=report-a-concern-link>Report a concern</a></div></div></div></li><li class="comment-wrapper not-sortable button-wrapper"><div class=comment-button-wrapper><a href="/login?originalPath=/articles/4-1521/v2&scrollTo=referee-response-11745" class=register-report-comment-button data-test-id=add-report-comment><button class="primary orange extra-padding"> COMMENT ON THIS REPORT </button></a></div></li></ul></div></div></div><div class="referee-response-container no-border last-report-for-version" data-reportID=11761><div class="f1r-article-mobile research-layout"><div class="version-border previous"></div></div><a name=referee-response-11761></a><div id=report-metadata-11761 class=is-hidden><input type=hidden name=reportId value=11761 /></div><div class="f1r-article-mobile-inline research-layout float-right margin-right-20"><span class="f1r-icon-circle-wrapper with-border white-bg big referee-reports"><span class="f1r-icon icon-86_approved status-green smaller" title=Approved></span></span></div><div class="article-count-box response-and-comment has-shadow"><div class=comments-count-title>Views</div><div class=comments-count-container>0</div><div class="article-interaction-info article-page"><div class="cite-article-popup-wrapper article-page-interaction-box"><div class="article-interaction-button cite-article-button" title="Cite this report" data-windowref=cite-report-popup-11761-2><span class="f1r-icon icon-82_quote"></span><a href="#" class="button-link cite-article-popup-link response-and-comment" title="Cite Report" data-test-id=cite-report-2>Cite</a></div><div id=cite-report-popup-11761-2 class="popup-window-wrapper is-hidden"><div class=cite-popup-background></div><div class="popup-window cite-this-article-box research-layout"><div class="popup-window-title small cite-title">How to cite this report:</div><span id=cite-report-text-11761-2><span class="article-title-and-info in-popup">Floor SN. Reviewer Report For: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences [version 2; peer review: 2 approved]</span>. <i>F1000Research</i> 2016, <b>4</b>:1521<br/>(<a class=new-orange href="https://doi.org/10.5256/f1000research.8143.r11761" target=_blank>https://doi.org/10.5256/f1000research.8143.r11761</a>) </span><div class=padding-top data-test-id=report-url> The direct URL for this report is:<br><a href="https://f1000research.com/articles/4-1521/v1#referee-response-11761" class=new-orange target=_blank>https://f1000research.com/articles/4-1521/v1#referee-response-11761</a></div><div class="popup-window-title small note margin-top-20 margin-bottom-20"><strong>NOTE:</strong> it is important to ensure the information in square brackets after the title is included in this citation. </div><div class=float-right><button class="secondary no-fill orange-text-and-border margin-right-20 close-cite-popup uppercase">Close</button><button class="secondary orange copy-cite-article-version uppercase js-clipboard" title="Copy the current citation details to the clipboard." data-clipboard-target="#cite-report-text-11761-2">Copy Citation Details</button></div></div></div></div></div></div><div class="referee-report-info-box referee-report-details-box"><span class="bold mobile-report-text">Reviewer Report</span>&nbsp;<span class=mobile-report-info>04 Jan 2016</span></div><span class="info-separation padding-bottom padding-left"><div><span class=bold>Stephen N. Floor<a href="http://orcid.org/0000-0002-9965-9694" target=_blank title="Open ORCID"><span class=orcid-logo-for-author-list></span></a></span><span class=f1r-article-desk-inline>,</span> Department of Molecular and Cell Biology, University of California, Berkeley, Berkeley, CA, USA&nbsp; </div></span><div class="research-layout referee-report-status-line"><span class="f1r-icon icon-86_approved status-green smaller" title=Approved></span><span class=status-text>Approved</span></div><div class=f1r-article-mobile><span class=article-pubinfo-mobile><div class="version-info-box views">VIEWS 0</div></span></div><div class=referee-report-content><div class=hidden-text><a href="https://doi.org/10.5256/f1000research.8143.r11761" target=_blank>https://doi.org/10.5256/f1000research.8143.r11761</a></div><span class=truncated-comment> Soneson, Love and Robinson tackle a crucial question for analysis of RNA deep sequencing data in this manuscript: what is the role of transcript diversity in the accuracy and statistical power associated with measurements of gene expression? The authors make ... <span class="f1r-article-desk-inline continue-reading-container response-and-comment increment-view-count" data-reportID=11761>Continue reading</span><div class="f1r-article-mobile research-layout margin-top-30 is-centered"><button class="primary orange-text white-bg bigger-text expand-button-mobile continue-reading-container show with-border"> READ ALL <span class="f1r-icon icon-14_more_small orange vmiddle big"></span></button></div></span><span class="hidden hidden-report-text"><div class=hidden-report-text__comment>Soneson, Love and Robinson tackle a crucial question for analysis of RNA deep sequencing data in this manuscript: what is the role of transcript diversity in the accuracy and statistical power associated with measurements of gene expression? The authors make and convincingly show three claims: gene-level estimation and inferences are more robust than those at the transcript-level, and incorporating transcript-level quantification into gene-level abundance leads to improved differential expression testing.&nbsp; The claims are convincingly proven, the manuscript is well written, and the subject matter is of considerable interest. Furthermore, the described R package <i>tximport </i>should be of broad interest to the RNA deep sequencing community.<br/>&nbsp;<br/>Overall comments:<br/>&nbsp;<br/>It may be useful to indicate explicitly in the text that the methods are contained within the (excellently written and formatted) supplementary material, as this was not apparent. It might be clearest to create a specific methods section that just references supplementary file 1.<br/>&nbsp;<br/>The clarity of scatter plots with more than ~hundreds of points (e.g. Figure 1A) could be improved by using partially transparent points to visualize density.<br/>&nbsp;<br/>Introduction:<br/>&nbsp;<br/>Paragraph 1: Cufflinks, RSEM and Bitseq are grouped with kallisto and Salmon and it is then stated that some of these methods bypass read alignment. It would be clearer if this were reworded to avoid the ambiguity as to which methods avoid read alignment.<br/>&nbsp;<br/>Paragraph 4: The third claim could be presented more clearly. While it is interesting that simple counting performs similarly to transcript-level quantification procedures, it seems more interesting to this reviewer that incorporating transcript-level information improves the accuracy of differential expression testing at the gene level. Perhaps these two concepts can be combined into one more concise point?<br/>&nbsp;<br/>Results:<br/>&nbsp;<br/>The assertion that simple counts tended to show a high degree of robustness against incompleteness of the annotation catalog, as evidenced from estimation errors after first removing (at random) 20% of the transcripts seems misleading since Salmon-derived gene-level abundances actually show a higher Spearman correlation than count-derived gene-level abundances when subjected to removing a random 20% of transcripts. Figure 1a bottom left shows that transcript-level abundances are strongly affected by removal of 20% of transcripts, but that gene-level abundances are not strongly changed whether estimated using counts or Salmon. This statement should be reworded.<br/>&nbsp;<br/>Two concerns are raised about DTE. It is certainly true that reads are spread across more features when performing DTE as opposed to DGE.However, it is not apparent why analysis of DTE involves grouping of transcripts together for interpretation. DTE implies analysis at the transcript level and therefore no grouping, while DGE could involve some level of grouping of transcripts or quantification at the gene level from the start. The clarity of this could be improved.<br/>&nbsp;<br/>It is a very interesting idea to separately frame questions regarding DGE and DTU, which should be adopted widely, as the two are separable questions.<br/>&nbsp;<br/>The authors state one possible workflow towards DGE analysis in the section Incorporating transcript-level estimates leads to more accurate DGE results. Alternative pipelines (e.g. cuffdiff) could be presented in brief.<br/>&nbsp;<br/>The observation that simplesum and featureCounts results are highly correlated and therefore that statistical methods based on the Negative Binomial distribution can be used on estimated counts seems of greater importance than is emphasized in the text. This should be elaborated upon in the discussion, since this means that estimated counts from kallisto, express, salmon, etc can be used directly by statistical packages assuming a NB distribution (edgeR, DESeq2, etc). This point is frequently debated in discussions of how to rigorously analyze sequencing data. The conclusion here that NB applies to estimated counts is thus quite important.<br/>&nbsp;<br/>Please explain the meaning of the name for each curve in the legend for Figure 3 (i.e. specify that avetxl means using the offset corresponding to average transcript length.<br/>&nbsp;<br/>Discussion:<br/>&nbsp;<br/>The assertion that gene-level results are more accurate, powerful and interpretable than transcript-level results seems an oversimplification given the result that incorporating transcript-level quantification leads to improved DGE detection performance (e.g. Fig 3).<br/>&nbsp;<br/>Please cite at minimum Roberts <i>et al.</i>, (2011) regarding sequence bias correction as this has been implemented in cufflinks, express and kallisto. Other relevant papers should also be included here, as attempts have been made to address both positional and sequence-specific bias in RNA sequencing data.<br/>&nbsp;<br/>Supplement:<br/>&nbsp;<br/>The usability of the supplemental info could be improved by substituting rasterized for vectorized plots for those with ~hundreds of points.<br/>&nbsp;<br/>Please explain the meaning of the name for each curve in the legend for Supplemental Figure 5.</div><p><b>References</b></p><a name=rep-ref-11761-1></a>1. Roberts A, Trapnell C, Donaghey J, Rinn JL, Pachter L: Improving RNA-Seq expression estimates by correcting for fragment bias.<i>Genome Biol</i>. 2011; <b>12</b> (3): R22 <a target=_blank href="http://www.ncbi.nlm.nih.gov/pubmed/21410973">PubMed Abstract</a> | <a target=_blank href="https://doi.org/10.1186/gb-2011-12-3-r22">Publisher Full Text</a><br><p style="margin-top: 10px;"><b class=mobile-block-display>Competing Interests:</b> No competing interests were disclosed.</p><span class="info-separation padding-top padding-bottom comment-statement-mobile"><div class=f1r-article-mobile><div class="content-divider report-info"></div></div><strong> I have read this submission. I believe that I have an appropriate level of expertise to confirm that it is of an acceptable scientific standard. </strong></span><div class=f1r-article-mobile><div class="content-divider report-info"></div></div><span class="f1r-article-desk hide-comment-container response-and-comment">Close</span><div class="f1r-article-mobile research-layout margin-top-30 is-centered"><button class="primary orange-text white-bg bigger-text expand-button-mobile hide-comment-container hide with-border"> READ LESS <span class="f1r-icon icon-10_less_small orange vmiddle big"></span></button></div></span><div class="f1r-article-mobile cite-report-wrapper"><a href="#" class=mobile-cite-report>CITE</a></div><div class="research-layout mobile-cite-report-popup-wrapper"><div class=bgmask></div><div class=mobile-cite-report-popup><span class="f1r-icon icon-88_not_approved close-cite-report-popup"></span><div class=cite-report-popup-title>CITE</div><span class=cite-report-popup-top-content> HOW TO CITE THIS REPORT </span><div class=cite-report-popup-content><span class=copy-cite-report id=cite-report-text-mobile-11761-2><span class=article-title-and-info>Floor SN. Reviewer Report For: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences [version 2; peer review: 2 approved]</span>. <i>F1000Research</i> 2016, <b>4</b>:1521 (<a class=new-orange href="https://doi.org/10.5256/f1000research.8143.r11761" target=_blank>https://doi.org/10.5256/f1000research.8143.r11761</a>) </span></div><div class=cite-report-popup-divider></div><div class=cite-report-popup-content data-test-id=report-url> The direct URL for this report is:<br><a href="https://f1000research.com/articles/4-1521/v1#referee-response-11761" class=new-orangetarget=_blank>https://f1000research.com/articles/4-1521/v1#referee-response-11761</a></div><div class=cite-report-popup-divider></div><div class=cite-report-popup-content> NOTE: <em>it is important to ensure the information in <b>square brackets after the title</b> is included in all citations of this article.</em></div><div class=cite-report-popup-buttons><button type=button class="primary orange extra-padding copy-cite-report-mobile js-clipboard" title="Copy the current citation details to the clipboard." data-clipboard-target="#cite-report-text-mobile-11761-2">COPY CITATION DETAILS</button></div></div></div><div class=report-concern-wrapper><a class=report-a-concern-link href="mailto:research@f1000.com?subject=Report a concern (7563) ">Report a concern</a></div></div><div class=comments-list-container id=referee-comments-11761-2><div class=f1r-article-desk><ul class=all-comment-wrapper data-reportID=11761><li class="comment-wrapper not-sortable " data-commentID=1821><div class="one-comment with-divider "><a name=article-comment-1821></a><div class="f1r-article-desk commenter-role"> Author Response 29 Feb 2016 </div><div><span class=name>Mark Robinson</span><span class=name>, </span><span class=commenter-affiliation>University of Zurich, Zurich, Switzerland</span></div><div class="f1r-article-mobile the-comment-date"> 29 Feb 2016 </div><div class=f1r-article-mobile><div class="article-pubinfo-mobile comments-section"><span class="version-info-box comments"> Author Response </span></div></div><div class="comment-content "><span class=truncated-comment> Thank you for taking the time to read and review our paper. In the revised version, we have improved the clarity and usability of the figures and the supplement by ... <span class="continue-reading-container response-and-comment">Continue reading</span></span><span class="full-comment hidden"><span id=full-comment-original-1821 class=hidden>Thank you for taking the time to read and review our paper. In the revised version, we have improved the clarity and usability of the figures and the supplement by using partially transparent points and extending the figure captions. We have also modified the main text to refer more clearly to the method information provided in the supplementary material, and improved the clarity of the text in several places, according to your suggestions.<br/>&nbsp;<br/>For additional comments regarding references to bias correction studies, please see responses to Rob Patro.</span><div class=padding-bottom id=full-comment-1821>Thank you for taking the time to read and review our paper. In the revised version, we have improved the clarity and usability of the figures and the supplement by using partially transparent points and extending the figure captions. We have also modified the main text to refer more clearly to the method information provided in the supplementary material, and improved the clarity of the text in several places, according to your suggestions.<br/>&nbsp;<br/>For additional comments regarding references to bias correction studies, please see responses to Rob Patro.</div><span class="disclosures competing-interests-display padding-top"><strong>Competing Interests:</strong><span class=c-i-d>No competing interests were disclosed.</span></span><span class="hide-comment-container response-and-comment">Close</span></span><div class="report-concern-wrapper last-comment"><a href="mailto:research@f1000.com?subject=Report a concern (7563) " class=report-a-concern-link>Report a concern</a></div></div></div></li><li class="comment-wrapper not-sortable"><div class="one-comment respond-or-comment"><a href="/login?originalPath=/articles/4-1521/v2&scrollTo=referee-response-11761" class=register-report-comment-button data-test-id=add-response-comment><span class=contracted></span>Respond or Comment</a></div></li></ul></div><div class="f1r-article-mobile research-layout"><div class=reader-comment-header>COMMENTS ON THIS REPORT</div><ul class=all-comment-wrapper data-reportID=11761><li class="comment-wrapper not-sortable response-on-mobile" data-commentID=1821><div class="one-comment with-divider "><a name=article-comment-1821></a><div class="f1r-article-desk commenter-role"> Author Response 29 Feb 2016 </div><div><span class=name>Mark Robinson</span><span class=name>, </span><span class=commenter-affiliation>University of Zurich, Zurich, Switzerland</span></div><div class="f1r-article-mobile the-comment-date"> 29 Feb 2016 </div><div class=f1r-article-mobile><div class="article-pubinfo-mobile comments-section"><span class="version-info-box comments"> Author Response </span></div></div><div class="comment-content "><span class=truncated-comment> Thank you for taking the time to read and review our paper. In the revised version, we have improved the clarity and usability of the figures and the supplement by ... <span class="continue-reading-container response-and-comment">Continue reading</span></span><span class="full-comment hidden"><span id=full-comment-original-1821 class=hidden>Thank you for taking the time to read and review our paper. In the revised version, we have improved the clarity and usability of the figures and the supplement by using partially transparent points and extending the figure captions. We have also modified the main text to refer more clearly to the method information provided in the supplementary material, and improved the clarity of the text in several places, according to your suggestions.<br/>&nbsp;<br/>For additional comments regarding references to bias correction studies, please see responses to Rob Patro.</span><div class=padding-bottom id=full-comment-1821>Thank you for taking the time to read and review our paper. In the revised version, we have improved the clarity and usability of the figures and the supplement by using partially transparent points and extending the figure captions. We have also modified the main text to refer more clearly to the method information provided in the supplementary material, and improved the clarity of the text in several places, according to your suggestions.<br/>&nbsp;<br/>For additional comments regarding references to bias correction studies, please see responses to Rob Patro.</div><span class="disclosures competing-interests-display padding-top"><strong>Competing Interests:</strong><span class=c-i-d>No competing interests were disclosed.</span></span><span class="hide-comment-container response-and-comment">Close</span></span><div class="report-concern-wrapper last-comment"><a href="mailto:research@f1000.com?subject=Report a concern (7563) " class=report-a-concern-link>Report a concern</a></div></div></div></li><li class="comment-wrapper not-sortable button-wrapper"><div class=comment-button-wrapper><a href="/login?originalPath=/articles/4-1521/v2&scrollTo=referee-response-11761" class=register-report-comment-button data-test-id=add-report-comment><button class="primary orange extra-padding"> COMMENT ON THIS REPORT </button></a></div></li></ul></div></div></div></div><div class="f1r-article-mobile research-layout"><div class="mobile-sections-divider before-comments"></div></div><div id=article-comments class="article-comments padding-bottom-20"><div class=current-article-comment-section><h2 class=main-title name=add-new-comment id=add-new-comment><span class="research-layout f1r-article-mobile-inline valign-middle"><span class="f1r-icon icon-104_comments size30"></span></span><span class=f1r-article-desk-inline>Comments on this article</span><span class=f1r-article-mobile-inline>Comments (4)</span><span class="f1r-article-mobile-inline float-right"><span class="f1r-icon icon-14_more_small"></span><span class="f1r-icon icon-10_less_small"></span></span></h2></div><div class=comments-list-container><div class="f1r-article-desk-inline referee-report-info-box referee-report-version-box"> Version 2 </div><div class="f1r-article-mobile research-layout mobile-version-info padding-top-30"><span class="mversion ">VERSION 2</span><span class=details>PUBLISHED 29 Feb 2016</span><span class="article-pubinfo-mobile versions-section"><div title="#springMessage("NewVersionReasonType.REVISED.tooltip")" class="version-info-box revised ">Revised</div></span></div><ul class=all-comment-wrapper data-reportID=NONE data-versionID=2><li class="comment-wrapper not-sortable " data-commentID=3453><div class="one-comment version-comments "><a name=article-comment-3453></a><div class="f1r-article-desk commenter-role"> Author Response 23 Feb 2018 </div><div><span class=name>Mark Robinson</span><span class=name>, </span><span class=commenter-affiliation>University of Zurich, Zurich, Switzerland</span></div><div class="f1r-article-mobile the-comment-date"> 23 Feb 2018 </div><div class=f1r-article-mobile><div class="article-pubinfo-mobile comments-section"><span class="version-info-box comments"> Author Response </span></div></div><div class="comment-content article-comment-list"><span class=truncated-comment> Written by Charlotte Soneson, Michael I Love and Mark D Robinson.<br/><br/>One of the main points of our paper was to point out and illustrate that defining the null hypothesis of ... <span class="continue-reading-container response-and-comment">Continue reading</span></span><span class="full-comment hidden"><span id=full-comment-original-3453 class=hidden>Written by Charlotte Soneson, Michael I Love and Mark D Robinson.<br/><br/>One of the main points of our paper was to point out and illustrate that defining the null hypothesis of interest is of great importance when it comes to choosing the right data processing and analysis approach (see e.g., the Introduction, or Table 2 for a summary). In particular, we considered three main null hypotheses on the gene level, testing for what we may denote: <ol><li>DGE (differential gene expression)</li><li>DTE+G (differential transcript expression aggregated to the gene level)</li><li>DTU (differential transcript usage).</li></ol> The first null hypothesis (tested in DGE analyses) is that the total &quot;transcriptional output&quot; of a gene* is the same between the compared groups. This is the null hypothesis considered in Figure 3, and indeed for most differential gene expression analyses to date. Now, consider for simplicity a gene with two isoforms of very different length, and assume that only the short isoform is expressed in experimental condition 1, and only the long isoform is expressed in condition 2, but that the total expression of the gene is the same in the two groups. Using the summarized gene counts (in the paper denoted as simplesum) to test the DGE null hypothesis will (falsely) detect a difference between the groups, since the observed gene count depends on the length of the expressed transcripts. Incorporating the average transcript length offsets, or using scaled TPMs instead of simplesum counts, addresses this problem and improves the inference (again, for the DGE null hypothesis) in situations with considerable differential transcript usage. We show this conceptually with (as stated) &quot;extreme&quot; simulated data, but we also illustrate that (as Dr. Pachter comments) for many real data sets, the overall effect is much smaller, and thus, simplesum aggregation indeed provides similar global performance as scaled TPMs. This does not exclude that there may be important effects for individual genes or for other data sets, and therefore we recommend users to follow the steps suggested in the tximport vignette (again, assuming that DGE is the relevant hypothesis to test).<br/><br/>It is important to recall that the concern that DTU among isoforms of different length could induce false positives in DGE analysis was the main issue that <a href="https://www.nature.com/articles/nbt.2450">Trapnell et al (2013)</a> had with competing count-based statistical methods. From the Results section of that paper:<br/><br/><i>When all of a gene's isoforms are up- or downregulated between two conditions, raw count methods recover true change in gene expression. However, when some isoforms are upregulated and others downregulated, raw count methods are inaccurate.</i><br/><br/>In the comment, Dr. Pachter notes that our average transcript length or scaled TPM corrections for performing DGE, which protect against potential false positives induced by DTU, may provide similar global performance as simplesum for a given dataset. This would imply that the major concern of Trapnell et al (2013) with count-based statistical methods may not be a large problem for every dataset. However, this does not invalidate the concern of Trapnell et al (2013), rather it means that the problem primarily affects individual genes with DTU among isoforms of different length, and this may not occur in every dataset, or may not affect the global operating characteristics.<br/><br/>The null hypothesis tested in Supplementary Figure 2 of the Ntranos/Yi et al paper mentioned by Dr. Pachter is very different from the DGE null hypothesis discussed above. In their figure, a gene is considered &quot;truly differential&quot; if any of its transcripts are truly differentially expressed between the compared groups. We consider this null hypothesis for bulk, full-length RNA-seq in our Supplementary Figure 9, and we show that, indeed, performing DGE on the simplesum counts detects more of the truly differential genes (defined as those with any transcript DE rather than where the total concentration of transcripts changes) than performing DGE by including the average transcript length offsets or using scaled TPMs. This is natural and expected, since the offsets are explicitly designed to eliminate the effect of DTU on the overall (aggregated) gene count, to avoid this affecting DGE analysis as described above. Thus, a gene as the one mentioned in the first paragraph will have different gene-level counts due (only) to the usage of different isoforms in the two conditions, not to a change in overall expression, and will be detected in a DGE analysis applied to summarized gene counts, but not if offsets are included. However, the important message of Supplementary Figure 9 is that neither of these gene-level summarization approaches should be applied if the goal is to find genes where any of the transcripts are differentially expressed (the hypothesis tested in Ntranos/Yi et al). Instead, in this situation, transcript-level information should be used. In our Supplementary Figure 9, we exemplify this by performing differential expression analysis on the individual transcript level, followed by aggregation of the test results to the gene level. As expected, this correctly detects many more of the genes with any transcript DE.<br/><br/>Thus, as we discuss in depth in our paper, the answer to the question &quot;How should I sum transcript counts on the gene level with tximport?&quot; depends on the null hypothesis that one is interested in testing. If the null hypothesis of interest is that the total transcriptional output of a gene is the same between conditions, incorporating the average transcript length offsets or using scaled (or length-scaled) TPMs will protect against effects of expressed transcript length on observed gene abundances (these are the steps one will find for DGE analysis in the tximport vignette). If the null hypothesis of interest is that none of the transcripts of a gene are differentially expressed, then the transcript-level abundances should not be aggregated to the gene level. Instead, we recommend using transcript-level information to answer this question (see, for example the methods papers for <a href="https://bioconductor.org/packages/DRIMSeq">DRIMSeq</a> or <a href="https://github.com/statOmics/stageR">stageR</a>).<br/><br/>In the context of transcript-level analysis, it is worth noting that statements about our F1000 paper written in <a href="https://liorpachter.wordpress.com/2018/02/15/gde%C2%B2-dge%C2%B2-dtu%C2%B2-dte%E2%82%81%C2%B2-dte%E2%82%82%C2%B2/">Dr. Pachters blog</a> (and at <a href="http://www.rna-seqblog.com/dont-bother-with-transcript-level-analysis/">RNASeqBlog</a>) are not true. In the 15 Feb 2018 version of Dr. Pachters blog, it is stated Soneson et al. 2016 suggest that while DTE and DTU may be appropriate in certain niche applications, generally its better to choose DGE, and they therefore advise not to bother with transcript-level analysis. However, we highly recommend transcript-level analysis, whether it be a combination of DGE and DTU, or via DTE+G; in particular, text from our Discussion is copied below with emphasis added:<br/><br/><i>Our results highlight the importance of carefully specifying the question of interest before selecting a statistical approach. Summarization of abundance estimates at the gene level before performing the statistical testing should be the method of choice if the interest is in finding changes in the overall transcriptional output of a gene. <b>However, it is suboptimal if the goal is to identify genes for which at least one of the transcripts show differences in transcriptional output, since it may miss genes where two transcripts change in opposite directions, or where a lowly expressed transcript changes</b>. For gene-level detection of DTE (that is, whether any transcript showed a change in expression between the conditions), statistical testing applied to aggregated gene counts led to reduced power and slightly inflated FDR compared to <b>performing the statistical test on the transcript level and aggregating results within genes</b> (Supplementary Figure 9). Statistical inference on aggregated transcript TPMs (scaledTPM) showed low power for detecting changes that did not affect the overall transcriptional output of the gene, as expected. <b>An alternative to DTE analysis, for potential improved interpretability, is to perform a combination of DGE and DTU analyses, both resulting in gene-level inferences. Table 2 summarizes our results and give suggested workflows for the different types of analyses we have considered.</b></i><br/><br/>Our main message overall is that analysts should think about and carefully specify the differential expression question of interest among the many possibilities.<br/><br/>* =concentration of transcript molecules, summed over all isoforms</span><div class=padding-bottom id=full-comment-3453>Written by Charlotte Soneson, Michael I Love and Mark D Robinson.<br/><br/>One of the main points of our paper was to point out and illustrate that defining the null hypothesis of interest is of great importance when it comes to choosing the right data processing and analysis approach (see e.g., the Introduction, or Table 2 for a summary). In particular, we considered three main null hypotheses on the gene level, testing for what we may denote: <ol><li>DGE (differential gene expression)</li><li>DTE+G (differential transcript expression aggregated to the gene level)</li><li>DTU (differential transcript usage).</li></ol> The first null hypothesis (tested in DGE analyses) is that the total &quot;transcriptional output&quot; of a gene* is the same between the compared groups. This is the null hypothesis considered in Figure 3, and indeed for most differential gene expression analyses to date. Now, consider for simplicity a gene with two isoforms of very different length, and assume that only the short isoform is expressed in experimental condition 1, and only the long isoform is expressed in condition 2, but that the total expression of the gene is the same in the two groups. Using the summarized gene counts (in the paper denoted as simplesum) to test the DGE null hypothesis will (falsely) detect a difference between the groups, since the observed gene count depends on the length of the expressed transcripts. Incorporating the average transcript length offsets, or using scaled TPMs instead of simplesum counts, addresses this problem and improves the inference (again, for the DGE null hypothesis) in situations with considerable differential transcript usage. We show this conceptually with (as stated) &quot;extreme&quot; simulated data, but we also illustrate that (as Dr. Pachter comments) for many real data sets, the overall effect is much smaller, and thus, simplesum aggregation indeed provides similar global performance as scaled TPMs. This does not exclude that there may be important effects for individual genes or for other data sets, and therefore we recommend users to follow the steps suggested in the tximport vignette (again, assuming that DGE is the relevant hypothesis to test).<br/><br/>It is important to recall that the concern that DTU among isoforms of different length could induce false positives in DGE analysis was the main issue that <a href="https://www.nature.com/articles/nbt.2450">Trapnell et al (2013)</a> had with competing count-based statistical methods. From the Results section of that paper:<br/><br/><i>When all of a gene's isoforms are up- or downregulated between two conditions, raw count methods recover true change in gene expression. However, when some isoforms are upregulated and others downregulated, raw count methods are inaccurate.</i><br/><br/>In the comment, Dr. Pachter notes that our average transcript length or scaled TPM corrections for performing DGE, which protect against potential false positives induced by DTU, may provide similar global performance as simplesum for a given dataset. This would imply that the major concern of Trapnell et al (2013) with count-based statistical methods may not be a large problem for every dataset. However, this does not invalidate the concern of Trapnell et al (2013), rather it means that the problem primarily affects individual genes with DTU among isoforms of different length, and this may not occur in every dataset, or may not affect the global operating characteristics.<br/><br/>The null hypothesis tested in Supplementary Figure 2 of the Ntranos/Yi et al paper mentioned by Dr. Pachter is very different from the DGE null hypothesis discussed above. In their figure, a gene is considered &quot;truly differential&quot; if any of its transcripts are truly differentially expressed between the compared groups. We consider this null hypothesis for bulk, full-length RNA-seq in our Supplementary Figure 9, and we show that, indeed, performing DGE on the simplesum counts detects more of the truly differential genes (defined as those with any transcript DE rather than where the total concentration of transcripts changes) than performing DGE by including the average transcript length offsets or using scaled TPMs. This is natural and expected, since the offsets are explicitly designed to eliminate the effect of DTU on the overall (aggregated) gene count, to avoid this affecting DGE analysis as described above. Thus, a gene as the one mentioned in the first paragraph will have different gene-level counts due (only) to the usage of different isoforms in the two conditions, not to a change in overall expression, and will be detected in a DGE analysis applied to summarized gene counts, but not if offsets are included. However, the important message of Supplementary Figure 9 is that neither of these gene-level summarization approaches should be applied if the goal is to find genes where any of the transcripts are differentially expressed (the hypothesis tested in Ntranos/Yi et al). Instead, in this situation, transcript-level information should be used. In our Supplementary Figure 9, we exemplify this by performing differential expression analysis on the individual transcript level, followed by aggregation of the test results to the gene level. As expected, this correctly detects many more of the genes with any transcript DE.<br/><br/>Thus, as we discuss in depth in our paper, the answer to the question &quot;How should I sum transcript counts on the gene level with tximport?&quot; depends on the null hypothesis that one is interested in testing. If the null hypothesis of interest is that the total transcriptional output of a gene is the same between conditions, incorporating the average transcript length offsets or using scaled (or length-scaled) TPMs will protect against effects of expressed transcript length on observed gene abundances (these are the steps one will find for DGE analysis in the tximport vignette). If the null hypothesis of interest is that none of the transcripts of a gene are differentially expressed, then the transcript-level abundances should not be aggregated to the gene level. Instead, we recommend using transcript-level information to answer this question (see, for example the methods papers for <a href="https://bioconductor.org/packages/DRIMSeq">DRIMSeq</a> or <a href="https://github.com/statOmics/stageR">stageR</a>).<br/><br/>In the context of transcript-level analysis, it is worth noting that statements about our F1000 paper written in <a href="https://liorpachter.wordpress.com/2018/02/15/gde%C2%B2-dge%C2%B2-dtu%C2%B2-dte%E2%82%81%C2%B2-dte%E2%82%82%C2%B2/">Dr. Pachters blog</a> (and at <a href="http://www.rna-seqblog.com/dont-bother-with-transcript-level-analysis/">RNASeqBlog</a>) are not true. In the 15 Feb 2018 version of Dr. Pachters blog, it is stated Soneson et al. 2016 suggest that while DTE and DTU may be appropriate in certain niche applications, generally its better to choose DGE, and they therefore advise not to bother with transcript-level analysis. However, we highly recommend transcript-level analysis, whether it be a combination of DGE and DTU, or via DTE+G; in particular, text from our Discussion is copied below with emphasis added:<br/><br/><i>Our results highlight the importance of carefully specifying the question of interest before selecting a statistical approach. Summarization of abundance estimates at the gene level before performing the statistical testing should be the method of choice if the interest is in finding changes in the overall transcriptional output of a gene. <b>However, it is suboptimal if the goal is to identify genes for which at least one of the transcripts show differences in transcriptional output, since it may miss genes where two transcripts change in opposite directions, or where a lowly expressed transcript changes</b>. For gene-level detection of DTE (that is, whether any transcript showed a change in expression between the conditions), statistical testing applied to aggregated gene counts led to reduced power and slightly inflated FDR compared to <b>performing the statistical test on the transcript level and aggregating results within genes</b> (Supplementary Figure 9). Statistical inference on aggregated transcript TPMs (scaledTPM) showed low power for detecting changes that did not affect the overall transcriptional output of the gene, as expected. <b>An alternative to DTE analysis, for potential improved interpretability, is to perform a combination of DGE and DTU analyses, both resulting in gene-level inferences. Table 2 summarizes our results and give suggested workflows for the different types of analyses we have considered.</b></i><br/><br/>Our main message overall is that analysts should think about and carefully specify the differential expression question of interest among the many possibilities.<br/><br/>* =concentration of transcript molecules, summed over all isoforms</div><span class="disclosures competing-interests-display padding-top"><strong>Competing Interests:</strong><span class=c-i-d>No competing interests were disclosed.</span></span><span class="hide-comment-container response-and-comment">Close</span></span><div class="report-concern-wrapper last-comment"><a href="mailto:research@f1000.com?subject=Report a concern (7563) " class=report-a-concern-link>Report a concern</a></div></div></div></li><li class="comment-wrapper not-sortable " data-commentID=3436><div class="one-comment version-comments "><a name=article-comment-3436></a><div class="f1r-article-desk commenter-role"> Reviewer Response 20 Feb 2018 </div><div><span class=name>Lior Pachter</span><span class=name>, </span><span class=commenter-affiliation>University of California Berkeley, Berkeley, USA</span></div><div class="f1r-article-mobile the-comment-date"> 20 Feb 2018 </div><div class=f1r-article-mobile><div class="article-pubinfo-mobile comments-section"><span class="version-info-box comments"> Reviewer Response </span></div></div><div class="comment-content article-comment-list"><span class=truncated-comment> Many users have inquired about which of the tximport options they should use when summing transcript-level abundances.&nbsp;The paper provides some mixed messaging in this regard, first stating that &quot;Accounting for ... <span class="continue-reading-container response-and-comment">Continue reading</span></span><span class="full-comment hidden"><span id=full-comment-original-3436 class=hidden>Many users have inquired about which of the tximport options they should use when summing transcript-level abundances.&nbsp;The paper provides some mixed messaging in this regard, first stating that &quot;Accounting for the potentially varying average transcript length across samples when performing DGE, either in the definition of the count matrix (scaledTPM) or by defining offsets (featureCounts_avetxl, simplesum_avetxl), led to considerably improved false discovery rate (FDR) control&quot; and then&nbsp;&quot;Experiments on various real data sets (<a>Supplementary Figure 20</a>) show only small differences in the collections of significant genes found with the simplesum and simplesum_avetxl approaches, suggesting that the extent of the problem in many real data sets is limited.&quot;<br/><br/> To shed further light on this question, we recently examined the performance of tximport in the single-cell RNA-Seq setting. In Supplementary Figure 2 of the paper <a href="https://www.biorxiv.org/content/early/2018/02/14/258566">Ntranos, Yi&nbsp;<i>et al.</i>&nbsp;2018</a>&nbsp;we see almost identical performance when using tximport (+DESeq2) with the simplesum, scaledTPM or lengthscaledTPM options (current options available via countsFromAbundance in tximport). Our result is from a simulation study where the simulation parameters are based on transcript changes seen in a biological dataset.<br/><br/> Our result indicates that simplesum may be adequate in practice (in our experiment even slightly outperforming the other normalizations), which is useful to know because the summation of transcript estimated counts to the gene-level is so simple to perform that it does not require tximport.</span><div class=padding-bottom id=full-comment-3436>Many users have inquired about which of the tximport options they should use when summing transcript-level abundances.&nbsp;The paper provides some mixed messaging in this regard, first stating that &quot;Accounting for the potentially varying average transcript length across samples when performing DGE, either in the definition of the count matrix (scaledTPM) or by defining offsets (featureCounts_avetxl, simplesum_avetxl), led to considerably improved false discovery rate (FDR) control&quot; and then&nbsp;&quot;Experiments on various real data sets (<a>Supplementary Figure 20</a>) show only small differences in the collections of significant genes found with the simplesum and simplesum_avetxl approaches, suggesting that the extent of the problem in many real data sets is limited.&quot;<br/><br/> To shed further light on this question, we recently examined the performance of tximport in the single-cell RNA-Seq setting. In Supplementary Figure 2 of the paper <a href="https://www.biorxiv.org/content/early/2018/02/14/258566">Ntranos, Yi&nbsp;<i>et al.</i>&nbsp;2018</a>&nbsp;we see almost identical performance when using tximport (+DESeq2) with the simplesum, scaledTPM or lengthscaledTPM options (current options available via countsFromAbundance in tximport). Our result is from a simulation study where the simulation parameters are based on transcript changes seen in a biological dataset.<br/><br/> Our result indicates that simplesum may be adequate in practice (in our experiment even slightly outperforming the other normalizations), which is useful to know because the summation of transcript estimated counts to the gene-level is so simple to perform that it does not require tximport.</div><span class="disclosures competing-interests-display padding-top"><strong>Competing Interests:</strong><span class=c-i-d>I have publicly accused the authors of another paper, Patro et al., Nature Methods 2017, of plagiarism. Mike Love (co-author of this paper) is a coauthor of the Patro et al. paper.</span></span><span class="hide-comment-container response-and-comment">Close</span></span><div class="report-concern-wrapper last-comment"><a href="mailto:research@f1000.com?subject=Report a concern (7563) " class=report-a-concern-link>Report a concern</a></div></div></div></li><li class="comment-wrapper not-sortable button-wrapper"><div class="one-comment version-comments add-the-comment"><a href="/login?originalPath=/articles/4-1521/v2&scrollTo=add-new-comments" class=register-report-comment-button data-test-id=add-comment><span class=f1r-article-desk><span class=contracted></span>Comment</span><span class="f1r-article-mobile research-layout "><button class="primary orange extra-padding comment-on-this-article">ADD YOUR COMMENT</button></span></a></div></li></ul><div class="f1r-article-mobile research-layout"><div class="mobile-sections-divider versions-divider"></div></div><div class="f1r-article-desk-inline referee-report-info-box referee-report-version-box"> Version 1 </div><div class="f1r-article-mobile research-layout mobile-version-info padding-top-30"><span class="mversion previous-version">VERSION 1</span><span class=details>PUBLISHED 30 Dec 2015</span><span class="article-pubinfo-mobile versions-section"></span></div><div class="comments-closed f1r-article-mobile"> Discussion is closed on this version, please comment on the latest version above. </div><ul class=all-comment-wrapper data-reportID=NONE data-versionID=1><li class="comment-wrapper not-sortable " data-commentID=1792><div class="one-comment version-comments "><a name=article-comment-1792></a><div class="f1r-article-desk commenter-role"> Author Response 02 Feb 2016 </div><div><span class=name>Charlotte Soneson</span><span class=name>, </span><span class=commenter-affiliation>University of Zurich, Switzerland</span></div><div class="f1r-article-mobile the-comment-date"> 02 Feb 2016 </div><div class=f1r-article-mobile><div class="article-pubinfo-mobile comments-section"><span class="version-info-box comments"> Author Response </span></div></div><div class="comment-content article-comment-list"><span class=truncated-comment> Hi Nick,<br/><br/>thanks for your comments.&nbsp;<br/><br/>Regarding the increased accuracy and robustness of gene-level estimates compared to transcript-level ones (even with the full annotation, bootstrap variances of gene estimates are lower than ... <span class="continue-reading-container response-and-comment">Continue reading</span></span><span class="full-comment hidden"><span id=full-comment-original-1792 class=hidden>Hi Nick,<br/><br/>thanks for your comments.&nbsp;<br/><br/>Regarding the increased accuracy and robustness of gene-level estimates compared to transcript-level ones (even with the full annotation, bootstrap variances of gene estimates are lower than those of transcript estimates), we agree that it is not surprising. However, we still found it worth pointing out since it implies that in some situations, gene-level statistical analyses may be preferred due to the increased precision of the input data.<br/><br/>For your second point, note that Figure 2 does not in fact compare DGE to DTE, but rather DTE summarized on gene- and transcript-level. You are of course right that just reducing the number of tests does not automatically result in higher power. Instead, the difference between the gene- and transcript-level results in Figure 2 is largely due to the different null hypotheses (as outlined in the text) and, consequently, the type of signal we require to call a feature (gene or transcript) significant. In fact, the gene-level summarized analysis is answering a somewhat &quot;easier&quot; question, from the sensitivity point of view. Consider for example a gene with 5 differentially expressed transcripts. On the gene level, we can reach a power of 100% for this gene if <b>any </b>of the transcripts is considered significantly DE. On the transcript level, we would need <b>all</b> of the transcripts to be significant to reach the same power. Looking at this particular data set, for many of the genes that are found as true positives with the gene level test, not all truly DE transcripts are actually detected. This explains the lower power of the transcript-level analysis. We will clarify this in the revised version. Regarding the choice of DTE vs DGE+DTU, it is clear that one solution will not always be the optimal choice, and it will likely depend on the particular problem as well as on the person interpreting the results. However, we have found that in many of our own collaborations, the biological question can be more clearly stated in terms of DGE+DTU (not necessarily performed sequentially, rather in parallel). Part of the reason for this discussion was to encourage researchers to think about what question they really want to answer before starting their analyses.&nbsp;<br/><br/>Finally, note that the only place where we are actually comparing &quot;summarizing DTE at the gene level&quot; and DGE is in Supplementary Figure 5 (on simulated data). In Figures 3-4, the question of interest is always comparing the total transcriptional output of a gene between conditions (i.e., DGE), and all methods are based on aggregating gene abundance estimates before the statistical test is applied. As you note, we don't see a big global effect of including offsets accounting for average transcript lengths for real data. However, there may of course still be important effects for individual genes, where isoforms of different lengths are expressed in different conditions.&nbsp;<br/><br/>Thanks again for your comments, and we are glad you found the paper useful!</span><div class=padding-bottom id=full-comment-1792>Hi Nick,<br/><br/>thanks for your comments.&nbsp;<br/><br/>Regarding the increased accuracy and robustness of gene-level estimates compared to transcript-level ones (even with the full annotation, bootstrap variances of gene estimates are lower than those of transcript estimates), we agree that it is not surprising. However, we still found it worth pointing out since it implies that in some situations, gene-level statistical analyses may be preferred due to the increased precision of the input data.<br/><br/>For your second point, note that Figure 2 does not in fact compare DGE to DTE, but rather DTE summarized on gene- and transcript-level. You are of course right that just reducing the number of tests does not automatically result in higher power. Instead, the difference between the gene- and transcript-level results in Figure 2 is largely due to the different null hypotheses (as outlined in the text) and, consequently, the type of signal we require to call a feature (gene or transcript) significant. In fact, the gene-level summarized analysis is answering a somewhat &quot;easier&quot; question, from the sensitivity point of view. Consider for example a gene with 5 differentially expressed transcripts. On the gene level, we can reach a power of 100% for this gene if <b>any </b>of the transcripts is considered significantly DE. On the transcript level, we would need <b>all</b> of the transcripts to be significant to reach the same power. Looking at this particular data set, for many of the genes that are found as true positives with the gene level test, not all truly DE transcripts are actually detected. This explains the lower power of the transcript-level analysis. We will clarify this in the revised version. Regarding the choice of DTE vs DGE+DTU, it is clear that one solution will not always be the optimal choice, and it will likely depend on the particular problem as well as on the person interpreting the results. However, we have found that in many of our own collaborations, the biological question can be more clearly stated in terms of DGE+DTU (not necessarily performed sequentially, rather in parallel). Part of the reason for this discussion was to encourage researchers to think about what question they really want to answer before starting their analyses.&nbsp;<br/><br/>Finally, note that the only place where we are actually comparing &quot;summarizing DTE at the gene level&quot; and DGE is in Supplementary Figure 5 (on simulated data). In Figures 3-4, the question of interest is always comparing the total transcriptional output of a gene between conditions (i.e., DGE), and all methods are based on aggregating gene abundance estimates before the statistical test is applied. As you note, we don't see a big global effect of including offsets accounting for average transcript lengths for real data. However, there may of course still be important effects for individual genes, where isoforms of different lengths are expressed in different conditions.&nbsp;<br/><br/>Thanks again for your comments, and we are glad you found the paper useful!</div><span class="disclosures competing-interests-display padding-top"><strong>Competing Interests:</strong><span class=c-i-d>No competing interests were disclosed.</span></span><span class="hide-comment-container response-and-comment">Close</span></span><div class="report-concern-wrapper last-comment"><a href="mailto:research@f1000.com?subject=Report a concern (7563) " class=report-a-concern-link>Report a concern</a></div></div></div></li><li class="comment-wrapper not-sortable " data-commentID=1787><div class="one-comment version-comments "><a name=article-comment-1787></a><div class="f1r-article-desk commenter-role"> Reader Comment (<span class=italic>Member of the F1000 Faculty</span>) 29 Jan 2016 </div><div><span class=name>Nick Schurch</span><span class=name>, </span><span class=commenter-affiliation>The Barton Group, Division of Computational Biology College of Life Sciences, University of Dundee, UK</span></div><div class="f1r-article-mobile the-comment-date"> 29 Jan 2016 </div><div class=f1r-article-mobile><div class="article-pubinfo-mobile comments-section"><span class="version-info-box comments"> Reader Comment </span><span class="version-info-box comments fm">Member of the F1000 Faculty</span></div></div><div class="comment-content article-comment-list"><span class=truncated-comment> I read this work with great interest for a literature review and found it engaging and informative. I thought I'd post my resulting thoughts here.<br/><br/>This paper focusses on the advantages ... <span class="continue-reading-container response-and-comment">Continue reading</span></span><span class="full-comment hidden"><span id=full-comment-original-1787 class=hidden>I read this work with great interest for a literature review and found it engaging and informative. I thought I'd post my resulting thoughts here.<br/><br/>This paper focusses on the advantages and disadvantages of performing differential expression analysis at either the gene level or the transcript level, relying heavily on simulated data. Initially the authors use simulated data to show that gene level estimations of expression are more stable then transcript level estimations. They do this by showing that with the removal of ~20% of the transcripts from the annotation, the quantification the transcript level estimations becomes less accurate while the gene level estimation remains relatively unchanged. I found this conclusion to be unsurprising; the transcripts are the primary annotation used for transcript level quantification but are not the primary annotation for the gene level estimation. Except in rare cases where the removal of the transcripts result in the removal of the parent gene or removal of the longest transcripts results in a considerable change in the length of the parent gene, the gene annotation will remain largely unaltered by the removal of transcripts.<br/><br/>The authors then consider the performance of differential expression at the gene level (DGE) vs differential expression at the transcript level (DTE), concluding that inference at the gene level is more robust, easier to interpret and has better statistical performance. While the argument for improved robustness is compelling, I found the arguments for improved inference and statistical performance a little misleading. The improved statistical performance of DGE is due essentially to performing fewer tests. By this argument, anything that reduced the number of tests always improves the inference (e.g., using a smaller annotation) which is clearly not always the case. The authors also suggest that rather than identifying DTE, a better approach with information summarized at the gene level may be to first identify which genes are changing and then identify differential isoform usage. There is no clear demonstration of the advantages of this approach or what the appropriate multiple testing correction is for the combined analysis. In particular, l find the interpretability of DTE far more straightforward than the combined approach.<br/><br/>The paper also examined the differences between the differential gene expression conclusions resulting from summarizing DTE at the gene level and DGE called on simple gene counts on both simulated and real data and find that, again somewhat unsurprisingly in my opinion, that the difference in the results is smaller and less obvious for real datasets than for the simulated data where they have added a strong signal. I found the evidence for using an adjusted transcript length is compelling despite the relatively small apparent impact it has for real datasets.<br/><br/>All in all I liked the paper and it was interesting to talk about it and discuss the issues it raises with my group.</span><div class=padding-bottom id=full-comment-1787>I read this work with great interest for a literature review and found it engaging and informative. I thought I'd post my resulting thoughts here.<br/><br/>This paper focusses on the advantages and disadvantages of performing differential expression analysis at either the gene level or the transcript level, relying heavily on simulated data. Initially the authors use simulated data to show that gene level estimations of expression are more stable then transcript level estimations. They do this by showing that with the removal of ~20% of the transcripts from the annotation, the quantification the transcript level estimations becomes less accurate while the gene level estimation remains relatively unchanged. I found this conclusion to be unsurprising; the transcripts are the primary annotation used for transcript level quantification but are not the primary annotation for the gene level estimation. Except in rare cases where the removal of the transcripts result in the removal of the parent gene or removal of the longest transcripts results in a considerable change in the length of the parent gene, the gene annotation will remain largely unaltered by the removal of transcripts.<br/><br/>The authors then consider the performance of differential expression at the gene level (DGE) vs differential expression at the transcript level (DTE), concluding that inference at the gene level is more robust, easier to interpret and has better statistical performance. While the argument for improved robustness is compelling, I found the arguments for improved inference and statistical performance a little misleading. The improved statistical performance of DGE is due essentially to performing fewer tests. By this argument, anything that reduced the number of tests always improves the inference (e.g., using a smaller annotation) which is clearly not always the case. The authors also suggest that rather than identifying DTE, a better approach with information summarized at the gene level may be to first identify which genes are changing and then identify differential isoform usage. There is no clear demonstration of the advantages of this approach or what the appropriate multiple testing correction is for the combined analysis. In particular, l find the interpretability of DTE far more straightforward than the combined approach.<br/><br/>The paper also examined the differences between the differential gene expression conclusions resulting from summarizing DTE at the gene level and DGE called on simple gene counts on both simulated and real data and find that, again somewhat unsurprisingly in my opinion, that the difference in the results is smaller and less obvious for real datasets than for the simulated data where they have added a strong signal. I found the evidence for using an adjusted transcript length is compelling despite the relatively small apparent impact it has for real datasets.<br/><br/>All in all I liked the paper and it was interesting to talk about it and discuss the issues it raises with my group.</div><span class="disclosures competing-interests-display padding-top"><strong>Competing Interests:</strong><span class=c-i-d>No competing interests were disclosed.</span></span><span class="hide-comment-container response-and-comment">Close</span></span><div class="report-concern-wrapper last-comment"><a href="mailto:research@f1000.com?subject=Report a concern (7563) " class=report-a-concern-link>Report a concern</a></div></div></div></li><li class="comment-wrapper not-sortable f1r-article-desk"><div class="bold padding-top padding-left-20"> Discussion is closed on this version, please comment on the latest version above. </div></li></ul></div></div></div></div><div id=article_main-column class="p-article__sidebar o-layout__item u-1/3 not-expanded js-article-sidebar"><div class="o-tab p-article__column-toggle-container"><button class="c-tab c-tab--left js-column-toggle p-article__column-toggle not-expanded " type=button data-target-main=article_main-column data-target-secondary=article_secondary-column><i class="c-tab__icon material-icons u-hide@expanded">keyboard_arrow_left</i><i class="c-tab__icon material-icons u-show@expanded">keyboard_arrow_right</i></button></div><div class=p-article__sidebar-content><div class="p-article__sidebar-scroller js-article-sidebar-scroller"><section class="p-article__sidebar-view js-article-sidebar-view js-article-sidebar-main u-pt u-pb--8" data-view=peer-review><div class="o-layout o-layout--flush"><div class="o-layout__item u-pl"><h3 class="u-mt--0 u-mb--2 t-h3 u-weight--md u-pl" data-test-id=article_sidebar_heading>Open Peer Review</h3></div><div class=o-layout__item><section class=""><div class="p-article__sidebar-highlight u-mb--2 u-pr--1"><div class="o-actions o-actions--middle"><div class=o-actions__primary><h4 class="u-mt--0 u-mb--0 u-ib u-middle t-h4 u-weight--md u-mr--1/2">Reviewer Status</h4><div class="c-referee-status__icons u-ib u-middle"><i class="c-icn--f1r-icon c-icn--approved " title=Approved #data-refInfo=13464-11761></i><i class="c-icn--f1r-icon c-icn--approved " title=Approved #data-refInfo=11706-11745></i></div></div><div class="o-actions__secondary _mdl-layout"><div class="c-block-tip p-article__sidebar-tooltip c-block-tip--below c-block-tip--small-arrow c-block-tip--sm-padding"><button type=button class="c-button c-button--icon c-button--text c-block-tip__toggle"><i class="material-icons c-button--icon__icon">info_outline</i></button><div class=c-block-tip__content><p class="t-body u-mt--0 u-mb--0"><em class=u-weight--md>Alongside their report, reviewers assign a status to the article:</em></p><dl class=c-definitions><dt class="c-definitions__term u-upper t-control u-weight--md u-black--high"><span class="u-ib u-middle"><i class="c-icn--f1r-icon c-icn--approved " title=Approved #data-refInfo=""></i></span><span class="u-middle u-ib u-ml--1/2">Approved</span></dt><dd class="c-definitions__description t-caption">The paper is scientifically sound in its current form and only minor, if any, improvements are suggested</dd><dt class="c-definitions__term u-upper t-control u-weight--md u-black--high"><span class="u-ib u-middle"><i class="c-icn--f1r-icon c-icn--reservations" title="Approved with Reservations" #data-refInfo=""></i></span><span class="u-middle u-ib u-ml--1/2">Approved with reservations</span></dt><dd class="c-definitions__description t-caption">Key revisions are required to address specific details and make the paper fully scientifically sound</dd><dt class="c-definitions__term u-upper t-control u-weight--md u-black--high"><span class="u-ib u-middle"><i class="c-icn--f1r-icon c-icn--not-approved " title="Not Approved" #data-refInfo=""></i></span><span class="u-middle u-ib u-ml--1/2">Not approved</span></dt><dd class="c-definitions__description t-caption">Fundamental flaws in the paper seriously undermine the findings and conclusions</dd></dl></div></div></div></div></div><div class="o-layout__item u-mb--1"><h4 class="u-mt--0 u-mb--0 u-ib u-middle t-h4 u-weight--md">Reviewer Reports</h4></div><table class="c-report-timeline u-mb--2"><thead class=c-report-timeline__headings><tr><th></th><th class="u-pb--1/2" colspan=2><em class="t-body u-weight--rg">Invited Reviewers</em></th></tr><tr class=c-report-timeline__headings-row><th></th><th class="c-report-timeline__headings-version p-article__color--dark">1</th><th class="c-report-timeline__headings-version p-article__color--dark">2</th></tr></thead><tbody><tr class="c-report-timeline__row c-report-timeline__row--empty c-report-timeline__row--selected "><th class=c-report-timeline__version><a href="https://f1000research.com/articles/4-1521/v2">Version 2</a><br/><span class=p-article__color--dark> (revision)<br/><span class=c-report-timeline__date>29 Feb 16</span></span></th><td class="c-report-timeline__report--empty c-report-timeline__cell c-report-timeline__report--has-prev "></td><td class="c-report-timeline__report--empty c-report-timeline__cell c-report-timeline__report--has-prev "></td></tr><tr class="c-report-timeline__row "><th class=c-report-timeline__version><a href="https://f1000research.com/articles/4-1521/v1">Version 1</a><br/><span class=p-article__color--dark><span class=c-report-timeline__date>30 Dec 15</span></span></th><td class="c-report-timeline__report c-report-timeline__cell "><a class="js-route c-report-timeline__report-link" title="04 Jan 2016" href="https://f1000research.com/articles/4-1521/v1#referee-response-11761" data-path="/reports/11761" target-attr=referee-response-11761 data-reportID=11761 style="cursor: pointer;"><i class="c-icn--f1r-icon c-icn--approved small" title=Approved #data-refInfo=""></i><span class="u-block c-report-timeline__report-read u-mt--1">read</span></a></td><td class="c-report-timeline__report c-report-timeline__cell "><a class="js-route c-report-timeline__report-link" title="12 Jan 2016" href="https://f1000research.com/articles/4-1521/v1#referee-response-11745" data-path="/reports/11745" target-attr=referee-response-11745 data-reportID=11745 style="cursor: pointer;"><i class="c-icn--f1r-icon c-icn--approved small" title=Approved #data-refInfo=""></i><span class="u-block c-report-timeline__report-read u-mt--1">read</span></a></td></tr></tbody></table><div class=o-layout__item><hr class="c-hr c-hr--low u-mb--2"><ol class=p-article__sidebar-referees-list><li class="u-pl--1 u-mb--1/2 t-small"><div data-test-id=sidebar_referee><strong data-test-id=sidebar_referee_name>Stephen N. Floor</strong><a href="http://orcid.org/0000-0002-9965-9694" target=_blank title="Open ORCID"><span class=orcid-logo-for-author-list></span></a>, University of California, Berkeley, Berkeley, USA </div></li><li class="u-pl--1 u-mb--1/2 t-small"><div data-test-id=sidebar_referee><strong data-test-id=sidebar_referee_name>Rob Patro</strong><a href="http://orcid.org/0000-0001-8463-1675" target=_blank title="Open ORCID"><span class=orcid-logo-for-author-list></span></a>, Stony Brook University, Stony Brook, USA </div></li></ol></div></section></div><section class="o-layout__item u-pl"><div class=u-pl><hr class="c-hr c-hr--low c-hr--md u-mb--3"><h4 class="t-h3 u-weight--md u-mt--0 u-mb--2">Comments on this article</h4><div class=u-mb--4><p class="u-mt--0 u-mb--1 t-h4"><a class=p-article__color--light href="#article-comments">All Comments</a><span class=" u-ib u-ml--1/2 p-article__color--light">(4)</span></p><a class=t-h4 href="/login?originalPath=/articles/4-1521/v2&scrollTo=add-new-comment" data-test-id=add-comment>Add a comment</a></div><hr class="c-hr c-hr--low c-hr--md u-mb--4"><div class="research-layout f1r-article-desk"><div class="heading6 c-ribbon-wrapper c-ribbon-wrapper--etoc f1000research "><div class=c-ribbon-wrapper__body>Sign up for content alerts</div></div></div><div class="research-layout sidebar-sign-up-form f1r-article-desk u-mb--4 "><form class=js-email-alert-signup action="#" method=POST data-email=tocAlertWeekly><input type=hidden name=isUserLoggedIn class=js-email-alert-signup-logged-in value=N /><input type=hidden name=userId class=js-email-alert-signup-user-id value=""/><input type=hidden name=frequency class=js-email-alert-signup-frequency value=WEEKLY /><div class="o-actions o-actions--middle"><div class=o-actions__primary><input type=email name=emailAddress class="form-input-field js-email-alert-signup-address u-1/1 u-bb" required=required placeholder=Email /></div><div class=o-actions__secondary><div class="_mdl-layout u-ml--1/2"><button class="mdl-button mdl-js-button mdl-button--colored mdl-button--small mdl-button--filled js-email-alert-signup-submit">Sign Up</button></div></div></div></form><div id=sidebar-sign-up-message class="section-text js-email-alert-signup-msg is-hidden">You are now signed up to receive this alert</div></div><section class=js-terms-container><hr class="c-hr c-hr--low c-hr--md u-mb--3"><h4 class="t-h3 u-weight--md u-mt--0 u-mb--1">Browse by related subjects</h4><div class="article-subcontainer article-subcontainer--sidebar"><ul class=js-terms-list></ul></div></section></div></section></div></section><div class="p-article__sidebar-view u-pl u-hide js-article-sidebar-view" data-view=report-11745><a class="u-block js-route c-link--basic p-article__sidebar-back" href="#" data-path="/"><i class="material-icons c-link__icon">keyboard_arrow_left</i><span class=c-link__text>Back to all reports</span></a><aside class="c-referee-report js-referee-report t-body u-pr--2 u-pb--8" data-reportid=11745 data-test-id=article_referee_report><div class=o-layout><header class="o-layout__item c-referee-report__header"><div class="o-actions o-actions--middle u-mb--1"><div class=o-actions__primary><h2 class="t-h3 u-weight--md u-mb--0 u-mt--0">Reviewer Report</h2></div><div class="o-actions__secondary u-hide@expanded"><span class="t-body u-weight--rg js-report-view-count u-ml--1 u-black--medium"><span class=js-referee-report-views>0</span> Views</span></div><div class="o-actions__secondary u-hide@expanded"><div class="c-block-tip p-article__sidebar-tooltip c-block-tip--below c-block-tip--small-arrow c-block-tip--sm-padding"><button type=button class="c-button c-button--icon c-button--text c-block-tip__toggle"><i class="material-icons c-button--icon__icon">copyright</i></button><div class="c-block-tip__content u-fs--normal"><span class=u-weight--bd> 2016 Patro R.</span> This is an open access peer review report distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" target=_blank>Creative Commons Attribution Licence</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. </div></div></div></div><p class="u-black--medium u-mt--0 u-mb--1"><em>12 Jan 2016 | for Version 1</em></p><div class="u-mb--1/2"><span class=bold>Rob Patro<a href="http://orcid.org/0000-0001-8463-1675" target=_blank title="Open ORCID"><span class=orcid-logo-for-author-list></span></a></span><span class=f1r-article-desk-inline>,</span> Department of Computer Science, Stony Brook University, Stony Brook, NY, USA&nbsp; </div><hr class="c-hr c-hr--low u-mt--2 u-mb--2 u-hide@expanded"></header><div class="o-layout__item c-referee-report__actions u-mb--2"><span class="t-body u-weight--rg js-report-view-count u-mt--0 u-mb--1 u-black--medium u-show@expanded"><span class=js-referee-report-views>0</span> Views</span><div id=copyrightInfo class="c-block-tip p-article__sidebar-tooltip c-block-tip--below c-block-tip--small-arrow u-show@expanded u-fr"><button type=button class="c-button c-button--icon c-button--text c-block-tip__toggle"><i class="material-icons c-button--icon__icon">copyright</i></button><div class="c-block-tip__content u-fs--normal"><span class=u-weight--bd> 2016 Patro R.</span> This is an open access peer review report distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" target=_blank>Creative Commons Attribution Licence</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. </div></div><a class="u-block c-link--basic u-mb--1 js-route" data-path="/reports/11745/cite" href="#" data-test-id=article_referee-report_cite-link><i class="material-icons c-link__icon">format_quote</i><span class="u-ml--1 c-link__text">Cite this report</span></a><a href="#report-responses-11745" class="u-block c-link--basic u-mb--1/2 js-route" data-path="/reports/11745/responses" href="#" data-test-id=article_referee-report_responses-link><i class="material-icons c-link__icon">speaker_notes</i><span class="u-ml--1 u-mr--1/2 c-link__text u-ib">Responses</span><span class="u-black--medium u-ib u-middle">(1)</span></a></div><div class="u-mb--1 u-show@expanded"></div></div><hr class="c-hr c-hr--low u-mt--0 u-mb--2"><div class="o-actions o-actions--middle u-mb--1"><div class="o-actions__primary p-article__color--dark"><i class="c-icn--f1r-icon c-icn--approved " title=Approved #data-refInfo=11706-11745></i><span class="p-article__report-status t-caption u-upper-spacing u-upper u-ib u-middle u-weight--md">Approved</span></div><div class="o-actions__secondary _mdl-layout"><div class="c-block-tip p-article__sidebar-tooltip c-block-tip--below c-block-tip--small-arrow c-block-tip--sm-padding"><button type=button class="c-button c-button--icon c-button--text c-block-tip__toggle"><i class="material-icons c-button--icon__icon">info_outline</i></button><div class=c-block-tip__content><p class="t-body u-mt--0 u-mb--0"><em class=u-weight--md>Alongside their report, reviewers assign a status to the article:</em></p><dl class=c-definitions><dt class="c-definitions__term u-upper t-control u-weight--md u-black--high"><span class="u-ib u-middle"><i class="c-icn--f1r-icon c-icn--approved " title=Approved #data-refInfo=""></i></span><span class="u-middle u-ib u-ml--1/2">Approved</span></dt><dd class="c-definitions__description t-caption">The paper is scientifically sound in its current form and only minor, if any, improvements are suggested</dd><dt class="c-definitions__term u-upper t-control u-weight--md u-black--high"><span class="u-ib u-middle"><i class="c-icn--f1r-icon c-icn--reservations" title="Approved with Reservations" #data-refInfo=""></i></span><span class="u-middle u-ib u-ml--1/2">Approved with reservations</span></dt><dd class="c-definitions__description t-caption">Key revisions are required to address specific details and make the paper fully scientifically sound</dd><dt class="c-definitions__term u-upper t-control u-weight--md u-black--high"><span class="u-ib u-middle"><i class="c-icn--f1r-icon c-icn--not-approved " title="Not Approved" #data-refInfo=""></i></span><span class="u-middle u-ib u-ml--1/2">Not approved</span></dt><dd class="c-definitions__description t-caption">Fundamental flaws in the paper seriously undermine the findings and conclusions</dd></dl></div></div></div></div><div class="">In this manuscript, the authors address a few questions of considerable (and perennial) interest in the analysis of RNA-seq data. Specifically, they provide evidence that, using available methods (e.g. DESeq2 / edgeR), assessing differential expression at the gene-level (DGE) is more robust than at the transcript level (DTE). Further, they convincingly argue that estimating abundance at the level of transcripts, and then aggregating these abundances to the gene level leads to improved estimation of differential gene expression. They demonstrate that one of the major factors in this improved estimation is the availability of a sample-specific feature length for each gene (derived from the abundance-weighted length of the expressed transcripts of this gene), which is not possible to obtain with any fixed gene model used by count-based methods. Finally, the authors argue that much of the analysis of interest at the transcript level does not actually require differential transcript expression testing, but rather can be inferred from a combination of DGE and differential transcript usage (DTU); this is an interesting proposition that merits further discussion and analysis. Overall, this is a well-written paper, with extensive and compelling supplementary and supporting data, that addresses a ubiquitous analysis task involving RNA-seq. &nbsp;It should be of broad interest to the community and makes a valuable contribution. The accompanying software, <i>tximport</i>, is user-friendly and makes it easy to apply the type of analysis recommended herein; it too should be widely useful.<br/><br/><b><u>Major comments:</u></b><br/><br/>It would be very useful to provide the equations used for calculating each of the abundance measured considered directly. Section 4 of the supplementary information is useful to this end, but the reader still has to search a bit to see exactly how each metric is computed (though the fantastic R-Markdown included with the figures means that these computations can be found explicitly).<br/><br/>Similarly, it would be useful to the reader to provide a description, in prose, of how specific experiments were performed (again, the reproducible nature of most of these experiments makes tracking down this information possible, but sometimes time-consuming). For example, how, precisely, was removal of transcripts handled at the level of the genome annotation? If a transcript consists only of constitutive exons, were all of those exons retained in the genome annotation used for STAR + featureCounts, while the transcript was removed in the Salmon index? &nbsp;<br/><br/>The result that transcript-level abundance estimation is more sensitive to the removal of transcripts than gene-level abundance estimation  this seems intuitive. However, I agree with Dr. Floors suggestion that:<br/><br/><i>&quot;The assertion that simple counts tended to show a high degree of robustness against incompleteness of the annotation catalog, as evidenced from estimation errors after first removing (at random) 20% of the transcripts seems misleading since Salmon-derived gene-level abundances actually show a higher Spearman correlation than count-derived gene-level abundances when subjected to removing a random 20% of transcripts.</i><br/><br/>I would suggest rewording this sentence, as the main result seems to be that gene-level analysis is more robust to an incomplete annotation than transcript-level analysis. Transcript-level abundance estimation followed by gene-level analysis seems to perform just as well (actually, better) than gene-level counting in this scenario.<br/><br/>The experiments in the section Incorporating transcript-level estimates leads to more accurate DGE results suggests the (reasonable) interpretation that the main benefit of incorporating transcript-level abundance estimates when assessing DGE is a more accurate measure of the feature length of the gene. The authors state  It is important to note that this improvement is entirely attributable to an improved handling of genes with changes in isoform composition between the conditions. This is supported by the fact that using the abundance-weighted average transcript length (i.e. offsets) with counting based approaches improves the results substantially. &nbsp;However, one other place where transcript-level abundance estimates are useful in the context of DGE is when assessing the expression of paralogous genes. While most multi-mapping reads that derive from different isoforms of the same gene will be uniquely mappable at the level of the genome, and hence will be included in the counts for that gene, reads that map ambiguously among paralogs may not be. In such cases, count-based methods do not have a principled way of apportioning a read between the paralogs involved, and discarding multi-mapping reads may negatively affect estimation of the abundance of the paralogs, even at the gene level. While this case is likely much less common than mis-estimation of DGE as a result of DTU, it is certainly of biological interest. I would suggest adding an analysis, restricted to sets of paralogous genes, comparing how the different approaches perform in this case. This may help to highlight the importance of not only deriving appropriately weighted and sample-dependent lengths for genes, but also on resolving multi-mapping ambiguity that occurs between genomically distinct loci.<br/><br/>The argument that most transcript-level analyses of interest may be addressed by looking at DGE in conjunction with DTU is an interesting one. It is certainly that case that not all tasks for which DTE is used actually require assessing differential expression at the transcript level. One issue with the DGE + DTU-based analysis which warrants further discussion in the manuscript is that I believe that this approach, too, would require correcting for multiple hypothesis testing. Specifically, one is testing both the DGE and the DTU hypotheses for each gene (or for a relevant subset of interest). The correction here is likely to be less harsh than in the case of assessing DTE, but is still worth discussing.<br/><br/>Minor comments:<br/><br/>As per Dr. Floors statement, Salmon (and Sailfish) also incorporate sequence-specific bias correction. Further, RSEM and Salmon (and a few other transcript-level abundance estimation tools) also incorporate the modeling of non-uniform fragment start position distributions. Of course, modeling a non-uniform start position distribution cannot overcome a complete lack of sampling in critical regions that might make determining transcript-level fragment assignment impossible, but it may help in properly apportioning an ambiguously-mapped fragment between transcripts depending on its relative position in each.<br/><br/>One potential added source of variability here is that all Salmon estimates presented in the manuscript make use of Salmons quasi-mapping of reads, while the STAR + featureCount pipeline makes use of traditional alignments. This is the primary intended usage mode of Salmon, and absolutely does represent a typical pipeline for methods that avoid alignment (Salmon, Sailfish, kallisto). However, it would probably be best to mention this as a potential (though likely negligible) additional source of variability.<br/><br/>In the discussion, the authors argue that ... it is therefore imperative to continue improving abundance estimation and inference methods applicable to individual transcripts, since misestimation can propagate to the gene level. This is, of course, an important and valid suggestion. Another direction, on which it would be useful to get the authors thoughts and suggestions, is the development of differential expression tools (at either the transcript or gene level) that can make use of the variance estimates that some tools (like Salmon) can provide. To what extent might incorporating this information help control false positive rates and improve DTE or even DGE estimates?</div><div class=u-mb--2></div><hr class="c-hr c-hr--low u-mb--2"><h5 class="t-body u-weight--md u-mt--0 u-mb--1/2">Competing Interests</h5><p class="u-mt--0 u-mb--2 u-italic">No competing interests were disclosed.</p><hr class="c-hr c-hr--low u-mb--3"><p class="u-mb--3 u-mt--0"><strong> I have read this submission. I believe that I have an appropriate level of expertise to confirm that it is of an acceptable scientific standard. </strong></p><div class="_mdl-layout u-mb--3"><a href="/login?originalPath=/articles/4-1521/v2&scrollTo=referee-response-11745" class="u-1/1 mdl-button mdl-js-button mdl-button--outline mdl-button--multi-line mdl-button--colored mdl-button--white-ish" data-test-id=article_referee-report_respond><i class="material-icons u-ib u-middle">reply</i><span class="p-article__sidebar-report-respond-text u-ib u-middle">Respond to this report</span></a></div><hr class="c-hr c-hr--low c-hr--md u-mb--3"><section id=report-responses-11745><h4 class="t-h3 u-weight--md u-mb--2 u-mt--0">Responses <span class="u-weight--rg u-black--medium t-body">(1)</span></h4><span class="c-hr c-hr--low u-mb--3"></span><article id=referee-report-comment-1820 class="c-comment js-referee-report-comment " data-commentid=1820 data-reportid=11745 data-test-id=article_referee_report_comment><header><div class="c-comment__role t-caption u-black--medium" data-test-id=article_referee-report-comment_roles><span class="u-weight--md u-upper u-ib u-mr--1/2">Author Response</span></div><div class="c-comment__date u-black--medium t-caption">29 Feb 2016</div><h5 class="c-comment__name t-body u-weight--rg u-mt--0 u-mb--1" data-test-id=article_referee-report-comment_name><strong class=u-weight--md>Mark Robinson, </strong><span class="c-comment__affiliations u-weight--rg">University of Zurich, Zurich, Switzerland</span></h5><div class="c-comment__body u-mb--1"><div class="c-read-more js-read-more t-body" data-lines=4><div class="c-read-more__content js-read-more-content "><div class=u-mb--1 data-test-id=article_referee-report-comment_text data-commentid=1820>Thank you for taking the time to read and review our paper. As per your suggestions, we have expanded the supplementary pdf document with equations defining each of the abundance measures that we used as well as a detailed description of the generation of the incomplete annotation files. We have also reworded the paragraph discussing the influence of an incomplete annotation catalog, and added further discussion points regarding multiple testing correction for DGE+DTU as well as regarding new methods incorporating variance estimates into the differential analysis. However, a deeper discussion of the latter point falls outside the scope of this article.<br/>&nbsp;<br/>We found your suggestion to restrict the abundance accuracy comparison to paralogous genes interesting, and the supplement has been expanded with several examples comparing the accuracy of gene-level abundance estimates from Salmon and from STAR+featureCounts, restricted to sets of paralogous genes. While we see a clear advantage of Salmon for many of the paralogous gene groups, the overall difference is only slightly larger for sets of paralogous genes compared to random sets of genes (as can be seen in the revised version of Dataset 1). In addition, we have added the following text to the relevant section in the results:<br/>&nbsp;<br/>...suggesting that the extent of the problem in many real data sets is limited... though some loss of sensitivity for certain genes may be encountered from discarding multi-mapping fragments, which may be recovered through the use of transcript abundance estimators such as Salmon.<br/>&nbsp;<br/>While we acknowledge that many of the popular transcript abundance estimation methods incorporate some type of bias estimation/correction (albeit not at the fragment level), and that it is definitely an important (and difficult) research area, a thorough discussion on the relative merits of different bias correction approaches is outside the scope of this paper. We have added citations to relevant literature on coverage bias, have simplified and clarified the last paragraph of the discussion, and have reworded the sentence about unidentifiable estimation due to coverage biases:<br/>&nbsp;<br/>While some extent of coverage variability might be alleviated by corrections for sequence- or position-specific biases, there remain cases where transcript expression cannot be inferred from data ( Figure 1C)</div></div><a href="#" class="c-read-more__toggle js-read-more-toggle t-caption"><span class=c-read-more__expand>View more</span><span class=c-read-more__contract>View less</span></a></div></div><div class=c-comment__footer><h5 class="t-body u-weight--md u-mt--0 u-mb--1/2">Competing Interests</h5><p class="u-mt--0 u-mb--1 u-italic" data-test-id=article_referee-report-comment_competing-interests>No competing interests were disclosed.</p></div><footer><div class="o-actions o-actions--middle"><div class=o-actions__primary><a href="/login?originalPath=/articles/4-1521/v2&scrollTo=referee-response-11745" class="js-referee-report-comment c-link--basic" data-test-id=article_referee-report-comment_respond><i class="material-icons c-link__icon">reply</i><span class=c-link__text>Respond</span></a></div><div class=o-actions__secondary><a href="mailto:research@f1000.com?subject=Report a concern (7563) " data-test-id=article_referee-report-comment_concern class="js-report-concern t-caption c-link c-link--inline">Report a concern</a></div></div></footer></header></article></section><script type="text/template" data-template=refereeReportCitation-11745 id=refereeReportCitation-11745 class=u-hide>
        <div class="u-black--high">
            <p id="referee-report-citation">Patro R. Referee Report For:  Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences [version 2; peer review: 2 approved]</span>. <i>F1000Research</i> 2016, <b>4</b>:1521 <br/>(<a class="new-orange" href="https://doi.org/10.5256/f1000research.8143.r11745" target="_blank">https://doi.org/10.5256/f1000research.8143.r11745)</a></p>

            <p class="c-modal__flush u-pl u-pr u-pt--3/2 u-pb--3/2 u-red u-bg--12"><strong>NOTE:</strong> it is important to ensure the information in square brackets after the title is included in this citation.</p>

            <p class="t-body">The direct URL for this report is:
                <br><a href="https://f1000research.com/articles/4-1521/v1#referee-response-11745" target="_blank">https://f1000research.com/articles/4-1521/v1#referee-response-11745</a></p>
        </div>
    </script></aside></div><div class="p-article__sidebar-view u-pl u-hide js-article-sidebar-view" data-view=report-11761><a class="u-block js-route c-link--basic p-article__sidebar-back" href="#" data-path="/"><i class="material-icons c-link__icon">keyboard_arrow_left</i><span class=c-link__text>Back to all reports</span></a><aside class="c-referee-report js-referee-report t-body u-pr--2 u-pb--8" data-reportid=11761 data-test-id=article_referee_report><div class=o-layout><header class="o-layout__item c-referee-report__header"><div class="o-actions o-actions--middle u-mb--1"><div class=o-actions__primary><h2 class="t-h3 u-weight--md u-mb--0 u-mt--0">Reviewer Report</h2></div><div class="o-actions__secondary u-hide@expanded"><span class="t-body u-weight--rg js-report-view-count u-ml--1 u-black--medium"><span class=js-referee-report-views>0</span> Views</span></div><div class="o-actions__secondary u-hide@expanded"><div class="c-block-tip p-article__sidebar-tooltip c-block-tip--below c-block-tip--small-arrow c-block-tip--sm-padding"><button type=button class="c-button c-button--icon c-button--text c-block-tip__toggle"><i class="material-icons c-button--icon__icon">copyright</i></button><div class="c-block-tip__content u-fs--normal"><span class=u-weight--bd> 2016 Floor S.</span> This is an open access peer review report distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" target=_blank>Creative Commons Attribution Licence</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. </div></div></div></div><p class="u-black--medium u-mt--0 u-mb--1"><em>04 Jan 2016 | for Version 1</em></p><div class="u-mb--1/2"><span class=bold>Stephen N. Floor<a href="http://orcid.org/0000-0002-9965-9694" target=_blank title="Open ORCID"><span class=orcid-logo-for-author-list></span></a></span><span class=f1r-article-desk-inline>,</span> Department of Molecular and Cell Biology, University of California, Berkeley, Berkeley, CA, USA&nbsp; </div><hr class="c-hr c-hr--low u-mt--2 u-mb--2 u-hide@expanded"></header><div class="o-layout__item c-referee-report__actions u-mb--2"><span class="t-body u-weight--rg js-report-view-count u-mt--0 u-mb--1 u-black--medium u-show@expanded"><span class=js-referee-report-views>0</span> Views</span><div id=copyrightInfo class="c-block-tip p-article__sidebar-tooltip c-block-tip--below c-block-tip--small-arrow u-show@expanded u-fr"><button type=button class="c-button c-button--icon c-button--text c-block-tip__toggle"><i class="material-icons c-button--icon__icon">copyright</i></button><div class="c-block-tip__content u-fs--normal"><span class=u-weight--bd> 2016 Floor S.</span> This is an open access peer review report distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" target=_blank>Creative Commons Attribution Licence</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. </div></div><a class="u-block c-link--basic u-mb--1 js-route" data-path="/reports/11761/cite" href="#" data-test-id=article_referee-report_cite-link><i class="material-icons c-link__icon">format_quote</i><span class="u-ml--1 c-link__text">Cite this report</span></a><a href="#report-responses-11761" class="u-block c-link--basic u-mb--1/2 js-route" data-path="/reports/11761/responses" href="#" data-test-id=article_referee-report_responses-link><i class="material-icons c-link__icon">speaker_notes</i><span class="u-ml--1 u-mr--1/2 c-link__text u-ib">Responses</span><span class="u-black--medium u-ib u-middle">(1)</span></a></div><div class="u-mb--1 u-show@expanded"></div></div><hr class="c-hr c-hr--low u-mt--0 u-mb--2"><div class="o-actions o-actions--middle u-mb--1"><div class="o-actions__primary p-article__color--dark"><i class="c-icn--f1r-icon c-icn--approved " title=Approved #data-refInfo=11706-11745></i><span class="p-article__report-status t-caption u-upper-spacing u-upper u-ib u-middle u-weight--md">Approved</span></div><div class="o-actions__secondary _mdl-layout"><div class="c-block-tip p-article__sidebar-tooltip c-block-tip--below c-block-tip--small-arrow c-block-tip--sm-padding"><button type=button class="c-button c-button--icon c-button--text c-block-tip__toggle"><i class="material-icons c-button--icon__icon">info_outline</i></button><div class=c-block-tip__content><p class="t-body u-mt--0 u-mb--0"><em class=u-weight--md>Alongside their report, reviewers assign a status to the article:</em></p><dl class=c-definitions><dt class="c-definitions__term u-upper t-control u-weight--md u-black--high"><span class="u-ib u-middle"><i class="c-icn--f1r-icon c-icn--approved " title=Approved #data-refInfo=""></i></span><span class="u-middle u-ib u-ml--1/2">Approved</span></dt><dd class="c-definitions__description t-caption">The paper is scientifically sound in its current form and only minor, if any, improvements are suggested</dd><dt class="c-definitions__term u-upper t-control u-weight--md u-black--high"><span class="u-ib u-middle"><i class="c-icn--f1r-icon c-icn--reservations" title="Approved with Reservations" #data-refInfo=""></i></span><span class="u-middle u-ib u-ml--1/2">Approved with reservations</span></dt><dd class="c-definitions__description t-caption">Key revisions are required to address specific details and make the paper fully scientifically sound</dd><dt class="c-definitions__term u-upper t-control u-weight--md u-black--high"><span class="u-ib u-middle"><i class="c-icn--f1r-icon c-icn--not-approved " title="Not Approved" #data-refInfo=""></i></span><span class="u-middle u-ib u-ml--1/2">Not approved</span></dt><dd class="c-definitions__description t-caption">Fundamental flaws in the paper seriously undermine the findings and conclusions</dd></dl></div></div></div></div><div class="">Soneson, Love and Robinson tackle a crucial question for analysis of RNA deep sequencing data in this manuscript: what is the role of transcript diversity in the accuracy and statistical power associated with measurements of gene expression? The authors make and convincingly show three claims: gene-level estimation and inferences are more robust than those at the transcript-level, and incorporating transcript-level quantification into gene-level abundance leads to improved differential expression testing.&nbsp; The claims are convincingly proven, the manuscript is well written, and the subject matter is of considerable interest. Furthermore, the described R package <i>tximport </i>should be of broad interest to the RNA deep sequencing community.<br/>&nbsp;<br/>Overall comments:<br/>&nbsp;<br/>It may be useful to indicate explicitly in the text that the methods are contained within the (excellently written and formatted) supplementary material, as this was not apparent. It might be clearest to create a specific methods section that just references supplementary file 1.<br/>&nbsp;<br/>The clarity of scatter plots with more than ~hundreds of points (e.g. Figure 1A) could be improved by using partially transparent points to visualize density.<br/>&nbsp;<br/>Introduction:<br/>&nbsp;<br/>Paragraph 1: Cufflinks, RSEM and Bitseq are grouped with kallisto and Salmon and it is then stated that some of these methods bypass read alignment. It would be clearer if this were reworded to avoid the ambiguity as to which methods avoid read alignment.<br/>&nbsp;<br/>Paragraph 4: The third claim could be presented more clearly. While it is interesting that simple counting performs similarly to transcript-level quantification procedures, it seems more interesting to this reviewer that incorporating transcript-level information improves the accuracy of differential expression testing at the gene level. Perhaps these two concepts can be combined into one more concise point?<br/>&nbsp;<br/>Results:<br/>&nbsp;<br/>The assertion that simple counts tended to show a high degree of robustness against incompleteness of the annotation catalog, as evidenced from estimation errors after first removing (at random) 20% of the transcripts seems misleading since Salmon-derived gene-level abundances actually show a higher Spearman correlation than count-derived gene-level abundances when subjected to removing a random 20% of transcripts. Figure 1a bottom left shows that transcript-level abundances are strongly affected by removal of 20% of transcripts, but that gene-level abundances are not strongly changed whether estimated using counts or Salmon. This statement should be reworded.<br/>&nbsp;<br/>Two concerns are raised about DTE. It is certainly true that reads are spread across more features when performing DTE as opposed to DGE.However, it is not apparent why analysis of DTE involves grouping of transcripts together for interpretation. DTE implies analysis at the transcript level and therefore no grouping, while DGE could involve some level of grouping of transcripts or quantification at the gene level from the start. The clarity of this could be improved.<br/>&nbsp;<br/>It is a very interesting idea to separately frame questions regarding DGE and DTU, which should be adopted widely, as the two are separable questions.<br/>&nbsp;<br/>The authors state one possible workflow towards DGE analysis in the section Incorporating transcript-level estimates leads to more accurate DGE results. Alternative pipelines (e.g. cuffdiff) could be presented in brief.<br/>&nbsp;<br/>The observation that simplesum and featureCounts results are highly correlated and therefore that statistical methods based on the Negative Binomial distribution can be used on estimated counts seems of greater importance than is emphasized in the text. This should be elaborated upon in the discussion, since this means that estimated counts from kallisto, express, salmon, etc can be used directly by statistical packages assuming a NB distribution (edgeR, DESeq2, etc). This point is frequently debated in discussions of how to rigorously analyze sequencing data. The conclusion here that NB applies to estimated counts is thus quite important.<br/>&nbsp;<br/>Please explain the meaning of the name for each curve in the legend for Figure 3 (i.e. specify that avetxl means using the offset corresponding to average transcript length.<br/>&nbsp;<br/>Discussion:<br/>&nbsp;<br/>The assertion that gene-level results are more accurate, powerful and interpretable than transcript-level results seems an oversimplification given the result that incorporating transcript-level quantification leads to improved DGE detection performance (e.g. Fig 3).<br/>&nbsp;<br/>Please cite at minimum Roberts <i>et al.</i>, (2011) regarding sequence bias correction as this has been implemented in cufflinks, express and kallisto. Other relevant papers should also be included here, as attempts have been made to address both positional and sequence-specific bias in RNA sequencing data.<br/>&nbsp;<br/>Supplement:<br/>&nbsp;<br/>The usability of the supplemental info could be improved by substituting rasterized for vectorized plots for those with ~hundreds of points.<br/>&nbsp;<br/>Please explain the meaning of the name for each curve in the legend for Supplemental Figure 5.</div><div class=u-mb--2></div><hr class="c-hr c-hr--low u-mb--2"><h5 class="t-body u-weight--md u-mt--0 u-mb--1/2"><b>References</b></h5><div class=u-mb--2><a name=rep-ref-11761-1></a>1. Roberts A, Trapnell C, Donaghey J, Rinn JL, Pachter L: Improving RNA-Seq expression estimates by correcting for fragment bias.<i>Genome Biol</i>. 2011; <b>12</b> (3): R22 <a target=_blank href="http://www.ncbi.nlm.nih.gov/pubmed/21410973">PubMed Abstract</a> | <a target=_blank href="https://doi.org/10.1186/gb-2011-12-3-r22">Publisher Full Text</a><br></div><hr class="c-hr c-hr--low u-mb--2"><h5 class="t-body u-weight--md u-mt--0 u-mb--1/2">Competing Interests</h5><p class="u-mt--0 u-mb--2 u-italic">No competing interests were disclosed.</p><hr class="c-hr c-hr--low u-mb--3"><p class="u-mb--3 u-mt--0"><strong> I have read this submission. I believe that I have an appropriate level of expertise to confirm that it is of an acceptable scientific standard. </strong></p><div class="_mdl-layout u-mb--3"><a href="/login?originalPath=/articles/4-1521/v2&scrollTo=referee-response-11761" class="u-1/1 mdl-button mdl-js-button mdl-button--outline mdl-button--multi-line mdl-button--colored mdl-button--white-ish" data-test-id=article_referee-report_respond><i class="material-icons u-ib u-middle">reply</i><span class="p-article__sidebar-report-respond-text u-ib u-middle">Respond to this report</span></a></div><hr class="c-hr c-hr--low c-hr--md u-mb--3"><section id=report-responses-11761><h4 class="t-h3 u-weight--md u-mb--2 u-mt--0">Responses <span class="u-weight--rg u-black--medium t-body">(1)</span></h4><span class="c-hr c-hr--low u-mb--3"></span><article id=referee-report-comment-1821 class="c-comment js-referee-report-comment " data-commentid=1821 data-reportid=11761 data-test-id=article_referee_report_comment><header><div class="c-comment__role t-caption u-black--medium" data-test-id=article_referee-report-comment_roles><span class="u-weight--md u-upper u-ib u-mr--1/2">Author Response</span></div><div class="c-comment__date u-black--medium t-caption">29 Feb 2016</div><h5 class="c-comment__name t-body u-weight--rg u-mt--0 u-mb--1" data-test-id=article_referee-report-comment_name><strong class=u-weight--md>Mark Robinson, </strong><span class="c-comment__affiliations u-weight--rg">University of Zurich, Zurich, Switzerland</span></h5><div class="c-comment__body u-mb--1"><div class="c-read-more js-read-more t-body" data-lines=4><div class="c-read-more__content js-read-more-content "><div class=u-mb--1 data-test-id=article_referee-report-comment_text data-commentid=1821>Thank you for taking the time to read and review our paper. In the revised version, we have improved the clarity and usability of the figures and the supplement by using partially transparent points and extending the figure captions. We have also modified the main text to refer more clearly to the method information provided in the supplementary material, and improved the clarity of the text in several places, according to your suggestions.<br/>&nbsp;<br/>For additional comments regarding references to bias correction studies, please see responses to Rob Patro.</div></div><a href="#" class="c-read-more__toggle js-read-more-toggle t-caption"><span class=c-read-more__expand>View more</span><span class=c-read-more__contract>View less</span></a></div></div><div class=c-comment__footer><h5 class="t-body u-weight--md u-mt--0 u-mb--1/2">Competing Interests</h5><p class="u-mt--0 u-mb--1 u-italic" data-test-id=article_referee-report-comment_competing-interests>No competing interests were disclosed.</p></div><footer><div class="o-actions o-actions--middle"><div class=o-actions__primary><a href="/login?originalPath=/articles/4-1521/v2&scrollTo=referee-response-11761" class="js-referee-report-comment c-link--basic" data-test-id=article_referee-report-comment_respond><i class="material-icons c-link__icon">reply</i><span class=c-link__text>Respond</span></a></div><div class=o-actions__secondary><a href="mailto:research@f1000.com?subject=Report a concern (7563) " data-test-id=article_referee-report-comment_concern class="js-report-concern t-caption c-link c-link--inline">Report a concern</a></div></div></footer></header></article></section><script type="text/template" data-template=refereeReportCitation-11761 id=refereeReportCitation-11761 class=u-hide>
        <div class="u-black--high">
            <p id="referee-report-citation">Floor SN. Referee Report For:  Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences [version 2; peer review: 2 approved]</span>. <i>F1000Research</i> 2016, <b>4</b>:1521 <br/>(<a class="new-orange" href="https://doi.org/10.5256/f1000research.8143.r11761" target="_blank">https://doi.org/10.5256/f1000research.8143.r11761)</a></p>

            <p class="c-modal__flush u-pl u-pr u-pt--3/2 u-pb--3/2 u-red u-bg--12"><strong>NOTE:</strong> it is important to ensure the information in square brackets after the title is included in this citation.</p>

            <p class="t-body">The direct URL for this report is:
                <br><a href="https://f1000research.com/articles/4-1521/v1#referee-response-11761" target="_blank">https://f1000research.com/articles/4-1521/v1#referee-response-11761</a></p>
        </div>
    </script></aside></div></div><script src="/js/shared_scripts/modal-dialogue.js"></script><script src="/js/shared_scripts/read-more.js"></script><script src="/js/article/article-router.js"></script><script src="/js/article/article-sidebar.js"></script><script src="/js/referee/new/referee_helpers.js"></script><script src="/js/article/article-column-toggle.js"></script></div></div></div></main><input type=hidden id=_articleVersionUrl value="https://f1000research.com/articles/4-1521/v2/"><div class=research-help id=about-referee-status><div class="research-layout research-help-content about-referee-status"><span class="close-research-help dark-cross" title=Close></span> Alongside their report, reviewers assign a status to the article: <div class="cf research-help-row"><span class="f1r-icon icon-86_approved status-green smaller" title=Approved></span><span class=research-help-text>Approved - the paper is scientifically sound in its current form and only minor, if any, improvements are suggested</span></div><div class="cf research-help-row"><span class="f1r-icon icon-87_approved_reservations status-green smaller" title="Approved with Reservations"></span><span class=research-help-text>Approved with reservations - key revisions are required to address specific details and make the paper fully scientifically sound</span></div><div class="cf research-help-row"><span class="f1r-icon icon-88_not_approved status-red small" title="Not Approved"></span><span class=research-help-text>Not approved - fundamental flaws in the paper seriously undermine the findings and conclusions</span></div></div></div><div id=datasets-info class=is-hidden><div id=dataset-DS0 plottable=false></div><div id=dataset-doi-DS0><p class=is-hidden><a href="https://f1000researchdata.s3.amazonaws.com/datasets/7563/4cc0b0d8-4853-4896-b66e-a4715dd3bf7b_sim1-quantification.html" class=link-for-dataset>Click here</a> to access the data.</p><div class=clearfix></div><div class=generated-resource-link-wrapper style="text-align: right;"><div class="modal-window-wrapper is-hidden research-layout"><div id=dataset-help-0 class="modal-window dataset-help-window"><span class=modal-window-close-button></span><h2 class=h2-title>The problem</h2><p>Spreadsheet data files may not format correctly if your computer is using different default delimiters (symbols used to separate values into separate cells) - a spreadsheet created in one region is sometimes misinterpreted by computers in other regions. You can change the regional settings on your computer so that the spreadsheet can be interpreted correctly.</p><h2 class=h2-title>How to fix it</h2><ul class="numbered-list no-padding"><li class=standard-padding>Save downloaded CSV file</li><li class=standard-padding>Open spreadsheet program (e.g. Excel)</li><li class=standard-padding>Click the &lsquo;Data&rsquo; tab at the top</li><li class=standard-padding>Click the &lsquo;From text&rsquo; icon (top left)</li><li class=standard-padding>Browse for downloaded CSV file, click &lsquo;Import&rsquo;</li><li class=standard-padding>Ensure &lsquo;Delimited&rsquo; radio button is selected, click &lsquo;Next&rsquo;</li><li class=standard-padding>Check one of the appropriate delimiter checkboxes (you can visualize the formatting by looking at the data preview below these options)</li><li class=standard-padding>Click &lsquo;Finish&rsquo;</li></ul></div></div><a href="#" class=download-help-link data-popup-id=dataset-help-0>Downloaded data do not display as expected?</a><a href="https://f1000researchdata.s3.amazonaws.com/datasets/7563/4cc0b0d8-4853-4896-b66e-a4715dd3bf7b_sim1-quantification.html" class="link-for-dataset general-white-orange-button with-hover margin-right"><span class="f1r-icon icon-76_download_file"></span>Download the data</a><div class="dataset-cite-article-popup-wrapper dataset-file"><div class=cite-dataset></div><div id=cite-dataset0-popup class="margin-top popup-window-wrapper is-hidden"><div class=cite-popup-background></div><div class="popup-window cite-this-dataset"><div class="popup-window-title medium">Dataset citation:</div> Soneson C, Love MI and Robinson MD. Dataset 1 in: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences. F1000Research 2016, <span class=bold>4</span>:1521 (https://doi.org/10.5256/f1000research.7563.d114722) <div class="float-right margin-top"><button class="general-white-orange-button close-cite-dataset-window">Close</button></div></div></div></div></div></div><div id=dataset-DS1 plottable=false></div><div id=dataset-doi-DS1><p class=is-hidden><a href="https://f1000researchdata.s3.amazonaws.com/datasets/7563/4197990b-04cf-48d9-832d-6a18cbd575af_sim2-quantification.html" class=link-for-dataset>Click here</a> to access the data.</p><div class=clearfix></div><div class=generated-resource-link-wrapper style="text-align: right;"><div class="modal-window-wrapper is-hidden research-layout"><div id=dataset-help-1 class="modal-window dataset-help-window"><span class=modal-window-close-button></span><h2 class=h2-title>The problem</h2><p>Spreadsheet data files may not format correctly if your computer is using different default delimiters (symbols used to separate values into separate cells) - a spreadsheet created in one region is sometimes misinterpreted by computers in other regions. You can change the regional settings on your computer so that the spreadsheet can be interpreted correctly.</p><h2 class=h2-title>How to fix it</h2><ul class="numbered-list no-padding"><li class=standard-padding>Save downloaded CSV file</li><li class=standard-padding>Open spreadsheet program (e.g. Excel)</li><li class=standard-padding>Click the &lsquo;Data&rsquo; tab at the top</li><li class=standard-padding>Click the &lsquo;From text&rsquo; icon (top left)</li><li class=standard-padding>Browse for downloaded CSV file, click &lsquo;Import&rsquo;</li><li class=standard-padding>Ensure &lsquo;Delimited&rsquo; radio button is selected, click &lsquo;Next&rsquo;</li><li class=standard-padding>Check one of the appropriate delimiter checkboxes (you can visualize the formatting by looking at the data preview below these options)</li><li class=standard-padding>Click &lsquo;Finish&rsquo;</li></ul></div></div><a href="#" class=download-help-link data-popup-id=dataset-help-1>Downloaded data do not display as expected?</a><a href="https://f1000researchdata.s3.amazonaws.com/datasets/7563/4197990b-04cf-48d9-832d-6a18cbd575af_sim2-quantification.html" class="link-for-dataset general-white-orange-button with-hover margin-right"><span class="f1r-icon icon-76_download_file"></span>Download the data</a><div class="dataset-cite-article-popup-wrapper dataset-file"><div class=cite-dataset></div><div id=cite-dataset1-popup class="margin-top popup-window-wrapper is-hidden"><div class=cite-popup-background></div><div class="popup-window cite-this-dataset"><div class="popup-window-title medium">Dataset citation:</div> Soneson C, Love MI and Robinson MD. Dataset 2 in: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences. F1000Research 2016, <span class=bold>4</span>:1521 (https://doi.org/10.5256/f1000research.7563.d114723) <div class="float-right margin-top"><button class="general-white-orange-button close-cite-dataset-window">Close</button></div></div></div></div></div></div><div id=dataset-DS2 plottable=false></div><div id=dataset-doi-DS2><p class=is-hidden><a href="https://f1000researchdata.s3.amazonaws.com/datasets/7563/5c81409a-b7af-47a2-9a47-d46fec5a7603_bottomly-quantification.html" class=link-for-dataset>Click here</a> to access the data.</p><div class=clearfix></div><div class=generated-resource-link-wrapper style="text-align: right;"><div class="modal-window-wrapper is-hidden research-layout"><div id=dataset-help-2 class="modal-window dataset-help-window"><span class=modal-window-close-button></span><h2 class=h2-title>The problem</h2><p>Spreadsheet data files may not format correctly if your computer is using different default delimiters (symbols used to separate values into separate cells) - a spreadsheet created in one region is sometimes misinterpreted by computers in other regions. You can change the regional settings on your computer so that the spreadsheet can be interpreted correctly.</p><h2 class=h2-title>How to fix it</h2><ul class="numbered-list no-padding"><li class=standard-padding>Save downloaded CSV file</li><li class=standard-padding>Open spreadsheet program (e.g. Excel)</li><li class=standard-padding>Click the &lsquo;Data&rsquo; tab at the top</li><li class=standard-padding>Click the &lsquo;From text&rsquo; icon (top left)</li><li class=standard-padding>Browse for downloaded CSV file, click &lsquo;Import&rsquo;</li><li class=standard-padding>Ensure &lsquo;Delimited&rsquo; radio button is selected, click &lsquo;Next&rsquo;</li><li class=standard-padding>Check one of the appropriate delimiter checkboxes (you can visualize the formatting by looking at the data preview below these options)</li><li class=standard-padding>Click &lsquo;Finish&rsquo;</li></ul></div></div><a href="#" class=download-help-link data-popup-id=dataset-help-2>Downloaded data do not display as expected?</a><a href="https://f1000researchdata.s3.amazonaws.com/datasets/7563/5c81409a-b7af-47a2-9a47-d46fec5a7603_bottomly-quantification.html" class="link-for-dataset general-white-orange-button with-hover margin-right"><span class="f1r-icon icon-76_download_file"></span>Download the data</a><div class="dataset-cite-article-popup-wrapper dataset-file"><div class=cite-dataset></div><div id=cite-dataset2-popup class="margin-top popup-window-wrapper is-hidden"><div class=cite-popup-background></div><div class="popup-window cite-this-dataset"><div class="popup-window-title medium">Dataset citation:</div> Soneson C, Love MI and Robinson MD. Dataset 3 in: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences. F1000Research 2016, <span class=bold>4</span>:1521 (https://doi.org/10.5256/f1000research.7563.d114724) <div class="float-right margin-top"><button class="general-white-orange-button close-cite-dataset-window">Close</button></div></div></div></div></div></div><div id=dataset-DS3 plottable=false></div><div id=dataset-doi-DS3><p class=is-hidden><a href="https://f1000researchdata.s3.amazonaws.com/datasets/7563/e22fa771-ee2b-486a-8cf4-31b645163d6b_gse64570_quantification.html" class=link-for-dataset>Click here</a> to access the data.</p><div class=clearfix></div><div class=generated-resource-link-wrapper style="text-align: right;"><div class="modal-window-wrapper is-hidden research-layout"><div id=dataset-help-3 class="modal-window dataset-help-window"><span class=modal-window-close-button></span><h2 class=h2-title>The problem</h2><p>Spreadsheet data files may not format correctly if your computer is using different default delimiters (symbols used to separate values into separate cells) - a spreadsheet created in one region is sometimes misinterpreted by computers in other regions. You can change the regional settings on your computer so that the spreadsheet can be interpreted correctly.</p><h2 class=h2-title>How to fix it</h2><ul class="numbered-list no-padding"><li class=standard-padding>Save downloaded CSV file</li><li class=standard-padding>Open spreadsheet program (e.g. Excel)</li><li class=standard-padding>Click the &lsquo;Data&rsquo; tab at the top</li><li class=standard-padding>Click the &lsquo;From text&rsquo; icon (top left)</li><li class=standard-padding>Browse for downloaded CSV file, click &lsquo;Import&rsquo;</li><li class=standard-padding>Ensure &lsquo;Delimited&rsquo; radio button is selected, click &lsquo;Next&rsquo;</li><li class=standard-padding>Check one of the appropriate delimiter checkboxes (you can visualize the formatting by looking at the data preview below these options)</li><li class=standard-padding>Click &lsquo;Finish&rsquo;</li></ul></div></div><a href="#" class=download-help-link data-popup-id=dataset-help-3>Downloaded data do not display as expected?</a><a href="https://f1000researchdata.s3.amazonaws.com/datasets/7563/e22fa771-ee2b-486a-8cf4-31b645163d6b_gse64570_quantification.html" class="link-for-dataset general-white-orange-button with-hover margin-right"><span class="f1r-icon icon-76_download_file"></span>Download the data</a><div class="dataset-cite-article-popup-wrapper dataset-file"><div class=cite-dataset></div><div id=cite-dataset3-popup class="margin-top popup-window-wrapper is-hidden"><div class=cite-popup-background></div><div class="popup-window cite-this-dataset"><div class="popup-window-title medium">Dataset citation:</div> Soneson C, Love MI and Robinson MD. Dataset 4 in: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences. F1000Research 2016, <span class=bold>4</span>:1521 (https://doi.org/10.5256/f1000research.7563.d114725) <div class="float-right margin-top"><button class="general-white-orange-button close-cite-dataset-window">Close</button></div></div></div></div></div></div><div id=dataset-DS4 plottable=false></div><div id=dataset-doi-DS4><p class=is-hidden><a href="https://f1000researchdata.s3.amazonaws.com/datasets/7563/080b3afa-c02e-407a-b08b-3a1e84aff2db_gse69244_quantification.html" class=link-for-dataset>Click here</a> to access the data.</p><div class=clearfix></div><div class=generated-resource-link-wrapper style="text-align: right;"><div class="modal-window-wrapper is-hidden research-layout"><div id=dataset-help-4 class="modal-window dataset-help-window"><span class=modal-window-close-button></span><h2 class=h2-title>The problem</h2><p>Spreadsheet data files may not format correctly if your computer is using different default delimiters (symbols used to separate values into separate cells) - a spreadsheet created in one region is sometimes misinterpreted by computers in other regions. You can change the regional settings on your computer so that the spreadsheet can be interpreted correctly.</p><h2 class=h2-title>How to fix it</h2><ul class="numbered-list no-padding"><li class=standard-padding>Save downloaded CSV file</li><li class=standard-padding>Open spreadsheet program (e.g. Excel)</li><li class=standard-padding>Click the &lsquo;Data&rsquo; tab at the top</li><li class=standard-padding>Click the &lsquo;From text&rsquo; icon (top left)</li><li class=standard-padding>Browse for downloaded CSV file, click &lsquo;Import&rsquo;</li><li class=standard-padding>Ensure &lsquo;Delimited&rsquo; radio button is selected, click &lsquo;Next&rsquo;</li><li class=standard-padding>Check one of the appropriate delimiter checkboxes (you can visualize the formatting by looking at the data preview below these options)</li><li class=standard-padding>Click &lsquo;Finish&rsquo;</li></ul></div></div><a href="#" class=download-help-link data-popup-id=dataset-help-4>Downloaded data do not display as expected?</a><a href="https://f1000researchdata.s3.amazonaws.com/datasets/7563/080b3afa-c02e-407a-b08b-3a1e84aff2db_gse69244_quantification.html" class="link-for-dataset general-white-orange-button with-hover margin-right"><span class="f1r-icon icon-76_download_file"></span>Download the data</a><div class="dataset-cite-article-popup-wrapper dataset-file"><div class=cite-dataset></div><div id=cite-dataset4-popup class="margin-top popup-window-wrapper is-hidden"><div class=cite-popup-background></div><div class="popup-window cite-this-dataset"><div class="popup-window-title medium">Dataset citation:</div> Soneson C, Love MI and Robinson MD. Dataset 5 in: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences. F1000Research 2016, <span class=bold>4</span>:1521 (https://doi.org/10.5256/f1000research.7563.d114726) <div class="float-right margin-top"><button class="general-white-orange-button close-cite-dataset-window">Close</button></div></div></div></div></div></div><div id=dataset-DS5 plottable=false></div><div id=dataset-doi-DS5><p class=is-hidden><a href="https://f1000researchdata.s3.amazonaws.com/datasets/7563/b0434f4d-253a-4998-8f44-ae109a76678c_gse72165_quantification.html" class=link-for-dataset>Click here</a> to access the data.</p><div class=clearfix></div><div class=generated-resource-link-wrapper style="text-align: right;"><div class="modal-window-wrapper is-hidden research-layout"><div id=dataset-help-5 class="modal-window dataset-help-window"><span class=modal-window-close-button></span><h2 class=h2-title>The problem</h2><p>Spreadsheet data files may not format correctly if your computer is using different default delimiters (symbols used to separate values into separate cells) - a spreadsheet created in one region is sometimes misinterpreted by computers in other regions. You can change the regional settings on your computer so that the spreadsheet can be interpreted correctly.</p><h2 class=h2-title>How to fix it</h2><ul class="numbered-list no-padding"><li class=standard-padding>Save downloaded CSV file</li><li class=standard-padding>Open spreadsheet program (e.g. Excel)</li><li class=standard-padding>Click the &lsquo;Data&rsquo; tab at the top</li><li class=standard-padding>Click the &lsquo;From text&rsquo; icon (top left)</li><li class=standard-padding>Browse for downloaded CSV file, click &lsquo;Import&rsquo;</li><li class=standard-padding>Ensure &lsquo;Delimited&rsquo; radio button is selected, click &lsquo;Next&rsquo;</li><li class=standard-padding>Check one of the appropriate delimiter checkboxes (you can visualize the formatting by looking at the data preview below these options)</li><li class=standard-padding>Click &lsquo;Finish&rsquo;</li></ul></div></div><a href="#" class=download-help-link data-popup-id=dataset-help-5>Downloaded data do not display as expected?</a><a href="https://f1000researchdata.s3.amazonaws.com/datasets/7563/b0434f4d-253a-4998-8f44-ae109a76678c_gse72165_quantification.html" class="link-for-dataset general-white-orange-button with-hover margin-right"><span class="f1r-icon icon-76_download_file"></span>Download the data</a><div class="dataset-cite-article-popup-wrapper dataset-file"><div class=cite-dataset></div><div id=cite-dataset5-popup class="margin-top popup-window-wrapper is-hidden"><div class=cite-popup-background></div><div class="popup-window cite-this-dataset"><div class="popup-window-title medium">Dataset citation:</div> Soneson C, Love MI and Robinson MD. Dataset 6 in: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences. F1000Research 2016, <span class=bold>4</span>:1521 (https://doi.org/10.5256/f1000research.7563.d114730) <div class="float-right margin-top"><button class="general-white-orange-button close-cite-dataset-window">Close</button></div></div></div></div></div></div></div><div class=article-interactive-content-container style="display: none;"><a name=f-template class=n-a></a><div class="interactive-content-wrapper padding-20"><img src="" class=interactive-content-image title="Open the interactive image display"><div class=interactive-content-title></div><div class=interactive-content-text></div><div class="f1r-article-desk interactive-content-ribbon" data-interactive-content-type=R-Script><div class=interactive-content-label>Adjust parameters to alter display</div><div class=interactive-content-button></div></div><div class="f1r-article-mobile mobile-interactive-note"> View on desktop for interactive features <img src="/img/icon/interactive_content.png" class="float-right margin-right-40"/></div><div class=clearfix></div></div></div><div id=article-interactive-omero-container class=article-interactive-omero-container style="display: none;"><div class=interactive-content-wrapper><div class="interactive-omero-button omero-content" title="Open the interactive content window." data-interactive-content-type=Omero></div><div class=has-interactive-content-image><span class=box-arrow></span><span class=box-middle>Includes Interactive Elements</span><span class=box-end></span></div><div class="fig panel clearfix" style="margin: 0; padding-bottom: 20px;"><a name=templatelink class=n-a></a><a target=_blank href="" class=link-for-omero-image><img src="" class=interactive-omero-image title="Open the image display window."></a><div class=caption><div class=interactive-content-title></div><div class=interactive-content-text></div></div><div class="is-hidden omero-image-list"></div></div><div class="f1r-article-mobile mobile-interactive-note omero"> View on desktop for interactive features <img src="/img/icon/interactive_content.png" class="float-right margin-right-40"/></div><div class=clearfix></div></div></div><div class="add-comment-container shadow-box is-hidden" id=save-comment-container><span id=save-comment-text class=intro-text>Edit comment</span><textarea id=new-comment name=new-comment class="global-textarea comment margin-bottom margin-top"></textarea><p><strong>Competing Interests</strong></p><textarea id=new-competing-interests name=competing-interests class="global-textarea competing-interests margin-bottom check-xss"></textarea><div class=clearfix></div><button id=cancelComment type=button class="general-white-orange-button float-right no-background-button margin-left"> Cancel </button><button id=save-comment-button commentId="" type=button class="general-white-orange-button float-right"> Save </button><div class=clearfix></div><div class="green-message margin-top is-hidden comment-is-saved">The comment has been saved.</div><div class="red-message margin-top is-hidden comment-not-added">An error has occurred. Please try again.</div><div class="red-message margin-top is-hidden comment-enter-text ucf">Your must enter a comment.</div><div class="red-message margin-top is-hidden comment-references-error references">References error.</div></div><div class="modal-window-wrapper is-hidden"><div id=conflicts-interests class="modal-window padding-20"><div class=modal-window__content><h2 class=h2-title>Competing Interests Policy</h2><p> Provide sufficient details of any financial or non-financial competing interests to enable users to assess whether your comments might lead a reasonable person to question your impartiality. Consider the following examples, but note that this is not an exhaustive list: </p><div class=heading5>Examples of 'Non-Financial Competing Interests'</div><ol class="numbered-list no-padding"><li class=standard-padding>Within the past 4 years, you have held joint grants, published or collaborated with any of the authors of the selected paper.</li><li class=standard-padding>You have a close personal relationship (e.g. parent, spouse, sibling, or domestic partner) with any of the authors.</li><li class=standard-padding>You are a close professional associate of any of the authors (e.g. scientific mentor, recent student).</li><li class=standard-padding>You work at the same institute as any of the authors.</li><li class=standard-padding>You hope/expect to benefit (e.g. favour or employment) as a result of your submission.</li><li class=standard-padding>You are an Editor for the journal in which the article is published.</li></ol><div class="heading5 padding-top">Examples of 'Financial Competing Interests'</div><ol class="numbered-list no-padding"><li class=standard-padding>You expect to receive, or in the past 4 years have received, any of the following from any commercial organisation that may gain financially from your submission: a salary, fees, funding, reimbursements.</li><li class=standard-padding>You expect to receive, or in the past 4 years have received, shared grant support or other funding with any of the authors.</li><li class=standard-padding>You hold, or are currently applying for, any patents or significant stocks/shares relating to the subject matter of the paper you are commenting on.</li></ol></div><div class="f1r-article-mobile research-layout"><span class=modal-window-close-button></span></div></div></div><div class="o-modal c-email-alert-popup js-email-alert-popup is-hidden"><div class="o-modal__body js-modal-body c-email-alert-popup__inner"><h1 class=c-email-alert-popup__title>Stay Updated</h1><p class=c-email-alert-popup__sub>Sign up for content alerts and receive a weekly or monthly email with all newly published articles</p><p><a href="/register?originalPath=" class="c-email-alert-popup__lnk c-email-alert-popup__button js-email-alert-popup-action">Register with F1000Research</a></p><p>Already registered? <a href="/login?originalPath=" class="c-email-alert-popup__lnk js-email-alert-popup-action">Sign in</a></p><p class=c-email-alert-popup__footer><a class="c-email-alert-popup__lnk js-email-alert-popup-cancel" href="#">Not now, thanks</a></p></div></div><div id=addCommentModal role=dialog aria-labelledby=addCommentModal_title aria-describedby=addCommentModal_description><div class="c-modal js-modal is-closed p-article__add-comment-modal c-modal--xlarge js-article-comment-modal c-modal--scroll "><aside class="c-modal__content u-black--high o-box u-bg--2 c-modal--scroll-always "><div class=c-modal__close><button type=button class="c-button c-button--icon c-button--medium c-button--full@hover js-modal__close"><i class="c-button--icon__icon material-icons">close</i></button></div><div id=addCommentModal_description class="c-modal__description js-modal__content t-h4 u-mt--0 u-mb--2 u-black--medium"><div class="js-add-comment-container t-caption u-black--high"><div class=u-weight--bd>PLEASE NOTE</div><div class=u-mt--1> If you are an <span class=u-weight--bd>AUTHOR</span> or <span class=u-weight--bd>REVIEWER</span> of this article, your &ldquo;User Comment&rdquo; will be automatically labelled Author Response or Reviewer Response <span class=u-weight--bd>if you are signed in with the account associated with this article</span>, otherwise your comment will be labelled &ldquo;Reader comment&rdquo;. </div><div class=u-mt--1> Additionally, if you are a <span class=u-weight--bd>REVIEWER</span> of this article, please go to your <a href="/my/referee">account</a> to submit your report, do not post it as a user comment. </div><div class="u-mt--1 u-mb--1"> If you do not have access to your original account, please <a href="mailto:research@f1000.com">contact us</a>. </div><div class="comments-note margin-bottom" id=accept-user-comments><input class=js-add-comment-accept-terms type=checkbox id=acceptedTermsAndConditions name=acceptedTermsAndConditions> I accept the <a href="/about/legal/usercommenttermsandconditions" target=_blank> User Comment Terms and Conditions</a><span class=required>&nbsp;</span></div><div class="default-error margin-top is-hidden comment-accept-conditions utac">Please confirm that you accept the User Comment Terms and Conditions.</div><p class=no-top-margin>All commenters must hold a formal affiliation as per our <a href="/about/policies#commentspolicy" target=_blank>Policies</a>. The information that you give us will be displayed next to your comment.</p><p>User comments must be in English, comprehensible and relevant to the article under discussion. We reserve the right to remove any comments that we consider to be inappropriate, offensive or otherwise in breach of the <a href="/about/legal/usercommenttermsandconditions" target=_blank>User Comment Terms and Conditions</a>. Commenters must not use a comment for personal attacks. When criticisms of the article are based on unpublished data, the data should be made available.</p><div class="research-layout registration-form u-mb--2"><div class="u-mb--1 u-mt--2"><strong>Affiliation</strong></div><div class=form-field><input type=text name=institution class="form-input-field check-xss js-add-comment-institution" placeholder="Organization *" autocomplete=off /><div class="default-error margin-top is-hidden comment-enter-institution institution">Please enter your organisation.</div></div><div class=form-field><input type=text name=place class="form-input-field check-xss js-add-comment-place" placeholder=Place></div><div class=form-field><div class="form-input-wrapper hundred-percent-wide"><div class="new-select-standard-wrapper half-width inline-display heading10"><div class="selected-option no-bg submission-form"></div><div class="faux-options-container is-closed"><div class=faux-options></div></div><div class="toggle-button arrow-toggle-modern f1r-icon icon-"></div><select id=country name=countryId class="is-hidden js-add-comment-country" autocomplete=off><option value=-1>Country *</option><option value=840>USA</option><option value=826>UK</option><option value=124>Canada</option><option value=156>China</option><option value=250>France</option><option value=276>Germany</option><optgroup label=-----------------------------------------------></optgroup><option value=4>Afghanistan</option><option value=248>Aland Islands</option><option value=8>Albania</option><option value=12>Algeria</option><option value=16>American Samoa</option><option value=20>Andorra</option><option value=24>Angola</option><option value=660>Anguilla</option><option value=10>Antarctica</option><option value=28>Antigua and Barbuda</option><option value=32>Argentina</option><option value=51>Armenia</option><option value=533>Aruba</option><option value=36>Australia</option><option value=40>Austria</option><option value=31>Azerbaijan</option><option value=44>Bahamas</option><option value=48>Bahrain</option><option value=50>Bangladesh</option><option value=52>Barbados</option><option value=112>Belarus</option><option value=56>Belgium</option><option value=84>Belize</option><option value=204>Benin</option><option value=60>Bermuda</option><option value=64>Bhutan</option><option value=68>Bolivia</option><option value=70>Bosnia and Herzegovina</option><option value=72>Botswana</option><option value=74>Bouvet Island</option><option value=76>Brazil</option><option value=86>British Indian Ocean Territory</option><option value=92>British Virgin Islands</option><option value=96>Brunei</option><option value=100>Bulgaria</option><option value=854>Burkina Faso</option><option value=108>Burundi</option><option value=116>Cambodia</option><option value=120>Cameroon</option><option value=124>Canada</option><option value=132>Cape Verde</option><option value=136>Cayman Islands</option><option value=140>Central African Republic</option><option value=148>Chad</option><option value=152>Chile</option><option value=156>China</option><option value=162>Christmas Island</option><option value=166>Cocos (Keeling) Islands</option><option value=170>Colombia</option><option value=174>Comoros</option><option value=178>Congo</option><option value=184>Cook Islands</option><option value=188>Costa Rica</option><option value=384>Cote d'Ivoire</option><option value=191>Croatia</option><option value=192>Cuba</option><option value=196>Cyprus</option><option value=203>Czech Republic</option><option value=180>Democratic Republic of the Congo</option><option value=208>Denmark</option><option value=262>Djibouti</option><option value=212>Dominica</option><option value=214>Dominican Republic</option><option value=218>Ecuador</option><option value=818>Egypt</option><option value=222>El Salvador</option><option value=226>Equatorial Guinea</option><option value=232>Eritrea</option><option value=233>Estonia</option><option value=231>Ethiopia</option><option value=238>Falkland Islands</option><option value=234>Faroe Islands</option><option value=583>Federated States of Micronesia</option><option value=242>Fiji</option><option value=246>Finland</option><option value=250>France</option><option value=254>French Guiana</option><option value=258>French Polynesia</option><option value=260>French Southern Territories</option><option value=266>Gabon</option><option value=268>Georgia</option><option value=276>Germany</option><option value=288>Ghana</option><option value=292>Gibraltar</option><option value=300>Greece</option><option value=304>Greenland</option><option value=308>Grenada</option><option value=312>Guadeloupe</option><option value=316>Guam</option><option value=320>Guatemala</option><option value=831>Guernsey</option><option value=324>Guinea</option><option value=624>Guinea-Bissau</option><option value=328>Guyana</option><option value=332>Haiti</option><option value=334>Heard Island and Mcdonald Islands</option><option value=336>Holy See (Vatican City State)</option><option value=340>Honduras</option><option value=344>Hong Kong</option><option value=348>Hungary</option><option value=352>Iceland</option><option value=356>India</option><option value=360>Indonesia</option><option value=364>Iran</option><option value=368>Iraq</option><option value=372>Ireland</option><option value=376>Israel</option><option value=380>Italy</option><option value=388>Jamaica</option><option value=392>Japan</option><option value=832>Jersey</option><option value=400>Jordan</option><option value=398>Kazakhstan</option><option value=404>Kenya</option><option value=296>Kiribati</option><option value=901>Kosovo (Serbia and Montenegro)</option><option value=414>Kuwait</option><option value=417>Kyrgyzstan</option><option value=418>Lao People's Democratic Republic</option><option value=428>Latvia</option><option value=422>Lebanon</option><option value=426>Lesotho</option><option value=430>Liberia</option><option value=434>Libya</option><option value=438>Liechtenstein</option><option value=440>Lithuania</option><option value=442>Luxembourg</option><option value=446>Macao</option><option value=807>Macedonia</option><option value=450>Madagascar</option><option value=454>Malawi</option><option value=458>Malaysia</option><option value=462>Maldives</option><option value=466>Mali</option><option value=470>Malta</option><option value=584>Marshall Islands</option><option value=474>Martinique</option><option value=478>Mauritania</option><option value=480>Mauritius</option><option value=175>Mayotte</option><option value=484>Mexico</option><option value=581>Minor Outlying Islands of the United States</option><option value=498>Moldova</option><option value=492>Monaco</option><option value=496>Mongolia</option><option value=499>Montenegro</option><option value=500>Montserrat</option><option value=504>Morocco</option><option value=508>Mozambique</option><option value=104>Myanmar</option><option value=516>Namibia</option><option value=520>Nauru</option><option value=524>Nepal</option><option value=530>Netherlands Antilles</option><option value=540>New Caledonia</option><option value=554>New Zealand</option><option value=558>Nicaragua</option><option value=562>Niger</option><option value=566>Nigeria</option><option value=570>Niue</option><option value=574>Norfolk Island</option><option value=580>Northern Mariana Islands</option><option value=408>North Korea</option><option value=578>Norway</option><option value=512>Oman</option><option value=586>Pakistan</option><option value=585>Palau</option><option value=275>Palestinian Territory</option><option value=591>Panama</option><option value=598>Papua New Guinea</option><option value=600>Paraguay</option><option value=604>Peru</option><option value=608>Philippines</option><option value=612>Pitcairn</option><option value=616>Poland</option><option value=620>Portugal</option><option value=630>Puerto Rico</option><option value=634>Qatar</option><option value=638>Reunion</option><option value=642>Romania</option><option value=643>Russian Federation</option><option value=646>Rwanda</option><option value=654>Saint Helena</option><option value=659>Saint Kitts and Nevis</option><option value=662>Saint Lucia</option><option value=666>Saint Pierre and Miquelon</option><option value=670>Saint Vincent and the Grenadines</option><option value=882>Samoa</option><option value=674>San Marino</option><option value=678>Sao Tome and Principe</option><option value=682>Saudi Arabia</option><option value=686>Senegal</option><option value=688>Serbia</option><option value=690>Seychelles</option><option value=694>Sierra Leone</option><option value=702>Singapore</option><option value=703>Slovakia</option><option value=705>Slovenia</option><option value=90>Solomon Islands</option><option value=706>Somalia</option><option value=710>South Africa</option><option value=239>South Georgia and the South Sandwich Is</option><option value=410>South Korea</option><option value=724>Spain</option><option value=144>Sri Lanka</option><option value=736>Sudan</option><option value=740>Suriname</option><option value=744>Svalbard and Jan Mayen</option><option value=748>Swaziland</option><option value=752>Sweden</option><option value=756>Switzerland</option><option value=760>Syria</option><option value=158>Taiwan</option><option value=762>Tajikistan</option><option value=834>Tanzania</option><option value=764>Thailand</option><option value=270>The Gambia</option><option value=528>The Netherlands</option><option value=626>Timor-Leste</option><option value=768>Togo</option><option value=772>Tokelau</option><option value=776>Tonga</option><option value=780>Trinidad and Tobago</option><option value=788>Tunisia</option><option value=792>Turkey</option><option value=795>Turkmenistan</option><option value=796>Turks and Caicos Islands</option><option value=798>Tuvalu</option><option value=800>Uganda</option><option value=826>UK</option><option value=804>Ukraine</option><option value=784>United Arab Emirates</option><option value=850>United States Virgin Islands</option><option value=858>Uruguay</option><option value=840>USA</option><option value=860>Uzbekistan</option><option value=548>Vanuatu</option><option value=862>Venezuela</option><option value=704>Vietnam</option><option value=876>Wallis and Futuna</option><option value=905>West Bank and Gaza Strip</option><option value=732>Western Sahara</option><option value=887>Yemen</option><option value=894>Zambia</option><option value=716>Zimbabwe</option></select></div></div><div class="default-error margin-top is-hidden comment-enter-country country">Please select your country.</div></div></div><textarea data-test-id=article_add-comment_comment name=new-comment class="js-add-comment-comment comment margin-bottom margin-top"></textarea><div class="default-error margin-top comment-enter-text comment-error is-hidden ">You must enter a comment.</div><label class="comments-note u-mt--2 u-mb--2" for=competingInterests_1><div class="u-mb--1 u-mt--2"><strong data-test-id=article_report-add-comment_competing-interests-title>Competing Interests</strong></div><p class="u-mb--2 u-mt--0" data-test-id=article_report-add-comment_competing-interests-description>Please disclose any <a href="#article-competing-intersts-policy" class=js-modal-competing-intersts-toggle>competing interests</a> that might be construed to influence your judgment of the article's or peer review report's validity or importance.</p></label><div id=article-competing-intersts-policy class=js-article-competing-interests-policy style="display: none;"><h2 class="h2-title u-mt--0 u-pt--0">Competing Interests Policy</h2><p> Provide sufficient details of any financial or non-financial competing interests to enable users to assess whether your comments might lead a reasonable person to question your impartiality. Consider the following examples, but note that this is not an exhaustive list: </p><div class=heading5>Examples of 'Non-Financial Competing Interests'</div><ol class="numbered-list no-padding"><li class=standard-padding>Within the past 4 years, you have held joint grants, published or collaborated with any of the authors of the selected paper.</li><li class=standard-padding>You have a close personal relationship (e.g. parent, spouse, sibling, or domestic partner) with any of the authors.</li><li class=standard-padding>You are a close professional associate of any of the authors (e.g. scientific mentor, recent student).</li><li class=standard-padding>You work at the same institute as any of the authors.</li><li class=standard-padding>You hope/expect to benefit (e.g. favour or employment) as a result of your submission.</li><li class=standard-padding>You are an Editor for the journal in which the article is published.</li></ol><div class="heading5 padding-top">Examples of 'Financial Competing Interests'</div><ol class="numbered-list no-padding"><li class=standard-padding>You expect to receive, or in the past 4 years have received, any of the following from any commercial organisation that may gain financially from your submission: a salary, fees, funding, reimbursements.</li><li class=standard-padding>You expect to receive, or in the past 4 years have received, shared grant support or other funding with any of the authors.</li><li class=standard-padding>You hold, or are currently applying for, any patents or significant stocks/shares relating to the subject matter of the paper you are commenting on.</li></ol></div><div id=competingInterests_1 class="c-inline-editor js-inline-editor js-form_input u-bb--all u-mb--2 js-comment-competing-interests" data-name=competing-interests data-rows=3><textarea id=competingInterests_1_input name=competing-interests placeholder="&nbsp;" required=false class="u-hide-visually js-inline-editor_input" tabindex=-1></textarea><div class="c-inline-editor__editor js-inline-editor_editor o-box c-box o-box--tiny u-bg--11" data-target=competingInterests_1_input role=textbox required=false data-placeholder="&nbsp;" contenteditable=true></div><span class="c-inline-editor__error js-inline-editor-message">Please state your competing interests</span></div><div data-test-id=article_add-comment_saved class="green-message comments is-hidden comment-is-saved">The comment has been saved.</div><div data-test-id=article_add-comment_error class="default-error comments is-hidden comment-not-added">An error has occurred. Please try again.</div><div class=clearfix></div><div class=js-hook></div></div><div class="c-modal__extra-message js-modal__extra-message t-h4 u-black--medium"></div></div><div class="c-modal__actions o-box__actions"><a href="#" data-test-id=article_add-comment_cancel class="c-button c-button--full js-modal__close c-button--secondary">Cancel</a><a href="#" data-test-id=article_add-comment_post class="c-button c-button--full js-modal__confirm c-button--primary">Post</a></div></aside></div></div><style>
                .at-icon-wrapper {
        background-size: 100% !important;
    }
</style><script src="/js/namespace.js"></script><script src="/js/constants.js"></script><script src="/js/utilities.js"></script><script src="/js/article/alert-signup.js"></script><script type='text/javascript'>
    var lTitle = "Differential analyses for RNA-seq: transcript-level...".replace("'", '');
    var linkedInUrl = "http://www.linkedin.com/shareArticle?url=https://f1000research.com/articles/4-1521/v2" + "&title=" + encodeURIComponent(lTitle) + "&summary=" + encodeURIComponent('Read the article by ');

    var citeulikeUrl = "http://www.citeulike.org/posturl?url=https://f1000research.com/articles/4-1521/v2&title=" + encodeURIComponent(lTitle);

    var deliciousUrl = "https://del.icio.us/post?url=https://f1000research.com/articles/4-1521/v2&title=" + encodeURIComponent(lTitle);

    var redditUrl = "http://reddit.com/submit?url=https://f1000research.com/articles/4-1521/v2" + "&title=" + encodeURIComponent(lTitle);

            linkedInUrl += encodeURIComponent('Soneson C et al.');
    
    var offsetTop = /chrome/i.test( navigator.userAgent ) ? 4 : -10; 
    var addthis_config = {
            ui_offset_top: offsetTop,
                                    services_compact : "facebook,twitter,www.linkedin.com,www.mendeley.com,www.citeulike.org,reddit.com",
            services_expanded : "facebook,twitter,www.linkedin.com,www.mendeley.com,www.citeulike.org,reddit.com",
            services_custom : [
                {
                    name: "LinkedIn",
                    url:  linkedInUrl,
                    icon:"/img/icon/at_linkedin.svg"
                },
                {
                    name: "Mendeley",
                    url:  "http://www.mendeley.com/import/?url=https://f1000research.com/articles/4-1521/v2/mendeley",
                    icon:"/img/icon/at_mendeley.svg"
                },
                {
                    name: "citeulike",
                    url:  citeulikeUrl,
                    icon:"/img/icon/at_citeulike.svg"
                },
                {
                    name: "Reddit",
                    url:  redditUrl,
                    icon:"/img/icon/at_reddit.svg"
                },
            ]
        };


    var addthis_share = {
            url: "https://f1000research.com/articles/4-1521",
            templates : {
                twitter : "Differential analyses for RNA-seq: transcript-level estimates.... Soneson C et al., published by " + 
               "@F1000Research"
       + ", https://f1000research.com/articles/4-1521/v2"
            }
        };

    if (typeof(addthis) != "undefined"){
        addthis.addEventListener('addthis.ready', checkCount);
        addthis.addEventListener('addthis.menu.share', checkCount);
    }

        $(".f1r-shares-twitter").attr("href", "https://twitter.com/intent/tweet?text=" + addthis_share.templates.twitter);
    $(".f1r-shares-facebook").attr("href", "https://www.facebook.com/sharer/sharer.php?u=" + addthis_share.url);
    $(".f1r-shares-linkedin").attr("href", addthis_config.services_custom[0].url);
    $(".f1r-shares-reddit").attr("href", addthis_config.services_custom[3].url);
    $(".f1r-shares-citeulike").attr("href", addthis_config.services_custom[2].url);
    $(".f1r-shares-mendelay").attr("href", addthis_config.services_custom[1].url);

    function checkCount(){
        setTimeout(function(){
            $(".addthis_button_expanded").each(function(){
                var count = $(this).text();
                if (count !== "" && count != "0")
                    $(this).removeClass("is-hidden");
                else
                    $(this).addClass("is-hidden");
            });
        }, 1000);
    }
</script><div id=citeReportModal role=dialog aria-labelledby=citeReportModal_title aria-describedby=citeReportModal_description><div class="c-modal js-modal is-closed c-modal--large js-cite-report-modal c-modal--scroll "><aside class="c-modal__content u-black--high o-box u-black--high js-overflow-scroll "><div class=c-modal__close><button type=button class="c-button c-button--icon c-button--medium c-button--full@hover js-modal__close"><i class="c-button--icon__icon material-icons">close</i></button></div><h1 id=citeReportModal_title class="c-modal__title t-h3 u-mt--0 u-mb--2 u-weight--md">How to cite this report</h1><div id=citeReportModal_description class="c-modal__description js-modal__content t-h4 u-mt--0 u-mb--2 u-black--medium"><div id="" class=js-report-citation-container>{{reportCitation}}</div><div class="c-modal__extra-message js-modal__extra-message t-h4 u-black--medium"></div></div><div class="c-modal__actions o-box__actions"><a href="#" class="c-button c-button--full js-modal__close c-button--secondary">Cancel</a><a href="#" title="Copy the current citation details to the clipboard." data-clipboard-target="#referee-report-citation" class="c-button c-button--full js-modal__confirm c-button--primary js-clipboard c-mini-tooltip--above">Copy Citation Details</a></div></aside></div></div><script src="/js/referee/new/referee_validators.js"></script><script src="/js/referee/new/referee_helpers.js"></script><script src="/js/referee/new/referee_checkbox-input.js"></script><script src="/js/referee/new/referee_inline-editor.js"></script><script type="text/javascript">
    $(function(){
        var gaCat = "F1000Research";
        if (gaCat === "") {
            gaCat = $("body").hasClass("wellcome-brand") ? "Wellcome Open Research" : "F1000Research";
        }
        GAHelper.track({category: gaCat, action: "Article Page: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences", label: "pageviews"});
        GAHelper.track({category: gaCat, action: "Article Type: Method Article", label: "Article Page"});
        $(".f1r-article-desk .collection-image").each(function (idx, el) {
            var whatChannel = $(el).find("a").attr("href"),
                channelName = $.trim($(el).parent().find(".collection-detail a").text()),
                gaRef = "(ID: " + whatChannel.replace("/collections/", "") + ") " + channelName;
            GAHelper.track({category: 'ChannelStats', action: "Article Page: Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences", label: gaRef});
        });
    });
</script><script>
    $(function(){R.ui.buttonDropdowns('.dropdown-for-downloads');});
    $(function(){R.ui.toolbarDropdowns('.toolbar-dropdown-for-downloads');});
</script><script src="/js/article/track_article.js" type="text/javascript"></script><script src="/js/comment_mgmt/comment_mgmt.js" type="text/javascript"></script><script type="text/javascript">
    $.get("/articles/acj/7563/8770")
</script><script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script><script type="text/javascript" src="/js/app/messenger.js"></script><script type="text/javascript" src="/js/article/article_mobiles.js"></script><script src="/js/vendor/clipboard.min.js"></script><script src="/js/shared_scripts/modal-dialogue.js"></script><script src="/js/shared_scripts/clipboard.js"></script><script src="/js/article/thesaurus-terms-display.js"></script><script>
    new F1000.Clipboard();
    new F1000.ThesaurusTermsDisplay("articles", "article", "8770");
</script><script>
    $(document).ready(function() {
        $( "#frame1" ).on('load', function() {
            var mydiv = $(this).contents().find("div");
            var h     = mydiv.height();
            console.log(h)
        });

        
        var tooltipLivingFigure = jQuery(".interactive-living-figure-label .icon-more-info"),
            titleLivingFigure = tooltipLivingFigure.attr("title");
        tooltipLivingFigure.simpletip({
            fixed: true,
            position: ["-115", "30"],
            baseClass: 'small-tooltip',
            content:titleLivingFigure + "<div class='tooltip-arrow'></div>"
        });
        tooltipLivingFigure.removeAttr("title");

        $("body").on("click", ".cite-living-figure", function(e) {
            e.preventDefault();
            var ref = $(this).attr("data-ref");
            $(this).closest(".living-figure-list-container").find("#" + ref).fadeIn(200);
        });
        $("body").on("click", ".close-cite-living-figure", function(e) {
            e.preventDefault();
            $(this).closest(".popup-window-wrapper").fadeOut(200);
        });

                $(document).on("mouseup", function(e) {
            var metricsContainer = $(".article-metrics-popover-wrapper");
            if (!metricsContainer.is(e.target) && metricsContainer.has(e.target).length === 0) {
                $(".article-metrics-close-button").click();
            }
        });

        var articleId = $('#articleId').val();

        if($("#main-article-count-box").attachArticleMetrics) {
            $("#main-article-count-box").attachArticleMetrics(articleId, {
                articleMetricsView: true
            });
        }
    });

    var figshareWidget = $(".new_figshare_widget");
    if (figshareWidget.length > 0) {
        window.figshare.load("f1000", function(Widget) {
            // Select a tag/tags defined in your page. In this tag we will place the widget.
            _.map(figshareWidget, function(el){
                var widget = new Widget({
                    articleId: $(el).attr("figshare_articleId")
                    //height:300 // this is the height of the viewer part. [Default: 550]
                });
                widget.initialize(); // initialize the widget
                widget.mount(el); // mount it in a tag that's on your page
                // this will save the widget on the global scope for later use from
                // your JS scripts. This line is optional.
                //window.widget = widget;
            });
        });
    }
</script>

<script>
    $(document).ready(function () {

        
        var reportIds = {
                           "11761": 334,
                           "11745": 316,
                           "11746": 0,
                           "11747": 0,
                           "11748": 0,
                    };

        $(".referee-response-container,.js-referee-report").each(function(index, el) {
            var reportId = $(el).attr("data-reportid"),
                reportCount = reportIds[reportId] || 0;
            $(el).find(".comments-count-container,.js-referee-report-views").html(reportCount);
        });

        var uuidInput = $("#article_uuid"),
            oldUUId = uuidInput.val(),
            newUUId = "26634e5a-3638-471d-b9ee-deb1e401fd6d";
        uuidInput.val(newUUId);

        $("a[href*='article_uuid=']").each(function(index, el) {
            var newHref = $(el).attr("href").replace(oldUUId, newUUId);
            $(el).attr("href", newHref);
        });

    });
</script>              </div>
        </div>

        
            
            <div class="o-page__footer sticky-email-wrapper">
                
                

                


<footer class="c-footer t-inverted">
    
    <div class="o-wrapper">
        <div class="o-layout">


                        
            <div class="o-layout__item u-mb--3">
                <div class="c-branding c-branding--research">
                    <img src="/img/research/F1000Research_white.svg" alt="F1000Research">
                </div>
            </div>


                        
            <div class="o-layout__item u-1/3@md u-mb--3">

                <span class="c-hr c-hr--thick c-hr--low u-mb--2"></span>
                
                <p class="t-h3 u-mt--0 u-mb--0">An open research publishing platform for life scientists that offers immediate publication and transparent peer review.</p>

            </div>


                        
            <div class="o-layout__item u-2/3@md">

                <span class="c-hr c-hr--thick c-hr--low u-mb--2"></span>
                
                <div class="o-layout">
                    <nav class="c-footer__nav">
                        
                            <div class="o-layout__item u-1/2@sm u-mb--3">
                                

                                <div class="o-columns o-columns--2">
                                    
                                                                            <a href="/browse/articles" class="t-body c-footer__nav-item "   >Browse</a>
                                                                            <a href="/gateways" class="t-body c-footer__nav-item "   >Gateways</a>
                                                                            <a href="/collections" class="t-body c-footer__nav-item "   >Collections</a>
                                                                            <a href="/about" class="t-body c-footer__nav-item "   >How it Works</a>
                                                                            <a href="https://blog.f1000.com/blogs/f1000research/" class="t-body c-footer__nav-item "   >Blog</a>
                                                                            <a href="/contact" class="t-body c-footer__nav-item "   >Contact</a>
                                                                            <a href="/developers" class="t-body c-footer__nav-item u-hide u-show@navbar"   >For Developers</a>
                                                                            <a href="/published/rss" class="t-body c-footer__nav-item "   title="RSS feed of published articles"  >RSS</a>
                                    
                                </div>

                            </div>

                            <div class="o-layout__item u-1/2@sm u-center u-right@sm u-mb--3">
                                
                                <div class="_mdl-layout">
                                    <a class="mdl-button mdl-js-button mdl-button--inverted mdl-button--no-shadow mdl-js-ripple-effect mdl-button--outline" href="/for-authors/publish-your-research"   data-test-id="footer_submit_research"  >Submit Your Research</a>
                                </div>

                            </div>

                    </nav>
                </div>
                
            </div>

            <div class="o-layout__item u-mb--2">
                <div class="c-footer__share">
                        <div class="c-footer__share">
        <span class="c-footer__share-icon" title="Open Access">
            <span class="f1r-icon icon-100_open_access license-icon"></span>
        </span>

        <a class="c-footer__share-icon" href="//creativecommons.org/licenses" target="_blank" title="Creative Commons License CC-BY">
            <span class="f1r-icon icon-116_cc license-icon license-icon-cc"></span>
            <span class="f1r-icon icon-117_ccby license-icon license-icon-cc"></span>
        </a>

        <a class="c-footer__share-icon" href="//creativecommons.org/about/cc0" target="_blank" title="Creative Commons License CC0">
            <span class="f1r-icon icon-118_cco license-icon"></span>
        </a>

    </div>
                </div>
            </div>


                        
            <div class="o-layout__item u-1/3@md u-mb--3">

                <span class="c-hr c-hr--low u-mb--3"></span>
                
                <p class="c-footer__social u-mt--0 u-mb--0 u-white--low-med">Follow us 
                    <a href="https://www.facebook.com/F1000Research" target="_blank" class="c-footer__social-icon f1r-icon icon-55_footer_facebook"></a>
                    <a href="https://www.linkedin.com/company/faculty-of-1000-ltd" target="_blank" class="c-footer__social-icon f1r-icon icon-56_footer_linkedin"></a>
                    <a href="https://twitter.com/#!/F1000Research" target="_blank" class="c-footer__social-icon f1r-icon icon-56_footer_twitter"></a>
                    <a href="http://www.youtube.com/user/F1000research" target="_blank" class="c-footer__social-icon f1r-icon icon-57_footer_youtube"></a></p>

            </div>


                        
            <div class="o-layout__item u-2/3@md u-right@md">

                <span class="c-hr c-hr--low u-mb--3"></span>
                
                <p class="t-caption u-white--low-med">&copy; 2012-2019 F1000 Research Ltd. ISSN 2046-1402 | <a href="/about/legal" class="copyrightLegal">Legal</a> | Partner of <a target="_blank" href="http://www.who.int/hinari/en/">HINARI</a>  &bull; <a target="_blank" href="http://crossref.org/">CrossRef</a> &bull; <a target="_blank" href="http://about.orcid.org/">ORCID</a> &bull; <a target="_blank" href="http://www.fairsharing.org">FAIRSharing</a></p>

            </div>
        </div>
    </div>

</footer>            </div>
        
    </div>
    <div class="js-cookie-spacer"></div>
    <div class="cookie-warning">
        <div class="instruction">The F1000Research website uses cookies. By continuing to browse the site, you are agreeing to our use of cookies. <a class="js-scroll-to" href="/about/legal/privacypolicy#use-of-cookies" data-scroll-target="#use-of-cookies">Find out more &raquo;</a></div>
        <div class="close-button"></div>
    </div>
    <script>
        R.templateTests.simpleTemplate = R.template('<p class="$variable.one">$text</p><p class="${variable.two}">$text</p><p class="$!variable.three">$text</p><p class="$!{variable.four}">$text</p><p class="${selector}.five">$text</p>');
        R.templateTests.runTests();

        var F1000platform = new F1000.Platform({
            name: "f1000research",
            displayName: "F1000Research",
            hostName: "f1000research.com",
            id: "1",
            editorialEmail: "research@f1000.com",
            infoEmail: "info@f1000.com",
            usePmcStats: true
        });

        $(function(){R.ui.dropdowns('.dropdown-for-authors, .dropdown-for-about, .dropdown-for-myresearch');});
        // $(function(){R.ui.dropdowns('.dropdown-for-referees');});

        $(document).ready(function () {
            if ($(".cookie-warning").is(":visible")) {
                $(".sticky").css("margin-bottom", "35px");
                $(".devices").addClass("devices-and-cookie-warning");
            }
            $(".cookie-warning .close-button").click(function (e) {
                $(".devices").removeClass("devices-and-cookie-warning");
                $(".sticky").css("margin-bottom", "0");
            });

            $("#tweeter-feed .tweet-message").each(function (i, message) {
                var self = $(message);
                self.html(linkify(self.html()));
            });

            $(".partner").on("mouseenter mouseleave", function() {
                $(this).find(".gray-scale, .colour").toggleClass("is-hidden");
            });
        });

    </script>

            
<div class="sign-in-popup">
	<!-- <a href="#" class="sign-in shadow">Sign in <span class="sign-in-image-active"></span></a> -->
	<a href="#" class="sign-in ${locale}">Sign In <span class="arrow-closed sign-in-arrow-padding arrow-opened"></span></a>
	<div class="sign-in-form">

            <form action="https://f1000research.com/j_spring_oauth_security_check" id="googleOAuth" method="post"  target="_top" >
                                    <input class="target-field" type="hidden" name="target" value="/articles/4-1521/v2"/>
                            <input id="google-remember-me" name="_spring_security_oauth_remember_me" type="hidden" value="true"/>
        <input id="system-google" name="system" type="hidden" value="GOOGLE"/>
    </form>
            <form action="https://f1000research.com/j_spring_oauth_security_check" id="facebookOAuth" method="post"  target="_top" >
                                    <input class="target-field" type="hidden" name="target" value="/articles/4-1521/v2"/>
                            <input id="facebook-remember-me" name="_spring_security_oauth_remember_me" type="hidden" value="true"/>
        <input id="system-fb" name="system" type="hidden" value="FACEBOOK"/>
    </form>
            <form action="https://f1000research.com/j_spring_oauth_security_check" id="orcidOAuth" method="post"  target="_top" >
                                    <input class="target-field" type="hidden" name="target" value="/articles/4-1521/v2"/>
                            <input id="orcid-remember-me" name="_spring_security_oauth_remember_me" type="hidden" value="true"/>
        <input id="system-orcid" name="system" type="hidden" value="ORCID"/>
    </form>
		<form id="sign-in-form" class="login-container" action="https://f1000research.com/login" method="post" name="f">
           <div id="sign-in-form-gfb-popup"></div>

                                                            <input class="target-field" type="hidden" name="target" value="/articles/4-1521/v2"/>
                            			<input type="text" name="username" id="signin-email-box" class="sign-in-input" placeholder="Email address" autocomplete="email">
			<input type="password" name="password" id="signin-password-box" class="sign-in-input" placeholder="Password" autocomplete="current-password">
			<div class="sign-in-remember">
                <div class="checkbox-wrapper">
    				<input type="checkbox" id="remember-me" name="remember_me" class="checkbox is-hidden">
                </div>
                <span class="checkbox-label">Remember me</span>
			</div>
			<a href="#" class="sign-in-link" id="forgot-password-link">Forgotten your password?</a>
			<div class="sign-in-button-container margin-top margin-left-20 margin-bottom">
				<button type="submit" id="sign-in-button" class="sign-in-buttons general-white-orange-button">Sign In</button>
				<button type="button" id="sign-in-cancel" class="sign-in-buttons sign-in-cancel-button margin-left">Cancel</button>
				<div class="clearfix"></div>
			</div>
			<div class="sign-in-error">Email or password not correct. Please try again</div>
			<div class="sign-in-loading">Please wait...</div>
		</form>
		<div class="forgot-password-container">
			
<script type="text/javascript">
	$(function(){
		// Note: All the setup needs to run against a name attribute and *not* the id due the clonish
		// nature of facebox...
		$("a[id=googleSignInButton]").click(function(event){
            event.preventDefault();
            $("input[id=oAuthSystem]").val("GOOGLE");
            $("form[id=oAuthForm]").submit();
        });
        $("a[id=facebookSignInButton]").click(function(event){
            event.preventDefault();
            $("input[id=oAuthSystem]").val("FACEBOOK");
            $("form[id=oAuthForm]").submit();
        });
        $("a[id=orcidSignInButton]").click(function(event){
            event.preventDefault();
            $("input[id=oAuthSystem]").val("ORCID");
            $("form[id=oAuthForm]").submit();
        });
	});
</script>

<span class="text first">
	If you've forgotten your password, please enter your email address below and we'll send you instructions on how to reset your password.
    <p>The email address should be the one you originally registered with F1000.</p>
</span>
<input name="email" class="sign-in-input" id="email-forgot-password" type="text" placeholder="Email address">
<div class="forgot-password-email-error">
	Email address not valid, please try again
</div>
<div class="forgot-password-google-email-error">
    <p>You registered with F1000 via Google, so we cannot reset your password.</p>
	<p>To sign in, please click <a href="#" id="googleSignInButton">here</a>.</p>
    <p>If you still need help with your Google account password, please click <a href="https://www.google.com/accounts/recovery">here</a>.</p>
</div>
<div class="forgot-password-facebook-email-error">
    <p>You registered with F1000 via Facebook, so we cannot reset your password.</p>
    <p>To sign in, please click <a href="#" id="facebookSignInButton">here</a>.</p>
	<p>If you still need help with your Facebook account password, please click <a href="https://www.facebook.com/recover/initiate">here</a>.</p>
</div>
<div class="clearfix"></div>
<div class="forgot-password-captcha-error">
	Code not correct, please try again
</div>
<div class="clearfix"></div>
<div class="sign-in-button-container margin-left-20 margin-bottom">
	<button type="button" id="sign-in-reset-password" class="sign-in-buttons general-white-orange-button">Reset password</button>
	<button type="button" id="forgot-password-cancel" class="sign-in-buttons sign-in-cancel-button margin-left">Cancel</button>
	<div class="clearfix"></div>
</div>
<span class="text last">
	Email <a href="mailto:"></a> for further assistance.
</span>
<form action="https://f1000research.com/j_spring_oauth_security_check" id="oAuthForm" method="post" target="_top">
                        <input class="target-field" type="hidden" name="target" value="/articles/4-1521/v2"/>
                <input id="oAuthSystem" name="system" type="hidden"/>
</form>
			<div class="forgot-password-server-error">Server error, please try again.</div>
			<div class="sign-in-success">
                <p>We have sent an email to <span id="email-value"></span>, please follow the instructions to reset your password.</p>
                <p>If you don't receive this email, please check your spam filters and/or contact .</p>
            </div>
			<div class="sign-in-loading">Please wait...</div>
		</div>

		<div class="sign-in-form-register-section">
			<div class="sign-in-button-container margin-left-20 margin-bottom">
				<a href="/register" title="Register"><button type="button" id="sign-in-register-button" class="sign-in-buttons general-white-orange-button">Register</button></a>
				<div class="clearfix"></div>
			</div>
		</div>

	</div>
</div>

<script type="text/javascript">
$(document).ready(function () {

    signIn.createSignInAsRow($("#sign-in-form-gfb-popup"));

    $(".target-field").each(function () {
        var uris = $(this).val().split("/");
        if (uris.pop() === "login") {
        	$(this).val(uris.toString().replace(",","/"));
        }
    });
});
</script>
        <div id="templateOverlay" class="is-hidden" hidden="hidden">
  <div class="o-overlay js-overlay is-hidden" hidden="hidden"></div>
</div>

<div id="templateExternalMessages" class="is-hidden" hidden="hidden">
  <div class="o-modal o-modal--auto@md js-external-messages is-hidden" hidden="hidden">
    <div class="o-modal__body">
      <section class="c-console">
        <div class="_mdl-layout c-console__bdy js-external-messages-body"></div>
        <footer class="_mdl-layout c-console__ftr o-flex o-flex--reverse js-external-messages-footer">
          <button type="button" class="mdl-button mdl-js-button mdl-button--raised mdl-button--colored c-console__btn js-external-messages-close" data-action="maintenance-close">I Understand</button>
        </footer>
      </section>
    </div>
  </div>
</div>

<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.22.1/moment.min.js"></script>

<script src="/js/namespace.js"></script>
<script src="/js/constants.js"></script>
<script src="/js/utilities.js"></script>

<script>
                  F1000.ExtenalMaintenanceItems = [
    {
      start: '2018-12-14 17Z',
      end: '2018-12-16 00Z',
      msg: 'Please note that on Saturday 15th December from 2pm (UTC) for up to 8 hours, ORCID services will not be available due to a scheduled outage by ORCID that we have no control over.<br><br>During this time, if you do not already have an ORCID iD connected to your account on this site, you will not be able to submit an article (as supplying your iD is a requirement for submission of articles; other content types are not affected).',
      cookieName: 'orcid20181215',
      editor: false
    },
    {
      start: '2018-11-22 12Z',
      end: '2018-11-22 17:30Z',
      msg: 'Please note that on Saturday 15th December from 2pm (UTC) for up to 8 hours, ORCID services will not be available due to a scheduled outage by ORCID that we have no control over.<br><br>During this time, if you do not already have an ORCID iD connected to your account on this site, you will not be able to submit an article (as supplying your iD is a requirement for submission of articles; other content types are not affected).<br><br>You can follow the progress on Twitter (<a href="https://twitter.com/ORCID_Org" target="_blank">@ORCID_Org</a>). For more information, please read <a href="https://orcid.org/blog/2018/11/20/orcid-registry-scheduled-outage-december-15-2018" target="_blank">ORCID\'s blog</a>.',
      cookieName: 'test20181122',
      editor: false
    }
  ];
</script>

<script src="/js/shared_scripts/cookie-helper.js"></script>
<script src="/js/shared_scripts/mdl-helper.js"></script>

<script src="/js/app/external-maintenance.js"></script>

                <script type="text/javascript">
            (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
            (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
            })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

            ga('create', 'UA-5646075-11', 'auto');
            ga('require', 'displayfeatures');
            ga('send', 'pageview');
        </script>
            
        <script type="text/javascript" src="/js/app/research.analytics.js"></script>

    <!-- Start of HubSpot Embed Code -->
    <script type="text/javascript" id="hs-script-loader" async defer src="//js.hs-scripts.com/4475190.js"></script>
    <!-- End of HubSpot Embed Code -->
</body>
</html>