---
title: "Lecture 18 - Cancer Informatics"
author: "ACGeffre"
date: "5/31/2019"
output: github_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

```{r, include = FALSE}
if (!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")
BiocManager::install()
```

For this worksheet, I will be using instructions from the [BGGN 213 website](https://bioboot.github.io/bggn213_S19/class-material/lecture18_part1_BGGN213_W19.html).

```{r, include = FALSE}
BiocManager::install(c("GenomicDataCommons", "TCGAbiolinks", "maftools"))
library(GenomicDataCommons) # allows us to connect to NCBI GDC
library(TCGAbiolinks) # Helps with integrative data analysis with GDC data
library(maftools) # Allows us to summarize, analyze and visualize Mutation Annotation Format (MAF) files
```


## Accessing the Genomic Data Commons
We use the bioconductor package "GenomicDataCommons" to connect to the NCBI [Genomic Data Commons](https://gdc.cancer.gov/search/node/TP53). First, we will check the status of our connection with the status() command.

```{r eval=FALSE}
status()
```

It says our status is okay, so that's great!Let's grab 

```{r}
projects <- getGDCprojects()
head(projects)
```
```{r}
cases_by_project <- cases() %>%
  facet("project.project_id") %>% # chunks out data by project ID (project ID is a facet of the data)
  aggregations() # Describes how many of each project there are
head(cases_by_project)
```

We can filter our list of projects and search for our particular target (in this case, we want the TCGA project looking at pancreatic cancers).
```{r}
x <- cases_by_project$project.project_id

# Make a custom color vector for our plot
colvec <- rep("lightblue", nrow(x))
colvec[x$key=="TCGA-PAAD"] <- "red"

# Plot with 'log' for y axis and rotate labels with 'las'
#par(___)  
barplot(x$doc_count, names.arg=x$key, log="y", col=colvec, las=2)
```

## Designing personal cancer vaccines

We will use material from BGGN 213 [Lecture 18](https://bioboot.github.io/bggn213_S19/class-material/lecture18_part2_BGGN213_W19/), from Barry Grant and Bjoern Peters.

>Background: To identify somatic mutations in a tumor, DNA from the tumor is sequenced and compared to DNA from normal tissue in the same individual using variant calling algorithms. Comparison of tumor sequences to those from normal tissue (rather than ‘the human genome’) is important to ensure that the detected differences are not germline mutations. To identify which of the somatic mutations leads to the production of aberrant proteins, the location of the mutation in the genome is inspected to identify non-synonymous mutations (i.e. those that fall into protein coding regions and change the encoded amino acid).

Let's read in our data:

```{r}
library(bio3d)
seqs <- read.fasta("lecture18_sequences.fa")
seqs
```

>Q1: Identify sequence regions that contain all 9-mer peptides that are only found in the tumor. Hint: You will need to first identify the sites of mutation in the above sequences and then extract the surrounding subsequence region. This subsequence should encompass all possible 9-mers in the tumor derived sequence. In other words extract the subsequence from 8 residues before and 8 residues after all point mutations in the tumor sequence.

```{r}
align <- seqaln(seqs)
ide <- conserv(seqs$ali, method="identity")
# This asks R to score how conserved residues are at each position in the alignmnet
mutant.sites <- which(ide < 1) 
# This step pops out any sites where the residue is not the same between sequences (i.e. there is less than perfect residue conservation)
mutant.sites
```
It appears that site 41, 65, 213, and 259 are worthwile to look at (note that everything after 259 we don't have sequence info on, so let's ignore them for now?). Alternatively, let's just remove the gaps:
```{r}
## Exclude gap possitions from analysis
gaps <- gap.inspect(seqs)
# gap.inspect() reports the number of gaps for a sequence and position in an alignment
gaps
```

Let's remake the mutant sites data set by searching just the positions that don't have gaps. 
```{r}
mutant.sites <- mutant.sites[mutant.sites %in% gaps$f.inds]
#f.inds is the set of sites that DON'T have gaps.  So we will only search those ones for mutant site. 
mutant.sites
```

```{r}
# We can make a list of start and end postitions for all possible 9mers (e.g. mutation site +/- 8 residues)
start.position <- mutant.sites - 8
end.position <-  mutant.sites + 8

```

```{r}
mutant.names <- paste0(seqs$ali["P53_wt",mutant.sites], mutant.sites,
                       seqs$ali["P53_mutant",mutant.sites])

## Sequence positions surounding each mutant site
start.position <- mutant.sites - 8
end.position <-  mutant.sites + 8

# Blank matrix to store sub-sequences
store.seqs <- matrix("-", nrow=length(mutant.sites), ncol=17)
rownames(store.seqs) <- mutant.names

## Extract each sub-sequence
for(i in 1:length(mutant.sites)) {
  store.seqs[i,] <- seqs$ali["P53_mutant",start.position[i]:end.position[i]]
}

store.seqs
```


```{r}
## First blank out the gap positions 
store.seqs[store.seqs == "-"] <- ""

## Output a FASTA file for further analysis
write.fasta(seqs=store.seqs, ids=mutant.names, file="subsequences.fa")
```

