<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <title>A survey of best practices for RNA-seq data analysis | Genome Biology | Full Text</title>

        

    <meta name="citation_abstract" content="RNA-sequencing (RNA-seq) has a wide variety of applications, but no single analysis pipeline can be used in all cases. We review all of the major steps in RNA-seq data analysis, including experimental design, quality control, read alignment, quantification of gene and transcript levels, visualization, differential gene expression, alternative splicing, functional analysis, gene fusion detection and eQTL mapping. We highlight the challenges associated with each step. We discuss the analysis of small RNAs and the integration of RNA-seq with other functional genomics techniques. Finally, we discuss the outlook for novel technologies that are changing the state of the art in transcriptomics."/>

    <meta name="description" content="RNA-sequencing (RNA-seq) has a wide variety of applications, but no single analysis pipeline can be used in all cases. We review all of the major steps in RNA-seq data analysis, including experimental design, quality control, read alignment, quantification of gene and transcript levels, visualization, differential gene expression, alternative splicing, functional analysis, gene fusion detection and eQTL mapping. We highlight the challenges associated with each step. We discuss the analysis of small RNAs and the integration of RNA-seq with other functional genomics techniques. Finally, we discuss the outlook for novel technologies that are changing the state of the art in transcriptomics."/>

    <meta name="journal_id" content="13059"/>

    <meta name="dc.title" content="A survey of best practices for RNA-seq data analysis"/>

    <meta name="dc.source" content="Genome Biology 2016 17:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2016-01-26"/>

    <meta name="dc.type" content="ReviewPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2016 Conesa et al."/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="RNA-sequencing (RNA-seq) has a wide variety of applications, but no single analysis pipeline can be used in all cases. We review all of the major steps in RNA-seq data analysis, including experimental design, quality control, read alignment, quantification of gene and transcript levels, visualization, differential gene expression, alternative splicing, functional analysis, gene fusion detection and eQTL mapping. We highlight the challenges associated with each step. We discuss the analysis of small RNAs and the integration of RNA-seq with other functional genomics techniques. Finally, we discuss the outlook for novel technologies that are changing the state of the art in transcriptomics."/>

    <meta name="prism.issn" content="1474-760X"/>

    <meta name="prism.publicationName" content="Genome Biology"/>

    <meta name="prism.publicationDate" content="2016-01-26"/>

    <meta name="prism.volume" content="17"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="ReviewPaper"/>

    <meta name="prism.startingPage" content="13"/>

    <meta name="prism.copyright" content="2016 Conesa et al."/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0881-8"/>

    <meta name="prism.doi" content="doi:10.1186/s13059-016-0881-8"/>

    <meta name="citation_pdf_url" content="https://genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-016-0881-8"/>

    <meta name="citation_fulltext_html_url" content="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0881-8"/>

    <meta name="citation_journal_title" content="Genome Biology"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="1474-760X"/>

    <meta name="citation_title" content="A survey of best practices for RNA-seq data analysis"/>

    <meta name="citation_volume" content="17"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2016/12"/>

    <meta name="citation_online_date" content="2016/01/26"/>

    <meta name="citation_firstpage" content="13"/>

    <meta name="citation_article_type" content="Review"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="dc.identifier" content="doi:10.1186/s13059-016-0881-8"/>

    <meta name="DOI" content="10.1186/s13059-016-0881-8"/>

    <meta name="citation_doi" content="10.1186/s13059-016-0881-8"/>

    <meta name="dc.creator" content="Ana Conesa"/>

    <meta name="dc.creator" content="Pedro Madrigal"/>

    <meta name="dc.creator" content="Sonia Tarazona"/>

    <meta name="dc.creator" content="David Gomez-Cabrero"/>

    <meta name="dc.creator" content="Alejandra Cervera"/>

    <meta name="dc.creator" content="Andrew McPherson"/>

    <meta name="dc.creator" content="Micha&#322; Wojciech Szcze&#347;niak"/>

    <meta name="dc.creator" content="Daniel J. Gaffney"/>

    <meta name="dc.creator" content="Laura L. Elo"/>

    <meta name="dc.creator" content="Xuegong Zhang"/>

    <meta name="dc.creator" content="Ali Mortazavi"/>

    <meta name="dc.subject" content="Animal Genetics and Genomics"/>

    <meta name="dc.subject" content="Human Genetics"/>

    <meta name="dc.subject" content="Plant Genetics &amp; Genomics"/>

    <meta name="dc.subject" content="Microbial Genetics and Genomics"/>

    <meta name="dc.subject" content="Bioinformatics"/>

    <meta name="dc.subject" content="Evolutionary Biology"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Methods; citation_title=Mapping and quantifying mammalian transcriptomes by RNA-Seq; citation_author=A Mortazavi, BA Williams, K McCue, L Schaeffer, B Wold; citation_volume=5; citation_publication_date=2008; citation_pages=1-8; citation_doi=10.1038/nmeth.1226; citation_id=CR1"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Methods; citation_title=Comprehensive comparative analysis of strand-specific RNA sequencing methods; citation_author=JZ Levin, M Yassour, X Adiconis, C Nusbaum, DA Thompson, N Friedman; citation_volume=7; citation_publication_date=2010; citation_pages=709-15; citation_doi=10.1038/nmeth.1491; citation_id=CR2"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=Transcriptome analysis by strand-specific sequencing of complementary DNA; citation_author=D Parkhomchuk, T Borodina, V Amstislavskiy, M Banaru, L Hallen, S Krobitsch; citation_volume=37; citation_publication_date=2009; citation_pages=e123; citation_doi=10.1093/nar/gkp596; citation_id=CR3"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Methods; citation_title=Analysis and design of RNA sequencing experiments for identifying isoform regulation; citation_author=Y Katz, ET Wang, EM Airoldi, CB Burge; citation_volume=7; citation_publication_date=2010; citation_pages=1009-15; citation_doi=10.1038/nmeth.1528; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Methods; citation_title=Computational methods for transcriptome annotation and quantification using RNA-seq; citation_author=M Garber, MG Grabherr, M Guttman, C Trapnell; citation_volume=8; citation_publication_date=2011; citation_pages=469-77; citation_doi=10.1038/nmeth.1613; citation_id=CR5"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Characterization and improvement of RNA-Seq precision in quantitative transcript expression profiling; citation_author=PP &#321;abaj, GG Leparc, BE Linggi, LM Markillie, HS Wiley, DP Kreil; citation_volume=27; citation_publication_date=2011; citation_pages=i383-91; citation_doi=10.1093/bioinformatics/btr247; citation_id=CR6"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Rev Genet; citation_title=Sequencing depth and coverage: key considerations in genomic analyses; citation_author=D Sims, I Sudbery, NE Ilott, A Heger, CP Ponting; citation_volume=15; citation_publication_date=2014; citation_pages=121-32; citation_doi=10.1038/nrg3642; citation_id=CR7"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Low-coverage single-cell mRNA sequencing reveals cellular heterogeneity and activated signaling pathways in developing cerebral cortex; citation_author=AA Pollen, TJ Nowakowski, J Shuga, X Wang, AA Leyrat, JH Lui; citation_volume=32; citation_publication_date=2014; citation_pages=1053-8; citation_doi=10.1038/nbt.2967; citation_id=CR8"/>

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Massively parallel single-cell RNA-seq for marker-free decomposition of tissues into cell types; citation_author=DA Jaitin, E Kenigsberg, H Keren-Shaul, N Elefant, F Paul, I Zaretsky; citation_volume=343; citation_publication_date=2014; citation_pages=776-9; citation_doi=10.1126/science.1247651; citation_id=CR9"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=Differential expression in RNA-seq: a matter of depth; citation_author=S Tarazona, F Garcia-Alcalde, J Dopazo, A Ferrer, A Conesa; citation_volume=21; citation_publication_date=2011; citation_pages=2213-23; citation_doi=10.1101/gr.124321.111; citation_id=CR10"/>

    <meta name="citation_reference" content="Andrews S. FASTQC. A quality control tool for high throughput sequence data. 
                    http://www.bioinformatics.babraham.ac.uk/projects/fastqc/
                    
                  . Accessed 29 September 2014."/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=NGSQC: cross-platform quality analysis pipeline for deep sequencing data; citation_author=M Dai, RC Thompson, C Maher, R Contreras-Galindo, MH Kaplan, DM Markovitz; citation_volume=11; citation_issue=Suppl 4; citation_publication_date=2010; citation_pages=S7; citation_doi=10.1186/1471-2164-11-S4-S7; citation_id=CR12"/>

    <meta name="citation_reference" content="FASTX-Toolkit. 
                    http://hannonlab.cshl.edu/fastx_toolkit/
                    
                  . Accessed 12 January 2016."/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data; citation_author=AM Bolger, M Lohse, B Usadel; citation_volume=30; citation_publication_date=2014; citation_pages=2114-20; citation_doi=10.1093/bioinformatics/btu170; citation_id=CR14"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=STAR: ultrafast universal RNA-seq aligner; citation_author=A Dobin, CA Davis, F Schlesinger, J Drenkow, C Zaleski, S Jha; citation_volume=29; citation_publication_date=2013; citation_pages=15-21; citation_doi=10.1093/bioinformatics/bts635; citation_id=CR15"/>

    <meta name="citation_reference" content="Picard. 
                    http://picard.sourceforge.net/
                    
                  . Accessed 12 January 2016."/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=RSeQC: quality control of RNA-seq experiments; citation_author=L Wang, S Wang, W Li; citation_volume=28; citation_publication_date=2012; citation_pages=2184-5; citation_doi=10.1093/bioinformatics/bts356; citation_id=CR17"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Qualimap: evaluating next-generation sequencing alignment data; citation_author=F Garc&#237;a-Alcalde, K Okonechnikov, J Carbonell, LM Cruz, S G&#246;tz, S Tarazona; citation_volume=28; citation_publication_date=2012; citation_pages=2678-9; citation_doi=10.1093/bioinformatics/bts503; citation_id=CR18"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=Data quality aware analysis of differential expression in RNA-seq with NOISeq R/Bioc package; citation_author=S Tarazona, P Furi&#243;-Tar&#237;, D Turr&#224;, AD Pietro, MJ Nueda, A Ferrer; citation_volume=43; citation_publication_date=2015; citation_pages=e140; citation_id=CR19"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=GC-content normalization for RNA-seq data; citation_author=D Risso, K Schwartz, G Sherlock, S Dudoit; citation_volume=12; citation_publication_date=2011; citation_pages=480; citation_doi=10.1186/1471-2105-12-480; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Methods; citation_title=Assessment of transcript reconstruction methods for RNA-seq; citation_author=T Steijger, JF Abril, PG Engstr&#246;m, F Kokocinski, TJ Hubbard, R Guig&#243;; citation_volume=10; citation_publication_date=2013; citation_pages=1177-84; citation_doi=10.1038/nmeth.2714; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Genome-guided transcript assembly by integrative analysis of RNA sequence data; citation_author=N Boley, MH Stoiber, BW Booth, KH Wan, RA Hoskins, PJ Bickel; citation_volume=32; citation_publication_date=2014; citation_pages=341-6; citation_doi=10.1038/nbt.2850; citation_id=CR22"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Identification of novel transcripts in annotated genomes using RNA-Seq; citation_author=A Roberts, H Pimentel, C Trapnell, L Pachter; citation_volume=27; citation_publication_date=2011; citation_pages=2325-9; citation_doi=10.1093/bioinformatics/btr355; citation_id=CR23"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=iReckon: simultaneous isoform discovery and abundance estimation from RNA-seq data; citation_author=AM Mezlini, EJ Smith, M Fiume, O Buske, GL Savich, S Shah; citation_volume=23; citation_publication_date=2013; citation_pages=519-29; citation_doi=10.1101/gr.142232.112; citation_id=CR24"/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Sparse linear modeling of next-generation mRNA sequencing (RNA-Seq) data for isoform discovery and abundance estimation; citation_author=JJ Li, CR Jiang, JB Brown, H Huang, PJ Bickel; citation_volume=108; citation_publication_date=2011; citation_pages=19867-72; citation_doi=10.1073/pnas.1113972108; citation_id=CR25"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=StringTie enables improved reconstruction of a transcriptome from RNA-seq reads; citation_author=M Pertea, GM Pertea, CM Antonescu, TC Chang, JT Mendell, SL Salzberg; citation_volume=33; citation_publication_date=2015; citation_pages=290-5; citation_doi=10.1038/nbt.3122; citation_id=CR26"/>

    <meta name="citation_reference" content="citation_journal_title=Stat Biosci; citation_title=Simultaneous isoform discovery and quantification from RNA-Seq; citation_author=D Hiller, WH Wong; citation_volume=5; citation_publication_date=2013; citation_pages=100-18; citation_doi=10.1007/s12561-012-9069-2; citation_id=CR27"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=AUGUSTUS: ab initio prediction of alternative transcripts; citation_author=M Stanke, O Keller, I Gunduz, A Hayes, S Waack, B Morgenstern; citation_volume=34; citation_publication_date=2006; citation_pages=W435-9; citation_doi=10.1093/nar/gkl200; citation_id=CR28"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Methods; citation_title=Systematic evaluation of spliced alignment programs for RNA-seq data; citation_author=PG Engstr&#246;m, T Steijger, B Sipos, GR Grant, A Kahles, G R&#228;tsch; citation_volume=10; citation_publication_date=2013; citation_pages=1185-91; citation_doi=10.1038/nmeth.2722; citation_id=CR29"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=SOAPdenovo-Trans: de novo transcriptome assembly with short RNA-Seq reads; citation_author=Y Xie, G Wu, J Tang, R Luo, J Patterson, S Liu; citation_volume=30; citation_publication_date=2014; citation_pages=1660-6; citation_doi=10.1093/bioinformatics/btu077; citation_id=CR30"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Oases: robust de novo RNA-seq assembly across the dynamic range of expression levels; citation_author=MH Schulz, DR Zerbino, M Vingron, E Birney; citation_volume=28; citation_publication_date=2012; citation_pages=1086-92; citation_doi=10.1093/bioinformatics/bts094; citation_id=CR31"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Full-length transcriptome assembly from RNA-seq data without a reference genome; citation_author=MG Grabherr, BJ Haas, M Yassour, JZ Levin, DA Thompson, I Amit; citation_volume=29; citation_publication_date=2011; citation_pages=644-52; citation_doi=10.1038/nbt.1883; citation_id=CR32"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Protoc; citation_title=
                              De novotranscript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis; citation_author=BJ Haas, A Papanicolaou, M Yassour, M Grabherr, PD Blood, J Bowden; citation_volume=8; citation_publication_date=2013; citation_pages=1494-512; citation_doi=10.1038/nprot.2013.084; citation_id=CR33"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Sailfish enables alignment-free isoform quantification from RNA-seq reads using lightweight algorithms; citation_author=R Patro, SM Mount, C Kingsford; citation_volume=32; citation_publication_date=2014; citation_pages=462-4; citation_doi=10.1038/nbt.2862; citation_id=CR34"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=HTSeq - a Python framework to work with high-throughput sequencing data; citation_author=S Anders, PT Pyl, W Huber; citation_volume=31; citation_publication_date=2015; citation_pages=166-9; citation_doi=10.1093/bioinformatics/btu638; citation_id=CR35"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=featureCounts: an efficient general purpose program for assigning sequence reads to genomic features; citation_author=Y Liao, GK Smyth, W Shi; citation_volume=30; citation_publication_date=2014; citation_pages=923-30; citation_doi=10.1093/bioinformatics/btt656; citation_id=CR36"/>

    <meta name="citation_reference" content="UCSC Genome Bioinformatics: Frequently Asked Questions: Data File Formats. 
                    https://genome.ucsc.edu/FAQ/FAQformat.html#format4
                    
                  . Accessed on 12 January 2016."/>

    <meta name="citation_reference" content="Pachter L. Models for transcript quantification from RNA-seq. arXiv.org. 2011. 
                    http://arxiv.org/abs/1104.3889
                    
                  . Accessed 6 January 2016."/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation; citation_author=C Trapnell, BA Williams, G Pertea, A Mortazavi, G Kwan, MJ Baren; citation_volume=28; citation_publication_date=2010; citation_pages=511-5; citation_doi=10.1038/nbt.1621; citation_id=CR39"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome; citation_author=B Li, CN Dewey; citation_volume=12; citation_publication_date=2011; citation_pages=323; citation_doi=10.1186/1471-2105-12-323; citation_id=CR40"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Methods; citation_title=Streaming fragment assignment for real-time analysis of sequencing experiments; citation_author=A Roberts, L Pachter; citation_volume=10; citation_publication_date=2013; citation_pages=71-3; citation_doi=10.1038/nmeth.2251; citation_id=CR41"/>

    <meta name="citation_reference" content="Bray N, Pimentel H, Melsted P, Pachter L. Near-optimal RNA-Seq quantification with kallisto. 
                    https://liorpachter.wordpress.com/2015/05/10/near-optimal-rna-seq-quantification-with-kallisto/
                    
                  . Accessed 6 January 2016."/>

    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=Improving RNA-seq expression estimates by correcting for fragment bias; citation_author=A Roberts, C Trapnell, J Donaghey, JL Rinn, L Pachter; citation_volume=12; citation_publication_date=2011; citation_pages=R22; citation_doi=10.1186/gb-2011-12-3-r22; citation_id=CR43"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=NURD: an implementation of a new method to estimate isoform expression from non-uniform RNA-seq data; citation_author=X Ma, X Zhang; citation_volume=14; citation_publication_date=2013; citation_pages=220; citation_doi=10.1186/1471-2105-14-220; citation_id=CR44"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=Evaluation of statistical methods for normalization and differential expression in mRNA-Seq experiments; citation_author=JH Bullard, E Purdom, KD Hansen, S Dudoit; citation_volume=11; citation_publication_date=2010; citation_pages=94; citation_doi=10.1186/1471-2105-11-94; citation_id=CR45"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=Biases in Illumina transcriptome sequencing caused by random hexamer priming; citation_author=K Hansen, S Brenner, S Dudoit; citation_volume=38; citation_publication_date=2010; citation_pages=e131; citation_doi=10.1093/nar/gkq224; citation_id=CR46"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=A scaling normalization method for differential expression analysis of RNA-seq data; citation_author=MD Robinson, A Oshlack; citation_volume=11; citation_publication_date=2010; citation_pages=R25; citation_doi=10.1186/gb-2010-11-3-r25; citation_id=CR47"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=Differential expression analysis for sequence count data; citation_author=S Anders, W Huber; citation_volume=11; citation_publication_date=2010; citation_pages=R106; citation_doi=10.1186/gb-2010-11-10-r106; citation_id=CR48"/>

    <meta name="citation_reference" content="citation_journal_title=Biostatistics; citation_title=Normalization, testing, and false discovery rate estimation for RNA-sequencing data; citation_author=J Li, DM Witten, IM Johnstone, R Tibshirani; citation_volume=13; citation_publication_date=2012; citation_pages=523-38; citation_doi=10.1093/biostatistics/kxr031; citation_id=CR49"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Differential analysis of gene regulation at transcript resolution with RNA-seq; citation_author=C Trapnell, DG Hendrickson, M Sauvageau, L Goff, JL Rinn, L Pachter; citation_volume=31; citation_publication_date=2012; citation_pages=46-53; citation_doi=10.1038/nbt.2450; citation_id=CR50"/>

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Statistical design and analysis of RNA sequencing data; citation_author=PL Auer, RW Doerge; citation_volume=185; citation_publication_date=2010; citation_pages=405-16; citation_doi=10.1534/genetics.110.114983; citation_id=CR51"/>

    <meta name="citation_reference" content="citation_journal_title=Biostatistics; citation_title=Adjusting batch effects in microarray expression data using Empirical Bayes methods; citation_author=WE Johnson, A Rabinovic, C Li; citation_volume=8; citation_publication_date=2007; citation_pages=118-27; citation_doi=10.1093/biostatistics/kxj037; citation_id=CR52"/>

    <meta name="citation_reference" content="citation_journal_title=Biostatistics; citation_title=ARSyN: a method for the identification and removal of systematic noise in multifactorial time course microarray experiments; citation_author=MJ Nueda, A Ferrer, A Conesa; citation_volume=13; citation_publication_date=2012; citation_pages=553-66; citation_doi=10.1093/biostatistics/kxr042; citation_id=CR53"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Moderated statistical tests for assessing differences in tag abundance; citation_author=MD Robinson, GK Smyth; citation_volume=23; citation_publication_date=2007; citation_pages=2881-7; citation_doi=10.1093/bioinformatics/btm453; citation_id=CR54"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=Voom: precision weights unlock linear model analysis tools for RNA-seq read counts; citation_author=CW Law, Y Chen, W Shi, GK Smyth; citation_volume=15; citation_publication_date=2014; citation_pages=R29; citation_doi=10.1186/gb-2014-15-2-r29; citation_id=CR55"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=A comparison of methods for differential expression analysis of RNA-seq data; citation_author=C Soneson, M Delorenzi; citation_volume=14; citation_publication_date=2013; citation_pages=91; citation_doi=10.1186/1471-2105-14-91; citation_id=CR56"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data; citation_author=MD Robinson, DJ McCarthy, GK Smyth; citation_volume=26; citation_publication_date=2010; citation_pages=139-40; citation_doi=10.1093/bioinformatics/btp616; citation_id=CR57"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2; citation_author=MI Love, W Huber, S Anders; citation_volume=15; citation_publication_date=2014; citation_pages=550; citation_doi=10.1186/s13059-014-0550-8; citation_id=CR58"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=baySeq: empirical Bayesian methods for identifying differential expression in sequence count data; citation_author=TJ Hardcastle, KA Kelly; citation_volume=11; citation_publication_date=2010; citation_pages=422; citation_doi=10.1186/1471-2105-11-422; citation_id=CR59"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=EBSeq: an empirical Bayes hierarchical model for inference in RNA-seq experiments; citation_author=N Leng, JA Dawson, JA Thomson, V Ruotti, AI Rissman, BM Smits; citation_volume=29; citation_publication_date=2013; citation_pages=1035-43; citation_doi=10.1093/bioinformatics/btt087; citation_id=CR60"/>

    <meta name="citation_reference" content="citation_journal_title=Stat Methods Med Res; citation_title=Finding consistent patterns: a nonparametric approach for identifying differential expression in RNA-Seq data; citation_author=J Li, R Tibshirani; citation_volume=22; citation_publication_date=2013; citation_pages=519-36; citation_doi=10.1177/0962280211428386; citation_id=CR61"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=DEGseq: an R package for identifying differentially expressed genes from RNA-seq data; citation_author=L Wang, Z Feng, X Wang, X Wang, X Zhang; citation_volume=26; citation_publication_date=2010; citation_pages=136-8; citation_doi=10.1093/bioinformatics/btp612; citation_id=CR62"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=Comprehensive evaluation of differential gene expression analysis methods for RNA-seq data; citation_author=F Rapaport, R Khanin, Y Liang, M Pirun, A Krek, P Zumbo; citation_volume=14; citation_publication_date=2013; citation_pages=R95; citation_doi=10.1186/gb-2013-14-9-r95; citation_id=CR63"/>

    <meta name="citation_reference" content="citation_journal_title=Brief Bioinform; citation_title=Comparison of software packages for detecting differential expression in RNA-seq studies; citation_author=F Seyednasrollah, A Laiho, LL Elo; citation_volume=16; citation_publication_date=2015; citation_pages=59-70; citation_doi=10.1093/bib/bbt086; citation_id=CR64"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=A hierarchical Bayesian model for comparing transcriptomes at the individual transcript isoform level; citation_author=S Zheng, L Chen; citation_volume=37; citation_publication_date=2009; citation_pages=e75; citation_doi=10.1093/nar/gkp282; citation_id=CR65"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Protoc; citation_title=Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks; citation_author=C Trapnell, A Roberts, L Goff, G Pertea, D Kim, DR Kelley; citation_volume=7; citation_publication_date=2012; citation_pages=562-78; citation_doi=10.1038/nprot.2012.016; citation_id=CR66"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=FDM: a graph-based statistical method to detect differential transcription using RNA-seq data; citation_author=D Singh, CF Orellana, Y Hu, CD Jones, Y Liu, DY Chiang; citation_volume=27; citation_publication_date=2011; citation_pages=2633-40; citation_doi=10.1093/bioinformatics/btr458; citation_id=CR67"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=rSeqDiff: detecting differential isoform expression from RNA-Seq data using hierarchical likelihood ratio test; citation_author=Y Shi, H Jiang; citation_volume=8; citation_publication_date=2013; citation_pages=e79448; citation_doi=10.1371/journal.pone.0079448; citation_id=CR68"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=Detecting differential usage of exons from RNA-seq data; citation_author=S Anders, A Reyes, W Huber; citation_volume=22; citation_publication_date=2012; citation_pages=2008-17; citation_doi=10.1101/gr.133744.111; citation_id=CR69"/>

    <meta name="citation_reference" content="citation_journal_title=Gene; citation_title=Identifying differentially spliced genes from two groups of RNA-seq samples; citation_author=W Wang, Z Qin, Z Feng, X Wang, X Zhang; citation_volume=518; citation_publication_date=2013; citation_pages=164-70; citation_doi=10.1016/j.gene.2012.11.045; citation_id=CR70"/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=rMATS: robust and flexible detection of differential alternative splicing from replicate RNA-Seq data; citation_author=S Shen, JW Park, ZX Lu, L Lin, MD Henry, YN Wu; citation_volume=111; citation_publication_date=2014; citation_pages=E5593-601; citation_doi=10.1073/pnas.1419161111; citation_id=CR71"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=Accurate detection of differential RNA processing; citation_author=P Drewe, O Stegle, L Hartmann, A Kahles, R Bohnert, A Wachter; citation_volume=41; citation_publication_date=2013; citation_pages=5189-98; citation_doi=10.1093/nar/gkt211; citation_id=CR72"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=DissSplice: the genome-wide detection of differential splicing events with RNA-seq; citation_author=Y Hu, Y Huang, Y Du, CF Orellana, D Singh, AR Johnson; citation_volume=41; citation_publication_date=2013; citation_pages=e39; citation_doi=10.1093/nar/gks1026; citation_id=CR73"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=ReadXplorer - visualization and analysis of mapped sequences; citation_author=R Hilker, KB Stadermann, D Doppmeier, J Kalinowski, J Stoye, J Straube; citation_volume=30; citation_publication_date=2014; citation_pages=2247-54; citation_doi=10.1093/bioinformatics/btu205; citation_id=CR74"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=The Human Genome Browser at UCSC; citation_author=WJ Kent, CW Sugnet, TS Furey, KM Roskin, TH Pringle, AM Zahler; citation_volume=12; citation_publication_date=2002; citation_pages=996-1006; citation_doi=10.1101/gr.229102. Article published online before print in May 2002; citation_id=CR75"/>

    <meta name="citation_reference" content="citation_journal_title=Brief Bioinformatics; citation_title=Integrative Genomics Viewer (IGV): high- performance genomics data visualization and exploration; citation_author=H Thorvaldsd&#243;ttir, JT Robinson, JP Mesirov; citation_volume=14; citation_publication_date=2012; citation_pages=178-92; citation_doi=10.1093/bib/bbs017; citation_id=CR76"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=Genome Maps, a new generation genome browser; citation_author=I Medina, F Salavert, R Sanchez, A Maria, R Alonso, P Escobar; citation_volume=41; citation_issue=Web Server issue; citation_publication_date=2013; citation_pages=W41-6; citation_doi=10.1093/nar/gkt530; citation_id=CR77"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Savant: genome browser for high-throughput sequencing data; citation_author=M Fiume, V Williams, A Brook, M Brudno; citation_volume=26; citation_publication_date=2010; citation_pages=1938-44; citation_doi=10.1093/bioinformatics/btq332; citation_id=CR78"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=RNAseqViewer: visualization tool for RNA-Seq data; citation_author=X Rog&#233;, X Zhang; citation_volume=30; citation_publication_date=2013; citation_pages=891-2; citation_doi=10.1093/bioinformatics/btt649; citation_id=CR79"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Quantitative visualization of alternative exon expression from RNA-seq data; citation_author=Y Katz, ET Wang, J Silterra, S Schwartz, B Wong, H Thorvaldsd&#243;ttir; citation_volume=31; citation_publication_date=2015; citation_pages=2400-2; citation_doi=10.1093/bioinformatics/btv034; citation_id=CR80"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=SplicePlot: a utility for visualizing splicing quantitative trait loci; citation_author=E Wu, T Nance, SB Montgomery; citation_volume=30; citation_publication_date=2014; citation_pages=1025-6; citation_doi=10.1093/bioinformatics/btt733; citation_id=CR81"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=SpliceSeq: a resource for analysis and visualization of RNA-Seq data on alternative splicing and its functional impacts; citation_author=MC Ryan, J Cleland, R Kim, WC Wong, JN Weinstein; citation_volume=28; citation_publication_date=2012; citation_pages=2385-7; citation_doi=10.1093/bioinformatics/bts452; citation_id=CR82"/>

    <meta name="citation_reference" content="citation_journal_title=Genomics; citation_title=Detection, annotation and visualization of alternative splicing from RNA-Seq data with SplicingViewer; citation_author=Q Liu, C Chen, E Shen, F Zhao, Z Sun, J Wu; citation_volume=99; citation_publication_date=2012; citation_pages=178-82; citation_doi=10.1016/j.ygeno.2011.12.003; citation_id=CR83"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=TraV: a genome context sensitive transcriptome browser; citation_author=S Dietrich, S Wiegand, H Liesegang; citation_volume=9; citation_publication_date=2014; citation_pages=e93677; citation_doi=10.1371/journal.pone.0093677; citation_id=CR84"/>

    <meta name="citation_reference" content="citation_journal_title=BioMed Res Int; citation_title=State-of-the-art fusion-finder algorithms sensitivity and specificity; citation_author=M Carrara, M Beccuti, F Lazzarato, F Cavallo, F Cordero, S Donatelli; citation_volume=15; citation_publication_date=2013; citation_pages=340620; citation_id=CR85"/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Chimeric transcript discovery by paired-end transcriptome sequencing; citation_author=CA Maher, N Palanisamy, JC Brenner, X Cao, S Kalyana-Sundaram, S Luo; citation_volume=106; citation_publication_date=2009; citation_pages=12353-8; citation_doi=10.1073/pnas.0904720106; citation_id=CR86"/>

    <meta name="citation_reference" content="citation_journal_title=Oncogene; citation_title=The landscape and therapeutic relevance of cancer-associated transcript fusions; citation_author=K Yoshihara, Q Wang, W Torres-Garcia, S Zheng, R Vegesna, H Kim; citation_volume=34; citation_publication_date=2015; citation_pages=4845-54; citation_doi=10.1038/onc.2014.406; citation_id=CR87"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS Comput Biol; citation_title=deFuse: an algorithm for gene fusion discovery in tumor RNA-seq data; citation_author=A McPherson, F Hormozdiari, A Zayed, R Giuliany, G Ha, MG Sun; citation_volume=7; citation_publication_date=2011; citation_pages=e1001138; citation_doi=10.1371/journal.pcbi.1001138; citation_id=CR88"/>

    <meta name="citation_reference" content="citation_journal_title=Gene Chromosomes Cancer; citation_title=Poly-gene fusion transcripts and chromothripsis in prostate cancer; citation_author=C Wu, AW Wyatt, A McPherson, D Lin, BJ McConeghy, F Mo; citation_volume=51; citation_publication_date=2012; citation_pages=1144-53; citation_doi=10.1002/gcc.21999; citation_id=CR89"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=Heterogeneity in the inter-tumor transcriptome of high risk prostate cancer; citation_author=AW Wyatt, F Mo, K Wang, B McConeghy, S Brahmbhatt, L Jong; citation_volume=15; citation_publication_date=2014; citation_pages=426; citation_doi=10.1186/s13059-014-0426-y; citation_id=CR90"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Commun; citation_title=The landscape of kinase fusions in cancer; citation_author=N Stransky, E Cerami, S Schalm, JL Kim, C Lengauer; citation_volume=5; citation_publication_date=2014; citation_pages=4846; citation_doi=10.1038/ncomms5846; citation_id=CR91"/>

    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Commonality but diversity in cancer gene fusions; citation_author=TH Rabbitts; citation_volume=137; citation_publication_date=2009; citation_pages=391-5; citation_doi=10.1016/j.cell.2009.04.034; citation_id=CR92"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Comrad: detection of expressed rearrangements by integrated analysis of RNA-Seq and low coverage genome sequence data; citation_author=A McPherson, C Wu, I Hajirasouliha, F Hormozdiari, F Hach, A Lapuk; citation_volume=27; citation_publication_date=2011; citation_pages=1481-8; citation_doi=10.1093/bioinformatics/btr184; citation_id=CR93"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=ChimeraScan: a tool for identifying chimeric transcription in sequencing data; citation_author=MK Iyer, AM Chinnaiyan, CA Maher; citation_volume=27; citation_publication_date=2011; citation_pages=2903-4; citation_doi=10.1093/bioinformatics/btr467; citation_id=CR94"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=Discovery of non-ETS gene fusions in human prostate cancer using next-generation RNA sequencing; citation_author=D Pflueger, S Terry, A Sboner, L Habegger, R Esgueva, PC Lin; citation_volume=21; citation_publication_date=2011; citation_pages=56-67; citation_doi=10.1101/gr.110684.110; citation_id=CR95"/>

    <meta name="citation_reference" content="citation_journal_title=RNA Biol; citation_title=mirTools 2.0 for non-coding RNA discovery, profiling, and functional annotation based on high-throughput sequencing; citation_author=J Wu, Q Liu, X Wang, J Zheng, T Wang, M You; citation_volume=10; citation_publication_date=2013; citation_pages=1087-92; citation_doi=10.4161/rna.25193; citation_id=CR96"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Methods; citation_title=Fast gapped-read alignment with Bowtie 2; citation_author=B Langmead, SL Salzberg; citation_volume=9; citation_publication_date=2012; citation_pages=357-9; citation_doi=10.1038/nmeth.1923; citation_id=CR97"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform; citation_author=H Li, R Durbin; citation_volume=25; citation_publication_date=2009; citation_pages=1754-60; citation_doi=10.1093/bioinformatics/btp324; citation_id=CR98"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=PatMaN: rapid alignment of short sequences to large databases; citation_author=K Pr&#252;fer, U Stenzel, M Dannemann, RE Green, M Lachmann, J Kelso; citation_volume=24; citation_publication_date=2008; citation_pages=1530-1; citation_doi=10.1093/bioinformatics/btn223; citation_id=CR99"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=MicroRazerS: rapid alignment of small RNA reads; citation_author=AK Emde, M Grunert, D Weese, K Reinert, SR Sperling; citation_volume=26; citation_publication_date=2010; citation_pages=123-4; citation_doi=10.1093/bioinformatics/btp601; citation_id=CR100"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=miRDeep*: an integrated application tool for miRNA identification from RNA sequencing data; citation_author=J An, J Lai, ML Lehman, CC Nelson; citation_volume=41; citation_publication_date=2013; citation_pages=727-37; citation_doi=10.1093/nar/gks1187; citation_id=CR101"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=miRDeep-P: a computational tool for analyzing the microRNA transcriptome in plants; citation_author=X Yang, L Li; citation_volume=27; citation_publication_date=2011; citation_pages=2614-5; citation_id=CR102"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=The UEA sRNA workbench: a suite of tools for analysing and visualizing next generation sequencing microRNA and small RNA datasets; citation_author=MB Stocks, S Moxon, D Mapleson, HC Woolfenden, I Mohorianu, L Folkes; citation_volume=28; citation_publication_date=2012; citation_pages=2059-61; citation_doi=10.1093/bioinformatics/bts311; citation_id=CR103"/>

    <meta name="citation_reference" content="citation_journal_title=RNA; citation_title=ShortStack: comprehensive annotation and quantification of small RNA genes; citation_author=MJ Axtell; citation_volume=19; citation_publication_date=2013; citation_pages=740-51; citation_doi=10.1261/rna.035279.112; citation_id=CR104"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=iMir: an integrated pipeline for high-throughput analysis of small non-coding RNA data obtained by smallRNA-Seq; citation_author=G Giurato, MR Filippo, A Rinaldi, A Hashim, G Nassa, M Ravo; citation_volume=14; citation_publication_date=2013; citation_pages=362; citation_doi=10.1186/1471-2105-14-362; citation_id=CR105"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=Gene ontology analysis for RNA-seq: accounting for selection bias; citation_author=MD Young, MJ Wakefield, GK Smyth, A Oshlack; citation_volume=11; citation_publication_date=2010; citation_pages=1-12; citation_doi=10.1186/gb-2010-11-2-r14; citation_id=CR106"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=GSVA: gene set variation analysis for microarray and RNA-Seq data; citation_author=S H&#228;nzelmann, R Castelo, J Guinney; citation_volume=14; citation_publication_date=2013; citation_pages=7; citation_doi=10.1186/1471-2105-14-7; citation_id=CR107"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=Gene set enrichment analysis of RNA-Seq data: integrating differential expression and splicing; citation_author=X Wang, MJ Cairns; citation_volume=14; citation_issue=Suppl 5; citation_publication_date=2013; citation_pages=S16; citation_doi=10.1186/1471-2105-14-S5-S16; citation_id=CR108"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Genet; citation_title=Gene Ontology: tool for the unification of biology; citation_author=M Ashburner, CA Ball, JA Blake, D Botstein, H Butler, JM Cherry; citation_volume=25; citation_publication_date=2000; citation_pages=25-9; citation_doi=10.1038/75556; citation_id=CR109"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Methods; citation_title=Orchestrating high-throughput genomic analysis with Bioconductor; citation_author=W Huber, VJ Carey, R Gentleman, S Anders, M Carlson, BS Carvalho; citation_volume=12; citation_publication_date=2015; citation_pages=115-21; citation_doi=10.1038/nmeth.3252; citation_id=CR110"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Protocols; citation_title=Systematic and integrative analysis of large gene lists using DAVID Bioinformatics Resources; citation_author=DW Huang, BT Sherman, RA Lempicki; citation_volume=4; citation_publication_date=2009; citation_pages=44-57; citation_doi=10.1038/nprot.2008.211; citation_id=CR111"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists; citation_author=DW Huang, BT Sherman, RA Lempicki; citation_volume=37; citation_publication_date=2009; citation_pages=1-13; citation_doi=10.1093/nar/gkn923; citation_id=CR112"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=Babelomics: an integrative platform for the analysis of transcriptomics, proteomics and genomic data with advanced functional profiling; citation_author=I Medina, J Carbonell, L Pulido, SC Madeira, S Goetz, A Conesa; citation_volume=38; citation_issue=suppl 2; citation_publication_date=2010; citation_pages=W210-3; citation_doi=10.1093/nar/gkq388; citation_id=CR113"/>

    <meta name="citation_reference" content="citation_journal_title=Brief Bioinformatics; citation_title=Swiss-Prot: juggling between evolution and stability; citation_author=A Bairoch, B Boeckmann, S Ferro, E Gasteiger; citation_volume=5; citation_publication_date=2004; citation_pages=39-55; citation_doi=10.1093/bib/5.1.39; citation_id=CR114"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=The Pfam protein families database; citation_author=RD Finn, A Bateman, J Clements, P Coggill, RY Eberhardt, SR Eddy; citation_volume=42; citation_issue=Database issue; citation_publication_date=2014; citation_pages=D222-30; citation_doi=10.1093/nar/gkt1223; citation_id=CR115"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=InterPro in 2011: new developments in the family and domain prediction database; citation_author=S Hunter, P Jones, A Mitchell, R Apweiler, TK Attwood, A Bateman; citation_volume=40; citation_issue=Database issue; citation_publication_date=2011; citation_pages=D306-12; citation_id=CR116"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research; citation_author=A Conesa, S G&#246;tz, JM Garc&#237;a-G&#243;mez, J Terol, M Tal&#243;n, M Robles; citation_volume=21; citation_publication_date=2005; citation_pages=3674-6; citation_doi=10.1093/bioinformatics/bti610; citation_id=CR117"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=Rfam: updates to the RNA families database; citation_author=PP Gardner, J Daub, JG Tate, EP Nawrocki, DL Kolbe, S Lindgreen; citation_volume=37; citation_issue=suppl 1; citation_publication_date=2009; citation_pages=D136-40; citation_doi=10.1093/nar/gkn766; citation_id=CR118"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=miRBase: annotating high confidence microRNAs using deep sequencing data; citation_author=A Kozomara, S Griffiths-Jones; citation_volume=42; citation_issue=Database issue; citation_publication_date=2014; citation_pages=D68-73; citation_doi=10.1093/nar/gkt1181; citation_id=CR119"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=MicroRNA targets in Drosophila
                           ; citation_author=AJ Enright, B John, U Gaul, T Tuschl, C Sander, DS Marks; citation_volume=5; citation_publication_date=2003; citation_pages=R1; citation_doi=10.1186/gb-2003-5-1-r1; citation_id=CR120"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS Genet; citation_title=Bayesian test for colocalisation between pairs of genetic association studies using summary statistics; citation_author=C Giambartolomei, D Vukcevic, EE Schadt, L Franke, AD Hingorani, C Wallace; citation_volume=10; citation_publication_date=2014; citation_pages=e1004383; citation_doi=10.1371/journal.pgen.1004383; citation_id=CR121"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Genetic variants regulating ORMDL3 expression contribute to the risk of childhood asthma; citation_author=MF Moffatt, M Kabesch, L Liang, AL Dixon, D Strachan, S Heath; citation_volume=448; citation_publication_date=2007; citation_pages=470-3; citation_doi=10.1038/nature06014; citation_id=CR122"/>

    <meta name="citation_reference" content="citation_journal_title=Trends Genet; citation_title=Revealing the architecture of gene regulation: the promise of eQTL studies; citation_author=Y Gilad, S Rifkin, J Pritchard; citation_volume=24; citation_publication_date=2008; citation_pages=408-15; citation_doi=10.1016/j.tig.2008.06.001; citation_id=CR123"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS Genet; citation_title=Global properties and functional complexity of human gene regulatory variation; citation_author=D Gaffney; citation_volume=9; citation_publication_date=2013; citation_pages=e1003501; citation_doi=10.1371/journal.pgen.1003501; citation_id=CR124"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Transcriptome genetics using second generation sequencing in a Caucasian population; citation_author=S Montgomery, M Sammeth, M Gutierrez-Arcelus, RP Lach, C Ingle, J Nisbett; citation_volume=464; citation_publication_date=2010; citation_pages=773-7; citation_doi=10.1038/nature08903; citation_id=CR125"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Understanding mechanisms underlying human gene expression variation with RNA sequencing; citation_author=JK Pickrell, JC Marioni, AA Pai, JF Degner, BE Engelhardt, E Nkadori; citation_volume=464; citation_publication_date=2010; citation_pages=768-72; citation_doi=10.1038/nature08872; citation_id=CR126"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Transcriptome and genome sequencing uncovers functional variation in humans; citation_author=T Lappalainen, M Sammeth, M Friedlander, PA &#8216;t Hoen, J Monlong, MA Rivas; citation_volume=501; citation_publication_date=2013; citation_pages=506-11; citation_doi=10.1038/nature12531; citation_id=CR127"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=Characterizing the genetic basis of transcriptome diversity through RNA-sequencing of 922 individuals; citation_author=A Battle, S Mostafavi, X Zhu, JB Potash, MM Weissman, J Shi; citation_volume=24; citation_publication_date=2014; citation_pages=14-24; citation_doi=10.1101/gr.155192.113; citation_id=CR128"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Rev Genet; citation_title=Genome-wide allele-specific analysis: insights into regulatory variation; citation_author=T Pastinen; citation_volume=11; citation_publication_date=2010; citation_pages=533-8; citation_doi=10.1038/nrg2815; citation_id=CR129"/>

    <meta name="citation_reference" content="citation_journal_title=Biometrics; citation_title=A statistical framework for eQTL mapping using RNA-seq data; citation_author=W Sun; citation_volume=68; citation_publication_date=2012; citation_pages=1-11; citation_doi=10.1111/j.1541-0420.2011.01654.x; citation_id=CR130"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Methods; citation_title=WASP: allele-specific for robust molecular quantitative trait locus discovery; citation_author=B Geijn, G McVicker, Y Gilad, JK Pritchard; citation_volume=12; citation_publication_date=2015; citation_pages=1061-3; citation_doi=10.1038/nmeth.3582; citation_id=CR131"/>

    <meta name="citation_reference" content="Kumasaka N, Knights AJ, Gaffney DJ. Fine-mapping cellular QTLs with RASQUAL and ATAC-seq. Nat Genet. 2015. doi: 
                    10.1038/ng.3467
                    
                  ."/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Statistical significance for genome-wide studies; citation_author=JD Storey, R Tibshirani; citation_volume=100; citation_publication_date=2003; citation_pages=9440-5; citation_doi=10.1073/pnas.1530509100; citation_id=CR133"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS Genet; citation_title=Genome-wide associations of gene expression variation in humans; citation_author=BE Stranger, MS Forrest, AG Clark, MJ Minichiello, S Deutsch, R Lyle; citation_volume=1; citation_publication_date=2005; citation_pages=e78; citation_doi=10.1371/journal.pgen.0010078; citation_id=CR134"/>

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Polarization of the effects of autoimmune and neurodegenerative risk alleles in leukocytes; citation_author=T Raj, K Rothamel, S Mostafavi, C Ye, MN Lee, JM Replogle; citation_volume=344; citation_publication_date=2014; citation_pages=519-23; citation_doi=10.1126/science.1249547; citation_id=CR135"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Matrix eQTL: ultra fast eQTL analysis via large matrix operations; citation_author=A Shabalin; citation_volume=28; citation_publication_date=2012; citation_pages=1353-8; citation_doi=10.1093/bioinformatics/bts163; citation_id=CR136"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Methods; citation_title=Comparative analysis of algorithms for integration of copy number and expression data; citation_author=R Louhimo, T Lepikhova, O Monni, S Hautaniemi; citation_volume=9; citation_publication_date=2012; citation_pages=351-5; citation_doi=10.1038/nmeth.1893; citation_id=CR137"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=Deep sequencing reveals distinct patterns of DNA methylation in prostate cancer; citation_author=JH Kim, SM Dhanasekaran, JR Prensner, X Cao, D Robinson, S Kalyana-Sundaram; citation_volume=21; citation_publication_date=2011; citation_pages=1028-41; citation_doi=10.1101/gr.119347.110; citation_id=CR138"/>

    <meta name="citation_reference" content="citation_journal_title=Sci Rep; citation_title=Genome-wide methylated CpG island profiles of melanoma cells reveal a melanoma coregulation network; citation_author=JL Li, J Mazar, C Zhong, GJ Faulkner, SS Govindarajan, Z Zhang; citation_volume=3; citation_publication_date=2013; citation_pages=2962; citation_id=CR139"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Syst Biol; citation_title=An integrative analysis of DNA methylation and RNA-Seq data for human heart, kidney and liver; citation_author=L Xie, B Weichel, JE Ohm, K Zhang; citation_volume=5; citation_issue=Suppl 3; citation_publication_date=2011; citation_pages=S4; citation_doi=10.1186/1752-0509-5-S3-S4; citation_id=CR140"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=Genetic analysis of DNA methylation and gene expression levels in whole blood of healthy human subjects; citation_author=KR Eijk, S Jong, MP Boks, T Langeveld, F Colas, JH Veldink; citation_volume=13; citation_publication_date=2012; citation_pages=636; citation_doi=10.1186/1471-2164-13-636; citation_id=CR141"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Epigenome-wide association data implicate DNA methylation as an intermediary of genetic risk in rheumatoid arthritis; citation_author=Y Liu, MJ Aryee, L Padyukov, MD Fallin, E Hesselberg, A Runarsson; citation_volume=31; citation_publication_date=2013; citation_pages=142-7; citation_doi=10.1038/nbt.2487; citation_id=CR142"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=An integrated analysis of molecular aberrations in NCI-60 cell lines; citation_author=C-H Yeang; citation_volume=11; citation_publication_date=2010; citation_pages=495; citation_id=CR143"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Med Genomics; citation_title=An empirical Bayes model for gene expression and methylation profiles in antiestrogen resistant breast cancer; citation_author=J Jeong, L Li, Y Liu, KP Nephew, YHM Huang, C Shen; citation_volume=3; citation_publication_date=2010; citation_pages=55; citation_doi=10.1186/1755-8794-3-55; citation_id=CR144"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=A systems-level integrative framework for genome-wide DNA methylation and gene expression data identifies differential gene expression modules under epigenetic control; citation_author=Y Jiao, M Widschwendter, AE Teschendorff; citation_volume=30; citation_publication_date=2014; citation_pages=2360-6; citation_doi=10.1093/bioinformatics/btu316; citation_id=CR145"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Protoc; citation_title=Target analysis by integration of transcriptome and ChIP-seq data with BETA; citation_author=S Wang, H Sun, J Ma, C Zang, C Wang, J Wang; citation_volume=8; citation_publication_date=2013; citation_pages=2502-15; citation_doi=10.1038/nprot.2013.150; citation_id=CR146"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Integrative analysis of 111 reference human epigenomes; citation_author=A Kundaje, W Meuleman, J Ernst, M Bilenky, A Yen; citation_volume=518; citation_publication_date=2015; citation_pages=317-30; citation_doi=10.1038/nature14248; citation_id=CR147"/>

    <meta name="citation_reference" content="citation_journal_title=BioData Min; citation_title=Uncovering correlated variability in epigenomic datasets using the Karhunen-Loeve transform; citation_author=P Madrigal, P Krajewski; citation_volume=8; citation_publication_date=2015; citation_pages=20; citation_doi=10.1186/s13040-015-0051-7; citation_id=CR148"/>

    <meta name="citation_reference" content="citation_journal_title=Front Cell Dev Biol; citation_title=Understanding gene regulatory mechanisms by integrating ChIP-seq and RNA-seq data: statistical solutions to biological problems; citation_author=C Angelini, V Costa; citation_volume=2; citation_publication_date=2014; citation_pages=51; citation_doi=10.3389/fcell.2014.00051; citation_id=CR149"/>

    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Circuitry and dynamics of human transcription factor regulatory networks; citation_author=S Neph, AB Stergachis, A Reynolds, R Sandstrom, E Borenstein, JA Stamatoyannopoulos; citation_volume=150; citation_publication_date=2012; citation_pages=1274-86; citation_doi=10.1016/j.cell.2012.04.040; citation_id=CR150"/>

    <meta name="citation_reference" content="citation_journal_title=J Biomed Inform; citation_title=miRWalk - database: prediction of possible miRNA binding sites by &#8216;walking&#8217; the genes of 3 genomes; citation_author=H Dweep, C Sticht, P Pandey, N Gretz; citation_volume=44; citation_publication_date=2011; citation_pages=839-47; citation_doi=10.1016/j.jbi.2011.05.002; citation_id=CR151"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=miRBase: tools for microRNA genomics; citation_author=S Griffiths-Jones, HK Saini, S Dongen, AJ Enright; citation_volume=36; citation_publication_date=2008; citation_pages=D154-8; citation_doi=10.1093/nar/gkm952; citation_id=CR152"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=CORNA: testing gene lists for regulation by microRNAs; citation_author=X Wu, M Watson; citation_volume=25; citation_publication_date=2009; citation_pages=832-3; citation_doi=10.1093/bioinformatics/btp059; citation_id=CR153"/>

    <meta name="citation_reference" content="citation_journal_title=IEEE Trans Nanobiosci; citation_title=BioVLAB-MMIA: a cloud environment for microRNA and mRNA integrated analysis (MMIA) on Amazon EC2; citation_author=H Lee, Y Yang, H Chae, S Nam, D Choi, P Tangchaisin; citation_volume=11; citation_publication_date=2012; citation_pages=266-72; citation_doi=10.1109/TNB.2011.2159621; citation_id=CR154"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=MicroRNA and mRNA integrated analysis (MMIA): a web tool for examining biological functions of microRNA expression; citation_author=S Nam, M Li, K Choi, C Balch, S Kim, KP Nephew; citation_volume=37; citation_publication_date=2009; citation_pages=W356-62; citation_doi=10.1093/nar/gkp294; citation_id=CR155"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=MAGIA, a web-based tool for miRNA and Genes Integrated Analysis; citation_author=G Sales, A Coppe, A Bisognin, S Bortoluzzi, C Romualdi; citation_volume=38; citation_publication_date=2010; citation_pages=W352-9; citation_doi=10.1093/nar/gkq423; citation_id=CR156"/>

    <meta name="citation_reference" content="Icay K, Chen P, Cervera C, Lehtonen R, Hautaniemi S. SePIA: RNA and smallRNA-sequence processing, integration, and analysis. 2015. 
                    http://anduril.org/sepia
                    
                  . Accessed 6 Jan 2016."/>

    <meta name="citation_reference" content="citation_journal_title=Mol Biosyst; citation_title=Global signatures of protein and mRNA expression levels; citation_author=AR Sousa, LO Penalva, EM Marcotte, C Vogel; citation_volume=5; citation_publication_date=2009; citation_pages=1512-26; citation_id=CR158"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Rev Genet; citation_title=Insights into the regulation of protein abundance from proteomic and transcriptomic analyses; citation_author=C Vogel, EM Marcotte; citation_volume=13; citation_publication_date=2012; citation_pages=227-32; citation_id=CR159"/>

    <meta name="citation_reference" content="citation_journal_title=Cell Rep; citation_title=Quantitative and qualitative proteome characteristics extracted from in-depth integrated genomics and proteomics analysis; citation_author=TY Low, S Heesch, H Toorn, P Giansanti, A Cristobal, P Toonen; citation_volume=5; citation_publication_date=2013; citation_pages=1469-78; citation_doi=10.1016/j.celrep.2013.10.041; citation_id=CR160"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=MassTRIX: mass translator into pathways; citation_author=K Suhre, P Schmitt-Kopplin; citation_volume=36; citation_issue=Web Server issue; citation_publication_date=2008; citation_pages=W481-4; citation_doi=10.1093/nar/gkn194; citation_id=CR161"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Paintomics: a web based tool for the joint visualization of transcriptomics and metabolomics data; citation_author=F Garc&#237;a-Alcalde, F Garc&#237;a-L&#243;pez, J Dopazo, A Conesa; citation_volume=27; citation_publication_date=2011; citation_pages=137-9; citation_doi=10.1093/bioinformatics/btq594; citation_id=CR162"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Syst Biol; citation_title=VANTED v2: a framework for systems biology applications; citation_author=H Rohn, A Junker, A Hartmann, E Grafahrend-Belau, H Treutler, M Klapperst&#252;ck; citation_volume=6; citation_publication_date=2012; citation_pages=139; citation_doi=10.1186/1752-0509-6-139; citation_id=CR163"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=SteinerNet: a web server for integrating &#8216;omic&#8217; data to discover hidden components of response pathways; citation_author=N Tuncbag, S McCallum, SS Huang, E Fraenkel; citation_volume=40; citation_publication_date=2012; citation_pages=W505-9; citation_doi=10.1093/nar/gks445; citation_id=CR164"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=A novel computational framework for simultaneous integration of multiple types of genomic data to identify microRNA-gene regulatory modules; citation_author=S Zhang, Q Li, J Liu, XJ Zhou; citation_volume=27; citation_publication_date=2011; citation_pages=i401-9; citation_doi=10.1093/bioinformatics/btr206; citation_id=CR165"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Integrating sequence, expression and interaction data to determine condition-specific miRNA regulation; citation_author=H-S Le, Z Bar-Joseph; citation_volume=29; citation_publication_date=2013; citation_pages=i89-97; citation_doi=10.1093/bioinformatics/btt231; citation_id=CR166"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=A modular framework for gene set analysis integrating multilevel omics data; citation_author=S Sass, F Buettner, NS Mueller, FJ Theis; citation_volume=41; citation_publication_date=2013; citation_pages=9622-33; citation_doi=10.1093/nar/gkt752; citation_id=CR167"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Syst Biol; citation_title=3Omics: a web-based systems biology tool for analysis, integration and visualization of human transcriptomic, proteomic and metabolomic data; citation_author=TC Kuo, TF Tian, YJ Tseng; citation_volume=7; citation_publication_date=2013; citation_pages=64; citation_doi=10.1186/1752-0509-7-64; citation_id=CR168"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Med; citation_title=Large-scale data integration framework provides a comprehensive view on glioblastoma multiforme; citation_author=K Ovaska, M Laakso, S Haapa-Paananen, R Louhimo, P Chen, V Aittom&#228;ki; citation_volume=2; citation_publication_date=2010; citation_pages=65; citation_doi=10.1186/gm186; citation_id=CR169"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=Galaxy: a comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences; citation_author=J Goecks, A Nekrutenko, J Taylor; citation_volume=11; citation_publication_date=2010; citation_pages=R86; citation_doi=10.1186/gb-2010-11-8-r86; citation_id=CR170"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=Chipster: user-friendly analysis software for microarray and other high-throughput data; citation_author=MA Kallio, JT Tuimala, T Hupponen, P Klemel&#228;, M Gentile, I Scheinin; citation_volume=12; citation_publication_date=2011; citation_pages=507; citation_doi=10.1186/1471-2164-12-507; citation_id=CR171"/>

    <meta name="citation_reference" content="citation_journal_title=J Proteome Res; citation_title=Tools to covisualize and coanalyze proteomic data with genomes and transcriptomes: validation of genes and alternative mRNA splicing; citation_author=CNI Pang, AP Tay, C Aya; citation_volume=13; citation_publication_date=2014; citation_pages=84-98; citation_doi=10.1021/pr400820p; citation_id=CR172"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Full-length mRNA-Seq from single-cell levels of RNA and individual circulating tumor cells; citation_author=D Ramsk&#246;ld, S Luo, YC Wang, R Li, Q Deng, OR Faridani; citation_volume=30; citation_publication_date=2012; citation_pages=777-82; citation_doi=10.1038/nbt.2282; citation_id=CR173"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Methods; citation_title=Smart-seq2 for sensitive full-length transcriptome profiling in single cells; citation_author=S Picelli, &#197;K Bj&#246;rklund, OR Faridani, S Sagasser, G Winberg, R Sandberg; citation_volume=10; citation_publication_date=2013; citation_pages=1096-8; citation_doi=10.1038/nmeth.2639; citation_id=CR174"/>

    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Highly parallel genome-wide expression profiling of individual cells using nanoliter droplets; citation_author=EZ Macosko, A Basu, R Satija, J Nemesh, K Shekhar, M Goldman; citation_volume=161; citation_publication_date=2015; citation_pages=1202-14; citation_doi=10.1016/j.cell.2015.05.002; citation_id=CR175"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=From single-cell to cell-pool transcriptomes: stochasticity in gene expression and RNA splicing; citation_author=GK Marinov, BA Williams, K McCue, GP Schroth, J Gertz, RM Myers; citation_volume=24; citation_publication_date=2014; citation_pages=496-510; citation_doi=10.1101/gr.161034.113; citation_id=CR176"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Methods; citation_title=Quantitative single-cell RNA-seq with unique molecular identifiers; citation_author=S Islam, A Zeisel, S Joost, G Manno, P Zajac, M Kasper; citation_volume=11; citation_publication_date=2014; citation_pages=163-6; citation_doi=10.1038/nmeth.2772; citation_id=CR177"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Methods; citation_title=Counting absolute numbers of molecules using unique molecular identifiers; citation_author=T Kivioja, A V&#228;h&#228;rautio, K Karlsson, M Bonke, M Enge, S Linnarsson; citation_volume=9; citation_publication_date=2011; citation_pages=72-4; citation_doi=10.1038/nmeth.1778; citation_id=CR178"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Methods; citation_title=Accounting for technical noise in single-cell RNA-seq experiments; citation_author=P Brennecke, S Anders, JK Kim, AA Ko&#322;odziejczyk, X Zhang, V Proserpio; citation_volume=10; citation_publication_date=2013; citation_pages=1093-5; citation_doi=10.1038/nmeth.2645; citation_id=CR179"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Rev Genet; citation_title=Computational and analytical challenges in single-cell transcriptomics; citation_author=O Stegle, SA Teichmann, JC Marioni; citation_volume=16; citation_publication_date=2015; citation_pages=133-45; citation_doi=10.1038/nrg3833; citation_id=CR180"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells; citation_author=C Trapnell, D Cacchiarelli; citation_volume=32; citation_publication_date=2014; citation_pages=381-6; citation_doi=10.1038/nbt.2859; citation_id=CR181"/>

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Single-cell DNA-methylation analysis reveals epigenetic chimerism in preimplantation embryos; citation_author=C Lorthongpanich, LF Cheow, S Balu, SR Quake, BB Knowles, WF Burkholder; citation_volume=341; citation_publication_date=2013; citation_pages=1110-2; citation_doi=10.1126/science.1240617; citation_id=CR182"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Single-cell chromatin accessibility reveals principles of regulatory variation; citation_author=JD Buenrostro, B Wu, UM Litzenburger, D Ruff, ML Gonzales, MP Snyder; citation_volume=523; citation_publication_date=2015; citation_pages=486-90; citation_doi=10.1038/nature14590; citation_id=CR183"/>

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Multiplex single-cell profiling of chromatin accessibility by combinatorial cellular indexing; citation_author=DA Cusanovich, R Daza, A Adey, HA Pliner, L Christiansen, KL Gunderson; citation_volume=348; citation_publication_date=2015; citation_pages=910-4; citation_doi=10.1126/science.aab1601; citation_id=CR184"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Comprehensive transcriptome analysis using synthetic long-read sequencing reveals molecular co-association of distant splicing events; citation_author=H Tilgner, F Jahanbani, T Blauwkamp, A Moshrefi, E Jaeger, F Chen; citation_volume=33; citation_publication_date=2015; citation_pages=736-42; citation_doi=10.1038/nbt.3242; citation_id=CR185"/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Characterization of the human ESC transcriptome by hybrid sequencing; citation_author=KF Au, V Sebastiano, PT Afshar, JD Durruthy, L Lee, BA Williams; citation_volume=110; citation_publication_date=2013; citation_pages=E4821-30; citation_doi=10.1073/pnas.1320101110; citation_id=CR186"/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Defining a personal, allele-specific, and single-molecule long-read transcriptome; citation_author=H Tilgner, F Grubert, D Sharon, MP Snyder; citation_volume=111; citation_publication_date=2014; citation_pages=9869-74; citation_doi=10.1073/pnas.1400447111; citation_id=CR187"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Improving PacBio long read accuracy by short read alignment; citation_author=KF Au, JG Underwood, L Lee, WH Wong; citation_volume=7; citation_publication_date=2012; citation_pages=e46679; citation_doi=10.1371/journal.pone.0046679; citation_id=CR188"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Sequencing technology does not eliminate biological variability; citation_author=KD Hansen, Z Wu, RA Irizarry, JT Leek; citation_volume=29; citation_publication_date=2011; citation_pages=572-3; citation_doi=10.1038/nbt.1910; citation_id=CR189"/>

    <meta name="citation_reference" content="citation_journal_title=J Comput Biol; citation_title=Calculating sample size estimates for RNA sequencing data; citation_author=SN Hart, TM Therneau, Y Zhang, GA Poland, JP Kocher; citation_volume=20; citation_publication_date=2013; citation_pages=970-8; citation_doi=10.1089/cmb.2012.0283; citation_id=CR190"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Scotty: a web tool for designing RNA-Seq experiments to measure differential gene expression; citation_author=MA Busby, C Stewart, CA Miller, KR Grzeda, GT Marth; citation_volume=29; citation_publication_date=2013; citation_pages=656-7; citation_doi=10.1093/bioinformatics/btt015; citation_id=CR191"/>

    <meta name="citation_reference" content="citation_journal_title=Biol Direct; citation_title=Transcript length bias in RNA-seq data confounds systems biology; citation_author=A Oshlack, MJ Wakefield; citation_volume=4; citation_publication_date=2009; citation_pages=14; citation_doi=10.1186/1745-6150-4-14; citation_id=CR192"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=How does multiple testing correction work?; citation_author=WS Noble; citation_volume=27; citation_publication_date=2009; citation_pages=1135-7; citation_doi=10.1038/nbt1209-1135; citation_id=CR193"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=subSeq: determining appropriate sequencing depth through efficient read subsampling; citation_author=DG Robinson, JD Storey; citation_volume=30; citation_publication_date=2014; citation_pages=3424-6; citation_doi=10.1093/bioinformatics/btu552; citation_id=CR194"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=RNA-seq differential expression studies: more sequence or more replication?; citation_author=Y Liu, J Zhou, KP White; citation_volume=30; citation_publication_date=2013; citation_pages=301-4; citation_doi=10.1093/bioinformatics/btt688; citation_id=CR195"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=A comprehensive assessment of RNA-seq accuracy, reproducibility and information content by the Sequencing Quality Control Consortium; citation_author=; citation_volume=32; citation_publication_date=2014; citation_pages=903-14; citation_doi=10.1038/nbt.2957; citation_id=CR196"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=Synthetic spike-in standards for RNA-seq experiments; citation_author=L Jiang, F Schlesinger, CA Davis, Y Zhang, R Li, M Salit; citation_volume=21; citation_publication_date=2011; citation_pages=1543-51; citation_doi=10.1101/gr.121095.111; citation_id=CR197"/>

    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Global regulation of promoter melting in naive lymphocytes; citation_author=F Kouzine, D Wojtowicz, A Yamane, W Resch, KR Kieffer-Kwon, R Bandle; citation_volume=153; citation_publication_date=2013; citation_pages=988-99; citation_doi=10.1016/j.cell.2013.04.033; citation_id=CR198"/>

    <meta name="citation_reference" content="citation_journal_title=Exp Cell Res; citation_title=Library preparation methods for next-generation sequencing: tone down the bias; citation_author=EL Dijk, Y Jaszczyszyn, C Thermes; citation_volume=322; citation_publication_date=2014; citation_pages=12-20; citation_doi=10.1016/j.yexcr.2014.01.008; citation_id=CR199"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=TopHat: discovering splice junctions with RNA-Seq; citation_author=C Trapnell, L Pachter, SL Salzberg; citation_volume=25; citation_publication_date=2009; citation_pages=1105-11; citation_doi=10.1093/bioinformatics/btp120; citation_id=CR200"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions; citation_author=D Kim, G Pertea, C Trapnell, H Pimentel, R Kelley, SL Salzberg; citation_volume=14; citation_publication_date=2013; citation_pages=R36; citation_doi=10.1186/gb-2013-14-4-r36; citation_id=CR201"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Fast and SNP-tolerant detection of complex variants and splicing in short reads; citation_author=TD Wu, S Nacu; citation_volume=26; citation_publication_date=2010; citation_pages=873-81; citation_doi=10.1093/bioinformatics/btq057; citation_id=CR202"/>

    <meta name="citation_reference" content="Jean G, Kahles A, Sreedharan VT, De Bona F, R&#228;tsch G. RNA-Seq read alignments with PALMapper. Curr Protoc Bioinformatics. 2010;11(6)."/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=MapSplice: accurate mapping of RNA-seq reads for splice junction discovery; citation_author=K Wang, D Singh, Z Zeng, SJ Coleman, Y Huang, GL Savich; citation_volume=38; citation_publication_date=2010; citation_pages=e178; citation_doi=10.1093/nar/gkq622; citation_id=CR204"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Methods; citation_title=The GEM mapper: fast, accurate and versatile alignment by filtration; citation_author=S Marco-Sola, M Sammeth, R Guig&#243;, P Ribeca; citation_volume=9; citation_publication_date=2012; citation_pages=1185-8; citation_doi=10.1038/nmeth.2221; citation_id=CR205"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=A comprehensive evaluation of ensembl, RefSeq, and UCSC annotations in the context of RNA-seq read mapping and gene quantification; citation_author=S Zhao, B Zhang; citation_volume=16; citation_publication_date=2015; citation_pages=97; citation_doi=10.1186/s12864-015-1308-8; citation_id=CR206"/>

    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome; citation_author=B Langmead, C Trapnell, M Pop, SL Salzberg; citation_volume=10; citation_publication_date=2009; citation_pages=R25; citation_doi=10.1186/gb-2009-10-3-r25; citation_id=CR207"/>

    <meta name="citation_reference" content="citation_journal_title=Am J Bot; citation_title=A comparison of statistical methods for detecting differentially expressed genes from RNA-seq data; citation_author=VM Kvam, P Liu, Y Si; citation_volume=99; citation_publication_date=2012; citation_pages=248-56; citation_doi=10.3732/ajb.1100340; citation_id=CR208"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=Efficient experimental design and analysis strategies for the detection of differential expression using RNA-Sequencing; citation_author=JA Robles, SE Qureshi, SJ Stephen, SR Wilson, CJ Burden, JM Taylor; citation_volume=13; citation_publication_date=2012; citation_pages=484; citation_doi=10.1186/1471-2164-13-484; citation_id=CR209"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=A comprehensive comparison of RNA-Seq-based transcriptome analysis from reads to differential gene expression and cross-comparison with microarrays: a case study in Saccharomyces cerevisiae
                           ; citation_author=I Nookaew, M Papini, N Pornputtapong, G Scalcinati, L Fagerberg, M Uhl&#233;n; citation_volume=40; citation_publication_date=2012; citation_pages=10084-97; citation_doi=10.1093/nar/gks804; citation_id=CR210"/>

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=ROTS: reproducible RNA-seq biomarker detector-prognostic markers for clear cell renal cell cancer; citation_author=F Seyednasrollah, K Rantanen, P Jaakkola, LL Elo; citation_volume=44; citation_issue=1; citation_publication_date=2016; citation_pages=e1; citation_doi=10.1093/nar/gkv806; citation_id=CR211"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=NPEBseq: nonparametric empirical bayesian-based procedure for differential expression analysis of RNA-seq data; citation_author=Y Bi, RV Davuluri; citation_volume=14; citation_publication_date=2013; citation_pages=262; citation_doi=10.1186/1471-2105-14-262; citation_id=CR212"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Next maSigPro: updating maSigPro bioconductor package for RNA-seq time series; citation_author=MJ Nueda, S Tarazona, A Conesa; citation_volume=30; citation_publication_date=2014; citation_pages=2598-602; citation_doi=10.1093/bioinformatics/btu333; citation_id=CR213"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Genet; citation_title=The Genotype-Tissue expression (GTEx) project; citation_author=; citation_volume=45; citation_publication_date=2013; citation_pages=580-5; citation_doi=10.1038/ng.2653; citation_id=CR214"/>

    <meta name="citation_author" content="Ana Conesa"/>

    <meta name="citation_author_institution" content="Institute for Food and Agricultural Sciences, Department of Microbiology and Cell Science, University of Florida, Gainesville, USA"/>

    <meta name="citation_author_institution" content="Centro de Investigaci&#243;n Pr&#237;ncipe Felipe, Genomics of Gene Expression Laboratory, Valencia, Spain"/>

    <meta name="citation_author" content="Pedro Madrigal"/>

    <meta name="citation_author_institution" content="Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK"/>

    <meta name="citation_author_institution" content="Wellcome Trust-Medical Research Council Cambridge Stem Cell Institute, Anne McLaren Laboratory for Regenerative Medicine, Department of Surgery, University of Cambridge, Cambridge, UK"/>

    <meta name="citation_author" content="Sonia Tarazona"/>

    <meta name="citation_author_institution" content="Centro de Investigaci&#243;n Pr&#237;ncipe Felipe, Genomics of Gene Expression Laboratory, Valencia, Spain"/>

    <meta name="citation_author_institution" content="Department of Applied Statistics, Operations Research and Quality, Universidad Polit&#233;cnica de Valencia, Valencia, Spain"/>

    <meta name="citation_author" content="David Gomez-Cabrero"/>

    <meta name="citation_author_institution" content="Unit of Computational Medicine, Department of Medicine, Karolinska Institutet, Karolinska University Hospital, Stockholm, Sweden"/>

    <meta name="citation_author_institution" content="Center for Molecular Medicine, Karolinska Institutet, Stockholm, Sweden"/>

    <meta name="citation_author_institution" content="Unit of Clinical Epidemiology, Department of Medicine, Karolinska University Hospital, Stockholm, Sweden"/>

    <meta name="citation_author_institution" content="Science for Life Laboratory, Solna, Sweden"/>

    <meta name="citation_author" content="Alejandra Cervera"/>

    <meta name="citation_author_institution" content="Systems Biology Laboratory, Institute of Biomedicine and Genome-Scale Biology Research Program, University of Helsinki, Helsinki, Finland"/>

    <meta name="citation_author" content="Andrew McPherson"/>

    <meta name="citation_author_institution" content="School of Computing Science, Simon Fraser University, Burnaby, Canada"/>

    <meta name="citation_author" content="Micha&#322; Wojciech Szcze&#347;niak"/>

    <meta name="citation_author_institution" content="Department of Bioinformatics, Institute of Molecular Biology and Biotechnology, Adam Mickiewicz University in Pozna&#324;, Pozna&#324;, Poland"/>

    <meta name="citation_author" content="Daniel J. Gaffney"/>

    <meta name="citation_author_institution" content="Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK"/>

    <meta name="citation_author" content="Laura L. Elo"/>

    <meta name="citation_author_institution" content="Turku Centre for Biotechnology, University of Turku and &#197;bo Akademi University, Turku, Finland"/>

    <meta name="citation_author" content="Xuegong Zhang"/>

    <meta name="citation_author_institution" content="Key Lab of Bioinformatics/Bioinformatics Division, TNLIST and Department of Automation, Tsinghua University, Beijing, China"/>

    <meta name="citation_author_institution" content="School of Life Sciences, Tsinghua University, Beijing, China"/>

    <meta name="citation_author" content="Ali Mortazavi"/>

    <meta name="citation_author_institution" content="Department of Developmental and Cell Biology, University of California, Irvine, Irvine, USA"/>

    <meta name="citation_author_institution" content="Center for Complex Biological Systems, University of California, Irvine, Irvine, USA"/>



        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>
        <meta name="format-detection" content="telephone=no">

        <link rel="shortcut icon"
      href=/static/images/favicons/bmc/favicon-9ff1bf1161.ico>
<link rel="apple-touch-icon"
      sizes="114x114"
      href=/static/images/favicons/bmc/app-icon-114x114-aa3514ee38.png>
<link rel="apple-touch-icon"
      sizes="144x144"
      href=/static/images/favicons/bmc/app-icon-144x144-67f5b60136.png>
<link rel="apple-touch-icon"
      sizes="180x180"
      href=/static/images/favicons/bmc/app-icon-180x180-230f795aee.png>
<link rel="icon"
      type="image/png"
      sizes="16x16"
      href=/static/images/favicons/bmc/favicon-16x16-25a0239eec.png>
<link rel="icon"
      type="image/png"
      sizes="32x32"
      href=/static/images/favicons/bmc/favicon-32x32-637712e609.png>
<link rel="icon"
      type="image/png"
      sizes="96x96"
      href=/static/images/favicons/bmc/favicon-96x96-9f5dc535c7.png>
<link rel="icon"
      type="image/png"
      sizes="194x194"
      href=/static/images/favicons/bmc/favicon-194x194-d2a2617de2.png>
<meta name="msapplication-TileColor"
      content="#e6e6e6">
<meta name="msapplication-TileImage"
      content=/static/images/favicons/bmc/app-icon-144x144-67f5b60136.png>
<meta name="theme-color"
      content="#1b3051">

        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        

<link rel="stylesheet" media="screen" href=/static/app-bmc/core-79235ff3ca.css>
<link rel="stylesheet" media="print" href=/static/app-bmc/print-7680b1a80b.css>

<link rel="stylesheet" id="js-mustard" href=/static/app-bmc/app-6bbf7dcb69.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">


        

        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>

        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'genomebiology.biomedcentral.com',
            siteWithPath: 'genomebiology.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s13059-016-0881-8',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            hasFigshareInvoked: false,
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script>

        <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script type="text/javascript">
document.addEventListener('dataLayerCreated', function() {
    var script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function () {
        var dL;
        var doi;
        var imprint;
        var bpids;
        var publisher;
        var pageType;
        var eventObject;

        for (var i = 0; i < window.dataLayer.length; i++) {
            if (window.dataLayer[i].event === "dataLayerCreated") {
                dL = window.dataLayer[i];
            }
        }

        if (!dL || !dL.content.article || !dL.content.article.doi) { return false;}

        doi = dL.content.article.doi;
        imprint = dL.content.contentInfo.imprint;
        bpids = dL.session.authentication.authenticationID;
        pageType = dL.page.category.pageType;

        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }

        eventObject = {'content_type': pageType, 'doi': doi};

        if (imprint === 'BioMed Central') {
            publisher = 'BMC';
        }
        else if (imprint === 'SpringerOpen') {
            publisher = 'SpringerOpen';
        }
        else {
            throw new Error('Event Tracker Error: The publisher has an unexpected value');
        }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }

        new EventTracker({'platform': publisher}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script>

        <!--Polyfills CustomEvent constructor in IE. Allows us to use events to manage race conditions in client side js-->
<script>
    (function () {
        if (typeof window.CustomEvent === "function") { return false; } // If IE polyfill not needed just return

        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: undefined };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script> 
        
    <script type="text/javascript">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/s13059-016-0881-8","articleType":"Review","peerReviewType":"Closed","supplement":null,"keywords":"Differential Expression Analysis;Reference Transcriptome;Transcript Identification;Transcript Discovery;Gene Transfer Format"},"contentInfo":{"imprint":"BioMed Central","title":"A survey of best practices for RNA-seq data analysis","publishedAt":1453766400000,"publishedAtDate":"2016-01-26","author":["Ana Conesa","Pedro Madrigal","Sonia Tarazona","David Gomez-Cabrero","Alejandra Cervera","Andrew McPherson","Michał Wojciech Szcześniak","Daniel J. Gaffney","Laura L. Elo","Xuegong Zhang","Ali Mortazavi"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"genomebiology.biomedcentral.com","volume":"17","issue":"1","title":"Genome Biology","type":"FLAGSHIP","journalID":13059,"gaCode":"UA-61881280-4","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Review","publishingSegment":"BMC flagships"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}},"event":"dataLayerCreated"}];
        var event = new CustomEvent('dataLayerCreated');
        document.dispatchEvent(event);
    </script>


        <script>
            
            var mustardcutlink = document.getElementById('js-mustard');
            if (mustardcutlink && window.matchMedia && window.matchMedia(mustardcutlink.media).matches) {
                window.config.mustardcut = true;
            }
        </script>
        <script>
            (function(d){
                var config = window.config;
                if (config && config.mustardcut && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                    if (window.config.publisherBrand === 'BioMed Central') {
                        d.style.setProperty('--font-family-sans', 'Europa');
                        d.style.setProperty('--font-family-serif', 'Noto Serif')
                    } else if (window.config.publisherBrand === 'SpringerOpen' || window.config.publisherBrand === 'Springer') {
                        d.style.setProperty('--font-family-sans', 'Source Sans Pro');
                    }
                }
            })(document.documentElement);
        </script>
        <script src="https://cdn.cookielaw.org/consent/7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0881-8"/>
    

        
        
        <meta property="og:url" content="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0881-8"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Genome Biology"/>
        <meta property="og:title" content="A survey of best practices for RNA-seq data analysis"/>
        <meta property="og:description" content="RNA-sequencing (RNA-seq) has a wide variety of applications, but no single analysis pipeline can be used in all cases. We review all of the major steps in RNA-seq data analysis, including experimental design, quality control, read alignment, quantification of gene and transcript levels, visualization, differential gene expression, alternative splicing, functional analysis, gene fusion detection and eQTL mapping. We highlight the challenges associated with each step. We discuss the analysis of small RNAs and the integration of RNA-seq with other functional genomics techniques. Finally, we discuss the outlook for novel technologies that are changing the state of the art in transcriptomics."/>
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/13059.jpg"/>
    
    </head>

    <body class="journal journal-fulltext"
          
          >
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    

        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-alert"><path d="M1 14h14v-.172a1 1 0 0 0-.293-.707L13 11.414V8A5 5 0 0 0 3 8v3h2.5a.5.5 0 1 1 0 1H2.414l-1.121 1.121a1 1 0 0 0-.293.707V14zm5 1a2 2 0 1 0 4 0H6zm-5 0a1 1 0 0 1-1-1v-.172a2 2 0 0 1 .586-1.414L2 11V8a6 6 0 1 1 12 0v3l1.414 1.414A2 2 0 0 1 16 13.828V14a1 1 0 0 1-1 1h-4a3 3 0 0 1-6 0H1zM8.997 2.082a1 1 0 1 0-1.993 0c-.336.057-.662.14-.977.25a2 2 0 1 1 3.945 0 5.958 5.958 0 0 0-.975-.25z"/></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-article"><path d="M14 14c0 .6.4 1 1 1h1c.5 0 1-.5 1-1V5h-3v9zm-1 0V1H1v13c0 .5.4 1 1 1h11.3c-.2-.3-.3-.6-.3-1zM0 14V1c0-.6.5-1 1-1h12c.6 0 1 .5 1 1v3h4v10c0 1.1-.9 2-2 2H2c-1.1 0-2-.9-2-2zM4 4v2h6V4H4zM3 3h8v4H3V3zm0 5h8v1H3V8zm0 2h8v1H3v-1zm0 2h8v1H3v-1z"/></symbol><symbol id="icon-chevron-right" viewBox="0 0 6 9"><path d="M1.14.854a.515.515 0 0 1 0-.708.467.467 0 0 1 .68 0l3.847 4.008a.5.5 0 0 1 0 .692L1.82 8.854a.467.467 0 0 1-.68 0 .515.515 0 0 1 0-.708L4.643 4.5 1.141.854z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><g filter="url(#a)"><path fill="#FFF" d="M0 31.01h60.643V0H0z"/></g></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download-rounded"><path d="M0 13c0-.556.449-1 1.002-1h9.996a.999.999 0 1 1 0 2H1.002A1.006 1.006 0 0 1 0 13zM7 1v6.8l2.482-2.482c.392-.392 1.022-.4 1.403-.02a1.001 1.001 0 0 1 0 1.417l-4.177 4.177a1.001 1.001 0 0 1-1.416 0L1.115 6.715a.991.991 0 0 1-.016-1.4 1 1 0 0 1 1.42.003L5 7.8V1c0-.55.444-.996 1-.996.552 0 1 .445 1 .996z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-editor"><path d="M10.2 16.9c1.3-.2 2.5-.7 3.5-1.5-.9-2.5-2.1-3.9-3.8-4.3V9.8c1.2-.4 2-1.5 2-2.8 0-1.7-1.3-3-3-3S6 5.3 6 7c0 1.3.8 2.4 2 2.8v1.3c-1.7.4-2.9 1.8-3.8 4.3 1 .8 2.2 1.3 3.5 1.5.5.1.9.1 1.3.1s.8 0 1.2-.1zm4.4-2.2C16.1 13.3 17 11.2 17 9c0-4.4-3.6-8-8-8S1 4.6 1 9c0 2.2.9 4.3 2.4 5.7.8-2 1.8-3.6 3.5-4.3C5.8 9.7 5 8.4 5 7c0-2.2 1.8-4 4-4s4 1.8 4 4c0 1.4-.8 2.7-1.9 3.4 1.6.7 2.7 2.3 3.5 4.3zM9 18c-5 0-9-4-9-9s4-9 9-9 9 4 9 9-4 9-9 9z"/></symbol><symbol id="icon-explore"><path d="M9 17c4.4 0 8-3.6 8-8s-3.6-8-8-8-8 3.6-8 8 3.6 8 8 8zm0 1c-5 0-9-4-9-9s4-9 9-9 9 4 9 9-4 9-9 9zm0-2.5c-.3 0-.5-.2-.5-.5s.2-.5.5-.5c3 0 5.4-2.4 5.5-5.3 0-.3.2-.5.5-.5s.5.2.5.5c-.1 3.5-3 6.3-6.5 6.3zm-.3-13c.3 0 .5.2.5.5s-.2.5-.4.5c-3 .1-5.3 2.6-5.3 5.5 0 .3-.2.5-.5.5s-.5-.2-.5-.5c0-3.5 2.7-6.3 6.2-6.5zm1.1 8.1L8.6 9.4c-.2-.2-.2-.5 0-.7.2-.2.5-.2.7 0l1.2 1.2 1.6-4-4.5 1.7-1.8 4.5 4-1.5zm4.1-6.5l-2.7 6.7c-.1.3-.3.5-.6.6l-6.7 2.7 2.7-6.7c.2-.4.4-.6.7-.7l6.6-2.6z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.6 10.6c2.2 0 4-1.7 4-4s-1.7-4-4-4-4 1.7-4 4 1.8 4 4 4zm4 1.4l1.5-1.5 3.7 3.6c.4.5.4 1.2 0 1.6-.4.4-1.1.4-1.6 0L10.6 12zm-4 1.2C2.9 13.2 0 10.3 0 6.6S2.9 0 6.6 0s6.6 2.9 6.6 6.6-2.9 6.6-6.6 6.6z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"/></filter></defs><symbol id="icon-alert"><path d="M1 14h14v-.172a1 1 0 0 0-.293-.707L13 11.414V8A5 5 0 0 0 3 8v3h2.5a.5.5 0 1 1 0 1H2.414l-1.121 1.121a1 1 0 0 0-.293.707V14zm5 1a2 2 0 1 0 4 0H6zm-5 0a1 1 0 0 1-1-1v-.172a2 2 0 0 1 .586-1.414L2 11V8a6 6 0 1 1 12 0v3l1.414 1.414A2 2 0 0 1 16 13.828V14a1 1 0 0 1-1 1h-4a3 3 0 0 1-6 0H1zM8.997 2.082a1 1 0 1 0-1.993 0c-.336.057-.662.14-.977.25a2 2 0 1 1 3.945 0 5.958 5.958 0 0 0-.975-.25z"/></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"/></symbol><symbol id="icon-article"><path d="M14 14c0 .6.4 1 1 1h1c.5 0 1-.5 1-1V5h-3v9zm-1 0V1H1v13c0 .5.4 1 1 1h11.3c-.2-.3-.3-.6-.3-1zM0 14V1c0-.6.5-1 1-1h12c.6 0 1 .5 1 1v3h4v10c0 1.1-.9 2-2 2H2c-1.1 0-2-.9-2-2zM4 4v2h6V4H4zM3 3h8v4H3V3zm0 5h8v1H3V8zm0 2h8v1H3v-1zm0 2h8v1H3v-1z"/></symbol><symbol id="icon-chevron-right" viewBox="0 0 6 9"><path d="M1.14.854a.515.515 0 0 1 0-.708.467.467 0 0 1 .68 0l3.847 4.008a.5.5 0 0 1 0 .692L1.82 8.854a.467.467 0 0 1-.68 0 .515.515 0 0 1 0-.708L4.643 4.5 1.141.854z"/></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"/></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><g filter="url(#a)"><path fill="#FFF" d="M0 31.01h60.643V0H0z"/></g></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"/></symbol><symbol id="icon-download-rounded"><path d="M0 13c0-.556.449-1 1.002-1h9.996a.999.999 0 1 1 0 2H1.002A1.006 1.006 0 0 1 0 13zM7 1v6.8l2.482-2.482c.392-.392 1.022-.4 1.403-.02a1.001 1.001 0 0 1 0 1.417l-4.177 4.177a1.001 1.001 0 0 1-1.416 0L1.115 6.715a.991.991 0 0 1-.016-1.4 1 1 0 0 1 1.42.003L5 7.8V1c0-.55.444-.996 1-.996.552 0 1 .445 1 .996z"/></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-editor"><path d="M10.2 16.9c1.3-.2 2.5-.7 3.5-1.5-.9-2.5-2.1-3.9-3.8-4.3V9.8c1.2-.4 2-1.5 2-2.8 0-1.7-1.3-3-3-3S6 5.3 6 7c0 1.3.8 2.4 2 2.8v1.3c-1.7.4-2.9 1.8-3.8 4.3 1 .8 2.2 1.3 3.5 1.5.5.1.9.1 1.3.1s.8 0 1.2-.1zm4.4-2.2C16.1 13.3 17 11.2 17 9c0-4.4-3.6-8-8-8S1 4.6 1 9c0 2.2.9 4.3 2.4 5.7.8-2 1.8-3.6 3.5-4.3C5.8 9.7 5 8.4 5 7c0-2.2 1.8-4 4-4s4 1.8 4 4c0 1.4-.8 2.7-1.9 3.4 1.6.7 2.7 2.3 3.5 4.3zM9 18c-5 0-9-4-9-9s4-9 9-9 9 4 9 9-4 9-9 9z"/></symbol><symbol id="icon-explore"><path d="M9 17c4.4 0 8-3.6 8-8s-3.6-8-8-8-8 3.6-8 8 3.6 8 8 8zm0 1c-5 0-9-4-9-9s4-9 9-9 9 4 9 9-4 9-9 9zm0-2.5c-.3 0-.5-.2-.5-.5s.2-.5.5-.5c3 0 5.4-2.4 5.5-5.3 0-.3.2-.5.5-.5s.5.2.5.5c-.1 3.5-3 6.3-6.5 6.3zm-.3-13c.3 0 .5.2.5.5s-.2.5-.4.5c-3 .1-5.3 2.6-5.3 5.5 0 .3-.2.5-.5.5s-.5-.2-.5-.5c0-3.5 2.7-6.3 6.2-6.5zm1.1 8.1L8.6 9.4c-.2-.2-.2-.5 0-.7.2-.2.5-.2.7 0l1.2 1.2 1.6-4-4.5 1.7-1.8 4.5 4-1.5zm4.1-6.5l-2.7 6.7c-.1.3-.3.5-.6.6l-6.7 2.7 2.7-6.7c.2-.4.4-.6.7-.7l6.6-2.6z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path fill="#676767" d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"/><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"/></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"/><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"/></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"/></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"/><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"/><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"/></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.6 10.6c2.2 0 4-1.7 4-4s-1.7-4-4-4-4 1.7-4 4 1.8 4 4 4zm4 1.4l1.5-1.5 3.7 3.6c.4.5.4 1.2 0 1.6-.4.4-1.1.4-1.6 0L10.6 12zm-4 1.2C2.9 13.2 0 10.3 0 6.6S2.9 0 6.6 0s6.6 2.9 6.6 6.6-2.9 6.6-6.6 6.6z"/></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"/></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"/></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"/></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"/></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"/><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"/></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"/><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"/><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"/></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"/><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"/><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"/><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"/><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"/><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"/><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"/><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"/><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"/><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"/><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"/><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"/><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"/><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"/><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"/><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"/></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/genomebiology/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s13059-016-0881-8;kwrd=Differential Expression Analysis,Reference Transcriptome,Transcript Identification,Transcript Discovery,Gene Transfer Format;pmc=L32030,B12008,L32020,L32010,L15001,L21001;sponsored=reviewsfromgenomebiology;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/genomebiology/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s13059-016-0881-8&amp;kwrd=Differential Expression Analysis,Reference Transcriptome,Transcript Identification,Transcript Discovery,Gene Transfer Format&amp;pmc=L32030,B12008,L32020,L32010,L15001,L21001&amp;sponsored=reviewsfromgenomebiology&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/genomebiology/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s13059-016-0881-8&amp;kwrd=Differential Expression Analysis,Reference Transcriptome,Transcript Identification,Transcript Discovery,Gene Transfer Format&amp;pmc=L32030,B12008,L32020,L32010,L15001,L21001&amp;sponsored=reviewsfromgenomebiology&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </div>

        
         
    
        <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>
    
    
        <header class="c-popup-search" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="c-navbar__container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization" />
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img alt="BMC" itemprop="logo" width="76" height="18" src=/static/images/bmc/logos/logo-bmc-white-flagship-8fa9bf7000.svg>
        <div class="c-logo__strapline">
            <img alt="Part of Springer Nature" width="173" height="16" src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-e7b1d6e67f.svg>
        </div>
    </a>
</div>

            </div>
            <button
                type="button"
                class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg"
                id="publisher-header-menu-button"
                data-toggle="collapse"
                data-test="publisher-header-menu-toggle"
                data-toggle-linked="publisher-header"
                data-target="publisher-header-search publisher-header-menu"
                aria-controls="publisher-header-menu publisher-header-search"
                aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link"
               href="//www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link"
               href="//www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link"
               href="//www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav c-navbar__nav--right">
                    <li class="c-navbar__item">
                        <button
                            type="button"
                            class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg"
                            data-toggle="collapse"
                            data-test="header-search-button"
                            data-target="publisher-header-search publisher-header-menu publisher-header-menu-button"
                            aria-controls="publisher-header-search publisher-header-menu"
                            aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                    <li class="c-navbar__item">
                        <a data-replace="true"
                           data-placeholder="/placeholder/v1/username"
                           class="c-navbar__link"
                           href="https://www.biomedcentral.com/login"
                           data-test="login-link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="c-flex">
            <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value=""/>
            <div>
                <button class="c-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


            </div>
        </div>
        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</header>

    

        
            <header class="c-journal-header c-journal-header--genome-biology ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner c-journal-header__inner--featured-journal">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
    <span class="c-journal-title__text  u-visually-hidden">Genome Biology</span>
    
        <div class="c-journal-title__featured-journal-lines c-journal-title__featured-journal-lines--genome-biology">
            <img src=/static/images/bmc/logos/genome-biology-logo-dffd3410b7.svg class="c-journal-title__featured-journal-logo" alt="Genome Biology logo" itemprop="logo">
        </div>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="c-navbar__container">
                        
                            <button type="button"
                                    class="c-navbar__link c-dropdown u-hidden-at-lg"
                                    data-toggle="collapse"
                                    data-test="journal-header-menu"
                                    data-toggle-linked="journal-header"
                                    data-target="journal-menu"
                                    aria-controls="journal-menu"
                                    aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--featured-journal">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-page-layout c-page-layout--fulltext" data-component="sticky-container">
                <div class="c-page-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Experimental design">Experimental design</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec3" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Analysis of the RNA-seq data">Analysis of the RNA-seq data</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec20" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Integration with other data types">Integration with other data types</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec27" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Outlook">Outlook</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec30" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Box 1. Number of replicates">Box 1. Number of replicates</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec31" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Box 2. Experiment execution choices">Box 2. Experiment execution choices</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec32" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Box 3. Mapping to a reference">Box 3. Mapping to a reference</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec33" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Box 4. Comparison of software tools for detecting differential gene and transcript expression">Box 4. Comparison of software tools for detecting differential gene and transcript expression</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-page-layout__main">
                    <article itemscope itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Review</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>
    

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">A survey of best practices for RNA-seq data analysis</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Ana&nbsp;Conesa</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup><a href="mailto:aconesa@ufl.edu" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Pedro&nbsp;Madrigal</span><sup><a href="#Aff3" class="AffiliationID">3</a>, <a href="#Aff4" class="AffiliationID">4</a></sup><a href="mailto:pm12@sanger.ac.uk" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Sonia&nbsp;Tarazona</span><sup><a href="#Aff2" class="AffiliationID">2</a>, <a href="#Aff5" class="AffiliationID">5</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">David&nbsp;Gomez-Cabrero</span><sup><a href="#Aff6" class="AffiliationID">6</a>, <a href="#Aff7" class="AffiliationID">7</a>, <a href="#Aff8" class="AffiliationID">8</a>, <a href="#Aff9" class="AffiliationID">9</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Alejandra&nbsp;Cervera</span><sup><a href="#Aff10" class="AffiliationID">10</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Andrew&nbsp;McPherson</span><sup><a href="#Aff11" class="AffiliationID">11</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Michał&nbsp;Wojciech&nbsp;Szcześniak</span><sup><a href="#Aff12" class="AffiliationID">12</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Daniel&nbsp;J.&nbsp;Gaffney</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Laura&nbsp;L.&nbsp;Elo</span><sup><a href="#Aff13" class="AffiliationID">13</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Xuegong&nbsp;Zhang</span><sup><a href="#Aff14" class="AffiliationID">14</a>, <a href="#Aff15" class="AffiliationID">15</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Ali&nbsp;Mortazavi</span><sup><a href="#Aff16" class="AffiliationID">16</a>, <a href="#Aff17" class="AffiliationID">17</a></sup><a href="mailto:ali.mortazavi@uci.edu" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">Genome Biology</span></span><span class="ArticleCitation_Year">2016</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">17</strong>:13</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/s13059-016-0881-8">https://doi.org/10.1186/s13059-016-0881-8</a></p><p class="Copyright">© &nbsp;Conesa et al.&nbsp;2016</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">26&nbsp;January&nbsp;2016</span></li></ul></div>
                        
                        
    
        <div class="c-alert c-alert--info" data-test="flash-info">
    <svg class="c-alert__icon c-icon c-icon--outline" width="18" height="18" aria-hidden="true">
        <use xlink:href="#icon-info"></use>
    </svg>
    <div data-test="flash-message">
        
            <p data-test="related-article">
                
                    
                        The <a href="/articles/10.1186/s13059-016-1047-4"
                               data-track="click"
                               data-track-category="Article page"
                               data-track-action="Click related articles"
                               data-track-label="10.1186/s13059-016-1047-4"
                               data-test="related-article-link">
                        Erratum</a> to this article has been published in <span class="italic">Genome Biology</span> 2016<span class="bold"> 17</span>:181
                    
                
            </p>
        
    </div>
</div>

    

                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par1" class="Para">RNA-sequencing (RNA-seq) has a wide variety of applications, but no single analysis pipeline can be used in all cases. We review all of the major steps in RNA-seq data analysis, including experimental design, quality control, read alignment, quantification of gene and transcript levels, visualization, differential gene expression, alternative splicing, functional analysis, gene fusion detection and eQTL mapping. We highlight the challenges associated with each step. We discuss the analysis of small RNAs and the integration of RNA-seq with other functional genomics techniques. Finally, we discuss the outlook for novel technologies that are changing the state of the art in transcriptomics.</p></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">Differential Expression Analysis</li><li class="c-keywords__item" itemprop="about">Reference Transcriptome</li><li class="c-keywords__item" itemprop="about">Transcript Identification</li><li class="c-keywords__item" itemprop="about">Transcript Discovery</li><li class="c-keywords__item" itemprop="about">Gene Transfer Format</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 SectionTypeIntroduction c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par29" class="Para">Transcript identification and the quantification of gene expression have been distinct core activities in molecular biology ever since the discovery of RNA’s role as the key intermediate between the genome and the proteome. The power of sequencing RNA lies in the fact that the twin aspects of discovery and quantification can be combined in a single high-throughput sequencing assay called RNA-sequencing (RNA-seq). The pervasive adoption of RNA-seq has spread well beyond the genomics community and has become a standard part of the toolkit used by the life sciences research community. Many variations of RNA-seq protocols and analyses have been published, making it challenging for new users to appreciate all of the steps necessary to conduct an RNA-seq study properly.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par30" class="Para">There is no optimal pipeline for the variety of different applications and analysis scenarios in which RNA-seq can be used. Scientists plan experiments and adopt different analysis strategies depending on the organism being studied and their research goals. For example, if a genome sequence is available for the studied organism, it should be possible to identify transcripts by mapping RNA-seq reads onto the genome. By contrast, for organisms without sequenced genomes, quantification would be achieved by first assembling reads de novo into contigs and then mapping these contigs onto the transcriptome. For well-annotated genomes such as the human genome, researchers may choose to base their RNA-seq analysis on the existing annotated reference transcriptome alone, or might try to identify new transcripts and their differential regulation. Furthermore, investigators might be interested only in messenger RNA isoform expression or microRNA (miRNA) levels or allele variant identification. Both the experimental design and the analysis procedures will vary greatly in each of these cases. RNA-seq can be used solo for transcriptome profiling or in combination with other functional genomics methods to enhance the analysis of gene expression. Finally, RNA-seq can be coupled with different types of biochemical assay to analyze many other aspects of RNA biology, such as RNA–protein binding, RNA structure, or RNA–RNA interactions. These applications are, however, beyond the scope of this review as we focus on ‘typical’ RNA-seq.</p><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par31" class="Para">Every RNA-seq experimental scenario could potentially have different optimal methods for transcript quantification, normalization, and ultimately differential expression analysis. Moreover, quality control checks should be applied pertinently at different stages of the analysis to ensure both reproducibility and reliability of the results. Our focus is to outline current standards and resources for the bioinformatics analysis of RNA-seq data. We do not aim to provide an exhaustive compilation of resources or software tools nor to indicate one best analysis pipeline. Rather, we aim to provide a commented guideline for RNA-seq data analysis. Figure&nbsp;<span class="InternalRef"><a href="#Fig1">1</a></span> depicts a generic roadmap for experimental design and analysis using standard Illumina sequencing. We also briefly list several data integration paradigms that have been proposed and comment on their potential and limitations. We finally discuss the opportunities as well as challenges provided by single-cell RNA-seq and long-read technologies when compared to traditional short-read RNA-seq.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO1">
                    <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs13059-016-0881-8/MediaObjects/13059_2016_881_Fig1_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs13059-016-0881-8/MediaObjects/13059_2016_881_Fig1_HTML.gif" alt="Fig. 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs13059-016-0881-8/MediaObjects/13059_2016_881_Fig1_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs13059-016-0881-8/MediaObjects/13059_2016_881_Fig1_HTML.gif" alt="Fig. 1"></img></noscript></a>
                  </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 1</span>
                      <p xmlns="" class="SimplePara">A generic roadmap for RNA-seq computational analyses. The major analysis steps are listed above the lines for pre-analysis, core analysis and advanced analysis. The key analysis issues for each step that are listed below the lines are discussed in the text. <strong class="EmphasisTypeBold">a</strong> Preprocessing includes experimental design, sequencing design, and quality control steps. <strong class="EmphasisTypeBold">b</strong> Core analyses include transcriptome profiling, differential gene expression, and functional profiling. <strong class="EmphasisTypeBold">c</strong> Advanced analysis includes visualization, other RNA-seq technologies, and data integration. Abbreviations: <em class="EmphasisTypeItalic">ChIP-seq</em> Chromatin immunoprecipitation sequencing, <em class="EmphasisTypeItalic">eQTL</em> Expression quantitative loci, <em class="EmphasisTypeItalic">FPKM</em> Fragments per kilobase of exon model per million mapped reads, <em class="EmphasisTypeItalic">GSEA</em> Gene set enrichment analysis, <em class="EmphasisTypeItalic">PCA</em> Principal component analysis, <em class="EmphasisTypeItalic">RPKM</em> Reads per kilobase of exon model per million reads, <em class="EmphasisTypeItalic">sQTL</em> Splicing quantitative trait loci, <em class="EmphasisTypeItalic">TF</em> Transcription factor, <em class="EmphasisTypeItalic">TPM</em> Transcripts per million</p>
                    </div></figcaption></figure>
                     </div></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Experimentaldesign" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Experimentaldesign">Experimental design</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Experimentaldesign" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par32" class="Para">A crucial prerequisite for a successful RNA-seq study is that the data generated have the potential to answer the biological questions of interest. This is achieved by first defining a good experimental design, that is, by choosing the library type, sequencing depth and number of replicates appropriate for the biological system under study, and second by planning an adequate execution of the sequencing experiment itself, ensuring that data acquisition does not become contaminated with unnecessary biases. In this section, we discuss both considerations.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par33" class="Para">One important aspect of the experimental design is the RNA-extraction protocol used to remove the highly abundant ribosomal RNA (rRNA), which typically constitutes over 90&nbsp;% of total RNA in the cell, leaving the 1–2&nbsp;% comprising messenger RNA (mRNA) that we are normally interested in. For eukaryotes, this involves choosing whether to enrich for mRNA using poly(A) selection or to deplete rRNA. Poly(A) selection typically requires a relatively high proportion of mRNA with minimal degradation as measured by RNA integrity number (RIN), which normally yields a higher overall fraction of reads falling onto known exons. Many biologically relevant samples (such as tissue biopsies) cannot, however, be obtained in great enough quantity or good enough mRNA integrity to produce good poly(A) RNA-seq libraries and therefore require ribosomal depletion. For bacterial samples, in which mRNA is not polyadenylated, the only viable alternative is ribosomal depletion. Another consideration is whether to generate strand-preserving libraries. The first generation of Illumina-based RNA-seq used random hexamer priming to reverse-transcribe poly(A)-selected mRNA. This methodology did not retain information contained on the DNA strand that is actually expressed [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>] and therefore complicates the analysis and quantification of antisense or overlapping transcripts. Several strand-specific protocols [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>], such as the widely used dUTP method, extend the original protocol by incorporating UTP nucleotides during the second cDNA synthesis step, prior to adapter ligation followed by digestion of the strand containing dUTP [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. In all cases, the size of the final fragments (usually less than 500&nbsp;bp for Illumina) will be crucial for proper sequencing and subsequent analysis. Furthermore, sequencing can involve single-end (SE) or paired-end (PE) reads, although the latter is preferable for de novo transcript discovery or isoform expression analysis [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>, <span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>]. Similarly, longer reads improve mappability and transcript identification [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>, <span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]. The best sequencing option depends on the analysis goals. The cheaper, short SE reads are normally sufficient for studies of gene expression levels in well-annotated organisms, whereas longer and PE reads are preferable to characterize poorly annotated transcriptomes.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par34" class="Para">Another important factor is sequencing depth or library size, which is the number of sequenced reads for a given sample. More transcripts will be detected and their quantification will be more precise as the sample is sequenced to a deeper level [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>]. Nevertheless, optimal sequencing depth again depends on the aims of the experiment. While some authors will argue that as few as five million mapped reads are sufficient to quantify accurately medium to highly expressed genes in most eukaryotic transcriptomes, others will sequence up to 100 million reads to quantify precisely genes and transcripts that have low expression levels [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]. When studying single cells, which have limited sample complexity, quantification is often carried out with just one million reads but may be done reliably for highly expressed genes with as few as 50,000 reads [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]; even 20,000 reads have been used to differentiate cell types in splenic tissue [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]. Moreover, optimal library size depends on the complexity of the targeted transcriptome. Experimental results suggest that deep sequencing improves quantification and identification but might also result in the detection of transcriptional noise and off-target transcripts [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]. Saturation curves can be used to assess the improvement in transcriptome coverage to be expected at a given sequencing depth [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par35" class="Para">Finally, a crucial design factor is the number of replicates. The number of replicates that should be included in a RNA-seq experiment depends on both the amount of technical variability in the RNA-seq procedures and the biological variability of the system under study, as well as on the desired statistical power (that is, the capacity for detecting statistically significant differences in gene expression between experimental groups). These two aspects are part of power analysis calculations (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1a</a></span>; Box 1).</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par36" class="Para">The adequate planning of sequencing experiments so as to avoid technical biases is as important as good experimental design, especially when the experiment involves a large number of samples that need to be processed in several batches. In this case, including controls, randomizing sample processing and smart management of sequencing runs are crucial to obtain error-free data (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1a</a></span>; Box 2).</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec3" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#AnalysisoftheRNAseqdata" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="AnalysisoftheRNAseqdata">Analysis of the RNA-seq data</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="AnalysisoftheRNAseqdata" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par37" class="Para">The actual analysis of RNA-seq data has as many variations as there are applications of the technology. In this section, we address all of the major analysis steps for a typical RNA-seq experiment, which involve quality control, read alignment with and without a reference genome, obtaining metrics for gene and transcript expression, and approaches for detecting differential gene expression. We also discuss analysis options for applications of RNA-seq involving alternative splicing, fusion transcripts and small RNA expression. Finally, we review useful packages for data visualization.</p><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">Quality-control checkpoints</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par38" class="Para">The acquisition of RNA-seq data consists of several steps — obtaining raw reads, read alignment and quantification. At each of these steps, specific checks should be applied to monitor the quality of the data (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1a</a></span>).</p>
                <section id="Sec5" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Raw reads</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par39" class="Para">Quality control for the raw reads involves the analysis of sequence quality, GC content, the presence of adaptors, overrepresented <em xmlns="" class="EmphasisTypeItalic">k</em>-mers and duplicated reads in order to detect sequencing errors, PCR artifacts or contaminations. Acceptable duplication, <em xmlns="" class="EmphasisTypeItalic">k</em>-mer or GC content levels are experiment- and organism-specific, but these values should be homogeneous for samples in the same experiments. We recommend that outliers with over 30&nbsp;% disagreement to be discarded. FastQC [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>] is a popular tool to perform these analyses on Illumina reads, whereas NGSQC [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>] can be applied to any platform. As a general rule, read quality decreases towards the 3’ end of reads, and if it becomes too low, bases should be removed to improve mappability. Software tools such as the FASTX-Toolkit [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>] and Trimmomatic [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>] can be used to discard low-quality reads, trim adaptor sequences, and eliminate poor-quality bases.</p>
                </section>
                <section id="Sec6" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Read alignment</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par40" class="Para">Reads are typically mapped to either a genome or a transcriptome, as will be discussed later. An important mapping quality parameter is the percentage of mapped reads, which is a global indicator of the overall sequencing accuracy and of the presence of contaminating DNA. For example, we expect between 70 and 90&nbsp;% of regular RNA-seq reads to map onto the human genome (depending on the read mapper used) [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>], with a significant fraction of reads mapping to a limited number of identical regions equally well (‘multi-mapping reads’). When reads are mapped against the transcriptome, we expect slightly lower total mapping percentages because reads coming from unannotated transcripts will be lost, and significantly more multi-mapping reads because of reads falling onto exons that are shared by different transcript isoforms of the same gene.</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par41" class="Para">Other important parameters are the uniformity of read coverage on exons and the mapped strand. If reads primarily accumulate at the 3’ end of transcripts in poly(A)-selected samples, this might indicate low RNA quality in the starting material. The GC content of mapped reads may reveal PCR biases. Tools for quality control in mapping include Picard [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>], RSeQC [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>] and Qualimap [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>].</p>
                </section>
                <section id="Sec7" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Quantification</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par42" class="Para">Once actual transcript quantification values have been calculated, they should be checked for GC content and gene length biases so that correcting normalization methods can be applied if necessary. If the reference transcriptome is well annotated, researchers could analyze the biotype composition of the sample, which is indicative of the quality of the RNA purification step. For example, rRNA and small RNAs should not be present in regular polyA longRNA preparations [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>, <span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>]. A number of R packages (such as NOISeq [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>] or EDASeq [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]) provide useful plots for quality control of count data.</p>
                </section>
                <section id="Sec8" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Reproducibility</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par43" class="Para">The quality-control steps described above involve individual samples. In addition, it is also crucial to assess the global quality of the RNA-seq dataset by checking on the reproducibility among replicates and for possible batch effects. Reproducibility among technical replicates should be generally high (Spearman R<sup>2</sup> &gt; 0.9) [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>], but no clear standard exists for biological replicates, as this depends on the heterogeneity of the experimental system. If gene expression differences exist among experimental conditions, it should be expected that biological replicates of the same condition will cluster together in a principal component analysis (PCA).</p>
                </section>
              </section><section id="Sec9" class="Section2 RenderAsSection2">
                <h3 class="Heading">Transcript identification</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par44" class="Para">When a reference genome is available, RNA-seq analysis will normally involve the mapping of the reads onto the reference genome or transcriptome to infer which transcripts are expressed. Mapping solely to the reference transcriptome of a known species precludes the discovery of new, unannotated transcripts and focuses the analysis on quantification alone. By contrast, if the organism does not have a sequenced genome, then the analysis path is first to assemble reads into longer contigs and then to treat these contigs as the expressed transcriptome to which reads are mapped back again for quantification. In either case, read coverage can be used to quantify transcript expression level (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1b</a></span>). A basic choice is whether transcript identification and quantification are done sequentially or simultaneously.</p>
                <section id="Sec10" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Alignment</h4>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par45" class="Para">Two alternatives are possible when a reference sequence is available: mapping to the genome or mapping to the annotated transcriptome (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2a, b</a></span>; Box 3). Regardless of whether a genome or transcriptome reference is used, reads may map uniquely (they can be assigned to only one position in the reference) or could be multi-mapped reads (multireads). Genomic multireads are primarily due to repetitive sequences or shared domains of paralogous genes. They normally account for a significant fraction of the mapping output when mapped onto the genome and should not be discarded. When the reference is the transcriptome, multi-mapping arises even more often because a read that would have been uniquely mapped on the genome would map equally well to all gene isoforms in the transcriptome that share the exon. In either case — genome or transcriptome mapping — transcript identification and quantification become important challenges for alternatively expressed genes.<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject" id="MO2">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs13059-016-0881-8/MediaObjects/13059_2016_881_Fig2_HTML.gif"><img class="c-lazy-image u-js-shown" src="https://media.springernature.com/lw5/springer-static/image/art%3A10.1186%2Fs13059-016-0881-8/MediaObjects/13059_2016_881_Fig2_HTML.gif" alt="Fig. 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs13059-016-0881-8/MediaObjects/13059_2016_881_Fig2_HTML.gif"></img><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2Fs13059-016-0881-8/MediaObjects/13059_2016_881_Fig2_HTML.gif" alt="Fig. 2"></img></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Fig. 2</span>
                          <p xmlns="" class="SimplePara">Read mapping and transcript identification strategies. Three basic strategies for regular RNA-seq analysis. <strong class="EmphasisTypeBold">a</strong> An annotated genome is available and reads are mapped to the genome with a gapped mapper. Next (novel) transcript discovery and quantification can proceed with or without an annotation file. Novel transcripts are then functionally annotated. <strong class="EmphasisTypeBold">b</strong> If no novel transcript discovery is needed, reads can be mapped to the reference transcriptome using an ungapped aligner. Transcript identification and quantification can occur simultaneously. <strong class="EmphasisTypeBold">c</strong> When no genome is available, reads need to be assembled first into contigs or transcripts. For quantification, reads are mapped back to the novel reference transcriptome and further analysis proceeds as in (<strong class="EmphasisTypeBold">b</strong>) followed by the functional annotation of the novel transcripts as in (<strong class="EmphasisTypeBold">a</strong>). Representative software that can be used at each analysis step are indicated in <em class="EmphasisTypeItalic">bold text</em>. Abbreviations: <em class="EmphasisTypeItalic">GFF</em> General Feature Format, <em class="EmphasisTypeItalic">GTF</em> gene transfer format, <em class="EmphasisTypeItalic">RSEM</em> RNA-Seq by Expectation Maximization</p>
                        </div></figcaption></figure>
                           </div>
                </section>
                <section id="Sec11" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Transcript discovery</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par46" class="Para">Identifying novel transcripts using the short reads provided by Illumina technology is one of the most challenging tasks in RNA-seq. Short reads rarely span across several splice junctions and thus make it difficult to directly infer all full-length transcripts. In addition, it is difficult to identify transcription start and end sites [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>], and tools such as GRIT [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>] that incorporate other data such as 5’ ends from CAGE or RAMPAGE typically have a better chance of annotating the major expressed isoforms correctly. In any case, PE reads and higher coverage help to reconstruct lowly expressed transcripts, and replicates are essential to resolve false-positive calls (that is, mapping artifacts or contaminations) at the low end of signal detection. Several methods, such as Cufflinks [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>], iReckon [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>], SLIDE [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>] and StringTie [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>], incorporate existing annotations by adding them to the possible list of isoforms. Montebello [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>] couples isoform discovery and quantification using a likelihood-based Monte Carlo algorithm to boost performance. Gene-finding tools such as Augustus [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>] can incorporate RNA-seq data to better annotate protein-coding transcripts, but perform worse on non-coding transcripts [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>]. In general, accurate transcript reconstruction from short reads is difficult, and methods typically show substantial disagreement [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>].</p>
                </section>
                <section id="Sec12" class="Section3 RenderAsSection3">
                  <h4 class="Heading">De novo transcript reconstruction</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par47" class="Para">When a reference genome is not available or is incomplete, RNA-seq reads can be assembled de novo (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2c</a></span>) into a transcriptome using packages such as SOAPdenovo-Trans [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>], Oases [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>], Trans-ABySS [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>] or Trinity [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>]. In general, PE strand-specific sequencing and long reads are preferred because they are more informative [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>]. Although it is impossible to assemble lowly expressed transcripts that lack enough coverage for a reliable assembly, too many reads are also problematic because they lead to potential misassembly and increased runtimes. Therefore, in silico reduction of the number of reads is recommended for deeply sequenced samples [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>]. For comparative analyses across samples, it is advisable to combine all reads from multiple samples into a single input in order to obtain a consolidated set of contigs (transcripts), followed by mapping back of the short reads for expression estimation [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>].</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par48" class="Para">Either with a reference or de novo, the complete reconstruction of transcriptomes using short-read Illumina technology remains a challenging problem, and in many cases de novo assembly results in tens or hundreds of contigs accounting for fragmented transcripts. Emerging long-read technologies, such as SMRT from Pacific Biosciences, provide reads that are long enough to sequence complete transcripts for most genes and are a promising alternative that is discussed further in the “Outlook” section below.</p>
                </section>
              </section><section id="Sec13" class="Section2 RenderAsSection2">
                <h3 class="Heading">Transcript quantification</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par49" class="Para">The most common application of RNA-seq is to estimate gene and transcript expression. This application is primarily based on the number of reads that map to each transcript sequence, although there are algorithms such as Sailfish that rely on <em xmlns="" class="EmphasisTypeItalic">k</em>-mer counting in reads without the need for mapping [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. The simplest approach to quantification is to aggregate raw counts of mapped reads using programs such as HTSeq-count [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>] or featureCounts [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>]. This gene-level (rather than transcript-level) quantification approach utilizes a gene transfer format (GTF) file [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>] containing the genome coordinates of exons and genes, and often discard multireads. Raw read counts alone are not sufficient to compare expression levels among samples, as these values are affected by factors such as transcript length, total number of reads, and sequencing biases. The measure RPKM (reads per kilobase of exon model per million reads) [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>] is a within-sample normalization method that will remove the feature-length and library-size effects. This measure and its subsequent derivatives FPKM (fragments per kilobase of exon model per million mapped reads), a within-sample normalized transcript expression measure analogous to RPKs, and TPM (transcripts per million) are the most frequently reported RNA-seq gene expression values. It should be noted that RPKM and FPKM are equivalent for SE reads and that FPKM can be converted into TPM using a simple formula [<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>]. The dichotomy of within-sample and between-sample comparisons has led to a lot of confusion in the literature. Correcting for gene length is not necessary when comparing changes in gene expression within the same gene across samples, but it is necessary for correctly ranking gene expression levels within the sample to account for the fact that longer genes accumulate more reads. Furthermore, programs such as Cufflinks that estimate gene length from the data can find significant differences in gene length between samples that cannot be ignored. TPMs, which effectively normalize for the differences in composition of the transcripts in the denominator rather than simply dividing by the number of reads in the library, are considered more comparable between samples of different origins and composition but can still suffer some biases. These must be addressed with normalization techniques such as TMM.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par50" class="Para">Several sophisticated algorithms have been developed to estimate transcript-level expression by tackling the problem of related transcripts’ sharing most of their reads. Cufflinks [<span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>] estimates transcript expression from a mapping to the genome obtained from mappers such as TopHat using an expectation-maximization approach that estimates transcript abundances. This approach takes into account biases such as the non-uniform read distribution along the gene length. Cufflinks was designed to take advantage of PE reads, and may use GTF information to identify expressed transcripts, or can infer transcripts de novo from the mapping data alone. Algorithms that quantify expression from transcriptome mappings include RSEM (RNA-Seq by Expectation Maximization) [<span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>], eXpress [<span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>], Sailfish [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>] and kallisto [<span class="CitationRef"><a href="#CR42" data-toggle="citation">42</a></span>] among others. These methods allocate multi-mapping reads among transcript and output within-sample normalized values corrected for sequencing biases [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>, <span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>, <span class="CitationRef"><a href="#CR43" data-toggle="citation">43</a></span>]. Additionally, the RSEM algorithm uses an expectation maximization approach that returns TPM values [<span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>]. NURD [<span class="CitationRef"><a href="#CR44" data-toggle="citation">44</a></span>] provides an efficient way of estimating transcript expression from SE reads with a low memory and computing cost.</p>
              </section><section id="Sec14" class="Section2 RenderAsSection2">
                <h3 class="Heading">Differential gene expression analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par51" class="Para">Differential expression analysis (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1b</a></span>) requires that gene expression values should be compared among samples. RPKM, FPKM, and TPM normalize away the most important factor for comparing samples, which is sequencing depth, whether directly or by accounting for the number of transcripts, which can differ significantly between samples. These approaches rely on normalizing methods that are based on total or effective counts, and tend to perform poorly when samples have heterogeneous transcript distributions, that is, when highly and differentially expressed features can skew the count distribution [<span class="CitationRef"><a href="#CR45" data-toggle="citation">45</a></span>, <span class="CitationRef"><a href="#CR46" data-toggle="citation">46</a></span>]. Normalization methods that take this into account are TMM [<span class="CitationRef"><a href="#CR47" data-toggle="citation">47</a></span>], DESeq [<span class="CitationRef"><a href="#CR48" data-toggle="citation">48</a></span>], PoissonSeq [<span class="CitationRef"><a href="#CR49" data-toggle="citation">49</a></span>] and UpperQuartile [<span class="CitationRef"><a href="#CR45" data-toggle="citation">45</a></span>], which ignore highly variable and/or highly expressed features. Additional factors that interfere with intra-sample comparisons include changes in transcript length across samples or conditions [<span class="CitationRef"><a href="#CR50" data-toggle="citation">50</a></span>], positional biases in coverage along the transcript (which are accounted for in Cufflinks), average fragment size [<span class="CitationRef"><a href="#CR43" data-toggle="citation">43</a></span>], and the GC contents of genes (corrected in the EDAseq package [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]). The NOISeq R package [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>] contains a wide variety of diagnostic plots to identify sources of biases in RNA-seq data and to apply appropriate normalization procedures in each case. Finally, despite these sample-specific normalization methods, batch effects may still be present in the data. These effects can be minimized by appropriate experimental design [<span class="CitationRef"><a href="#CR51" data-toggle="citation">51</a></span>] or, alternatively, removed by batch-correction methods such as COMBAT [<span class="CitationRef"><a href="#CR52" data-toggle="citation">52</a></span>] or ARSyN [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>, <span class="CitationRef"><a href="#CR53" data-toggle="citation">53</a></span>]. These approaches, although initially developed for microarray data, have been shown to work well with normalized RNA-seq data (STATegra project, unpublished).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par52" class="Para">As RNA-seq quantification is based on read counts that are absolutely or probabilistically assigned to transcripts, the first approaches to compute differential expression used discrete probability distributions, such as the Poisson or negative binomial [<span class="CitationRef"><a href="#CR48" data-toggle="citation">48</a></span>, <span class="CitationRef"><a href="#CR54" data-toggle="citation">54</a></span>]. The negative binomial distribution (also known as the gamma-Poisson distribution) is a generalization of the Poisson distribution, allowing for additional variance (called overdispersion) beyond the variance expected from randomly sampling from a pool of molecules that are characteristic of RNA-seq data. However, the use of discrete distributions is not required for accurate analysis of differential expression as long as the sampling variance of small read counts is taken into account (most important for experiments with small numbers of replicates). Methods for transforming normalized counts of RNA-seq reads while learning the variance structure of the data have been shown to perform well in comparison to the discrete distribution approaches described above [<span class="CitationRef"><a href="#CR55" data-toggle="citation">55</a></span>, <span class="CitationRef"><a href="#CR56" data-toggle="citation">56</a></span>]. Moreover, after extensive normalization (including TMM and batch removal), the data might have lost their discrete nature and be more akin to a continuous distribution.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par53" class="Para">Some methods, such as the popular edgeR [<span class="CitationRef"><a href="#CR57" data-toggle="citation">57</a></span>], take as input raw read counts and introduce possible bias sources into the statistical model to perform an integrated normalization as well as a differential expression analysis. In other methods, the differential expression requires the data to be previously normalized to remove all possible biases. DESeq2, like edgeR, uses the negative binomial as the reference distribution and provides its own normalization approach [<span class="CitationRef"><a href="#CR48" data-toggle="citation">48</a></span>, <span class="CitationRef"><a href="#CR58" data-toggle="citation">58</a></span>]. baySeq [<span class="CitationRef"><a href="#CR59" data-toggle="citation">59</a></span>] and EBSeq [<span class="CitationRef"><a href="#CR60" data-toggle="citation">60</a></span>] are Bayesian approaches, also based on the negative binomial model, that define a collection of models to describe the differences among experimental groups and to compute the posterior probability of each one of them for each gene. Other approaches include data transformation methods that take into account the sampling variance of small read counts and create discrete gene expression distributions that can be analyzed by regular linear models [<span class="CitationRef"><a href="#CR55" data-toggle="citation">55</a></span>]. Finally, non-parametric approaches such as NOISeq [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>] or SAMseq [<span class="CitationRef"><a href="#CR61" data-toggle="citation">61</a></span>] make minimal assumptions about the data and estimate the null distribution for inferential analysis from the actual data alone. For small-scale studies that compare two samples with no or few replicates, the estimation of the negative binomial distribution can be noisy. In such cases, simpler methods based on the Poisson distribution, such as DEGseq [<span class="CitationRef"><a href="#CR62" data-toggle="citation">62</a></span>], or on empirical distributions (NOISeq [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]) can be an alternative, although it should be strongly stressed that, in the absence of biological replication, no population inference can be made and hence any <em xmlns="" class="EmphasisTypeItalic">p</em> value calculation is invalid. Methods that analyze RNA-seq data without replicates therefore only have exploratory value. Considering the drop in price of sequencing, we recommend that RNA-seq experiments have a minimum of three biological replicates when sample availability is not limiting to allow all of the differential expression methods to leverage reproducibility between replicates.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par54" class="Para">Recent independent comparison studies have demonstrated that the choice of the method (or even the version of a software package) can markedly affect the outcome of the analysis and that no single method is likely to perform favorably for all datasets [<span class="CitationRef"><a href="#CR56" data-toggle="citation">56</a></span>, <span class="CitationRef"><a href="#CR63" data-toggle="citation">63</a></span>, <span class="CitationRef"><a href="#CR64" data-toggle="citation">64</a></span>] (Box 4). We therefore recommend thoroughly documenting the settings and version numbers of programs used and considering the repetition of important analyses using more than one package.</p>
              </section><section id="Sec15" class="Section2 RenderAsSection2">
                <h3 class="Heading">Alternative splicing analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par55" class="Para">Transcript-level differential expression analysis can potentially detect changes in the expression of transcript isoforms from the same gene, and specific algorithms for alternative splicing-focused analysis using RNA-seq have been proposed. These methods fall into two major categories. The first approach integrates isoform expression estimation with the detection of differential expression to reveal changes in the proportion of each isoform within the total gene expression. One such early method, BASIS, used a hierarchical Bayesian model to directly infer differentially expressed transcript isoforms [<span class="CitationRef"><a href="#CR65" data-toggle="citation">65</a></span>]. CuffDiff2 estimates isoform expression first and then compares their differences. By integrating the two steps, the uncertainty in the first step is taken into consideration when performing the statistical analysis to look for differential isoform expression [<span class="CitationRef"><a href="#CR66" data-toggle="citation">66</a></span>]. The flow difference metric (FDM) uses aligned cumulative transcript graphs from mapped exon reads and junction reads to infer isoforms and the Jensen-Shannon divergence to measure the difference [<span class="CitationRef"><a href="#CR67" data-toggle="citation">67</a></span>]. Recently, Shi and Jiang [<span class="CitationRef"><a href="#CR68" data-toggle="citation">68</a></span>] proposed a new method, rSeqDiff, that uses a hierarchical likelihood ratio test to detect differential gene expression without splicing change and differential isoform expression simultaneously. All these approaches are generally hampered by the intrinsic limitations of short-read sequencing for accurate identification at the isoform level, as discussed in the RNA-seq Genome Annotation Assessment Project paper [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par56" class="Para">The so-called ‘exon-based’ approach skips the estimation of isoform expression and detects signals of alternative splicing by comparing the distributions of reads on exons and junctions of the genes between the compared samples. This approach is based on the premise that differences in isoform expression can be tracked in the signals of exons and their junctions. DEXseq [<span class="CitationRef"><a href="#CR69" data-toggle="citation">69</a></span>] and DSGSeq [<span class="CitationRef"><a href="#CR70" data-toggle="citation">70</a></span>] adopt a similar idea to detect differentially spliced genes by testing for significant differences in read counts on exons (and junctions) of the genes. rMATS detects differential usage of exons by comparing exon-inclusion levels defined with junction reads [<span class="CitationRef"><a href="#CR71" data-toggle="citation">71</a></span>]. rDiff detects differential isoform expression by comparing read counts on alternative regions of the gene, either with or without annotated alternative isoforms [<span class="CitationRef"><a href="#CR72" data-toggle="citation">72</a></span>]. DiffSplice uses alignment graphs to identify alternative splicing modules (ASMs) and identifies differential splicing using signals of the ASMs [<span class="CitationRef"><a href="#CR73" data-toggle="citation">73</a></span>]. The advantage of exon or junction methods is their greater accuracy in identifying individual alternative splicing events. Exon-based methods are appropriate if the focus of the study is not on whole isoforms but on the inclusion and exclusion of specific exons and the functional protein domains (or regulatory features, in case of untranslated region exons) that they contain.</p>
              </section><section id="Sec16" class="Section2 RenderAsSection2">
                <h3 class="Heading">Visualization</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par57" class="Para">Visualization of RNA-seq data (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1c</a></span>) is, in general terms, similar to that of any other type of genomic sequencing data, and it can be done at the level of reads (using ReadXplorer [<span class="CitationRef"><a href="#CR74" data-toggle="citation">74</a></span>], for example) or at the level of processed coverage (read pileup), unnormalized (for example, total count) or normalized, using genome browsers such as the UCSC browser [<span class="CitationRef"><a href="#CR75" data-toggle="citation">75</a></span>], Integrative Genomics Viewer (IGV) [<span class="CitationRef"><a href="#CR76" data-toggle="citation">76</a></span>] (Figure S1a in Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>), Genome Maps [<span class="CitationRef"><a href="#CR77" data-toggle="citation">77</a></span>], or Savant [<span class="CitationRef"><a href="#CR78" data-toggle="citation">78</a></span>]. Some visualization tools are specifically designed for visualizing multiple RNA-seq samples, such as RNAseqViewer [<span class="CitationRef"><a href="#CR79" data-toggle="citation">79</a></span>], which provides flexible ways to display the read abundances on exons, transcripts and junctions. Introns can be hidden to better display signals on the exons, and the heatmaps can help the visual comparison of signals on multiple samples (Figure S1b, c in Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>). However, RNAseqViewer is slower than IGV.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par58" class="Para">Some of the software packages for differential gene expression analysis (such as DESeq2 or DEXseq in Bioconductor) have functions to enable the visualization of results, whereas others have been developed for visualization-exclusive purposes, such as CummeRbund (for CuffDiff [<span class="CitationRef"><a href="#CR66" data-toggle="citation">66</a></span>]) or Sashimi plots, which can be used to visualize differentially spliced exons [<span class="CitationRef"><a href="#CR80" data-toggle="citation">80</a></span>]. The advantage of Sashimi plots is that their display of junction reads is more intuitive and aesthetically pleasing when the number of samples is small (Figure S1d in Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>). Sashimi, structure, and hive plots for splicing quantitative trait loci (sQTL) can be obtained using SplicePlot [<span class="CitationRef"><a href="#CR81" data-toggle="citation">81</a></span>]. Splice graphs can be produced using SpliceSeq [<span class="CitationRef"><a href="#CR82" data-toggle="citation">82</a></span>], and SplicingViewer [<span class="CitationRef"><a href="#CR83" data-toggle="citation">83</a></span>] plots splice junctions and alternative splicing events. TraV [<span class="CitationRef"><a href="#CR84" data-toggle="citation">84</a></span>] is a visualization tool that integrates data analysis, but its analytical methods are not applicable to large genomes.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par59" class="Para">Owing to the complexity of transcriptomes, efficient display of multiple layers of information is still a challenge. All of the tools are evolving rapidly and we can expect more comprehensive tools with desirable features to be available soon. Nevertheless, the existing tools are of great value for exploring results for individual genes of biological interest to assess whether particular analyses’ results can withstand detailed scrutiny or to reveal potential complications caused by artifacts, such as 3’ biases or complicated transcript structures. Users should visualize changes in read coverage for genes that are deemed important or interesting on the basis of their analysis results to evaluate the robustness of their conclusions.</p>
              </section><section id="Sec17" class="Section2 RenderAsSection2">
                <h3 class="Heading">Gene fusion discovery</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par60" class="Para">The discovery of fused genes that can arise from chromosomal rearrangements is analogous to novel isoform discovery, with the added challenge of a much larger search space as we can no longer assume that the transcript segments are co-linear on a single chromosome. Artifacts are common even using state-of-the-art tools, which necessitates post-processing using heuristic filters [<span class="CitationRef"><a href="#CR85" data-toggle="citation">85</a></span>]. Artifacts primarily result from misalignment of read sequences due to polymorphisms, homology, and sequencing errors. Families of homologous genes, and highly polymorphic genes such as the HLA genes, produce reads that cannot be easily mapped uniquely to their location of origin in the reference genome. For genes with very high expression, the small but non-negligible sequencing error rate of RNA-seq will produce reads that map incorrectly to homologous loci. Filtering highly polymorphic genes and pairs of homologous genes is recommended [<span class="CitationRef"><a href="#CR86" data-toggle="citation">86</a></span>, <span class="CitationRef"><a href="#CR87" data-toggle="citation">87</a></span>]. Also recommended is the filtering of highly expressed genes that are unlikely to be involved in gene fusions, such as ribosomal RNA [<span class="CitationRef"><a href="#CR86" data-toggle="citation">86</a></span>]. Finally, a low ratio of chimeric to wild-type reads in the vicinity of the fusion boundary may indicate spurious mis-mapping of reads from a highly expressed gene (the transcript allele fraction described by Yoshihara et al. [<span class="CitationRef"><a href="#CR87" data-toggle="citation">87</a></span>]).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par61" class="Para">Given successful prediction of chimeric sequences, the next step is the prioritization of gene fusions that have biological impact over more expected forms of genomic variation. Examples of expected variation include immunoglobulin (IG) rearrangements in tumor samples infiltrated by immune cells, transiently expressed transposons and nuclear mitochondrial DNA, and read-through chimeras produced by co-transcription of adjacent genes [<span class="CitationRef"><a href="#CR88" data-toggle="citation">88</a></span>]. Care must be taken with filtering in order not to lose events of interest. For example, removing all fusions involving an IG gene may remove real IG fusions in lymphomas and other blood disorders; filtering fusions for which both genes are from the IG locus is preferred [<span class="CitationRef"><a href="#CR88" data-toggle="citation">88</a></span>]. Transiently expressed genomic breakpoint sequences that are associated with real gene fusions often overlap transposons; these should be filtered unless they are associated with additional fusion isoforms from the same gene pair [<span class="CitationRef"><a href="#CR89" data-toggle="citation">89</a></span>]. Read-through chimeras are easily identified as predictions involving alternative splicing between adjacent genes. Where possible, fusions should be filtered by their presence in a set of control datasets [<span class="CitationRef"><a href="#CR87" data-toggle="citation">87</a></span>]. When control datasets are not available, artifacts can be identified by their presence in a large number of unrelated datasets, after excluding the possibility that they represent true recurrent fusions [<span class="CitationRef"><a href="#CR90" data-toggle="citation">90</a></span>, <span class="CitationRef"><a href="#CR91" data-toggle="citation">91</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par62" class="Para">Strong fusion-sequence predictions are characterized by distinct subsequences that each align with high specificity to one of the fused genes. As alignment specificity is highly correlated with sequence length, a strong prediction sequence is longer, with longer subsequences from each gene. Longer reads and larger insert sizes produce longer predicted sequences; thus, we recommend PE RNA-seq data with larger insert size over SE datasets or datasets with short insert size. Another indicator of prediction strength is splicing. For most known fusions, the genomic breakpoint is located in an intron of each gene [<span class="CitationRef"><a href="#CR92" data-toggle="citation">92</a></span>] and the fusion boundary coincides with a splice site within each gene. Furthermore, fusion isoforms generally follow the splicing patterns of wild-type genes. Thus, high confidence predictions have fusion boundaries coincident with exon boundaries and exons matching wild-type exons [<span class="CitationRef"><a href="#CR91" data-toggle="citation">91</a></span>]. Fusion discovery tools often incorporate some of the aforementioned ideas to rank fusion predictions [<span class="CitationRef"><a href="#CR93" data-toggle="citation">93</a></span>, <span class="CitationRef"><a href="#CR94" data-toggle="citation">94</a></span>], though most studies apply additional custom heuristic filters to produce a list of high-quality fusion candidates [<span class="CitationRef"><a href="#CR90" data-toggle="citation">90</a></span>, <span class="CitationRef"><a href="#CR91" data-toggle="citation">91</a></span>, <span class="CitationRef"><a href="#CR95" data-toggle="citation">95</a></span>].</p>
              </section><section id="Sec18" class="Section2 RenderAsSection2">
                <h3 class="Heading">Small RNAs</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par63" class="Para">Next-generation sequencing represents an increasingly popular method to address questions concerning the biological roles of small RNAs (sRNAs). sRNAs are usually 18–34 nucleotides in length, and they include miRNAs, short-interfering RNAs (siRNAs), PIWI-interacting RNAs (piRNAs), and other classes of regulatory molecules. sRNA-seq libraries are rarely sequenced as deeply as regular RNA-seq libraries because of a lack of complexity, with a typical range of 2–10 million reads. Bioinformatics analysis of sRNA-seq data differs from standard RNA-seq protocols (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1c</a></span>). Ligated adaptor sequences are first trimmed and the resulting read-length distribution is computed. In animals, there are usually peaks for 22 and 23 nucleotides, whereas in plants there are peaks for 21- and 24-nucleotide redundant reads. For instance, miRTools 2.0 [<span class="CitationRef"><a href="#CR96" data-toggle="citation">96</a></span>], a tool for prediction and profiling of sRNA species, uses by default reads that are 18–30 bases long. The threshold value depends on the application, and in case of miRNAs is usually in the range of 19–25 nucleotides.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par64" class="Para">As in standard RNA-seq, sRNA reads must then be aligned to a reference genome or transcriptome sequences using standard tools, such as Bowtie2 [<span class="CitationRef"><a href="#CR97" data-toggle="citation">97</a></span>], STAR [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>], or Burrows-Wheeler Aligner (BWA) [<span class="CitationRef"><a href="#CR98" data-toggle="citation">98</a></span>]. There are, however, some aligners (such as PatMaN [<span class="CitationRef"><a href="#CR99" data-toggle="citation">99</a></span>] and MicroRazerS [<span class="CitationRef"><a href="#CR100" data-toggle="citation">100</a></span>]) that have been designed to map short sequences with preset parameter value ranges suited for optimal alignment of short reads. The mapping itself may be performed with or without mismatches, the latter being used more commonly. In addition, reads that map beyond a predetermined set number of locations may be removed as putatively originating from repetitive elements. In the case of miRNAs, usually 5–20 distinct mappings per genome are allowed. sRNA reads are then simply counted to obtain expression values. However, users should also verify that their sRNA reads are not significantly contaminated by degraded mRNA, for example, by checking whether a miRNA library shows unexpected read coverage over the body of highly expressed genes such as <em xmlns="" class="EmphasisTypeItalic">GAPDH</em> or <em xmlns="" class="EmphasisTypeItalic">ACTB</em>.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par65" class="Para">Further analysis steps include comparison with known sRNAs and de novo identification of sRNAs. There are class-specific tools for this purpose, such as miRDeep [<span class="CitationRef"><a href="#CR101" data-toggle="citation">101</a></span>] and miRDeep-P [<span class="CitationRef"><a href="#CR102" data-toggle="citation">102</a></span>] for animal and plant miRNAs, respectively, or the trans-acting siRNA prediction tool at the UEA sRNA Workbench [<span class="CitationRef"><a href="#CR103" data-toggle="citation">103</a></span>]. Tools such as miRTools 2.0 [<span class="CitationRef"><a href="#CR96" data-toggle="citation">96</a></span>], ShortStack [<span class="CitationRef"><a href="#CR104" data-toggle="citation">104</a></span>], and iMir [<span class="CitationRef"><a href="#CR105" data-toggle="citation">105</a></span>] also exist for comprehensive annotation of sRNA libraries and for identification of diverse classes of sRNAs.</p>
              </section><section id="Sec19" class="Section2 RenderAsSection2">
                <h3 class="Heading">Functional profiling with RNA-seq</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par66" class="Para">The last step in a standard transcriptomics study (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1b</a></span>) is often the characterization of the molecular functions or pathways in which differentially expressed genes (DEGs) are involved. The two main approaches to functional characterization that were developed first for microarray technology are (a) comparing a list of DEGs against the rest of the genome for overrepresented functions, and (b) gene set enrichment analysis (GSEA), which is based on ranking the transcriptome according to a measurement of differential expression. RNA-seq biases such as gene length complicate the direct applications of these methods for count data and hence RNA-seq-specific tools have been proposed. For example, GOseq [<span class="CitationRef"><a href="#CR106" data-toggle="citation">106</a></span>] estimates a bias effect (such as gene length) on differential expression results and adapts the traditional hypergeometric statistic used in the functional enrichment test to account for this bias. Similarly, the Gene Set Variation Analysis (GSVA) [<span class="CitationRef"><a href="#CR107" data-toggle="citation">107</a></span>] or SeqGSEA [<span class="CitationRef"><a href="#CR108" data-toggle="citation">108</a></span>] packages also combine splicing and implement enrichment analyses similar to GSEA.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par67" class="Para">Functional analysis requires the availability of sufficient functional annotation data for the transcriptome under study. Resources such as Gene Ontology [<span class="CitationRef"><a href="#CR109" data-toggle="citation">109</a></span>], Bioconductor [<span class="CitationRef"><a href="#CR110" data-toggle="citation">110</a></span>], DAVID [<span class="CitationRef"><a href="#CR111" data-toggle="citation">111</a></span>, <span class="CitationRef"><a href="#CR112" data-toggle="citation">112</a></span>] or Babelomics [<span class="CitationRef"><a href="#CR113" data-toggle="citation">113</a></span>] contain annotation data for most model species. However, novel transcripts discovered during de novo transcriptome assembly or reconstruction would lack at least some functional information and therefore annotation is necessary for functional profiling of those results. Protein-coding transcripts can be functionally annotated using orthology by searching for similar sequences in protein databases such as SwissProt [<span class="CitationRef"><a href="#CR114" data-toggle="citation">114</a></span>] and in databases that contain conserved protein domains such as Pfam [<span class="CitationRef"><a href="#CR115" data-toggle="citation">115</a></span>] and InterPro [<span class="CitationRef"><a href="#CR116" data-toggle="citation">116</a></span>]. The use of standard vocabularies such as the Gene Ontology (GO) allows for some exchangeability of functional information across orthologs. Popular tools such as Blast2GO [<span class="CitationRef"><a href="#CR117" data-toggle="citation">117</a></span>] allow massive annotation of complete transcriptome datasets against a variety of databases and controlled vocabularies. Typically, between 50 and 80&nbsp;% of the transcripts reconstructed from RNA-seq data can be annotated with functional terms in this way. However, RNA-seq data also reveal that an important fraction of the transcriptome is lacking protein-coding potential. The functional annotation of these long non-coding RNAs is more challenging as their conservation is often less pronounced than that of protein-coding genes. The Rfam database [<span class="CitationRef"><a href="#CR118" data-toggle="citation">118</a></span>] contains most well-characterized RNA families, such as ribosomal or transfer RNAs, while mirBase [<span class="CitationRef"><a href="#CR119" data-toggle="citation">119</a></span>] or Miranda [<span class="CitationRef"><a href="#CR120" data-toggle="citation">120</a></span>] are specialized in miRNAs. These resources can be used for similarity-based annotation of short non-coding RNAs, but no standard functional annotation procedures are available yet for other RNA types such as the long non-coding RNAs.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec20" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Integrationwithotherdatatypes" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Integrationwithotherdatatypes">Integration with other data types</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Integrationwithotherdatatypes" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par68" class="Para">The integration of RNA-seq data with other types of genome-wide data (Fig.&nbsp;<span class="InternalRef"><a href="#Fig1">1c</a></span>) allows us to connect the regulation of gene expression with specific aspects of molecular physiology and functional genomics. Integrative analyses that incorporate RNA-seq data as the primary gene expression readout that is compared with other genomic experiments are becoming increasingly prevalent. Below, we discuss some of the additional challenges posed by such analyses.</p><section id="Sec21" class="Section2 RenderAsSection2">
                <h3 class="Heading">DNA sequencing</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par69" class="Para">The combination of RNA and DNA sequencing can be used for several purposes, such as single nucleotide polymorphism (SNP) discovery, RNA-editing analyses, or expression quantitative trait loci (eQTL) mapping. In a typical eQTL experiment, genotype and transcriptome profiles are obtained from the same tissue type across a relatively large number of individuals (&gt;50) and correlations between genotype and expression levels are then detected. These associations can unravel the genetic basis of complex traits such as height [<span class="CitationRef"><a href="#CR121" data-toggle="citation">121</a></span>], disease susceptibility [<span class="CitationRef"><a href="#CR122" data-toggle="citation">122</a></span>] or even features of genome architecture [<span class="CitationRef"><a href="#CR123" data-toggle="citation">123</a></span>, <span class="CitationRef"><a href="#CR124" data-toggle="citation">124</a></span>]. Large eQTL studies have shown that genetic variation affects the expression of most genes [<span class="CitationRef"><a href="#CR125" data-toggle="citation">125</a></span>–<span class="CitationRef"><a href="#CR128" data-toggle="citation">128</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par70" class="Para">RNA-seq has two major advantages over array-based technologies for detecting eQTLs. First, it can identify variants that affect transcript processing. Second, reads that overlap heterozygous SNPs can be mapped to maternal and paternal chromosomes, enabling quantification of allele-specific expression within an individual [<span class="CitationRef"><a href="#CR129" data-toggle="citation">129</a></span>]. Allele-specific signals provide additional information about a genetic effect on transcription, and a number of computational methods have recently become available that leverage these signals to boost power for association mapping [<span class="CitationRef"><a href="#CR130" data-toggle="citation">130</a></span>–<span class="CitationRef"><a href="#CR132" data-toggle="citation">132</a></span>]. One challenge of this approach is the computational burden, as billions of gene–SNP associations need to be tested; bootstrapping or permutation-based approaches [<span class="CitationRef"><a href="#CR133" data-toggle="citation">133</a></span>] are frequently used [<span class="CitationRef"><a href="#CR134" data-toggle="citation">134</a></span>, <span class="CitationRef"><a href="#CR135" data-toggle="citation">135</a></span>]. Many studies have focused on testing only SNPs in the <em xmlns="" class="EmphasisTypeItalic">cis</em> region surrounding the gene in question, and computationally efficient approaches have been developed recently to allow extremely swift mapping of eQTLs genome-wide [<span class="CitationRef"><a href="#CR136" data-toggle="citation">136</a></span>]. Moreover, the combination of RNA-seq and re-sequencing can be used both to remove false positives when inferring fusion genes [<span class="CitationRef"><a href="#CR88" data-toggle="citation">88</a></span>] and to analyze copy number alterations [<span class="CitationRef"><a href="#CR137" data-toggle="citation">137</a></span>].</p>
              </section><section id="Sec22" class="Section2 RenderAsSection2">
                <h3 class="Heading">DNA methylation</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par71" class="Para">Pairwise DNA-methylation and RNA-seq integration, for the most part, has consisted of the analysis of the correlation between DEGs and methylation patterns [<span class="CitationRef"><a href="#CR138" data-toggle="citation">138</a></span>–<span class="CitationRef"><a href="#CR140" data-toggle="citation">140</a></span>]. General linear models [<span class="CitationRef"><a href="#CR141" data-toggle="citation">141</a></span>–<span class="CitationRef"><a href="#CR143" data-toggle="citation">143</a></span>], logistic regression models [<span class="CitationRef"><a href="#CR143" data-toggle="citation">143</a></span>] and empirical Bayes model [<span class="CitationRef"><a href="#CR144" data-toggle="citation">144</a></span>] have been attempted among other modeling approaches. The statistically significant correlations that were observed, however, accounted for relatively small effects. An interesting shift away from focusing on individual gene–CpG methylation correlations is to use a network-interaction-based approach to analyze RNA-seq in relation to DNA methylation. This approach identifies one or more sets of genes (also called modules) that have coordinated differential expression and differential methylation [<span class="CitationRef"><a href="#CR145" data-toggle="citation">145</a></span>].</p>
              </section><section id="Sec23" class="Section2 RenderAsSection2">
                <h3 class="Heading">Chromatin features</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par72" class="Para">The combination of RNA-seq and transcription factor (TF) chromatin immunoprecipitation sequencing (ChIP-seq) data can be used to remove false positives in ChIP-seq analysis and to suggest the activating or repressive effect of a TF on its target genes. For example, BETA [<span class="CitationRef"><a href="#CR146" data-toggle="citation">146</a></span>] uses differential gene expression in combination with peaks from ChIP-seq experiments to call TF targets. In addition, ChIP-seq experiments involving histone modifications have been used to understand the general role of these epigenomic changes on gene expression [<span class="CitationRef"><a href="#CR147" data-toggle="citation">147</a></span>, <span class="CitationRef"><a href="#CR148" data-toggle="citation">148</a></span>]. Other RNA-ChIP-sequencing integrative approaches are reviewed in [<span class="CitationRef"><a href="#CR149" data-toggle="citation">149</a></span>]. Integration of open chromatin data such as that from FAIRE-seq and DNase-seq with RNA-seq has mostly been limited to verifying the expression status of genes that overlap a region of interest [<span class="CitationRef"><a href="#CR150" data-toggle="citation">150</a></span>]. DNase-seq can be used for genome-wide footprinting of DNA-binding factors, and this in combination with the actual expression of genes can be used to infer active transcriptional networks [<span class="CitationRef"><a href="#CR150" data-toggle="citation">150</a></span>].</p>
              </section><section id="Sec24" class="Section2 RenderAsSection2">
                <h3 class="Heading">MicroRNAs</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par73" class="Para">Integration of RNA-seq and miRNA-seq data has the potential to unravel the regulatory effects of miRNAs on transcript steady-state levels. This analysis is challenging, however, because of the very noisy nature of miRNA target predictions, which hampers analyses based on correlations between miRNAs and their target genes. Associations might be found in databases such as mirWalk [<span class="CitationRef"><a href="#CR151" data-toggle="citation">151</a></span>] and miRBase [<span class="CitationRef"><a href="#CR152" data-toggle="citation">152</a></span>] that offer target prediction according to various algorithms. Tools such as CORNA [<span class="CitationRef"><a href="#CR153" data-toggle="citation">153</a></span>], MMIA [<span class="CitationRef"><a href="#CR154" data-toggle="citation">154</a></span>, <span class="CitationRef"><a href="#CR155" data-toggle="citation">155</a></span>], MAGIA [<span class="CitationRef"><a href="#CR156" data-toggle="citation">156</a></span>], and SePIA [<span class="CitationRef"><a href="#CR157" data-toggle="citation">157</a></span>] refine predictions by testing for significant associations between genes, miRNAs, pathways and GO terms, or by testing the relatedness or anticorrelation of the expression profiles of both the target genes and the associated miRNAs. In general, we recommend using miRNA–mRNA associations that are predicted by several algorithms. For example, in mouse, we found that requiring miRNA–mRNA association in five databases resulted in about 50 target mRNA predictions per miRNA (STATegra observations).</p>
              </section><section id="Sec25" class="Section2 RenderAsSection2">
                <h3 class="Heading">Proteomics and metabolomics</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par74" class="Para">Integration of RNA-seq with proteomics is controversial because the two measurements show generally low correlation (~0.40 [<span class="CitationRef"><a href="#CR158" data-toggle="citation">158</a></span>, <span class="CitationRef"><a href="#CR159" data-toggle="citation">159</a></span>]). Nevertheless, pairwise integration of proteomics and RNA-seq can be used to identify novel isoforms. Unreported peptides can be predicted from RNA-seq data and then used to complement databases normally queried in mass spectrometry as done by Low et al. [<span class="CitationRef"><a href="#CR160" data-toggle="citation">160</a></span>]. Furthermore, post-translational editing events may be identified if peptides that are present in the mass spectrometry analysis are absent from the expressed genes of the RNA-seq dataset. Integration of transcriptomics with metabolomics data has been used to identify pathways that are regulated at both the gene expression and the metabolite level, and tools are available that visualize results within the pathway context (MassTRIX [<span class="CitationRef"><a href="#CR161" data-toggle="citation">161</a></span>], Paintomics [<span class="CitationRef"><a href="#CR162" data-toggle="citation">162</a></span>], VANTED v2 [<span class="CitationRef"><a href="#CR163" data-toggle="citation">163</a></span>], and SteinerNet [<span class="CitationRef"><a href="#CR164" data-toggle="citation">164</a></span>]).</p>
              </section><section id="Sec26" class="Section2 RenderAsSection2">
                <h3 class="Heading">Integration and visualization of multiple data types</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par75" class="Para">Integration of more than two genomic data types is still at its infancy and not yet extensively applied to functional sequencing techniques, but there are already some tools that combine several data types. SNMNMF [<span class="CitationRef"><a href="#CR165" data-toggle="citation">165</a></span>] and PIMiM [<span class="CitationRef"><a href="#CR166" data-toggle="citation">166</a></span>] combine mRNA and miRNA expression data with protein–protein, DNA–protein, and miRNA–mRNA interaction networks to identify miRNA–gene regulatory modules. MONA [<span class="CitationRef"><a href="#CR167" data-toggle="citation">167</a></span>] combines different levels of functional genomics data, including mRNA, miRNA, DNA methylation, and proteomics data to discover altered biological functions in the samples being studied. Paintomics can integrate any type of functional genomics data into pathway analysis, provided that the features can be mapped onto genes or metabolites [<span class="CitationRef"><a href="#CR162" data-toggle="citation">162</a></span>]. 3Omics [<span class="CitationRef"><a href="#CR168" data-toggle="citation">168</a></span>] integrates transcriptomics, metabolomics and proteomics data into regulatory networks.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par76" class="Para">In all cases, integration of different datasets is rarely straightforward because each data type is analyzed separately with its own tailored algorithms that yield results in different formats. Tools that facilitate format conversions and the extraction of relevant results can help; examples of such workflow construction software packages include Anduril [<span class="CitationRef"><a href="#CR169" data-toggle="citation">169</a></span>], Galaxy [<span class="CitationRef"><a href="#CR170" data-toggle="citation">170</a></span>] and Chipster [<span class="CitationRef"><a href="#CR171" data-toggle="citation">171</a></span>]. Anduril was developed for building complex pipelines with large datasets that require automated parallelization. The strength of Galaxy and Chipster is their usability; visualization is a key component of their design. Simultaneous or integrative visualization of the data in a genome browser is extremely useful for both data exploration and interpretation of results. Browsers can display in tandem mappings from most next-generation sequencing technologies, while adding custom tracks such as gene annotation, nucleotide variation or ENCODE datasets. For proteomics integration, the PG Nexus pipeline [<span class="CitationRef"><a href="#CR172" data-toggle="citation">172</a></span>] converts mass spectrometry data to mappings that are co-visualized with RNA-seq alignments.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec27" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Outlook" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Outlook">Outlook</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Outlook" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par77" class="Para">RNA-seq has become the standard method for transcriptome analysis, but the technology and tools are continuing to evolve. It should be noted that the agreement between results obtained from different tools is still unsatisfactory and that results are affected by parameter settings, especially for genes that are expressed at low levels. The two major highlights in the current application of RNA-seq are the construction of transcriptomes from small amounts of starting materials and better transcript identification from longer reads. The state of the art in both of these areas is changing rapidly, but we will briefly outline what can be done now and what can be expected in the near future.</p><section id="Sec28" class="Section2 RenderAsSection2">
                <h3 class="Heading">Single-cell RNA-seq</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par78" class="Para">Single-cell RNA-seq (scRNA-seq) is one of the newest and most active fields of RNA-seq with its unique set of opportunities and challenges. Newer protocols such as Smart-seq [<span class="CitationRef"><a href="#CR173" data-toggle="citation">173</a></span>] and Smart-seq2 [<span class="CitationRef"><a href="#CR174" data-toggle="citation">174</a></span>] have enabled us to work from very small amounts of starting mRNA that, with proper amplification, can be obtained from just a single cell. The resulting single-cell libraries enable the identification of new, uncharacterized cell types in tissues. They also make it possible to measure a fascinating phenomenon in molecular biology, the stochasticity of gene expression in otherwise identical cells within a defined population. In this context, single cell studies are meaningful only when a set of individual cell libraries are compared with the cell population, with the aim of identifying subgroups of multiple cells with distinct combinations of expressed genes. Differences may be due to naturally occurring factors such as stage of the cell cycle, or may reflect rare cell types such as cancer stem cells. Recent rapid progress in methodologies for single-cell preparation, including the availability of single-cell platforms such as the Fluidigm C1 [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>], has increased the number of individual cells analyzed from a handful to 50–90 per condition up to 800 cells at a time. Other methods, such as DROP-seq [<span class="CitationRef"><a href="#CR175" data-toggle="citation">175</a></span>], can profile more than 10,000 cells at a time. This increased number of single-cell libraries in each experiment directly allows for the identification of smaller subgroups within the population.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par79" class="Para">The small amount of starting material and the PCR amplification limit the depth to which single-cell libraries can be sequenced productively, often to less than a million reads. Deeper sequencing for scRNA-seq will do little to improve quantification as the number of individual mRNA molecules in a cell is small (in the order of 100–300,000 transcripts) and only a fraction of them are successfully reverse-transcribed to cDNA [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>, <span class="CitationRef"><a href="#CR176" data-toggle="citation">176</a></span>]; but deeper sequencing is potentially useful for discovering and measuring allele-specific expression, as additional reads could provide useful evidence.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par80" class="Para">Single-cell transcriptomes typically include about 3000–8000 expressed genes, which is far fewer than are counted in the transcriptomes of the corresponding pooled populations. The challenge is to distinguish the technical noise that results from a lack of sensitivity at the single-molecule level [<span class="CitationRef"><a href="#CR173" data-toggle="citation">173</a></span>] (where capture rates of around 10–50&nbsp;% result in the frequent loss of the most lowly expressed transcripts) from true biological noise where a transcript might not be transcribed and present in the cell for a certain amount of time while the protein is still present. The inclusion of added reference transcripts and the use of unique molecule identifiers (UMIs) have been applied to overcome amplification bias and to improve gene quantification [<span class="CitationRef"><a href="#CR177" data-toggle="citation">177</a></span>, <span class="CitationRef"><a href="#CR178" data-toggle="citation">178</a></span>]. Methods that can quantify gene-level technical variation allow us to focus on biological variation that is likely to be of interest [<span class="CitationRef"><a href="#CR179" data-toggle="citation">179</a></span>]. Typical quality-control steps involve setting aside libraries that contain few reads, libraries that have a low mapping rate, and libraries that have zero expression levels for housekeeping genes, such as <em xmlns="" class="EmphasisTypeItalic">GAPDH</em> and <em xmlns="" class="EmphasisTypeItalic">ACTB</em>, that are expected to be expressed at a detectable level.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par81" class="Para">Depending on the chosen single-cell protocol and the aims of the experiment, different bulk RNA-seq pipelines and tools can be used for different stages of the analysis as reviewed by Stegle et al. [<span class="CitationRef"><a href="#CR180" data-toggle="citation">180</a></span>]. Single-cell libraries are typically analyzed by mapping to a reference transcriptome (using a program such as RSEM) without any attempt at new transcript discovery, although at least one package maps to the genome (Monocle [<span class="CitationRef"><a href="#CR181" data-toggle="citation">181</a></span>]). While mapping onto the genome does result in a higher overall read-mapping rate, studies that are focused on gene expression alone with fewer reads per cell tend to use mapping to the reference transcriptome for the sake of simplicity. Other single-cell methods have been developed to measure single-cell DNA methylation [<span class="CitationRef"><a href="#CR182" data-toggle="citation">182</a></span>] and single-cell open chromatin using ATAC-seq [<span class="CitationRef"><a href="#CR183" data-toggle="citation">183</a></span>, <span class="CitationRef"><a href="#CR184" data-toggle="citation">184</a></span>]. At present, we can measure only one functional genomic data-type at a time in the same single cell, but we can expect that in the near future we will be able to recover the transcriptome of a single cell simultaneously with additional functional data.</p>
              </section><section id="Sec29" class="Section2 RenderAsSection2">
                <h3 class="Heading">Long-read sequencing</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par82" class="Para">The major limitation of short-read RNA-seq is the difficulty in accurately reconstructing expressed full-length transcripts from the assembly of reads. This is particularly complicated in complex transcriptomes, where different but highly similar isoforms of the same gene are expressed, and for genes that have many exons and possible alternative promoters or 3’ ends. Long-read technologies, such as Pacific-Biosciences (PacBio) SMRT and Oxford Nanopore, that were initially applied to genome sequencing are now being used for transcriptomics and have the potential to overcome this assembly problem. Long-read sequencing provides amplification-free, single-molecule sequencing of cDNAs that enables recovery of full-length transcripts without the need for an assembly step. PacBio adds adapters to the cDNA molecule and creates a circularized structure that can be sequenced with multiple passes within one single long read. The Nanopore GridION system can directly sequence RNA strands by using RNA processive enzymes and RNA-specific bases. Another interesting technology was previously known as Moleculo (now Illumina’s TruSeq synthetic long-read technology), where Illumina library preparation is multiplexed and restricted to a limited number of long DNA molecules that are separately bar-coded and pooled back for sequencing. As one barcode corresponds to a limited number of molecules, assembly is greatly simplified and unambiguous reconstruction to long contigs is possible. This approach has recently been published for RNA-seq analysis [<span class="CitationRef"><a href="#CR185" data-toggle="citation">185</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par83" class="Para">PacBio RNA-seq is the long-read approach with the most publications to date. The technology has proven useful for unraveling isoform diversity at complex loci [<span class="CitationRef"><a href="#CR186" data-toggle="citation">186</a></span>], and for determining allele-specific expression from single reads [<span class="CitationRef"><a href="#CR187" data-toggle="citation">187</a></span>]. Nevertheless, long-read sequencing has its own set of limitations, such as a still high error rate that limits de novo transcript identifications and forces the technology to leverage the reference genome [<span class="CitationRef"><a href="#CR188" data-toggle="citation">188</a></span>]. Moreover, the relatively low throughput of SMRT cells hampers the quantification of transcript expression. These two limitations can be addressed by matching PacBio experiments with regular, short-read RNA-seq. The accurate and abundant Illumina reads can be used both to correct long-read sequencing errors and to quantify transcript levels [<span class="CitationRef"><a href="#CR189" data-toggle="citation">189</a></span>]. Updates in PacBio chemistry are increasing sequencing lengths to produce reads with a sufficient number of passes over the cDNA molecule to autocorrect sequencing errors. This will eventually improve sequencing accuracy and allow for genome-free determination of isoform-resolved transcriptomes.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec30" class="Section1 RenderAsSection1 SectionTypeBox c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#BoxNumberofreplicates" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="BoxNumberofreplicates">Box 1. Number of replicates</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="BoxNumberofreplicates" data-test="collapse-section"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par84" class="Para">Three factors determine the number of replicates required in a RNA-seq experiment. The first factor is the variability in the measurements, which is influenced by the technical noise and the biological variation. While reproducibility in RNA-seq is usually high at the level of sequencing [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>, <span class="CitationRef"><a href="#CR45" data-toggle="citation">45</a></span>], other steps such as RNA extraction and library preparation are noisier and may introduce biases in the data that can be minimized by adopting good experimental procedures (Box 2). Biological variability is particular to each experimental system and is harder to control [<span class="CitationRef"><a href="#CR189" data-toggle="citation">189</a></span>]. Nevertheless, biological replication is required if inference on the population is to be made, with three replicates being the minimum for any inferential analysis. For a proper statistical power analysis, estimates of the within-group variance and gene expression levels are required. This information is typically not available beforehand but can be obtained from similar experiments. The exact power will depend on the method used for differential expression analysis, and software packages exist that provide a theoretical estimate of power over a range of variables, given the within-group variance of the samples, which is intrinsic to the experiment [<span class="CitationRef"><a href="#CR190" data-toggle="citation">190</a></span>, <span class="CitationRef"><a href="#CR191" data-toggle="citation">191</a></span>]. Table&nbsp;<span class="InternalRef"><a href="#Tab1">1</a></span> shows an example of statistical power calculations over a range of fold-changes (or effect sizes) and number of replicates in a human blood RNA-seq sample sequenced at 30 million mapped reads. It should be noted that these estimates apply to the average gene expression level, but as dynamic ranges in RNA-seq data are large, the probability that highly expressed genes will be detected as differentially expressed is greater than that for low-count genes [<span class="CitationRef"><a href="#CR192" data-toggle="citation">192</a></span>]. For methods that return a false discovery rate (FDR), the proportion of genes that are highly expressed out of the total set of genes being tested will also influence the power of detection after multiple testing correction [<span class="CitationRef"><a href="#CR193" data-toggle="citation">193</a></span>]. Filtering out genes that are expressed at low levels prior to differential expression analysis reduces the severity of the correction and may improve the power of detection [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>]. Increasing sequencing depth also can improve statistical power for lowly expressed genes [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>, <span class="CitationRef"><a href="#CR194" data-toggle="citation">194</a></span>], and for any given sample there exists a level of sequencing at which power improvement is best achieved by increasing the number of replicates [<span class="CitationRef"><a href="#CR195" data-toggle="citation">195</a></span>]. Tools such as Scotty are available to calculate the best trade-off between sequencing depth and replicate number given some budgetary constraints [<span class="CitationRef"><a href="#CR191" data-toggle="citation">191</a></span>].<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                      <p xmlns="" class="SimplePara">Statistical power to detect differential expression varies with effect size, sequencing depth and number of replicates</p>
                    </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></col><col></col><col></col><col></col></colgroup><thead><tr class="header"><th>&nbsp;</th><th colspan="3"><p xmlns="" class="SimplePara">Replicates per group</p></th></tr><tr class="header"><th>&nbsp;</th><th><p xmlns="" class="SimplePara">3</p></th><th><p xmlns="" class="SimplePara">5</p></th><th><p xmlns="" class="SimplePara">10</p></th></tr></thead><tbody><tr class="noclass"><td colspan="4"><p xmlns="" class="SimplePara">Effect size (fold change)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> 1.25</p></td><td><p xmlns="" class="SimplePara">17&nbsp;%</p></td><td><p xmlns="" class="SimplePara">25&nbsp;%</p></td><td><p xmlns="" class="SimplePara">44&nbsp;%</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> 1.5</p></td><td><p xmlns="" class="SimplePara">43&nbsp;%</p></td><td><p xmlns="" class="SimplePara">64&nbsp;%</p></td><td><p xmlns="" class="SimplePara">91&nbsp;%</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> 2</p></td><td><p xmlns="" class="SimplePara">87&nbsp;%</p></td><td><p xmlns="" class="SimplePara">98&nbsp;%</p></td><td><p xmlns="" class="SimplePara">100&nbsp;%</p></td></tr><tr class="noclass"><td colspan="4"><p xmlns="" class="SimplePara">Sequencing depth (millions of reads)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> 3</p></td><td><p xmlns="" class="SimplePara">19&nbsp;%</p></td><td><p xmlns="" class="SimplePara">29&nbsp;%</p></td><td><p xmlns="" class="SimplePara">52&nbsp;%</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> 10</p></td><td><p xmlns="" class="SimplePara">33&nbsp;%</p></td><td><p xmlns="" class="SimplePara">51&nbsp;%</p></td><td><p xmlns="" class="SimplePara">80&nbsp;%</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"> 15</p></td><td><p xmlns="" class="SimplePara">38&nbsp;%</p></td><td><p xmlns="" class="SimplePara">57&nbsp;%</p></td><td><p xmlns="" class="SimplePara">85&nbsp;%</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                    <p xmlns="" class="SimplePara">Example of calculations for the probability of detecting differential expression in a single test at a significance level of 5&nbsp;%, for a two-group comparison using a Negative Binomial model, as computed by the RNASeqPower package of Hart et al. [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR190" data-toggle="citation">190</a></span>]. For a fixed within-group variance (package default value), the statistical power increases with the difference between the two groups (effect size), the sequencing depth, and the number of replicates per group. This table shows the statistical power for a gene with 70 aligned reads, which was the median coverage for a protein-coding gene for one whole-blood RNA-seq sample with 30 million aligned reads from the GTEx Project [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR214" data-toggle="citation">214</a></span>]</p>
                  </div></figure>
                     </div></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec31" class="Section1 RenderAsSection1 SectionTypeBox c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#BoxExperimentexecutionchoices" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="BoxExperimentexecutionchoices">Box 2. Experiment execution choices</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="BoxExperimentexecutionchoices" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par85" class="Para">RNA-seq library preparation and sequencing procedures include a number of steps (RNA fragmentation, cDNA synthesis, adapter ligation, PCR amplification, bar-coding, and lane loading) that might introduce biases into the resulting data [<span class="CitationRef"><a href="#CR196" data-toggle="citation">196</a></span>]. Including exogenous reference transcripts (‘spike-ins’) is useful both for quality control [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>, <span class="CitationRef"><a href="#CR197" data-toggle="citation">197</a></span>] and for library-size normalization [<span class="CitationRef"><a href="#CR198" data-toggle="citation">198</a></span>]. For bias minimization, we recommend following the suggestions made by Van Dijk et al. [<span class="CitationRef"><a href="#CR199" data-toggle="citation">199</a></span>], such as the use of adapters with random nucleotides at the extremities or the use of chemical-based fragmentation instead of RNase III-based fragmentation. If the RNA-seq experiment is large and samples have to be processed in different batches and/or Illumina runs, caution should be taken to randomize samples across library preparation batches and lanes so as to avoid technical factors becoming confounded with experimental factors. Another option, when samples are individually barcoded and multiple Illumina lanes are needed to achieve the desired sequencing depth, is to include all samples in each lane, which would minimize any possible lane effect.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec32" class="Section1 RenderAsSection1 SectionTypeBox c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#BoxMappingtoareference" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="BoxMappingtoareference">Box 3. Mapping to a reference</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="BoxMappingtoareference" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par86" class="Para">Mapping to a reference genome allows for the identification of novel genes or transcripts, and requires the use of a gapped or spliced mapper as reads may span splice junctions. The challenge is to identify splice junctions correctly, especially when sequencing errors or differences with the reference exist or when non-canonical junctions and fusion transcripts are sought. One of the most popular RNA-seq mappers, TopHat, follows a two-step strategy in which unspliced reads are first mapped to locate exons, then unmapped reads are split and aligned independently to identify exon junctions [<span class="CitationRef"><a href="#CR200" data-toggle="citation">200</a></span>, <span class="CitationRef"><a href="#CR201" data-toggle="citation">201</a></span>]. Several other mappers exist that are optimized to identify SNPs or indels (GSNAP [<span class="CitationRef"><a href="#CR202" data-toggle="citation">202</a></span>], PALMapper [<span class="CitationRef"><a href="#CR203" data-toggle="citation">203</a></span>] MapSplice [<span class="CitationRef"><a href="#CR204" data-toggle="citation">204</a></span>]), detect non-canonical splice junctions (STAR [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>], MapSplice [<span class="CitationRef"><a href="#CR204" data-toggle="citation">204</a></span>]), achieve ultra-fast mapping (GEM [<span class="CitationRef"><a href="#CR205" data-toggle="citation">205</a></span>]) or map long-reads (STAR [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>]). Important parameters to consider during mapping are the strandedness of the RNA-seq library, the number of mismatches to accept, the length and type of reads (SE or PE), and the length of sequenced fragments. In addition, existing gene models can be leveraged by supplying an annotation file to some read mapper in order to map exon coordinates accurately and to help in identifying splicing events. The choice of gene model can also have a strong impact on the quantification and differential expression analysis [<span class="CitationRef"><a href="#CR206" data-toggle="citation">206</a></span>]. We refer the reader to [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>] for a comprehensive comparison of RNA-seq mappers. If the transcriptome annotation is comprehensive (for example, in mouse or human), researchers may choose to map directly to a Fasta-format file of all transcript sequences for all genes of interests. In this case, no gapped alignment is needed and unspliced mappers such as Bowtie [<span class="CitationRef"><a href="#CR207" data-toggle="citation">207</a></span>] can be used (Fig.&nbsp;<span class="InternalRef"><a href="#Fig2">2b</a></span>). Mapping to the transcriptome is generally faster but does not allow de novo transcript discovery.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec33" class="Section1 RenderAsSection1 SectionTypeBox c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#BoxComparisonofsoftwaretoolsfordetectingdifferentialgeneandtranscriptexpression" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="BoxComparisonofsoftwaretoolsfordetectingdifferentialgeneandtranscriptexpression">Box 4. Comparison of software tools for detecting differential gene and transcript expression</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="BoxComparisonofsoftwaretoolsfordetectingdifferentialgeneandtranscriptexpression" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par87" class="Para">Many statistical methods are available for detecting differential gene or transcript expression from RNA-seq data, and a major practical challenge is how to choose the most suitable tool for a particular data analysis job. Most comparison studies have focused on simulated datasets [<span class="CitationRef"><a href="#CR56" data-toggle="citation">56</a></span>, <span class="CitationRef"><a href="#CR208" data-toggle="citation">208</a></span>, <span class="CitationRef"><a href="#CR209" data-toggle="citation">209</a></span>] or on samples to which exogenous RNA (‘spike-in’) has been added in known quantities [<span class="CitationRef"><a href="#CR63" data-toggle="citation">63</a></span>, <span class="CitationRef"><a href="#CR196" data-toggle="citation">196</a></span>]. This enables a direct assessment of the sensitivity and specificity of the methods as well as their FDR control. As simulations typically rely on specific statistical distributions or on limited experimental datasets and as spike-in datasets represent only technical replicates with minimal variation, comparisons using simulated datasets have been complemented with more practical comparisons in real datasets with true biological replicates [<span class="CitationRef"><a href="#CR64" data-toggle="citation">64</a></span>, <span class="CitationRef"><a href="#CR210" data-toggle="citation">210</a></span>, <span class="CitationRef"><a href="#CR211" data-toggle="citation">211</a></span>].</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par88" class="Para">As yet, no clear consensus has been reached regarding the best practices and the field is continuing to evolve rapidly. However, some common findings have been made in multiple comparison studies and in different study settings. First, specific caution is needed with all the methods when the number of replicate samples is very small or for genes that are expressed at very low levels [<span class="CitationRef"><a href="#CR55" data-toggle="citation">55</a></span>, <span class="CitationRef"><a href="#CR64" data-toggle="citation">64</a></span>, <span class="CitationRef"><a href="#CR209" data-toggle="citation">209</a></span>]. Among the tools, limma has been shown to perform well under many circumstances and it is also the fastest to run [<span class="CitationRef"><a href="#CR56" data-toggle="citation">56</a></span>, <span class="CitationRef"><a href="#CR63" data-toggle="citation">63</a></span>, <span class="CitationRef"><a href="#CR64" data-toggle="citation">64</a></span>]. DESeq and edgeR perform similarly in ranking genes but are often relatively conservative or too liberal, respectively, in controlling FDR [<span class="CitationRef"><a href="#CR63" data-toggle="citation">63</a></span>, <span class="CitationRef"><a href="#CR209" data-toggle="citation">209</a></span>, <span class="CitationRef"><a href="#CR210" data-toggle="citation">210</a></span>]. SAMseq performs well in terms of FDR but presents an acceptable sensitivity when the number of replicates is relatively high, at least 10 [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>, <span class="CitationRef"><a href="#CR55" data-toggle="citation">55</a></span>, <span class="CitationRef"><a href="#CR209" data-toggle="citation">209</a></span>]. NOISeq and NOISeqBIO (the adaptation of NOISeq for biological replication) are more efficient in avoiding false positive calls at the cost of some sensitivity but perform well with different numbers of replicates [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>, <span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>, <span class="CitationRef"><a href="#CR212" data-toggle="citation">212</a></span>]. Cuffdiff and Cuffdiff2 have performed surprisingly poorly in the comparisons [<span class="CitationRef"><a href="#CR56" data-toggle="citation">56</a></span>, <span class="CitationRef"><a href="#CR63" data-toggle="citation">63</a></span>]. This probably reflects the fact that detecting differential expression at the transcript level remains challenging and involves uncertainties in assigning the reads to alternative isoforms. In a recent comparison, BitSeq compared favorably to other transcript-level packages such as Cuffdiff2 [<span class="CitationRef"><a href="#CR196" data-toggle="citation">196</a></span>]. Besides the actual performance, other issues affecting the choice of the tool include ease of installation and use, computational requirements, and quality of documentation and instructions. Finally, an important consideration when choosing an analysis method is the experimental design. While some of the differential expression tools can only perform a pair-wise comparison, others such as edgeR [<span class="CitationRef"><a href="#CR57" data-toggle="citation">57</a></span>], limma-voom [<span class="CitationRef"><a href="#CR55" data-toggle="citation">55</a></span>], DESeq [<span class="CitationRef"><a href="#CR48" data-toggle="citation">48</a></span>], DESeq2 [<span class="CitationRef"><a href="#CR58" data-toggle="citation">58</a></span>], and maSigPro [<span class="CitationRef"><a href="#CR213" data-toggle="citation">213</a></span>] can perform multiple comparisons, include different covariates or analyze time-series data.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section ArticleNotes" id="ArticleNotes"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Notes" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Notes">Notes</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Notes"><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteCrossLinking"><p xmlns="" class="SimplePara">An erratum to this article can be found at <span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://dx.doi.org/10.1186/s13059-016-1047-4" target="_blank" class="ext-link"><span class="RefSource">http://dx.doi.org/10.1186/s13059-016-1047-4</span></a></span>.</p></aside></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Abbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Abbreviations">Abbreviations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Abbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>ASM:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par2" class="Para">Alternative splicing module</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>ChIP-seq:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par3" class="Para">Chromatin immunoprecipitation sequencing</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>DEG:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par4" class="Para">Differentially expressed genes</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>eQTL:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par5" class="Para">Expression quantitative loci</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>FDR:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par6" class="Para">False discovery rate</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>FPKM:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par7" class="Para">Fragments per kilobase of exon model per million mapped reads</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>GO:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par8" class="Para">Gene Ontology</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>GSEA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par9" class="Para">Gene set enrichment analysis</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>GTF:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par10" class="Para">Gene transfer format</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>IG:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par11" class="Para">Immunoglobulin</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>IGV:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par12" class="Para">Integrative Genomics Viewer</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>miRNA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par13" class="Para">MicroRNA</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>mRNA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par14" class="Para">Messenger RNA</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>PCA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par15" class="Para">Principal component analysis</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>PE read:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par16" class="Para">Paired-end read</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>RNA-seq:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par17" class="Para">RNA-sequencing</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>RPKM:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par18" class="Para">Reads per kilobase of exon model per million reads</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>rRNA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par19" class="Para">Ribosomal RNA</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>RSEM:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par20" class="Para">RNA-Seq by Expectation Maximization</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>scRNA-seq:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par21" class="Para">Single-cell RNA-seq</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>SE read:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par22" class="Para">Single-end read</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>siRNA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par23" class="Para">Short-interfering RNA</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>SNP:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par24" class="Para">Single nucleotide polymorphism</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>sQTL:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par25" class="Para">Splicing quantitative trait loci</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>sRNA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par26" class="Para">Small RNA</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>TF:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par27" class="Para">Transcription factor</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>TPM:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par28" class="Para">Transcripts per million</p>
                  </dd></dl>
              </div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgements</h3><p xmlns="" class="SimplePara">The authors would like to thank Michael Love and Harold Pimentel for helpful suggestions on the initial draft of the manuscript. AC, ST, AM, DGC were supported by the FP7 STATegra project (grant 36000). Research in AC’s laboratory was supported by MINECO grant BIO2012-40244 and co-funded with European Regional Development Funds (ERDF). Research in PM’s laboratory is supported by ERC starting grant Relieve-IMDs and by a core support grant from the Wellcome Trust and MRC to the Wellcome Trust-Medical Research Council Cambridge Stem Cell Institute. XZ was supported by the National Basic Research Program of China (2012CB316504). LLE was supported by JDRF (grant number 2-2013-32) and by the Sigrid Juselius Foundation. ACe was supported by the Academy of Finland (Center of Excellence in Cancer Genetics Research).</p></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="License">
                <p xmlns="" class="SimplePara">
                           <strong class="EmphasisTypeBold">Open Access</strong>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/4.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/licenses/by/4.0/</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/publicdomain/zero/1.0/" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/publicdomain/zero/1.0/</span></a></span>) applies to the data made available in this article, unless otherwise stated.</p>
              </div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Additional files</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" id="Par89" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a href="https://static-content.springer.com/esm/art%3A10.1186%2Fs13059-016-0881-8/MediaObjects/13059_2016_881_MOESM1_ESM.pdf"><span class="caption-number">Additional file 1: Figure S1.</span></a>
                          <span xmlns="" class="SimplePara">Screenshots of RNA-seq data visualization. <strong class="EmphasisTypeBold">a</strong> Integrative Genomics Viewer (IGV) [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR77" data-toggle="citation">77</a></span>] display of a gene detected as differentially expressed between the two groups of samples by DEGseq [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR62" data-toggle="citation">62</a></span>]. The bottom track in the right panel is the gene annotation. The tracks are five samples from each group. <strong class="EmphasisTypeBold">b</strong> RNAseqViewer [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR80" data-toggle="citation">80</a></span>] display of the same data as in (<strong class="EmphasisTypeBold">a</strong>). <strong class="EmphasisTypeBold">c</strong> RNAseqViewer heatmap display of a gene detected as differentially spliced between two groups by both DSGSeq [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR70" data-toggle="citation">70</a></span>] and DEXSeq [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR69" data-toggle="citation">69</a></span>]. Introns are hidden in the display to emphasize the signals on the exons. <strong class="EmphasisTypeBold">d</strong> MISO [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR81" data-toggle="citation">81</a></span>] display of another gene detected as differentially spliced, with junction reads illustrated. (PDF 1152 kb)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Competing interests</h3><p xmlns="" class="SimplePara">The authors declare that they have no competing interests.</p></aside><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Authors’ contributions</h3><p xmlns="" class="SimplePara">ACo, PM and AM conceived the idea and shaped the structure of the manuscript. ACo drafted the experimental design, alignment and functional profiling sections and integrated contributions from all authors. PM drafted the visualization and de novo transcript reconstruction sections, and coordinated author contributions. ST drafted the quality-control and differential expression sections. DGC drafted the experimental design and integration sections. ACe contributed to drafting the integration section. AMP drafted the transcript fusion section. MWS drafted the small RNA section. DG drafted the eQTL section. LLE drafted the software comparison for differential expression section. LLE and XZ drafted the transcript isoform analysis sections. XZ contributed to drafting the visualization section. AM drafted the introduction and outlook sections and globally edited the manuscript. All authors read and approved the final manuscript.</p></aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">Institute for Food and Agricultural Sciences, Department of Microbiology and Cell Science, University of Florida, Gainesville, FL&nbsp;32603, USA</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">Centro de Investigación Príncipe Felipe, Genomics of Gene Expression Laboratory, 46012&nbsp;Valencia, Spain</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3"><span class="AffiliationNumber">(3)</span><div class="AffiliationText">Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SA, UK</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff4"><span class="AffiliationNumber">(4)</span><div class="AffiliationText">Wellcome Trust-Medical Research Council Cambridge Stem Cell Institute, Anne McLaren Laboratory for Regenerative Medicine, Department of Surgery, University of Cambridge, Cambridge, CB2 0SZ, UK</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff5"><span class="AffiliationNumber">(5)</span><div class="AffiliationText">Department of Applied Statistics, Operations Research and Quality, Universidad Politécnica de Valencia, 46020&nbsp;Valencia, Spain</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff6"><span class="AffiliationNumber">(6)</span><div class="AffiliationText">Unit of Computational Medicine, Department of Medicine, Karolinska Institutet, Karolinska University Hospital, 171 77&nbsp;Stockholm, Sweden</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff7"><span class="AffiliationNumber">(7)</span><div class="AffiliationText">Center for Molecular Medicine, Karolinska Institutet, 17177&nbsp;Stockholm, Sweden</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff8"><span class="AffiliationNumber">(8)</span><div class="AffiliationText">Unit of Clinical Epidemiology, Department of Medicine, Karolinska University Hospital, L8, 17176&nbsp;Stockholm, Sweden</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff9"><span class="AffiliationNumber">(9)</span><div class="AffiliationText">Science for Life Laboratory, 17121&nbsp;Solna, Sweden</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff10"><span class="AffiliationNumber">(10)</span><div class="AffiliationText">Systems Biology Laboratory, Institute of Biomedicine and Genome-Scale Biology Research Program, University of Helsinki, 00014&nbsp;Helsinki, Finland</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff11"><span class="AffiliationNumber">(11)</span><div class="AffiliationText">School of Computing Science, Simon Fraser University, Burnaby, V5A 1S6, BC, Canada</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff12"><span class="AffiliationNumber">(12)</span><div class="AffiliationText">Department of Bioinformatics, Institute of Molecular Biology and Biotechnology, Adam Mickiewicz University in Poznań, 61-614&nbsp;Poznań, Poland</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff13"><span class="AffiliationNumber">(13)</span><div class="AffiliationText">Turku Centre for Biotechnology, University of Turku and Åbo Akademi University, FI-20520&nbsp;Turku, Finland</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff14"><span class="AffiliationNumber">(14)</span><div class="AffiliationText">Key Lab of Bioinformatics/Bioinformatics Division, TNLIST and Department of Automation, Tsinghua University, Beijing, 100084, China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff15"><span class="AffiliationNumber">(15)</span><div class="AffiliationText">School of Life Sciences, Tsinghua University, Beijing, 100084, China</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff16"><span class="AffiliationNumber">(16)</span><div class="AffiliationText">Department of Developmental and Cell Biology, University of California, Irvine, Irvine, CA&nbsp;92697-2300, USA</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff17"><span class="AffiliationNumber">(17)</span><div class="AffiliationText">Center for Complex Biological Systems, University of California, Irvine, Irvine, CA&nbsp;92697, USA</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mortazavi A, Williams BA, McCue K, Schaeffer L, Wold B. Mapping and quantifying mammalian transcriptomes by RNA-Seq. Nat Methods. 2008;5:1–8.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nmeth.1226"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mapping%20and%20quantifying%20mammalian%20transcriptomes%20by%20RNA-Seq&amp;author=A.%20Mortazavi&amp;author=BA.%20Williams&amp;author=K.%20McCue&amp;author=L.%20Schaeffer&amp;author=B.%20Wold&amp;journal=Nat%20Methods&amp;volume=5&amp;pages=1-8&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Levin JZ, Yassour M, Adiconis X, Nusbaum C, Thompson DA, Friedman N, et al. Comprehensive comparative analysis of strand-specific RNA sequencing methods. Nat Methods. 2010;7:709–15.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20711195"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3005310"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nmeth.1491"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comprehensive%20comparative%20analysis%20of%20strand-specific%20RNA%20sequencing%20methods&amp;author=JZ.%20Levin&amp;author=M.%20Yassour&amp;author=X.%20Adiconis&amp;author=C.%20Nusbaum&amp;author=DA.%20Thompson&amp;author=N.%20Friedman&amp;journal=Nat%20Methods&amp;volume=7&amp;pages=709-15&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Parkhomchuk D, Borodina T, Amstislavskiy V, Banaru M, Hallen L, Krobitsch S, et al. Transcriptome analysis by strand-specific sequencing of complementary DNA. Nucleic Acids Res. 2009;37:e123.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19620212"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2764448"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkp596"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Transcriptome%20analysis%20by%20strand-specific%20sequencing%20of%20complementary%20DNA&amp;author=D.%20Parkhomchuk&amp;author=T.%20Borodina&amp;author=V.%20Amstislavskiy&amp;author=M.%20Banaru&amp;author=L.%20Hallen&amp;author=S.%20Krobitsch&amp;journal=Nucleic%20Acids%20Res&amp;volume=37&amp;pages=e123&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Katz Y, Wang ET, Airoldi EM, Burge CB. Analysis and design of RNA sequencing experiments for identifying isoform regulation. Nat Methods. 2010;7:1009–15.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21057496"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3037023"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nmeth.1528"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Analysis%20and%20design%20of%20RNA%20sequencing%20experiments%20for%20identifying%20isoform%20regulation&amp;author=Y.%20Katz&amp;author=ET.%20Wang&amp;author=EM.%20Airoldi&amp;author=CB.%20Burge&amp;journal=Nat%20Methods&amp;volume=7&amp;pages=1009-15&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Garber M, Grabherr MG, Guttman M, Trapnell C. Computational methods for transcriptome annotation and quantification using RNA-seq. Nat Methods. 2011;8:469–77.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21623353"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nmeth.1613"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Computational%20methods%20for%20transcriptome%20annotation%20and%20quantification%20using%20RNA-seq&amp;author=M.%20Garber&amp;author=MG.%20Grabherr&amp;author=M.%20Guttman&amp;author=C.%20Trapnell&amp;journal=Nat%20Methods&amp;volume=8&amp;pages=469-77&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Łabaj PP, Leparc GG, Linggi BE, Markillie LM, Wiley HS, Kreil DP. Characterization and improvement of RNA-Seq precision in quantitative transcript expression profiling. Bioinformatics. 2011;27:i383–91.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3117338"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btr247"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21685096"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Characterization%20and%20improvement%20of%20RNA-Seq%20precision%20in%20quantitative%20transcript%20expression%20profiling&amp;author=PP.%20%C5%81abaj&amp;author=GG.%20Leparc&amp;author=BE.%20Linggi&amp;author=LM.%20Markillie&amp;author=HS.%20Wiley&amp;author=DP.%20Kreil&amp;journal=Bioinformatics&amp;volume=27&amp;pages=i383-91&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sims D, Sudbery I, Ilott NE, Heger A, Ponting CP. Sequencing depth and coverage: key considerations in genomic analyses. Nat Rev Genet. 2014;15:121–32.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24434847"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nrg3642"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Sequencing%20depth%20and%20coverage%3A%20key%20considerations%20in%20genomic%20analyses&amp;author=D.%20Sims&amp;author=I.%20Sudbery&amp;author=NE.%20Ilott&amp;author=A.%20Heger&amp;author=CP.%20Ponting&amp;journal=Nat%20Rev%20Genet&amp;volume=15&amp;pages=121-32&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pollen AA, Nowakowski TJ, Shuga J, Wang X, Leyrat AA, Lui JH, et al. Low-coverage single-cell mRNA sequencing reveals cellular heterogeneity and activated signaling pathways in developing cerebral cortex. Nat Biotechnol. 2014;32:1053–8.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25086649"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4191988"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.2967"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Low-coverage%20single-cell%20mRNA%20sequencing%20reveals%20cellular%20heterogeneity%20and%20activated%20signaling%20pathways%20in%20developing%20cerebral%20cortex&amp;author=AA.%20Pollen&amp;author=TJ.%20Nowakowski&amp;author=J.%20Shuga&amp;author=X.%20Wang&amp;author=AA.%20Leyrat&amp;author=JH.%20Lui&amp;journal=Nat%20Biotechnol&amp;volume=32&amp;pages=1053-8&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jaitin DA, Kenigsberg E, Keren-Shaul H, Elefant N, Paul F, Zaretsky I, et al. Massively parallel single-cell RNA-seq for marker-free decomposition of tissues into cell types. Science. 2014;343:776–9.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24531970"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4412462"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1126/science.1247651"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Massively%20parallel%20single-cell%20RNA-seq%20for%20marker-free%20decomposition%20of%20tissues%20into%20cell%20types&amp;author=DA.%20Jaitin&amp;author=E.%20Kenigsberg&amp;author=H.%20Keren-Shaul&amp;author=N.%20Elefant&amp;author=F.%20Paul&amp;author=I.%20Zaretsky&amp;journal=Science&amp;volume=343&amp;pages=776-9&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tarazona S, Garcia-Alcalde F, Dopazo J, Ferrer A, Conesa A. Differential expression in RNA-seq: a matter of depth. Genome Res. 2011;21:2213–23.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21903743"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3227109"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.124321.111"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Differential%20expression%20in%20RNA-seq%3A%20a%20matter%20of%20depth&amp;author=S.%20Tarazona&amp;author=F.%20Garcia-Alcalde&amp;author=J.%20Dopazo&amp;author=A.%20Ferrer&amp;author=A.%20Conesa&amp;journal=Genome%20Res&amp;volume=21&amp;pages=2213-23&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Andrews S. FASTQC. A quality control tool for high throughput sequence data. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank" class="ext-link"><span class="RefSource">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</span></a></span>. Accessed 29 September 2014.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dai M, Thompson RC, Maher C, Contreras-Galindo R, Kaplan MH, Markovitz DM, et al. NGSQC: cross-platform quality analysis pipeline for deep sequencing data. BMC Genomics. 2010;11 Suppl 4:S7.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21143816"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3005923"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2164-11-S4-S7"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=NGSQC%3A%20cross-platform%20quality%20analysis%20pipeline%20for%20deep%20sequencing%20data&amp;author=M.%20Dai&amp;author=RC.%20Thompson&amp;author=C.%20Maher&amp;author=R.%20Contreras-Galindo&amp;author=MH.%20Kaplan&amp;author=DM.%20Markovitz&amp;journal=BMC%20Genomics&amp;volume=11&amp;issue=Suppl%204&amp;pages=S7&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">FASTX-Toolkit. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://hannonlab.cshl.edu/fastx_toolkit/" target="_blank" class="ext-link"><span class="RefSource">http://hannonlab.cshl.edu/fastx_toolkit/</span></a></span>. Accessed 12 January 2016.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bolger AM, Lohse M, Usadel B. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics. 2014;30:2114–20.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24695404"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4103590"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btu170"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;author=AM.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2114-20&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dobin A, Davis CA, Schlesinger F, Drenkow J, Zaleski C, Jha S, et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics. 2013;29:15–21.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23104886"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/bts635"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=STAR%3A%20ultrafast%20universal%20RNA-seq%20aligner&amp;author=A.%20Dobin&amp;author=CA.%20Davis&amp;author=F.%20Schlesinger&amp;author=J.%20Drenkow&amp;author=C.%20Zaleski&amp;author=S.%20Jha&amp;journal=Bioinformatics&amp;volume=29&amp;pages=15-21&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Picard. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://picard.sourceforge.net/" target="_blank" class="ext-link"><span class="RefSource">http://picard.sourceforge.net/</span></a></span>. Accessed 12 January 2016.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang L, Wang S, Li W. RSeQC: quality control of RNA-seq experiments. Bioinformatics. 2012;28:2184–5.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22743226"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/bts356"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=RSeQC%3A%20quality%20control%20of%20RNA-seq%20experiments&amp;author=L.%20Wang&amp;author=S.%20Wang&amp;author=W.%20Li&amp;journal=Bioinformatics&amp;volume=28&amp;pages=2184-5&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">García-Alcalde F, Okonechnikov K, Carbonell J, Cruz LM, Götz S, Tarazona S, et al. Qualimap: evaluating next-generation sequencing alignment data. Bioinformatics. 2012;28:2678–9.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22914218"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/bts503"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Qualimap%3A%20evaluating%20next-generation%20sequencing%20alignment%20data&amp;author=F.%20Garc%C3%ADa-Alcalde&amp;author=K.%20Okonechnikov&amp;author=J.%20Carbonell&amp;author=LM.%20Cruz&amp;author=S.%20G%C3%B6tz&amp;author=S.%20Tarazona&amp;journal=Bioinformatics&amp;volume=28&amp;pages=2678-9&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tarazona S, Furió-Tarí P, Turrà D, Pietro AD, Nueda MJ, Ferrer A, et al. Data quality aware analysis of differential expression in RNA-seq with NOISeq R/Bioc package. Nucleic Acids Res. 2015;43:e140.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26184878"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4666377"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Data%20quality%20aware%20analysis%20of%20differential%20expression%20in%20RNA-seq%20with%20NOISeq%20R%2FBioc%20package&amp;author=S.%20Tarazona&amp;author=P.%20Furi%C3%B3-Tar%C3%AD&amp;author=D.%20Turr%C3%A0&amp;author=AD.%20Pietro&amp;author=MJ.%20Nueda&amp;author=A.%20Ferrer&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=e140&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Risso D, Schwartz K, Sherlock G, Dudoit S. GC-content normalization for RNA-seq data. BMC Bioinformatics. 2011;12:480.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22177264"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3315510"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2105-12-480"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=GC-content%20normalization%20for%20RNA-seq%20data&amp;author=D.%20Risso&amp;author=K.%20Schwartz&amp;author=G.%20Sherlock&amp;author=S.%20Dudoit&amp;journal=BMC%20Bioinformatics&amp;volume=12&amp;pages=480&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Steijger T, Abril JF, Engström PG, Kokocinski F, Hubbard TJ, Guigó R, et al. Assessment of transcript reconstruction methods for RNA-seq. Nat Methods. 2013;10:1177–84.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24185837"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nmeth.2714"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Assessment%20of%20transcript%20reconstruction%20methods%20for%20RNA-seq&amp;author=T.%20Steijger&amp;author=JF.%20Abril&amp;author=PG.%20Engstr%C3%B6m&amp;author=F.%20Kokocinski&amp;author=TJ.%20Hubbard&amp;author=R.%20Guig%C3%B3&amp;journal=Nat%20Methods&amp;volume=10&amp;pages=1177-84&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Boley N, Stoiber MH, Booth BW, Wan KH, Hoskins RA, Bickel PJ, et al. Genome-guided transcript assembly by integrative analysis of RNA sequence data. Nat Biotechnol. 2014;32:341–6.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24633242"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4037530"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.2850"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome-guided%20transcript%20assembly%20by%20integrative%20analysis%20of%20RNA%20sequence%20data&amp;author=N.%20Boley&amp;author=MH.%20Stoiber&amp;author=BW.%20Booth&amp;author=KH.%20Wan&amp;author=RA.%20Hoskins&amp;author=PJ.%20Bickel&amp;journal=Nat%20Biotechnol&amp;volume=32&amp;pages=341-6&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Roberts A, Pimentel H, Trapnell C, Pachter L. Identification of novel transcripts in annotated genomes using RNA-Seq. Bioinformatics. 2011;27:2325–9.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21697122"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btr355"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20novel%20transcripts%20in%20annotated%20genomes%20using%20RNA-Seq&amp;author=A.%20Roberts&amp;author=H.%20Pimentel&amp;author=C.%20Trapnell&amp;author=L.%20Pachter&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2325-9&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Mezlini AM, Smith EJ, Fiume M, Buske O, Savich GL, Shah S, et al. iReckon: simultaneous isoform discovery and abundance estimation from RNA-seq data. Genome Res. 2013;23:519–29.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23204306"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3589540"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.142232.112"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=iReckon%3A%20simultaneous%20isoform%20discovery%20and%20abundance%20estimation%20from%20RNA-seq%20data&amp;author=AM.%20Mezlini&amp;author=EJ.%20Smith&amp;author=M.%20Fiume&amp;author=O.%20Buske&amp;author=GL.%20Savich&amp;author=S.%20Shah&amp;journal=Genome%20Res&amp;volume=23&amp;pages=519-29&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li JJ, Jiang CR, Brown JB, Huang H, Bickel PJ. Sparse linear modeling of next-generation mRNA sequencing (RNA-Seq) data for isoform discovery and abundance estimation. Proc Natl Acad Sci U S A. 2011;108:19867–72.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22135461"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3250192"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.1113972108"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Sparse%20linear%20modeling%20of%20next-generation%20mRNA%20sequencing%20%28RNA-Seq%29%20data%20for%20isoform%20discovery%20and%20abundance%20estimation&amp;author=JJ.%20Li&amp;author=CR.%20Jiang&amp;author=JB.%20Brown&amp;author=H.%20Huang&amp;author=PJ.%20Bickel&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=108&amp;pages=19867-72&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pertea M, Pertea GM, Antonescu CM, Chang TC, Mendell JT, Salzberg SL. StringTie enables improved reconstruction of a transcriptome from RNA-seq reads. Nat Biotechnol. 2015;33:290–5.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25690850"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4643835"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.3122"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=StringTie%20enables%20improved%20reconstruction%20of%20a%20transcriptome%20from%20RNA-seq%20reads&amp;author=M.%20Pertea&amp;author=GM.%20Pertea&amp;author=CM.%20Antonescu&amp;author=TC.%20Chang&amp;author=JT.%20Mendell&amp;author=SL.%20Salzberg&amp;journal=Nat%20Biotechnol&amp;volume=33&amp;pages=290-5&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hiller D, Wong WH. Simultaneous isoform discovery and quantification from RNA-Seq. Stat Biosci. 2013;5:100–18.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23888185"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s12561-012-9069-2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Simultaneous%20isoform%20discovery%20and%20quantification%20from%20RNA-Seq&amp;author=D.%20Hiller&amp;author=WH.%20Wong&amp;journal=Stat%20Biosci&amp;volume=5&amp;pages=100-18&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stanke M, Keller O, Gunduz I, Hayes A, Waack S, Morgenstern B. AUGUSTUS: <em class="EmphasisTypeItalic">ab initio</em> prediction of alternative transcripts. Nucleic Acids Res. 2006;34:W435–9.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16845043"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1538822"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkl200"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=AUGUSTUS%3A%20ab%20initio%20prediction%20of%20alternative%20transcripts&amp;author=M.%20Stanke&amp;author=O.%20Keller&amp;author=I.%20Gunduz&amp;author=A.%20Hayes&amp;author=S.%20Waack&amp;author=B.%20Morgenstern&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=W435-9&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Engström PG, Steijger T, Sipos B, Grant GR, Kahles A, Rätsch G, et al. Systematic evaluation of spliced alignment programs for RNA-seq data. Nat Methods. 2013;10:1185–91.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24185836"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4018468"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nmeth.2722"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Systematic%20evaluation%20of%20spliced%20alignment%20programs%20for%20RNA-seq%20data&amp;author=PG.%20Engstr%C3%B6m&amp;author=T.%20Steijger&amp;author=B.%20Sipos&amp;author=GR.%20Grant&amp;author=A.%20Kahles&amp;author=G.%20R%C3%A4tsch&amp;journal=Nat%20Methods&amp;volume=10&amp;pages=1185-91&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xie Y, Wu G, Tang J, Luo R, Patterson J, Liu S, et al. SOAPdenovo-Trans: de novo transcriptome assembly with short RNA-Seq reads. Bioinformatics. 2014;30:1660–6.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24532719"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btu077"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=SOAPdenovo-Trans%3A%20de%20novo%20transcriptome%20assembly%20with%20short%20RNA-Seq%20reads&amp;author=Y.%20Xie&amp;author=G.%20Wu&amp;author=J.%20Tang&amp;author=R.%20Luo&amp;author=J.%20Patterson&amp;author=S.%20Liu&amp;journal=Bioinformatics&amp;volume=30&amp;pages=1660-6&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schulz MH, Zerbino DR, Vingron M, Birney E. Oases: robust de novo RNA-seq assembly across the dynamic range of expression levels. Bioinformatics. 2012;28:1086–92.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22368243"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3324515"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/bts094"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Oases%3A%20robust%20de%20novo%20RNA-seq%20assembly%20across%20the%20dynamic%20range%20of%20expression%20levels&amp;author=MH.%20Schulz&amp;author=DR.%20Zerbino&amp;author=M.%20Vingron&amp;author=E.%20Birney&amp;journal=Bioinformatics&amp;volume=28&amp;pages=1086-92&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Grabherr MG, Haas BJ, Yassour M, Levin JZ, Thompson DA, Amit I, et al. Full-length transcriptome assembly from RNA-seq data without a reference genome. Nat Biotechnol. 2011;29:644–52.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21572440"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3571712"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.1883"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Full-length%20transcriptome%20assembly%20from%20RNA-seq%20data%20without%20a%20reference%20genome&amp;author=MG.%20Grabherr&amp;author=BJ.%20Haas&amp;author=M.%20Yassour&amp;author=JZ.%20Levin&amp;author=DA.%20Thompson&amp;author=I.%20Amit&amp;journal=Nat%20Biotechnol&amp;volume=29&amp;pages=644-52&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Haas BJ, Papanicolaou A, Yassour M, Grabherr M, Blood PD, Bowden J, et al. <em class="EmphasisTypeItalic">De novo</em>transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis. Nat Protoc. 2013;8:1494–512.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23845962"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nprot.2013.084"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20De%20novotranscript%20sequence%20reconstruction%20from%20RNA-seq%20using%20the%20Trinity%20platform%20for%20reference%20generation%20and%20analysis&amp;author=BJ.%20Haas&amp;author=A.%20Papanicolaou&amp;author=M.%20Yassour&amp;author=M.%20Grabherr&amp;author=PD.%20Blood&amp;author=J.%20Bowden&amp;journal=Nat%20Protoc&amp;volume=8&amp;pages=1494-512&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Patro R, Mount SM, Kingsford C. Sailfish enables alignment-free isoform quantification from RNA-seq reads using lightweight algorithms. Nat Biotechnol. 2014;32:462–4.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24752080"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4077321"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.2862"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Sailfish%20enables%20alignment-free%20isoform%20quantification%20from%20RNA-seq%20reads%20using%20lightweight%20algorithms&amp;author=R.%20Patro&amp;author=SM.%20Mount&amp;author=C.%20Kingsford&amp;journal=Nat%20Biotechnol&amp;volume=32&amp;pages=462-4&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Anders S, Pyl PT, Huber W. HTSeq - a Python framework to work with high-throughput sequencing data. Bioinformatics. 2015;31:166–9.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25260700"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btu638"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=HTSeq%20-%20a%20Python%20framework%20to%20work%20with%20high-throughput%20sequencing%20data&amp;author=S.%20Anders&amp;author=PT.%20Pyl&amp;author=W.%20Huber&amp;journal=Bioinformatics&amp;volume=31&amp;pages=166-9&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liao Y, Smyth GK, Shi W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Bioinformatics. 2014;30:923–30.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24227677"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btt656"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=featureCounts%3A%20an%20efficient%20general%20purpose%20program%20for%20assigning%20sequence%20reads%20to%20genomic%20features&amp;author=Y.%20Liao&amp;author=GK.%20Smyth&amp;author=W.%20Shi&amp;journal=Bioinformatics&amp;volume=30&amp;pages=923-30&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">UCSC Genome Bioinformatics: Frequently Asked Questions: Data File Formats. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://genome.ucsc.edu/FAQ/FAQformat.html#format4" target="_blank" class="ext-link"><span class="RefSource">https://genome.ucsc.edu/FAQ/FAQformat.html#format4</span></a></span>. Accessed on 12 January 2016.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pachter L. Models for transcript quantification from RNA-seq. arXiv.org. 2011. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://arxiv.org/abs/1104.3889" target="_blank" class="ext-link"><span class="RefSource">http://arxiv.org/abs/1104.3889</span></a></span>. Accessed 6 January 2016.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR39" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Trapnell C, Williams BA, Pertea G, Mortazavi A, Kwan G, van Baren MJ, et al. Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation. Nat Biotechnol. 2010;28:511–5.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20436464"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3146043"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.1621"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation&amp;author=C.%20Trapnell&amp;author=BA.%20Williams&amp;author=G.%20Pertea&amp;author=A.%20Mortazavi&amp;author=G.%20Kwan&amp;author=MJ.%20Baren&amp;journal=Nat%20Biotechnol&amp;volume=28&amp;pages=511-5&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR40" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li B, Dewey CN. RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome. BMC Bioinformatics. 2011;12:323.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21816040"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3163565"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2105-12-323"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=RSEM%3A%20accurate%20transcript%20quantification%20from%20RNA-Seq%20data%20with%20or%20without%20a%20reference%20genome&amp;author=B.%20Li&amp;author=CN.%20Dewey&amp;journal=BMC%20Bioinformatics&amp;volume=12&amp;pages=323&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR41" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Roberts A, Pachter L. Streaming fragment assignment for real-time analysis of sequencing experiments. Nat Methods. 2013;10:71–3.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23160280"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nmeth.2251"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Streaming%20fragment%20assignment%20for%20real-time%20analysis%20of%20sequencing%20experiments&amp;author=A.%20Roberts&amp;author=L.%20Pachter&amp;journal=Nat%20Methods&amp;volume=10&amp;pages=71-3&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR42" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bray N, Pimentel H, Melsted P, Pachter L. Near-optimal RNA-Seq quantification with kallisto. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://liorpachter.wordpress.com/2015/05/10/near-optimal-rna-seq-quantification-with-kallisto/" target="_blank" class="ext-link"><span class="RefSource">https://liorpachter.wordpress.com/2015/05/10/near-optimal-rna-seq-quantification-with-kallisto/</span></a></span>. Accessed 6 January 2016.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR43" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Roberts A, Trapnell C, Donaghey J, Rinn JL, Pachter L. Improving RNA-seq expression estimates by correcting for fragment bias. Genome Biol. 2011;12:R22.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21410973"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3129672"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/gb-2011-12-3-r22"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Improving%20RNA-seq%20expression%20estimates%20by%20correcting%20for%20fragment%20bias&amp;author=A.%20Roberts&amp;author=C.%20Trapnell&amp;author=J.%20Donaghey&amp;author=JL.%20Rinn&amp;author=L.%20Pachter&amp;journal=Genome%20Biol&amp;volume=12&amp;pages=R22&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR44" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ma X, Zhang X. NURD: an implementation of a new method to estimate isoform expression from non-uniform RNA-seq data. BMC Bioinformatics. 2013;14:220.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23837734"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3717102"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2105-14-220"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=NURD%3A%20an%20implementation%20of%20a%20new%20method%20to%20estimate%20isoform%20expression%20from%20non-uniform%20RNA-seq%20data&amp;author=X.%20Ma&amp;author=X.%20Zhang&amp;journal=BMC%20Bioinformatics&amp;volume=14&amp;pages=220&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR45" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bullard JH, Purdom E, Hansen KD, Dudoit S. Evaluation of statistical methods for normalization and differential expression in mRNA-Seq experiments. BMC Bioinformatics. 2010;11:94.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20167110"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2838869"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2105-11-94"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evaluation%20of%20statistical%20methods%20for%20normalization%20and%20differential%20expression%20in%20mRNA-Seq%20experiments&amp;author=JH.%20Bullard&amp;author=E.%20Purdom&amp;author=KD.%20Hansen&amp;author=S.%20Dudoit&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=94&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR46" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hansen K, Brenner S, Dudoit S. Biases in Illumina transcriptome sequencing caused by random hexamer priming. Nucleic Acids Res. 2010;38:e131.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20395217"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2896536"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkq224"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Biases%20in%20Illumina%20transcriptome%20sequencing%20caused%20by%20random%20hexamer%20priming&amp;author=K.%20Hansen&amp;author=S.%20Brenner&amp;author=S.%20Dudoit&amp;journal=Nucleic%20Acids%20Res&amp;volume=38&amp;pages=e131&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR47" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Robinson MD, Oshlack A. A scaling normalization method for differential expression analysis of RNA-seq data. Genome Biol. 2010;11:R25.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20196867"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2864565"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/gb-2010-11-3-r25"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20scaling%20normalization%20method%20for%20differential%20expression%20analysis%20of%20RNA-seq%20data&amp;author=MD.%20Robinson&amp;author=A.%20Oshlack&amp;journal=Genome%20Biol&amp;volume=11&amp;pages=R25&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR48" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Anders S, Huber W. Differential expression analysis for sequence count data. Genome Biol. 2010;11:R106.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20979621"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3218662"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/gb-2010-11-10-r106"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Differential%20expression%20analysis%20for%20sequence%20count%20data&amp;author=S.%20Anders&amp;author=W.%20Huber&amp;journal=Genome%20Biol&amp;volume=11&amp;pages=R106&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR49" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li J, Witten DM, Johnstone IM, Tibshirani R. Normalization, testing, and false discovery rate estimation for RNA-sequencing data. Biostatistics. 2012;13:523–38.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22003245"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/biostatistics/kxr031"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Normalization%2C%20testing%2C%20and%20false%20discovery%20rate%20estimation%20for%20RNA-sequencing%20data&amp;author=J.%20Li&amp;author=DM.%20Witten&amp;author=IM.%20Johnstone&amp;author=R.%20Tibshirani&amp;journal=Biostatistics&amp;volume=13&amp;pages=523-38&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR50" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Trapnell C, Hendrickson DG, Sauvageau M, Goff L, Rinn JL, Pachter L. Differential analysis of gene regulation at transcript resolution with RNA-seq. Nat Biotechnol. 2012;31:46–53.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23222703"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.2450"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Differential%20analysis%20of%20gene%20regulation%20at%20transcript%20resolution%20with%20RNA-seq&amp;author=C.%20Trapnell&amp;author=DG.%20Hendrickson&amp;author=M.%20Sauvageau&amp;author=L.%20Goff&amp;author=JL.%20Rinn&amp;author=L.%20Pachter&amp;journal=Nat%20Biotechnol&amp;volume=31&amp;pages=46-53&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR51" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Auer PL, Doerge RW. Statistical design and analysis of RNA sequencing data. Genetics. 2010;185:405–16.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20439781"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2881125"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1534/genetics.110.114983"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Statistical%20design%20and%20analysis%20of%20RNA%20sequencing%20data&amp;author=PL.%20Auer&amp;author=RW.%20Doerge&amp;journal=Genetics&amp;volume=185&amp;pages=405-16&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR52" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Johnson WE, Rabinovic A, Li C. Adjusting batch effects in microarray expression data using Empirical Bayes methods. Biostatistics. 2007;8:118–27.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16632515"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/biostatistics/kxj037"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Adjusting%20batch%20effects%20in%20microarray%20expression%20data%20using%20Empirical%20Bayes%20methods&amp;author=WE.%20Johnson&amp;author=A.%20Rabinovic&amp;author=C.%20Li&amp;journal=Biostatistics&amp;volume=8&amp;pages=118-27&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR53" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nueda MJ, Ferrer A, Conesa A. ARSyN: a method for the identification and removal of systematic noise in multifactorial time course microarray experiments. Biostatistics. 2012;13:553–66.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22085896"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/biostatistics/kxr042"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=ARSyN%3A%20a%20method%20for%20the%20identification%20and%20removal%20of%20systematic%20noise%20in%20multifactorial%20time%20course%20microarray%20experiments&amp;author=MJ.%20Nueda&amp;author=A.%20Ferrer&amp;author=A.%20Conesa&amp;journal=Biostatistics&amp;volume=13&amp;pages=553-66&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR54" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Robinson MD, Smyth GK. Moderated statistical tests for assessing differences in tag abundance. Bioinformatics. 2007;23:2881–7.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17881408"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btm453"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Moderated%20statistical%20tests%20for%20assessing%20differences%20in%20tag%20abundance&amp;author=MD.%20Robinson&amp;author=GK.%20Smyth&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2881-7&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR55" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Law CW, Chen Y, Shi W, Smyth GK. Voom: precision weights unlock linear model analysis tools for RNA-seq read counts. Genome Biol. 2014;15:R29.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24485249"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4053721"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/gb-2014-15-2-r29"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Voom%3A%20precision%20weights%20unlock%20linear%20model%20analysis%20tools%20for%20RNA-seq%20read%20counts&amp;author=CW.%20Law&amp;author=Y.%20Chen&amp;author=W.%20Shi&amp;author=GK.%20Smyth&amp;journal=Genome%20Biol&amp;volume=15&amp;pages=R29&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR56" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Soneson C, Delorenzi M. A comparison of methods for differential expression analysis of RNA-seq data. BMC Bioinformatics. 2013;14:91.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23497356"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2105-14-91"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3608160"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20comparison%20of%20methods%20for%20differential%20expression%20analysis%20of%20RNA-seq%20data&amp;author=C.%20Soneson&amp;author=M.%20Delorenzi&amp;journal=BMC%20Bioinformatics&amp;volume=14&amp;pages=91&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR57" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Robinson MD, McCarthy DJ, Smyth GK. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics. 2010;26:139–40.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19910308"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btp616"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=edgeR%3A%20a%20Bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data&amp;author=MD.%20Robinson&amp;author=DJ.%20McCarthy&amp;author=GK.%20Smyth&amp;journal=Bioinformatics&amp;volume=26&amp;pages=139-40&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR58" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Love MI, Huber W, Anders S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 2014;15:550.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25516281"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302049"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s13059-014-0550-8"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&amp;author=MI.%20Love&amp;author=W.%20Huber&amp;author=S.%20Anders&amp;journal=Genome%20Biol&amp;volume=15&amp;pages=550&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR59" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hardcastle TJ, Kelly KA. baySeq: empirical Bayesian methods for identifying differential expression in sequence count data. BMC Bioinformatics. 2010;11:422.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20698981"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928208"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2105-11-422"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=baySeq%3A%20empirical%20Bayesian%20methods%20for%20identifying%20differential%20expression%20in%20sequence%20count%20data&amp;author=TJ.%20Hardcastle&amp;author=KA.%20Kelly&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=422&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR60" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Leng N, Dawson JA, Thomson JA, Ruotti V, Rissman AI, Smits BM, et al. EBSeq: an empirical Bayes hierarchical model for inference in RNA-seq experiments. Bioinformatics. 2013;29:1035–43.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23428641"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3624807"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btt087"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=EBSeq%3A%20an%20empirical%20Bayes%20hierarchical%20model%20for%20inference%20in%20RNA-seq%20experiments&amp;author=N.%20Leng&amp;author=JA.%20Dawson&amp;author=JA.%20Thomson&amp;author=V.%20Ruotti&amp;author=AI.%20Rissman&amp;author=BM.%20Smits&amp;journal=Bioinformatics&amp;volume=29&amp;pages=1035-43&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR61" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li J, Tibshirani R. Finding consistent patterns: a nonparametric approach for identifying differential expression in RNA-Seq data. Stat Methods Med Res. 2013;22:519–36.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22127579"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1177/0962280211428386"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Finding%20consistent%20patterns%3A%20a%20nonparametric%20approach%20for%20identifying%20differential%20expression%20in%20RNA-Seq%20data&amp;author=J.%20Li&amp;author=R.%20Tibshirani&amp;journal=Stat%20Methods%20Med%20Res&amp;volume=22&amp;pages=519-36&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR62" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang L, Feng Z, Wang X, Wang X, Zhang X. DEGseq: an R package for identifying differentially expressed genes from RNA-seq data. Bioinformatics. 2010;26:136–8.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19855105"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btp612"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=DEGseq%3A%20an%20R%20package%20for%20identifying%20differentially%20expressed%20genes%20from%20RNA-seq%20data&amp;author=L.%20Wang&amp;author=Z.%20Feng&amp;author=X.%20Wang&amp;author=X.%20Wang&amp;author=X.%20Zhang&amp;journal=Bioinformatics&amp;volume=26&amp;pages=136-8&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR63" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rapaport F, Khanin R, Liang Y, Pirun M, Krek A, Zumbo P, et al. Comprehensive evaluation of differential gene expression analysis methods for RNA-seq data. Genome Biol. 2013;14:R95.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24020486"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4054597"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/gb-2013-14-9-r95"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comprehensive%20evaluation%20of%20differential%20gene%20expression%20analysis%20methods%20for%20RNA-seq%20data&amp;author=F.%20Rapaport&amp;author=R.%20Khanin&amp;author=Y.%20Liang&amp;author=M.%20Pirun&amp;author=A.%20Krek&amp;author=P.%20Zumbo&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=R95&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR64" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Seyednasrollah F, Laiho A, Elo LL. Comparison of software packages for detecting differential expression in RNA-seq studies. Brief Bioinform. 2015;16:59–70.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bib/bbt086"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24300110"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparison%20of%20software%20packages%20for%20detecting%20differential%20expression%20in%20RNA-seq%20studies&amp;author=F.%20Seyednasrollah&amp;author=A.%20Laiho&amp;author=LL.%20Elo&amp;journal=Brief%20Bioinform&amp;volume=16&amp;pages=59-70&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR65" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zheng S, Chen L. A hierarchical Bayesian model for comparing transcriptomes at the individual transcript isoform level. Nucleic Acids Res. 2009;37:e75.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19417075"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2691848"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkp282"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20hierarchical%20Bayesian%20model%20for%20comparing%20transcriptomes%20at%20the%20individual%20transcript%20isoform%20level&amp;author=S.%20Zheng&amp;author=L.%20Chen&amp;journal=Nucleic%20Acids%20Res&amp;volume=37&amp;pages=e75&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR66" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Trapnell C, Roberts A, Goff L, Pertea G, Kim D, Kelley DR, et al. Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks. Nat Protoc. 2012;7:562–78.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22383036"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3334321"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nprot.2012.016"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Differential%20gene%20and%20transcript%20expression%20analysis%20of%20RNA-seq%20experiments%20with%20TopHat%20and%20Cufflinks&amp;author=C.%20Trapnell&amp;author=A.%20Roberts&amp;author=L.%20Goff&amp;author=G.%20Pertea&amp;author=D.%20Kim&amp;author=DR.%20Kelley&amp;journal=Nat%20Protoc&amp;volume=7&amp;pages=562-78&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR67" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Singh D, Orellana CF, Hu Y, Jones CD, Liu Y, Chiang DY, et al. FDM: a graph-based statistical method to detect differential transcription using RNA-seq data. Bioinformatics. 2011;27:2633–40.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21824971"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3179659"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btr458"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=FDM%3A%20a%20graph-based%20statistical%20method%20to%20detect%20differential%20transcription%20using%20RNA-seq%20data&amp;author=D.%20Singh&amp;author=CF.%20Orellana&amp;author=Y.%20Hu&amp;author=CD.%20Jones&amp;author=Y.%20Liu&amp;author=DY.%20Chiang&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2633-40&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR68" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shi Y, Jiang H. rSeqDiff: detecting differential isoform expression from RNA-Seq data using hierarchical likelihood ratio test. PLoS One. 2013;8:e79448.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24260225"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3832546"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0079448"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=rSeqDiff%3A%20detecting%20differential%20isoform%20expression%20from%20RNA-Seq%20data%20using%20hierarchical%20likelihood%20ratio%20test&amp;author=Y.%20Shi&amp;author=H.%20Jiang&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e79448&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR69" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Anders S, Reyes A, Huber W. Detecting differential usage of exons from RNA-seq data. Genome Res. 2012;22:2008–17.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22722343"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3460195"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.133744.111"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Detecting%20differential%20usage%20of%20exons%20from%20RNA-seq%20data&amp;author=S.%20Anders&amp;author=A.%20Reyes&amp;author=W.%20Huber&amp;journal=Genome%20Res&amp;volume=22&amp;pages=2008-17&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR70" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang W, Qin Z, Feng Z, Wang X, Zhang X. Identifying differentially spliced genes from two groups of RNA-seq samples. Gene. 2013;518:164–70.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23228854"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.gene.2012.11.045"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Identifying%20differentially%20spliced%20genes%20from%20two%20groups%20of%20RNA-seq%20samples&amp;author=W.%20Wang&amp;author=Z.%20Qin&amp;author=Z.%20Feng&amp;author=X.%20Wang&amp;author=X.%20Zhang&amp;journal=Gene&amp;volume=518&amp;pages=164-70&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR71" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shen S, Park JW, Lu ZX, Lin L, Henry MD, Wu YN, et al. rMATS: robust and flexible detection of differential alternative splicing from replicate RNA-Seq data. Proc Natl Acad Sci U S A. 2014;111:E5593–601.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25480548"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4280593"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.1419161111"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=rMATS%3A%20robust%20and%20flexible%20detection%20of%20differential%20alternative%20splicing%20from%20replicate%20RNA-Seq%20data&amp;author=S.%20Shen&amp;author=JW.%20Park&amp;author=ZX.%20Lu&amp;author=L.%20Lin&amp;author=MD.%20Henry&amp;author=YN.%20Wu&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=111&amp;pages=E5593-601&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR72" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Drewe P, Stegle O, Hartmann L, Kahles A, Bohnert R, Wachter A, et al. Accurate detection of differential RNA processing. Nucleic Acids Res. 2013;41:5189–98.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23585274"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3664801"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkt211"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Accurate%20detection%20of%20differential%20RNA%20processing&amp;author=P.%20Drewe&amp;author=O.%20Stegle&amp;author=L.%20Hartmann&amp;author=A.%20Kahles&amp;author=R.%20Bohnert&amp;author=A.%20Wachter&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=5189-98&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR73" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hu Y, Huang Y, Du Y, Orellana CF, Singh D, Johnson AR, et al. DissSplice: the genome-wide detection of differential splicing events with RNA-seq. Nucleic Acids Res. 2013;41:e39.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23155066"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gks1026"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=DissSplice%3A%20the%20genome-wide%20detection%20of%20differential%20splicing%20events%20with%20RNA-seq&amp;author=Y.%20Hu&amp;author=Y.%20Huang&amp;author=Y.%20Du&amp;author=CF.%20Orellana&amp;author=D.%20Singh&amp;author=AR.%20Johnson&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=e39&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR74" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hilker R, Stadermann KB, Doppmeier D, Kalinowski J, Stoye J, Straube J, et al. ReadXplorer - visualization and analysis of mapped sequences. Bioinformatics. 2014;30:2247–54.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24790157"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4217279"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btu205"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=ReadXplorer%20-%20visualization%20and%20analysis%20of%20mapped%20sequences&amp;author=R.%20Hilker&amp;author=KB.%20Stadermann&amp;author=D.%20Doppmeier&amp;author=J.%20Kalinowski&amp;author=J.%20Stoye&amp;author=J.%20Straube&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2247-54&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR75" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kent WJ, Sugnet CW, Furey TS, Roskin KM, Pringle TH, Zahler AM, et al. The Human Genome Browser at UCSC. Genome Res. 2002;12:996–1006.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12045153"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC186604"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.229102.%20Article%20published%20online%20before%20print%20in%20May%202002"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20Human%20Genome%20Browser%20at%20UCSC&amp;author=WJ.%20Kent&amp;author=CW.%20Sugnet&amp;author=TS.%20Furey&amp;author=KM.%20Roskin&amp;author=TH.%20Pringle&amp;author=AM.%20Zahler&amp;journal=Genome%20Res&amp;volume=12&amp;pages=996-1006&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR76" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Thorvaldsdóttir H, Robinson JT, Mesirov JP. Integrative Genomics Viewer (IGV): high- performance genomics data visualization and exploration. Brief Bioinformatics. 2012;14:178–92.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22517427"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3603213"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bib/bbs017"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Integrative%20Genomics%20Viewer%20%28IGV%29%3A%20high-%20performance%20genomics%20data%20visualization%20and%20exploration&amp;author=H.%20Thorvaldsd%C3%B3ttir&amp;author=JT.%20Robinson&amp;author=JP.%20Mesirov&amp;journal=Brief%20Bioinformatics&amp;volume=14&amp;pages=178-92&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR77" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Medina I, Salavert F, Sanchez R, de Maria A, Alonso R, Escobar P, et al. Genome Maps, a new generation genome browser. Nucleic Acids Res. 2013;41(Web Server issue):W41–6.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23748955"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3692043"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkt530"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome%20Maps%2C%20a%20new%20generation%20genome%20browser&amp;author=I.%20Medina&amp;author=F.%20Salavert&amp;author=R.%20Sanchez&amp;author=A.%20Maria&amp;author=R.%20Alonso&amp;author=P.%20Escobar&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;issue=Web%20Server%20issue&amp;pages=W41-6&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR78" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Fiume M, Williams V, Brook A, Brudno M. Savant: genome browser for high-throughput sequencing data. Bioinformatics. 2010;26:1938–44.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20562449"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3271355"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btq332"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Savant%3A%20genome%20browser%20for%20high-throughput%20sequencing%20data&amp;author=M.%20Fiume&amp;author=V.%20Williams&amp;author=A.%20Brook&amp;author=M.%20Brudno&amp;journal=Bioinformatics&amp;volume=26&amp;pages=1938-44&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR79" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rogé X, Zhang X. RNAseqViewer: visualization tool for RNA-Seq data. Bioinformatics. 2013;30:891–2.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24215023"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btt649"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=RNAseqViewer%3A%20visualization%20tool%20for%20RNA-Seq%20data&amp;author=X.%20Rog%C3%A9&amp;author=X.%20Zhang&amp;journal=Bioinformatics&amp;volume=30&amp;pages=891-2&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR80" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Katz Y, Wang ET, Silterra J, Schwartz S, Wong B, Thorvaldsdóttir H, et al. Quantitative visualization of alternative exon expression from RNA-seq data. Bioinformatics. 2015;31:2400–2.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25617416"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4542614"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btv034"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Quantitative%20visualization%20of%20alternative%20exon%20expression%20from%20RNA-seq%20data&amp;author=Y.%20Katz&amp;author=ET.%20Wang&amp;author=J.%20Silterra&amp;author=S.%20Schwartz&amp;author=B.%20Wong&amp;author=H.%20Thorvaldsd%C3%B3ttir&amp;journal=Bioinformatics&amp;volume=31&amp;pages=2400-2&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR81" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wu E, Nance T, Montgomery SB. SplicePlot: a utility for visualizing splicing quantitative trait loci. Bioinformatics. 2014;30:1025–6.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24363378"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btt733"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=SplicePlot%3A%20a%20utility%20for%20visualizing%20splicing%20quantitative%20trait%20loci&amp;author=E.%20Wu&amp;author=T.%20Nance&amp;author=SB.%20Montgomery&amp;journal=Bioinformatics&amp;volume=30&amp;pages=1025-6&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR82" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ryan MC, Cleland J, Kim R, Wong WC, Weinstein JN. SpliceSeq: a resource for analysis and visualization of RNA-Seq data on alternative splicing and its functional impacts. Bioinformatics. 2012;28:2385–7.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22820202"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3436850"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/bts452"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=SpliceSeq%3A%20a%20resource%20for%20analysis%20and%20visualization%20of%20RNA-Seq%20data%20on%20alternative%20splicing%20and%20its%20functional%20impacts&amp;author=MC.%20Ryan&amp;author=J.%20Cleland&amp;author=R.%20Kim&amp;author=WC.%20Wong&amp;author=JN.%20Weinstein&amp;journal=Bioinformatics&amp;volume=28&amp;pages=2385-7&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR83" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liu Q, Chen C, Shen E, Zhao F, Sun Z, Wu J. Detection, annotation and visualization of alternative splicing from RNA-Seq data with SplicingViewer. Genomics. 2012;99:178–82.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22226708"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.ygeno.2011.12.003"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Detection%2C%20annotation%20and%20visualization%20of%20alternative%20splicing%20from%20RNA-Seq%20data%20with%20SplicingViewer&amp;author=Q.%20Liu&amp;author=C.%20Chen&amp;author=E.%20Shen&amp;author=F.%20Zhao&amp;author=Z.%20Sun&amp;author=J.%20Wu&amp;journal=Genomics&amp;volume=99&amp;pages=178-82&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR84" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dietrich S, Wiegand S, Liesegang H. TraV: a genome context sensitive transcriptome browser. PLoS One. 2014;9:e93677.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24709941"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3977867"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0093677"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=TraV%3A%20a%20genome%20context%20sensitive%20transcriptome%20browser&amp;author=S.%20Dietrich&amp;author=S.%20Wiegand&amp;author=H.%20Liesegang&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e93677&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR85" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Carrara M, Beccuti M, Lazzarato F, Cavallo F, Cordero F, Donatelli S, et al. State-of-the-art fusion-finder algorithms sensitivity and specificity. BioMed Res Int. 2013;15:340620.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=State-of-the-art%20fusion-finder%20algorithms%20sensitivity%20and%20specificity&amp;author=M.%20Carrara&amp;author=M.%20Beccuti&amp;author=F.%20Lazzarato&amp;author=F.%20Cavallo&amp;author=F.%20Cordero&amp;author=S.%20Donatelli&amp;journal=BioMed%20Res%20Int&amp;volume=15&amp;pages=340620&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR86" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Maher CA, Palanisamy N, Brenner JC, Cao X, Kalyana-Sundaram S, Luo S, et al. Chimeric transcript discovery by paired-end transcriptome sequencing. Proc Natl Acad Sci U S A. 2009;106:12353–8.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19592507"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2708976"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.0904720106"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Chimeric%20transcript%20discovery%20by%20paired-end%20transcriptome%20sequencing&amp;author=CA.%20Maher&amp;author=N.%20Palanisamy&amp;author=JC.%20Brenner&amp;author=X.%20Cao&amp;author=S.%20Kalyana-Sundaram&amp;author=S.%20Luo&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=106&amp;pages=12353-8&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR87" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yoshihara K, Wang Q, Torres-Garcia W, Zheng S, Vegesna R, Kim H, et al. The landscape and therapeutic relevance of cancer-associated transcript fusions. Oncogene. 2015;34:4845–54.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25500544"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/onc.2014.406"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20landscape%20and%20therapeutic%20relevance%20of%20cancer-associated%20transcript%20fusions&amp;author=K.%20Yoshihara&amp;author=Q.%20Wang&amp;author=W.%20Torres-Garcia&amp;author=S.%20Zheng&amp;author=R.%20Vegesna&amp;author=H.%20Kim&amp;journal=Oncogene&amp;volume=34&amp;pages=4845-54&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR88" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">McPherson A, Hormozdiari F, Zayed A, Giuliany R, Ha G, Sun MG, et al. deFuse: an algorithm for gene fusion discovery in tumor RNA-seq data. PLoS Comput Biol. 2011;7:e1001138.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21625565"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3098195"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pcbi.1001138"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=deFuse%3A%20an%20algorithm%20for%20gene%20fusion%20discovery%20in%20tumor%20RNA-seq%20data&amp;author=A.%20McPherson&amp;author=F.%20Hormozdiari&amp;author=A.%20Zayed&amp;author=R.%20Giuliany&amp;author=G.%20Ha&amp;author=MG.%20Sun&amp;journal=PLoS%20Comput%20Biol&amp;volume=7&amp;pages=e1001138&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR89" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wu C, Wyatt AW, McPherson A, Lin D, McConeghy BJ, Mo F, et al. Poly-gene fusion transcripts and chromothripsis in prostate cancer. Gene Chromosomes Cancer. 2012;51:1144–53.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1002/gcc.21999"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Poly-gene%20fusion%20transcripts%20and%20chromothripsis%20in%20prostate%20cancer&amp;author=C.%20Wu&amp;author=AW.%20Wyatt&amp;author=A.%20McPherson&amp;author=D.%20Lin&amp;author=BJ.%20McConeghy&amp;author=F.%20Mo&amp;journal=Gene%20Chromosomes%20Cancer&amp;volume=51&amp;pages=1144-53&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR90" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wyatt AW, Mo F, Wang K, McConeghy B, Brahmbhatt S, Jong L, et al. Heterogeneity in the inter-tumor transcriptome of high risk prostate cancer. Genome Biol. 2014;15:426.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25155515"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4169643"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s13059-014-0426-y"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Heterogeneity%20in%20the%20inter-tumor%20transcriptome%20of%20high%20risk%20prostate%20cancer&amp;author=AW.%20Wyatt&amp;author=F.%20Mo&amp;author=K.%20Wang&amp;author=B.%20McConeghy&amp;author=S.%20Brahmbhatt&amp;author=L.%20Jong&amp;journal=Genome%20Biol&amp;volume=15&amp;pages=426&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR91" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stransky N, Cerami E, Schalm S, Kim JL, Lengauer C. The landscape of kinase fusions in cancer. Nat Commun. 2014;5:4846.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25204415"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4175590"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/ncomms5846"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20landscape%20of%20kinase%20fusions%20in%20cancer&amp;author=N.%20Stransky&amp;author=E.%20Cerami&amp;author=S.%20Schalm&amp;author=JL.%20Kim&amp;author=C.%20Lengauer&amp;journal=Nat%20Commun&amp;volume=5&amp;pages=4846&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR92" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rabbitts TH. Commonality but diversity in cancer gene fusions. Cell. 2009;137:391–5.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19410533"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.cell.2009.04.034"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Commonality%20but%20diversity%20in%20cancer%20gene%20fusions&amp;author=TH.%20Rabbitts&amp;journal=Cell&amp;volume=137&amp;pages=391-5&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR93" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">McPherson A, Wu C, Hajirasouliha I, Hormozdiari F, Hach F, Lapuk A, et al. Comrad: detection of expressed rearrangements by integrated analysis of RNA-Seq and low coverage genome sequence data. Bioinformatics. 2011;27:1481–8.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21478487"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btr184"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comrad%3A%20detection%20of%20expressed%20rearrangements%20by%20integrated%20analysis%20of%20RNA-Seq%20and%20low%20coverage%20genome%20sequence%20data&amp;author=A.%20McPherson&amp;author=C.%20Wu&amp;author=I.%20Hajirasouliha&amp;author=F.%20Hormozdiari&amp;author=F.%20Hach&amp;author=A.%20Lapuk&amp;journal=Bioinformatics&amp;volume=27&amp;pages=1481-8&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR94" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Iyer MK, Chinnaiyan AM, Maher CA. ChimeraScan: a tool for identifying chimeric transcription in sequencing data. Bioinformatics. 2011;27:2903–4.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21840877"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3187648"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btr467"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=ChimeraScan%3A%20a%20tool%20for%20identifying%20chimeric%20transcription%20in%20sequencing%20data&amp;author=MK.%20Iyer&amp;author=AM.%20Chinnaiyan&amp;author=CA.%20Maher&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2903-4&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR95" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pflueger D, Terry S, Sboner A, Habegger L, Esgueva R, Lin PC, et al. Discovery of non-ETS gene fusions in human prostate cancer using next-generation RNA sequencing. Genome Res. 2011;21:56–67.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21036922"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3012926"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.110684.110"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Discovery%20of%20non-ETS%20gene%20fusions%20in%20human%20prostate%20cancer%20using%20next-generation%20RNA%20sequencing&amp;author=D.%20Pflueger&amp;author=S.%20Terry&amp;author=A.%20Sboner&amp;author=L.%20Habegger&amp;author=R.%20Esgueva&amp;author=PC.%20Lin&amp;journal=Genome%20Res&amp;volume=21&amp;pages=56-67&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR96" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wu J, Liu Q, Wang X, Zheng J, Wang T, You M, et al. mirTools 2.0 for non-coding RNA discovery, profiling, and functional annotation based on high-throughput sequencing. RNA Biol. 2013;10:1087–92.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23778453"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3849156"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.4161/rna.25193"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=mirTools%202.0%20for%20non-coding%20RNA%20discovery%2C%20profiling%2C%20and%20functional%20annotation%20based%20on%20high-throughput%20sequencing&amp;author=J.%20Wu&amp;author=Q.%20Liu&amp;author=X.%20Wang&amp;author=J.%20Zheng&amp;author=T.%20Wang&amp;author=M.%20You&amp;journal=RNA%20Biol&amp;volume=10&amp;pages=1087-92&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR97" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Langmead B, Salzberg SL. Fast gapped-read alignment with Bowtie 2. Nat Methods. 2012;9:357–9.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22388286"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3322381"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nmeth.1923"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=SL.%20Salzberg&amp;journal=Nat%20Methods&amp;volume=9&amp;pages=357-9&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR98" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li H, Durbin R. Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics. 2009;25:1754–60.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19451168"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705234"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btp324"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR99" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Prüfer K, Stenzel U, Dannemann M, Green RE, Lachmann M, Kelso J. PatMaN: rapid alignment of short sequences to large databases. Bioinformatics. 2008;24:1530–1.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18467344"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2718670"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btn223"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=PatMaN%3A%20rapid%20alignment%20of%20short%20sequences%20to%20large%20databases&amp;author=K.%20Pr%C3%BCfer&amp;author=U.%20Stenzel&amp;author=M.%20Dannemann&amp;author=RE.%20Green&amp;author=M.%20Lachmann&amp;author=J.%20Kelso&amp;journal=Bioinformatics&amp;volume=24&amp;pages=1530-1&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR100" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Emde AK, Grunert M, Weese D, Reinert K, Sperling SR. MicroRazerS: rapid alignment of small RNA reads. Bioinformatics. 2010;26:123–4.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19880369"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btp601"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MicroRazerS%3A%20rapid%20alignment%20of%20small%20RNA%20reads&amp;author=AK.%20Emde&amp;author=M.%20Grunert&amp;author=D.%20Weese&amp;author=K.%20Reinert&amp;author=SR.%20Sperling&amp;journal=Bioinformatics&amp;volume=26&amp;pages=123-4&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR101" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">An J, Lai J, Lehman ML, Nelson CC. miRDeep*: an integrated application tool for miRNA identification from RNA sequencing data. Nucleic Acids Res. 2013;41:727–37.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23221645"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gks1187"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=miRDeep%2A%3A%20an%20integrated%20application%20tool%20for%20miRNA%20identification%20from%20RNA%20sequencing%20data&amp;author=J.%20An&amp;author=J.%20Lai&amp;author=ML.%20Lehman&amp;author=CC.%20Nelson&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=727-37&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR102" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yang X, Li L. miRDeep-P: a computational tool for analyzing the microRNA transcriptome in plants. Bioinformatics. 2011;27:2614–5.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21775303"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=miRDeep-P%3A%20a%20computational%20tool%20for%20analyzing%20the%20microRNA%20transcriptome%20in%20plants&amp;author=X.%20Yang&amp;author=L.%20Li&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2614-5&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR103" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stocks MB, Moxon S, Mapleson D, Woolfenden HC, Mohorianu I, Folkes L, et al. The UEA sRNA workbench: a suite of tools for analysing and visualizing next generation sequencing microRNA and small RNA datasets. Bioinformatics. 2012;28:2059–61.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22628521"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3400958"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/bts311"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20UEA%20sRNA%20workbench%3A%20a%20suite%20of%20tools%20for%20analysing%20and%20visualizing%20next%20generation%20sequencing%20microRNA%20and%20small%20RNA%20datasets&amp;author=MB.%20Stocks&amp;author=S.%20Moxon&amp;author=D.%20Mapleson&amp;author=HC.%20Woolfenden&amp;author=I.%20Mohorianu&amp;author=L.%20Folkes&amp;journal=Bioinformatics&amp;volume=28&amp;pages=2059-61&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR104" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Axtell MJ. ShortStack: comprehensive annotation and quantification of small RNA genes. RNA. 2013;19:740–51.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23610128"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3683909"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1261/rna.035279.112"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=ShortStack%3A%20comprehensive%20annotation%20and%20quantification%20of%20small%20RNA%20genes&amp;author=MJ.%20Axtell&amp;journal=RNA&amp;volume=19&amp;pages=740-51&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR105" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Giurato G, De Filippo MR, Rinaldi A, Hashim A, Nassa G, Ravo M, et al. iMir: an integrated pipeline for high-throughput analysis of small non-coding RNA data obtained by smallRNA-Seq. BMC Bioinformatics. 2013;14:362.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24330401"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3878829"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2105-14-362"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=iMir%3A%20an%20integrated%20pipeline%20for%20high-throughput%20analysis%20of%20small%20non-coding%20RNA%20data%20obtained%20by%20smallRNA-Seq&amp;author=G.%20Giurato&amp;author=MR.%20Filippo&amp;author=A.%20Rinaldi&amp;author=A.%20Hashim&amp;author=G.%20Nassa&amp;author=M.%20Ravo&amp;journal=BMC%20Bioinformatics&amp;volume=14&amp;pages=362&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR106" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Young MD, Wakefield MJ, Smyth GK, Oshlack A. Gene ontology analysis for RNA-seq: accounting for selection bias. Genome Biol. 2010;11:1–12.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/gb-2010-11-2-r14"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Gene%20ontology%20analysis%20for%20RNA-seq%3A%20accounting%20for%20selection%20bias&amp;author=MD.%20Young&amp;author=MJ.%20Wakefield&amp;author=GK.%20Smyth&amp;author=A.%20Oshlack&amp;journal=Genome%20Biol&amp;volume=11&amp;pages=1-12&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR107" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hänzelmann S, Castelo R, Guinney J. GSVA: gene set variation analysis for microarray and RNA-Seq data. BMC Bioinformatics. 2013;14:7.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23323831"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3618321"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2105-14-7"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=GSVA%3A%20gene%20set%20variation%20analysis%20for%20microarray%20and%20RNA-Seq%20data&amp;author=S.%20H%C3%A4nzelmann&amp;author=R.%20Castelo&amp;author=J.%20Guinney&amp;journal=BMC%20Bioinformatics&amp;volume=14&amp;pages=7&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR108" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang X, Cairns MJ. Gene set enrichment analysis of RNA-Seq data: integrating differential expression and splicing. BMC Bioinformatics. 2013;14 Suppl 5:S16.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23734663"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3622641"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2105-14-S5-S16"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Gene%20set%20enrichment%20analysis%20of%20RNA-Seq%20data%3A%20integrating%20differential%20expression%20and%20splicing&amp;author=X.%20Wang&amp;author=MJ.%20Cairns&amp;journal=BMC%20Bioinformatics&amp;volume=14&amp;issue=Suppl%205&amp;pages=S16&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR109" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM, et al. Gene Ontology: tool for the unification of biology. Nat Genet. 2000;25:25–9.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10802651"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3037419"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/75556"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Gene%20Ontology%3A%20tool%20for%20the%20unification%20of%20biology&amp;author=M.%20Ashburner&amp;author=CA.%20Ball&amp;author=JA.%20Blake&amp;author=D.%20Botstein&amp;author=H.%20Butler&amp;author=JM.%20Cherry&amp;journal=Nat%20Genet&amp;volume=25&amp;pages=25-9&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR110" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Huber W, Carey VJ, Gentleman R, Anders S, Carlson M, Carvalho BS, et al. Orchestrating high-throughput genomic analysis with Bioconductor. Nat Methods. 2015;12:115–21.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25633503"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4509590"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nmeth.3252"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Orchestrating%20high-throughput%20genomic%20analysis%20with%20Bioconductor&amp;author=W.%20Huber&amp;author=VJ.%20Carey&amp;author=R.%20Gentleman&amp;author=S.%20Anders&amp;author=M.%20Carlson&amp;author=BS.%20Carvalho&amp;journal=Nat%20Methods&amp;volume=12&amp;pages=115-21&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR111" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Huang DW, Sherman BT, Lempicki RA. Systematic and integrative analysis of large gene lists using DAVID Bioinformatics Resources. Nat Protocols. 2009;4:44–57.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nprot.2008.211"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Systematic%20and%20integrative%20analysis%20of%20large%20gene%20lists%20using%20DAVID%20Bioinformatics%20Resources&amp;author=DW.%20Huang&amp;author=BT.%20Sherman&amp;author=RA.%20Lempicki&amp;journal=Nat%20Protocols&amp;volume=4&amp;pages=44-57&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR112" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Huang DW, Sherman BT, Lempicki RA. Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists. Nucleic Acids Res. 2009;37:1–13.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkn923"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Bioinformatics%20enrichment%20tools%3A%20paths%20toward%20the%20comprehensive%20functional%20analysis%20of%20large%20gene%20lists&amp;author=DW.%20Huang&amp;author=BT.%20Sherman&amp;author=RA.%20Lempicki&amp;journal=Nucleic%20Acids%20Res&amp;volume=37&amp;pages=1-13&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR113" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Medina I, Carbonell J, Pulido L, Madeira SC, Goetz S, Conesa A, et al. Babelomics: an integrative platform for the analysis of transcriptomics, proteomics and genomic data with advanced functional profiling. Nucleic Acids Res. 2010;38 suppl 2:W210–3.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20478823"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2896184"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkq388"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Babelomics%3A%20an%20integrative%20platform%20for%20the%20analysis%20of%20transcriptomics%2C%20proteomics%20and%20genomic%20data%20with%20advanced%20functional%20profiling&amp;author=I.%20Medina&amp;author=J.%20Carbonell&amp;author=L.%20Pulido&amp;author=SC.%20Madeira&amp;author=S.%20Goetz&amp;author=A.%20Conesa&amp;journal=Nucleic%20Acids%20Res&amp;volume=38&amp;issue=suppl%202&amp;pages=W210-3&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR114" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bairoch A, Boeckmann B, Ferro S, Gasteiger E. Swiss-Prot: juggling between evolution and stability. Brief Bioinformatics. 2004;5:39–55.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15153305"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bib/5.1.39"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Swiss-Prot%3A%20juggling%20between%20evolution%20and%20stability&amp;author=A.%20Bairoch&amp;author=B.%20Boeckmann&amp;author=S.%20Ferro&amp;author=E.%20Gasteiger&amp;journal=Brief%20Bioinformatics&amp;volume=5&amp;pages=39-55&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR115" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Finn RD, Bateman A, Clements J, Coggill P, Eberhardt RY, Eddy SR, et al. The Pfam protein families database. Nucleic Acids Res. 2014;42(Database issue):D222–30.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24288371"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkt1223"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20Pfam%20protein%20families%20database&amp;author=RD.%20Finn&amp;author=A.%20Bateman&amp;author=J.%20Clements&amp;author=P.%20Coggill&amp;author=RY.%20Eberhardt&amp;author=SR.%20Eddy&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;issue=Database%20issue&amp;pages=D222-30&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR116" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hunter S, Jones P, Mitchell A, Apweiler R, Attwood TK, Bateman A, et al. InterPro in 2011: new developments in the family and domain prediction database. Nucleic Acids Res. 2011;40(Database issue):D306–12.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22096229"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3245097"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=InterPro%20in%202011%3A%20new%20developments%20in%20the%20family%20and%20domain%20prediction%20database&amp;author=S.%20Hunter&amp;author=P.%20Jones&amp;author=A.%20Mitchell&amp;author=R.%20Apweiler&amp;author=TK.%20Attwood&amp;author=A.%20Bateman&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;issue=Database%20issue&amp;pages=D306-12&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR117" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Conesa A, Götz S, García-Gómez JM, Terol J, Talón M, Robles M. Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research. Bioinformatics. 2005;21:3674–6.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16081474"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/bti610"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Blast2GO%3A%20a%20universal%20tool%20for%20annotation%2C%20visualization%20and%20analysis%20in%20functional%20genomics%20research&amp;author=A.%20Conesa&amp;author=S.%20G%C3%B6tz&amp;author=JM.%20Garc%C3%ADa-G%C3%B3mez&amp;author=J.%20Terol&amp;author=M.%20Tal%C3%B3n&amp;author=M.%20Robles&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3674-6&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR118" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gardner PP, Daub J, Tate JG, Nawrocki EP, Kolbe DL, Lindgreen S, et al. Rfam: updates to the RNA families database. Nucleic Acids Res. 2009;37 suppl 1:D136–40.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18953034"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkn766"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Rfam%3A%20updates%20to%20the%20RNA%20families%20database&amp;author=PP.%20Gardner&amp;author=J.%20Daub&amp;author=JG.%20Tate&amp;author=EP.%20Nawrocki&amp;author=DL.%20Kolbe&amp;author=S.%20Lindgreen&amp;journal=Nucleic%20Acids%20Res&amp;volume=37&amp;issue=suppl%201&amp;pages=D136-40&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR119" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kozomara A, Griffiths-Jones S. miRBase: annotating high confidence microRNAs using deep sequencing data. Nucleic Acids Res. 2014;42(Database issue):D68–73.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24275495"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkt1181"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=miRBase%3A%20annotating%20high%20confidence%20microRNAs%20using%20deep%20sequencing%20data&amp;author=A.%20Kozomara&amp;author=S.%20Griffiths-Jones&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;issue=Database%20issue&amp;pages=D68-73&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR120" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Enright AJ, John B, Gaul U, Tuschl T, Sander C, Marks DS. MicroRNA targets in <em class="EmphasisTypeItalic">Drosophila</em>. Genome Biol. 2003;5:R1.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14709173"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC395733"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/gb-2003-5-1-r1"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MicroRNA%20targets%20in%20Drosophila%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=AJ.%20Enright&amp;author=B.%20John&amp;author=U.%20Gaul&amp;author=T.%20Tuschl&amp;author=C.%20Sander&amp;author=DS.%20Marks&amp;journal=Genome%20Biol&amp;volume=5&amp;pages=R1&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR121" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Giambartolomei C, Vukcevic D, Schadt EE, Franke L, Hingorani AD, Wallace C, et al. Bayesian test for colocalisation between pairs of genetic association studies using summary statistics. PLoS Genet. 2014;10:e1004383.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24830394"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4022491"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pgen.1004383"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Bayesian%20test%20for%20colocalisation%20between%20pairs%20of%20genetic%20association%20studies%20using%20summary%20statistics&amp;author=C.%20Giambartolomei&amp;author=D.%20Vukcevic&amp;author=EE.%20Schadt&amp;author=L.%20Franke&amp;author=AD.%20Hingorani&amp;author=C.%20Wallace&amp;journal=PLoS%20Genet&amp;volume=10&amp;pages=e1004383&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR122" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Moffatt MF, Kabesch M, Liang L, Dixon AL, Strachan D, Heath S, et al. Genetic variants regulating ORMDL3 expression contribute to the risk of childhood asthma. Nature. 2007;448:470–3.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17611496"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature06014"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20variants%20regulating%20ORMDL3%20expression%20contribute%20to%20the%20risk%20of%20childhood%20asthma&amp;author=MF.%20Moffatt&amp;author=M.%20Kabesch&amp;author=L.%20Liang&amp;author=AL.%20Dixon&amp;author=D.%20Strachan&amp;author=S.%20Heath&amp;journal=Nature&amp;volume=448&amp;pages=470-3&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR123" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gilad Y, Rifkin S, Pritchard J. Revealing the architecture of gene regulation: the promise of eQTL studies. Trends Genet. 2008;24:408–15.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18597885"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2583071"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.tig.2008.06.001"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Revealing%20the%20architecture%20of%20gene%20regulation%3A%20the%20promise%20of%20eQTL%20studies&amp;author=Y.%20Gilad&amp;author=S.%20Rifkin&amp;author=J.%20Pritchard&amp;journal=Trends%20Genet&amp;volume=24&amp;pages=408-15&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR124" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gaffney D. Global properties and functional complexity of human gene regulatory variation. PLoS Genet. 2013;9:e1003501.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23737752"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3667745"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pgen.1003501"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Global%20properties%20and%20functional%20complexity%20of%20human%20gene%20regulatory%20variation&amp;author=D.%20Gaffney&amp;journal=PLoS%20Genet&amp;volume=9&amp;pages=e1003501&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR125" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Montgomery S, Sammeth M, Gutierrez-Arcelus M, Lach RP, Ingle C, Nisbett J, et al. Transcriptome genetics using second generation sequencing in a Caucasian population. Nature. 2010;464:773–7.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20220756"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature08903"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Transcriptome%20genetics%20using%20second%20generation%20sequencing%20in%20a%20Caucasian%20population&amp;author=S.%20Montgomery&amp;author=M.%20Sammeth&amp;author=M.%20Gutierrez-Arcelus&amp;author=RP.%20Lach&amp;author=C.%20Ingle&amp;author=J.%20Nisbett&amp;journal=Nature&amp;volume=464&amp;pages=773-7&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR126" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pickrell JK, Marioni JC, Pai AA, Degner JF, Engelhardt BE, Nkadori E, et al. Understanding mechanisms underlying human gene expression variation with RNA sequencing. Nature. 2010;464:768–72.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20220758"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3089435"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature08872"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Understanding%20mechanisms%20underlying%20human%20gene%20expression%20variation%20with%20RNA%20sequencing&amp;author=JK.%20Pickrell&amp;author=JC.%20Marioni&amp;author=AA.%20Pai&amp;author=JF.%20Degner&amp;author=BE.%20Engelhardt&amp;author=E.%20Nkadori&amp;journal=Nature&amp;volume=464&amp;pages=768-72&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR127" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lappalainen T, Sammeth M, Friedlander M, ‘t Hoen PA, Monlong J, Rivas MA, et al. Transcriptome and genome sequencing uncovers functional variation in humans. Nature. 2013;501:506–11.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24037378"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3918453"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature12531"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Transcriptome%20and%20genome%20sequencing%20uncovers%20functional%20variation%20in%20humans&amp;author=T.%20Lappalainen&amp;author=M.%20Sammeth&amp;author=M.%20Friedlander&amp;author=PA.%20%E2%80%98t%20Hoen&amp;author=J.%20Monlong&amp;author=MA.%20Rivas&amp;journal=Nature&amp;volume=501&amp;pages=506-11&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR128" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Battle A, Mostafavi S, Zhu X, Potash JB, Weissman MM, Shi J, et al. Characterizing the genetic basis of transcriptome diversity through RNA-sequencing of 922 individuals. Genome Res. 2014;24:14–24.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24092820"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3875855"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.155192.113"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Characterizing%20the%20genetic%20basis%20of%20transcriptome%20diversity%20through%20RNA-sequencing%20of%20922%20individuals&amp;author=A.%20Battle&amp;author=S.%20Mostafavi&amp;author=X.%20Zhu&amp;author=JB.%20Potash&amp;author=MM.%20Weissman&amp;author=J.%20Shi&amp;journal=Genome%20Res&amp;volume=24&amp;pages=14-24&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR129" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pastinen T. Genome-wide allele-specific analysis: insights into regulatory variation. Nat Rev Genet. 2010;11:533–8.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20567245"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nrg2815"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome-wide%20allele-specific%20analysis%3A%20insights%20into%20regulatory%20variation&amp;author=T.%20Pastinen&amp;journal=Nat%20Rev%20Genet&amp;volume=11&amp;pages=533-8&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR130" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sun W. A statistical framework for eQTL mapping using RNA-seq data. Biometrics. 2012;68:1–11.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21838806"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1541-0420.2011.01654.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20statistical%20framework%20for%20eQTL%20mapping%20using%20RNA-seq%20data&amp;author=W.%20Sun&amp;journal=Biometrics&amp;volume=68&amp;pages=1-11&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR131" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">van de Geijn B, McVicker G, Gilad Y, Pritchard JK. WASP: allele-specific for robust molecular quantitative trait locus discovery. Nat Methods. 2015;12:1061–3.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26366987"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4626402"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nmeth.3582"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=WASP%3A%20allele-specific%20for%20robust%20molecular%20quantitative%20trait%20locus%20discovery&amp;author=B.%20Geijn&amp;author=G.%20McVicker&amp;author=Y.%20Gilad&amp;author=JK.%20Pritchard&amp;journal=Nat%20Methods&amp;volume=12&amp;pages=1061-3&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR132" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kumasaka N, Knights AJ, Gaffney DJ. Fine-mapping cellular QTLs with RASQUAL and ATAC-seq. Nat Genet. 2015. doi: <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1038/ng.3467" target="_blank" class="ext-link"><span class="RefSource">10.1038/ng.3467</span></a></span>.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR133" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Storey JD, Tibshirani R. Statistical significance for genome-wide studies. Proc Natl Acad Sci U S A. 2003;100:9440–5.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12883005"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC170937"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.1530509100"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Statistical%20significance%20for%20genome-wide%20studies&amp;author=JD.%20Storey&amp;author=R.%20Tibshirani&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=100&amp;pages=9440-5&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR134" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stranger BE, Forrest MS, Clark AG, Minichiello MJ, Deutsch S, Lyle R, et al. Genome-wide associations of gene expression variation in humans. PLoS Genet. 2005;1:e78.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16362079"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1315281"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pgen.0010078"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome-wide%20associations%20of%20gene%20expression%20variation%20in%20humans&amp;author=BE.%20Stranger&amp;author=MS.%20Forrest&amp;author=AG.%20Clark&amp;author=MJ.%20Minichiello&amp;author=S.%20Deutsch&amp;author=R.%20Lyle&amp;journal=PLoS%20Genet&amp;volume=1&amp;pages=e78&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR135" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Raj T, Rothamel K, Mostafavi S, Ye C, Lee MN, Replogle JM, et al. Polarization of the effects of autoimmune and neurodegenerative risk alleles in leukocytes. Science. 2014;344:519–23.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24786080"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4910825"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1126/science.1249547"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Polarization%20of%20the%20effects%20of%20autoimmune%20and%20neurodegenerative%20risk%20alleles%20in%20leukocytes&amp;author=T.%20Raj&amp;author=K.%20Rothamel&amp;author=S.%20Mostafavi&amp;author=C.%20Ye&amp;author=MN.%20Lee&amp;author=JM.%20Replogle&amp;journal=Science&amp;volume=344&amp;pages=519-23&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR136" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Shabalin A. Matrix eQTL: ultra fast eQTL analysis via large matrix operations. Bioinformatics. 2012;28:1353–8.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22492648"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3348564"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/bts163"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Matrix%20eQTL%3A%20ultra%20fast%20eQTL%20analysis%20via%20large%20matrix%20operations&amp;author=A.%20Shabalin&amp;journal=Bioinformatics&amp;volume=28&amp;pages=1353-8&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR137" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Louhimo R, Lepikhova T, Monni O, Hautaniemi S. Comparative analysis of algorithms for integration of copy number and expression data. Nat Methods. 2012;9:351–5.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22327835"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nmeth.1893"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20algorithms%20for%20integration%20of%20copy%20number%20and%20expression%20data&amp;author=R.%20Louhimo&amp;author=T.%20Lepikhova&amp;author=O.%20Monni&amp;author=S.%20Hautaniemi&amp;journal=Nat%20Methods&amp;volume=9&amp;pages=351-5&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR138" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kim JH, Dhanasekaran SM, Prensner JR, Cao X, Robinson D, Kalyana-Sundaram S, et al. Deep sequencing reveals distinct patterns of DNA methylation in prostate cancer. Genome Res. 2011;21:1028–41.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21724842"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3129246"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.119347.110"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Deep%20sequencing%20reveals%20distinct%20patterns%20of%20DNA%20methylation%20in%20prostate%20cancer&amp;author=JH.%20Kim&amp;author=SM.%20Dhanasekaran&amp;author=JR.%20Prensner&amp;author=X.%20Cao&amp;author=D.%20Robinson&amp;author=S.%20Kalyana-Sundaram&amp;journal=Genome%20Res&amp;volume=21&amp;pages=1028-41&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR139" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Li JL, Mazar J, Zhong C, Faulkner GJ, Govindarajan SS, Zhang Z, et al. Genome-wide methylated CpG island profiles of melanoma cells reveal a melanoma coregulation network. Sci Rep. 2013;3:2962.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24129253"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3797435"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genome-wide%20methylated%20CpG%20island%20profiles%20of%20melanoma%20cells%20reveal%20a%20melanoma%20coregulation%20network&amp;author=JL.%20Li&amp;author=J.%20Mazar&amp;author=C.%20Zhong&amp;author=GJ.%20Faulkner&amp;author=SS.%20Govindarajan&amp;author=Z.%20Zhang&amp;journal=Sci%20Rep&amp;volume=3&amp;pages=2962&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR140" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Xie L, Weichel B, Ohm JE, Zhang K. An integrative analysis of DNA methylation and RNA-Seq data for human heart, kidney and liver. BMC Syst Biol. 2011;5 Suppl 3:S4.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22784623"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3287572"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1752-0509-5-S3-S4"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=An%20integrative%20analysis%20of%20DNA%20methylation%20and%20RNA-Seq%20data%20for%20human%20heart%2C%20kidney%20and%20liver&amp;author=L.%20Xie&amp;author=B.%20Weichel&amp;author=JE.%20Ohm&amp;author=K.%20Zhang&amp;journal=BMC%20Syst%20Biol&amp;volume=5&amp;issue=Suppl%203&amp;pages=S4&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR141" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Van Eijk KR, de Jong S, Boks MP, Langeveld T, Colas F, Veldink JH, et al. Genetic analysis of DNA methylation and gene expression levels in whole blood of healthy human subjects. BMC Genomics. 2012;13:636.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23157493"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3583143"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2164-13-636"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20analysis%20of%20DNA%20methylation%20and%20gene%20expression%20levels%20in%20whole%20blood%20of%20healthy%20human%20subjects&amp;author=KR.%20Eijk&amp;author=S.%20Jong&amp;author=MP.%20Boks&amp;author=T.%20Langeveld&amp;author=F.%20Colas&amp;author=JH.%20Veldink&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=636&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR142" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liu Y, Aryee MJ, Padyukov L, Fallin MD, Hesselberg E, Runarsson A, et al. Epigenome-wide association data implicate DNA methylation as an intermediary of genetic risk in rheumatoid arthritis. Nat Biotechnol. 2013;31:142–7.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23334450"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3598632"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.2487"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Epigenome-wide%20association%20data%20implicate%20DNA%20methylation%20as%20an%20intermediary%20of%20genetic%20risk%20in%20rheumatoid%20arthritis&amp;author=Y.%20Liu&amp;author=MJ.%20Aryee&amp;author=L.%20Padyukov&amp;author=MD.%20Fallin&amp;author=E.%20Hesselberg&amp;author=A.%20Runarsson&amp;journal=Nat%20Biotechnol&amp;volume=31&amp;pages=142-7&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR143" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yeang C-H. An integrated analysis of molecular aberrations in NCI-60 cell lines. BMC Bioinformatics. 2010;11:495.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20925909"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2984587"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=An%20integrated%20analysis%20of%20molecular%20aberrations%20in%20NCI-60%20cell%20lines&amp;author=C-H.%20Yeang&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=495&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR144" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jeong J, Li L, Liu Y, Nephew KP, Huang YHM, Shen C. An empirical Bayes model for gene expression and methylation profiles in antiestrogen resistant breast cancer. BMC Med Genomics. 2010;3:55.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21108837"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3003621"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1755-8794-3-55"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=An%20empirical%20Bayes%20model%20for%20gene%20expression%20and%20methylation%20profiles%20in%20antiestrogen%20resistant%20breast%20cancer&amp;author=J.%20Jeong&amp;author=L.%20Li&amp;author=Y.%20Liu&amp;author=KP.%20Nephew&amp;author=YHM.%20Huang&amp;author=C.%20Shen&amp;journal=BMC%20Med%20Genomics&amp;volume=3&amp;pages=55&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR145" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jiao Y, Widschwendter M, Teschendorff AE. A systems-level integrative framework for genome-wide DNA methylation and gene expression data identifies differential gene expression modules under epigenetic control. Bioinformatics. 2014;30:2360–6.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24794928"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btu316"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20systems-level%20integrative%20framework%20for%20genome-wide%20DNA%20methylation%20and%20gene%20expression%20data%20identifies%20differential%20gene%20expression%20modules%20under%20epigenetic%20control&amp;author=Y.%20Jiao&amp;author=M.%20Widschwendter&amp;author=AE.%20Teschendorff&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2360-6&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR146" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang S, Sun H, Ma J, Zang C, Wang C, Wang J, et al. Target analysis by integration of transcriptome and ChIP-seq data with BETA. Nat Protoc. 2013;8:2502–15.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24263090"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4135175"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nprot.2013.150"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Target%20analysis%20by%20integration%20of%20transcriptome%20and%20ChIP-seq%20data%20with%20BETA&amp;author=S.%20Wang&amp;author=H.%20Sun&amp;author=J.%20Ma&amp;author=C.%20Zang&amp;author=C.%20Wang&amp;author=J.%20Wang&amp;journal=Nat%20Protoc&amp;volume=8&amp;pages=2502-15&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR147" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Roadmap Epigenomics Consortium, Kundaje A, Meuleman W, Ernst J, Bilenky M, Yen A, et al. Integrative analysis of 111 reference human epigenomes. Nature. 2015;518:317–30.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4530010"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature14248"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Integrative%20analysis%20of%20111%20reference%20human%20epigenomes&amp;author=A.%20Kundaje&amp;author=W.%20Meuleman&amp;author=J.%20Ernst&amp;author=M.%20Bilenky&amp;author=A.%20Yen&amp;journal=Nature&amp;volume=518&amp;pages=317-30&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR148" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Madrigal P, Krajewski P. Uncovering correlated variability in epigenomic datasets using the Karhunen-Loeve transform. BioData Min. 2015;8:20.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26140054"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4488123"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s13040-015-0051-7"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Uncovering%20correlated%20variability%20in%20epigenomic%20datasets%20using%20the%20Karhunen-Loeve%20transform&amp;author=P.%20Madrigal&amp;author=P.%20Krajewski&amp;journal=BioData%20Min&amp;volume=8&amp;pages=20&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR149" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Angelini C, Costa V. Understanding gene regulatory mechanisms by integrating ChIP-seq and RNA-seq data: statistical solutions to biological problems. Front Cell Dev Biol. 2014;2:51.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25364758"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4207007"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3389/fcell.2014.00051"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Understanding%20gene%20regulatory%20mechanisms%20by%20integrating%20ChIP-seq%20and%20RNA-seq%20data%3A%20statistical%20solutions%20to%20biological%20problems&amp;author=C.%20Angelini&amp;author=V.%20Costa&amp;journal=Front%20Cell%20Dev%20Biol&amp;volume=2&amp;pages=51&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR150" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Neph S, Stergachis AB, Reynolds A, Sandstrom R, Borenstein E, Stamatoyannopoulos JA. Circuitry and dynamics of human transcription factor regulatory networks. Cell. 2012;150:1274–86.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22959076"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3679407"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.cell.2012.04.040"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Circuitry%20and%20dynamics%20of%20human%20transcription%20factor%20regulatory%20networks&amp;author=S.%20Neph&amp;author=AB.%20Stergachis&amp;author=A.%20Reynolds&amp;author=R.%20Sandstrom&amp;author=E.%20Borenstein&amp;author=JA.%20Stamatoyannopoulos&amp;journal=Cell&amp;volume=150&amp;pages=1274-86&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR151" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dweep H, Sticht C, Pandey P, Gretz N. miRWalk - database: prediction of possible miRNA binding sites by ‘walking’ the genes of 3 genomes. J Biomed Inform. 2011;44:839–47.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21605702"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.jbi.2011.05.002"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=miRWalk%20-%20database%3A%20prediction%20of%20possible%20miRNA%20binding%20sites%20by%20%E2%80%98walking%E2%80%99%20the%20genes%20of%203%20genomes&amp;author=H.%20Dweep&amp;author=C.%20Sticht&amp;author=P.%20Pandey&amp;author=N.%20Gretz&amp;journal=J%20Biomed%20Inform&amp;volume=44&amp;pages=839-47&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR152" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Griffiths-Jones S, Saini HK, van Dongen S, Enright AJ. miRBase: tools for microRNA genomics. Nucleic Acids Res. 2008;36:D154–8.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17991681"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkm952"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=miRBase%3A%20tools%20for%20microRNA%20genomics&amp;author=S.%20Griffiths-Jones&amp;author=HK.%20Saini&amp;author=S.%20Dongen&amp;author=AJ.%20Enright&amp;journal=Nucleic%20Acids%20Res&amp;volume=36&amp;pages=D154-8&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR153" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wu X, Watson M. CORNA: testing gene lists for regulation by microRNAs. Bioinformatics. 2009;25:832–3.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19181683"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2654799"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btp059"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=CORNA%3A%20testing%20gene%20lists%20for%20regulation%20by%20microRNAs&amp;author=X.%20Wu&amp;author=M.%20Watson&amp;journal=Bioinformatics&amp;volume=25&amp;pages=832-3&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR154" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lee H, Yang Y, Chae H, Nam S, Choi D, Tangchaisin P, et al. BioVLAB-MMIA: a cloud environment for microRNA and mRNA integrated analysis (MMIA) on Amazon EC2. IEEE Trans Nanobiosci. 2012;11:266–72.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1109/TNB.2011.2159621"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=BioVLAB-MMIA%3A%20a%20cloud%20environment%20for%20microRNA%20and%20mRNA%20integrated%20analysis%20%28MMIA%29%20on%20Amazon%20EC2&amp;author=H.%20Lee&amp;author=Y.%20Yang&amp;author=H.%20Chae&amp;author=S.%20Nam&amp;author=D.%20Choi&amp;author=P.%20Tangchaisin&amp;journal=IEEE%20Trans%20Nanobiosci&amp;volume=11&amp;pages=266-72&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR155" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nam S, Li M, Choi K, Balch C, Kim S, Nephew KP. MicroRNA and mRNA integrated analysis (MMIA): a web tool for examining biological functions of microRNA expression. Nucleic Acids Res. 2009;37:W356–62.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19420067"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2703907"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkp294"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MicroRNA%20and%20mRNA%20integrated%20analysis%20%28MMIA%29%3A%20a%20web%20tool%20for%20examining%20biological%20functions%20of%20microRNA%20expression&amp;author=S.%20Nam&amp;author=M.%20Li&amp;author=K.%20Choi&amp;author=C.%20Balch&amp;author=S.%20Kim&amp;author=KP.%20Nephew&amp;journal=Nucleic%20Acids%20Res&amp;volume=37&amp;pages=W356-62&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR156" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sales G, Coppe A, Bisognin A, Bortoluzzi S, Romualdi C. MAGIA, a web-based tool for miRNA and Genes Integrated Analysis. Nucleic Acids Res. 2010;38:W352–9.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20484379"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2896126"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkq423"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MAGIA%2C%20a%20web-based%20tool%20for%20miRNA%20and%20Genes%20Integrated%20Analysis&amp;author=G.%20Sales&amp;author=A.%20Coppe&amp;author=A.%20Bisognin&amp;author=S.%20Bortoluzzi&amp;author=C.%20Romualdi&amp;journal=Nucleic%20Acids%20Res&amp;volume=38&amp;pages=W352-9&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR157" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Icay K, Chen P, Cervera C, Lehtonen R, Hautaniemi S. SePIA: RNA and smallRNA-sequence processing, integration, and analysis. 2015. <span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://anduril.org/sepia" target="_blank" class="ext-link"><span class="RefSource">http://anduril.org/sepia</span></a></span>. Accessed 6 Jan 2016.</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR158" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">de Sousa AR, Penalva LO, Marcotte EM, Vogel C. Global signatures of protein and mRNA expression levels. Mol Biosyst. 2009;5:1512–26.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Global%20signatures%20of%20protein%20and%20mRNA%20expression%20levels&amp;author=AR.%20Sousa&amp;author=LO.%20Penalva&amp;author=EM.%20Marcotte&amp;author=C.%20Vogel&amp;journal=Mol%20Biosyst&amp;volume=5&amp;pages=1512-26&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR159" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Vogel C, Marcotte EM. Insights into the regulation of protein abundance from proteomic and transcriptomic analyses. Nat Rev Genet. 2012;13:227–32.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22411467"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3654667"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Insights%20into%20the%20regulation%20of%20protein%20abundance%20from%20proteomic%20and%20transcriptomic%20analyses&amp;author=C.%20Vogel&amp;author=EM.%20Marcotte&amp;journal=Nat%20Rev%20Genet&amp;volume=13&amp;pages=227-32&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR160" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Low TY, van Heesch S, van den Toorn H, Giansanti P, Cristobal A, Toonen P. Quantitative and qualitative proteome characteristics extracted from in-depth integrated genomics and proteomics analysis. Cell Rep. 2013;5:1469–78.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24290761"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.celrep.2013.10.041"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Quantitative%20and%20qualitative%20proteome%20characteristics%20extracted%20from%20in-depth%20integrated%20genomics%20and%20proteomics%20analysis&amp;author=TY.%20Low&amp;author=S.%20Heesch&amp;author=H.%20Toorn&amp;author=P.%20Giansanti&amp;author=A.%20Cristobal&amp;author=P.%20Toonen&amp;journal=Cell%20Rep&amp;volume=5&amp;pages=1469-78&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR161" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Suhre K, Schmitt-Kopplin P. MassTRIX: mass translator into pathways. Nucleic Acids Res. 2008;36(Web Server issue):W481–4.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18442993"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2447776"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkn194"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MassTRIX%3A%20mass%20translator%20into%20pathways&amp;author=K.%20Suhre&amp;author=P.%20Schmitt-Kopplin&amp;journal=Nucleic%20Acids%20Res&amp;volume=36&amp;issue=Web%20Server%20issue&amp;pages=W481-4&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR162" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">García-Alcalde F, García-López F, Dopazo J, Conesa A. Paintomics: a web based tool for the joint visualization of transcriptomics and metabolomics data. Bioinformatics. 2011;27:137–9.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21098431"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btq594"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Paintomics%3A%20a%20web%20based%20tool%20for%20the%20joint%20visualization%20of%20transcriptomics%20and%20metabolomics%20data&amp;author=F.%20Garc%C3%ADa-Alcalde&amp;author=F.%20Garc%C3%ADa-L%C3%B3pez&amp;author=J.%20Dopazo&amp;author=A.%20Conesa&amp;journal=Bioinformatics&amp;volume=27&amp;pages=137-9&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR163" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rohn H, Junker A, Hartmann A, Grafahrend-Belau E, Treutler H, Klapperstück M, et al. VANTED v2: a framework for systems biology applications. BMC Syst Biol. 2012;6:139.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23140568"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3610154"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1752-0509-6-139"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=VANTED%20v2%3A%20a%20framework%20for%20systems%20biology%20applications&amp;author=H.%20Rohn&amp;author=A.%20Junker&amp;author=A.%20Hartmann&amp;author=E.%20Grafahrend-Belau&amp;author=H.%20Treutler&amp;author=M.%20Klapperst%C3%BCck&amp;journal=BMC%20Syst%20Biol&amp;volume=6&amp;pages=139&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR164" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tuncbag N, McCallum S, Huang SS, Fraenkel E. SteinerNet: a web server for integrating ‘omic’ data to discover hidden components of response pathways. Nucleic Acids Res. 2012;40:W505–9.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22638579"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3394335"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gks445"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=SteinerNet%3A%20a%20web%20server%20for%20integrating%20%E2%80%98omic%E2%80%99%20data%20to%20discover%20hidden%20components%20of%20response%20pathways&amp;author=N.%20Tuncbag&amp;author=S.%20McCallum&amp;author=SS.%20Huang&amp;author=E.%20Fraenkel&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=W505-9&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR165" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang S, Li Q, Liu J, Zhou XJ. A novel computational framework for simultaneous integration of multiple types of genomic data to identify microRNA-gene regulatory modules. Bioinformatics. 2011;27:i401–9.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21685098"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3117336"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btr206"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20novel%20computational%20framework%20for%20simultaneous%20integration%20of%20multiple%20types%20of%20genomic%20data%20to%20identify%20microRNA-gene%20regulatory%20modules&amp;author=S.%20Zhang&amp;author=Q.%20Li&amp;author=J.%20Liu&amp;author=XJ.%20Zhou&amp;journal=Bioinformatics&amp;volume=27&amp;pages=i401-9&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR166" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Le H-S, Bar-Joseph Z. Integrating sequence, expression and interaction data to determine condition-specific miRNA regulation. Bioinformatics. 2013;29:i89–97.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23813013"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3694655"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btt231"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Integrating%20sequence%2C%20expression%20and%20interaction%20data%20to%20determine%20condition-specific%20miRNA%20regulation&amp;author=H-S.%20Le&amp;author=Z.%20Bar-Joseph&amp;journal=Bioinformatics&amp;volume=29&amp;pages=i89-97&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR167" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sass S, Buettner F, Mueller NS, Theis FJ. A modular framework for gene set analysis integrating multilevel omics data. Nucleic Acids Res. 2013;41:9622–33.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23975194"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3834824"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkt752"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20modular%20framework%20for%20gene%20set%20analysis%20integrating%20multilevel%20omics%20data&amp;author=S.%20Sass&amp;author=F.%20Buettner&amp;author=NS.%20Mueller&amp;author=FJ.%20Theis&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=9622-33&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR168" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kuo TC, Tian TF, Tseng YJ. 3Omics: a web-based systems biology tool for analysis, integration and visualization of human transcriptomic, proteomic and metabolomic data. BMC Syst Biol. 2013;7:64.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23875761"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3723580"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1752-0509-7-64"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=3Omics%3A%20a%20web-based%20systems%20biology%20tool%20for%20analysis%2C%20integration%20and%20visualization%20of%20human%20transcriptomic%2C%20proteomic%20and%20metabolomic%20data&amp;author=TC.%20Kuo&amp;author=TF.%20Tian&amp;author=YJ.%20Tseng&amp;journal=BMC%20Syst%20Biol&amp;volume=7&amp;pages=64&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR169" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ovaska K, Laakso M, Haapa-Paananen S, Louhimo R, Chen P, Aittomäki V, et al. Large-scale data integration framework provides a comprehensive view on glioblastoma multiforme. Genome Med. 2010;2:65.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20822536"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3092116"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/gm186"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Large-scale%20data%20integration%20framework%20provides%20a%20comprehensive%20view%20on%20glioblastoma%20multiforme&amp;author=K.%20Ovaska&amp;author=M.%20Laakso&amp;author=S.%20Haapa-Paananen&amp;author=R.%20Louhimo&amp;author=P.%20Chen&amp;author=V.%20Aittom%C3%A4ki&amp;journal=Genome%20Med&amp;volume=2&amp;pages=65&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR170" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Goecks J, Nekrutenko A, Taylor J. Galaxy: a comprehensive approach for supporting accessible, reproducible, and transparent computational research in the life sciences. Genome Biol. 2010;11:R86.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20738864"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2945788"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/gb-2010-11-8-r86"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Galaxy%3A%20a%20comprehensive%20approach%20for%20supporting%20accessible%2C%20reproducible%2C%20and%20transparent%20computational%20research%20in%20the%20life%20sciences&amp;author=J.%20Goecks&amp;author=A.%20Nekrutenko&amp;author=J.%20Taylor&amp;journal=Genome%20Biol&amp;volume=11&amp;pages=R86&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR171" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kallio MA, Tuimala JT, Hupponen T, Klemelä P, Gentile M, Scheinin I, et al. Chipster: user-friendly analysis software for microarray and other high-throughput data. BMC Genomics. 2011;12:507.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21999641"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3215701"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2164-12-507"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Chipster%3A%20user-friendly%20analysis%20software%20for%20microarray%20and%20other%20high-throughput%20data&amp;author=MA.%20Kallio&amp;author=JT.%20Tuimala&amp;author=T.%20Hupponen&amp;author=P.%20Klemel%C3%A4&amp;author=M.%20Gentile&amp;author=I.%20Scheinin&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=507&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR172" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pang CNI, Tay AP, Aya C. Tools to covisualize and coanalyze proteomic data with genomes and transcriptomes: validation of genes and alternative mRNA splicing. J Proteome Res. 2014;13:84–98.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24152167"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1021/pr400820p"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Tools%20to%20covisualize%20and%20coanalyze%20proteomic%20data%20with%20genomes%20and%20transcriptomes%3A%20validation%20of%20genes%20and%20alternative%20mRNA%20splicing&amp;author=CNI.%20Pang&amp;author=AP.%20Tay&amp;author=C.%20Aya&amp;journal=J%20Proteome%20Res&amp;volume=13&amp;pages=84-98&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR173" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ramsköld D, Luo S, Wang YC, Li R, Deng Q, Faridani OR, et al. Full-length mRNA-Seq from single-cell levels of RNA and individual circulating tumor cells. Nat Biotechnol. 2012;30:777–82.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22820318"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3467340"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.2282"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Full-length%20mRNA-Seq%20from%20single-cell%20levels%20of%20RNA%20and%20individual%20circulating%20tumor%20cells&amp;author=D.%20Ramsk%C3%B6ld&amp;author=S.%20Luo&amp;author=YC.%20Wang&amp;author=R.%20Li&amp;author=Q.%20Deng&amp;author=OR.%20Faridani&amp;journal=Nat%20Biotechnol&amp;volume=30&amp;pages=777-82&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR174" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Picelli S, Björklund ÅK, Faridani OR, Sagasser S, Winberg G, Sandberg R. Smart-seq2 for sensitive full-length transcriptome profiling in single cells. Nat Methods. 2013;10:1096–8.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24056875"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nmeth.2639"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Smart-seq2%20for%20sensitive%20full-length%20transcriptome%20profiling%20in%20single%20cells&amp;author=S.%20Picelli&amp;author=%C3%85K.%20Bj%C3%B6rklund&amp;author=OR.%20Faridani&amp;author=S.%20Sagasser&amp;author=G.%20Winberg&amp;author=R.%20Sandberg&amp;journal=Nat%20Methods&amp;volume=10&amp;pages=1096-8&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR175" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Macosko EZ, Basu A, Satija R, Nemesh J, Shekhar K, Goldman M, et al. Highly parallel genome-wide expression profiling of individual cells using nanoliter droplets. Cell. 2015;161:1202–14.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26000488"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4481139"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.cell.2015.05.002"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Highly%20parallel%20genome-wide%20expression%20profiling%20of%20individual%20cells%20using%20nanoliter%20droplets&amp;author=EZ.%20Macosko&amp;author=A.%20Basu&amp;author=R.%20Satija&amp;author=J.%20Nemesh&amp;author=K.%20Shekhar&amp;author=M.%20Goldman&amp;journal=Cell&amp;volume=161&amp;pages=1202-14&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR176" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Marinov GK, Williams BA, McCue K, Schroth GP, Gertz J, Myers RM, et al. From single-cell to cell-pool transcriptomes: stochasticity in gene expression and RNA splicing. Genome Res. 2014;24:496–510.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24299736"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3941114"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.161034.113"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=From%20single-cell%20to%20cell-pool%20transcriptomes%3A%20stochasticity%20in%20gene%20expression%20and%20RNA%20splicing&amp;author=GK.%20Marinov&amp;author=BA.%20Williams&amp;author=K.%20McCue&amp;author=GP.%20Schroth&amp;author=J.%20Gertz&amp;author=RM.%20Myers&amp;journal=Genome%20Res&amp;volume=24&amp;pages=496-510&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR177" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Islam S, Zeisel A, Joost S, La Manno G, Zajac P, Kasper M, et al. Quantitative single-cell RNA-seq with unique molecular identifiers. Nat Methods. 2014;11:163–6.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24363023"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nmeth.2772"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Quantitative%20single-cell%20RNA-seq%20with%20unique%20molecular%20identifiers&amp;author=S.%20Islam&amp;author=A.%20Zeisel&amp;author=S.%20Joost&amp;author=G.%20Manno&amp;author=P.%20Zajac&amp;author=M.%20Kasper&amp;journal=Nat%20Methods&amp;volume=11&amp;pages=163-6&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR178" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kivioja T, Vähärautio A, Karlsson K, Bonke M, Enge M, Linnarsson S, et al. Counting absolute numbers of molecules using unique molecular identifiers. Nat Methods. 2011;9:72–4.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22101854"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nmeth.1778"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Counting%20absolute%20numbers%20of%20molecules%20using%20unique%20molecular%20identifiers&amp;author=T.%20Kivioja&amp;author=A.%20V%C3%A4h%C3%A4rautio&amp;author=K.%20Karlsson&amp;author=M.%20Bonke&amp;author=M.%20Enge&amp;author=S.%20Linnarsson&amp;journal=Nat%20Methods&amp;volume=9&amp;pages=72-4&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR179" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Brennecke P, Anders S, Kim JK, Kołodziejczyk AA, Zhang X, Proserpio V, et al. Accounting for technical noise in single-cell RNA-seq experiments. Nat Methods. 2013;10:1093–5.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24056876"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nmeth.2645"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Accounting%20for%20technical%20noise%20in%20single-cell%20RNA-seq%20experiments&amp;author=P.%20Brennecke&amp;author=S.%20Anders&amp;author=JK.%20Kim&amp;author=AA.%20Ko%C5%82odziejczyk&amp;author=X.%20Zhang&amp;author=V.%20Proserpio&amp;journal=Nat%20Methods&amp;volume=10&amp;pages=1093-5&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR180" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stegle O, Teichmann SA, Marioni JC. Computational and analytical challenges in single-cell transcriptomics. Nat Rev Genet. 2015;16:133–45.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25628217"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nrg3833"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Computational%20and%20analytical%20challenges%20in%20single-cell%20transcriptomics&amp;author=O.%20Stegle&amp;author=SA.%20Teichmann&amp;author=JC.%20Marioni&amp;journal=Nat%20Rev%20Genet&amp;volume=16&amp;pages=133-45&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR181" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Trapnell C, Cacchiarelli D. The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells. Nat Biotechnol. 2014;32:381–6.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24658644"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4122333"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.2859"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20dynamics%20and%20regulators%20of%20cell%20fate%20decisions%20are%20revealed%20by%20pseudotemporal%20ordering%20of%20single%20cells&amp;author=C.%20Trapnell&amp;author=D.%20Cacchiarelli&amp;journal=Nat%20Biotechnol&amp;volume=32&amp;pages=381-6&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR182" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lorthongpanich C, Cheow LF, Balu S, Quake SR, Knowles BB, Burkholder WF, et al. Single-cell DNA-methylation analysis reveals epigenetic chimerism in preimplantation embryos. Science. 2013;341:1110–2.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24009393"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1126/science.1240617"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Single-cell%20DNA-methylation%20analysis%20reveals%20epigenetic%20chimerism%20in%20preimplantation%20embryos&amp;author=C.%20Lorthongpanich&amp;author=LF.%20Cheow&amp;author=S.%20Balu&amp;author=SR.%20Quake&amp;author=BB.%20Knowles&amp;author=WF.%20Burkholder&amp;journal=Science&amp;volume=341&amp;pages=1110-2&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR183" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Buenrostro JD, Wu B, Litzenburger UM, Ruff D, Gonzales ML, Snyder MP, et al. Single-cell chromatin accessibility reveals principles of regulatory variation. Nature. 2015;523:486–90.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26083756"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4685948"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nature14590"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Single-cell%20chromatin%20accessibility%20reveals%20principles%20of%20regulatory%20variation&amp;author=JD.%20Buenrostro&amp;author=B.%20Wu&amp;author=UM.%20Litzenburger&amp;author=D.%20Ruff&amp;author=ML.%20Gonzales&amp;author=MP.%20Snyder&amp;journal=Nature&amp;volume=523&amp;pages=486-90&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR184" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Cusanovich DA, Daza R, Adey A, Pliner HA, Christiansen L, Gunderson KL, et al. Multiplex single-cell profiling of chromatin accessibility by combinatorial cellular indexing. Science. 2015;348:910–4.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25953818"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4836442"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1126/science.aab1601"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Multiplex%20single-cell%20profiling%20of%20chromatin%20accessibility%20by%20combinatorial%20cellular%20indexing&amp;author=DA.%20Cusanovich&amp;author=R.%20Daza&amp;author=A.%20Adey&amp;author=HA.%20Pliner&amp;author=L.%20Christiansen&amp;author=KL.%20Gunderson&amp;journal=Science&amp;volume=348&amp;pages=910-4&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR185" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tilgner H, Jahanbani F, Blauwkamp T, Moshrefi A, Jaeger E, Chen F, et al. Comprehensive transcriptome analysis using synthetic long-read sequencing reveals molecular co-association of distant splicing events. Nat Biotechnol. 2015;33:736–42.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25985263"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4832928"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.3242"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comprehensive%20transcriptome%20analysis%20using%20synthetic%20long-read%20sequencing%20reveals%20molecular%20co-association%20of%20distant%20splicing%20events&amp;author=H.%20Tilgner&amp;author=F.%20Jahanbani&amp;author=T.%20Blauwkamp&amp;author=A.%20Moshrefi&amp;author=E.%20Jaeger&amp;author=F.%20Chen&amp;journal=Nat%20Biotechnol&amp;volume=33&amp;pages=736-42&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR186" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Au KF, Sebastiano V, Afshar PT, Durruthy JD, Lee L, Williams BA, et al. Characterization of the human ESC transcriptome by hybrid sequencing. Proc Natl Acad Sci U S A. 2013;110:E4821–30.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24282307"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3864310"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.1320101110"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20human%20ESC%20transcriptome%20by%20hybrid%20sequencing&amp;author=KF.%20Au&amp;author=V.%20Sebastiano&amp;author=PT.%20Afshar&amp;author=JD.%20Durruthy&amp;author=L.%20Lee&amp;author=BA.%20Williams&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=110&amp;pages=E4821-30&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR187" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tilgner H, Grubert F, Sharon D, Snyder MP. Defining a personal, allele-specific, and single-molecule long-read transcriptome. Proc Natl Acad Sci U S A. 2014;111:9869–74.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24961374"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4103364"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.1400447111"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Defining%20a%20personal%2C%20allele-specific%2C%20and%20single-molecule%20long-read%20transcriptome&amp;author=H.%20Tilgner&amp;author=F.%20Grubert&amp;author=D.%20Sharon&amp;author=MP.%20Snyder&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=111&amp;pages=9869-74&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR188" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Au KF, Underwood JG, Lee L, Wong WH. Improving PacBio long read accuracy by short read alignment. PLoS One. 2012;7:e46679.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23056399"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3464235"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0046679"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Improving%20PacBio%20long%20read%20accuracy%20by%20short%20read%20alignment&amp;author=KF.%20Au&amp;author=JG.%20Underwood&amp;author=L.%20Lee&amp;author=WH.%20Wong&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e46679&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR189" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hansen KD, Wu Z, Irizarry RA, Leek JT. Sequencing technology does not eliminate biological variability. Nat Biotechnol. 2011;29:572–3.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21747377"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3137276"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.1910"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Sequencing%20technology%20does%20not%20eliminate%20biological%20variability&amp;author=KD.%20Hansen&amp;author=Z.%20Wu&amp;author=RA.%20Irizarry&amp;author=JT.%20Leek&amp;journal=Nat%20Biotechnol&amp;volume=29&amp;pages=572-3&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR190" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hart SN, Therneau TM, Zhang Y, Poland GA, Kocher JP. Calculating sample size estimates for RNA sequencing data. J Comput Biol. 2013;20:970–8.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23961961"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3842884"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1089/cmb.2012.0283"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Calculating%20sample%20size%20estimates%20for%20RNA%20sequencing%20data&amp;author=SN.%20Hart&amp;author=TM.%20Therneau&amp;author=Y.%20Zhang&amp;author=GA.%20Poland&amp;author=JP.%20Kocher&amp;journal=J%20Comput%20Biol&amp;volume=20&amp;pages=970-8&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR191" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Busby MA, Stewart C, Miller CA, Grzeda KR, Marth GT. Scotty: a web tool for designing RNA-Seq experiments to measure differential gene expression. Bioinformatics. 2013;29:656–7.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23314327"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3582267"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btt015"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Scotty%3A%20a%20web%20tool%20for%20designing%20RNA-Seq%20experiments%20to%20measure%20differential%20gene%20expression&amp;author=MA.%20Busby&amp;author=C.%20Stewart&amp;author=CA.%20Miller&amp;author=KR.%20Grzeda&amp;author=GT.%20Marth&amp;journal=Bioinformatics&amp;volume=29&amp;pages=656-7&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR192" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Oshlack A, Wakefield MJ. Transcript length bias in RNA-seq data confounds systems biology. Biol Direct. 2009;4:14.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19371405"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2678084"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1745-6150-4-14"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Transcript%20length%20bias%20in%20RNA-seq%20data%20confounds%20systems%20biology&amp;author=A.%20Oshlack&amp;author=MJ.%20Wakefield&amp;journal=Biol%20Direct&amp;volume=4&amp;pages=14&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR193" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Noble WS. How does multiple testing correction work? Nat Biotechnol. 2009;27:1135–7.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20010596"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2907892"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt1209-1135"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=How%20does%20multiple%20testing%20correction%20work%3F&amp;author=WS.%20Noble&amp;journal=Nat%20Biotechnol&amp;volume=27&amp;pages=1135-7&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR194" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Robinson DG, Storey JD. subSeq: determining appropriate sequencing depth through efficient read subsampling. Bioinformatics. 2014;30:3424–6.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25189781"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4296149"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btu552"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=subSeq%3A%20determining%20appropriate%20sequencing%20depth%20through%20efficient%20read%20subsampling&amp;author=DG.%20Robinson&amp;author=JD.%20Storey&amp;journal=Bioinformatics&amp;volume=30&amp;pages=3424-6&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR195" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Liu Y, Zhou J, White KP. RNA-seq differential expression studies: more sequence or more replication? Bioinformatics. 2013;30:301–4.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24319002"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3904521"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btt688"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=RNA-seq%20differential%20expression%20studies%3A%20more%20sequence%20or%20more%20replication%3F&amp;author=Y.%20Liu&amp;author=J.%20Zhou&amp;author=KP.%20White&amp;journal=Bioinformatics&amp;volume=30&amp;pages=301-4&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR196" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">SEQC/MAQC-III Consortium. A comprehensive assessment of RNA-seq accuracy, reproducibility and information content by the Sequencing Quality Control Consortium. Nat Biotechnol. 2014;32:903–14.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nbt.2957"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20comprehensive%20assessment%20of%20RNA-seq%20accuracy%2C%20reproducibility%20and%20information%20content%20by%20the%20Sequencing%20Quality%20Control%20Consortium&amp;journal=Nat%20Biotechnol&amp;volume=32&amp;pages=903-14&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR197" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jiang L, Schlesinger F, Davis CA, Zhang Y, Li R, Salit M, et al. Synthetic spike-in standards for RNA-seq experiments. Genome Res. 2011;21:1543–51.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21816910"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3166838"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.121095.111"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Synthetic%20spike-in%20standards%20for%20RNA-seq%20experiments&amp;author=L.%20Jiang&amp;author=F.%20Schlesinger&amp;author=CA.%20Davis&amp;author=Y.%20Zhang&amp;author=R.%20Li&amp;author=M.%20Salit&amp;journal=Genome%20Res&amp;volume=21&amp;pages=1543-51&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR198" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kouzine F, Wojtowicz D, Yamane A, Resch W, Kieffer-Kwon KR, Bandle R, et al. Global regulation of promoter melting in naive lymphocytes. Cell. 2013;153:988–99.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23706737"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684982"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.cell.2013.04.033"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Global%20regulation%20of%20promoter%20melting%20in%20naive%20lymphocytes&amp;author=F.%20Kouzine&amp;author=D.%20Wojtowicz&amp;author=A.%20Yamane&amp;author=W.%20Resch&amp;author=KR.%20Kieffer-Kwon&amp;author=R.%20Bandle&amp;journal=Cell&amp;volume=153&amp;pages=988-99&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR199" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Van Dijk EL, Jaszczyszyn Y, Thermes C. Library preparation methods for next-generation sequencing: tone down the bias. Exp Cell Res. 2014;322:12–20.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24440557"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.yexcr.2014.01.008"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Library%20preparation%20methods%20for%20next-generation%20sequencing%3A%20tone%20down%20the%20bias&amp;author=EL.%20Dijk&amp;author=Y.%20Jaszczyszyn&amp;author=C.%20Thermes&amp;journal=Exp%20Cell%20Res&amp;volume=322&amp;pages=12-20&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR200" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Trapnell C, Pachter L, Salzberg SL. TopHat: discovering splice junctions with RNA-Seq. Bioinformatics. 2009;25:1105–11.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19289445"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btp120"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2672628"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=TopHat%3A%20discovering%20splice%20junctions%20with%20RNA-Seq&amp;author=C.%20Trapnell&amp;author=L.%20Pachter&amp;author=SL.%20Salzberg&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1105-11&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR201" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kim D, Pertea G, Trapnell C, Pimentel H, Kelley R, Salzberg SL. TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions. Genome Biol. 2013;14:R36.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23618408"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4053844"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/gb-2013-14-4-r36"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=TopHat2%3A%20accurate%20alignment%20of%20transcriptomes%20in%20the%20presence%20of%20insertions%2C%20deletions%20and%20gene%20fusions&amp;author=D.%20Kim&amp;author=G.%20Pertea&amp;author=C.%20Trapnell&amp;author=H.%20Pimentel&amp;author=R.%20Kelley&amp;author=SL.%20Salzberg&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=R36&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR202" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wu TD, Nacu S. Fast and SNP-tolerant detection of complex variants and splicing in short reads. Bioinformatics. 2010;26:873–81.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20147302"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2844994"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btq057"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Fast%20and%20SNP-tolerant%20detection%20of%20complex%20variants%20and%20splicing%20in%20short%20reads&amp;author=TD.%20Wu&amp;author=S.%20Nacu&amp;journal=Bioinformatics&amp;volume=26&amp;pages=873-81&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR203" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jean G, Kahles A, Sreedharan VT, De Bona F, Rätsch G. RNA-Seq read alignments with PALMapper. Curr Protoc Bioinformatics. 2010;11(6).<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Jean%20G%2C%20Kahles%20A%2C%20Sreedharan%20VT%2C%20De%20Bona%20F%2C%20R%C3%A4tsch%20G.%20RNA-Seq%20read%20alignments%20with%20PALMapper.%20Curr%20Protoc%20Bioinformatics.%202010%3B11%286%29."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR204" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang K, Singh D, Zeng Z, Coleman SJ, Huang Y, Savich GL, et al. MapSplice: accurate mapping of RNA-seq reads for splice junction discovery. Nucleic Acids Res. 2010;38:e178.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20802226"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2952873"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkq622"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MapSplice%3A%20accurate%20mapping%20of%20RNA-seq%20reads%20for%20splice%20junction%20discovery&amp;author=K.%20Wang&amp;author=D.%20Singh&amp;author=Z.%20Zeng&amp;author=SJ.%20Coleman&amp;author=Y.%20Huang&amp;author=GL.%20Savich&amp;journal=Nucleic%20Acids%20Res&amp;volume=38&amp;pages=e178&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR205" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Marco-Sola S, Sammeth M, Guigó R, Ribeca P. The GEM mapper: fast, accurate and versatile alignment by filtration. Nat Methods. 2012;9:1185–8.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23103880"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nmeth.2221"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20GEM%20mapper%3A%20fast%2C%20accurate%20and%20versatile%20alignment%20by%20filtration&amp;author=S.%20Marco-Sola&amp;author=M.%20Sammeth&amp;author=R.%20Guig%C3%B3&amp;author=P.%20Ribeca&amp;journal=Nat%20Methods&amp;volume=9&amp;pages=1185-8&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR206" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhao S, Zhang B. A comprehensive evaluation of ensembl, RefSeq, and UCSC annotations in the context of RNA-seq read mapping and gene quantification. BMC Genomics. 2015;16:97.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25765860"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4339237"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/s12864-015-1308-8"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20comprehensive%20evaluation%20of%20ensembl%2C%20RefSeq%2C%20and%20UCSC%20annotations%20in%20the%20context%20of%20RNA-seq%20read%20mapping%20and%20gene%20quantification&amp;author=S.%20Zhao&amp;author=B.%20Zhang&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=97&amp;publication_year=2015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR207" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Langmead B, Trapnell C, Pop M, Salzberg SL. Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome Biol. 2009;10:R25.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19261174"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2690996"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/gb-2009-10-3-r25"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;author=B.%20Langmead&amp;author=C.%20Trapnell&amp;author=M.%20Pop&amp;author=SL.%20Salzberg&amp;journal=Genome%20Biol&amp;volume=10&amp;pages=R25&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR208" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kvam VM, Liu P, Si Y. A comparison of statistical methods for detecting differentially expressed genes from RNA-seq data. Am J Bot. 2012;99:248–56.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22268221"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3732/ajb.1100340"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20comparison%20of%20statistical%20methods%20for%20detecting%20differentially%20expressed%20genes%20from%20RNA-seq%20data&amp;author=VM.%20Kvam&amp;author=P.%20Liu&amp;author=Y.%20Si&amp;journal=Am%20J%20Bot&amp;volume=99&amp;pages=248-56&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR209" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Robles JA, Qureshi SE, Stephen SJ, Wilson SR, Burden CJ, Taylor JM. Efficient experimental design and analysis strategies for the detection of differential expression using RNA-Sequencing. BMC Genomics. 2012;13:484.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22985019"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3560154"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2164-13-484"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Efficient%20experimental%20design%20and%20analysis%20strategies%20for%20the%20detection%20of%20differential%20expression%20using%20RNA-Sequencing&amp;author=JA.%20Robles&amp;author=SE.%20Qureshi&amp;author=SJ.%20Stephen&amp;author=SR.%20Wilson&amp;author=CJ.%20Burden&amp;author=JM.%20Taylor&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=484&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR210" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nookaew I, Papini M, Pornputtapong N, Scalcinati G, Fagerberg L, Uhlén M, et al. A comprehensive comparison of RNA-Seq-based transcriptome analysis from reads to differential gene expression and cross-comparison with microarrays: a case study in <em class="EmphasisTypeItalic">Saccharomyces cerevisiae</em>. Nucleic Acids Res. 2012;40:10084–97.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22965124"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3488244"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gks804"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20comprehensive%20comparison%20of%20RNA-Seq-based%20transcriptome%20analysis%20from%20reads%20to%20differential%20gene%20expression%20and%20cross-comparison%20with%20microarrays%3A%20a%20case%20study%20in%20Saccharomyces%20cerevisiae%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;author=I.%20Nookaew&amp;author=M.%20Papini&amp;author=N.%20Pornputtapong&amp;author=G.%20Scalcinati&amp;author=L.%20Fagerberg&amp;author=M.%20Uhl%C3%A9n&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=10084-97&amp;publication_year=2012"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR211" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Seyednasrollah F, Rantanen K, Jaakkola P, Elo LL. ROTS: reproducible RNA-seq biomarker detector-prognostic markers for clear cell renal cell cancer. Nucleic Acids Res. 2016;44(1):e1. doi:<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="https://doi.org/10.1093/nar/gkv806" target="_blank" class="ext-link"><span class="RefSource">10.1093/nar/gkv806</span></a></span>.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26264667"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkv806"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=ROTS%3A%20reproducible%20RNA-seq%20biomarker%20detector-prognostic%20markers%20for%20clear%20cell%20renal%20cell%20cancer&amp;author=F.%20Seyednasrollah&amp;author=K.%20Rantanen&amp;author=P.%20Jaakkola&amp;author=LL.%20Elo&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;issue=1&amp;pages=e1&amp;publication_year=2016&amp;doi=10.1093%2Fnar%2Fgkv806"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR212" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bi Y, Davuluri RV. NPEBseq: nonparametric empirical bayesian-based procedure for differential expression analysis of RNA-seq data. BMC Bioinformatics. 2013;14:262.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23981227"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3765716"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2105-14-262"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=NPEBseq%3A%20nonparametric%20empirical%20bayesian-based%20procedure%20for%20differential%20expression%20analysis%20of%20RNA-seq%20data&amp;author=Y.%20Bi&amp;author=RV.%20Davuluri&amp;journal=BMC%20Bioinformatics&amp;volume=14&amp;pages=262&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR213" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nueda MJ, Tarazona S, Conesa A. Next maSigPro: updating maSigPro bioconductor package for RNA-seq time series. Bioinformatics. 2014;30:2598–602.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24894503"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4155246"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btu333"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Next%20maSigPro%3A%20updating%20maSigPro%20bioconductor%20package%20for%20RNA-seq%20time%20series&amp;author=MJ.%20Nueda&amp;author=S.%20Tarazona&amp;author=A.%20Conesa&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2598-602&amp;publication_year=2014"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR214" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">GTEx Consortium. The Genotype-Tissue expression (GTEx) project. Nat Genet. 2013;45:580–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/ng.2653"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20Genotype-Tissue%20expression%20%28GTEx%29%20project&amp;journal=Nat%20Genet&amp;volume=45&amp;pages=580-5&amp;publication_year=2013"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conesaetal" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conesaetal">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Conesaetal"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;Conesa et al.&nbsp;2016</div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-page-layout__side u-text-sm u-margin-top-md-at-lt-md">
                    <aside>
                        <div class="c-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="//genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-016-0881-8" data-track="click" data-track-category="PDF Download"
            data-track-action="Click download article PDF"
            data-track-label="10.1186/s13059-016-0881-8">Download PDF</a>
        </p>
    
                            
                            
    <button type="button"
       class="c-button c-button--expandable"
       data-toggle="collapse"
       data-target="export-citations"
       data-test="export-citations-button"
       data-sticky-update="true"
       aria-controls="export-citations"
       aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/s13059-016-0881-8"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s13059-016-0881-8?format=refman&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/s13059-016-0881-8"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s13059-016-0881-8?format=refman&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/s13059-016-0881-8"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s13059-016-0881-8?format=refman&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/s13059-016-0881-8"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s13059-016-0881-8?format=endnote&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/s13059-016-0881-8"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s13059-016-0881-8?format=endnote&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/s13059-016-0881-8"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s13059-016-0881-8?format=endnote&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/s13059-016-0881-8"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s13059-016-0881-8?format=bibtex&flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/s13059-016-0881-8"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s13059-016-0881-8?format=bibtex&flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/s13059-016-0881-8"
                       href="http://citation-needed.springer.com/v2/references/10.1186/s13059-016-0881-8?format=bibtex&flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
        <div id="articleCollections" class="c-separator">
            <h3 class="p-section-title">In these collections</h3>
            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--lg">
                
                    <li class="c-list-group__item">
                        <a href="https://www.biomedcentral.com/collections/reviewsfromgenomebiology"
                           title="Reviews"
                           data-track="click"
                           data-track-category="Article Page|Article Page"
                           data-track-action="Click collection link|Click collection link"
                           data-track-label="reviewsfromgenomebiology|10.1186/s13059-016-0881-8">Reviews</a>
                    </li>
                
            </ul>
        </div>
    
 
                        <div class="c-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/s13059-016-0881-8 #accessCount-container"></div>
                                <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/s13059-016-0881-8 #count-container"></div>
                                <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/s13059-016-0881-8 #altmetric-container"></div>
                            </ul>
                        </div>
                        
    <div class="c-separator">
        <div class="js-shown">
            <h3 class="p-section-title">Share this article</h3>
            <ul data-test="social-media-share-buttons" class="c-social-links">
                <li class="c-social-links__item">
                    <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click"
                            data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s13059-016-0881-8">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page"
                              data-track-action="Share Twitter" data-track-label="10.1186/s13059-016-0881-8">Share on Twitter</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true"
                             data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/s13059-016-0881-8">
                            <use xlink:href="#icon-twitter"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click"
                            data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s13059-016-0881-8">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page"
                              data-track-action="Share Facebook" data-track-label="10.1186/s13059-016-0881-8">Share on Facebook</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true"
                             data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/s13059-016-0881-8">
                            <use xlink:href="#icon-facebook"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click"
                            data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s13059-016-0881-8">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page"
                              data-track-action="Share LinkedIn" data-track-label="10.1186/s13059-016-0881-8">Share on LinkedIn</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true"
                             data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/s13059-016-0881-8">
                            <use xlink:href="#icon-linkedin"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click"
                            data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s13059-016-0881-8">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page"
                              data-track-action="Share Weibo" data-track-label="10.1186/s13059-016-0881-8">Share on Weibo</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true"
                             data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/s13059-016-0881-8">
                            <use xlink:href="#icon-weibo"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click"
                            data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s13059-016-0881-8">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page"
                              data-track-action="Share Reddit" data-track-label="10.1186/s13059-016-0881-8">Share on Reddit</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true"
                             data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/s13059-016-0881-8">
                            <use xlink:href="#icon-reddit"></use>
                        </svg>
                    </button>
                </li>
            </ul>
        </div>
        
            <div data-sharedit="container">
                <a class="c-button u-margin-top-sm"
                   href="/articles/10.1186/s13059-016-0881-8/sharedit"
                   rel="nofollow"
                   data-sharedit="button"
                   data-track="click"
                   data-track-category="sharedit"
                   data-track-action="click article sharedit button"
                   data-track-label="10.1186/s13059-016-0881-8">Get shareable link</a>
            </div>
        
    </div>
 
                        
    
        <div class="c-separator u-js-shown">
            <h3 class="p-section-title">See updates</h3>
            <button class="c-crossmark-button" data-target="crossmark" data-test="crossmark" data-track="click" data-track-category="Article Page" data-track-action="Click CrossMark" data-track-label="10.1186/s13059-016-0881-8">
                <img id="crossmark-icon" src="https://crossmark-cdn.crossref.org/widget/v2.0/logos/CROSSMARK_Color_horizontal.svg" alt="Crossmark logo" />
            </button>
        </div>
    
 
                        <div class="c-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/s13059-016-0881-8" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2Fs13059-016-0881-8&volume=17&issue=1&title=A+survey+of+best+practices+for+RNA-seq+data+analysis&author_name=Ana+Conesa&start_page=1&end_page=19">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">Genome Biology</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 1474-760X</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="mailto:editorial@genomebiology.com">editorial@genomebiology.com</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s13059-016-0881-8' alt=""/>

         
    <footer>
        
            <div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        
        <div class="c-flex c-flex--wrap c-flex--space-between" data-test="publisher-footer-menu">
            <div class="c-flex">
                
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/contact-us">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
                    
                
            </div>
            <div class="u-margin-bottom-lg">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="c-flex u-reset-list" data-test="footer-social-links">
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://twitter.com/biomedcentral"
                               class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-twitter-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://www.facebook.com/BioMedCentral"
                               class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-facebook-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="http://www.weibo.com/biomedcentral"
                               class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-weibo-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="//www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        
        <div class="c-sn-footer">
    <div class="u-container">
        <img src=/static/images/bmc/logos/logo-springernature-8bd4e8d82f.svg class="c-sn-footer__logo" alt="Springer Nature" itemprop="logo">
        <p data-test="copyright"> &#169; 2019 BioMed Central Ltd unless otherwise stated. Part of
            <a href="https://www.springernature.com" itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

        
    </footer>

    </div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

        
<script>
    (function() {
        if (window.config && window.config.mustardcut) {
            var appScript = document.createElement('script');
            
            appScript.src = '/static/js/app-bundle-daf7e91d7a.js';
            
            appScript.async = false;
            document.body.appendChild(appScript);
        }
    })();
</script>

    </body>
</html>

